{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "QAsystem.ipynb",
      "provenance": [],
      "collapsed_sections": [],
      "include_colab_link": true
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "name": "python"
    },
    "accelerator": "GPU"
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/sam505/Machine_Learning/blob/master/QAsystem.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "nlqZlGix8Qx5",
        "outputId": "934479e9-41c6-4132-c529-ed497a08839e"
      },
      "source": [
        "!pip install transformers"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: transformers in /usr/local/lib/python3.7/dist-packages (4.11.3)\n",
            "Requirement already satisfied: sacremoses in /usr/local/lib/python3.7/dist-packages (from transformers) (0.0.46)\n",
            "Requirement already satisfied: huggingface-hub>=0.0.17 in /usr/local/lib/python3.7/dist-packages (from transformers) (0.0.19)\n",
            "Requirement already satisfied: filelock in /usr/local/lib/python3.7/dist-packages (from transformers) (3.3.0)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.7/dist-packages (from transformers) (2.23.0)\n",
            "Requirement already satisfied: tokenizers<0.11,>=0.10.1 in /usr/local/lib/python3.7/dist-packages (from transformers) (0.10.3)\n",
            "Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.7/dist-packages (from transformers) (2019.12.20)\n",
            "Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.7/dist-packages (from transformers) (4.62.3)\n",
            "Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.7/dist-packages (from transformers) (6.0)\n",
            "Requirement already satisfied: importlib-metadata in /usr/local/lib/python3.7/dist-packages (from transformers) (4.8.1)\n",
            "Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.7/dist-packages (from transformers) (1.19.5)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.7/dist-packages (from transformers) (21.0)\n",
            "Requirement already satisfied: typing-extensions in /usr/local/lib/python3.7/dist-packages (from huggingface-hub>=0.0.17->transformers) (3.7.4.3)\n",
            "Requirement already satisfied: pyparsing>=2.0.2 in /usr/local/lib/python3.7/dist-packages (from packaging>=20.0->transformers) (2.4.7)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.7/dist-packages (from importlib-metadata->transformers) (3.6.0)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests->transformers) (2.10)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests->transformers) (3.0.4)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/dist-packages (from requests->transformers) (1.24.3)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests->transformers) (2021.5.30)\n",
            "Requirement already satisfied: joblib in /usr/local/lib/python3.7/dist-packages (from sacremoses->transformers) (1.0.1)\n",
            "Requirement already satisfied: click in /usr/local/lib/python3.7/dist-packages (from sacremoses->transformers) (7.1.2)\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.7/dist-packages (from sacremoses->transformers) (1.15.0)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "RrGcKdQh7PIF",
        "outputId": "82f6fc23-21b8-42b5-d2cc-15670d74f0d5"
      },
      "source": [
        "# import the necessary libraries\n",
        "import nltk\n",
        "import string\n",
        "import re\n",
        "import pandas as pd\n",
        "import json\n",
        "import inflect\n",
        "import torch\n",
        "from google.colab import files\n",
        "from nltk.corpus import stopwords\n",
        "from nltk.tokenize import word_tokenize\n",
        "from nltk.stem import WordNetLemmatizer\n",
        "from nltk.tokenize import word_tokenize\n",
        "from nltk.stem.porter import PorterStemmer\n",
        "from nltk.tokenize import sent_tokenize\n",
        "from torch.utils.data import DataLoader\n",
        "from sklearn.model_selection import train_test_split\n",
        "from transformers import pipeline\n",
        "from termcolor import  colored\n",
        "from tqdm import tqdm\n",
        "import logging\n",
        "from transformers import DistilBertForQuestionAnswering, BertTokenizerFast, DistilBertTokenizer, BertForQuestionAnswering, AutoTokenizer, AdamW, DistilBertTokenizerFast\n",
        "\n",
        "nltk.download('stopwords')\n",
        "nltk.download('averaged_perceptron_tagger')"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[nltk_data] Downloading package stopwords to /root/nltk_data...\n",
            "[nltk_data]   Package stopwords is already up-to-date!\n",
            "[nltk_data] Downloading package averaged_perceptron_tagger to\n",
            "[nltk_data]     /root/nltk_data...\n",
            "[nltk_data]   Package averaged_perceptron_tagger is already up-to-\n",
            "[nltk_data]       date!\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "True"
            ]
          },
          "metadata": {},
          "execution_count": 2
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "8DHzUcxy1eiR"
      },
      "source": [
        "**Preparing Data**"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "aKNXwbXd1k4_"
      },
      "source": [
        "retrieve data in a dataframe"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "paaYmPnPykQS"
      },
      "source": [
        "def json_to_df(json_file):\n",
        "    arrayForDF = []\n",
        "    for current_sub in json_file['data']:\n",
        "        for current_cntxt in current_sub['paragraphs']:\n",
        "            cntxt = current_cntxt['context']\n",
        "            for current_ques in current_cntxt['qas']:\n",
        "                ques = current_ques['question']\n",
        "                if(len(ques)>2):\n",
        "                    is_impossible = current_ques['is_impossible']\n",
        "                    if(is_impossible == False):\n",
        "                        for ans in current_ques['answers']:\n",
        "                            answer_text = ans['text']\n",
        "                            answer_start = ans['answer_start']\n",
        "                            record = {\n",
        "                                \"answer\": ans,\n",
        "                                \"answer_text\" : answer_text,\n",
        "                                \"answer_start\": answer_start,\n",
        "                                \"question\" : ques,\n",
        "                                \"context\" : cntxt,\n",
        "                            }\n",
        "                            arrayForDF.append(record)\n",
        "    df = pd.DataFrame(arrayForDF)\n",
        "    return df"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "rVUFVLwd6sJh",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 560
        },
        "outputId": "ee3946a5-3076-4504-e4e1-73eaa0fbbaed"
      },
      "source": [
        "with open(r'/content/data.json') as f:  \n",
        "  data = json.load(f)  \n",
        "\n",
        "print(data)\n",
        "json_to_df(data).head(10)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{'data': [{'paragraphs': [{'qas': [{'question': 'What is the main cause of HIV-1 infection in children?', 'id': 262, 'answers': [{'text': 'Mother-to-child transmission (MTCT) is the main cause of HIV-1 infection in children worldwide. ', 'answer_start': 370}], 'is_impossible': False}, {'question': 'What plays the crucial role in the Mother to Child Transmission of HIV-1 and what increases the risk', 'id': 276, 'answers': [{'text': 'DC-SIGNR plays a crucial role in MTCT of HIV-1 and that impaired placental DC-SIGNR expression increases risk of transmission.', 'answer_start': 2003}], 'is_impossible': False}, {'question': 'How many children were infected by HIV-1 in 2008-2009, worldwide?', 'id': 278, 'answers': [{'text': 'more than 400,000 children were infected worldwide, mostly through MTCT and 90% of them lived in sub-Saharan Africa. ', 'answer_start': 2291}], 'is_impossible': False}, {'question': 'What is the role of C-C Motif Chemokine Ligand 3 Like 1 (CCL3L1) in mother to child transmission of HIV-1?', 'id': 316, 'answers': [{'text': 'High copy numbers of CCL3L1, a potent HIV-1 suppressive ligand for CCR5, are associated with higher chemokine production and lower risk of MTCT of HIV-1 among South African infants', 'answer_start': 28143}], 'is_impossible': False}, {'question': 'What is DC-GENR and where is  it expressed?', 'id': 305, 'answers': [{'text': 'Dendritic cell-specific ICAM-grabbing non-integrin-related (DC-SIGNR, also known as CD209L or liver/lymph node-specific ICAM-grabbing non-integrin (L-SIGN)) can interact with a plethora of pathogens including HIV-1 and is expressed in placental capillary endothelial cells', 'answer_start': 3207}], 'is_impossible': False}, {'question': 'How does the presence of DC-SIGNR affect the MTCT of HIV-1?', 'id': 306, 'answers': [{'text': 'the presence of DC-SIGNR at the placental endothelial cell surface may protect infants from HIV-1 infection by capturing virus and promoting its degradation/presentation. ', 'answer_start': 28910}], 'is_impossible': False}, {'question': 'Why do low levels of DC-SIGNR enhance Mother to Child Transmission of HIV-1?', 'id': 307, 'answers': [{'text': 'in placenta containing low levels of DC-SIGNR, HIV-1 would preferentially binds CCR5 on endothelial cells resulting in a loss of placental barrier integrity and enhanced passage of maternal HIV-1-infected cells in foetal circulation leading to MTCT of HIV-1', 'answer_start': 29090}], 'is_impossible': False}, {'question': 'What is the percentage of Mother to Child Transmission of HIV-1, when there is no intervention?', 'id': 277, 'answers': [{'text': 'Without specific interventions, the rate of HIV-1 mother-tochild transmission (MTCT) is approximately 15-45%', 'answer_start': 2137}], 'is_impossible': False}, {'question': 'Does C-C chemokine receptor type 5 (CCR5) affect the transmission of HIV-1?', 'id': 312, 'answers': [{'text': 'Genetic variants in CCR5 have been shown to influence vertical transmission of HIV-1. CCR5 promoter variants resulting in higher expression of the receptor were associated with increased risk of MTCT of HIV-1 among sub-Saharan Africans', 'answer_start': 27719}], 'is_impossible': False}, {'question': 'How does Mannanose Binding Lectin (MBL) affect elimination of HIV-1 pathogen?', 'id': 318, 'answers': [{'text': 'Mannose-binding lectin (MBL) is an innate immune receptor synthesised in the liver and secreted in the bloodstream in response to inflammation signal. MBL promotes pathogen elimination by opsonization and phagocytosis,', 'answer_start': 28335}], 'is_impossible': False}, {'question': \"How can CCR5's effect in HIV-1 transmission be reduced?\", 'id': 321, 'answers': [{'text': 'The 32-pb deletion polymorphism in CCR5 has be shown to protect from vertical transmission of HIV-1', 'answer_start': 27966}], 'is_impossible': False}], 'context': \"Functional Genetic Variants in DC-SIGNR Are Associated with Mother-to-Child Transmission of HIV-1\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752805/\\n\\nBoily-Larouche, Geneviève; Iscache, Anne-Laure; Zijenah, Lynn S.; Humphrey, Jean H.; Mouland, Andrew J.; Ward, Brian J.; Roger, Michel\\n2009-10-07\\nDOI:10.1371/journal.pone.0007211\\nLicense:cc-by\\n\\nAbstract: BACKGROUND: Mother-to-child transmission (MTCT) is the main cause of HIV-1 infection in children worldwide. Given that the C-type lectin receptor, dendritic cell-specific ICAM-grabbing non-integrin-related (DC-SIGNR, also known as CD209L or liver/lymph node–specific ICAM-grabbing non-integrin (L-SIGN)), can interact with pathogens including HIV-1 and is expressed at the maternal-fetal interface, we hypothesized that it could influence MTCT of HIV-1. METHODS AND FINDINGS: To investigate the potential role of DC-SIGNR in MTCT of HIV-1, we carried out a genetic association study of DC-SIGNR in a well-characterized cohort of 197 HIV-infected mothers and their infants recruited in Harare, Zimbabwe. Infants harbouring two copies of DC-SIGNR H1 and/or H3 haplotypes (H1-H1, H1-H3, H3-H3) had a 3.6-fold increased risk of in utero (IU) (P = 0.013) HIV-1 infection and a 5.7-fold increased risk of intrapartum (IP) (P = 0.025) HIV-1 infection after adjusting for a number of maternal factors. The implicated H1 and H3 haplotypes share two single nucleotide polymorphisms (SNPs) in promoter region (p-198A) and intron 2 (int2-180A) that were associated with increased risk of both IU (P = 0.045 and P = 0.003, respectively) and IP (P = 0.025, for int2-180A) HIV-1 infection. The promoter variant reduced transcriptional activity in vitro. In homozygous H1 infants bearing both the p-198A and int2-180A mutations, we observed a 4-fold decrease in the level of placental DC-SIGNR transcripts, disproportionately affecting the expression of membrane-bound isoforms compared to infant noncarriers (P = 0.011). CONCLUSION: These results suggest that DC-SIGNR plays a crucial role in MTCT of HIV-1 and that impaired placental DC-SIGNR expression increases risk of transmission.\\n\\nText: Without specific interventions, the rate of HIV-1 mother-tochild transmission (MTCT) is approximately 15-45% [1] . UNAIDS estimates that last year alone, more than 400,000 children were infected worldwide, mostly through MTCT and 90% of them lived in sub-Saharan Africa. In the most heavilyaffected countries, such as Zimbabwe, HIV-1 is responsible for one third of all deaths among children under the age of five. MTCT of HIV-1 can occur during pregnancy (in utero, IU), delivery (intrapartum, IP) or breastfeeding (postpartum, PP). High maternal viral load, low CD4 cells count, vaginal delivery, low gestational age have all been identified as independent factors associated with MTCT of HIV-1 [1] . Although antiretrovirals can reduce MTCT to 2%, limited access to timely diagnostics and drugs in many developing world countries limits the potential impact of this strategy. A better understanding of the mechanisms acting at the maternal-fetal interface is crucial for the design of alternative interventions to antiretroviral therapy for transmission prevention.\\n\\nDendritic cell-specific ICAM-grabbing non-integrin-related (DC-SIGNR, also known as CD209L or liver/lymph node-specific ICAM-grabbing non-integrin (L-SIGN)) can interact with a plethora of pathogens including HIV-1 and is expressed in placental capillary endothelial cells [2] . DC-SIGNR is organized in three distinct domains, an N-terminal cytoplasmic tail, a repeat region containing seven repeat of 23 amino acids and a C-terminal domain implicated in pathogen binding. Alternative splicing of DC-SIGNR gene leads to the production of a highly diversify isoforms repertoire which includes membrane-bound and soluble isoforms [3] . It has been proposed that interaction between DC-SIGNR and HIV-1 might enhance viral transfer to other susceptible cell types [2] but DC-SIGNR can also internalize and mediate proteasome-dependant degradation of viruses [4] that may differently affect the outcome of infection.\\n\\nGiven the presence of DC-SIGNR at the maternal-fetal interface and its interaction with HIV-1, we hypothesized that it could influence MTCT of HIV-1. To investigate the potential role of DC-SIGNR in MTCT of HIV-1, we carried out a genetic association study of DC-SIGNR in a well-characterized cohort of HIV-infected mothers and their infants recruited in Zimbabwe, and identified specific DC-SIGNR variants associated with increased risks of HIV transmission. We further characterized the functional impact of these genetic variants on DC-SIGNR expression and show that they affect both the level and type of DC-SIGNR transcripts produced in the placenta.\\n\\nSamples consisted of stored DNA extracts obtained from 197 mother-child pairs co-enrolled immediately postpartum in the ZVITAMBO Vitamin A supplementation trial (Harare, Zimbabwe) and followed at 6 weeks, and 3-monthly intervals up to 24 months. The ZVITAMBO project was a randomized placebocontrolled clinical trial that enrolled 14,110 mother-child pairs, between November 1997 and January 2000, with the main objective of investigating the impact of immediate postpartum vitamin A supplementation on MTCT of HIV-1. The samples used in the present study were from mother-child pairs randomly assigned to the placebo group of the ZVITAMBO project. Antiretroviral prophylaxis for HIV-1-positive antenatal women was not available in the Harare public-sector during ZVITAMBO patient recruitment. The samples were consecutively drawn from two groups: 97 HIV-1-positive mother/HIV-1-positive child pairs and 100 HIV-1-positive mother/HIV-negative child pairs. Mother's serological status was determined by ELISA and confirmed by Western Blot. Infants were considered to be infected if they were HIV-1 seropositive at 18 months or older and had two or more positive HIV-1-DNA polymerase chain reaction (PCR) results at earlier ages. 100 infants were considered to be uninfected as they were ELISA negative at 18 months or older and had two DNA PCR negative results from samples collected at a younger age. Of the 97 HIV-1-infected infants, 57 were infected IU, 11 were infected IP, and 17 were infected PP as determined by PCR analyses of blood samples collected at birth, 6 weeks, 3 and 6 months of age and according to the following definitions adapted from Bryson and colleagues [5] . Briefly, infants who were DNA PCR positive at birth were infected IU. Infants with negative PCR results from sample obtained at birth but who become positive by 6 weeks of age were infected IP. Infants with negative PCR results at birth and 6 weeks of age but who subsequently became DNA PCR positive were considered to be infected during the PP period. In the analysis comparing the 3 different modes of MTCT, 12 HIV-1-infected infants were excluded because the PCR results were not available at 6 weeks of age. Full methods for recruitment, baseline characteristics collection, laboratory procedures have been described elsewhere [6] .\\n\\nThe nucleotide sequence variation of the entire promoter, coding and part of 39-UTR regions of DC-SIGNR gene in the study population was determined previously [7] . Haplotype reconstruction was performed using Bayesian statistical method implemented in PHASE [8] , version 2.1.1, using single nucleotide polymorphism (SNP) with a minimum allele frequency (MAF) of 2%. We applied the algorithm five times, using different randomly generated seeds, and consistent results were obtained across runs ( Figure 1 ). Fifteen haplotype-tagged SNPs (htSNPs) were identified by the HaploBlockFinder software [9] with a MAF $5%. These htSNPs were genotyped in the 197 infants by direct PCR sequencing analysis as we have described previously [7] . The DC-SIGNR exon 4 repeat region genotype was determined by PCR amplification followed by migration in 1.5% agarose gels [10] . DNA sequences in the promoter region were analysed with the TESS interface (http//:www.cbil.upenn.edu/tess) for putative transcription factors binding sites using the TRANSFAC database.\\n\\nLuciferase reporter assays using pGL2-Basic vector were performed in order to investigate the functional effect of mutations on DC-SIGNR promoter activity. Genomic DNA from subjects homozygous for the promoter variants and WT was amplified from nucleotide position 2715 to 21 and cloned between the BglII and HindIII multiple cloning sites in the pGL2-Basic vector which harbours a reporter firefly luciferase gene downstream (Invitrogen Canada inc, Burlington, Canada). All recombinants clones were verified by DNA sequencing. The firefly luciferase test reporter vector was co-transfected at a ratio of 10:1 with the constitutive expressor of Renilla luciferase, phRL-CMV (Promega, Madison, WI, USA). We cultured HeLa cells in 6 wells plates (2610 5 cells) and transfected them the following day using lipofectamine (Invitrogen) according to the manufacturer. Cells were lysed and luciferase assays were performed using 20 mg of protein extract according to the manufacturer (Promega) at 44 h post-transfection. Firefly luciferase activity was normalized to Renilla luciferase activity. 0 mg, 0,5 mg or 1 mg CMV-Tat vector was transfected with LTR-Luc as a positive control in these experiments. We carried out lucierase assays in triplicate in three independent experiments. Results are expressed as mean6 standard error of the mean (S.E.M).\\n\\nFirst-term placental tissues were obtained from abortions following voluntary interruption of pregnancy at CHUM Hôpital Saint-Luc (Montreal, Canada). Tissues from 3 H1 (associated with MTCT of HIV-1) and 3 H15 (wild-type) homozygous haplotypes were used to analyse possible differences in isoform expression. Total placental RNAs were extracted by MasterPure DNA and RNA Extraction Kit (Epicentre Biotechnologies, Madison, WI, USA) according to the manufacturer. Fragments corresponding to the DC-SIGNR coding region were reversed transcribed (RT) and then amplified by nested PCR with the following primers; RT primers RR, first PCR RF and RR and second PCR RcF and RcR according to Liu and colleagues [11] . 1 mg of total RNA was reverse transcribed with Expand RT (Roche Applied Science, Indianapolis, IN, USA) according to the manufacturer and were PCR-amplified with DNA Platinum Taq Polymerase (Invitrogen). Major PCR products from the second PCR reaction were gel extracted with the Qiagen Gel Extraction Kit (Qiagen Canada inc, Mississauga, ON, Canada) and cloned using the TOPO TA Cloning Kit for sequencing (Invitrogen). For each placenta, 15 different clones were randomly selected and amplified with M13 primers and sequenced with ABI PRISM 3100 capillary automated sequencer (Applied Biosystems, Foster City, CA, USA). Sequences were analysed and aligned with GeneBank reference sequence NM_014257 using Lasergene software (DNA Stars, Madison, WI, USA).\\n\\nQuantitative expression of DC-SIGNR isoforms 1,5 mg of placental RNA was reverse transcribed using 2.5 mM of Oligo dT 20 and Expand RT in 20 ml volume according to the manufacturer (Roche Applied Science). 15 ng of total cDNA in a final volume of 20 ml was used to perform quantitative real-time PCR using Universal Express SYBR GreenER qPCR Supermix (Invitrogen) on a Rotor Gene Realtime Rotary Analyser (Corbett Life Science, Sydney, Australia). Samples from 2 subjects in each group were used because RNA quality of others was not suitable for a qRT-PCR analysis. Amplification of all DC-SIGNR isoforms was performed using an exon 5 specific primer pair (Table S1 ). Membrane-bound isoforms were amplified using primers specific for exon 3, corresponding to the common trans-membrane domain of DC-SIGNR. Primers were targeted to the exon-exon junction and RNA extracts were treated with DNase (Fermantas International inc, Burlington, ON, Canada) to avoid amplification of contaminant DNA. Standard curves (50-500 000 copies per reaction) were generated using serial dilution of a full-length DC-SIGNR or commercial GAPDH (Invitrogen) plasmid DNA. All qPCR reactions had efficiencies ranging from 99% to 100%, even in the presence of 20 ng of non-specific nucleic acids, and therefore could be compared. The copy number of unknown samples was estimated by placing the measured PCR cycle number (crossing threshold) on the standard curve. To correct for differences in both RNA quality and quantity between samples, the expression levels of transcripts were normalised to the reference GAPDH gene transcripts. GAPDH primer sequences were kindly provided by A. Mes-Masson at the CHUM. The results are presented as target gene copy number per 10 5 copies of GAPDH. The ratio of membrane-bound isoforms was calculated as E3/E5. Soluble isoforms were calculated by subtracting membrane-bound from total isoforms. We carried out qPCR assays in triplicate in three independent experiments. Results are expressed as mean6S.E.M.\\n\\nStatistical analysis was performed using the GraphPad PRISM 5.0 for Windows (GraphPad Software inc, San Diego, CA, USA). Differences in baseline characteristics and genotypic frequencies of haplotypes or htSNPs were compared between groups using the x 2 analysis or Fisher's exact test. Logistic regression analysis was used to estimate odds ratios (OR) for each genotype and baseline risk factors. Multiple logistic regression was used to define independent predictors identified as significant in the crude analysis. ORs and 95% confidence interval were calculated with the exact method. Comparisons of continuous variables between groups were assessed with the unpaired two-tailed Student's t test when variables were normally distributed and with the Mann-Whitney U test when otherwise. Differences were considered significant at P,0.05.\\n\\nWritten informed consent was obtained from all mothers who participated in the study and the ZVITAMBO trial and the investigation reported in this paper were approved by The \\n\\nWe carried out an association study of DC-SIGNR polymorphism in 197 infants born to untreated HIV-1-infected mothers recruited in Harare, Zimbabwe. Among them, 97 infants were HIV-1-infected and 100 infants remained uninfected. Of the 97 HIV-1-infected infants, 57 were infected IU, 11 were infected IP, and 17 were infected PP. Timing of infection was not determined for 12 HIV-1-infected infants. Baseline characteristics of mothers and infants are presented in Table 1 . Maternal age and CD4 cell count, child sex, mode of delivery, duration of membrane rupture and gestational age were similar among all groups. However, maternal viral load .29 000 copies/ml was associated with increased risk in both IU and PP with odds ratios (OR) of 3.64 (95% CI = 1.82-7.31, P = 0.0002) and 4.45 (95% CI = 1.50-13.2, P = 0.0045) for HIV-1 transmission, respectively.\\n\\nFifteen haplotype-tagged SNPs (htSNPs) corresponding to the 15 major DC-SIGNR haplotypes ( Figure 1 ) described among Zimbabweans [7] were genotyped in our study samples (Tables S2  and S3 ). H1 (31%) and H3 (11%) were the most frequent haplotypes observed (Figure 1 ). Being homozygous for the H1 haplotype was associated with increased risk of both IU (OR: 4.42, P = 0.022) and PP (OR: 7.31, P = 0.016) HIV-1 transmission ( Table 2) . Infants harbouring two copy combinations of H1 and/ or H3 haplotypes (H1-H1, H1-H3 or H3-H3) had increased risk of IU (OR: 3.42, P = 0.007) and IP (OR: 5.71, P = 0.025) but not PP (P = 0.098) HIV-1 infection compared to infant noncarriers ( Table 2 ). The latter associations remained significant after adjustment was made for the maternal viral load for both IU (OR: 3.57, 95% CI = 1.30-9.82, P = 0.013) and IP (OR: 5.71, 95% CI = 1.40-23.3, P = 0.025) HIV-1 transmission. The H1 and H3 haplotypes share a cluster of mutations (p-198A, int2-391C, int2-180A, ex4RPT, int5+7C) ( Figure 1 ). Of these, the p-198A and int2-180A variants were significantly associated with MTCT of HIV-1 (Table S2 ). In the unadjusted regression analysis, homozygous infants for the p-198A and int2-180A variants had increased risk of IU (OR: 2.07 P = 0.045, OR: 3.78, P = 0.003, respectively) and IP (OR: 2.47, P = 0.17, O.R: 5.71, P = 0.025, respectively) HIV-1 infection compared to heterozygote infants or noncarriers (Table 3) . When adjustment was made for maternal factors, only the association with the int2-180A variant remained significant for IU (OR: 3.83, 95% CI = 1.42-10.4, P = 0.008) and IP (O.R: 5.71, 95% CI = 1.40-23.3, P = 0.025) HIV-1 transmission. Thus, infants homozygous for DC-SIGNR variant int2-180A contained in H1 and H3 haplotypes were 4-fold to 6-fold more likely to be infected by HIV-1 during pregnancy or at delivery, respectively.\\n\\nAlternative splicing of the DC-SIGNR gene in the placenta produces both membrane-bound and soluble isoform repertoires [3] . The relative proportion of membrane bound and soluble DC-SIGNR could plausibly influence the susceptibility to HIV-1 infection [11] . We therefore hypothesized that the DC-SIGNR mutations associated with MTCT of HIV-1 would have an impact on both the level of DC-SIGNR expression and in the isoform repertoire produced. We investigated DC-SIGNR transcript expression in first-term placentas obtained after elective abortion.\\n\\nWe cloned DC-SIGNR from placental tissues by RT-PCR from 3 homozygous H1 samples containing both the DC-SIGNR p-198AA and int2-180AA variants associated with HIV-1 transmission and 3 homozygous wild-type (WT) (p-198CC, int2-180GG) samples. Fifteen clones per sample were randomly selected for sequencing. As expected, we found an extensive repertoire of DC-SIGNR transcripts in all samples with 9 to 16 different isoforms per individual. A total of 65 distinct transcripts were identified ( Figure S1 ), of which 3 were full-length transcripts. 64 of the sequenced clones contained a total of 69 amino acid substitutions with 3 new C termini and 2 premature stop codons. However, the diversity was mostly attributable to the entire deletion of exon 2 or exon 3 or to variations in the length of the neck region (exon 4) of DC-SIGNR. The deletion of exon 3 eliminates the trans-membrane domain of the protein and leads to the expression of soluble DC-SIGNR isoforms [3] . Interestingly, the abundance of membrane-bound isoforms in placental tissues of the H1 homozygotes appears to be lower than that observed in samples from WT individuals ( Figure S1 ). The deletion of exon 3 was confirmed by sequencing and we hypothesize that the skipping of exon 3, could be due to the presence of the int2-180A mutation observed in infants with the H1 haplotype. In fact, this intron mutation is located 180 bp downstream from exon 3 and potentially modifies splicing events (Figure 2A ). We confirmed that the variation in transcript proportions seen between the two groups was also reflected at the level of mRNA expression in the placenta. To quantify membrane-bound vs soluble isoforms in placental samples from homozygous H1 and WT infants, we amplified the exon 5 (E5) sequence present in all DC-SIGNR isoforms (total transcripts). We then amplified exon 3 (E3) which is deleted in the soluble forms and then calculated the E3:E5 ratio. We found that placental tissues from homozygous H1 infants express a significantly lower proportion of membrane-bound DC-SIGNR (18%) compared to that in WT individuals (36%) (P = 0.004) ( Figure 2B ) suggesting that exon 3 skipping happens more frequently in presence of the DC-SIGNR int2-180A variant associated with MTCT of HIV-1.\\n\\nThe DC-SIGNR int2-180A variant is always transmitted with the promoter mutation p-198A (Figure 1 ). In the unadjusted regression analysis, the p-198A variant was significantly associated with IU but not with IP and PP HIV-1 transmission (Table 3) . Computational transcription factor binding site analysis predicts Table 1 . Baseline characteristics of mother and infants risk factors for intrauterine (IU), intrapartum (IP) and postpartum (PP) mother-to-child HIV-1 transmission. Figure 3A ). The luciferase activity of the p-198A variant construct was significantly lower than that of the WT p-198C promoter construct (p-198C/A ratio = 2, P = 0.006) ( Figure 3B ) suggesting that DC-SIGNR p-198A affects promoter activity. The other promoter mutants (p-577C and p-323A) observed in the Zimbabwean population did not affect DC-SIGNR transcription in this assay ( Figure S2 ). To determine the net impact of the DC-SIGNR p-198A mutation on DC-SIGNR expression in the placenta, we quantitated the absolute number of total and membrane-bound DC-SIGNR transcripts in the H1 homozygote and wild-type placental samples as described earlier. The total number of DC-SIGNR transcripts was determined to be 6856213 (DC-SIGNR copies6S.E.M per 10 5 GAPDH copies) in the placental samples from homozygous H1 infants and was 4-fold lower compared to that found in placentas from WT individuals (27816638, P = 0.011) ( Figure 3C ). As suggested earlier, the int2-180A mutation might induce exon 3 skipping leading to a lower production of membrane-bound DC-SIGNR. Although, the decrease in the total number of DC-SIGNR transcripts in H1 homozygous placental samples containing both the p-198AA and int2-180AA variants affected the proportion of membrane-bound and soluble isoforms, the effect of these mutations was more pronounced on the membrane-bound isoforms with an 8-fold decrease (H1 = 117636.2 vs WT = 9906220.6, P = 0.003) compared to a 3-fold decrease in total soluble isoforms (H1 = 5686181.9 vs WT = 19256495.3, P = 0.03) ( Figure 3C ). Therefore, DC-SIGNR p-198A and int2-180A mutations associated with MTCT of HIV-1 significantly decreased the level of total placental DC-SIGNR transcripts, disproportionately affecting the membrane-bound isoform production. Table 3 . Associations between infant DC-SIGNR promoter p-198 and intron 2 (int2)-180 variants and intrauterine (IU), intrapartum (IP) and postpartum (PP) mother-to-child HIV-1 transmission. \\n\\nOur genetic results, supported by expression assay in placenta, suggest the involvement of DC-SIGNR in MTCT of HIV-1. Homozygosity for the haplotype H1 was associated with IU transmission in the unadjusted regression analysis. However, the association disappeared after adjustment was made for the maternal factors presumably because of the small number of H1 homozygote infants analysed in each groups. H1 and H3 were the most frequent haplotypes observed in the study population and they share a cluster of mutations (Figure 1 ). Grouping haplotypes H1 and H3 increased the power of the study and permitted the identification of specific DC-SIGNR mutations associated with MTCT of HIV-1. Indeed, two mutations shared by haplotypes H1 and H3 were associated with vertical transmission of HIV-1. The int2-180A was associated with a 4-fold increased risk of IU and 6fold increased risk of IP after adjustment for the maternal factors. Although the p-198A variant was associated with IU transmission, the association disappeared after adjustment was made for the maternal viral load. Nevertheless, we showed that this mutation reduces DC-SIGNR transcriptional activity in vitro and produces lower level of DC-SIGNR transcripts in placental tissues in combination with the int2-180A variant. Since int2-180A is always transmitted with p-198A on the MTCT associated combined haplotypes H1/H3, whereas p-198A is carried on other nonassociated haplotypes (Figure 1) , we can speculate that the p-198A mutation alone may have a minor effect in vivo whereas in combination with the int2-180A variant, they both act to reduce the level of placental DC-SIGNR expression resulting in an increased risk of MTCT of HIV-1.\\n\\nThe majority of IU transmission occurs during the last trimester of pregnancy (reviewed in [12] ). Full-term placenta samples were not available for the current study and the expression assays were performed on first-term placental tissues. A previous study looking at DC-SIGNR placental isoforms repertoire in full-term placenta samples demonstrated similar diversity of DC-SIGNR transcripts as in the first-term placental tissues studied herein [3] . However, since levels of DC-SIGNR expression have never been compared between the different terms of pregnancy, it is not known whether DC-SIGNR expression varies during the course of pregnancy. Nevertheless, it is reasonable to assume that the inter-individual differences in both DC-SIGNR isoform repertoire and transcript levels observed between the H1 and WT homozygous infants would be reflected throughout the pregnancy. To date, most studies have focused on the potential role of DC-SIGNR in trans infection of HIV-1 in vitro [2, 10] . However, the multiple mechanisms involved in trans infection and redundancy among C-type lectin functions make it difficult to determine the actual participation of DC-SIGNR in this mode of infection in vivo [13, 14] . The strong correlation we observed between MTCT of HIV-1 and DC-SIGNR genetic variants producing low levels of DC-SIGNR in the placenta suggested that mechanisms other than DC-SIGNR-mediated trans infection might operate during vertical transmission of HIV-1. For example, DC-SIGNR has also been shown to function as a HIV-1 antigen-capturing receptor [15] . Chan and colleagues recently demonstrated that DC-SIGNR transfected CHO cells diminish SARS-CoV titers by enhanced capture and degradation of the virus in a proteasome-dependent manner [4] . Since endothelial cells express MHC-I and II, degraded viral antigens could then be presented to immune cells to elicit an adaptive immune response [16, 17] . The HIV-1 coreceptor CCR5, but not CD4, is co-expressed with DC-SIGNR on placental and blood-brain barrier (BBB) endothelial cells [18, 19] . HIV-1 gp120 binding to CCR5 receptor on endothelial cells compromises BBB integrity and enhances monocytes adhesion and transmigration across the BBB [20, 21] . It is thus possible that reduced expression of DC-SIGNR, particularly the membranebound isoforms, on placental capillary endothelial cells might favour HIV-1 binding to CCR5 receptor, instead of DC-SIGNR receptor, facilitating the migration of maternal HIV-1-infected cells across the placental barrier resulting in IU transmission of HIV-1.\\n\\nThe int2-180A variant contained in the H1 and H3 haplotypes was associated with IP transmission suggesting that DC-SIGNR also affect transmission of HIV-1 during delivery. Little is known about the mechanisms underlying transmission of HIV-1 during delivery. Passage through the birth canal could potentially expose infants through a mucosal portal entry (presumably ophthalmic, skin, or gastrointestinal), whereas placental insult during delivery (physical or inflammatory) may enhance transplacental passage of maternal HIV-1-infected cells into foetal circulation [22, 23] . Such process called microtransfusion has been proposed in regards to the results obtain in a Malawian cohort. Kweik and colleagues found a significant association between levels of maternal DNA in umbilical cord blood and IP transmission of HIV-1 suggesting that passage of maternal infected cells through the placenta is likely to occur during delivery [22] . Thus, in a similar fashion as suggested earlier for IU transmission, the relatively lower level of DC-SIGNR in the placenta of homozygous infants harbouring the int2-180A variant could promote HIV-1 binding to CCR5 receptor on endothelial cells affecting the placental barrier integrity and facilitating the passage of maternal infected cells in foetal circulation during delivery.\\n\\nBeside DC-SIGNR, other HIV-1 receptors are known to influence MTCT of HIV-1 (reviewed in [24] ). Genetic variants in CCR5 have been shown to influence vertical transmission of HIV-1. CCR5 promoter variants resulting in higher expression of the receptor were associated with increased risk of MTCT of HIV-1 among sub-Saharan Africans [25, 26] . The 32-pb deletion polymorphism in CCR5 has be shown to protect from vertical transmission of HIV-1 [27] , but this variant is virtually absent among African populations [28] . High copy numbers of CCL3L1, a potent HIV-1 suppressive ligand for CCR5, are associated with higher chemokine production and lower risk of MTCT of HIV-1 among South African infants [29, 30] . Mannose-binding lectin (MBL) is an innate immune receptor synthesised in the liver and secreted in the bloodstream in response to inflammation signal. MBL promotes pathogen elimination by opsonization and phagocytosis, and reduced expression of MBL resulting from polymorphism in coding and non-coding regions has been associated with an increased risk of MTCT of HIV-1 [31, 32] .\\n\\nIn this study, we demonstrate for the first time, the potential functional impact of DC-SIGNR mutations on its expression in the placenta and in vertical transmission of HIV-1. We believe that the presence of DC-SIGNR at the placental endothelial cell surface may protect infants from HIV-1 infection by capturing virus and promoting its degradation/presentation. However, in placenta containing low levels of DC-SIGNR, HIV-1 would preferentially binds CCR5 on endothelial cells resulting in a loss of placental barrier integrity and enhanced passage of maternal HIV-1-infected cells in foetal circulation leading to MTCT of HIV-1. This mechanism may also apply to other vertically-transmitted pathogens known to interact with DC-SIGNR such as HIV-2, hepatitis C and dengue viruses and warrant further investigation. \\n\\nAssociations between child DC-SIGNR exon 4 repeated region genotypes and mother-to-child HIV-1 transmission.CI, Confidence interval; N, number; NA; not applicable; OR, odds ratio a P-value as determined by the Chi-square test. b Comparison between genotype and all others. Found at: doi:10.1371/journal.pone.0007211.s003 (0.05 MB DOC) Figure S1 DC-SIGNR transcripts repertoire in placenta. Major RT-PCR products from RNA extract from 3 homozygous H1 and 3 homozygous WT placenta samples were purified, cloned and sequenced. Sequenced were analysed according to NCBI reference sequence NM_014257. CT; cytoplasmic tail, TM; trans-membrane domain; WT; wild-type Found at: doi:10.1371/journal.pone.0007211.s004 (0.11 MB DOC) Figure S2 Effect of DC-SIGNR promoter variant on transcriptional activity in luciferase reporter assay in vitro in transfected HeLa cells. Relative luciferase expression from pGL2-Basic, parental vector without promoter. Expression DC-SIGNR promoter constructs, spanning p-577C variant or p-323A variant were calculated relatively to this value. Data are presented in mean values6S.E.M of three independent experiments performed in triplicate. One-way ANOVA test followed by the Dunnett test for multiple comparison was used to compare the relative luciferase expression of the p-557C and p-323A variant reporters against the wild-type (WT) construct (not significant). 0 mg, 0,5 mg or 1 mg CMV-Tat vector was transfected with LTR-Luc as a positive control in these experiments.\", 'document_id': 630}]}, {'paragraphs': [{'qas': [{'question': 'What is IFITM?', 'id': 568, 'answers': [{'text': 'interferon-induced transmembrane', 'answer_start': 353}], 'is_impossible': False}, {'question': 'How many cysteine residues are contained in the first transmembrane domain of IFITM3?', 'id': 569, 'answers': [{'text': 'three', 'answer_start': 701}], 'is_impossible': False}, {'question': 'What inhibits S-palmitoylation?', 'id': 570, 'answers': [{'text': '2-bromopalmitic acid (2BP)', 'answer_start': 1509}], 'is_impossible': False}, {'question': 'What interaction is inhibited by the presence of 2-bromopalmitic acid (2BP)?', 'id': 571, 'answers': [{'text': 'IFITM5 with FKBP11', 'answer_start': 1807}], 'is_impossible': False}, {'question': 'What is a function associated with IFITM5?', 'id': 572, 'answers': [{'text': 'bone formation factor.', 'answer_start': 2195}], 'is_impossible': False}, {'question': 'What regulates the antiviral activity of IFITM3?', 'id': 573, 'answers': [{'text': 'S-palmitoylation on the protein', 'answer_start': 3574}], 'is_impossible': False}, {'question': 'What is another name for IFITM5?', 'id': 574, 'answers': [{'text': 'bonerestricted IFITM-like (BRIL) protein', 'answer_start': 6107}], 'is_impossible': False}, {'question': 'Why is the expression of IFITM5 not promoted by interferons?', 'id': 575, 'answers': [{'text': 'the region upstream of the ifitm5 gene lacks the interferon regulatory elements', 'answer_start': 6335}], 'is_impossible': False}, {'question': 'What is the amino acid similarity between IFITM5 and the other IFITM proteins?', 'id': 576, 'answers': [{'text': '~ 65% similarity', 'answer_start': 6693}], 'is_impossible': False}, {'question': 'What is the amino acid similarity between IFITM 1, IFITM 2, and IFITM 3?', 'id': 577, 'answers': [{'text': '~ 85% similarity', 'answer_start': 6742}], 'is_impossible': False}, {'question': 'What amino acid might be involved in calcium binding in the C-terminal region of a protein?', 'id': 580, 'answers': [{'text': 'aspartate', 'answer_start': 6817}], 'is_impossible': False}], 'context': 'Role of S-Palmitoylation on IFITM5 for the Interaction with FKBP11 in Osteoblast Cells\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC3776769/\\n\\nTsukamoto, Takashi; Li, Xianglan; Morita, Hiromi; Minowa, Takashi; Aizawa, Tomoyasu; Hanagata, Nobutaka; Demura, Makoto\\n2013-09-18\\nDOI:10.1371/journal.pone.0075831\\nLicense:cc-by\\n\\nAbstract: Recently, one of the interferon-induced transmembrane (IFITM) family proteins, IFITM3, has become an important target for the activity against influenza A (H1N1) virus infection. In this protein, a post-translational modification by fatty acids covalently attached to cysteine, termed S-palmitoylation, plays a crucial role for the antiviral activity. IFITM3 possesses three cysteine residues for the S-palmitoylation in the first transmembrane (TM1) domain and in the cytoplasmic (CP) loop. Because these cysteines are well conserved in the mammalian IFITM family proteins, the S-palmitoylation on these cysteines is significant for their functions. IFITM5 is another IFITM family protein and interacts with the FK506-binding protein 11 (FKBP11) to form a higher-order complex in osteoblast cells, which induces the expression of immunologically relevant genes. In this study, we investigated the role played by S-palmitoylation of IFITM5 in its interaction with FKBP11 in the cells, because this interaction is a key process for the gene expression. Our investigations using an established reporter, 17-octadecynoic acid (17-ODYA), and an inhibitor for the S-palmitoylation, 2-bromopalmitic acid (2BP), revealed that IFITM5 was S-palmitoylated in addition to IFITM3. Specifically, we found that cysteine residues in the TM1 domain and in the CP loop were S-palmitoylated in IFITM5. Then, we revealed by immunoprecipitation and western blot analyses that the interaction of IFITM5 with FKBP11 was inhibited in the presence of 2BP. The mutant lacking the S-palmitoylation site in the TM1 domain lost the interaction with FKBP11. These results indicate that the S-palmitoylation on IFITM5 promotes the interaction with FKBP11. Finally, we investigated bone nodule formation in osteoblast cells in the presence of 2BP, because IFITM5 was originally identified as a bone formation factor. The experiment resulted in a morphological aberration of the bone nodule. This also indicated that the S-palmitoylation contributes to bone formation.\\n\\nText: The interferon-induced transmembrane (IFITM) protein family (also known as the Fragilis family in mice) is a part of the dispanin family [1] and is composed of double-transmembrane α-helices connected by a cytoplasmic (CP) loop and extracellular (EC) amino-and carboxyl-terminal polypeptide sequences (Figure 1-A) . The IFITM proteins are evolutionarily conserved in vertebrates [2] . Recent genomic research has revealed that there are 5 IFITM members in humans (IFITM1, 2, 3, 5 and 10) and 7 members in mice (IFITM1, 2, 3, 5, 6, 7, and 10). These proteins play roles in diverse biological processes, such as germ cell maturation during gastrulation (IFITM1-3) [3] [4] [5] , cell-to-cell adhesion (IFITM1) [6] [7] [8] , antiviral activity (IFITM1-3) [9] [10] [11] [12] [13] [14] [15] [16] [17] , and bone formation (IFITM5) [18] [19] [20] [21] [22] , although the detailed functions of IFITM6, 7, and 10 are unknown at present. In particular, IFITM3 has been a target of intensive studies on its activity against influenza A (H1N1) virus infection and internalization [9] [10] [11] [12] [13] [14] .\\n\\nIn 2010, Dr. Yount and co-workers reported that the antiviral activity of IFITM3 is dependent on S-palmitoylation on the protein [10] . The S-palmitoylation [23] is a post-translational modification on proteins by C 16 saturated-fatty acids (palmitic acids) covalently attached to certain cysteine residues via a thioester linkage (Figure 1-B) . The modification is reversibly catalyzed by protein acyltransferases and acylprotein thioesterases, and confers unique properties to the protein, such as membrane binding and targeting, immunoreactivity,\\n\\nAmino-acid sequence alignment of IFITM5, IFITM1, IFITM2, and IFITM3 derived from mice. The conserved residues are highlighted in black. The three conserved cysteines are highlighted in red and numbered based on the sequence of IFITM5 (top) and IFITM3 (bottom). The residues unique in IFITM5 are highlighted in gray. The first and the second transmembrane domains, the extracellular sequences, and the cytoplasmic loop are indicated by arrows and denoted as TM1 and TM2, EC, and the CP loop, respectively. The TM domains were predicted by SOSUI. The aspartates at the C-terminal region in IFITM5 are shown in blue. B) The schematic illustration of the protein S-palmitoylation. The C 16 -palmitic acid is attached to cysteine via a thioester linkage. The palmitoylation and depalmitoylation are catalyzed by protein acyltransferases and acylprotein thioesterases, respectively. In this study, hydroxylamine, NH 2 OH, was used to reduce the thioester linkage. C) The amino acid sequence identity (similarity) among IFITM5, IFITM1, IFITM2, and IFITM3 is summarized. doi: 10.1371/journal.pone.0075831.g001 and protein-protein interaction. The authors revealed that IFITM3 is S-palmitoylated on three membrane proximal cysteines, Cys71 and Cys72 in the first transmembrane (TM1) domain, and Cys105 in the CP loop (Figure 1-A) [10] . In addition, IFITM3 lacking the S-palmitoylation is not clustered in the cell membrane and significantly diminishes the antiviral activity. Moreover, the cysteines in IFITM2, Cys70, Cys71, and Cys104 are also palmitoylated in the same manner, which affects the intracellular localization [24] . A resent study has revealed that murine IFITM1 has four cysteine residues (Cys49, Cys50, Cys83, and Cys103) for the S-palmitoylation, which is required for the antiviral activity and the protein stability [25] . The other IFITM family members also possess these cysteines (Figure 1-A) , and thus the role of the Spalmitoylation on the cysteines should be significant for the functions of IFITM proteins.\\n\\nHere, we focused on IFITM5, which is also known as bonerestricted IFITM-like (BRIL) protein [18] . Among the IFITM family proteins, IFITM5 is unique. (i) Expression of IFITM5: Unlike the other IFITM family proteins, the expression of IFITM5 is not induced by interferons because the region upstream of the ifitm5 gene lacks the interferon regulatory elements [26] . Furthermore, the expression of IFITM5 is mostly restricted to osteoblast cells [18, 19, 27] , while the other IFITM proteins are expressed ubiquitously (ii). Amino-acid sequence similarity: The amino acid sequence of IFITM5 is relatively dissimilar to IFITM1-3 proteins (~ 65% similarity), while IFITM1-3 proteins share ~ 85% similarity with each other (Figure 1 -C). In addition, IFITM5 has an aspartate-rich domain in the C-terminal region, which could be involved in calcium binding (Figure 1 -A) [26] . (iii) Role of IFITM5 in bone formation: The expression of IFITM5 is associated with mineralization during the bone formation process in osteoblast cells [18] [19] [20] [21] . Previous studies have confirmed the expression of IFITM5 in bone tissues in mice, rats, humans and tammar wallabies [2] . The ifitm5-gene knockout mice have smaller bones [19] . Moreover, the knockdown of the ifitm5 gene by small hairpin RNA induces a decrease in bone nodule formation, whereas overexpression of the gene in UMR106 cells has been shown to increase calcium uptake and bone nodule formation [18] . (iv) Role of IFITM5 for immune activity: Recent studies have revealed that IFITM5 interacts with the FK506-binding protein 11 (FKBP11) to form IFITM5-FKBP11-CD81-the prostaglandin F2 receptor negative regulator (FPRP) complex [28] . When the complex is formed, the expressions of 5 interferon-induced genes are induced, including bone marrow stromal cell antigen 2 (Bst2), interferon inducible protein 1 (Irgm), interferoninduced protein with tetratricopeptide repeats 3 (Ifit3), b(2)microglobulin (B2m), and MHC class I antigen gene. Consequently, these results indicate that IFITM5 is involved not only in the bone formation but also in the immune system activity.\\n\\nIn this study, we investigated the S-palmitoylation of IFITM5 and its role in the interaction with FKBP11 in mouse osteoblast cells. Cells transfected by a plasmid DNA encoding mouse IFITM5 were grown in the presence of an established chemical reporter, 17-octadecynoic acid (17-ODYA) [29, 30] , or an inhibitor for the S-palmitoylation, 2-bromopalmitic acid (2BP) [31] . The biochemical assays using these compounds revealed that the wild-type IFITM5 is S-palmitoylated. To identify the Spalmitoylation site in IFITM5, we prepared cysteine-substituted mutants, IFITM5-C86A, -C52A/C53A, and -C52A/53A/86A (Cys-less). The chemical reporter assay suggested that at least two out of three cysteines in IFITM5 are S-palmitoylated. The interaction of IFITM5 with FKBP11 was examined by immunoprecipitation assay, resulting in the loss of the interaction in the presence of 2BP. The same result was obtained in the two mutants, C52A/C53A and Cys-less. These results suggested that the S-palmitoylation on Cys52 and/or Cys53 in the TM1 domain of IFITM5 is necessary for the interaction with FKBP11. On the other hand, Cys86 in the CP loop of IFITM5 was S-palmitoylated but not involved in the interaction. Because this interaction is important for the immunologically relevant gene expression, it was indicated that the role of the S-palmitoylation is to promote the interaction of IFITM5 with FKBP11 and to regulate the immune activity in the osteoblast cells. The possible interaction mechanism and the effect of the S-palmitoylation on the bone nodule formation will be discussed.\\n\\nFor mammalian cell expression, plasmid vectors of wild-type IFITM5 (IFITM5-WT) and FLAG-fused FKBP11 (FKBP11-FLAG) were constructed by inserting the cloned genes into a pBApo-CMV Neo expression vector (Takara Bio, Shiga, Japan). The details of the recombinant DNA constructs were the same as described previously [19] . The genes of IFITM5 mutants (IFITM5-C86A, -C52A/53A, and -C52A/C53A/C86A (Cys-less)) were prepared using a QuikChange site-directed mutagenesis kit (Stratagene, La Jolla, CA). The plasmid vectors of FLAG-fused IFITM5-WT, -C52A/53A, and Cys-less were constructed by inserting the cloned genes into the pBApo-CMV Neo expression vector. For E. coli cell expression, the plasmid vector of IFITM5-WT was constructed by inserting the cloned gene into a pET22b (Novagen, Madison, WI) expression vector. The forward primer 5\\'-GGAATTCCATATGGACACTTCATATCCCCGTG-3\\' and the reverse primer 5\\'-CCGCTCGAGGTTATAGTCCTCCTCATCAAACTTGG-3\\' were used to amplify the gene encoding the entire IFITM5 from the plasmid vector for mammalian cell expression described above. The underlined letters denote an NdeI and an XhoI cleavage site, respectively. The plasmids of IFITM5 mutants were prepared using a QuikChange site-directed mutagenesis kit. The sense and anti-sense primers used were 5\\'-GGCAGTATGGCTCCAAAGCCAAGGCGTACAACATCCTGG CTGC-3\\' and 5\\'-GCAGCCAGGATGTTGTACGCCTTGGCTTTGGAGCCATACT GCC-3\\' for IFITM5-C86A; and 5\\'-GCACGATGTACCTGAATCTGGCGGCGCTTGGATTCCTGG CGC-3\\' and 5\\'-GCGCCAGGAATCCAAGCGCCGCCAGATTCAGGTACATCG TGC-3\\' for IFITM5-C52A/C53A, respectively (Sigma-Aldrich, St. Louis, MO).\\n\\nOsteoblast-like MC3T3 cells were provided by the RIKEN, Cell Bank (RCB 1126). The procedures for cell culture, transfection, and protein expression were the same as reported previously. When necessary, 2-bromopalmitic acid (2BP; Wako, Osaka, Japan) and 17-octadecynoic acid (17-ODYA; Sigma-Aldrich) were dissolved in 99.5% dimethyl sulfoxide (DMSO; Wako) and added to differentiation medium at concentrations of 100 μM and 50 μM in less than 0.1% DMSO, respectively [30, 31] .\\n\\nWild-type and mutant IFITM5 proteins were also produced using an E. coli recombinant expression system. E. coli BL21(DE3) cells transformed by the expression plasmid were grown at 37°C in LB medium containing 50 μg/mL ampicillin. After four-hour induction by 1 mM isopropyl β-Dthiogalactopyranoside (IPTG), cells were harvested by centrifugation (6,400 × g for 10 min at 4°C). The cells were suspended in 50 mM Tris-HCl buffer (pH 8) and disrupted by a French press (Ohtake, Tokyo, Japan) (100 MPa × 4 times). The crude membrane fraction was collected by ultracentrifugation (178,000 × g for 90 min at 4°C). The collected fraction was solubilized with 1.5% n-dodecyl-β-Dmaltopyranoside (DDM) (Dojindo Lab, Kumamoto, Japan) in 50 mM Tris-HCl, pH 8, containing 0.3 M NaCl and 5 mM imidazole. After the ultracentrifugation, the supernatant was incubated with Ni 2+ -NTA agarose resin (Qiagen, Hilden, Germany). The resin was applied to a chromatography column and washed with 50 mM imidazole containing 50 mM Tris-HCl (pH 8), 0.3 M NaCl and 0.1% DDM. The DDM-solubilized IFITM5 was collected by elution with the same buffer containing 0.3 M imidazole. The sample media were replaced by the appropriate buffer solution by two passages over a PD-10 column (GE Healthcare UK, Ltd., Amersham Place, England).\\n\\nThe experimental details are described in previous reports [19, 28] . Briefly, total proteins were extracted from the osteoblast cells which co-expressed IFITM5 and FKBP11-FLAG using a total protein extraction kit (BioChain Institute Inc., Newark, CA). Then, the cell lysate was incubated with anti-FLAG M2 agarose gel (Sigma-Aldrich) at 4°C for 2 h. To recover FKBP11-FLAG, 500 ng/μL 3 × FLAG peptide (Sigma-Aldrich) dissolved in Tris-buffered saline was added to the collected gel at 4°C for 1 h. The recovered proteins and the cell lysate containing total proteins were analyzed by SDS-PAGE (15% ePAGEL; ATTO, Tokyo, Japan) and western blot. The anti-IFITM5 polyclonal antibody, which was prepared from the amino-terminal peptide sequence (TSYPREDPRAPSSRC), and anti-FLAG monoclonal antibody (Sigma-Aldrich) were used as primary antibodies. The HRP-conjugated goat anti-rabbit IgG (H+L) (Zymed Laboratories, San Francisco, CA) and goat anti-mouse IgG (H+L) (Sigma-Aldrich) antibodies were used as secondary antibodies for the anti-IFITM5 and anti-FLAG primary antibodies, respectively. The proteins were detected by chemiluminescent reaction (MercK-Millipore, Billerica, MA).\\n\\nThe cell lysate extracted from the osteoblast cells metabolically labeled by 17-ODYA was incubated with anti-FLAG M2 agarose gel to obtain purified FLAG-fused IFITM5 proteins. The 17-ODYA-labeled proteins were chemically labeled with azide-PEG 3 -5(6)-carboxytetramethylrhodamine (TAMRA-azide; Click Chemistry Tools, Scottsdale, AZ) with reference to previous studies [10, 29, 30, 32] and the manufacturer\\'s guide. The proteins separated by SDS-PAGE were visualized using a 532-nm laser for excitation and the fluorescence by TAMRA (565 nm) was detected using a 575nm long-path filter (Typhoon FLA 9000; GE Healthcare).\\n\\nThe subcultured osteoblast MC3T3 cells were seeded at a density of 5,000 cells/cm 2 in 40 mm dishes and cultured in α-Modified Eagle\\'s Medium (α-MEM; Sigma-Aldrich) containing 10% (v/v) fetal bovine serum (FBS; Nichirei Biosciences Inc., Tokyo, Japan). On the next day, this was replaced with differentiation medium, containing 2 mM glycerophosphate and 50 μg/mL sodium ascorbate at final concentrations, to induce osteoblast differentiation. When necessary, 100 μM 2BP in less than 0.1% DMSO, or 0.1% DMSO alone was added to the differentiation medium at final concentrations. All cultures were incubated at 37°C in a humidified atmosphere containing 5% CO 2 for 27 days. Mineralized nodules were stained with Alizarin Red S (Sigma-Aldrich). The standard staining procedure was used. The mineralized nodules were checked every three days.\\n\\nTo identify the S-palmitoylation on IFITM5, the osteoblast cells harboring the plasmid DNA encoding IFITM5-WT were cultured in the absence and presence of 2BP, which inhibits the S-palmitoylation (Figure 2-A) [31] . Then, the cell lysate containing total protein was extracted for use in the SDS-PAGE and western blot analyses. For purposes of comparison, E. coli cells were also cultured in the absence of 2BP and the cell lysate was extracted. Figure 2 -B shows the results of the western blot assay for IFITM5-WT expressed in the osteoblast and the E. coli cells. In the osteoblast cells, IFITM5-WT exhibited a single band near the 17.4 kDa molecular-mass marker (see lane 1) in the absence of 2BP. However, in the presence of 2BP (see lane 4), the band appeared at a lower position than that in the absence of 2BP (lane 1). These results suggested that IFITM5-WT has high and low molecular-mass forms in the absence and presence of 2BP, respectively. The S-palmitoylation is a reversible reaction, and therefore is depalmitoylated by a strong reductant such as hydroxylamine [10] . Following hydroxylamine treatment (see lane 2), the band appeared at the same position as in the presence of 2BP (lane 4). In prokaryote E. coli cells, the post-translational modification S-Palmitoylation on IFITM5 PLOS ONE | www.plosone.org does not occur. Hence, the band was also observed at the same lower position (see lane 3). In the case of IFITM3, the palmitoylation was also reported to induce a change in mobility on electrophoresis, just as in our present results [10] .\\n\\nFor direct observation of the S-palmitoylation, an established chemical reporter, 17-ODYA (Figure 2-C) , was used. The osteoblast cells harboring the plasmid encoding IFITM5-WT were cultured in the presence of 17-ODYA to label the protein metabolically. Following the extraction and the purification of the cell lysate, the labeled IFITM5-WT was ligated with TAMRA-azide according to the Cu(I)-catalyzed [3+2] azidealkyne cycloaddition method [10, 29, 30, 32 ]. An in-gel fluorescence image of the 17-ODYA-TAMRA-labeled IFITM5-WT (see lane 2 in Figure 2 -D) showed that IFITM5 was Spalmitoylated in the osteoblast cells. The FLAG-tag attached to IFITM5 has no influence on the modification and chemical labeling (lanes 1 and 5). In addition, after the hydroxylamine treatment (see lane 6), the fluorescence became weak because of the dissociation of 17-ODYA from IFITM5, which was the same mechanism as the dissociation of the palmitic acid from IFITM5 by reduction as described above (lane 2 of Figure 2-B) .\\n\\nTherefore, we concluded that the IFITM5 expressed in the native osteoblast cells is S-palmitoylated. In addition, the bands corresponding to the high and the low molecular-mass forms shown in western blot analysis were tentatively assigned to the S-palmitoylated and the depalmitoylated forms, respectively.\\n\\nAs described above in the Introduction, cysteine residues are the substrate for S-palmitoylation. IFITM5 possesses three cysteines, Cys52 and Cys53 in the TM1 domain, and Cys86 in the CP loop (Figure 1-A) . All of these cysteines are highly conserved among the mammalian IFITM family proteins (Figure 3-A) . To identify the modification site in IFITM5, we prepared cysteine-substituted mutants, IFITM5-C52A/C53A, -C86A, and -C52A/C53A/C86A (Cys-less). The osteoblast cells harboring each plasmid were cultured in the absence of 2BP, and then the cell lysate was extracted. Figure 3 -B shows the results of the western blot detecting the expression of all the mutants in the osteoblast cells. In the C52A/C53A and Cys-less mutants (see lanes 2 and 4), the low molecular-mass form was detected. This result indicates that either Cys52 or Cys53 is involved in the S-palmitoylation. In addition, as shown in Figure  2 -D, strong and weak fluorescence were detected in the C52A/ C53A mutant in the absence and presence of hydroxylamine (lanes 3 and 7) , respectively, but not in the Cys-less mutant (lanes 4 and 8) . These results suggested that the rest of the cysteine in the C52A/C53A mutant, Cys86, is S-palmitoylated and the Cys-less mutant completely lost the S-palmitoylation because all the cysteines were substituted. Therefore, we concluded that Cys86, plus one or two other cysteine residues in IFITM5, i.e., Cys52 and/or Cys53, are S-palmitoylated. In addition, it was found that the S-palmitoylation on the TM1 domain has a major effect on the mobility in the gel (lower panel of Figure 2 -D and Figure 3-B) . Therefore, we hereafter refer to the high and low molecular-mass forms as the TM1palmitoylated and the TM1-depalmitoylated forms, respectively. Finally, we reassigned the bands shown in the western blot analysis as follows: IFITM5-WT is fully palmitoylated, the C86A mutant is partially palmitoylated at Cys52 and/or Cys53, the C52A/C53A mutant is partially palmitoylated at Cys86, and the Cys-less mutant is completely depalmitoylated.\\n\\nPrevious studies have revealed that IFITM5 interacts with FKBP11 [19] . FKBP11 belongs to the FK506-binding protein family and has a transmembrane domain. The interaction between IFITM5 and FKBP11 is important for the immune activity because formation of the IFITM5-FKBP11-CD81-FPRP complex induces the expression of interferon-induced genesnamely, the Bst2, Irgm, Ifit3, B2m, and MHC class I antigen gene [28] . To investigate the effect of the S-palmitoylation on the interaction of IFITM5 with FKBP11, we carried out an immunoprecipitation assay. The osteoblast cells co-transfected by the plasmids encoding IFITM5-WT and FKBP11-FLAG were cultured in the absence and the presence of 2BP. Then, the extracted cell-lysate was incubated with anti-FLAG agarose gel. The gel was washed several times. Finally, the proteins were competitively eluted by the addition of FLAG peptide. If IFITM5 interacted with FKBP11, it was expected that IFITM5 The conserved cysteines are highlighted in orange and numbered. In the lower panel, the numbers given in parenthesis correspond to the residual number for IFITM2. For the calculation of probability, a total of 23 IFITM2, 23 IFITM3, and 17 IFITM5 sequences derived from mammalian species in the Kyoto Encyclopedia of Genes and Genomes (KEGG) database were used. Sequence alignment was carried out using CLUSTALW. Sequence logos were generated using WEBLOGO 3. B) Western blot for the wild-type and cysteine-substituted mutants of IFITM5 expressed in the osteoblast cells. For detection, the anti-IFITM5 antibody was used as a primary antibody. The upper arrow indicates that C52 and/or C53 in the TM1 domain is Spalmitoylated (lanes 1 and 3) . The C52A/C53A (lane 2) and Cys-less (lane 4) mutants are partially and completely depalmitoylated. The experiment was carried out 2 times. doi: 10.1371/journal.pone.0075831.g003 would be obtained during this step and detected by immunoblotting. Figure 4 -A shows the results of the western blot for the co-immunoprecipitation of IFITM5-WT with FKBP-FLAG. The band corresponding to FKBP11 appeared in all the lanes (upper panel). Lanes 1 and 2 are controls to ensure that IFITM5 and FKBP11 are both contained in the cell lysate before the immunoprecipitation. The controls also ensured that IFITM5 was S-palmitoylated in the absence of 2BP (see lane 1), whereas IFITM5 was not S-palmitoylated in the presence of 2BP (see lane 2). After the immunoprecipitation, a single band corresponding to the S-palmitoylated IFITM5 appeared in the absence of 2BP (see lane 3), indicating the interaction of the Spalmitoylated IFITM5 with FKBP11. However, in the presence of 2BP, no band corresponding to IFITM5 appeared (see lane 4) , indicating that the two molecules do not interact with each other. These results suggest that the S-palmitoylation on IFITM5 contributes to the interaction with FKBP11.\\n\\nNext, we further investigated the relationship between the Spalmitoylation and the interaction with FKBP11 by using the IFITM5 mutants described above. The osteoblast cells cotransfected by the plasmids encoding IFITM5 mutants (C52A/ C53A, C86A, and Cys-less) and FKBP11-FLAG were cultured. The immunoprecipitation assay was carried out in the same way as described above. Figure 4 -B shows the results of the western blot for the co-immunoprecipitation of the wild-type and the IFITM5 mutants with FKBP11. Figure 4 -C shows the results of the control experiment using the cell lysate before the immunoprecipitation. As described in the previous section 3-3, the band corresponding to FKBP11 appeared in all the lanes (upper panels) because the immunoprecipitation was carried out using the anti-FLAG agarose gel. In the lower panel of Figure 4 -B, single bands were observed for the IFITM5-WT and -C86A mutant (lanes 1 and 3) but not for the -C52A/C53A and Cys-less mutants (lanes 2 and 4) . This result indicates that the wild-type and the C86A mutant interact with FKBP11, whereas the other two mutants do not. Interestingly, this tendency mirrored the trend for the S-palmitoylation profiles, which means that Cys52 and/or Cys53 in the TM1 domain of the IFITM5-WT and -C86A mutants is S-palmitoylated, whereas these residues are not S-palmitoylated in the C52A/C53A and Cys-less mutants (see Figures 2-D, 3 -B and the lower panel of Figure 4 -C). Because the S-palmitoylation contributes to the IFITM5-FKBP11 interaction, as described in the previous section 3-3 (also in Figure 4-A) , the results of Figure 4 -B suggest that the mutants which lost the S-palmitoylation site(s), Cys52 and/or Cys53, are not able to interact with FKBP11. In other words, the S-palmitoylation on these cysteines is necessary for the interaction of IFITM5 with FKBP11.\\n\\nAs described above in the Introduction, previous studies have revealed that IFITM5 also contributes to bone formation [18] [19] [20] [21] . Therefore, we investigated the influence of Spalmitoylation on the bone nodule formation in osteoblast cells, in which native IFITM5 is expressed. Figure 5 shows the time-dependent nodule formation in the absence and the presence of 2BP ( Figure 5-A and -B) . Figure 5 -C shows the results of the control trial to verify the effect of DMSO, which was used as the solvent for 2BP, on the nodule formation. The mineralized nodule was stained with Alizarin Red, which reacts with deposited calcium. In Figure 5 -D, the area of the mineralized nodule was plotted against experimental time. In the absence of 2BP (Figure 5-A, -C, and -D) , the mineralization was started 15 days after the initiation of the cell differentiation (Day 0). On the other hand, in the presence of 2BP ( Figure 5-B and -D) , the nodule was formed on Day 12. The halftime for the maximum mineralization in the presence of 2BP was estimated to be 7 days earlier than that in the absence of 2BP (Figure 5-D) . In addition, differences in the form of the mineralized nodules were observed. Figure 5 -E shows an enlarged view of each nodule on Day 21. The stained nodules were diffused in the presence of 2BP (panel b), whereas in the absence of 2BP the nodules formed a large cluster (panels a and c). Therefore, our observations in this study suggested that the S-palmitoylation affects the bone nodule formation in the osteoblast cells.\\n\\nIn this study, we confirmed the S-palmitoylation on IFITM5 in the osteoblast cells, which was the same as that previously reported for IFITM3 and IFITM2. As reported previously, in IFITM3 and IFITM2, which share 85% sequence similarity (Figure 1-C) , two cysteines in the TM1 domain (Cys71 and Cys72 for IFITM3, Cys70 and Cys71 for IFITM2) and one cysteine in the CP loop (Cys105 for IFITM3, Cys104 for IFITM2) are all S-palmitoylated in cells [10, 24] . On the other hand, although IFITM5 shares 68% and 66% sequence similarity to IFITM3 and IFITM2, respectively, more than one cysteine in the TM1 domain (Cys52 or Cys53) and one cysteine in the CP loop (Cys86) are S-palmitoylated. Taking into account the high conservation of three cysteines in the IFITM proteins (Figures 1-A and 3-A) , all the cysteines in IFITM5 may be involved in the S-palmitoylation just as in the case of IFITM3 and IFITM2 [10, 24] .\\n\\nThe roles of the S-palmitoylation on IFITM3 have been studied intensively, and the S-palmitoylation has been shown to be crucial for the correct positioning in the membrane and the resistance to viral infection and internalization [10] (the roles are summarized in Figure 6 -A and discussed in detail below). A recent study has revealed that the S-palmitoylation on IFITM2 is also important for the protein clustering in the membrane [24] . However, we do not know the role of the Spalmitoylation of IFITM5 for the clustering in the membrane at present because we have not yet succeeded in obtaining a proper antibody for immunohistochemistry, despite our allocating much time to the search and considering a considerable number of antibodies.\\n\\nDr. Hanagata and co-workers previously reported that IFITM5 lacking the TM1 domain and the CP loop, which and IFITM5 (lower panels), the anti-FLAG and the anti-IFITM5 antibodies were used as primary antibodies, respectively. Arrows indicate the existence of each protein and the S-palmitoylation on IFITM5. A) Western blot for the co-immunoprecipitation of the wild-type IFITM5 with the FLAG-fused FKBP11 (FKBP11-FLAG) in the osteoblast cells in the absence and the presence of 2BP (denoted as \"-\" and \"+\", respectively). Lanes 1 and 2 are the results for the control trials used to verify the existence of IFITM5 and FKBP11 before the immunoprecipitation, and Lanes 3 and 4 show the results after the immunoprecipitation. The experiment was repeated 3 times. B) Western blot for the co-immunoprecipitation of the wild-type and the cysteine-substituted mutants of IFITM5 with FKBP11-FLAG in the osteoblast cells. The band corresponding to FLAG peptide is not shown because of the smaller molecular-mass of FLAG peptide relative to FKBP11-FLAG. C) The control experiment of Figure 4 -B used to verify that IFITM5 and FKBP11 were both present in the cell lysate before the immunoprecipitation. The experiment was repeated 2 times. A) The functional mechanism of IFITM3 is summarized from previous studies. (i) IFITM3 is S-palmitoylated at Cys71, Cys72, and Cys105, (ii) which induces clustering and correct positioning in the membrane, (iii) resulting in the antiviral activity against influenza virus. B) The functional mechanism of IFITM5 is summarized by combining the results from the present and the previous studies. (i) Cys86, plus one or two other cysteine residues in IFITM5, i.e., Cys52 and/or Cys53, are S-palmitoylated (ii). The S-palmitoylation allows IFITM5 to interact with FKBP11 in the osteoblast cells (iii). The dissociation of CD9 from the FKBP11-CD81-FPRP/CD9 complex is induced by formation of the IFITM5-FKBP11-CD81-FPRP complex and leads to the immunologically relevant gene expression. IFITM5 also contributes to the bone formation, but it is unknown which states as described in (i)-(iii) are important for the bone formation at present.At present, no interactive protein has been identified in IFITM3 and IFITM2. On the other hand, IFITM5 interacts with the partner protein, FKBP11, and the S-palmitoylation clearly makes a significant contribution to the interaction. Therefore, IFITM5 forms a hetero-oligomer in the cell membrane for its physiological function.\\n\\ncontain the relevant modification sites, lost the ability to interact with FKBP11 [19] . In the present study, we determined that the S-palmitoylation on Cys52 and/or Cys53 in the TM1 domain is necessary for the interaction. From these results, we speculate that Cys52 and Cys53 face toward the interaction surface with FKBP11, and therefore IFITM5 and FKBP11 interact with each other through the palmitic acid(s) attached to the cysteine(s) (summarized in Figure 6 -B, discussed in detail later). Our investigation revealed that Cys86 is involved in the Spalmitoylation but does not contribute to the interaction with FKBP11. We speculate that some other residues in the CP loop located near the TM1 domain make some contribution to the interaction.\\n\\nPrevious investigations also revealed that IFITM5 expressed in the heterologous fibroblast NIH3T3 cells exhibited direct interactions with CD81, the B cell receptor-associated protein 31 (BCAP31), and the hydroxysteroid (17-beta) dehydrogenase 7 (HSD17b7). These three proteins bind to the IFITM5 without the S-palmitoylation (low molecular-mass form; see Figure 3 -b in ref [19] . and Figure 1 -B in ref [28] .). In the fibroblast cells, the S-palmitoylation on IFITM5 is insufficient [19] . These interactions are not observed in the native osteoblast cells, and therefore are nonspecific. Taking these facts into consideration, we speculate that the S-palmitoylation on IFITM5 promotes the specific interaction with FKBP11 in the osteoblast cells.\\n\\nThe role played by the S-palmitoylation of IFITM5 in immune activity of the osteoblast cells will be discussed by combining the results from the present and the previous studies. A specific interaction between IFITM5 and FKBP11 should be necessary to form the IFITM5-FKBP11-CD81-FPRP complex. CD81, also known as TAPA-1, is a member of the tetraspanin membrane protein family and a component of the B-cell coreceptor complex which mediates the B-cell signaling for immune responses. When forming this complex, CD9, a partner protein with CD81, dissociates from the FKBP11-CD81-FPRP/CD9 complex and consequently induces the osteoblastspecific expression of the interferon-induced genes, Bst2, Irgm, Ifit3, B2m, and the MHC class I antigen gene [28] . If the Spalmitoylation-mediated specific interaction of IFITM5 with FKBP11 were lost, the IFITM5-FKBP11-CD81-FPRP complex would not be formed, and consequently the interferon-induced gene expression would be inhibited because CD9 would remain associated with the FKBP11-CD81-FPRP/CD9 complex. In this respect, we speculate that IFITM5 is involved in the immune system activity in the osteoblast cells and the interaction of the S-palmitoylated IFITM5 with FKBP11 regulates the immune activity.\\n\\nIn addition, it was suggested that the S-palmitoylation on IFITM5 contributes to the bone nodule formation, including morphology and time for mineralization, in the osteoblast cells ( Figure 5 ). It is difficult to conclude at present that the lack of the S-palmitoylation on IFITM5 causes the diffusion of the bone nodules (panel b of Figure 5 -E); we can say, however, that IFITM5 will probably not be S-palmitoylated in the cells in the presence of 2BP. While 2BP is commonly used as an inhibitor of palmitoylation, it also targets many metabolic enzymes [33, 34] . Thus, it is also difficult to interpret the results of the long-term incubation of the osteoblast cells in the presence of 2BP. In any case, these are interesting and key observations in terms of clarifying the role played by the S-palmitoylation of IFITM5 in bone formation, and further studies are required. Figure 6 describes a possible mechanism of the interaction of IFITM5 with FKBP11 and the role of IFITM5 in the osteoblast cell function by means of a comparison with IFITM3. In the case of IFITM3, as shown in Figure 6 -A, the following are observed. (i) The three cysteines are all S-palmitoylated (ii). The S-palmitoylation leads to the clustering and the correct positioning of IFITM3 molecules in the membrane (iii). The Spalmitoylation and the following clustering are crucial for the resistance to the influenza virus. When IFITM3 lacks the Spalmitoylation, the IFITM3 molecules do not cluster, which leads to the significant decrease in the antiviral activity.\\n\\nOn the other hand, Figure 6 -B shows that the following observations are made in the case of IFITM5. (i) Cys86, plus one or two other cysteine residues in IFITM5, i.e., Cys52 and/or Cys53, are S-palmitoylated (ii). The S-palmitoylated IFITM5 is able to interact specifically with FKBP11. The interaction is presumed to be mediated by the palmitic acid(s) attached to the cysteine(s) facing toward the interaction surface on FKBP11. Cys86 is involved in the S-palmitoylation but not in the interaction of IFITM5 with FKBP11. At present, however, little is known about the role of the S-palmitoylation of IFITM5 for the localization in the membrane. When the S-palmitoylation affects the localization of IFITM5 as in the case of IFITM3 [10] , the S-palmitoylated IFITM5 molecules should be localized in the membrane or the depalmitoylated molecules should be delocalized. The loss of the interaction between IFITM5 and FKBP11 could be due to a relocalization of the depalmitoylated IFITM5 that prevents its association with FKBP11 (iii). The Spalmitoylated IFITM5 interacts with the FKBP11-CD81-FPRP/CD9 complex through FKBP11, which induces the dissociation of CD9 from the complex and the expression of 5 immunologically relevant genes. Finally, IFITM5 forms the IFITM5-FKBP11-CD81-FPRP complex. It is unknown at present which of the three states (i)~(iii) illustrated in Figure 6 -B is important for the bone mineralization of the osteoblast cells. The lack of the S-palmitoylation influences the interaction with FKBP11, which could account for the following complex formation and gene expression. In addition, the bone nodule formation is also affected. Note that the role of the Spalmitoylation has been involved in the bone formation [35] . It is indicated that the S-palmitoylation on IFITM5 plays roles not only for the regulation of the immune activity but also for the bone formation.\\n\\nIn conclusion, we have revealed the S-palmitoylation on IFITM5 and its role in the interaction with FKBP11. Not only the immune activity but also the bone mineralization in the osteoblast cells is affected by the S-palmitoylation. In general, the functional role of the S-palmitoylation is different for each protein [36] . For many proteins, the palmitoylation and depalmitoylation cycle is constitutive and regulated by enzymes. Based on the present results, it is difficult to address (i) whether the S-palmitoylation on IFITM5 is constitutive or regulated, or (ii) when and where IFITM5 is S-palmitoylated in the osteoblast cells. Further studies are required and are currently underway.', 'document_id': 650}]}, {'paragraphs': [{'qas': [{'question': 'What is the size of bovine coronavirus?', 'id': 917, 'answers': [{'text': '31 kb', 'answer_start': 840}], 'is_impossible': False}, {'question': 'What is the molecular structure of bovine coronavirus?', 'id': 918, 'answers': [{'text': 'single-stranded, linear, and nonsegmented RNA', 'answer_start': 784}], 'is_impossible': False}, {'question': 'How many nucleotides does bovine coronavirus contain?', 'id': 919, 'answers': [{'text': '30,847 nucleotides', 'answer_start': 1873}], 'is_impossible': False}, {'question': 'What is the size of the orf1ab gene in bovine coronavirus?', 'id': 920, 'answers': [{'text': '20kb', 'answer_start': 2034}], 'is_impossible': False}, {'question': \"Is the orf1ab gene at the 3' or 5' end of the bovine coronavirus genome?\", 'id': 921, 'answers': [{'text': '5= side', 'answer_start': 2067}], 'is_impossible': False}], 'context': \"First Complete Genome Sequence of a French Bovine coronavirus Strain\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC5477389/\\n\\nSHA: eef0ecf5b8e7b179dadaef967e65f2ab68f021e1\\n\\nAuthors: Kin, Nathalie; Guerard, Pauline; Diancourt, Laure; Caro, Valérie; Vabret, Astrid; Ar Gouilh, Meriadeg\\nDate: 2017-05-25\\nDOI: 10.1128/genomea.00319-17\\nLicense: cc-by\\n\\nAbstract: We sequenced the first Bovine coronavirus (BCoV) complete genome sequence from France. This BCoV was directly sequenced from a fecal sample collected from a calf in Normandy in 2014.\\n\\nText: B ovine coronavirus (BCoV) belongs to the Nidovirales order, the Coronaviridae family, the Coronavirinae subfamily, and the Betacoronavirus (https://talk.ictvonline.org/ ICTV/proposals/2008.085-122V.v4.Coronaviridae.pdf). Its genome is a single-stranded, linear, and nonsegmented RNA of around 31 kb. BCoV is responsible for respiratory and enteric diseases in cattle, particularly during winter (1, 2) . To date, the 19 complete BCoV genome sequences available in GenBank databases (consulted on 17 January 2017) originated from the United States or Asia. Here, we report the first complete genome sequence of a BCoV detected in France.\\n\\nThe BCoV/FRA-EPI/CAEN/2014/13 strain was obtained from a fecal sample collected from a 1-week-old calf in Normandy in 2014. The presence of BCoV in the fecal sample was assessed using an in-house reverse transcription-PCR (RT-PCR) targeting the M gene (3). A cDNA library was synthesized using SuperScript III (Invitrogen, Carlsbad, CA, USA) and hexamers. The complete genome sequencing of overlapping PCR products was carried out in both directions, using original primers and Sanger's dideoxy sequencing. Sequencing reactions were performed as previously described (3). Sequences were assembled and annotated using the Geneious software (version 5.1.6). We obtained a sequence counting 30,847 nucleotides. The orf1ab, HE, S, ns5, E, M, and N genes of the obtained BCoV were submitted to a Blastn analysis. According to these analyses, the orf1ab (20kb nucleotides, located at the 5= side of the genome) gene is closely related to the Dromedary camel coronavirus (DcCoV) HKU23-23-362F strain from the United Arab Emirates (accession no. KF906251), with a nucleotide identity of 99.19%. Conversely, the NS2, HE, S, ns5, and M genes are closely related to the BCoV Bubalus/Italy/179/07-11 strain (accession no. EU019216), with nucleotide identities of 99.88%, 99.45%, 99.02%, 98.79%, and 99.28%, respectively. The E gene is closely related to the Chinese Bovine coronavirus strain BCV-AKS-01 (accession no. KU886219), with a nucleotide identity of 100%. Finally, the highest Blastn score for the N gene was found with the American enteric BCoV-ENT (accession no. AF391541), associated with a nucleotide identity of 100%.\\n\\nMultiple-sequence alignment, including 20 BCoVs and 10 clade A betacoronaviruses closely related to BCoV from North America, two DcCoVs from the United Arab Emirates, and two Human coronavirus OC43 (HCoV-OC43) strains from France, was performed using the Muscle algorithm implemented in MEGA7 (4, 5) . The phylogenetic analysis on the orf1ab confirms that BCoV/FRA-EPI/CAEN/2014/13 is closely related to the Dromedary camel coronavirus (DcCoV) HKU23-23-362F. The orf1ab gene of these two viruses together clustered separately from that of BCoV and BCoV-like viruses from North America and Asia. This finding also confirms the results from our previous analysis on partial genomes in which nsp12, S, and N genes of American and Asian BCoVs group together in a cluster tentatively named C 1 . The nsp12 and N coding regions of BCoVs from France and DcCoVs from the United Arab Emirates clustered together in C 2 . The DcCoV S gene individualized from both HCoV-OC43 and BCoV S genes. Potential recombination events could be at the origin of DcCoV.\\n\\nAccession number(s). The complete genome sequence sequence of the BCoV/FRA-EPI/CAEN/2014/13 isolate has been deposited in GenBank under the accession number KX982264.\", 'document_id': 1546}]}, {'paragraphs': [{'qas': [{'question': 'What is a significant cause of Influenze like illness among healthy adolescents and adults presenting for medical evaluation?', 'id': 1658, 'answers': [{'text': 'HCoV', 'answer_start': 5069}], 'is_impossible': False}, {'question': 'What is the most common species of Human Coronavirus among adults?\\n', 'id': 1659, 'answers': [{'text': 'HCoV-OC43', 'answer_start': 3997}], 'is_impossible': False}, {'question': 'Which Human Coronavirus showed species specific clinical characteristics of its infection?', 'id': 1660, 'answers': [{'text': 'HCoV‐HKU1', 'answer_start': 956}], 'is_impossible': False}, {'question': 'What causes the outbreak of SARS and MERS.', 'id': 1717, 'answers': [{'text': 'Highly virulent species of HCoV', 'answer_start': 1276}], 'is_impossible': False}, {'question': 'What is the case fatality rate of SARS and MERS?', 'id': 1718, 'answers': [{'text': 'ranged from 14% to 45%', 'answer_start': 1448}], 'is_impossible': False}, {'question': 'What were the common HCOV strains in the 5 year USA study?', 'id': 1719, 'answers': [{'text': ' HCoV-OC43 and HCoV-229E', 'answer_start': 4100}], 'is_impossible': False}, {'question': 'Which species are more prevalent but less severe?', 'id': 1720, 'answers': [{'text': 'HCoV-HKU1, HCoV-OC43, HCoV-NL63, and HCoV-229E', 'answer_start': 1517}], 'is_impossible': False}], 'context': 'Species‐specific clinical characteristics of human coronavirus infection among otherwise healthy adolescents and adults\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC5820427/\\n\\nSHA: edfe02a438fa9b667313da8f03614303fc2a4a14\\n\\nAuthors: Bouvier, Monique; Chen, Wei‐Ju; Arnold, John C.; Fairchok, Mary P.; Danaher, Patrick J.; Lalani, Tahaniyat; Malone, Leslie; Mor, Deepika; Ridoré, Michelande; Burgess, Timothy H.; Millar, Eugene V.\\nDate: 2018-02-02\\nDOI: 10.1111/irv.12538\\nLicense: cc-by\\n\\nAbstract: Human coronavirus (HCoV) is a known cause of influenza‐like illness (ILI). In a multisite, observational, longitudinal study of ILI among otherwise healthy adolescents and adults, 12% of subjects were PCR‐positive for HCoV. The distribution of species was as follows: HCoV‐OC43 (34%), HCoV‐229E (28%), HCoV‐NL63 (22%), and HCoV‐HKU1 (16%). We did not observe species‐specific differences in the clinical characteristics of HCoV infection, with the exception of HCoV‐HKU1, for which the severity of gastrointestinal symptoms trended higher on the fourth day of illness.\\n\\nText: Clinical manifestations of human coronavirus (HCoV) infection range from a mild, self-limiting illness of the upper respiratory tract to an acute respiratory distress syndrome with a high mortality rate.\\n\\nHighly virulent species of HCoV were responsible for outbreaks of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS); case-fatality rates ranged from 14% to 45%. [1] [2] [3] By contrast, other HCoV species (HCoV-HKU1, HCoV-OC43, HCoV-NL63, and HCoV-229E) are much more prevalent, much less severe, and common causes of influenza-like illness (ILI). [4] [5] [6] [7] [8] [9] [10] [11] Five previous studies have described the species-specific clinical characteristics of HCoV infection among adults. 6, 7, [10] [11] [12] In two of these studies, a significant proportion of the study population had underlying medical conditions. 6, 7 Herein, we describe, among a cohort of otherwise healthy adolescents and adults with influenza-like illness (ILI), the species-specific prevalence and severity of symptoms associated with HCoV infection. 13 Patients 0-65 years of age and presenting for care <72 hours after onset of ILI symptoms were recruited for study participation. ILI was defined as a temperature ≥100.4°F and sore throat or one of the following respiratory symptoms: cough, sputum production, shortness of breath, or chest pain. Both inpatient and outpatient subjects were eligible to participate. Patients with underlying medical conditions (eg, diabetes, chronic obstructive pulmonary disease, severe asthma), women with a high-risk or complicated pregnancy, and patients with a poorly controlled psychiatric disorder were excluded. Information on patient demographics and presence/severity of symptoms at the time of enrollment was collected by in-person interview. Participants were then instructed on the use of a daily diary to record the presence/severity of symptoms for 7 days following initial symptom onset. Symptom severity was rated on an ordinal scale from 0 (none) to 3 (severe). Symptom severity scores were quantified using the following five measures: (i) individual symptom score for 20 symptoms, (ii) the upper respiratory symptom score, calculated as the sum of severity scores for earache, runny nose, sore throat, and sneezing, (iii) the lower respiratory symptom score, calculated as the sum of severity scores for cough, difficulty breathing, hoarseness, and chest discomfort, (iv) the gastrointestinal symptom score, calculated as the sum of severity scores for diarrhea, vomiting, anorexia, nausea, and (Table 1) .\\n\\nThere was season-to-season variability in the leading causes of \\n\\nThe findings of our study, conducted over a 5-year period at five geographically dispersed sites in the USA, demonstrate that human coronavirus (HCoV) is an important cause of influenza-like illness (ILI) ranged from 4% to 22%. [8] [9] [10] [11] 14 Additionally, we found HCoV-OC43\\n\\nto be the most common species among adults, as has been reported elsewhere. 8, 9, 11, 12, 14 HCoV-OC43 and HCoV-229E were the most common strains in alternate seasons, reflecting a season-to-season variability of HCoV strain circulation that has been reported in other multiyear studies. 4 8 The mechanisms by which this particular species elicits these symptoms are not known.\\n\\nThe strengths of this study of HCoV in otherwise healthy adolescents and adults include its multisite and multiyear design, the use of a multiplex diagnostic panel, the prospective collection of symptom data, and the use of a symptom severity scale similar to what has been employed previously. 15 One important limitation of this study was our selective recruitment of individuals who had presented to a healthcare facility for care of an ILI. Therefore, our cases are not representative of HCoV infection in the community, where individuals with mild, self-limiting illness due to HCoV opt not to seek medical care for the management of their ILI.\\n\\nIn summary, we have shown that HCoV is a significant cause of ILI among otherwise healthy adolescents and adults presenting for medical evaluation. Although there were differences in species distribution by age group, we did not detect any differences between species with respect to the clinical spectrum of disease.', 'document_id': 1545}]}, {'paragraphs': [{'qas': [{'question': 'What is required for a Hepatitis B infection in cells?', 'id': 2996, 'answers': [{'text': 'both intracellular and cell-surface factors', 'answer_start': 1158}], 'is_impossible': False}, {'question': 'What regulates the broad, but less specific, virus-cell interaction in a hepatitis B infection?', 'id': 2997, 'answers': [{'text': 'heparan sulfates in the membrane proteins', 'answer_start': 1727}], 'is_impossible': False}, {'question': 'Which protein domain of the Hepatitis B envelope is necessary for infection?', 'id': 2998, 'answers': [{'text': 'Nterminus of HBV preS1 (amino acids 1-47) ', 'answer_start': 2168}], 'is_impossible': False}, {'question': 'Where is NTCP located in the body?', 'id': 2999, 'answers': [{'text': 'lateral surface (canalicular) of hepatocytes', 'answer_start': 4322}], 'is_impossible': False}, {'question': 'What does the NTCP protein mediate?', 'id': 3000, 'answers': [{'text': 'bile acid transport ', 'answer_start': 4383}], 'is_impossible': False}, {'question': 'Is NTCP sufficient to allow HBV infection?', 'id': 3001, 'answers': [{'text': 'not sufficient', 'answer_start': 5012}], 'is_impossible': False}, {'question': 'Why is NTCP thought to not be sufficient for HBV infection?', 'id': 3002, 'answers': [{'text': 'the majority of HepaRG cells were found to express NPCT but not to be infected', 'answer_start': 5054}], 'is_impossible': False}], 'context': 'One step closer to an experimental infection system for Hepatitis B Virus? --- the identification of sodium taurocholate cotransporting peptide as a viral receptor\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC3562259/\\n\\nSHA: f4f36a8e9fee64d59ccf22b724c7dab345102658\\n\\nAuthors: Chen, Pei-Jer; Wu, T-C\\nDate: 2013-01-11\\nDOI: 10.1186/2045-3701-3-2\\nLicense: cc-by\\n\\nAbstract: Following the successful cloning of receptor for SARS coronavirus a few years ago, Dr. Wenhui Li and colleagues raised attention again by publishing a possible receptor for hepatitis B virus in eLife. We will briefly review the significance of this finding and the future prospects of hepatitis B research.\\n\\nText: Among the five hepatotropic hepatitis viruses, only hepatitis B virus (HBV) and its satellite hepatitis D virus (HDV) still wait for the development of an in vitro infection system in cell culture. One hepatocellular carcinoma (HCC) cell line, HepaRG, can be infected at a modest efficiency after weeks of culture and induced differentiation [1] . Even primary human hepatocytes rapidly lose the capacity for HBV infection after brief cell culture. The HBV infection demands both intracellular and cell-surface factors. The intracellular requirements appear less stringent, as after transfection of HBV DNA into many HCC cell lines or mouse liver, which cannot be infected naturally, the viral genome is expressed and replicates actively. Thus, the failure of HBV infection is considered largely to be due to strict restriction on the interaction between HBV virions and the cell membrane.\\n\\nThe molecules on the cell membrane needed for HBV infection can be divided into two classes: low affinity and high affinity molecules. Among others, the heparan sulfates in the membrane proteins mediate the broad, but less specific, virus-cell interaction. However, the high affinity membrane partners for HBV remain elusive (the carboxypeptidase D found for duck hepatitis B virus may be the only serious contender [2] ).\\n\\nHBV envelope protein, namely the surface antigens, plays an essential role in the infection process. Both genetic and functional examination identified one domain in the Nterminus of HBV preS1 (amino acids 1-47) necessary for infection. This domain has been shown to function as a direct mediator for HBV by binding presumably cellular corresponding receptor(s) [3] . More importantly, the myristoylated peptide is shown to effectively block HBV infection in primary human hepatocytes and in the human hepatocytechimera mouse at a nanomolar concentration [4] . In fact, a clinical trial testing the efficacy of this peptide in preventing HBV infection has been ongoing [5] . Clearly, this preS1 peptide can be a useful probe to pull out the interacting cellular factors, including specific viral receptors.\\n\\nYan et al. have taken a reasonable approach to fish out possible HBV receptor(s) [6] . They engineered the first 2-47 amino acid peptide from PreS1 to increase its capacity to be cross-linked with proteins interacting with the cell membrane, without affecting its binding specificity. In order to obtain sufficient materials after cross-linking, they adopted the Tupaia hepatocytes, instead of human hepatocytes, for the experiments. The strategies actually brought down many membrane proteins, but in comparison with the negative control (homologous peptide without specific binding), they identified one cellular protein, NTCP (sodium taurocholate cotransporting peptide) by LC/MS/MS. The same protein was pulled down from human hepatocytes as well. The authors further produced HCC cell lines stably expressing NTCP and subsequently infected them with HBV or HDV.\\n\\nImmunofluorescence staining clearly demonstrated the expression of HBV and HDV proteins in these cell lines, suggestive of a successful viral infection. In addition, they documented a 2-4-fold increase of viral RNA and DNA after infection in the cell line by real-time PCR. They also showed a Southern blot supporting the presence of HBV covalently closed circular DNA in the infected cell, a well-recognized marker for productive HBV infection. Finally, they identified a stretch of 10 amino acids in the NTCP transmembrane domain, as the motif directly interacting with the PreS1 peptide.\\n\\nNTCP is a transmembrane protein, usually located in the lateral surface (canalicular) of hepatocytes, which mediates bile acid transport [7] . Geographically, it is a good candidate for an HBV receptor. Moreover, the authors could convert the cell lines previously non-permissible to HBV infection to permissible by over-expression of NTCP, again supporting its possible role in the HBV infection process. This can be a critical and long-awaited discovery toward understanding HBV receptors and establishing an experimental HBV infection system.\\n\\nLooking forward, we need to understand how NTCP interacts with both HBV envelope proteins and with other cellular proteins, especially through the motif embedded in the cell membrane. NTCP itself is not sufficient to allow HBV infection, as the majority of HepaRG cells were found to express NPCT but not to be infected [8] . NTCP might initiate or mediate molecular interactions that can overcome the cell-surface restrictions for viral entry. Such cooperative cellular or viral factors have to be discovered and demonstrated to enhance the efficiency of viral infection, at a level comparable to a natural one (hundreds or thousands fold viral amplification). For example, the authors can use the NTCP-expressing cell lines as the starting materials to systemically identify other factors (maybe carboxypeptidase D) and make these cell lines more productive and permissive to HBV infection. In the near future, standard virological assays for HBV infections, including Northern or Western blots, are expected to demonstrate the successful HBV infections in vitro.\\n\\nThe HBV research community has searched for HBV receptors for decades. Many candidates have been discovered and then discarded. The current study, however, took advantage of a well-documented viral peptide required for HBV entry in combination with a state-of-the-art proteomics platform. As a Chinese proverb says \"a thousand-mile journey starts from one incremental step\". As such, the identification of NTCP as a potential viral receptor for HBV may serve as an important initial step for this journey, leading to the development of an HBV infection system to facilitate the HBV research and hepatitis B treatment.', 'document_id': 1552}]}, {'paragraphs': [{'qas': [{'question': 'What kinds of viruses are Japanese encephalitis virus(JEV), tick-borne encephalitis virus(TBEV), eastern equine encephalitis virus (EEEV), sindbis virus(SV), and dengue virus(DV)?', 'id': 3003, 'answers': [{'text': 'arboviruses', 'answer_start': 1906}], 'is_impossible': False}, {'question': 'What are the current clinically-available methods to detect encephalitis viral antigens?', 'id': 3004, 'answers': [{'text': 'ELISA and IFA', 'answer_start': 2177}], 'is_impossible': False}, {'question': 'What methods exist for detecting multiple antigens simultaneously in a one-sample, laboratory test?', 'id': 3005, 'answers': [{'text': 'two-dimensional gel electrophoresis , protein chip, mass spectrometry, and suspension array technology', 'answer_start': 2474}], 'is_impossible': False}, {'question': \"How many antigens could be detected by Liew's multiplex ELISA test?\", 'id': 3006, 'answers': [{'text': '9', 'answer_start': 2923}], 'is_impossible': False}, {'question': 'What kind of antibodies were used in the ELISA-array assay?', 'id': 3007, 'answers': [{'text': 'monoclonal', 'answer_start': 3480}], 'is_impossible': False}, {'question': 'How was the ELISA assay validated?', 'id': 3008, 'answers': [{'text': 'using cultured viruses and inoculated chicken eggs with patient sera', 'answer_start': 3618}], 'is_impossible': False}, {'question': 'What capture antibodies were used in the study?', 'id': 3009, 'answers': [{'text': '4D5, 2B5, 1F1, 2B8, 4F9, and 4E11', 'answer_start': 4365}], 'is_impossible': False}, {'question': 'What was the spotting concentration range for the capture antibodies?', 'id': 3010, 'answers': [{'text': 'from 0.2 to 0.0125 mg/ml', 'answer_start': 9091}], 'is_impossible': False}, {'question': 'How was the proper spotting concentration determined?', 'id': 3011, 'answers': [{'text': 'combination of minimized cross reaction and higher signal intensity', 'answer_start': 9317}], 'is_impossible': False}, {'question': 'How was cross reaction detection determined?', 'id': 3012, 'answers': [{'text': 'by applying JEV, YF, and DV cultures', 'answer_start': 9606}], 'is_impossible': False}, {'question': 'How was the ELISA-array assay validated?', 'id': 3013, 'answers': [{'text': 'using cultured viruses and inoculated chicken eggs with patient sera', 'answer_start': 3618}], 'is_impossible': False}], 'context': 'Development of an ELISA-array for simultaneous detection of five encephalitis viruses\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC3305475/\\n\\nSHA: ef2b8f83d5a3ab8ae35e4b51fea6d3ed9eb49122\\n\\nAuthors: Kang, Xiaoping; Li, Yuchang; Fan, Li; Lin, Fang; Wei, Jingjing; Zhu, Xiaolei; Hu, Yi; Li, Jing; Chang, Guohui; Zhu, Qingyu; Liu, Hong; Yang, Yinhui\\nDate: 2012-02-27\\nDOI: 10.1186/1743-422x-9-56\\nLicense: cc-by\\n\\nAbstract: Japanese encephalitis virus(JEV), tick-borne encephalitis virus(TBEV), and eastern equine encephalitis virus (EEEV) can cause symptoms of encephalitis. Establishment of accurate and easy methods by which to detect these viruses is essential for the prevention and treatment of associated infectious diseases. Currently, there are still no multiple antigen detection methods available clinically. An ELISA-array, which detects multiple antigens, is easy to handle, and inexpensive, has enormous potential in pathogen detection. An ELISA-array method for the simultaneous detection of five encephalitis viruses was developed in this study. Seven monoclonal antibodies against five encephalitis-associated viruses were prepared and used for development of the ELISA-array. The ELISA-array assay is based on a \"sandwich\" ELISA format and consists of viral antibodies printed directly on 96-well microtiter plates, allowing for direct detection of 5 viruses. The developed ELISA-array proved to have similar specificity and higher sensitivity compared with the conventional ELISAs. This method was validated by different viral cultures and three chicken eggs inoculated with infected patient serum. The results demonstrated that the developed ELISA-array is sensitive and easy to use, which would have potential for clinical use.\\n\\nText: Japanese encephalitis virus(JEV), tick-borne encephalitis virus(TBEV), eastern equine encephalitis virus (EEEV), sindbis virus(SV), and dengue virus(DV) are arboviruses and cause symptoms of encephalitis, with a wide range of severity and fatality rates [1] . Establishment of an accurate and easy method for detection of these viruses is essential for the prevention and treatment of associated infectious diseases. Currently, ELISA and IFA are the methods which are clinically-available for the detection of encephalitis viral antigens, but they could only detect one pathogen in one assay [2, 3] .\\n\\nThere are a variety of different methods available for identifying multiple antigens in one sample simultaneously, such as two-dimensional gel electrophoresis , protein chip, mass spectrometry, and suspension array technology [4] [5] [6] . However, the application of these techniques on pathogen detection is still in an early phase, perhaps due to the complicated use and high cost.\\n\\nAntibody arrays for simultaneous multiple antigen quantification are considered the most accurate methods [7] [8] [9] [10] . Liew [11] validated one multiplex ELISA for the detection of 9 antigens; Anderson [12] used microarray ELISA for multiplex detection of antibodies to tumor antigens in breast cancer, and demonstrated that ELISA-based array assays had the broadest dynamic range and lowest sample volume requirements compared with the other assays.\\n\\nHowever, the application of ELISA-based arrays is currently limited to detection of cancer markers or interleukins; no detection of pathogens has been reported. In this study, we developed an ELISA-based array for the simultaneous detection of five encephalitis viruses. Seven specific monoclonal antibodies were prepared against five encephalitis viruses and used to establish an ELISA-array assay. The assay was validated using cultured viruses and inoculated chicken eggs with patient sera. The results demonstrated that this method combined the advantage of ELISA and protein array (multiplex and ease of use) and has potential for the identification of clinical encephalitis virus.\\n\\nMonoclonal antibodies were prepared from hybridoma cell lines constructed by Prof. Zhu et al. Purification was conducted by immunoaffinity chromatography on protein G affinity sepharose [13] . Specific monoclonal antibodies (4D5 against JEV, 2B5 against TBEV, 1F1 against SV, 2B8 against serotype 2 DV, 4F9 against serotype 4 DV, 4E11 against EEEV, and 2A10 against Flavivirus) were selected for this study. All of the antibodies were raised according to standard procedures.\\n\\nUsing 4D5, 2B5, 1F1, 2B8, 4F9, and 4E11 as capture antibodies, detection antibodies (2A10, 1 F1, and 4E11) were coupled to biotin-NHS ester(Pierce, Germany) at 4°C for 3 h according to the manufacturer\\'s instructions. Unincorporated biotin was removed by Desalt spin column (Pierce). Immunologic reactions were reported by Streptavidin-HRP (CWBIO, Beijing, China) and Super Signal ELISA Femto Maximum sensitive substrate. Purified goat-anti mouse antibody was used as a positive control.\\n\\nJEV and DV were cultured in C6/36 cells; SV, TBEV, and EEEV were cultured in BHK-21 cells. The culture of TBEV and EEEV was conducted in biosafety level 3 facility, however, JEV, DV and SV were conducted in biosafety level 2 facility. Viral titers were determined by the 50% tissue culture infectious dose (TCID 50 ) method. All the cultures were inactivated by 0.025% β-propionolactone at 4°C overnight, then 37°C for 1 h to decompose β-propionolactone.\\n\\nAntibodies were spotted using a BIODOT machine (BD6000;California, USA) on ELISA plates (30 nl/dot). The plates were blocked with 3% BSA-PBS in 37°C for 1 h, followed by washing 3 times with PBS containing 0.1% Tween-20 for 2 min each. Then, the plates were dried, sealed, and stored at 4°C before use [11] .\\n\\nWhen spotting, different spotting buffers and concentrations of capture monoclonal antibodies were evaluated to optimize the ELISA-array assay. The optimization was evaluated by dot morphology and signal intensity. The tested spotting buffers included 1 × phosphate buffer saline (PBS), PBS +20% glycerol, and 1 × PBS + 20% glycerol+0.004% Triton-X100. A range of monoclonal antibody concentrations (0.0125, 0.025, 0.05, 0.1, and 0.2 mg/ml) were compared.\\n\\nFollowing a double antibody sandwich format, printed plates were incubated sequentially with inactivated viral cultures, biotin-labeled detecting antibody, HPR-labeled avidin, and substrate, followed by signal evaluation.\\n\\nAntigen binding was performed in PBS(containing 0.1% Tween-20 and 5% FCS) at 37°C for 2 h, followed by washing 3 times(1 × PBS containing 0.1% Tween-20). Incubation of ELISA plates with biotinylated detecting antibody cocktails was performed in PBS (containing 0.1% Tween-20 and 5% FCS) at 37°C for 2 h. After washing, specific binding of the detecting antibodies was reported by streptavidin-HRP and stained with Super Signal ELISA Femto Maximum sensitive substrate (Thermo scientific, Rockford, USA) [11, 14, 15] . Visualization of the plate was performed in AE 1000 cool CCD image analyzer(Beijing BGI GBI Biotech Company., LTD, China). The signal intensity and background of each spot was read out and recorded with \"Monster\"software. The positive signals were defined as a signal value > 400 and a signal value (sample)/signal value (negative) > 2.\\n\\nThe identical antibodies used in the ELISA-array format were also tested in a conventional ELISA format to determine the difference in sensitivity and specificity of the two methods. The conventional ELISAs were performed at the same time as the ELISA-array assays to ensure similar reaction conditions. The conventional ELISAs were performed in an identical maner to the ELISA-array, except that antibodies were coated at a concentration of 2 μg/mL in PBS (pH 7.4), and substrate TMB was used instead of Super Signal ELISA Femto Maximum sensitive substrate [16, 17] .\\n\\nThree serum samples were collected from patients with nervous system symptoms and histories of tick bites. The serum samples were treated with penicillin and streptomycin, then inoculated into the allantoic cavities of chicken eggs. 3 days later, the liquid was collected and divided into two portions (one for inactivation and one for RNA extraction). The RNA and inactivated samples were stored at -70°C before use.\\n\\nRNA was extracted from the inoculated chicken eggs using a RNeasy mini kit (Qiagen Inc., Valencia, CA, USA) according to the manufacturer\\'s instructions. All RNA extraction procedures were conducted at BSL-3 facilities. The primers and probes were used as previously described [18] . The real-time RT-PCR was conducted with a Quti-teck q-RT-PCR Kit (Qiagen Inc,). The reaction consisted of 10 μL of 2 × reaction buffer (0.2 μL reverse transcription enzyme, and 250 nmol/l primers and probes). RNA and deionized water were added to a final volume of 20 μl. PCR was performed with a LightCycler 2.0 (Roche, Switzerland) [19] .\\n\\nOptimization of the ELISA-array assay\\n\\nThe spotted array layout is depicted in Figure 1 and the efficacy of three different spotting buffers on the quality of the printed ELISA-arrays were investigated by spot morphology observation and signal intensity comparison.\\n\\nThe spotting concentration of the capture antibodies varied from 0.2 to 0.0125 mg/ml (each was serially diluted 2-fold). The efficacy of the spotting concentration of the capture antibodies was evaluated by virus culture detection, the proper spotting concentration was determined by a combination of minimized cross reaction and higher signal intensity. Figure 1 illustrates the array layout and Figure 2 demonstrates the result of the three spotting buffers and spot concentration of antibody 2B5 by TBE virus culture detection. Cross reaction detection was also conducted by applying JEV, YF, and DV cultures.\\n\\nSpot morphology observation (Figures 2a, b , and 2c) demonstrated that spotting buffer containing PBS with 20% glycerol produced tailed spot morphology; buffers containing PBS alone and PBS with 20% glycerol +0.004% Triton-X100 gave good spot morphology (round and full). Buffers containing PBS with 20% glycerol and PBS with 20% glycerol+0.004% Triton-X100 produced higher signal intensities than PBS alone. Thus, PBS with 20% glycerol+0.004% Triton-X100 was adopted as the optimized spotting buffer for subsequent experiments. Simultaneously, the spot concentration evaluation suggested that 0.05 mg/ml was optimal. At this concentration, the signal intensity was higher and the cross-reaction did not appear (Figure 2d ). Consequently, spotting concentration optimization of other capture antibodies (4D5, 1F1, 4E11, and 2B8) demonstrated that 0.05 mg/ml was also suitable(data not shown).\\n\\nThe optimized ELISA array layout is shown in Figure  3 , which was applied in the following experiments. \\n\\nSuccessful detection of viral pathogens requires a test with high sensitivity and specificity. To evaluate the performance of the designed antibody arrays, the specificity and sensitivity of the individual analytes were examined. By testing serially-diluted viral cultures, including DV-2, DV-4, JEV, TBE, SV, and EEEV, the sensitivity of ELISAarray and the identical conventional ELISA were compared ( Table 1 ). The detection limit of the two methods was compared and demonstrated. The cross-reactivity test was conducted using BHK-21 and vero cell lysate, Yellow fever virus (YFV) cultures (5 × 10 5 TCID 50 /ml, West Nile virus(WNV) cultures(2 × 10 6 TCID 50 /ml), and Western equine encephalitis virus(1 × 10 7 TCID 50 /ml). The results demonstrated that neither the ELISA-array nor traditional ELISA displayed cross-reactivity.\\n\\nEqual volumes of cultured TBEV, JEV, DV-2, DV-4, SV, and EEEV were prepared for single sample detection; two or three of the cultures were mixed for multiplex detection. A cocktail of biotin conjugated antibody (2A10, 4E11, and 1F1) was used in all tests. The results demonstrated that for all virus combinations, each virus was detected specifically, with no false-positive or-negative results (Figures 4 and 5) .\\n\\nChicken eggs inoculated with infected human serum were used for validation of the ELISA-array assay. All samples showed high reaction signals with capture antibody 2B5, which was specific for TBEV ( Figure 6b ). The ELISA-array assay suggested that the three patients were all infected with TBEV.\\n\\nTo verify the results tested by ELISA-array, RNA extracted from chicken eggs was applied to a real time-RT-PCR assay using primers and probes targeting TBEV. The results were also positive (Figure 6a) . The consensus detection results confirmed that the ELISAarray assay was reliable.\\n\\nTo be widely used in the clinical setting, the detection system should be easy to use and can be performed by untrained staff with little laboratory and experimental experience. Moreover, when the volume of the clinical samples is limited and an increasing number of pathogens per sample needs to be tested, the detecting system should be high-throughput to allow detection of multiple pathogens simultaneously [6, 20, 21] . Multiple detection, easy to use, and affordability are requirements for detection methods in the clinical setting. Thus, an ELISA-array, which combines the advantages of ELISA and protein array, meets the above requirements.\\n\\nIt has been reported that an ELISA-array has been used in the diagnosis of cancer and auto-allergic disease [7, 12] ; however, No study has reported the detection of viral pathogens. In this study, we developed a multiplex ELISA-based method in a double-antibody sandwich format for the simultaneous detection of five encephalitis-associated viral pathogens.\\n\\nThe production of a reliable antibody chip for identification of microorganisms requires careful screening of capture of antibodies [14] . Cross-reactivity must be minimized and the affinity of the antibody is as important as the specificity. First, we prepared and screened 23 monoclonal antibodies against eight viruses and verified the specificity and affinity to the target viruses by an immunofluorescence assay. Then, the antibodies were screened by an ELISA-array with a double-antibody sandwich ELISA format. The antibodies which produced cross-reactivity and low-positive signals were excluded. Finally, six antibodies were selected as capture antibodies. Another monoclonal antibody, 2A10, which could specifically react with all viruses in the genus Flavivirus was used for detecting antibody against DV, JEV, and TBEV. For the detection of EEEV and SV, although the detecting and trapping antibodies were the same (1F1 and 4E11, respectively), the antibodies produced excellent positive signals. The epitope was not defined; however, we suspect that the antibodies both target the surface of the virions. As one virion exits as, many with the same epitope appear, thus no interference occurred using the same antibody in the double-antibody sandwich format assay.\\n\\nCurrently, the availability of antibodies suitable for an array format diagnostic assay is a major problem. In the ELISA-array assay, this problem exists as well. Because of the limitation of available antibodies, this assay could only detect 5 pathogens. In the future, with increasing numbers of suitable antibodies, especially specific antibodies against Flavivirus, this ELISAarray might be able to test more pathogens and be of greater potential use. To make the assay more amenable to multiple virus detection, the assay protocol was optimized. In addition to the dotting buffer, the capture antibody concentration and the different virus inactivation methods (heating and β-propiolactone) were also compared and evaluated. Heat inactivation was performed by heating the viral cultures at 56°C for 1 h, and β-propiolactone inactivation was performed by adding β-propiolactone into the retains better antigenicity than the heat-inactivation method. Thus, β-propiolactone treatment was chosen as the virus-inactivation method. A conventional ELISA is a standard method in many diagnostic laboratories. We compared the ELISA-array with a conventional ELISA and confirmed that the advantage of the ELISA-array was evident with comparable specificity and higher sensitivity than ELISA. The time required for the ELISA-array is significantly less than for conventional ELISA (4 h vs. a minimum of 6 h, respectively). Furthermore, less IgG is required for printing than for coating ELISA plates. Coating of a single well in microtiter plate requires 100 μl of a 1 μg/ml antibody solution, which is equivalent to 100 ng of IgG. For the ELISA-array, only 30 nl of a 50 μg/ml antibody solution is required for each spot, which is equivalent to 1.5 ng of IgG. With the characteristics of ease of use, sensitivity, specificity, and accuracy, the ELISA-array assay would be widely accepted for clinical use.', 'document_id': 1553}]}, {'paragraphs': [{'qas': [{'question': 'In 2010, how many cases of tuberculosis were estimated in China?', 'id': 3014, 'answers': [{'text': '108 per 100,000', 'answer_start': 1869}], 'is_impossible': False}, {'question': 'What is the population of Shandong province?', 'id': 3015, 'answers': [{'text': '94 million', 'answer_start': 2484}], 'is_impossible': False}, {'question': 'What was the purpose of this study?', 'id': 3016, 'answers': [{'text': 'estimate the TB prevalence in Shandong', 'answer_start': 2837}], 'is_impossible': False}, {'question': 'What was the age range for the people surveyed?', 'id': 3017, 'answers': [{'text': '15 years old or above', 'answer_start': 3026}], 'is_impossible': False}, {'question': 'How was the survey designed?', 'id': 3018, 'answers': [{'text': 'in accordance with WHO recommendations', 'answer_start': 3393}], 'is_impossible': False}, {'question': 'Was was the sample size?', 'id': 3019, 'answers': [{'text': '52500', 'answer_start': 3532}], 'is_impossible': False}, {'question': 'How were the clusters selected?', 'id': 3020, 'answers': [{'text': 'A stratified multi stage random sampling', 'answer_start': 3841}], 'is_impossible': False}, {'question': 'How many people were in a community cluster?', 'id': 3021, 'answers': [{'text': '1250 to 1750', 'answer_start': 4236}], 'is_impossible': False}, {'question': 'Who was excluded from the study?', 'id': 3022, 'answers': [{'text': 'Military barracks and prisons', 'answer_start': 4627}], 'is_impossible': False}, {'question': 'When was the study conducted?', 'id': 3023, 'answers': [{'text': 'March to June 2010', 'answer_start': 4731}], 'is_impossible': False}, {'question': 'Who conducted the study?', 'id': 3024, 'answers': [{'text': 'clinicians, public health doctors, radiologists, laboratory technicians and nurses', 'answer_start': 4780}], 'is_impossible': False}, {'question': 'What medium was used to collect the sputum samples?', 'id': 3025, 'answers': [{'text': 'Löwenstein-Jensen medium', 'answer_start': 6549}], 'is_impossible': False}, {'question': 'What was the response rate for the study?', 'id': 3026, 'answers': [{'text': '95% to 97%', 'answer_start': 9615}], 'is_impossible': False}, {'question': 'What was the average age of a study participant?', 'id': 3027, 'answers': [{'text': '46 years', 'answer_start': 9776}], 'is_impossible': False}, {'question': 'What was the prevalence rate in Shandong in 2010 for sputum positive cases of tuberculosis?', 'id': 3028, 'answers': [{'text': '22.1', 'answer_start': 12601}], 'is_impossible': False}, {'question': 'What was the most striking finding of the study regarding tuberculosis patients?', 'id': 3029, 'answers': [{'text': 'a large proportion of TB patients did not present consistent cough', 'answer_start': 14313}], 'is_impossible': False}, {'question': 'How many cases of sputum positive tuberculosis patients had no persistent cough?', 'id': 3030, 'answers': [{'text': '45%', 'answer_start': 14655}], 'is_impossible': False}, {'question': 'How many tuberculosis patients in Shandong were over 65 years old?', 'id': 3031, 'answers': [{'text': 'over half', 'answer_start': 18779}], 'is_impossible': False}], 'context': \"Changes in pulmonary tuberculosis prevalence: evidence from the 2010 population survey in a populous province of China\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC3890533/\\n\\nSHA: eef61bdfa49b8652fd660b5b8b7e74cf51922505\\n\\nAuthors: Wei, Xiaolin; Zhang, Xiulei; Yin, Jia; Walley, John; Beanland, Rachel; Zou, Guanyang; Zhang, Hongmei; Li, Fang; Liu, Zhimin; Zee, Benny CY; Griffiths, Sian M\\nDate: 2014-01-11\\nDOI: 10.1186/1471-2334-14-21\\nLicense: cc-by\\n\\nAbstract: BACKGROUND: This paper reports findings from the prevalence survey conducted in Shandong China in 2010, a province with a population of 94 million. This study aimed to estimate TB prevalence of the province in 2010 in comparison with the 2000 survey; and to compare yields of TB cases from different case finding approaches. METHODS: A population based, cross-sectional survey was conducted using multi-stage random cluster sampling. 54,279 adults participated in the survey with a response rate of 96%. Doctors interviewed and classified participants as suspected TB cases if they presented with persistent cough, abnormal chest X-ray (CXRAY), or both. Three sputum specimens of all suspected cases were collected and sent for smear microscopy and culture. RESULTS: Adjusted prevalence rate of bacteriologically confirmed cases was 34 per 100,000 for adults in Shandong in 2010. Compared to the 2000 survey, TB prevalence has declined by 80%. 53% of bacteriologically confirmed cases did not present persistent cough. The yield of bacteriologically confirmed cases was 47% by symptom screening and 95% by CXRAY. Over 50% of TB cases were among over 65’s. CONCLUSIONS: The prevalence rate of bacteriologically confirmed cases was significantly reduced compared with 2000. The survey raised challenges to identify TB cases without clear symptoms.\\n\\nText: China, with an estimated prevalence of all TB cases of 108 per 100,000 in 2010, has the second highest TB burden in the world, accounting for 13% of all cases worldwide [1] . The World Health organization (WHO) estimated that China had reached the targets of 85% treatment success by 1993 and 70% case detection rate by 2005 [2] . National TB prevalence surveys were conducted in China in 1979 China in , 1990 China in , 2000 , and 2010 [4] . Survey results provide more accurate estimates for TB prevalence rates than the WHO estimates and can be used to assess the likelihood of China achieving global targets for TB prevalence.\\n\\nShandong province has a population of 94 million. It is a relatively developed province with a per capita GDP 1.6 times of the national average in 2010 [5] . The prevalence rate of TB in Shandong was lower compared with the average rate of China in 2000 [3] . Population representative samples were drawn in Shandong in the surveys of 2000 and 2010 using similar methods. The study aimed to estimate the TB prevalence in Shandong based on the 2010 survey, and compare results of the two cross sectional surveys.\\n\\nThe target population of the TB prevalence survey was residents of 15 years old or above who had lived in the selected clusters for more than 6 months. A population based, cross-sectional survey was conducted using multistage random cluster sampling method.\\n\\nThe survey employed the same sampling methods as the China national survey in 2010, which was similar to the sampling methods used in 2000 [6] . The design of the surveys was in accordance with WHO recommendations [7] . The design effect factor due to cluster sampling was estimated at 1.28 [8] . A sample size of 52500 adults (≧15 years old), in 35 clusters, was calculated based on detecting a change of 20% in prevalence rate of TB smear positive cases compared with the rate of the 2000 survey (95 per 100,000), with a probability greater than 95% and 95% power, accounting for 90% response rate of participants [9] .\\n\\nA stratified multi stage random sampling was used to select the 35 clusters within 17 prefectures in Shandong province. The number of clusters was randomly allocated in proportion to the provincial population at the prefectural, county/district and township levels. A cluster was defined as a community (a village in the rural area or a resident community in an urban area) with a population of 1250 to 1750 adults (i.e., those of 15 years or older). If the community contained less than 1250 adult residents, the neighboring community to the north was annexed. If the community or combined communities containing more than 1750 adults, we randomly selected households and then included all adults in the household for the survey until the total number of selected adults reached 1750. Military barracks and prisons located in the cluster were excluded [7] .\\n\\nThe survey was conducted from March to June 2010 by survey teams consisting of clinicians, public health doctors, radiologists, laboratory technicians and nurses. Local media was used to promote awareness of the survey. Community workers conducted a house-to-house census to update the database of residents, inform survey participants and obtain informed consent. The study did not involve children under 15 years old. Written informed consent was obtained from all participants of 16 years old or above. While from those of 15 years old, written informed consents were obtained from their parents or guardians. All documents were properly stored in the Shandong Chest Hospital. Ethical approvals for the study and consent procedures were obtained from the Institutional Review Board (IRB) of Shandong Chest Hospital (NIH register numberIRB00006010).\\n\\nThose who agreed to participate in the survey were invited to the county TB dispensary, where they completed a consultation with a trained clinical TB doctor regarding any symptoms suggestive of TB, such as persistent cough (lasting two weeks or longer), haemoptysis, weight loss and fever. All participants had a chest X-ray (CXRAY) taken that then were reviewed by a panel of radiologists. Those with symptoms or CXRAY films suggestive of TB were classified as suspected TB cases. All suspected cases were asked to produce three sputum samples, one at the time of consultation, another at night and the third in the early morning of the following day. Identified suspects completed an additional questionnaire regarding their social-economic situation, smoking status, and the presence of TB related symptoms in the preceding six months (cough, fever, weight loss, chest pain and haemoptysis).\\n\\nSputum smears were conducted in local TB dispensaries. All sputum samples were cultured using the Löwenstein-Jensen medium in the provincial laboratory within 24 hours using cold chain transportation. Samples were excluded from TB when non-tuberculosis bacilli were identified from the culture. All sputum smear and culture were conducted strictly according the national TB laboratory external quality control measure, which is in consistent with the WHO TB prevalence survey guideline [7] . TB classification was made according to the China national TB guideline [10] . A positive smear had at least one acid fast bacillus identified during examination of at least 100 fields. Participants with positive sputum smear specimens were classified as sputum positive cases. Those with positive smear or culture sputum specimens were classified as sputum bacteriologically confirmed cases. Those being culture negative with abnormal CXRAY suggestive of TB and having been ruled out from other diseases by clinicians and radiologists were classified as CXRAY suggestive bacteriologically negative cases. Due to resource limitations the recommendation of broad-spectrum antimicrobial agents to confirm the diagnosis of negative TB cases was not applied in this survey [11] . Newly diagnosed cases were distinguished from previously diagnosed cases through checks during the interviews and against the TB registration system. Initial diagnosis was made by a group of local clinicians and radiologists. Subsequently, samples and CXRAY films of all suspected and confirmed cases were re-assessed by a group of senior clinicians and radiologists at provincial and national levels. CXRAY films of 100% of those scored as abnormal and 10% random sampling of those scored as normal were transferred for independent reading. The provincial laboratory team randomly examined one slide from the three samples of sputum positive cases, all three samples of CXRAY suggestive TB cases, and randomly selected 10% of the non-TB cases.\\n\\nPrevalence estimates of sputum positive, bacteriologically confirmed and all TB cases were calculated. In all analyses, population weightings were employed to adjust for the stratified multi-stage sampling design effect [8] . The survey results in 2010 and 2000 were standardized against the age structures of China's census population in 2010. The 2000 TB prevalence survey included all age groups [12] . The WHO recommended method was used to enable comparison between the two surveys that prevalence rates of child TB were assumed to reduce to the same extent as adult TB from 2000 to 2010 [13] . Subgroup analysis in gender, age groups and urban/rural residence were conducted. Case identification rate was calculated as the number of cases identified by a screening method over all suspected cases found by the method. Yields of the symptom consultation and CXRAY were calculated as a proportion of the total number of bacteriologically confirmed cases.\\n\\nThe survey selected 17 urban clusters and 18 rural clusters. It covered a total population of 89,093, of which 56,671 were eligible for the survey (Figure 1 ). The response rate ranged from 95% to 97% in different clusters. 54,279 participants attended clinical consultation and were examined by CXRAY. Among them, 47% were males. The average age was 46 years with 14% of 65 years and older. A total of 572 suspected TB cases were found. Of these, 264 (46%) were identified based on CXRAY abnormalities, 228 (40%) were based on persistent cough, 80 (14%) were based on both. The survey diagnosed 172 new cases, including 19 new bacteriologically confirmed cases (including 11 sputum and culture positive cases, and 8 sputum negative but culture positive cases) and 153 CXRAY suggestive bacteriologically negative cases. The survey also identified 11 existing cases registered on the national TB program. In addition, the survey found four cases with culture positive non-tuberculosis bacilli, and excluded them from TB patients.\\n\\nAll participants of the survey were first screened by symptoms and CXRAY. Those who had symptoms of consistent cough or haemoptysis, or CXRAY abnormalities were then screened by smear and culture. Case identification rates of new bacteriologically confirmed cases from the suspected cases were significantly higher with CXRAY as a primary tool (Figure 1 , 3.8%, P = 0.012) and further increased by both symptom screen of persistent cough and CXRAY (10%, P < 0.001) compared with symptom screen alone (0.4%). The same pattern of case identification rate was observed in the sputum positive cases (7.5%, 1.9% and 0% respectively). The proportion reporting persistent cough was not significantly higher among bacteriologically confirmed cases compared with other suspects (P = 0.565). The symptom consultation alone identified 308 suspects, including 6 (1.9%) sputum smear positive TB and 9 (2.9%) bacteriologically confirmed TB. Among the 344 suspects with CXRAY abnormalities, 11 (3.2%) had sputum positive TB and 18 (5.2%) had bacteriologically confirmed TB. The yield of bacteriologically confirmed cases was 47.4% by screening consultation and 94.7% by CXRAY. In the population of over 65 years old, symptom consultation and the CXRAY identified 174 and 182 suspected cases respectively, yielding5 (2.9%) and 9 (4.9%) of bacteriologically confirmed cases. Yields of bacteriologically confirmed cases were 55.6% by symptom consultation and 100% by CXRAY among over 65's.\\n\\nOf the 512 suspected cases that completed the additional questionnaire, 42% were farmers and 31% were current smokers (Table 1) . Per capita household income of bacteriologically confirmed cases was less than 50% of that of the non-TB cases (P < 0.05). Though smoking rate was higher among TB cases compared with non-TB cases, no significant differences were found (P > 0.05). Of the ten bacteriologically confirmed cases not presenting with persistent cough at the prevalence survey, one coughed for two days, one had chest pain, and the other eight had no symptoms of TB in the last six months.\\n\\nThe crude prevalence rate in Shandong in 2010 of sputum positive cases was 22.1 (95% CI: 9.6-34.6), bacteriologically confirmed cases was 36.8 (95% CI: 17.8-55.8), and all cases were 337.1 (95% CI: 254.1-420.0) per 100,000 in adult population ( Table 2 ). The adjusted prevalence rates of the whole population in Shandong were17.8 (95% CI: 8.3-17.5), 27.8 (95% CI: 14.8-28.0) and 239.4 (95% CI: 179.9-298.9) per 100,000 in 2010. A remarkable decline of 82.0%, 80.2% and 31.4% was observed in TB prevalence rates of sputum positive, bacteriologically confirmed, and all cases, respectively, compared to the adjusted rates in 2000 [12] . Large declines were observed in males between 40 and 65 years old, and in females over 60 years old ( Figure 2) .\\n\\nThe adjusted prevalence rates in the adult population were 21.4 (95% CI: 10.0-32.8), 33.5 (95% CI: 17.8-49.2) and 285.8 (95% CI: 254.2-356.4) for sputum positive cases, bacteriologically confirmed cases and all cases, respectively. Significant differences regarding adjusted TB prevalence rates were observed between males and females, over 65's and 15 to 64 years old, in rural and urban areas ( Table 2 , P < 0.001). The male to female ratios were 5.5 in sputum positive cases and 2.8 in bacteriologically confirmed cases, while the ratios climbed to 6.0 and 4.1, respectively, among those over 65 years. The majority of TB patients, 54.5% of sputum positive cases and 47.3% of bacteriologically confirmed cases, were from people 65 years or older. The ratio between over 65's and 15 to 64 years old was 8.4 in sputum positive cases and 5.9 in bacteriologically confirmed cases. The ratio between rural and urban areas was 2.7 in sputum positive cases and 4.8 in bacteriologically confirmed cases.\\n\\nThe most striking finding was that a large proportion of TB patients did not present consistent cough. Passive case finding is the routine practice in developing countries where sputum microscopy is performed to identify TB cases among people with persistent cough. A large proportion of TB cases may be missed using this method as 53% of bacteriologically confirmed cases and 45% sputum positive cases in this study had no persistent cough but were identified through abnormal CXRAY. Nearly half of bacteriologically confirmed cases reported no symptoms in the last six months. This finding, although initially surprising, is consistent with reports from Vietnam (47% of bacteriologically confirmed cases not presenting persistent cough) [14] , Myanmar (38%) and Ethiopia (48%) [13] . CXRAY was sensitive in detecting TB cases, as yields of bacteriologically confirmed cases were much higher by CXRAY compared with by symptom screening, as reported in Vietnam [15] and some high HIV prevalence settings [16, 17] . CXRAY, though expensive at the initial installment, may improve TB case finding due to its short turnover time and high throughput [18] . Our findings suggest that the strategy of case finding using CXRAY followed by sputum or culture as the primary and secondary screening tests could be more effective, especially among the population of over 65 year olds, as the yields were higher in over 65's compared with the general Table 2 Prevalence rates of sputum positive TB cases, bacteriologically confirmed TB cases and all cases in Shandong, China, 2010 No population. Although using CXRAY to examine everyone is not feasible, it can be used in routine elder physical examinations. The China public health package now covers free CXRAY for elders, as well annual employee body examinations provided free CXRAY.\\n\\nIn this survey, only one sputum positive patient had been detected and treated by the national program, though specific clinical consultation was conducted to identify any patients who have been diagnosed and treated for TB before. This may reflect the difference between the active case finding approach in the survey and the passive casing finding approach in practice. Nevertheless, it indicated that a large proportion of bacteriologically confirmed TB cases are missed by the national TB program.\\n\\nAnother notable change is the sharp decline of the proportion of sputum positive cases, which accounted for 30.5% of all cases in the 2000 survey but was reduced to 6.6% in the 2010 survey. The proportion of notified sputum cases out of all TB cases in Shandong also declined from 80.9% in 2005 to 64.6% in 2010 [19] .\\n\\nThe prevalence rate of bacteriologically confirmed cases has reduced by 80% in the last decade in Shandong, compared with a national decline of 45% (from 216/ 100,000 in 2000 to 119/ 100,000 in 2010) [4] . The rapid decline of TB prevalence rate of bacteriologically confirmed cases in the recent decade may be attributed to China's strengthened public health system following the outbreak of severe acute respiratory syndrome in 2003 [2] . Another reason may be due to improved reporting of TB cases in the online communicable disease reporting system, and the improved collaboration between public hospitals and TB dispensaries [20] . Other factors such as social economic development may also have played an important role in the reduction of TB prevalence, as found in a study of TB notification rates trends in 134 countries [21] .\\n\\nThe adjusted prevalence rate of bacteriologically confirmed cases in Shandong was lower than the WHO estimates for China in 2010 [1] . But the national prevalence rates of bacteriologically confirmed cases, 119/100,000 in 2010 [4] , was higher than the WHO estimate, 108/ 100,000, even the survey did not collect negative and extra-pulmonary TB cases. Vietnam reported similar findings in its 2006 survey [14] . One reason is that prevalence surveys results are based on active case finding while WHO estimates are based on notification rates from passive case finding. A re-evaluation of the reported TB prevalence in China is needed based on the recent survey.\\n\\nCXRAY suggestive bacteriologically negative cases may be smear or culture negative TB cases if they had any TB symptoms, while some may be caused by suboptimal smear or culture. As reported in China's previous surveys [3, 22] , including these cases as TB cases may result in an over-estimate of all pulmonary cases [23] .\\n\\nThe survey revealed that over half of the TB patients were 65 years and older in Shandong, while the over 65's were more likely to present with abnormal CXRAY and persistent cough. Similar trends have been documented in other developed cities such as Hong Kong and Singapore [24] . These high rates may reflect the higher TB rates in the past and decline in immunity in the over 65's. How to treat elders with TB and other complications such as diabetes remains an ongoing challenge in China and similar settings.\\n\\nThe survey results can be generalized to the Shandong population of 94 million or similar international settings with middle income and middle TB prevalence levels. The patterns of the TB epidemic found in Shandong, i.e., the proportion of patients with symptoms, ratios between urban and rural areas, men and women, were similar to those found in the national survey [4] . However, the prevalence rates cannot be extrapolated to western provinces in China with a higher TB prevalence. For logistical reasons, the eligible population did not include adults staying in the sampled clusters less than 6 months, which was the same practice in the 2000 survey. However, shortterm migrants may have a potentially higher prevalence of TB than the general population [25] . This may result in a lower estimate of the true prevalence rate. The survey did not collect social-economic indicators, smoking status and HIV status of all participants, so comparisons between TB cases and all non-TB patients are not available. However, the HIV prevalence in Shandong China is below 0.01%, and would not significantly alter the TB prevalence rate. In addition, the survey did not evaluate child TB and extra pulmonary TB. Discussions of using CXRAY as a screening tool was on the technical aspect, but not on costing side as we did not conduct any cost effectiveness analysis or the social willingness to pay for such a strategy in similar settings.\\n\\nThis study has shown that the prevalence of bacteriologically confirmed TB in Shandong has reduced substantially over the last decade. Importantly, the majority of these cases did not present with persistent cough and the proportion of sputum positive cases has declined sharply. Further studies are recommended to assess the feasibility of adopting CXRAY in the existing health care services to detect TB cases and the cost effectiveness of such intervention. \\n\\nThe authors declare that they have no competing interests.\", 'document_id': 1557}]}, {'paragraphs': [{'qas': [{'question': 'What enzymes have been reported to be linked with severity of infection and various pathological conditions caused by microorganisms?', 'id': 3041, 'answers': [{'text': 'cysteine proteases', 'answer_start': 2276}], 'is_impossible': False}, {'question': 'At what temperatures was the assay completed?', 'id': 3042, 'answers': [{'text': '32uC, 37uC, 42uC', 'answer_start': 9753}], 'is_impossible': False}, {'question': 'What criteria sets the guideline for drug-like properties?', 'id': 3043, 'answers': [{'text': \"Lipinski's ''Rule of five''\", 'answer_start': 6881}], 'is_impossible': False}, {'question': 'What could be novel candidates as potent inhibitors of papain like cysteine proteases in resistant microorganisms?', 'id': 3044, 'answers': [{'text': '1-substituted pyridylimidazo[1,5-a]pyridine derivatives', 'answer_start': 22962}], 'is_impossible': False}], 'context': \"Design, Synthesis, Evaluation and Thermodynamics of 1-Substituted Pyridylimidazo[1,5-a]Pyridine Derivatives as Cysteine Protease Inhibitors\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC3734177/\\n\\nSHA: ee8483f8f2cc5fe38be4e565eae3af9d0bb8220b\\n\\nAuthors: Khan, Mohd Sajid; Baig, Mohd Hassan; Ahmad, Saheem; Siddiqui, Shapi Ahmad; Srivastava, Ashwini Kumar; Srinivasan, Kumar Venkatraman; Ansari, Irfan A.\\nDate: 2013-08-05\\nDOI: 10.1371/journal.pone.0069982\\nLicense: cc-by\\n\\nAbstract: Targeting papain family cysteine proteases is one of the novel strategies in the development of chemotherapy for a number of diseases. Novel cysteine protease inhibitors derived from 1-pyridylimidazo[1,5-a]pyridine representing pharmacologically important class of compounds are being reported here for the first time. The derivatives were initially designed and screened in silico by molecular docking studies against papain to explore the possible mode of action. The molecular interaction between the compounds and cysteine protease (papain) was found to be very similar to the interactions observed with the respective epoxide inhibitor (E-64c) of papain. Subsequently, compounds were synthesized to validate their efficacy in wet lab experiments. When characterized kinetically, these compounds show their K(i) and IC(50) values in the range of 13.75 to 99.30 µM and 13.40 to 96.50 µM, respectively. The thermodynamics studies suggest their binding with papain hydrophobically and entropically driven. These inhibitors also inhibit the growth of clinically important different types of Gram positive and Gram negative bacteria having MIC(50) values in the range of 0.6–1.4 µg/ml. Based on Lipinski’s rule of Five, we also propose these compounds as potent antibacterial prodrugs. The most active antibacterial compound was found to be 1-(2-pyridyl)-3-(2-hydroxyphenyl)imidazo[1,5-a]pyridine (3a).\\n\\nText: Cysteine-protease inhibitors (CPI) have gained considerable attention over the last couple of decades and many classes of compounds are currently in human clinical trials for a number of diseases. Interest in papain family cysteine proteases as chemotherapeutic targets is derived from the recognition that they are critical to the life cycle or pathogenicity of many microorganisms. The cysteine proteases from Streptococcus sp. (streptopain) [1] , Staphylococcus sp. (staphopain) [2] , Plasmodium falciparum (falcipain-1, -2, and -3) and Trypanosoma cruzi (cruzipain) [3] are some of the most widely studied members of papain family which have been reported to be linked with severity of infection and various pathological conditions caused by these microorganisms.\\n\\nThe activation of the kallikrein-kinin pathway, which could be activated by more than sixteen bacterial proteases, is a mechanism that some pathogens exploit to ensure that there is a supply of nutrients to the site of infection by increasing vascular permeability. This has been shown to occur in infections with several microbial species, including Pseudomonas, Serratia, Clostridium, Candida, Bacteroides, Porphyromonas and Staphylococcus sp. [4] . Many bacteria secrete several nonspecific proteases e.g. Pseudomonas, Serratia, Streptococcus, Staphylococcus and Bacteroides sp. have potent metallo-, cysteine and serine proteases with broad ranges of activities [5] . The critical role of bacterial proteases in virulence was successfully demonstrated by eliminating the proteaseencoding gene in P. gingivalis [6] .\\n\\nRecently described cystatin superfamily of proteins comprises both eukaryotic and prokaryotic cysteine protease inhibitors [7] . Human cystatins C, D and S, rat cystatins A and S, chicken cystatin and oryza cystatin have been reported to inhibit the replication of certain viruses and bacteria [8] although it has not yet been directly demonstrated that these effects are due to the protease inhibitory capacity of the cystatins [9] . The key role of cysteine proteases in microbial infections, coupled with the relative lack of redundancy compared to mammalian systems has made microbial proteases attractive targets for the development of novel chemotherapeutic approaches [10, 11] .\\n\\nImidazopyridine ring systems represent an important class of compounds not only for their theoretical interest but also from a pharmacological point of view. They have been shown to possess a broad range of useful pharmacological activities [12] including antigastric, antisecretory, local anesthetic, antiviral, antianxiety, antibacterial, antifungal, antihelminthic, antiprotozoal, anticonvulsant, gastrointestinal, antiulcer (Zolmidine), anxiolytic (Alpidem), hypnotic (Zolpidem) and immunomodulatory [13] . The nature and the position of the substituents on the pyridinic moiety influence these pharmacological activities. These imidazopyridine heterocyclic structures form part of the skeleton of natural alkaloids, neuromuscular blocking agents [14] , reversible inhibitors of the H + , K + -ATPase enzymes with a potent antisecretory activity, and are known to be sedative hypnotics of the nervous system [15] . In this study, we have proposed kinetically and thermodynamically characterized 1-substituted pyridylimidazo[1,5-a]pyridine derivatives as a potent and novel cysteine protease inhibitors which also acts as antibacterial agents. \\n\\nThe crystal structure of papain was extracted from Protein Data Bank (PDB code: 1PE6) [16] . All the water molecules and heteroatoms were removed and hydrogen atoms were added to the protein. CharMm forcefield [17] was applied and the structure was subjected to energy minimization for 1000 steps using steepest descent method. The chemical structures of all the synthesized compounds were generated using chemdraw and were subsequently converted into 3D format using CORINA. A series of docking experiments were carried out with all the designed 1substituted pyridylimidazo[1,5-a]pyridine derivatives against papain using AutoDock Tools 4.0 [18] for possible cysteine-protease inhibitory activities. The compounds were selected on the basis of their binding energies and those reflecting good binding affinity were further analyzed on in silico platform. As a parameter for the molecular docking, the Lamarckian genetic algorithm, a combination between the genetic algorithm and the local search Pseudo-Solis and Wets algorithm, was employed. A grid box of 60660660 Å was generated around active site of papain making sure those inhibitors can freely rotate inside the grid. The number of docking runs was set to 10. Each docking was repeated five times, having in the end a total of 50 docking runs, to check the precision of results. The finally obtained docked complexes were subsequently visualized using PyMol [19] . The work was further authenticated in the wet lab after its detailed analysis on in silico platform.\\n\\nThe designed derivatives were filtered by Lipinski's ''Rule of five'' that sets the criteria for drug-like properties. Drug likeness is a property that is most often used to characterize novel lead compounds [20] . According to this rule, poor absorption is expected if MW .500, log P.5, hydrogen bond donors .5, and hydrogen bond acceptors .10 [21] . In silico absorption, distribution, metabolism and excretion (ADME) properties of these derivatives were also predicted using following online bioinformatics tools.\\n\\nN http://www.organic-chemistry.org. N http://mobyle.rpbs.univ-paris-diderot.fr/cgi-bin/portal. py? Form = admetox N https://secure.chemsilico.com/pages/submit.php\\n\\nThe above study gave us an idea about the existence of possible mutagenic and tumorigenic properties in synthesized compounds. The result obtained helped us to screen out the synthesized compounds for their further usage as potent leads.\\n\\nBased on the results of docking studies, ten derivatives of 1pyridylimidazo[1,5-a]pyridine were synthesized according to Siddiqui et al., 2006 [22] which are named as follows: 1- \\n\\nThe capacity of the 1-pyridylimidazo[1,5-a]pyridine derivatives to inhibit cysteine proteases was tested using papain as the model enzyme. The proteolytic activity of the reaction mixtures was determined using Bz-DL-Arg-pNA as the chromogenic substrate [23] . To solutions of active papain (final concentration: 0.05 mM) were added concentrated solutions of the different derivatives to final concentrations of 0.2 mM. After incubation for 30 min at 37uC, the substrate solution was added and after a further incubation for 20 min the reaction was stopped by the addition of 5% trichloric acid (TCA) acidified with 2.25% HCl and the absorbance of the reaction mixture was determined at a wavelength of 410 nm by Microplate Manager 4.0 (Bio-Rad laboratories). The same procedure was used at 32uC and 42uC for thermodynamics studies. The kinetic parameters for the substrate hydrolysis were determined by measuring the initial rate of enzymatic activity. The inhibition constant K i was determined by Dixon method [24] and also by the Lineweaver-Burk equation. The K m value was calculated from the double-reciprocal equation by fitting the data into the computer software Origin 6.1. The Lineweaver-Burk plot was used to determine the types of inhibition. For the kinetic analysis and rate constant determinations, the assays were carried out in triplicate, and the average value was considered throughout this work. Temperature dependence of the inhibition constants was used to determine the thermodynamic parameters. Changes in enthalpy (DH) were determined from the Van't Hoff plots by using the equation,\\n\\nWhere DH is enthalpy change, R is gas constant, DS is entropy change and T is the absolute temperature. The entropy change was obtained from the equation,\\n\\nThe assay was done at different temperatures (32uC, 37uC, 42uC) calculating various K i of 1-pyridylimidazo[1,5-a]pyridine derivatives with papain as model enzyme.\\n\\nThe disk diffusion method [25] was used for the preliminary antibacterial evaluation of 1-pyridylimidazo[1,5-a]pyridine derivatives. The MIC 50 of these derivatives, showing inhibition in the preliminary tests, were further determined by the microtitre plate technique using micro dilution method [26] . In brief, the bacterial strains (S. aureus, P. vulgaris, Group D Streptococci, Bacillus sp., E. coli, P. aeruginosa and S. morganii)) were grown and diluted to 2610 5 colony-forming units (CFU)/ml in sodium phosphate buffer (SPB) containing 0.03% Luria-Bertani (LB) broth. The synthesized derivatives were dissolved in DMSO and their serial dilution was performed in 50 mL of LB medium in 96-well microtitre plate to achieve the required concentrations (0.1-10 mg/ml) with bacterial inoculums (5610 4 CFU per well). DMSO was taken as negative control and Ceftriaxone and clotrimazole were taken as positive control. After incubation at 37uC overnight, the MICs were taken as the lowest inhibitor concentration at which the bacterial growth was inhibited. The average of three values was calculated and that was the MIC for the test material and bacterial strain.\\n\\nFor the agar plate count method [27] , 25 mL aliquots of bacteria at 1610 5 CFU/ml in SPB containing 0.03% LB broth were incubated with 25 mL of diluted compounds for 2 h at 37uC. The mixtures of bacteria and compounds were serially diluted 10-fold with SPB and plated on LB plates that were incubated at 37uC overnight. Bacterial colonies were enumerated the following day.\\n\\nAfter having determined the MICs, bacterial strains from the wells of the microtitre plate with no visible bacterial growth were removed for serial sub cultivation of 2 ml into another new microtitre plate containing 100 ml of broth per well and further incubated for 24 h. The lowest concentration with no visible growth was defined as MBC [28], indicating 99.5% killing of the original inoculum. The absorbance of each well was measured at a wavelength of 620 nm by Microplate Manager 4.0 (Bio-Rad laboratories) and compared with a blank. Solvent (DMSO) was used as a negative control. Three replicates were done for each compound and experiment was repeated two times.\\n\\nBacteria use their cysteine proteases for pathogenecity as could be depicted from the structure of Cif homolog in Burkholderia pseudomallei (CHBP) which reveals a papain-like fold and a conserved Cys-His-Gln catalytic triad [29] . It has been proven that bacterial pathogens have a unique papain-like hydrolytic activity to block the normal host cell cycle progression as the core of an avirulence (Avr) protein (AvrPphB) from the plant pathogen Pseudomonas syringae, resembles the papain-like cysteine proteases. The similarity of this AvrPphB protein with papain includes the catalytic triad of Cys-98, His-212, and Asp-227 in the AvrPphB active site [30] .\\n\\nTurk et al. have proposed, on the basis of kinetic and structural studies, that papain has seven subsites at the active site but only five subsites are important which can bind to an amino acid residue of the substrate [31] . A variety of intermediates are generated when papain reacts with substrate or an inhibitor [2] . Like serine proteases, cysteine proteases tend to have relatively shallow, solvent-exposed active sites that can accommodate short substrate/inhibitor segments of protein loops (e.g. from endogenous inhibitors such as cystatins) or strands. The inhibitor Table 3 . Name, Structure, IC50 & K i of 1-substituted pyridylimidazo[1,5-a]pyridine derivatives against cysteine protease papain.\\n\\nType of inhibition Ki (mM) IC 50 (mM)\\n\\nNon-Competitive 13.7 13.4 compound bound to protease with a combination of hydrogen bonds and hydrophobic interactions. As a part of our investigation in developing novel and efficient cysteine protease inhibitors, ten 1-substituted pyridylimidazo [1,5a] pyridine derivatives (3a-j) were primarily designed and screened on the basis of their docking energies against papain to elucidate their possible mode of action. It was found that these compounds were specific inhibitors of cysteine protease, papain and didn't show inhibition against other types of proteases like serine, aspartic or metalloproteases. They are specific for CA clan of cysteine protease and didn't show any significant inhibition against other clans of cysteine proteases.\\n\\nThese new compounds were devised based on the knowledge of ability of a protein to alter its conformation to accommodate a binding ligand and enabled us to directly compare the relative positions of the residue in the binding pocket. Molecular docking study provided the structural insight into the binding of these compounds (3a-j) (Figure 1 ) within the active site of papain which mainly consist of a catalytic triad of Cys 25, His 159 and Asp 175 [32] . Moreover, role of other residues present in the active site of papain, playing important role in the accommodation of compounds have also been revealed. Initially, docking was performed with all the designed compounds (3a-j) against papain, a known cysteine protease enzyme and in this context, we observed very interesting results where our proposed inhibitors (3a-j) take advantage of aromatic and hydrophilic residues by making a variety of interactions with target enzyme. Although, compounds 3e-j gave significant results when docked with papain but during evaluation of antibacterial properties in wet lab experiments, they gave insignificant results (data not shown). Therefore, only four compounds were considered for discussion and further experiments like kinetic and thermodynamic studies to characterize these compounds as potent pro-inhibitors, were performed (3a-d).\\n\\nThe findings of the above study have shown that the molecular interactions between the compounds 3a-d and papain were very similar to the interactions observed for E-64c, a derivative of naturally occurring epoxide inhibitor (E-64c) (Figure 1 ) of cysteine proteases [31, 32] , with papain; especially with regard to the hydrogen bonding and hydrophobic interactions of the ligands with conserved residues in the catalytic binding site (Figure 2 A-D). Several papain residues participated in hydrophobic interactions with compounds 3a-d, including Gln19, Cys25, Gly66 and Asp158. The pyridine moieties of compounds 3a-d interact with S2 site of papain which includes (Tyr61, Asn64, Gly65 & Tyr67) amino acids (Figure 2 A-D) . The active site residues that were found to be key player in the interaction of compounds within the active site (mostly through hydrophobic interactions) were Cys25, Tyr61, His159 and Trp177, while Trp177, Gln19 were found to me making hydrogen bonds only with compound 3a. Besides this many other residues were also found to be actively involved ( Table 1) . Furthermore, the binding energies for the compound 3a, 3b, 3c and 3d with papain were found to be 26.12, 25.76, 26.84 and 25.62 Kcal/mol respectively, which were in great agreement with our wet lab experiments; shall be discussed later ( Table 1) . This confirmed the accuracy of our docking protocol. Since, the binding energy is a direct measure of strength of interaction and our compounds 3a-d showed stronger binding within the active site of papain in comparison to the inhibitor E-64c (DG: 24.04 Kcal/mol), therefore, the results suggest that these 1-substituted pyridylimidazo[1,5-a]pyridine derivatives (3ad) could be potent inhibitors of papain like cysteine proteases.\\n\\nThe in silico interaction of compounds 3a-d with papain, which were observed as discussed above, was validated with wet lab Table 5 . Prediction of antibacterial compounds as drugs (http://www.organic-chemistry.org). Table 2) . Interestingly, the observed in silico binding energies for the compounds 3a-d against papain were found to be in great agreement (standard error 62 Kcal/mol) with the value of free energy of binding (DG) observed during thermodynamics studies ( Table 1 and 2) . Similarly, enthalpy change (DH) of the binding was negative whereas entropy (DS) change of the binding was positive which indicated the exothermic and entropically driven nature of binding. This pattern of temperature dependence is characteristic of hydrophobic interaction [33] . As discussed earlier that all the compounds (3a-d) were found to interact with the active site residues of papain through hydrophobic interactions at most instances during in silico studies, the same was observed by the analysis of Van't Hoff plots for all the proposed inhibitors at three different temperatures (32uC, 37uC and 42uC) in wet lab experiments ( Figure 3) . This proves the importance of these types of interactions in the positioning of compounds within the active site. Hence, thermodynamics as well as in silico study reveals that hydrophobic interactions favor binding of these proposed inhibitors with papain like cysteine proteases. Further wet lab results proposed the non competitive interaction of compounds (3a, 3c & 3d) with papain except for compound 3b which showed competitive interaction. In sum up, the above results of molecular docking studies and thermodynamic analysis of compounds 3a-d with papain showed that these compounds have the potential to be novel and unique cysteine protease inhibitors.\\n\\nIn the current study, the cysteine protease inhibitory activity of synthesized derivatives of 1-substituted pyridylimidazo[1,5-a] pyridine (3a-d)) was also performed against papain and the inhibition constants (K i ) for the above said enzyme were observed to be 13.70, 23.20, 90.00 and 99.30 mM for compounds 3a, 3b, 3c and 3d respectively ( Table 3) . Furthermore, the calculated IC 50 values were also found to be 13.40, 21.17, 94.50 and 96.50 mM for compounds 3a, 3b, 3c and 3d respectively ( Table 3) . Except compound 3b, rest of the compounds showed non competitive, reversible inhibitions but all the compounds irrespective of types of binding, showed hydrophobic and entropically driven interaction. These derivatives (3a-j) were eventually evaluated for their antibacterial activities against seven clinically important microbes (S. aureus, P. vulgaris, Group D Streptococci, Bacillus sp., E. coli, P. aeruginosa and S. morganii). Here, we are showing the data of only four compounds (3a-d) because of their significant results ( Table 4 ). All the compounds strictly followed the pattern of antiprotease activity towards bacterial growth except P. vulgaris and E. coli at one instance each (Table 4) . Since compound 3c & 3d do not have much difference in their IC50 values (3c-94.5 mM and 3d-96.5 mM) against cysteine protease, papain and hence in antibacterial activity in all instances except one. It might be random due to so close in IC50 values. Compounds 3c & 3d are having much difference in their IC50 values (3b-21.17 mM and 3c-94.5 mM) and they showed exact pattern for their antibacterial activity for all microbes except for E. coli at one instance. Although, E. coli does contain six major cysteine proteases but none belong to the CA clan of papain. It is argued that these compounds also inhibited the cysteine proteases of other clan than papain but with low efficacy.\\n\\nSince, pyridylimidazo[1,5-a]pyridine derivatives is absolutely new scaffold towards antibacterial agents and hence, not any standard compound(s) of same scaffold is available for reference. So, Clotrimazole (1-[(2-chlorophenyl)(diphenyl)methyl]-1H-imidazole), an imidazole derivatives and Ceftriaxone (third-generation cephalosporin antibiotic with broad spectrum activity against Gram-positive and Gram-negative bacteria) have been used as positive control whereas DMSO has been used as negative control. All the above mentioned bacterial species have been shown to secrete certain cysteine proteases which play very important role in the pathogenecity of different diseases caused by these microorganisms. The minimum inhibitory concentration (MICs) of compounds (3a-d) ( Table 4 ) against all tested bacteria except E. coli and P. vulgaris, were observed to be in great agreement with their respective inhibition constant (K i )/IC 50 values against papain (Table 3 ) which clearly indicates that these compounds have the potential to inhibit the papain like cysteine proteases of these pathogens. The partition coefficient (logP) is a well-established measure of the compound's lipophilicity. The distribution of calculated logP (cLogP) values of a majority of drugs in the market is in the range of zero to five. All the compounds studied except 3d, showed good agreement with the criteria laid down for the prediction of a compound to be a potential drug ( Table 5 ). All the compounds do not show any threat against toxicity risk assessment except compound 3d which showed threat as tumorogenic effect due to the presence of isobutyl group. Among all the tested compounds, compound 3a was the most potent whose MIC was the lowest among all the tested compounds and showed maximum drug score and positive values for drug likeness.\\n\\nIn summary, the results of the present study have established that 1-substituted pyridylimidazo[1,5-a]pyridine derivatives could be candidate for novel and potent inhibitors of papain like cysteine proteases, which play deleterious role in the progression of different diseases caused by diverse microorganisms. Therefore, this group of compounds could be the subject of future research to confront the challenges with resistant microorganisms that is a major threat globally.\\n\\nFile S1 Types of inhibitions with Ki (Compounds 3a-3d).\\n\\n(DOC)\", 'document_id': 1565}]}, {'paragraphs': [{'qas': [{'question': 'What method is useful in administering small molecules for systemic delivery to the body?', 'id': 1612, 'answers': [{'text': 'Intranasal', 'answer_start': 3238}], 'is_impossible': False}, {'question': 'Why is the nasal mucosa useful in the delivery of small molecules into the body?', 'id': 1615, 'answers': [{'text': 'the surface area can result in rapid absorption of the medication into the blood', 'answer_start': 3446}], 'is_impossible': False}, {'question': 'What are the most common methods of inhaled delivery of medications?', 'id': 1616, 'answers': [{'text': 'Metered dose inhalers (MDIs) and dry powder inhalers (DPIs)', 'answer_start': 4624}], 'is_impossible': False}, {'question': 'What medications have shown good promise to in vivo delivery via dry powder inhalers?', 'id': 1617, 'answers': [{'text': 'insulin [39] and low-molecular-weight heparin [40]', 'answer_start': 5473}], 'is_impossible': False}, {'question': 'How are siRNAs typically delivered for systemic effect?', 'id': 1618, 'answers': [{'text': 'intratracheal or intranasal delivery', 'answer_start': 5840}], 'is_impossible': False}, {'question': 'What structures form the human airway?', 'id': 1619, 'answers': [{'text': 'respiratory bronchioles, alveolar ducts, and alveolar sacs', 'answer_start': 6995}], 'is_impossible': False}, {'question': 'What size of particle has been shown to be most effective in the delivery to the lower airway?', 'id': 1620, 'answers': [{'text': '1-5 μm', 'answer_start': 7557}], 'is_impossible': False}, {'question': 'What are the essential conditions in siRNA delivery to effectively produce gene silencing in the lungs?', 'id': 1621, 'answers': [{'text': 'delivered to their site of action, be stable, enter the target cells, and be present in the cytoplasm at sufficient concentration', 'answer_start': 10025}], 'is_impossible': False}], 'context': 'RNAi Therapeutic Platforms for Lung Diseases\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC3816685/\\n\\nFujita, Yu; Takeshita, Fumitaka; Kuwano, Kazuyoshi; Ochiya, Takahiro\\n2013-02-06\\nDOI:10.3390/ph6020223\\nLicense:cc-by\\n\\nAbstract: RNA interference (RNAi) is rapidly becoming an important method for analyzing gene functions in many eukaryotes and holds promise for the development of therapeutic gene silencing. The induction of RNAi relies on small silencing RNAs, which affect specific messenger RNA (mRNA) degradation. Two types of small RNA molecules, i.e. small interfering RNAs (siRNAs) and microRNAs (miRNAs), are central to RNAi. Drug discovery studies and novel treatments of siRNAs are currently targeting a wide range of diseases, including various viral infections and cancers. Lung diseases in general are attractive targets for siRNA therapeutics because of their lethality and prevalence. In addition, the lung is anatomically accessible to therapeutic agents via the intrapulmonary route. Recently, increasing evidence indicates that miRNAs play an important role in lung abnormalities, such as inflammation and oncogenesis. Therefore, miRNAs are being targeted for therapeutic purposes. In this review, we present strategies for RNAi delivery and discuss the current state-of-the-art RNAi-based therapeutics for various lung diseases.\\n\\nText: traditional surgery, Bivas-Benita et al. reported that no mortality occurred as a result of the use of the endotracheal technique. Endotracheal applications are currently being used by many practitioners in the pulmonary field [22, 34] ; this is useful for studying pulmonary drug delivery in mice. However, the approach is more complex in humans because an artificial path for the delivery of drugs into the lungs is used. Therefore, the method is being used in animal models to test and evaluate its reliability for possible clinical applications. Intratracheal route: under anesthesia, the trachea is exposed surgically, and a tube or needle is inserted through an incision made between the tracheal rings. Complications, such as vascular injury and air leakage, are possible due to the tracheotomy. (b) Endotracheal route: siRNAs are sprayed directly from the mouth into the lungs using a MicroSprayer ® aerolizer (Penn-Century, Philadelphia, PA, USA) and a laryngoscope. It is important to maintain a clear view of the trachea during the procedure.\\n\\nIntranasal delivery is another common method of pulmonary drug application in animal studies. In many studies, in vivo success has been demonstrated in delivering siRNAs to the lungs intranasally [22, 35, 36 ]. An experimental setup of intranasal delivery by spray or droplet is simple and painless for the animal. Although the success in delivering siRNAs intranasally in rodents cannot be completely extrapolated to human use because of the significant differences in lung anatomy [37] , this approach has potential for the clinical application of siRNAs. Phase II clinical trials have been initiated for the treatment of respiratory syncytial virus (RSV) infection, making use of intranasal application of naked chemically modified siRNA molecules that target viral gene products [17, 38] (see Section 3.1.1. for details).\\n\\nIntranasal entry has long been used to administer small molecules, such as proteins, for systemic delivery. Because the nasal mucosa is highly vascularized, delivery of a thin epithelium of medication across the surface area can result in rapid absorption of the medication into the blood. Therefore, siRNAs administered intranasally might be deposited in the nose, and some of them may be unable to reach the lower respiratory tract. In fact, it has been reported that intranasal application of unformulated siRNAs resulted in lower delivery efficiency and homogeneous pulmonary distribution than that achieved with intratracheal application [31] . The intranasal method is suitable for some lung diseases, such as upper respiratory infection by RSV, and it also has potential for systemic delivery rather than pulmonary delivery of siRNAs. Therefore, it is important to consider the route of administration in animal studies when assessing the delivery and therapeutic efficacy of a formulation for pulmonary delivery. Careful choice of efficient delivery in response to the condition of lung diseases is necessary.\\n\\nThe use of aerosols to deliver medication to the lungs has a long history. Administration by inhalation is a popular and non-invasive method of delivering agents into the lungs. There are several inhalation devices available for the delivery of drugs into the lungs. Metered dose inhalers (MDIs) and dry powder inhalers (DPIs) are the most common modes of inhaled delivery. MDIs are the most commonly used inhalers for several lung diseases, such as asthma, bronchitis, and chronic obstructive pulmonary disease (COPD), and a spacer is an external device that is attached to an MDI to allow for better drug delivery by enhanced actuation and inhalation coordination. For most MDIs, the propellant is one or more gases called chlorofluorocarbons (CFCs). Although CFCs in drugs are safe for patients to inhale, they are harmful to the environment. Therefore, further development of inhalable siRNAs may not be the best way forward. DPIs are devices that deliver medication to the lungs in the form of dry powder. The use of DPIs has already shown promise for the in vivo delivery of therapeutic macromolecules such as insulin [39] and low-molecular-weight heparin [40] ; thus, it could be a better device for delivering siRNAs to the lungs. The advantages of DPIs are improved stability and sterility of biomolecules over liquid aerosols and propellant-free formation.\\n\\nAlthough drugs are commonly delivered to the lungs by inhalation, most in vivo studies using siRNAs have relied on intratracheal or intranasal delivery. The reason could be the difficulty in formulating inhalable siRNAs and maintaining the stability during the delivery process. A suitable carrier is also needed to protect nucleic acids from degradation due to shear force and increased temperature during the drying process. The use of spray-drying as a technique for engineering dry powder formulations of siRNA nanoparticles, which might enable the local delivery of biologically active siRNA directly to the lung tissue, has been demonstrated [24, 25] . In the future, the technique is desirable to estimate the in vivo study on siRNA therapy for inhalation. In the long term, we anticipate that there will be more sophisticated devices for clinical use and that those currently being developed will be more suitable.\\n\\nThere are two main barriers to efficient pulmonary siRNA delivery to the cells of the lung. The first is the complex, branched anatomy of the lungs and biomechanical barriers, such as the mucus layer covering the airway cells [41, 42] (Figure 2) . A remarkable feature of the respiratory tract is its high degree of branching. Airway consists of respiratory bronchioles, alveolar ducts, and alveolar sacs. All of these structures bear alveoli, the tiny air sacs in which the gas exchange takes place. It is generally acknowledged that the critical factor for efficient siRNA delivery depends on the properties of RNAi drug particles in terms of size, charge, shape, velocity and density. For efficient pulmonary siRNA delivery, the particles must be deposited in the lower respiratory tract. Deposition in the airway is affected by the particle size and patient\\'s pulmonary function. A particle size between 1-5 μm is found to be the most appropriate for deposition at the lower respiratory tract [23] . In addition, the presence of mucus and surfactant proteins, the mucociliary clearance actions, and phagocytosis by macrophages present major barriers to targeted pulmonary delivery. Therefore, delivery systems usually require delivery vectors, and these vectors need to be designed in order to maximize the siRNA deposition to the diseased area of the respiratory tract. Besides, the extracellular barriers to siRNA delivery also depend on physiological features of the respiratory tract, which may change with the disease stage and characteristics of the patient. At the active stage of lung disease, the physiological conditions of the airways might change and have significant impact on the efficiency of the pulmonary delivery system. During infection, inflammation, and allergic reaction, there is an increase in mucus secretion along with the impaired mucociliary clearance [43] . Moreover, asthma and COPD are both chronic inflammatory conditions of the lung associated with structural \"remodeling\" that is inappropriate to the maintenance of normal lung function [44] . The airway wall thickness, the high viscosity, and the composition of the mucus layer might be altered in patients who have inflammatory lung diseases. Figure 2 . Extracellular barriers to pulmonary siRNA delivery. The anatomical feature of the respiratory tract is its high degree of branching. The mucus lines the respiratory epithelium from the nasal cavity to the terminal bronchioles. The deposited particles on the ciliated epithelial cells are rapidly cleared by the mucociliary clearance actions. Mucus and mucociliary clearance of mucus-trapped particles is a pulmonary defense mechanism as a physiological barrier. In the alveolar, clara cells and type II alveolar cells secrete on the surface of the alveolar epithelium, forming a thin layer of pulmonary surfactants. The surfactants act as the main barrier for siRNA delivery because they reduce the transfection efficiency. In addition, the macrophages located in the alveoli rapidly engulf the foreign particles by phagocytosis. The particles taken up into the macrophages are subsequently degraded inside the cells. These factors present major barriers to targeted pulmonary delivery.\\n\\nThe second is the airway cell membrance and its intracellular barriers ( Figure 3 ). For efficient gene silencing in the lungs, siRNAs must be delivered to their site of action, be stable, enter the target cells, and be present in the cytoplasm at sufficient concentration. Once the siRNAs reach the target cells, they must be trafficked into the cytoplasm and taken up by Argonaute (Ago)2/RNA-induced silencing complex (RISC), which degrades mRNAs and, subsequently, suppresses the sequence-specific gene expression. For efficient endocytosis to occur, particles should be under 150 nm in size. Particles within this size range could also avoid macrophage uptake and delayed lung clearance [45] . The physicochemical properties of siRNAs also play a significant role in crossing the biological membrane. Despite their small size, the negative charge and chemical degradability of siRNA molecules prevent them from readily crossing biological membranes. Therefore, efficient siRNA delivery approaches need to overcome this limitation by facilitating cellular uptake. One of the main functions of a delivery vector is to facilitate the cellular uptake of siRNAs [46] . The electrostatic complexation of siRNA molecules with cationic lipids and polymers helps to mask their net negative charge. The positively charged siRNA carrier complex interacts with anionic proteoglycans on the cell membrance, forms an endocytic vesicle, and enters the cells by endocytosis [47] . After cellular internalization, the siRNA carrier complex in endocytic vesicles is transported along microtubules to lysosomes that are co-localized with the microtubule-organizing center. To avoid lysosomal degradation, siRNAs must escape from the endosome into the cytoplasm, where they can associate with the RNAi machinery. Endosomal escape is a major barrier for efficient siRNA delivery [48, 49] . The endosomal entrapment and lysosomal degradation of siRNA and carriers contribute to the low transfection efficiency and is a major difficulty for delivery vectors. An ideal delivery agent should protect siRNAs from enzymatic degradation, facilitate cellular uptake, and promote endosomal escape inside the cells with negligible toxicity.\\n\\nMultiple approaches for the delivery of siRNAs have been reported, ranging from the relatively simple direct administration of saline-formulated siRNAs to lipid-based and polymer-based nanoparticle approaches and siRNA conjugation and complexation approaches [50] . The negative charge and chemical degradability of siRNAs under physiologically relevant conditions make its delivery a major challenge. Accordingly, the delivery of siRNAs usually requires a vector or carriers for their transfection into the target cells. In general, both viral and non-viral vectors are being assessed for siRNA delivery to lung cells. Some viral vectors, such as retroviruses and adenoviruses, have been demonstrated to mediate gene silencing in an in vitro lung model [51] and to induce RNAi in a range of animal tissues [52] . Recently, Guo et al. showed that lentivirus-mediated siRNA was used to specifically knock down the expression of nuclear protein 1 (NUPR1) in vivo, which resulted in inhibited tumor growth [53] . However, viral-based delivery has several disadvantages. The immune response to viruses not only impedes gene delivery but also has the potential to cause severe complications [54] . Recent well-documented cases, such as the death of Jesse Gelsinger due to complications related with an adenoviral delivery vector, highlight this problem [55] . In addition, some viral vectors may insert their genome at random positions in the host chromosome, which eventually restrict the gene function [56] . . Intracellular barriers to pulmonary siRNA delivery. Barriers to cellular internalization are dependent on the surface properties of siRNA and carriers (e.g., charge and size). After siRNAs are successfully taken into the target cells by endocytosis, the main barriers for delivering siRNAs to its site of action are the endosomal entrapment and lysosomal degradation of siRNA and carriers. To direct target-gene silencing, the siRNAs need to escape from the endosome into the cytoplasm, where they associate with the Ago2/RNA-induced silencing complex (RISC) to direct the cleavage of mRNAs bearing complementary binding sites.\\n\\nAs an alternative to viral vectors, non-viral vectors, including lipid and polymer-based vectors, have been generally used for the delivery of siRNAs to the lungs due to their reduced toxicity [57] . Ongoing research into the transfection of primary cells and whole organisms with siRNA using non-viral transfection agents has produced some promising results. Lipid-based delivery vectors are successfully used to deliver siRNA in vitro and in vivo [58] . Cationic lipids are composed of positively charged head, a linker and hydrophobic. In general, lipid-based complexes are easy to formulate and good transfection efficacy is achieved due to interaction with negative charged cell membrance. Many commercial siRNA transfection agents are lipid-based delivery system, some of which are also employed for pulmonary delivery-DharmFECT [30] , Oligofectamine [59] , Lipofectamine [60] and TransIT-TKO [35] . Similarly, cationic polymers have also been assessed for siRNA delivery to lung cells. Cationic polymer polyethylenimine (PEI) is widely used for siRNA delivery [13, 61] . PEI is considered as the gold standard for in vitro gene delivery and its transfection efficiency depends on the molecular weight and degree of branching.\\n\\nOn the other hand, lipid-based vectors can also induce toxicity and non-specific activation of inflammatory cytokine and interferon responses [62, 63] . Although polymer-based vectors elicit a relatively less strong immune response than lipid-based vectors, effective siRNA delivery to a local area in lung diseases requires more attention to the development of non-toxic delivery vectors. An important point for siRNA-mediated inhibition of gene expression is whether the observed effects are specific rather than due to off-target effects and free from potential interferon responses [64, 65] . Interestingly, some studies have shown that it was possible to administer \"naked siRNAs\" to mice and down-regulate an endogenous or exogenous target without inducing an interferon response [66] .\\n\\nThe term \"naked siRNAs\" refers to the delivery of siRNAs without any delivery vectors. Naked siRNAs are degraded by serum endonucleases and are usually removed by glomerular filtration, resulting in a short plasma half-life of < 10 min. Thus, some studies of systemic delivery of naked siRNAs have failed to achieve the downregulation of the targeted gene [67, 68] . In contrast, there have also been some successes of locally delivering naked siRNAs to the lungs [15, 16, 20, 31] . A few of them reported that the use of delivery vectors showed no significant difference in gene silencing efficiency compared to that of naked siRNAs [16, 35] . Indeed, in one clinical trial, the delivery of naked siRNAs for the treatment of RSV has been used [17, 38] . This successful evidence can be because that naked siRNAs for clinical applications are highly chemically modified to prevent nuclease-induced degradation and presumably minimize immune stimulatory effects. Although it is unclear how the naked siRNAs cross the cell membrane, gain access to the cytoplasm, and remain intact to perform their biological action, both animal and human trials have been conducted successfully, showing the efficacy of naked siRNAs (ALN-RSV01) that were administered intranasally. This explanation has not been confirmed, but the physiological damage of respiratory epithelial cells caused by viral infection may have possibly influenced the mystery. The active change in airway epithelial cell membrance caused by infectious disease might affect cellular internalization. Naked siRNA delivery has some advantages, such as simple formation and the absence of toxicity or inflammatory responses that are usually associated with delivery vectors. Nevertheless, the advantage of naked siRNAs over delivery vectors in the treatment of lung diseases is controversial [69, 70] . Further in vivo investigations about both naked siRNAs and non-viral vectors are required.\\n\\nLung disease is a major cause of death, and diminished quality of life is responsible for the suffering of many patients. Various lung diseases make life extremely difficult for the patients, and severe cases of these lung diseases can result in death. The high death rates associated with lung cancer are partially due to the fact that it is unfortunately difficult to cure. Above all, COPD is the fourth-leading cause of death in most industrialized countries and is predicted to become third by 2020 [71] . Therefore, decisive action is needed to stem the rising health and economic burden this represents. Chronic lung diseases, such as COPD and asthma, are disorders of the airways largely related to the presence of persistent inflammation. The approval of inhaled corticosteroids pioneered a new generation of therapy in treating chronic inflammatory diseases. This was the first time that an anti-inflammatory product was available to reduce the characteristic lung inflammation in airways and the associated obstruction. Corticosteroids are still an important therapeutic intervention. However, they are used with limitations in COPD and moderate to severe asthma. Likewise, the treatment of various refractory lung diseases also depends on systemic corticosteroid therapy. Many of these patients also suffered various side effects from systemic corticosteroid use, such as weight gain and uncontrolled hyperglycemia. Treatment of lung disease using cell-specific targeting as well as RNAi techniques represents a novel strategy and could possibly provide new opportunities in nanomedicine. Pulmonary applications of siRNA in in vivo conditions are frequently studied and often result in clinical trials [57, 72] . The findings of recent clinical studies of pulmonary RNAi therapeutics are discussed.\\n\\nSince the discovery of RNAi, the therapeutic potential of siRNAs has been rapidly recognized. In 2004, the first human clinical trial of RNAi-based therapy was initiated for the treatment of age-related macular degeneration with a siRNA targeting VEGF-receptor 1 delivered intravitreally [73] . Many studies have been conducted over the past few years that involve the delivery of siRNAs to the lungs for the treatment of various lung diseases. Delivery to the lungs will be most important to moving siRNA technology into the clinic. A number of siRNA-based therapies are being evaluated in clinical trials for the treatment of different conditions, including lung diseases such as asthma and RSV infection. Table 1 is a summary of clinical trials of siRNA-based therapeutics [74] . \\n\\nSiRNA shows potential for the treatment of various pulmonary viral infections, and it has been reported that siRNA-based therapeutics can also be used in the treatment of influenza [13] , parainfluenza virus [35] , severe acute respiratory syndrome (SARS) [14] , and RSV [35] . Above all, RSV is the most promising therapeutic target of siRNAs.\\n\\nRSV is a common cause of serious respiratory infections in infants and children. It also produces significant morbidity and mortality in adult immunocompromised or elderly populations [75] . An RSV vaccine is not available, and the only approved antiviral therapy for RSV is undesirable for pediatric patients due to its potential teratogenicity and limited effectiveness. Thus, a safe and efficacious RSV therapy has long been awaited for both pediatric and adult patients. RNAi-based therapy has shown promising effects in murine models of RSV infection [35] . The siRNA, ALN-RSV01, is directed against the mRNA encoding the N-protein of RSV that exhibits specific in vitro and in vivo anti-RSV activity. It is delivered without a delivery vector as a nasal spray and targets the upper respiratory tract instead of the lower lung area. ALN-RSV01 has undergone complete phase I intranasal and inhalation studies in healthy adults and has been found to be generally well tolerated [38] . Additionally, ALN-RSV01 has been evaluated in a randomized, double-blind, placebo-controlled phase II trial in lung transplant patients with RSV respiratory tract infection [76] . The administration of ALN-RSV01 to RSV infected lung transplant patients was safe and well tolerated and associated with a statistically significant improvement in symptoms. Based on these results, a larger multinational, randomized, double-blind Phase IIb trial of ALN-RSV01 has been initiated in lung transplant patients to confirm and extend these findings.\\n\\nCancer is a major target of RNAi-based therapy, as oncogenes, mutated tumor suppressor genes, and several other genes contributing to tumor progression are potentially important targets for gene silencing by RNAi. Lung cancer is one of the most frequent tumors worldwide with regard to incidence rates and mortality. Patients with lung cancer are commonly diagnosed at an advanced stage of the disease and have limited therapeutic options. Although the knowledge regarding the genetic and molecular basis of lung cancer has regularly increased, the median survival rates of individuals with advanced lung cancer are still poor.\\n\\nRNAi-based therapy is an attractive strategy for the development of more effective anticancer therapies with reduced treatment-related toxicity. The major advantage of RNAi therapeutics in cancer might be the simultaneous targeting of multiple genes belonging to different cellular pathways that are involved in tumor progression. The simultaneously inhibition of several genes would also minimize the risk of drug resistance normally encountered with small molecule-based therapies, involving siRNAs and miRNAs. There have already been significant improvements in siRNAs for primary or metastatic lung cancer treatment by targeting oncogenes such as Akt1 [9] , Wilms tumor 1 (WT1) [12] , overexpressed genes such as the insulin-like growth factor receptor 1 (IGF-1R) [77] , NUPR1 [53] and EZH2 [78] . Some of these studies have successfully shown the efficacy of RNAi-based therapy through intrapulmonary administration of siRNAs with non-viral vectors. Although strategies to minimize off-target and nonspecific immune stimulatory effects must be devised, these data suggest that the silencing of the target gene with siRNAs is an attractive strategy for the prevention and treatment of primary and metastatic lung cancer. There are currently some clinical trials in progress estimating the safety and efficacy of siRNA-based drugs for cancer treatment. Atu027, a siRNA-lipoplex targeted against protein kinase N3 (PKN3), prevented lung metastasis in a phase I trial of various cancer models [79] . PKN3 is a downstream effector of the phosphoinositide 3-kinase (PI3K) signaling pathway [80] , which regulates diverse cellular responses, including development, growth, and survival [81] . Recently, PKN3 has also been considered as a suitable therapeutic target for modulating tumor angiogenesis because loss of function analysis with Atu027 in cultured primary endothelial cells showed an essential role of PKN3 for endothelial tube formation and migration [79] . Atu027 can be considered as a potential siRNA for preventing lung metastasis and might be suitable for preventing hematogenous metastasis combined with conventional cancer therapy.\\n\\nInflammatory lung disease, also called COPD, includes a wide range of lung ailments. These related diseases include asthma, pulmonary fibrosis, and chronic bronchitis. They are influenced by a combination of environmental, genetic, and epigenetic components [82] . COPD is a chronic inflammatory disease of the airways. This disease is hallmarked by airflow that is not fully reversible. Systemic and local airway inflammation has been implicated in the pathogenesis of COPD [83] . COPD is mainly associated with tobacco smoking, and recent studies investigating the pathophysiology of emphysema have demonstrated that cigarette smoke can cause cells to enter cellular senescence.\\n\\nSmoking might cause cells to senesce due to DNA damage through increased cell turnover, which in turn leads to accelerated telomere shortening [84] . Lately, a lot of studies have investigated the role of cellular senescence in the development and progression of COPD [85] . Although several medication classes, including inhaled corticosteroids, are used for COPD treatment, none of these medications have been shown to significantly improve long-term lung function during the progression of the disease. Current interventions that have been shown to improve mortality in COPD are cessation of smoking and delivery of supplemental oxygen when hypoxemia is present.\\n\\nMany people are developing COPD, and the cause of this condition is complicated and not thoroughly understood. One key factor is genetic susceptibility. Some studies have shown a large genetic contribution to the variability in pulmonary function and COPD [86, 87] . Polymorphisms in multiple genes have been reported to be associated with COPD [87] , such as transcription factor [e.g. nuclear factor-kappa B (NFκB)] [88] , extracellular matrix (e.g., matrix metalloproteinase-12 (MMP-12)) [89, 90] , cytokines [e.g. tumor necrosis factor (TNF)-α] [91] , chemokines [e.g. interleukins (IL)-8, IL-8 receptor and chemokine receptor (CCR)1] [92, 93] , and apoptosis (e.g., caspase-3 and vascular endothelial growth factor (VEGF)) [94, 95] . Many of these have been identified as possible targets for therapeutic intervention using molecule inhibitors or antagonists. Although several new treatments that target the inflammatory process are now in clinical development, such as TNF-α inhibitors and I-kappaB kinase complex 2 (IKK2) inhibitors [96, 97] , clinical trials with siRNAs have never been performed in COPD. The delay of drug development for COPD might be due to the relatively recent emergence of research addressing the molecular basis of COPD. Furthermore, more research is needed to understand the essential molecular mechanisms about the pathogenesis of COPD and to develop monitoring techniques to support the development of RNAi therapies. Currently, no available treatments reduce the progression of COPD or suppress the inflammation in small airways and lung parenchyma. The RNAi-based approach for the key molecules also has potential implications for the treatment of COPD.\\n\\nAsthma is also a chronic inflammatory disease of the airways characterized by variable and recurring symptoms and reversible airflow obstruction. The World Health Organization estimates that 300 million people are currently affected and that, by the year 2025, another 100 million will be affected by the disease [98] . Inhaled corticosteroids are very effective in mild asthma because they improve symptoms and decrease exacerbations. However, in moderate and severe asthma, inhaled corticosteroids have important therapeutic limitations. Although corticosteroids remain an important therapeutic intervention for inflammatory lung diseases, their use is not always completely effective and is associated with side effects. Due to such limitations, it is clear that there is a need for new types of medications that can treat and improve the prognosis of moderate to severe asthma.\\n\\nMany target genes have been identified that participate in the pathogenesis of asthma. The most promising targets include genes coding for cytokines (IL-4, IL5, and IL-13), cytokine and chemokine receptors (IL-4 receptor and CCR3), and tyrosine kinases [spleen tyrosine kinase (Syk) and LCK/YES-related novel tyrosine kinase (Lyn)], as well as for transcription factors [signal transducers and activators of transcription 1 (STAT1), STAT6, GATA3, and NFκB] that are involved in asthma [19, 99, 100] . The genes that have been assessed as siRNA targets for the treatment of asthma in preclinical models are reported [101] . Currently, in a clinical trial for asthma, Excellair TM (ZaBeCor, Bala Cynwyd, PA, USA), a siRNA that targets Syk, is being used. The kinase is involved in signaling from a B cell receptor and is a key regulator of downstream signaling cascades that ultimately lead to the activation of several pro-inflammatory transcription factors. It has been reported that antisense oligonucleotides administered by aerosol were potent to decrease Syk expression, mediator release from alveolar macrophages, and Syk-dependent pulmonary inflammation [102] . Moreover, inhibition of inflammatory mediators was shown in a study using siRNA targeting Syk in airway epithelial cells [103] . Following the successful results of the company\\'s Phase I clinical trial, a Phase II trial for its asthma drug candidate Excellair TM has already been initiated. Some of the current treatments for asthma and other inflammatory conditions, such as TNF-α inhibitors or leukotriene inhibitors, inhibit only one of the mediators of inflammation. In contrast, siRNA targeting Syk seeks to inhibit an initial signaling step of inflammation and, thereby, prevent the release of multiple inflammatory mediators. Overall, recent progress of siRNAs to the lungs has also improved the therapeutic feasibility of RNAi for inflammatory lung diseases. The rapid progress will put siRNA-based therapeutics on a fast track to the clinic.\\n\\nMiRNAs are small endogenous noncoding RNAs that regulate gene expression by repressing translation or promoting the degradation of their target mRNA. MiRNAs regulate gene expression by binding to the 3′ untranslated region (UTR) of their target mRNAs and mediating mRNA degradation or translational inhibition. In the human genome, transcripts of approximately 60% of all mRNAs are estimated to be targeted by miRNAs [104] . According to their function, miRNAs play an important role in cellular processes as development, proliferation, and apoptosis of pulmonary pathologies [105] . A growing number of miRNAs have been shown to be involved in different lung diseases. This evidence makes miRNAs a promising technology for current and future therapeutic development. We discuss the role of some miRNAs in various lung diseases as well as the possible future of these discoveries in clinical applications. Table 2 shows the summary of miRNAs in therapeutic development. At this point, a miRNA-based therapy has already entered a phase II clinical trial. \\n\\nThere is evidence that upregulation or downregulation of miRNAs is critical for lung homeostasis and, thus, may contribute to the development of pathological pulmonary conditions. Many studies have focused on the role of miRNAs in inflammatory lung diseases, such as COPD [116, 117] , pulmonary fibrosis [118] [119] [120] [121] , and asthma [122] [123] [124] [125] (Table 3) . [130] [117, 129] The pathogenesis of COPD is attributed to not only chronic inflammation in the airways but also systemic inflammation [131] . Cigarette smoking is the main risk factor for the development of COPD.\\n\\nSmoking has been shown to cause biological change in the gene expression of the lungs [132] , and there are some reports about smoking-related miRNAs [117, 129, 130] . However, there are few reports that focus on the miRNAs related to the pathogenesis of this disease with systemic inflammatory components. Recent study on pulmonary fibroblasts of COPD patients presents less expression of miR-146a after stimulation with proinflammatory cytokines when compared with non-COPD subjects with similar smoking histories [127] . The downregulation of miR-146a resulted in a prolonged mRNA half-life of cyclooxygenase-2, thus increasing prostaglandin E2 in fibroblasts from COPD subjects.\\n\\nMoreover, Ezzie et al. researched the difference of miRNA profiles expressed in the lungs of smokers with and without COPD. They concluded that miR-223 and miR-1274a were the most affected miRNAs in subjects with COPD [126] . Yet, COPD is a complex, multi-component, and heterogeneous disorder with a number of different pathological processes and subgroups with their own characteristics and natural history [133] . A better understanding of the complexity of the disease and potential clinical relevance of the identified miRNAs is needed.\\n\\nPulmonary fibrosis can be caused by an identifiable irritation to the lungs, but, in many cases, the cause is unknown, and the therapeutic possibilities are limited. Cigarette smoking is one of the most recognized risk factors for the development of pulmonary fibrosis. This disorder is mainly accompanied by increased expression of the key fibrotic mediator transforming growth factor β (TGF-β) and other cytokines produced at the lesion of active fibrosis [128] . Recently, it was reported that miRNAs may play an important regulatory role in the pulmonary fibrotic change in the lungs. The downregulation of let-7d in idiopathic pulmonary fibrosis (IPF) resulted in increased collagen deposition and alveolar septal thickening [119] . In addition, Liu et al. reported that the oncogenic miR-21 was found to be upregulated in IPF patients and in the murine lungs with bleomycin-induced fibrosis [118] . Although these miRNAs may be potential therapeutic targets because their expression is related to the regulation of TGF-β, the factor is necessary but not sufficient for pathologic fibrosis of the lungs. Pulmonary fibrosis is also a complicated illness that can have many different causes.\\n\\nFocus on the role of miRNAs in asthma has recently increased. Asthma is an inflammatory disease of the airway that is characterized by an abnormal response of T helper-2 (Th2)-type CD4+T lymphocytes against inhaled allergens [134] . In a different asthmatic mouse model, there was an observed increase in the expression of miR-21 in the lungs [123] . This report might contribute to the understanding of the inflammatory mechanism in the airway through the inhibition of IL-12, favoring the Th2 lymphocyte response. A toll-like receptor 4 (TLR4)-induced Th2 lymphocyte induces high expression of miR-126, and selective blockade of miR-126 suppressed the asthmatic phenotype [124] . In addition, airway remodeling is a characteristic feature of asthma and has important functional implications. Rodriguez et al. have shown that miR-155 is related to the development of inflammatory infiltration into the lung and airway remodeling [122] . Thus, some studies present a functional connection between miRNA expression and asthma pathogenesis and suggest that targeting miRNAs in the airways may lead to anti-inflammatory treatments for allergic asthma. Despite the evidence from experimental models, the expression profiling of miRNAs in airway biopsies from patients with mild asthma before and after treatment with inhaled corticosteroids and in healthy volunteers revealed no differences in miRNA expression [135] . Further investigations about the role of miRNAs related to asthma pathogenesis are required.\\n\\nAlthough the basic evidence of miRNA biology is still providing new insights, applications of miRNA-based therapy for inflammatory lung diseases are less advanced than those for lung cancer [136] . One reason for this could be that the disease heterogeneity is caused by the effects of many environmental air pollutants, including smoke and volatile organic compounds. The presence of several risk factors makes the understanding of the pathogenesis of inflammatory lung diseases complicated. Understanding the role that miRNAs play in the modulation of gene expression, leading to sustain the pathogenesis of lung diseases, is important for the development of new therapies that focus on the prevention of disease progression and symptom relief.\\n\\nGiven the significant roles that miRNAs play in multiple pathways of lung carcinogenesis, increasing efforts are dedicated to the research and development of miRNA-based therapies, including restoring functions of tumor suppressive miRNAs or inhibiting oncogenic miRNAs. The development of miRNA-based therapies for lung cancer is growing prosperously with the help of new RNAi technologies. Compared to siRNA-based therapies, which are already in clinical trials, miRNAs are less toxic and have the potential to target multiple genes. The difficulty associated with miRNA delivery is mainly equal to that of siRNAs. The critical problems for the development of this therapy are effective delivery into target sites, potency of the therapy, and elimination of off-target effects [137] .\\n\\nThere are two strategies as the therapeutic applications of miRNAs for lung cancer [138] . One strategy is miRNA replacement therapy, which involves the re-introduction of a tumor suppressor miRNA mimic to restore a loss of the function. MiRNA mimics are synthetic RNA duplexes designed to mimic the endogenous functions of miRNAs with chemical modifications for stability and cellular uptake. The concept of miRNA replacement therapy is most exemplified by the let-7 miRNA. let-7 is a tumor-suppressor miRNA in non-small-cell lung cancer that inversely correlates with the expression of the RAS oncoprotein, a key cancer gene [139] . Intranasal administration of let-7 mimic into mouse models of lung cancer significantly reduced tumor growth, suggesting that miRNA replacement therapy is indeed promising [106, 140, 141] . Another miRNA that shows the value of miRNA replacement is provided by miR-34a [107, 142] . Local and/or systemic delivery of a synthetic miR-34a mimic led to accumulation of miR-34a in the tumor tissue and inhibition of lung tumor growth. Lately, Ling et al. also showed that tumor suppressor miR-22 exhibited anti-lung cancer activity through post-transcriptional regulation of ErbB3 [143] . Thus, therapeutic miRNA mimics have a powerful potential by attacking multiple genes relevant to several diseases. However, it is necessary to pay attention to the potential toxicity in normal tissues under conditions in which the therapeutic delivery of miRNA mimics will lead to an accumulation of exogenous miRNAs in normal cells [138] . Although the assumptions are well founded, there is still insufficient evidence for toxicity caused by miRNA mimics. Indeed, several in vivo studies failed to reveal side effects caused by the miRNA mimics and suggested that delivery of miRNA mimics to normal tissues was well tolerated [107, 141] . It will be important to research miRNA mimic-induced effects in normal cells and to carefully assess toxicity before using them in clinical practice.\\n\\nThe second strategy is directed toward a gain of function and aims to inhibit oncomiRs by using anti-miRNAs. Chemical modifications, such as 2\\'-O-methyl-group and locked nucleic acid (LNA), would increase oligo stability against nucleases [144] . Antisense oligonucleotides contained in these modifications are termed antagomirs or \"LNA-antimiRs\" [144, 145] . They are oligonucleotides with sequences complementary to the endogenous miRNA and inhibit the specific miRNA function. An LNA-antimiR against miR-122 has been shown to effectively silence miR-122 in non-human primates [145] , and the findings support the potential of these compounds as a new class of therapeutics. Moreover, it has also been reported that anti-miR-150 delivered into lung tumor xenografts in mice led to inhibited tumor growth [146] . Relative to studies on miRNA mimics, studies with antisense oligonucleotides have shown effective evidence with naked oligonucleotides. This illustrates the potential of chemical modifications of oligonucleotides to improve their stability, resistance to RNase, and pharmacologic properties. Therefore, inhibition of miRNA function by chemically modified antimiR oligonucleotides has become an important and widely used approach. Recent data from the first phase II study in patients with chronic HCV infection treated with the LNA-modified antimiR-122 showed that this compound was well tolerated and provided continuing viral suppression.\\n\\nAn increasing number of studies have examined the therapeutic potential of miRNAs. Recently, the evidence of roles for miRNAs in determining drug resistance has emerged [147] . Cytotoxic and molecular target drugs have been widely used in the treatment of advanced lung cancer; unfortunately, many cases are still refractory to chemotherapy. In this situation, combining miRNA mimics or antimiR with chemotherapy may potentiate the efficacy of the cancer treatment in the future. In addition, miRNAs related with cancer stem cells may significantly broaden the field of miRNA-based therapy and suggest that miRNAs can be potential tools to kill cancer cells associated with therapy resistance, recurrence, and metastasis [108, 148] . Hence, the main challenge is the successful delivery and chemical modifications of the therapeutic miRNAs to the target tissue without harming normal tissues.\\n\\nRNAi-based approaches provide a promising therapeutic modality for the treatment of various lung diseases. One of the greatest challenges in RNAi-based therapy continues to be the delivery method of the therapeutic siRNAs and miRNAs to the target cells. Pulmonary delivery applications are very attractive, since they tend to be non-invasive, are locally restricted, and can be administered by the patient. A realistic therapeutic intervention, such as aerosolization, can enhance drug delivery to the site of action and decrease systemic exposure of the patient to the therapy, thereby reducing off-target effects. The advancement of pulmonary siRNA delivery to the clinic illustrates that RNAi-based therapy holds a central place in the future treatment of lung diseases. On the other hand, miRNAs have the opportunity to target multiple genes in a fine-tuned manner, and the miRNA-based therapy will provide an attractive anti-tumor and anti-inflammatory approach for various lung diseases. In particular, anti-miRNA therapy by chemically modified antimiR oligonucleotides has become a potential therapy for lung diseases because the oligonucleotides can be successfully delivered without delivery vectors. Increased evidence has indicated that miRNAs fulfill causative roles in a variety of lung diseases and have prompted investigations into their potential as therapeutic targets. Further understanding of the detailed mechanisms of RNAi-based therapy and investigations of more effective delivery methods are required for future development. These novel approaches could open new avenues for various lung diseases and improve the clinical outcome of the patients.', 'document_id': 641}]}, {'paragraphs': [{'qas': [{'question': 'How long is the SAIBK gene?', 'id': 1622, 'answers': [{'text': '27,534 nucleotides', 'answer_start': 1847}], 'is_impossible': False}, {'question': 'How many open reading frames are in the SAIBK gene?', 'id': 1623, 'answers': [{'text': '10', 'answer_start': 1958}], 'is_impossible': False}, {'question': 'What virus has the closest genetic identity with the SAIBK gene?', 'id': 1624, 'answers': [{'text': 'Chinese IBV strain SC021202', 'answer_start': 2061}], 'is_impossible': False}], 'context': 'Complete Genome Sequence of a Nephropathogenic Infectious Bronchitis Virus Strain Isolated in China\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC3795213/\\n\\nSHA: f2df4fc3c60338755fd23da3d7e01c0455e20745\\n\\nAuthors: Yang, Jing-tian; Ma, Bing-cun\\nDate: 2013-10-10\\nDOI: 10.1128/genomea.00815-13\\nLicense: cc-by\\n\\nAbstract: Infectious bronchitis virus (IBV) causes tremendous economic losses to the poultry industry. Here, we report the complete genome analysis results for a new natural recombination nephropathogenic IBV strain named SAIBK, which was isolated in the Sichuan province of China in 2005.\\n\\nText: tagious and acute disease in domestic chickens, belongs to group III of the genus Coronavirus in the family Coronaviridae (1) . It is an enveloped, unsegmented, positive-sense, single-stranded RNA (ssRNA) virus and has a genome of approximately 27.6 kb (2) . Recently, many epidemiological analysis reports have suggested that nephropathogenic IBVs have become increasingly prevalent (3) (4) (5) (6) in China. In this work, the complete genome sequence of an isolate named SAIBK was analyzed and recombination was detected between SAIBK and some previously reported IBVs.\\n\\nA rapid amplification of cDNA ends (RACE) kit (TaKaRa, Japan) was used to obtain the 5= and 3= ends of the genome. Other parts were amplified by 19 primers with overlap between each fragment and were cloned into the pMD19-T vector (TaKaRa, Japan). All fragments were sequenced three times by Sangon Biotech (Shanghai, China). The sequenced fragments were assembled using the SeqMan software program (DNAStar, Inc.). Sequence alignment was conducted and a phylogenetic tree was constructed using the software program MEGA5 (7). Recombination analysis was performed using the RDP 4.14 (8) and SimPlot 3.5.1 (9) software programs.\\n\\nThe complete genome of the SAIBK strain is 27,534 nucleotides (nt) in length, including the poly(A) tail. It has a classical IBV genome organization with 10 open reading frames (ORFs):\\n\\nThe genome sequence of SAIBK shows the highest identity (94.3%) to the Chinese IBV strain SC021202 (GenBank accession no. EU714029) and the lowest identity (85.8%) to two Chinese IBV strains, BJ (GenBank accession no. AY319651) and DY07 (GenBank accession no. HM245923). It has lower nucleotide identities of 88.1%, 87.9%, and 87.7% to the most popularly used IBV vaccine strains, H120, H52, and M41, respectively.\\n\\nPhylogenetic analysis of the complete genome results indicated that the SAIBK strain clusters into the same branch as the IBV YN strain (GenBank accession no. JF893452) and the SC021202 strain (GenBank accession no. EU714029). The S1 subunit of the IBV genome is the major determinant of serotype (10) (11) (12) (13) , and S1 analysis indicated that the SAIBK strain has a 4/91-like serotype.\\n\\nThe employed recombination detection methods revealed that SAIBK is a chimera virus, with recombination by the SC021202 strain as a major parent and the H120 vaccine strain as a minor parent. The first and second recombination regions were located at positions 7231 to 9126 and 13437 to 14473 in genes 1a and 1b, respectively. There were two other recombination regions detected at positions 951 to 1067 and 5393 to 5605 of SAIBK, which were recombined with the SC021202 strain as a major parent and the H52 vaccine strain as a minor parent. The recombination detection results suggested that SAIBK is possibly a chimera virus derived from the popularly used vaccine strains H120 and H52 and the field strain SC021202, and the SC021202 strain was isolated from chickens vaccinated with H120 in the Sichuan province of China in 2003 (14) . This result revealed that the field IBVs in Sichuan Province have undergone genetic recombination and are possibly emerging as new mutant strains, such as SAIBK.\\n\\nNucleotide sequence accession number. The complete genome sequence of the SAIBK isolate was submitted to GenBank and assigned the accession no. DQ288927.', 'document_id': 1547}]}, {'paragraphs': [{'qas': [{'question': 'How many surgical masks or respirators have past studies projected will be required for a pandemic in the United States?', 'id': 257, 'answers': [{'text': 'an estimated 7.3 billion', 'answer_start': 6383}], 'is_impossible': False}, {'question': 'What is the acronym MERS-CoV?', 'id': 248, 'answers': [{'text': 'Middle East respiratory syndrome coronavirus', 'answer_start': 722}], 'is_impossible': False}, {'question': 'What are the critical factors that determine the effect of an epidemic?', 'id': 249, 'answers': [{'text': 'Transmissibility and severity', 'answer_start': 474}], 'is_impossible': False}, {'question': 'When did the World Health Organization (WHO) officially declare the 2019-nCoV epidemic as a Public Health Emergency of International Concern?', 'id': 250, 'answers': [{'text': 'January 30, 2020', 'answer_start': 7759}], 'is_impossible': False}, {'question': 'What influenza virus was identified in China in 2013?', 'id': 251, 'answers': [{'text': 'H7N9', 'answer_start': 5591}], 'is_impossible': False}, {'question': 'What past research has been done on severe, single-wave pandemics?', 'id': 252, 'answers': [{'text': 'After a new influenza virus (H7N9) was identified in China in 2013, a series of modeling articles described the effect of, and level of preparedness for, a severe, single-wave pandemic in the United States.', 'answer_start': 5562}], 'is_impossible': False}, {'question': 'What is a clinical attack rate?', 'id': 253, 'answers': [{'text': 'the proportion of individuals who become ill with or die from a disease in a population initially uninfected', 'answer_start': 5816}], 'is_impossible': False}, {'question': 'What was the clinical attack rate in the 2009 H1N1 pandemic?', 'id': 254, 'answers': [{'text': '20%', 'answer_start': 5985}], 'is_impossible': False}, {'question': 'What is the estimated R0 of COVID-19?', 'id': 255, 'answers': [{'text': '2.2', 'answer_start': 7158}], 'is_impossible': False}, {'question': 'How many ventilators have past studies projected will be required for a pandemic in the United States?\\n', 'id': 1657, 'answers': [{'text': '35\\u202f000 to 60\\u202f000', 'answer_start': 6331}], 'is_impossible': False}], 'context': 'Preparation for Possible Sustained Transmission of 2019 Novel Coronavirus\\nLessons From Previous Epidemics\\nhttps://jamanetwork.com/journals/jama/fullarticle/2761285\\nFebruary 11, 2020\\nDavid L. Swerdlow, MD1; Lyn Finelli, DrPH, MS2\\nAuthor Affiliations Article Information\\nJAMA. 2020;323(12):1129-1130. doi:10.1001/jama.2020.1960\\nCOVID-19 Resource Center\\nrelated articles icon Related\\nArticles\\nauthor interview icon Interviews\\nAudio Interview (25:53)\\nCOVID-19 Update From China\\nTransmissibility and severity are the 2 most critical factors that determine the effect of an epidemic. Neither the 2009 pandemic influenza A(H1N1) virus ([H1N1]pdm09) pandemic or the severe acute respiratory syndrome coronavirus (SARS-CoV) or the Middle East respiratory syndrome coronavirus (MERS-CoV) epidemics had the combination of both high transmissibility and severity. Control strategies are driven by this combination. R0, the basic reproduction number, is a commonly used measure of transmissibility and is defined as the number of additional persons one case infects over the course of their illness. An R0 of less than 1 indicates the infection will die out “eventually.” An R0 of greater than 1 indicates the infection has the potential for sustained transmission.\\n\\nFor example, influenza A(H1N1)pdm09, first identified in southern California on April 15, 2009, was highly transmissible. By May 5, 2009, influenza A(H1N1)pdm09 had spread to 41 US states and 21 countries.1 While influenza A(H1N1)pdm09 was highly transmissible, it was not severe. Initial estimates of the R0 of influenza A(H1N1)pdm09 were 1.7.2 Although an estimated 201\\u202f200 respiratory deaths due to influenza A(H1N1)pdm09 occurred during the first year of the pandemic, the number of deaths per population was 30\\u2009times\\u2009lower than that seen during the 1968 influenza pandemic, 1000 times less than the 1918 pandemic, and even less than typical seasonal influenza epidemics (estimated by the World Health Organization [WHO] to be 250\\u202f000 to 500\\u202f000 per year, although estimation methods differ).3 Influenza A(H1N1)pdm09 was highly transmissible but not severe.\\n\\nSARS-CoV (2003) and MERS-CoV (2012-current) cause severe disease, but despite the initial R0 estimations of greater than 2.0 for SARS-CoV (indicating sustained and even worldwide transmission could occur), and some large outbreaks, neither were as transmissible as initial concerns suggested. SARS-CoV caused 8098 reported cases and 774 deaths (case-fatality rate, 9.6%) in 37 countries before the epidemic was controlled. Control was thought to have been possible because a high proportion of cases were severe, making it easier to rapidly identify and isolate infected individuals. In addition, the virus was present at lower levels in upper airway secretions. There was no secondary transmission in the United States from the 8 imported cases, although in Toronto, Canada, a single importation is thought to have led to about 400 cases and 44 deaths. Later estimates of R0 were less than 1, indicating that SARS-CoV may not have been capable of sustained transmission, especially in the setting of control measures.4\\n\\nSimilarly, MERS-CoV appears to have high severity and low transmissibility. Since 2012, MERS-CoV has caused 2494 reported cases and 858 deaths (case-fatality rate, 34%) in 27 countries. MERS-CoV has also caused some rapid outbreaks, mainly in hospitals in Saudi Arabia, Jordan, and South Korea, but estimates of MERS-CoV R0 are less than 1, and thus far it has been contained.5\\n\\nCan a respiratory virus that is both transmissible and severe be contained? In preparation for an influenza pandemic, the US Department of Health and Human Services’ Pandemic Influenza Plan included a combination of nonpharmaceutical (border and school closing, infection control measures) and pharmaceutical (antiviral prophylaxis, vaccines) interventions meant to be used in combination to interrupt or slow influenza transmission. Despite implementation of some of these interventions, influenza A(H1N1)pdm09 spread to 120 countries in 3 months.\\n\\nWith the emergence of MERS-CoV in the Middle East, a preparedness plan was developed that included a surveillance plan, laboratory testing, and contact tracing guidance. Infection control guidance was developed for use in health care settings and traveler guidance was developed for the public.6 The US Centers for Disease Control and Prevention (CDC) distributed MERS-CoV polymerase chain reaction test kits to state health departments. Two cases were imported into the United States. Contacts were traced, including household, hospital, and airline contacts. No secondary cases were identified in the United States. MERS-CoV was thought to be severe and control measures relied on recognition of suspect cases. However, during a hospital outbreak in Jeddah, Saudi Arabia, among hospitalized patients only 5 of 53 (9%) health care–associated cases had documented presence in the same room as a patient with MERS.5 Despite the high case-fatality rate (an important measure of severity), MERS cases can be asymptomatic and mild (25% in one outbreak). Although it is not known how often asymptomatic or mildly symptomatic patients transmit MERS, initiating comprehensive measures such as isolating patients suspected of having or having been exposed to the virus and using personal protective equipment when caring for them may be extremely difficult because so many patients have mild and nonspecific symptoms.\\n\\nIs the world ready for a respiratory virus with high transmissibility and severity? After a new influenza virus (H7N9) was identified in China in 2013, a series of modeling articles described the effect of, and level of preparedness for, a severe, single-wave pandemic in the United States.7 In scenarios that used clinical attack rates (the proportion of individuals who become ill with or die from a disease in a population initially uninfected) of 20% to 30% (for comparison the clinical attack rate was 20% in the first year of the 2009 H1N1 pandemic), depending on severity there would be an estimated 669\\u202f000 to 4.3 million hospitalizations and an estimated 54\\u202f000 to 538\\u202f000 deaths without any interventions in the United States. The models suggested that without a vaccine, school closures would be unlikely to affect the pandemic, an estimated 35\\u202f000 to 60\\u202f000 ventilators would be needed, up to an estimated 7.3 billion surgical masks or respirators would be required, and perhaps most important, if vaccine development did not start before the virus was introduced, it was unlikely that a significant number of hospitalizations and deaths could be averted due to the time it takes to develop, test, manufacture, and distribute a vaccine.\\n\\nIt is impossible to know what will happen so early in this novel 2019 coronavirus (2019-nCoV) epidemic. The scope, morbidity, and mortality will depend on the combination of severity and transmissibility. Numerous experts have “nowcasted” how many cases have occurred and forecasted how many cases will likely occur. A recent study suggests rapid person to person transmission can occur.8 Disease modelers have estimated R0 to be 2.2.9 The University of Hong Kong estimates the outbreak could infect more than 150\\u202f000 persons per day in China at its peak.\\n\\nIs 2019-nCoV infection severe? To date approximately 14% of cases of 2019-nCoV have been described as severe by WHO, with a case-fatality rate of 2.1%.10 Estimates of severity are usually higher in the beginning of an epidemic due to the identification of the most severely affected cases and decline as the epidemic progresses. However, because many infected persons have not yet recovered and may still die, the case-fatality rate and severity could be underestimated. On January 30, 2020, WHO officially declared the 2019-nCoV epidemic as a Public Health Emergency of International Concern, indicating its concern that countries aside from China could be affected by 2019-nCoV.\\n\\nIn preparing for possible sustained transmission of 2019-nCoV beyond China, applicable lessons from previous experiences with epidemics/pandemics of respiratory viruses should be carefully considered to better control and mitigate potential consequences. Influenza preparedness plans have been developed that aim to stop, slow, or limit the spread of an influenza pandemic to the United States. These plans address limiting domestic spread and mitigating disease but also sustaining infrastructure and reducing the adverse effects of the pandemic on the economy and society. These plans would be useful to enact during the 2019-nCoV epidemic should the United States experience sustained transmission. Countries have been successful in the past and there is nothing yet to predict that this time it is likely to be worse. Effective prevention and control will not be easy if there is sustained transmission and will require the full attention of public health, federal and local governments, the private sector, and every citizen.\\n\\nBack to topArticle Information\\nCorresponding Author: David L. Swerdlow, MD, Clinical Epidemiology Lead, Medical Development and Scientific/Clinical Affairs, Pfizer Vaccines, 500 Arcola Rd, Collegeville, PA 19426 (david.swerdlow@pfizer.com).\\n\\nPublished Online: February 11, 2020. doi:10.1001/jama.2020.1960\\n\\nConflict of Interest Disclosures: Dr Swerdlow reports owning stock and stock options in Pfizer Inc. Dr Swerdlow also reports providing a one-time consultation consisting of an overview of SARS and MERS epidemiology to GLG Consulting and receiving an honorarium. Dr Finelli reports owning stock in Merck and Co.\\n\\nFunding/Support: Pfizer Inc provided salary support for Dr Swerdlow.\\n\\nRole of the Funder/Sponsor: Pfizer Inc reviewed the manuscript and approved the decision to submit the manuscript for publication.\\n\\nReferences\\n1.\\nSwerdlow  DL, Finelli  L, Bridges  CB.  2009 H1N1 influenza pandemic: field and epidemiologic investigations in the United States at the start of the first pandemic of the 21st century.  Clin Infect Dis. 2011;52(suppl 1):S1-S3. doi:10.1093/cid/ciq005PubMedGoogle ScholarCrossref\\n2.\\nBalcan  D, Hu  H, Goncalves  B,  et al. Seasonal transmission potential and activity peaks of the new influenza A(H1N1): a Monte Carlo likelihood analysis based on human mobility.  BMC Medicine. 2009;7(45). doi:10.1186/1741-7015-7-45\\n3.\\nDawood  FS, Iuliano  AD, Reed  C,  et al.  Estimated global mortality associated with the first 12 months of 2009 pandemic influenza A H1N1 virus circulation: a modelling study.  Lancet Infect Dis. 2012;12(9):687-695. doi:10.1016/S1473-3099(12)70121-4PubMedGoogle ScholarCrossref\\n4.\\nChowell  G, Castillo-Chavez  C, Fenimore  PW, Kribs-Zaleta  CM, Arriola  L, Hyman  JM.  Model parameters and outbreak control for SARS.  Emerg Infect Dis. 2004;10(7):1258-1263. doi:10.3201/eid1007.030647PubMedGoogle ScholarCrossref\\n5.\\nKillerby  ME, Biggs  HM, Midgley  CM, Gerber  SI, Watson  JT.  Middle East respiratory syndrome coronavirus transmission.  Emerg Infect Dis. 2020;26(2):191-198. doi:10.3201/eid2602.190697PubMedGoogle ScholarCrossref\\n6.\\nRasmussen  SA, Watson  AK, Swerdlow  DL.  Middle East respiratory syndrome (MERS).  Microbiol Spectr. 2016;4(3). doi:10.1128/microbiolspec.EI10-0020-2016PubMedGoogle Scholar\\n7.\\nSwerdlow  DL, Pillai  SK, Meltzer  MI, eds.  CDC modeling efforts in response to a potential public health emergency: influenza A(H7N9) as an example.  Clin Infect Dis. 2015;60(suppl):S1-S63. https://academic.oup.com/cid/issue/60/suppl_1.Google Scholar\\n8.\\nWang  D, Hu  B, Hu  C,  et al.  Clinical characteristics of 138 hospitalized patients with 2019 novel coronavirus–infected pneumonia in Wuhan, China.  JAMA. Published online February 7, 2020. doi:10.1001/jama.2020.1585\\nArticlePubMedGoogle Scholar\\n9.\\nLi  Q, Guan  X, Wu  P,  et al.  Early transmission dynamics in Wuhan, China, of novel coronavirus–infected pneumonia.  N Engl J Med. Published online January 29, 2020. doi:10.1056/NEJMoa2001316PubMedGoogle Scholar\\n10.\\nWorld Health Organization. Novel coronavirus (2019-nCoV) situation reports. https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/. Accessed February 4, 2020.\\nComment\\n2 Comments for this articleEXPAND ALL\\nFebruary 12, 2020\\nUnderstanding R and Disease Control\\nOz Mansoor | Public Health Physician, Wellington\\nThe message, that we need to prepare for a pandemic is vital. But the article misreports some key ideas. Firstly, SARS was not controlled \"because a high proportion of cases were severe.\" While that helped , it was because cases were not infectious before some days after symptom onset (usually in the second week of illness). This gave more time for case identification and isolation. And most cases did not pass on infection to anybody, but a few spread to many. When all such individuals were identified and isolated, spread stopped.\\n\\nUnfortunately, the new virusappears to be spreading from people much earlier in the course of illness, and even with mild symptoms - which was never documented for SARS. However, it is not clear that it is any different or better at spread between people, and perhaps with the same pattern of most cases not causing further spread.\\n\\nSecondly, the R0, the basic reproduction number, is correctly described as the average number of infections each case causes. But it lacks two key ideas: 1) the 0 after the R implies the native state, which is a fully susceptible population and without any control measures. R is the effectiive number and can include the impact of control measures.\\n\\nTo claim that it was the lack of transmissibility, rather than the control measures that ended SARS, is not based on any evidence. And it ignores the heroic efforts of affected countries.\\n\\nElimination of SARS demonstrated the potential of globally coordinated collective action, as well as the damage caused by ignorance and prejudice. Most seem to have already forgotten the lessons of SARS.CONFLICT OF INTEREST: Worked for WHO/WPRO in SARS responseREAD MORE\\nFebruary 24, 2020\\nCOVID 19: a global presence and not only a new pathogen?\\nGiuliano Ramadori, Professor of Medicine | University Clinic, Göttingen, Germany\\nIn the winter season there comes the time of upper and lower respiratory tract infections characterised by cough, dyspnea and eventually fever (influenza-like illness).Some of the patients, especially older people living alone affected by the disease ,may need hospitalization and eventually intensive care. In many of the cases who are hospitalized nasal and/or tracheal fluid are examined for viral or bacterial agents. Only in less than 50% of the cases influenza viruses are considered to be the cause of the disease.In the rest of the cases diagnostic procedure for human coronaviruses is not performed routinely. One of the fourdifferent Human Coronaviruses (HuCoV: 229E,NL 63,0C43 and HKU1) can however be found in up to 30% ofpatients negative for influenza viruses (1). Chinese scientists in Wuhan, who had to deal with an increasing number of acute respiratory tract diseases resembling viral pneumonia, performed deep sequencing analysis from samples taken from the lower respiratory tract and found a \"novel\" coronavirus. The sequence of the complete genome was made public. At the same time, however, the notice from Wuhan brought to mind the SARS- and MERS-epidemics. The measures taken by the Chinese- and WHO-authorities are now well known.\\n\\nRecently about 150 new cases have been identified in northern Italy and health authorities are still looking for case 0 (the source). Is it possible that COVID-19 was already existent in Italy -- and not only in Italy but possibly everywhere in the world -- and that newly available nucleotide sequence allows now to find the cause of previously undefined influenza-like illness?\\n\\nREFERENCE\\n\\n1. Benezit F et al.:Non-influenza respiratory viruses in adult patients admitted with influenza-like illness:a 3- year prospective multicenter study.Infection, 13 february 2020, https://doi.org/10.1007/s15010-019-01388-1).CONFLICT OF INTEREST: None ReportedREAD MORE\\nSee More About\\nGlobal Health Public Health Pulmonary Medicine Infectious Diseases Influenza\\nDownload PDF\\nCite This\\nPermissionsComment\\nCME & MOC\\n Coronavirus Resource Center\\n\\n\\nTrending\\nOpinion is learning has multimedia\\nUS Emergency Legal Responses to Novel Coronavirus—Balancing Public Health and Civil Liberties\\nMarch 24, 2020\\nOpinion is learning has multimedia\\n2019 Novel Coronavirus—Important Information for Clinicians\\nMarch 17, 2020\\nResearch is learning has multimedia\\nClinical Characteristics of Patients With Novel Coronavirus (2019-nCoV) Infection Hospitalized in Beijing, China\\nMarch 17, 2020\\nSelect Your Interests\\nJOB LISTINGS ON JAMA CAREER CENTER®\\nACADEMIC CARDIOLOGIST: HEART FAILURE SPECIALIST\\nPhoenix, Arizona\\nNONINVASIVE CARDIOLOGIST\\nWest Grove, Pennsylvania\\nCARDIOLOGIST\\nPhoenixville, Pennsylvania\\nCARDIAC INTENSIVIST FACULTY\\nWest Reading, Pennsylvania\\nCLINICAL FACULTY: CARDIOLOGY / ELECTROPHYSIOLOGIST\\nPhoenix, Arizona\\nSee more at JAMA Career Center\\nOthers Also Liked\\nCoronavirus Dx Emergency Use Authorizations Progressing Rapidly Despite Criticism\\nMadeleine Johnson, 360Dx, 2020\\nAnalysis of therapeutic targets for SARS-CoV-2 and discovery of potential drugs by computational methods\\nCanrong Wu, Acta Pharmaceutica Sinica B, 2020\\nCommercial Labs Step up Coronavirus Test Efforts After FDA Guidance\\n360Dx, 2020\\nPowered by\\nTrending\\nUS Emergency Legal Responses to Novel Coronavirus—Balancing Public Health and Civil Liberties\\nJAMA Opinion March 24, 2020\\nPractical Aspects of Otolaryngologic Clinical Services During the COVID-19 Epidemic\\nJAMA Otolaryngology–Head & Neck Surgery Opinion March 20, 2020\\n2019 Novel Coronavirus—Important Information for Clinicians\\nJAMA Opinion March 17, 2020\\nJAMA\\nCONTENT\\nHome New Online Current Issue\\nJOURNAL INFORMATION\\nFor Authors Editors & Publishers RSS Contact Us\\nJN Learning / CME Store Apps Jobs Institutions Reprints & Permissions\\nJournal Cover\\nSubscribe\\nGo\\nJAMA Network\\nPUBLICATIONS\\nJAMA JAMA Network Open JAMA Cardiology JAMA Dermatology JAMA Facial Plastic Surgery JAMA Health Forum JAMA Internal Medicine JAMA Neurology JAMA Oncology JAMA Ophthalmology JAMA Otolaryngology–Head & Neck Surgery JAMA Pediatrics JAMA Psychiatry JAMA Surgery Archives of Neurology & Psychiatry (1919-1959)\\nSITES\\nAMA Manual of Style Art and Images in Psychiatry Breast Cancer Screening Guidelines Colorectal Screening Guidelines Declaration of Helsinki Depression Screening Guidelines Evidence-Based Medicine: An Oral History Fishbein Fellowship Genomics and Precision Health Health Disparities Hypertension Guidelines JAMA Network Audio JAMA Network Conferences Machine Learning Med Men Medical Education Opioid Management Guidelines Peer Review Congress Research Ethics Sepsis and Septic Shock Statins and Dyslipidemia Topics and Collections\\nFEATURED ARTICLES\\nACS Breast Cancer Screening Guideline CDC Guideline for Prescribing Opioids CDC Guideline for Prevention of Surgical Site Infections Consensus Definitions for Sepsis and Septic Shock Global Burden of Cancer, 1990-2016 Global Burden of Disease in Children, 1990-2013 Global Burden of Hypertension, 1990-2015 Global Firearm Mortality, 1990-2016 Health Care Spending in the US and Other High-Income Countries Income and Life Expectancy in the US JNC 8 Guideline for Management of High Blood Pressure President Obama on US Health Care Reform Screening for Colorectal Cancer Screening for Depression in Adults Screening for Prostate Cancer Statins for Primary Prevention of Cardiovascular Disease The State of US Health, 1990-2016 US Burden of Cardiovascular Disease, 1990-2016 WMA Declaration of Helsinki, 7th Revision\\nBLOGS\\nJAMA Health Forum AMA Style Insider\\nINFORMATION FOR\\nAuthors Institutions & Librarians Advertisers Subscription Agents Employers & Job Seekers Media\\nJAMA NETWORK PRODUCTS\\nAMA Manual of Style JAMAevidence JN Listen Peer Review Congress\\nJN LEARNING\\nHome CME Quizzes State CME Audio / Podcast Courses Clinical Challenge CME Atrial Fibrillation Course Marijuana Course Penicillin Allergy Course Cervical Cancer Screening Course CME / MOC Reporting Preferences About CME & MOC\\nHelp\\nSubscriptions & Renewals Email Subscriptions Update Your Address Contact Us Frequently Asked Questions\\nJAMA CAREER CENTER\\nPhysician Job Listings\\n\\nGet the latest from JAMA\\nEmail address\\nSign Up\\nPrivacy Policy | Terms of Use\\nJama Network Logo\\n© 2020 American Medical Association. All Rights Reserved.\\nTerms of Use| Privacy Policy| Accessibility Statement\\n\\nSilverchair Logo\\n', 'document_id': 187}]}, {'paragraphs': [{'qas': [{'question': 'How is exhaled breath condensate used in viral research?', 'id': 5193, 'answers': [{'text': 'the isolation of respiratory viruses', 'answer_start': 548}], 'is_impossible': False}, {'question': 'How many patients were i this study?', 'id': 5194, 'answers': [{'text': '102', 'answer_start': 609}], 'is_impossible': False}, {'question': 'What was the conclusion of this study?', 'id': 5195, 'answers': [{'text': 'EBC collection using the RTube™ is not reliable for diagnosis of respiratory infections', 'answer_start': 1673}], 'is_impossible': False}, {'question': 'How long did the patient breath into the RTube?', 'id': 5196, 'answers': [{'text': '10 minutes', 'answer_start': 4107}], 'is_impossible': False}, {'question': 'What followed the reverse transcription step in the analysis?', 'id': 5197, 'answers': [{'text': 'denaturation step', 'answer_start': 9620}], 'is_impossible': False}, {'question': 'What was the last step in the analysis?', 'id': 5198, 'answers': [{'text': 'amplification step', 'answer_start': 9670}], 'is_impossible': False}, {'question': 'What percentage of the patients were between 20 and 30 years old in this study?', 'id': 5199, 'answers': [{'text': '52%', 'answer_start': 9888}], 'is_impossible': False}, {'question': 'Why is EBC an attractive method for screening?', 'id': 5200, 'answers': [{'text': 'easy to perform and can be conducted in a home environment', 'answer_start': 12629}], 'is_impossible': False}], 'context': \"Exhaled breath condensate sampling is not a new method for detection of respiratory viruses\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059288/\\n\\nSHA: f3b46e7e8f58799207cc44515f859c1daf5e4dfc\\n\\nAuthors: Houspie, Lieselot; De Coster, Sarah; Keyaerts, Els; Narongsack, Phouthalack; De Roy, Rikka; Talboom, Ive; Sisk, Maura; Maes, Piet; Verbeeck, Jannick; Van Ranst, Marc\\nDate: 2011-03-04\\nDOI: 10.1186/1743-422x-8-98\\nLicense: cc-by\\n\\nAbstract: BACKGROUND: Exhaled breath condensate (EBC) sampling has been considered an inventive and novel method for the isolation of respiratory viruses. METHODS: In our study, 102 volunteers experiencing upper airway infection were recruited over the winter and early spring of 2008/2009 and the first half of the winter of 2009/2010. Ninety-nine EBCs were successfully obtained and screened for 14 commonly circulating respiratory viruses. To investigate the efficiency of virus isolation from EBC, a nasal swab was taken in parallel from a subset of volunteers. The combined use of the ECoVent device with the RTube™ allowed the registration of the exhaled volume and breathing frequency during collection. In this way, the number of exhaled viral particles per liter air or per minute can theoretically be estimated. RESULTS: Viral screening resulted in the detection of 4 different viruses in EBC and/or nasal swabs: Rhinovirus, Human Respiratory Syncytial Virus B, Influenza A and Influenza B. Rhinovirus was detected in 6 EBCs and 1 EBC was Influenza B positive. We report a viral detection rate of 7% for the EBCs, which is much lower than the detection rate of 46.8% observed using nasal swabs. CONCLUSION: Although very promising, EBC collection using the RTube™ is not reliable for diagnosis of respiratory infections.\\n\\nText: Human respiratory tract infections represent the most commonly encountered infections worldwide. In the majority of cases, the etiology of these infections remains undetermined due to rapid convalescence after infection. Respiratory tract infections in healthy adults can be caused by a variety of pathogens and the detection of these agents is currently based on their isolation from nasal swabs (NS), bronchoalveolar lavages (BAL), nasopharyngeal aspirates and sputum samples. The acquisition of these specimens by semi-invasive and invasive techniques is often unpleasant for the patient. Therefore, exhaled breath condensate (EBC) analysis has recently been explored as a new and non-invasive method to monitor lung inflammation and pulmonary disease such as chronic obstructive pulmonary disease (COPD), asthma, cystic fibrosis, lung cancer etc. EBCs mainly consist of water vapour but a small fraction contains respiratory droplets derived from the airway lining fluid [1, 2] . This observation has created a growing interest in the use of EBC as a new sampling method for the screening of respiratory viruses infecting the upper airways. At first, investigators suspected that turbulence of the inhaled air was responsible for the aerosolisation of the respiratory fluid. However, the effect of the turbulent airflow is limited to the upper airways since the turbulent airflow becomes laminar as it reaches the smaller bronchial airways and alveoli. Recently, the bronchiole fluid film burst model has been described [3] . This model suggests that aerosols are produced during inhalation by the bursting of fluid bubbles present in the bronchioles.\\n\\nThe aim of this study was to investigate whether the EBC collection method was suited for the efficient condensation of aerosolised virus particles during normal breathing and to explore the isolation of respiratory viruses in the condensate. Therefore we screened the EBC samples with virus specific PCR assays targeting 14 \\n\\nIn this study, 102 EBCs were collected from otherwise healthy volunteers showing respiratory or flu-like symptoms (defined in Table 1 ), using a commercially available condenser (RTube™, Respiratory Research Inc., Charlottesville, Virginia, USA). The patient was instructed to breath orally at tidal volumes into a mouthpiece attached to a condenser for 10 minutes. No nose clips were used during collection and saliva contamination was avoided by the presence of a one-way valve and the T-shaped section of the mouthpiece.\\n\\nIn a first part of the study that started during the winter and spring of 2008/2009, 70 EBC samples were collected from patients who voluntary presented themselves to our laboratory. The majority of these volunteers were students that responded to the information leaflet, distributed in the university buildings of the Catholic University of Leuven. The samples were collected with the aluminium cooler sleeve chilled at -80°C.\\n\\nIn the fall and first half of the winter of 2009/2010, 32 condensates were collected from patients who presented themselves to their general practitioner. Due to practical circumstances, the condensates were collected with the cooler chilled at -20°C. For 13 out of 32 collections, the RTube™ was connected by a custom made connectingpiece to the ECoVent (Jaeger, Germany). This device registers ventilatory parameters such as the exhaled volume, breathing frequency and tidal volume. Additionally, a NS was obtained in parallel with the condensate collection from each patient.\\n\\nAll EBCs were immediately stored at -20°C. Nasal swabs (NS) were refrigerated. After viral DNA and RNA extraction, EBC samples and nasal swabs were stored at -80°C. Three specimens were excluded from the study due to incorrect condensate collection. A short questionnaire was used to document the date of birth, the severity of respiratory complaints and to record the days of symptomatic illness from all volunteers. This study was approved by the Medical Ethics Committee of the University Hospital of Leuven and informed consents were received from all participants.\\n\\nViral DNA and RNA were isolated with the QIAamp MinElute Virus kit (Qiagen, Westburg, The Netherlands) according to the instruction manual. EBC extracts were eluted in 60 μl elution buffer and NS extracts in 110 μl elution buffer.\\n\\nThe breath condensates were screened for 11 respiratory RNA viruses (CoV NL63, E229 and OC43, RV, HMPV, InfA&B and PIV1-4) [4] [5] [6] [7] using a OneStep RT-PCR Kit (Qiagen, Westburg, The Netherlands) in a 50 μl reaction containing 10 μl of the extracted RNA, 0.6 μM of forward and reverse primers (Table 2), 1.5 μl One Step Enzyme Mix, 10 μl 5 × One Step RT-PCR Buffer and 400 μM of each dNTP. For adenovirus screening, a DNA PCR was carried out for which the amplification reaction mix contained 0.5 μM forward primer (AdFW) and reverse primer (AdRV), 0.4 mM dNTPs, 10 μl Buffer C and 1 U Taq polymerase in a final volume of 50 μl. The PCR primers used were located in conserved regions of the genomes of the respiratory pathogens ( Table 2 ). The reactions were carried out in a T3000 Thermocycler 48 (Westburg, Leusden, The Netherlands) with an initial reverse transcription step for RNA viruses at 50°C for 30 min, followed by PCR activation at 95°C for 30 s, 45 cycles of amplification followed by a final extension step for 10 min at 72°C. The DNA amplification program was initiated with a denaturation step at 94°C for 3 min, followed by 45 cycles of 94°C for 30 s, 55°C for 30 s and a final extension step at 72°C for 1 min. The amplicons were subjected to a 6% polyacrylamide gel and visualised under UV light by staining with ethidium bromide. PCR products were purified using the Invitek MSB Spin PCRapace Kit and cycle sequenced in forward and reverse direction using the ABI PRISM Big-Dye Termination Cycle Sequencing Ready Reaction kit (Applied Biosystems, Foster City, CA, USA). Sequence analysis was performed with the ABI3130 Genetic Analyser (Applied Biosystems, Foster City, CA, USA). Consensus sequences were obtained using the SeqMan II software (DNASTAR, Madison, Wis.). For samples from HRSV was detected using a RT-PCR assay as previously described [8, 9] . In brief, a multiplex mix was prepared in a final volume of 25 μl using 5 μl extracted RNA, 12.5 μl of Eurogentec One-Step Reverse Transcriptase qPCR Master Mix containing ROX as a passive reference, 0.125 μl Euroscript + RT & RNase inhibitor (Eurogentec, Seraing, Belgium) 200 nM of HRSV-A and -B specific forward and reverse primers and 100 nM of HRSV-A and -B MGB probes. cRNA standards were constructed using the MEGAshortscript T7 kit (Ambion, Austin, TX, USA) and spectrophotometrically quantified.\\n\\nThe viral load of RV positive samples were quantified by qRT-PCR as described in the manuscript published by Lu and coworkers [10] . The Eurogentec One-Step Reverse Transcriptase qPCR kit was used for preparation of the master mix as described above. The primerset \\n\\nHRSV-AF F 669-695 ctgtgatagarttccaacaaaagaaca [8, 9] HRSV-AF F 718-745 agttacacctgcattaacactaaattcc [8, 9] HRSV-BN N 435-458 ggctccagaatataggcatgattc [8, 9] HRSV-BN N 480-508 tggttattacaagaagagcagctatacacagt [8, 9] MGB probes and probe, located in 5'UTR, were added to a final concentration of 1 μM and 0.1 μM, respectively. cRNA standards were constructed based on the PCR product of sample 1 using the MegaScript kit (Ambion, Austin, TX, USA). Quantification was performed with a spectrophotometer at 260 nm and converted to the molecule number [11] . Tenfold serial dilutions, allowing detection in a range of 8.6 × 10 6 to 8.6 × 10 2 RNA copies were used. The RT-PCR assays were carried out on a ABI PRISM 7500 Sequence Detection System (Applied Biosystems, Foster City, CA, USA). An initial reverse transcription step was performed at 48°C for 30 min, followed by a denaturation step at 95°C for 10 min. Finally, an amplification step of 45 cycli at 95°C for 15 sec and 1 min at 60°C was completed. (37.5%) men, with a median age of 29 (range 9 -46 years). Age and gender was missing for 2 participants of the second group. In total, 52% of the participants were between 20-30 years old. Only 6% were younger than 20 years old and 3% were older than 70 years. In totality, 80 patients (78.4%) were already feeling ill for 1 to 7 days at the day the sample was obtained. Seven volunteers (6.8%) were symptomatic for 8 to 14 days and 9 participants (8.8%) were already ill for more than 14 days at the day of sample collection. Data on the duration of symptoms was lacking for 6 patients. Almost all volunteers experienced at least 2 symptoms except for two patients (Table 1) . Forty-seven (46.1%) volunteers complained about a constant runny or stuffy nose, 43 (42.2%) had frequent sneezing events and 38 (37.3%) participants had a serious sore throat (Table 1) .\\n\\nIn a first part of the study, we collected 70 EBCs. Screening of the EBCs for 14 respiratory viruses (Table 2) , showed 5 RV (7.1%) positive samples (Table 3 ). In a second part, we collected 32 EBCs from patients that presented themselves to their general practitioner. Two of these EBCs were positive for one of the 14 investigated respiratory viruses, 1 for RV and 1 for InfB. To inspect the detection rate of respiratory viruses in the condensate, a NS was taken from this second group of volunteers for comparison. In 15 out of 32 NS (46.8%), one or more viral pathogens were isolated. Viral screening of the NS resulted in the detection of RV, InfA (subtype H1N1) and HRSV-B. Quantification of the HRSV-B viral load demonstrated for samples 72 and 101 viral titers of 8.0 × 10 4 RNA copies/ml and 6.8 × 10 7 RNA copies/ml respectively. The RV RT-PCR assay did not allow the quantification of all samples that tested positive for RV by PCR ( Table 3) . Presence of the same pathogen in both the EBC and the NS was confirmed for only 1 sample: sample 71, which tested positive for RV in both the EBC and the NS. For sample 81, RV was detected in the NS and analysis of the EBC demonstrated an InfB infection.\\n\\nFor EBC samples that were collected in the fall and winter of 2009/2010, measurements with the ECoVent in (Table 3 , sample 81) was positive for InfB when using the RTube™ in combination with the EcoVent. In theory, the viral generation rate (number of viral RNA copies exhaled per minute) can be predicted by quantification of the exhaled viral load. Then, an estimation of the RNA copies per litre exhaled air or per minute can be calculated. Quantification of the exhaled InfB would allow us to predict the generation rate for this virus. Due to insufficient sample volume, we could not determine the number of RNA copies in the sample.\\n\\nCollection of exhaled breath condensates is a novel and non-invasive method for obtaining samples of the upper respiratory tract. The collection of EBC is easy to perform and can be conducted in a home environment. This method is much more agreeable for the patient when compared to the unpleasant and invasive collection of nasal swabs, BAL, aspirates, etc. This aspect renders the method very attractive for routine laboratory diagnostics of viral infections. Most studies that perform breath analyses for viral detection use modified face masks, with a removable central region in electret or a removable Teflon filter on which exhaled particles impact [12] [13] [14] . With the RTube™ collection device, aerosolized particles of the airway lining fluid are precipitated into a condensate when the breath is cooled which serves as an immediate starting point for molecular testing.\\n\\nUntil now, this is the study with the largest subset of volunteers that investigated EBC as a specimen for the detection of respiratory viruses. Previous studies reported the inclusion of a limited subset of participants and investigated the presence of a limited number of viruses in the breath samples. The study performed by Fabian and colleagues, included 12 volunteers [12] . Huynh and co-workers recruited 9 volunteers for exhaled breath sampling [13] . In the study by Stelzer-Braid et al., 50 EBCs were analysed [14] and St-George et al. report the participation of 12 adults [15] . These studies have focused on the detection of InfA and -B, PIV1-3, HRSV and HMPV, while we have screened the samples for a panel of 14 commonly circulating respiratory viruses. Based on the analysis of 99 EBCs (3 EBCs were excluded), our results support the exhalation of RV and InfB in 7% of our samples. Since many of the volunteers had already been experiencing symptoms for 1 to 7 days, we initially presumed that they were already recovering from the infection and were no longer exhaling the virus. For common cold infections it is suggested that a person may already be infectious for 1 or 2 days before experiencing any symptoms. However, in a second part of our study we started collecting EBCs in parallel with nasal swabs from patients presenting themselves to their medical doctor, 1 to 3 days after onset of symptoms. Only for 1 condensate the same pathogen was detected in both the EBC and the NS. The detection rate for respiratory viral pathogens in the NS was 46.8% which is much higher than the 7% detection rate in the EBCs. The low detection of virus positive condensates can therefore not be attributed to the fact that volunteers were no longer infectious. The discrepant detection rate between samples may also be explained by different severity of respiratory infection, since comparator samples were of different parts of the respiratory tract. Patients that delivered a positive NS may have possibly suffered from an upper airway infection whereas EBC positive volunteers may have experienced a more advanced, lower respiratory tract infection. However, the effect of nasal inhalation on EBC collection, guiding formed particles in the upper respiratory tract to the lower compartments, in stead of oral inhalation was not investigated. Patients with positive EBC samples were experiencing symptoms for maximum two days at the time of collection. However, this was not different for 7 patients with positive NS. Six patients that provided positive NS were experiencing symptoms for a longer period at the time of collection (Table 3 ). In the group of volunteers that provided an EBC negative or EBC and NS negative sample, the manifestation of symptoms were reported ranging from 1 day to more than two weeks. When reported symptoms were compared between EBC positive patients (7) and NS positive patients (15) , 27% and 33% in the positive NS group experienced shivering and muscle pain whereas this symptom was not indicated by any patient of the EBC positive group. In all groups fever, headache, watering eyes, stuffed nose, frequent sneezing, sore throat and coughing were reported.\\n\\nVolunteers were not diagnosed with other pathogens before participation in the study. Since we did not test these samples for other than viral pathogens, we can not exclude the possibility that some of the negative NS are positive for bacteria or other pathogens causing respiratory illness. Recently, one study reported a detection rate of 5% for influenza in EBC [15] . This is in the same range of the detection rate that we report for respiratory viruses in general. Other studies with a limited number of patients, describe a markedly higher sensitivity of 33 to 36% [12] [13] [14] but the higher percentage may be due to the low number of participants subjects were included [12] . Remarkably, the studies reporting this higher detection rate used collections masks, while the study using the RTube™ reported comparable findings. Face masks consist of electret which trap viruses based on permanently charged fibres [13] . In addition, the Teflon filter has 2 μm pores which will retain all larger particles. Possibly, the lower detection rate can partly be explained by the fact that the RTube™ is manufactured in polypropylene and does not possess a virus attracting and filtering feature like the aforementioned materials.\\n\\nThe qRT-PCR developed by Lu and coworkers for the detection of RV, did not allow the assessment of the viral load present in the EBC samples [10] . Also for 4 NS, the viral titer remained undetermined, probably due to the limited sensitivity of the assay. For diagnosis, more sensitive methods might be necessary to detect respiratory viruses present in EBC since it is unpredictable how diluted the viral particles in the specimen are. Recently, nested qRT-PCR assays have been developed to allow a more sensitive detection of viruses in aerosols [16] .\\n\\nAlso person-dependent factors, such as the number of particles produced, the exhaled volume and the age of the patient, have been suggested to play an important role for exhalation of viral particles. The participants that were recruited in the study of Fabian and coworkers were 12 years of age and older [12] . For hospitalized children a much higher rate of virus positive samples is reported [14] . In our study, the majority of volunteers were between 20 and 30 years old. Only two children less than 10 years and 3 elderly people (> 70 years) were included. One of the children tested positive for InfA in the NS, but the infection was not confirmed in the EBC.\\n\\nFor influenza, an exhaled generation rate of <3.2 to 20 influenza RNA copies per minute was predicted by quantifying the virus aerosols that impacted on a removable Teflon filter of a collection mask [12] . We used the RTube™ in combination with the ECoVent, that allowed the registration of additional ventilation parameters such as breathing frequency and exhaled volume. In this way, when the number of RNA copies in the EBC is quantified, the amount of viral particles that are exhaled per litre or per minute can be estimated. Unfortunately, we were not able to predict a virus generation rate for InfB since viral load remained undetermined.\\n\\nAlthough an inventive, new and promising method, EBC collected by the RTube™ does not appear to be appropriate for diagnosis of respiratory infections. Nonetheless, this method may provide an alternative for current sample procurement for epidemiological studies of circulating viruses. This technique also confirms the observation that viruses are able to disseminate through normal breathing, particularly RV.\\n\\nIn addition, EBC collection from patients during respiratory infections may be further investigated for biomarker patterns. In calves that were experimentally infected with bovine RSV, an increase in leukotriene B 4 , indicating oxidative stress, was observed. This increased level was also associated with the development of bronchial hyperresponsiveness [17] . In humans, a transiently elevated H 2 O 2 level was observed during common cold infection. This marker returned to baseline values when volunteers recovered from infection. H 2 O 2 has also been recognized as an interesting marker in asthma, where it is associated with chronic lower airway inflammation [18] . In InfA infected volunteers, an increased CO level was observed during upper respiratory infection. This observation might imply that CO is an indicator of airway inflammation or represents one of the host defence mechanisms against viral infection [19] . Therefore, a better identification of the biomarker signature in condensates of individuals experiencing a viral infection might imply interesting findings towards the identification of markers reflecting inflammation or antiviral protection. This may contribute to the biomarker profiles established for diseases like asthma and COPD, for which viral infections are suggested to trigger or exacerbate symptoms [20] .\", 'document_id': 1582}]}, {'paragraphs': [{'qas': [{'question': 'What is the leading cause of death among children after the age of 1 month?', 'id': 502, 'answers': [{'text': 'Pneumonia remains the leading cause of death in children outside the neonatal period,', 'answer_start': 295}], 'is_impossible': False}, {'question': 'How has the number  of  childhood pneumonia been reduced?', 'id': 504, 'answers': [{'text': 'New conjugate vaccines against Haemophilus influenzae type b and Streptococcus pneumoniae have contributed to decreases in radiologic, clinical and complicated pneumonia cases', 'answer_start': 566}], 'is_impossible': False}, {'question': 'What percentage of childhood deaths are due to  pneumonia?', 'id': 507, 'answers': [{'text': ' approximately 900,000 of the estimated 6.3 million child deaths in 2013 ', 'answer_start': 1438}], 'is_impossible': False}, {'question': 'How has the childhood population grown in the last two  decades?', 'id': 510, 'answers': [{'text': 'global childhood population from 605 million in 2000 to 664 million in 2015 ', 'answer_start': 2274}], 'is_impossible': False}, {'question': 'What is the reduction in the number of childhood pneumonia cases?', 'id': 511, 'answers': [{'text': 'Recent data suggest that there has been a 25% decrease in the incidence of pneumonia, from 0.29 episodes per child year in low-and middle-income countries in 2000, to 0.22 episodes per child year in 2010 [3] . This is substantiated by a 58% decrease in pneumonia-associated disability-adjusted life years between 1990 and 2013, from 186 million to 78 million ', 'answer_start': 2356}], 'is_impossible': False}, {'question': 'How much  is the reduction in the childhood pneumonia deaths?', 'id': 512, 'answers': [{'text': 'Pneumonia deaths decreased from 1.8 million in 2000 to 900,000 in 2013', 'answer_start': 2772}], 'is_impossible': False}, {'question': 'Childhood pneumonia rate for  high income countries vs low and middle income countries.', 'id': 513, 'answers': [{'text': 'The incidence in high-income countries is estimated at 0.015 episodes per child year, compared to 0.22 episodes per child year in low-and middle-income countries [3] . On average, 1 in 66 children in high-income countries is affected by pneumonia per year, compared to 1 in 5 children in low-and middle-income countries.', 'answer_start': 3310}], 'is_impossible': False}, {'question': 'What  percentage of childhood pneumonia deaths occur outside hospital in low and middle  income countries?', 'id': 514, 'answers': [{'text': 'up to 81% of severe pneumonia deaths occur outside a hospital ', 'answer_start': 3756}], 'is_impossible': False}, {'question': 'Case Fatality Rates for Childhood Pneumonia in high income vs low and middle income countries.', 'id': 515, 'answers': [{'text': 'the case fatality rate is estimated to be almost 10-fold higher in low-and middle-income countries as compared to high-income countries', 'answer_start': 3872}], 'is_impossible': False}, {'question': 'How can childhood pneumonia affect the subsequent health of a person?', 'id': 516, 'answers': [{'text': 'Early life pneumonia can impair longterm lung health by decreasing lung function [6] . Severe or recurrent pneumonia can have a worse effect on lung function; increasing evidence suggests that chronic obstructive pulmonary disease might be related to early childhood pneumonia', 'answer_start': 4098}], 'is_impossible': False}, {'question': 'What is the increase in the risk of respiratory disease after having childhood pneumonia.', 'id': 517, 'answers': [{'text': 'The risk of developing at least one of the major sequelae was estimated as 6% after an ambulatory pneumonia event and 14% after an episode of hospitalized pneumonia.', 'answer_start': 4662}], 'is_impossible': False}, {'question': 'Which is the best method to identify pneumonia in a person?', 'id': 518, 'answers': [{'text': 'Chest radiologic changes have been considered the gold standard for defining a pneumonia event', 'answer_start': 5040}], 'is_impossible': False}, {'question': 'What is responsible for the reduction of radiologic pneumonia?', 'id': 520, 'answers': [{'text': 'Widespread use of pneumococcal conjugate vaccination and Haemophilus influenzae type B conjugate vaccination has decreased the incidence of radiologic pneumonia.', 'answer_start': 6100}], 'is_impossible': False}, {'question': 'What is the percentage reduction in pneumonia cases due to  vaccination?', 'id': 521, 'answers': [{'text': 'Haemophilus influenzae type B conjugate vaccination in high-burden communities, the vaccination was associated with an 18% decrease in radiologic pneumonia [13] . Introduction of pneumococcal conjugate vaccination was associated with a 26% decrease in radiologic pneumonia in California between 1995 and 1998 [14] . In vaccine efficacy trials in low-and middle-income countries, pneumococcal conjugate vaccination reduced radiologic pneumonia by 37% in the Gambia [15] , 25% in South Africa [16] and 26% in the Philippines ', 'answer_start': 6343}], 'is_impossible': False}, {'question': 'What is the revised WHO definition of Bacterial Pneumonia?', 'id': 522, 'answers': [{'text': ' A revised case definition of \"presumed bacterial pneumonia\" has been introduced, and this definition includes pneumonia cases with WHO-defined alveolar consolidation, as well as those with other abnormal chest radiograph infiltrates and a serum C-reactive protein of at least 40 mg/L ', 'answer_start': 7735}], 'is_impossible': False}, {'question': 'What is the reduction in bacterial pneumonia under the revised WHO  definition of bacterial pneumonia?', 'id': 524, 'answers': [{'text': ' Using the revised definition, the 10-valent pneumococcal conjugate vaccine (pneumococcal conjugate vaccination-10), had a vaccine efficacy of 22% in preventing presumed bacterial pneumonia in young children in South America [22] , and pneumococcal conjugate vaccination-13 had a vaccine efficacy of 39% in preventing presumed bacterial pneumonia in children older than 16 weeks who were not infected with human immunodeficiency virus (HIV) in South Africa [21]', 'answer_start': 8257}], 'is_impossible': False}, {'question': 'What caused the increase in the  incidence of empyema in children in the recent past?', 'id': 530, 'answers': [{'text': ' An increased incidence of empyema in children was noted in some high-income countries following pneumococcal conjugate vaccination-7 introduction, and this was attributed to pneumococcal serotypes not included in pneumococcal conjugate vaccination-7, especially 3 and 19A', 'answer_start': 9031}], 'is_impossible': False}, {'question': 'How have the incidence Empyema been reduced?', 'id': 532, 'answers': [{'text': 'These trends have been reversed since the introduction of pneumococcal conjugate vaccination-13. Data from the United States suggest that empyema decreased by 50% in children younger than 5 years ', 'answer_start': 9813}], 'is_impossible': False}, {'question': 'What pneumonia related or chest conditions indicate the need for child radiography?', 'id': 533, 'answers': [{'text': 'chest radiography should not be routinely performed in children with ambulatory pneumonia [31] [32] [33] . Indications for chest radiography include hospitalization, severe hypoxemia or respiratory distress, failed initial antibiotic therapy, or suspicion for other diseases (tuberculosis, inhaled foreign body) or complications. ', 'answer_start': 10334}], 'is_impossible': False}, {'question': 'What chest diseases and pneumonia were identified as leading causes prior to  the availability of vaccines?', 'id': 534, 'answers': [{'text': 'prior to availability of new conjugate vaccines confirmed S. pneumoniae and H. influenzae type B as the most important bacterial causes of pneumonia, with Staphylococcus aureus and Klebsiella pneumoniae associated with some severe cases. Respiratory syncytial virus was the leading viral cause, identified in 15-40% of pneumonia cases, followed by influenza A and B, parainfluenza, human metapneumovirus and adenovirus', 'answer_start': 11677}], 'is_impossible': False}, {'question': 'Why has pertussis immunity in infants has decreased in infants?', 'id': 535, 'answers': [{'text': 'Because pertussis immunity after acellular pertussis vaccination is less long-lasting than immunity after wild-type infection or whole-cell vaccination, many women of child-bearing age have waning pertussis antibody levels. ', 'answer_start': 13760}], 'is_impossible': False}, {'question': 'What is the effect of childhood tuberculosis in childhood pneumonia?', 'id': 536, 'answers': [{'text': 'A recent systematic review of tuberculosis as a comorbidity of childhood pneumonia reported culture-confirmed disease in about 8% of cases [50] . Because intrathoracic tuberculosis disease is only culture-confirmed in a minority of cases, the true burden could be even higher; tuberculosis could therefore be an important contributor to childhood pneumonia incidence and mortality in high-prevalence areas', 'answer_start': 15402}], 'is_impossible': False}, {'question': 'What are the risk factors in childhood pneumonia?', 'id': 537, 'answers': [{'text': ' incomplete or inadequate vaccination must be considered as a major preventable risk factor for childhood pneumonia. Other risk factors include low birth weight, which is associated with 3.2 times increased odds of severe pneumonia in low-and middle-income countries, and 1.8 times increased odds in high-income countries [51] . Similarly, lack of exclusive breastfeeding for the first 4 months of life increases odds of severe pneumonia by 2.7 times in low-and middle-income countries and 1.3 times in highincome countries. ', 'answer_start': 16115}], 'is_impossible': False}, {'question': 'How does air pollution affect the incidence of childhood pneumonia?', 'id': 538, 'answers': [{'text': ' Indoor air pollution from use of solid or biomass fuels increases odds of pneumonia by 1.6 times; lack of measles vaccination by the end of the first year of age increases odds of pneumonia by 1.8 times ', 'answer_start': 16987}], 'is_impossible': False}, {'question': 'What is the strongest risk factor for childhood pneumonia?', 'id': 539, 'answers': [{'text': 'The single strongest risk factor for pneumonia is HIV infection, which is especially prevalent in children in sub-Saharan Africa. HIV-infected children have 6 times increased odds of developing severe pneumonia or of death compared to HIV-uninfected children', 'answer_start': 17514}], 'is_impossible': False}, {'question': 'What is the global coverage of influenza and pneumonia vaccines?', 'id': 540, 'answers': [{'text': ' By the end of 2015, Haemophilus influenzae type B conjugate vaccination had been introduced in 73 countries, with global coverage estimated at 68%. However, inequities are still apparent among regions: in the Americas coverage is estimated at 90%, while in the Western Pacific it is only 25%. By 2015, pneumococcal conjugate vaccination had been introduced into 54 countries, with global coverage of 35% for three doses of pneumococcal conjugate vaccination for infant populations', 'answer_start': 18325}], 'is_impossible': False}, {'question': 'Is influenza vaccination during pregnancy safe? How long does it protect the child?', 'id': 541, 'answers': [{'text': 'Influenza vaccination during pregnancy is safe, provides reasonable maternal protection against influenza, and also protects infants for a limited period from confirmed influenza infection (vaccine efficacy 63% in Bangladesh [63] and 50.4% in South Africa [64] ). However as antibody levels drop sharply after birth, infant protection does not persist much beyond 8 weeks', 'answer_start': 19437}], 'is_impossible': False}, {'question': 'What is emphyema?', 'id': 923, 'answers': [{'text': 'a rare complication of pneumonia', 'answer_start': 8998}], 'is_impossible': False}, {'question': 'How is the term end point consolidation described  with regard to pneumonia diagnosis?', 'id': 4573, 'answers': [{'text': 'as a dense or fluffy opacity that occupies a portion or whole of a lobe, or the entire lung.', 'answer_start': 5539}], 'is_impossible': False}], 'context': 'Community-acquired pneumonia in children — a changing spectrum of disease\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC5608782/\\n\\nSHA: eecb946b106a94f26a79a964f0160e8e16f79f42\\n\\nAuthors: le Roux, David M.; Zar, Heather J.\\nDate: 2017-09-21\\nDOI: 10.1007/s00247-017-3827-8\\nLicense: cc-by\\n\\nAbstract: Pneumonia remains the leading cause of death in children outside the neonatal period, despite advances in prevention and management. Over the last 20 years, there has been a substantial decrease in the incidence of childhood pneumonia and pneumonia-associated mortality. New conjugate vaccines against Haemophilus influenzae type b and Streptococcus pneumoniae have contributed to decreases in radiologic, clinical and complicated pneumonia cases and have reduced hospitalization and mortality. The importance of co-infections with multiple pathogens and the predominance of viral-associated disease are emerging. Better access to effective preventative and management strategies is needed in low- and middle-income countries, while new strategies are needed to address the residual burden of disease once these have been implemented.\\n\\nText: Pneumonia has been the leading cause of death in children younger than 5 years for decades. Although there have been substantial decreases in overall child mortality and in pneumonia-specific mortality, pneumonia remains the major single cause of death in children outside the neonatal period, causing approximately 900,000 of the estimated 6.3 million child deaths in 2013 [1] . Substantial advances have occurred in the understanding of risk factors and etiology of pneumonia, in development of standardized case definitions, and in prevention with the production of improved vaccines and in treatment. Such advances have led to changes in the epidemiology, etiology and mortality from childhood pneumonia. However in many areas access to these interventions remains sub-optimal, with large inequities between and within countries and regions. In this paper we review the impact of recent preventative and management advances in pneumonia epidemiology, etiology, radiologic presentation and outcome in children.\\n\\nThe overall burden of childhood pneumonia has been reduced substantially over the last decade, despite an increase in the global childhood population from 605 million in 2000 to 664 million in 2015 [2] . Recent data suggest that there has been a 25% decrease in the incidence of pneumonia, from 0.29 episodes per child year in low-and middle-income countries in 2000, to 0.22 episodes per child year in 2010 [3] . This is substantiated by a 58% decrease in pneumonia-associated disability-adjusted life years between 1990 and 2013, from 186 million to 78 million as estimated in the Global Burden of Disease study [1] . Pneumonia deaths decreased from 1.8 million in 2000 to 900,000 in 2013 [1] . These data do not reflect the full impact of increasingly widespread use of pneumococcal conjugate vaccine in low-and middle-income countries because the incidence of pneumonia and number of deaths are likely to decrease still further as a result of this widespread intervention [4] .\\n\\nNotwithstanding this progress, there remains a disproportionate burden of disease in low-and middle-income countries, where more than 90% of pneumonia cases and deaths occur. The incidence in high-income countries is estimated at 0.015 episodes per child year, compared to 0.22 episodes per child year in low-and middle-income countries [3] . On average, 1 in 66 children in high-income countries is affected by pneumonia per year, compared to 1 in 5 children in low-and middle-income countries. Even within low-and middleincome countries there are regional inequities and challenges with access to health care services: up to 81% of severe pneumonia deaths occur outside a hospital [5] . In addition to a higher incidence of pneumonia, the case fatality rate is estimated to be almost 10-fold higher in low-and middle-income countries as compared to high-income countries [3, 5] .\\n\\nChildhood pneumonia can also lead to significant morbidity and chronic disease. Early life pneumonia can impair longterm lung health by decreasing lung function [6] . Severe or recurrent pneumonia can have a worse effect on lung function; increasing evidence suggests that chronic obstructive pulmonary disease might be related to early childhood pneumonia [7, 8] . A meta-analysis of the risk of long-term outcomes after childhood pneumonia categorized chronic respiratory sequelae into major (restrictive lung disease, obstructive lung disease, bronchiectasis) and minor (chronic bronchitis, asthma, abnormal pulmonary function) groups [9] . The risk of developing at least one of the major sequelae was estimated as 6% after an ambulatory pneumonia event and 14% after an episode of hospitalized pneumonia. Because respiratory diseases affect almost 1 billion people globally and are a major cause of mortality and morbidity [10] , childhood pneumonia might contribute to substantial morbidity across the life course.\\n\\nChest radiologic changes have been considered the gold standard for defining a pneumonia event [11] because clinical findings can be subjective and clinical definitions of pneumonia can be nonspecific. In 2005, to aid in defining outcomes of pneumococcal vaccine studies, the World Health Organization\\'s (WHO) standardized chest radiograph description defined a group of children who were considered most likely to have pneumococcal pneumonia [12] . The term \"end-point consolidation\" was described as a dense or fluffy opacity that occupies a portion or whole of a lobe, or the entire lung. \"Other infiltrate\" included linear and patchy densities, peribronchial thickening, minor patchy infiltrates that are not of sufficient magnitude to constitute primary end-point consolidation, and small areas of atelectasis that in children can be difficult to distinguish from consolidation. \"Primary end-point pneumonia\" included either end-point consolidation or a pleural effusion associated with a pulmonary parenchymal infiltrate (including \"other\" infiltrate).\\n\\nWidespread use of pneumococcal conjugate vaccination and Haemophilus influenzae type B conjugate vaccination has decreased the incidence of radiologic pneumonia. In a review of four randomized controlled trials and two case-control studies of Haemophilus influenzae type B conjugate vaccination in high-burden communities, the vaccination was associated with an 18% decrease in radiologic pneumonia [13] . Introduction of pneumococcal conjugate vaccination was associated with a 26% decrease in radiologic pneumonia in California between 1995 and 1998 [14] . In vaccine efficacy trials in low-and middle-income countries, pneumococcal conjugate vaccination reduced radiologic pneumonia by 37% in the Gambia [15] , 25% in South Africa [16] and 26% in the Philippines [17] .\\n\\nThe WHO radiologic case definition was not intended to distinguish bacterial from viral etiology but rather to define a sub-set of pneumonia cases in which pneumococcal infection was considered more likely and to provide a set of standardized definitions through which researchers could achieve broad agreement in reporting chest radiographs. However, despite widespread field utilization, there are concerns regarding inter-observer repeatability. There has been good consensus for the description of lobar consolidation but significant disagreement on the description of patchy and perihilar infiltrates [18, 19] . In addition, many children with clinically severe lung disease do not have primary end-point pneumonia: in one pre-pneumococcal conjugate vaccination study, only 34% of children hospitalized with pneumonia had primary end-point pneumonia [20] . A revised case definition of \"presumed bacterial pneumonia\" has been introduced, and this definition includes pneumonia cases with WHO-defined alveolar consolidation, as well as those with other abnormal chest radiograph infiltrates and a serum C-reactive protein of at least 40 mg/L [21, 22] . This definition has been shown to have greater sensitivity than the original WHO radiologic definition of primary end-point pneumonia for detecting the burden of pneumonia prevented by pneumococcal conjugate vaccination [23] . Using the revised definition, the 10-valent pneumococcal conjugate vaccine (pneumococcal conjugate vaccination-10), had a vaccine efficacy of 22% in preventing presumed bacterial pneumonia in young children in South America [22] , and pneumococcal conjugate vaccination-13 had a vaccine efficacy of 39% in preventing presumed bacterial pneumonia in children older than 16 weeks who were not infected with human immunodeficiency virus (HIV) in South Africa [21] . Thus there is convincing evidence that pneumococcal conjugate vaccination decreases the incidence of radiologic pneumonia; however there is no evidence to suggest that pneumococcal conjugate vaccination modifies the radiologic appearance of pneumococcal pneumonia.\\n\\nEmpyema is a rare complication of pneumonia. An increased incidence of empyema in children was noted in some high-income countries following pneumococcal conjugate vaccination-7 introduction, and this was attributed to pneumococcal serotypes not included in pneumococcal conjugate vaccination-7, especially 3 and 19A [24] . In the United States, evidence from a national hospital database suggests that the incidence of empyema increased 1.9-fold between 1996 and 2008 [25] . In Australia, the incidence rate ratio increased by 1.4 times when comparing the pre-pneumococcal conjugate vaccination-7 period (1998 to 2004) to the post-pneumococcal conjugate vaccination-7 period (2005 to 2010) [26] . In Scotland, incidence of empyema in children rose from 6.5 per million between 1981 and 1998, to 66 per million in 2005 [27] . These trends have been reversed since the introduction of pneumococcal conjugate vaccination-13. Data from the United States suggest that empyema decreased by 50% in children younger than 5 years [28] ; similarly, data from the United Kingdom and Scotland showed substantial reduction in pediatric empyema following pneumococcal conjugate vaccination-13 introduction [29, 30] .\\n\\nSeveral national guidelines from high-income countries, as well as the WHO recommendations for low-and middleincome countries, recommend that chest radiography should not be routinely performed in children with ambulatory pneumonia [31] [32] [33] . Indications for chest radiography include hospitalization, severe hypoxemia or respiratory distress, failed initial antibiotic therapy, or suspicion for other diseases (tuberculosis, inhaled foreign body) or complications. However, point-of-care lung ultrasound is emerging as a promising modality for diagnosing childhood pneumonia [34] .\\n\\nIn addition to the effect on radiologic pneumonia, pneumococcal conjugate vaccination reduces the risk of hospitalization from viral-associated pneumonia, probably by reducing bacterial-viral co-infections resulting in severe disease and hospitalization [35] . An analysis of ecological and observational studies of pneumonia incidence in different age groups soon after introduction of pneumococcal conjugate vaccination-7 in Canada, Italy, Australia, Poland and the United States showed decreases in all-cause pneumonia hospitalizations ranging from 15% to 65% [36] . In the United States after pneumococcal conjugate vaccination-13 replaced pneumococcal conjugate vaccination-7, there was a further 17% decrease in hospitalizations for pneumonia among children eligible for the vaccination, and a further 12% decrease among unvaccinated adults [28] .\\n\\nA systematic review of etiology studies prior to availability of new conjugate vaccines confirmed S. pneumoniae and H. influenzae type B as the most important bacterial causes of pneumonia, with Staphylococcus aureus and Klebsiella pneumoniae associated with some severe cases. Respiratory syncytial virus was the leading viral cause, identified in 15-40% of pneumonia cases, followed by influenza A and B, parainfluenza, human metapneumovirus and adenovirus [37] .\\n\\nMore recent meta-analyses of etiology data suggest a changing pathogen profile, with increasing recognition that clinical pneumonia is caused by the sequential or concurrent interaction of more than one organism. Severe disease in particular is often caused by multiple pathogens. With high coverage of pneumococcal conjugate vaccination and Haemophilus influenzae type B conjugate vaccination, viral pathogens increasingly predominate [38] . In recent case-control studies, at least one virus was detected in 87% of clinical pneumonia cases in South Africa [39] , while viruses were detected in 81% of radiologic pneumonia cases in Sweden [40] . In a large multi-center study in the United States, viral pathogens were detected in 73% of children hospitalized with radiologic pneumonia, while bacteria were detected in only 15% of cases [41] . A meta-analysis of 23 case-control studies of viral etiology in radiologically confirmed pneumonia in children, completed up to 2014, reported good evidence of causal attribution for respiratory syncytial virus, influenza, metapneumovirus and parainfluenza virus [42] . However there was no consistent evidence that many other commonly described viruses, including rhinovirus, adenovirus, bocavirus and coronavirus, were more commonly isolated from cases than from controls. Further attribution of bacterial etiology is difficult because it is often not possible to distinguish colonizing from pathogenic bacteria when they are isolated from nasal specimens [43] .\\n\\nAnother etiology is pertussis. In the last decade there has also been a resurgence in pertussis cases, especially in highincome countries [44] . Because pertussis immunity after acellular pertussis vaccination is less long-lasting than immunity after wild-type infection or whole-cell vaccination, many women of child-bearing age have waning pertussis antibody levels. Their infants might therefore be born with low transplacental anti-pertussis immunoglobulin G levels, making them susceptible to pertussis infection before completion of the primary vaccination series [45] . In 2014, more than 40,000 pertussis cases were reported to the Centers for Disease Control and Prevention in the United States; in some states, population-based incidence rates are higher than at any time in the last 70 years [44] . In contrast, most low-and middleincome countries use whole-cell pertussis vaccines and the numbers of pertussis cases in those countries were stable or decreasing until 2015 [46] . However recent evidence from South Africa (where the acellular vaccine is used) shows an appreciable incidence of pertussis among infants presenting with acute pneumonia: 2% of clinical pneumonia cases among infants enrolled in a birth cohort were caused by pertussis [39] , and 3.7% of infants and young children presenting to a tertiary academic hospital had evidence of pertussis infection [47] .\\n\\nSimilarly, childhood tuberculosis is a major cause of morbidity and mortality in many low-and middle-income countries, and Mycobacterium tuberculosis has increasingly been recognized as a pathogen in acute pneumonia in children living in high tuberculosis-prevalence settings. Postmortem studies of children dying from acute respiratory illness have commonly reported M. tuberculosis [48, 49] . A recent systematic review of tuberculosis as a comorbidity of childhood pneumonia reported culture-confirmed disease in about 8% of cases [50] . Because intrathoracic tuberculosis disease is only culture-confirmed in a minority of cases, the true burden could be even higher; tuberculosis could therefore be an important contributor to childhood pneumonia incidence and mortality in high-prevalence areas.\\n\\nChildhood pneumonia and clinically severe disease result from a complex interaction of host and environmental risk factors [37] . Because of the effectiveness of pneumococcal conjugate vaccination and Haemophilus influenzae type B conjugate vaccination for prevention of radiologic and clinical pneumonia, incomplete or inadequate vaccination must be considered as a major preventable risk factor for childhood pneumonia. Other risk factors include low birth weight, which is associated with 3.2 times increased odds of severe pneumonia in low-and middle-income countries, and 1.8 times increased odds in high-income countries [51] . Similarly, lack of exclusive breastfeeding for the first 4 months of life increases odds of severe pneumonia by 2.7 times in low-and middle-income countries and 1.3 times in highincome countries. Markers of undernutrition are strong risk factors for pneumonia in low-and middle-income countries only, with highly significant odds ratios for underweight for age (4.5), stunting (2.6) and wasting (2.8) . Household crowding has uniform risk, with odds ratios between 1.9 and 2.3 in both low-and middle-income countries and high-income countries. Indoor air pollution from use of solid or biomass fuels increases odds of pneumonia by 1.6 times; lack of measles vaccination by the end of the first year of age increases odds of pneumonia by 1.8 times [51] . It is estimated that the prevalence of these critical risk factors in low-and middle-income countries decreased by 25% between 2000 and 2010, contributing to reductions in pneumonia incidence and mortality in low-and middle-income countries, even in countries where conjugate vaccines have not been available [3] .\\n\\nThe single strongest risk factor for pneumonia is HIV infection, which is especially prevalent in children in sub-Saharan Africa. HIV-infected children have 6 times increased odds of developing severe pneumonia or of death compared to HIV-uninfected children [52] . Since the effective prevention of mother-to-child transmission of HIV, there is a growing population of HIV-exposed children who are uninfected; their excess risk of pneumonia, compared to HIV unexposed children, has been described as 1.3-to 3.4-fold higher [53] [54] [55] [56] [57] .\\n\\nThe pneumococcal conjugate vaccination and Haemophilus influenzae type B conjugate vaccination have been effective tools to decrease pneumonia incidence, severity and mortality [58, 59] . However, equitable coverage and access to vaccines remains sub-optimal. By the end of 2015, Haemophilus influenzae type B conjugate vaccination had been introduced in 73 countries, with global coverage estimated at 68%. However, inequities are still apparent among regions: in the Americas coverage is estimated at 90%, while in the Western Pacific it is only 25%. By 2015, pneumococcal conjugate vaccination had been introduced into 54 countries, with global coverage of 35% for three doses of pneumococcal conjugate vaccination for infant populations [60] . To address this issue, the WHO\\'s Global Vaccine Access Plan initiative was launched to make life-saving vaccines more equitably available. In addition to securing guarantees for financing of vaccines, the program objectives include building political will in low-and middle-income countries to commit to immunization as a priority, social marketing to individuals and communities, strengthening health systems and promoting relevant local research and development innovations [61] .\\n\\nMaternal vaccination to prevent disease in the youngest infants has been shown to be effective for tetanus, influenza and pertussis [62] . Influenza vaccination during pregnancy is safe, provides reasonable maternal protection against influenza, and also protects infants for a limited period from confirmed influenza infection (vaccine efficacy 63% in Bangladesh [63] and 50.4% in South Africa [64] ). However as antibody levels drop sharply after birth, infant protection does not persist much beyond 8 weeks [65] . Recently respiratory syncytial virus vaccination in pregnancy has been shown to be safe and immunogenic, and a phase-3 clinical trial of efficacy at preventing respiratory syncytial virus disease in infants is under way [66] . Within a decade, respiratory syncytial virus in infancy might be vaccine-preventable, with further decreases in pneumonia incidence, morbidity and mortality [67] .\\n\\nImproved access to health care, better nutrition and improved living conditions might contribute to further decreases in childhood pneumonia burden. The WHO Integrated Global Action Plan for diarrhea and pneumonia highlights many opportunities to protect, prevent and treat children [68] . Breastfeeding rates can be improved by programs that combine education and counseling interventions in homes, communities and health facilities, and by promotion of baby-friendly hospitals [69] . Improved home ventilation, cleaner cooking fuels and reduction in exposure to cigarette smoke are essential interventions to reduce the incidence and severity of pneumonia [70, 71] . Prevention of pediatric HIV is possible by providing interventions to prevent mother-to-child transmission [72] . Early infant HIV testing and early initiation of antiretroviral therapy and cotrimoxazole prophylaxis can substantially reduce the incidence of community-acquired pneumonia among HIV-infected children [73] . Community-based interventions reduce pneumonia mortality and have the indirect effect of improved-careseeking behavior [58] . If these cost-effective interventions were scaled up, it is estimated that 67% of pneumonia deaths in lowand middle-income countries could be prevented by 2025 [58] .\\n\\nCase management of pneumonia is a strategy by which severity of disease is classified as severe or non-severe. All children receive early, appropriate oral antibiotics, and severe cases are referred for parenteral antibiotics. When implemented in highburden areas before the availability of conjugate vaccines, case management as part of Integrated Management of Childhood Illness was associated with a 27% decrease in overall child mortality, and 42% decrease in pneumonia-specific mortality [74] . However the predominance of viral causes of pneumonia and low case fatality have prompted concern about overuse of antibiotics. Several randomized controlled trials comparing oral antibiotics to placebo for non-severe pneumonia have been performed [75] [76] [77] and others are ongoing [78] . In two studies, performed in Denmark and in India, outcomes of antibiotic and placebo treatments were equivalent [76, 77] . In the third study, in Pakistan, there was a non-significant 24% vs. 20% rate of failure in the placebo group, which was deemed to be non-equivalent to the antibiotic group [75] . Furthermore, because WHO-classified non-severe pneumonia and bronchiolitis might be considered within a spectrum of lower respiratory disease, many children with clinical pneumonia could actually have viral bronchiolitis, for which antibiotics are not beneficial [79] . This has been reflected in British [33] and Spanish [31] national pneumonia guidelines, which do not recommend routine antibiotic treatment for children younger than 2 years with evidence of pneumococcal conjugate vaccination who present with non-severe pneumonia. The United States\\' national guidelines recommend withholding antibiotics in children up to age 5 years presenting with non-severe pneumonia [32] . However, given the high mortality from pneumonia in low-and middle-income countries, the lack of easy access to care, and the high prevalence of risk factors for severe disease, revised World Health Organization pneumonia guidelines still recommend antibiotic treatment for all children who meet the WHO pneumonia case definitions [80] .\\n\\nUse of supplemental oxygen is life-saving, but this is not universally available in low-and middle-income countries; it is estimated that use of supplemental oxygen systems could reduce mortality of children with hypoxic pneumonia by 20% [81] . Identifying systems capacity to increase availability of oxygen in health facilities, and identifying barriers to further implementation are among the top 15 priorities for future childhood pneumonia research [82] . However, up to 81% of pneumonia deaths in 2010 occurred outside health facilities [5] , so there are major challenges with access to health services and health-seeking behavior of vulnerable populations. Identifying and changing the barriers to accessing health care is an important area with the potential to impact the survival and health of the most vulnerable children [82] .\\n\\nMuch progress has been made in decreasing deaths caused by childhood pneumonia. Improved socioeconomic status and vaccinations, primarily the conjugate vaccines (against Haemophilus influenzae and pneumococcus), have led to substantial reductions in the incidence and severity of childhood pneumonia. Stronger strategies to prevent and manage HIV have reduced HIV-associated pneumonia deaths. However, despite the substantial changes in incidence, etiology and radiology globally, there remain inequities in access to care and availability of effective interventions, especially in low-and middle-income countries. Effective interventions need to be more widely available and new interventions developed for the residual burden of childhood pneumonia.', 'document_id': 1571}]}, {'paragraphs': [{'qas': [{'question': 'What factors make H5N1 a worldwide threat to public health?', 'id': 5217, 'answers': [{'text': 'rapid spread and high pathogenicity', 'answer_start': 1731}], 'is_impossible': False}, {'question': 'What are the symptoms of H5N1 infection in humans?', 'id': 5218, 'answers': [{'text': 'fever, encephalitis, pneumonia, and severe acute respiratory syndrome (SARS)', 'answer_start': 1837}], 'is_impossible': False}, {'question': 'Name some medications used to treat influenza.', 'id': 5219, 'answers': [{'text': 'zanamivir (Relenza) and oseltamivir (Tamiflu)', 'answer_start': 2356}], 'is_impossible': False}, {'question': 'Why have antiretrovirals medications had limited benefit in treating influenza?', 'id': 5220, 'answers': [{'text': 'drug-resistance and frequent antigenic mutation', 'answer_start': 2472}], 'is_impossible': False}, {'question': 'What is the focus of the current study?', 'id': 5221, 'answers': [{'text': 'effect of EAP against H5N1 influenza infection', 'answer_start': 4447}], 'is_impossible': False}, {'question': 'What is the result of the current study?', 'id': 5222, 'answers': [{'text': 'the anti-H5N1 effects of EAP offer an alternative strategy for developing antiinfluenza agents', 'answer_start': 4621}], 'is_impossible': False}, {'question': 'How do the polysaccharides in plants effect the immune response?', 'id': 5223, 'answers': [{'text': 'enhance the secretion of cytokines and chemokines, such as TNF-, IL-6, IL-8, and IL-12', 'answer_start': 14834}], 'is_impossible': False}, {'question': 'What does this study demonstrate?', 'id': 5224, 'answers': [{'text': 'EAP leaf extract is a prophylactic and immune enhancement agent against H5N1 influenza virus infection', 'answer_start': 21976}], 'is_impossible': False}, {'question': 'In this study, how did treatment of EAP after infection affect survival?', 'id': 5225, 'answers': [{'text': 'did not provide a survival advantage', 'answer_start': 19815}], 'is_impossible': False}], 'context': 'Immunomodulatory Activity and Protective Effects of Polysaccharide from Eupatorium adenophorum Leaf Extract on Highly Pathogenic H5N1 Influenza Infection\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC3789439/\\n\\nSHA: efba2008a6ccf1ad2614aebd79a6a741ea6538b9\\n\\nAuthors: Jin, Yi; Zhang, Yuewei; Wan, Chunyan; Wang, Hongjun; Hou, Lingyu; Chang, Jianyu; Fan, Kai; Xie, Xiangming\\nDate: 2013-09-18\\nDOI: 10.1155/2013/194976\\nLicense: cc-by\\n\\nAbstract: The development of novel broad-spectrum, antiviral agents against H5N1 infection is urgently needed. In this study, we evaluated the immunomodulatory activities and protective effect of Eupatorium adenophorum polysaccharide (EAP) against the highly pathogenic H5N1 subtype influenza virus. EAP treatment significantly increased the production of IL-6, TNF-α, and IFN-γ both in vivo and in vitro as measured by qPCR and ELISA. In a mouse infection model, intranasal administration of EAP at a dose of 25 mg/kg body weight prior to H5N1 viral challenge efficiently inhibited viral replication, decreased lung lesions, and increased survival rate. We further evaluated the innate immune recognition of EAP, as this process is regulated primarily Dectin-1 and mannose receptor (MR). These results indicate that EAP may have immunomodulatory properties and a potential prophylactic effect against H5N1 influenza infection. Our investigation suggests an alternative strategy for the development of novel antiinfluenza agents and benefits of E. adenophorum products.\\n\\nText: Highly pathogenic H5N1 subtype influenza virus can be transmitted directly from poultry to human and cause acute respiratory infections. Pandemic influenza virus H5N1 posed a worldwide threat to the public health because of rapid spread and high pathogenicity [1, 2] . The symptoms in animals or humans infected with H5N1 include fever, encephalitis, pneumonia, and severe acute respiratory syndrome (SARS) [3, 4] . The World Health Organization reported 622 human cases of highly pathogenic H5N1 influenza virus infection, including 371 deaths (a mortality rate >50%), from 2003 to 2013 (http://www.who.int/ influenza/human animal interface/H5N1 cumulative table archives/en/index.html). Currently, the most effective preventive measure against the influenza virus is vaccination. Several antiinfluenza medications have been widely used, including zanamivir (Relenza) and oseltamivir (Tamiflu).\\n\\nUnfortunately, their benefits have been significantly restricted by drug-resistance and frequent antigenic mutation [5, 6] . Therefore, the development of novel antiinfluenza agents against the H5N1 subtype is very important.\\n\\nThe invasive plant Eupatorium adenophorum, native to Central America, has a strong ability to adapt to different environments all over the world. This plant first invaded southern Yunnan Province (China) in the 1940s from Burma and Vietnam, and quickly spread across southwestern China throughout the 1950s [7, 8] . Over the past 50 years, E. adenophorum has seriously impacted the ecological environment in China\\'s middle subtropical zones, including Yunnan, Guizhou, Sichuan, and Guangxi Provinces, by encroaching farmlands, pasture fields, and forests [7] . Manual, chemical, or biological control of E. adenophorum has hindered its comprehensive development and utilization for economic benefit. Many bioactive components isolated from E. adenophorum have shown antimicrobial activity and immunomodulating 2 Evidence-Based Complementary and Alternative Medicine properties [9] . In a recent study, the anti-inflammatory properties of ethanolic leaf extract was evaluated [10] . However, there have been few reports addressing the bioactivity of E. adenophorum polysaccharide (EAP).\\n\\nThe immunomodulating properties and therapeutic potential of a large number of botanical polysaccharides have been reported [11] . Several polysaccharides from Cordyceps militaris, Portulaca oleracea, Gracilaria lemaneiformis, Gyrodinium impudium, and Panax ginseng have been described as efficacious antiinfluenza agents against H1N1 and H3N2 strains [12] [13] [14] [15] . In recent reports, polysaccharidebased adjuvants enhanced the immunogenicity and improved the protective efficacy of H5N1 vaccines in animal infection models [16, 17] . However, to our knowledge there have not been any reports regarding the treatment with EAP against highly pathogenic H5N1 influenza.\\n\\nIn the present study, we investigated the potential effect of EAP against H5N1 influenza infection in a mouse model. Immune enhancement effects and the innate immune recognition of EAP were also evaluated. Our results suggest the anti-H5N1 effects of EAP offer an alternative strategy for developing antiinfluenza agents and the utilization of E. adenophorum products.\\n\\nVirus. The H5N1 influenza virus (A/bar-headed goose/ Qinghai/1/2010) used in this study was isolated from Qinghai Lake in May 2010. This isolate is highly pathogenic in poultry, mouse, and Madin-Darby canine kidney (MDCK) cells. The virus was propagated in MDCK cells at 37 ∘ C for 48 h, and the viral supernatant was harvested, aliquoted, and stored at −80 ∘ C. Viral titers were determined by plaque assay as described previously [18] .\\n\\nAnimal and Cells. 8-10-week-old Female BALB/c mice were obtained from Vital River Laboratories (Beijing, China), and the original breeding pairs were purchased from Charles River (Beijing, China). Mice were raised in independent ventilated cages (IVC) and received pathogen-free food and water. Animal treatments were governed by the Regulations of Experimental Animals of Beijing Authority, and approved by the Animal Ethics Committee of the China Agriculture University.\\n\\nThe mouse leukemic monocyte macrophage Raw 264.7 cell line, human lung adenocarcinoma epithelial A549 cell line, and Madin-Darby canine kidney (MDCK) cell lines were provided by the Cell Resource Center of Peking Union Medical College. The cells were cultured and maintained according to the supplier\\'s recommendations.\\n\\nYunnan province, China. The leaves were sliced and dried in shade. 100 g dried materials were powdered in a mixer and then filtered with 40 meshes. Leaf powder was extracted by ultrasonic treatment with 1000 mL of distilled water for 45 min. The supernatant was collected and the precipitate resuspended in 1000 mL of distilled water and again extracted by ultrasonic treatment for 30 min. The resulting supernatant was combined with that obtained from the first ultrasonic treatment. The final aqueous fraction was evaporated to dryness in a rotary evaporator. The residue obtained was dissolved in distilled water and kept frozen at 4 ∘ C.\\n\\nThe extract was centrifuged at 3000 g/min for 25 min and concentrated under 80 ∘ C for 8 h to prepare polysaccharide. The supernatant was then deproteinized using the Sevag method, and dialyzed against water for 48 h. The final liquid was mixed with three-fold volume of 95% ethanol (v/v) and centrifuged at 3000 g/min for 10 min. The precipitates were successively washed with absolute ethanol, ether, and dried under vacuum at 40 ∘ C to obtained the crude polysaccharide (yield = 1.2%). EAP content was determined by the phenol-H 2 SO 4 method [19] .\\n\\nVitro. 2.5 mL A549 and Raw 264.7 cells (4 × 10 5 /mL) per well were plated in 6-well plates and cultured at 37 ∘ C under 5% CO 2 for 24 h. Media was removed and 2.5 mL culture medium containing different concentrations of EAP (50, 100, 200 g/mL) was added to each well. Controls were treated with phosphate-buffered saline (PBS). Cells were collected 36 h after treatment for RNA extraction and quantitative polymerase chain reaction (qPCR).\\n\\nAssay. Mice were administrated EAP at a dose of 5, 10, 25, or 50 mg/kg body weight, intranasally once daily for 5 days before the challenge. Control mice were administered PBS using the same schedule. Influenza virus stocks were diluted in PBS. Mice were anesthetized with Zotile (Virbac, France) intramuscularly at 15 mg/kg (body weight) and then infected intranasally with 120 plaqueforming units (PFU) of H5N1 influenza virus in 50 L. The lung tissue of five mice per group was collected on day 0 before challenge for qPCR and ELISA. Lung tissue from another five mice on day 3 postinfection was collected for plaque assay and qPCR. Ten mice per group were observed for survival for 14 days and body weights recorded.\\n\\n2.6. Plaque Assay. MDCK cells were cultured in DMEM (Hyclone Laboratories, Logan, UT, USA) containing 10% FBS (Hyclone Laboratories), 100 U/mL penicillin, and 100 g/mL streptomycin (Invitrogen, San Diego, CA, USA). Lung tissue supernatant was diluted 10-fold and added to a cell monolayer covered by semisolid agar containing 0.5 g/mL of trypsin TPCK (Sigma-Aldrich, St. Louis, MO, USA). Plates were incubated at 37 ∘ C, 5% CO 2 for 60-72 h and stained with 1% crystal violet.\\n\\nTotal RNA from 1 × 10 6 cells or 10 mg lung tissue were prepared by Trizol (Invitrogen) according to the manufacturer\\'s instructions. DNaseItreated RNA (0.2 g) was reverse transcribed into cDNA using random primers. The expression of the hemagglutinin (HA) gene of H5N1 influenza virus was detected by qPCR using the Power SYBR Green PCR Master Mix kit (Applied Biosystems, Foster City, CA, USA). The following primers AGG CAC CA-3  5 -CTC CTT AAT GTC ACG CAC GAT TTC-3  h IL-6  5 -CCT TCG GTC CAG TTG CCT TCT-3  5 -CCA GTG CCT CTT TGC TGC TTT C-3  h IFN were used: forward primer, 5 -CGC AGT ATT CAG AAG AAG CAAGAC-3 ; and reverse primer, 5 -TCC ATA AGG ATA GAC CAG CTA CCA-3 . The reaction was run on an ABI 7500 thermal cycler with an initial denaturation step at 95 ∘ C for 10 min, followed by 40 cycles of 95 ∘ C for 15 s, 56 ∘ C for 30 s, and 72 ∘ C for 40 s. The copy number of the HA gene was calculated by 7500 software v2.0 (Applied Biosystems) using an HA-containing plasmid of known concentration as a standard.\\n\\nRelative qPCR was performed for other eight genes: hactin, h IL-6, h IFN-, and hTNF-for A549 cells; mactin, mTLR-2, mTLR-4, mDectin-1, mMR, mIL-6, mIFN-, and mTNF-for Raw264.7 cells. The sequences of primers were shown in Table 1 . The reaction was run with 95 ∘ C for 10 min, followed by 40 cycles of denaturation at 95 ∘ C for 15 sec, annealing at 52 ∘ C for 30 s, and extension at 72 ∘ C for 40 s. The fold change in gene expression was normalized to controls (naive mice) by 2 −ΔΔCT using -actin as an internal standard [20] .\\n\\n2.8. ELISA. IL-6, TNF-, and IFN-levels in lung were tested with ELISA kits (Boster, Wuhan, China) according to the manufacturer\\'s protocol. One gram of lung tissue from each mouse was ground in 1 mL PBS and centrifuged for 20 min at 5000 rpm. The supernatants were collected and diluted 10fold for ELISA. 2.10. Statistical Analysis. The statistical analysis was performed using one-way ANOVAs with SPSS 12.0 (SPSS Taiwan Corp., Taiwan), and < 0.05 was considered significant.\\n\\nMany botanical polysaccharides exhibit an immunomodulatory effect [11] . To determine the immunomodulatory properties of EAP, we investigated the potential effect of the polysaccharides on A549 and Raw264.7 cells. Cells were treated with various concentrations of EAP (50, 100, 200 g/mL) for 36 h. The mRNA levels of IL-6, TNF-, and IFN-were detected by qPCR. Figure 1 shows the immunomodulatory activities of EAP in vitro. Various concentrations of EAP triggered a strong secretion of IL-6, TNF-, and IFN-in a dosedependent manner both in A549 cells (Figures 1(a)-1(c) ) and Raw264.7 cells (Figures 1(d) -1(f)) compared with the PBS treatment group.\\n\\nTo test whether EAP could protect H5N1 infected mice, mice were treated with EAP at a dose of 5, 10, 25, or 50 mg/kg body weight intranasally once daily for 5 days prior to viral challenge with 120 PFU. Ten mice per group were monitored for 14 days for the survival rate. As shown in Figure 2 (a), all mice receiving PBS died at day 11. Mice administrated 25 mg/kg EAP had a survival rate of 50% at day 14, which was significantly higher than those receiving PBS (by log rank analysis). EAP treatment of 10 mg/kg and 50 mg/kg also appeared to have a survival advantage, but not statistically significant. This result suggests that the protective effect of EAP against H5N1 infection requires a moderate dose. EAP treatment also alleviated weight loss in infected mice (Figure 2(b) ).\\n\\nTo determine the viral load in the lung of the infected mice, plaque assays and qPCR were performed. The pulmonary viral titers in the EAP (25 mg/kg) group were significantly lower than the titers in the mice that received PBS at day 3 postinfection (Figures 2(c) and 2(d) ). These data clearly indicate that intranasal administration of EAP controls H5N1 viral replication and improves survival rates in a mouse model. The protective effect of EAP against H5N1 virus is likely due to its immunomodulatory properties. To detect IL-6, TNF-, and IFN-expression, lungs of five mice per group were collected at day 0 before infection and tested by qPCR and ELISA. The mRNA levels in the EAP group (25 mg/kg) were significantly higher than those in the PBS control (naive mice) (Figures 3(a)-3(c) ). Soluble cytokine levels at day 0 were measured by ELISA, and results were consistent with the qPCR results, even though IFN-production in the EAP group was not significantly higher than that of the PBS group ( = 0.0599) (Figures 3(g)-3(i) ). These results suggest that EAP increases the IL-6, TNF-, and IFN-production.\\n\\nIL-6, TNF-, and IFN-expression at day 3 postinfection was determined by qPCR. In contrast, TNF-mRNA levels following EAP (25 mg/kg) treatment were significantly lower than those in the PBS group (Figure 3(e) ), while IL-6 and IFN-expression were only slightly lower (not significant) (Figures 3(d) and 3(f) ). These results may be explained by a higher viral load, and the more severe inflammatory response in PBS treated mice.\\n\\nExcessive inflammation can cause severe lung lesions during H5N1 influenza infection. To evaluate histopathological changes in the lungs of infected mice, tissues of each group at day 3 postinfection were examined. The lungs of PBS treated mice exhibited a severe inflammation response, characterized by interstitial edema, inflammatory cellular infiltration around small blood vessels, alveolar lumen flooded with edema fluid mixed with exfoliated alveolar epithelial cells, and a thickening of alveolar walls (Figures 4(c) and 4(d) ). The lungs of EAP (25 mg/kg) treated mice exhibited milder lesions than those receiving PBS, characterized by signs of bronchopneumonia with interstitial edema, and inflammatory cell infiltration around small blood vessels (Figures 4(a) and 4(b) ). Viral loads and inflammatory cytokine production in the lung were correlated; suggesting that EAP treatment reduces lung lesions in H5N1 infected mice.\\n\\nPolysaccharides derived from many plants enhance the secretion of cytokines and chemokines, such as TNF-, IL-6, IL-8, and IL-12 [11] . This immunomodulatory effect is mediated mainly through recognition of polysaccharide polymers by several pattern recognition receptors (PRRs). To determine which receptor contributes directly to the innate immune recognition of EAP, Toll-like receptor 2 (TLR2), TLR4, Dectin-1, and mannose receptor (MR) were examined by qPCR both in vivo and in vitro. Mice were treated with EAP at a dose of 25 mg/kg body weight intranasally once daily for 5 days, with control mice receiving PBS. Lung total RNA was prepared for qPCR. The expression of Dectin-1 and MR in EAP treated mice was significantly elevated compared with controls, while expression of TLR2 and TLR4 were slightly higher, but not statistically significant (Figure 5(a) ). In vitro assay showed similar trends. As shown in Figure 5 (b), Raw264.7 cells were treated with 200 g/mL EPA for 36 h before qPCR. Dectin-1 and MR levels were significantly higher, while expression of TLR2 and TLR4 did not change. These data suggest that EAP recognition occurred mainly via the Dectin-1 and MR pathway.\\n\\nIn this study, we evaluated the immunomodulatory activities and protective effect of EAP against H5N1 influenza infection in a mouse model. To our knowledge, these findings are the first to show the anti-H5N1 effect of EAP. Intranasal administration of EAP prior to H5N1 viral challenge improved survival rates of infected mice with a corresponding reduction of pulmonary viral load. The anti-H5N1 effect was very likely due to the innate immune recognition of EAP and the secretion of innate immune mediators (IL-6, TNFand IFN-) before infection. Furthermore, the effect of EAP on PRR expression (including TLR2, TLR4, Dectin-1, and MR) was determined both in vivo and in vitro. These results suggest that the innate immune recognition of EAP was dependent upon the activation of the Dectin-1 and MR pathways. Our data demonstrate the feasibility of using EAP as a novel immunomodulatory agent against influenza infection. Unfortunately, the sugar composition of EAP has not been characterized.\\n\\nThe emergence of new drug-resistant strains resulting from antigenic drift limits the therapeutic benefits of vaccination and antiviral agents in controlling influenza [6, 21, 22] . Thus, development of novel broad-spectrum antiinfluenza strategies is urgently needed. Most botanical polysaccharides are ideal candidates for novel immunomodulatory agents due to their nontoxic properties and fewer side effects compared with bacterially derived polysaccharides. A number of polysaccharides isolated from plant and fungi exhibit effective antiviral benefits against influenza A virus (including H1N1 and H3N2 subtypes) [12] [13] [14] [15] . The use of polysaccharides as immunomodulatory agent in anti-H5N1 studies is rare. In this paper, our data show the immunomodulatory activities of EAP both in vivo and in vitro. EAP treatment elevated the production of IL-6, TNF-, and IFNand provides a survival advantage in H5N1 infected mice. The survival rate following EAP pretreatment (25 mg/kg body weight) was significantly higher than in mice receiving PBS (50% to 0%).\\n\\nIn previous reports, high levels of proinflammatory cytokines and chemokines (including TNF-, IL-6 and IFN-) were detected during H5N1 infection [23, 24] . This \"cytokine storm\" leads to the severe respiratory symptoms and host immune injury. Thus, H5N1-induced cytokine storms are hypothesized to be the main cause of mortality, and the use of anti-inflammatory agents may therefore provide a therapeutic effect [25, 26] . However, it is unclear whether the lack of proinflammatory cytokines (such as TNFand IL-6) facilitates viral clearance. Interestingly, knockout 8\\n\\nEvidence-Based Complementary and Alternative Medicine mice deficient in TNF-, TNF-receptor, IL-6, MIP-1 , and IL-1R or steroid-treated, wild-type mice did not have a survival advantage compared with wild-type mice following H5N1 influenza infection [27, 28] . Interestingly, prophylactic treatment of TLR3 agonist PolyICLC, which strongly upregulates cytokine production, provides protection against H1N1 and H5N1 infections [29, 30] . These conflicting studies may be explained in that the inflammatory response helps clear the virus, while aggravating host pathological damage. Elevated production of cytokines, such as IL-6, TNF-, and IFNare very important for viral clearance in the early stage of infection by activating the innate immune system. Once the viral infection has triggered a cytokine storm due to the high viral load, the inflammatory response causes severe pathological injury or even death. In this case, receiving an immunomodulator alone cannot help animal to survive [25] . This likely explains why immunomodulator treatment prior to viral infection results in a better survival rate [26, 30] . In our study, treatment of EAP shortly after infection or 24 h postinfection did not provide a survival advantage (data not show).\\n\\nThe antiinfluenza properties of IL-6, TNF-, and IFNhave been discussed in many studies, despite their participation in cytokine storms triggered by influenza infection. IL-6 plays an important role in protecting against influenza A virus as it is required for viral clearance and essential for animal survival [31] . TNF-has been reported to exert a defensive effect against influenza infection in vitro [32] . IFN-treatment in the early stages of influenza infection improves the survival rate in mouse models [33] . In addition, high levels of IFN-secretion stimulated by ginseng polysaccharides provide an antiinfluenza effect in vivo [12] . In this report, intranasal administration of EAP before H5N1 challenge elevates expression of IL-6, TNF-, and IFNcompared with mice receiving PBS. The high levels of these mediators contribute to the viral clearance and antiviral response. Pulmonary viral titers following EAP treatment were lower at day 3 postinfection. In contrast, IL-6 and IFN-mRNA levels were slightly lower, while TNF-production was significantly lower than that of PBS group. Regarding the excessive inflammation induced by H5N1 virus, massive secretion of mediators contributes to lung injury rather than an antiviral response. Therefore, the timing of EAP treatment as a prophylactic agent is very important.\\n\\nThe immunomodulatory activities of botanical polysaccharides are thought to be mediated by several PRRs [11] . In this study, we examined the mRNA levels of TLR2, TLR4, Dectin-1, and MR after EAP treatment. EAP was found to upregulate Dectin-1 and MR mRNA expressions significantly both in vivo and in vitro. Our hypothesis is that the innate immune recognition of EAP is driven mainly via a Dectin-1 and MR dependent pathway. Binding to these receptors, EAP may activate complex intracellular signaling pathways, and increase cytokine production, leading to an antiviral response. Thus, the protection against H5N1 by EAP treatment is less likely to cause drug resistance, and may represent a broad-spectrum antiinfluenza effect.\\n\\nIn conclusion, our study demonstrates that EAP leaf extract is a prophylactic and immune enhancement agent against H5N1 influenza virus infection. Treatment with EAP effectively inhibits H5N1 viral replication and improves animal survival. This approach offers an alternative strategy for antiinfluenza immunomodulatory agent development, and benefits the utilization of E. adenophorum products.', 'document_id': 1585}]}, {'paragraphs': [{'qas': [{'question': 'What viruses have been responsible for most common childhood acute respiratory track infections (ARTI)?', 'id': 550, 'answers': [{'text': 'The most frequently reported viruses include respiratory syncytial virus (RSV), influenza viruses A and B (IAV, IBV), parainfluenza viruses (PIVs), human rhinovirus (HRV) and adenovirus (ADV), ', 'answer_start': 2534}], 'is_impossible': False}, {'question': 'Are there any vaccines against to protect against respiratory viral infections?', 'id': 553, 'answers': [{'text': 'Currently, there are no approved vaccines or medications available for most of the respiratory viruses', 'answer_start': 3172}], 'is_impossible': False}], 'context': \"A 3-year prospective study of the epidemiology of acute respiratory viral infections in hospitalized children in Shenzhen, China\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC4181804/\\n\\nSHA: ef5fe7296ec8baf90d974cf5737af0da3ed403ea\\n\\nAuthors: He, Ying; Lin, Guang-Yu; Wang, Qiong; Cai, Xiao-Ying; Zhang, Yin-Hui; Lin, Chuang-Xing; Lu, Chang-Dong; Lu, Xue-Dong\\nDate: 2014-05-14\\nDOI: 10.1111/irv.12257\\nLicense: cc-by\\n\\nAbstract: BACKGROUND: The epidemiology of local viral etiologies is essential for the management of viral respiratory tract infections. Limited data are available in China to describe the epidemiology of viral respiratory infections, especially in small–medium cities and rural areas. OBJECTIVES: To determine the viral etiology and seasonality of acute respiratory infections in hospitalized children, a 3-year study was conducted in Shenzhen, China. METHODS: Nasopharyngeal aspirates from eligible children were collected. Influenza and other respiratory viruses were tested by molecular assays simultaneously. Data were analyzed to describe the frequency and seasonality. RESULTS: Of the 2025 children enrolled in the study, 971 (48·0%) were positive for at least one viral pathogen, in which 890 (91·7%) were <4 years of age. The three most prevalent viruses were influenza A (IAV; 35·8%), respiratory syncytial virus (RSV; 30·5%) and human rhinovirus (HRV; 21·5%). Co-infections were found in 302 cases (31·1%), and dual viral infection was dominant. RSV, HRV and IAV were the most frequent viral agents involved in co-infection. On the whole, the obvious seasonal peaks mainly from March to May were observed with peak strength varying from 1 year to another. CONCLUSIONS: This study provides a basic profile of the epidemiology of acute respiratory viral infection in hospitalized children in Shenzhen. The spectrum of viruses in the study site is similar to that in other places, but the seasonality is closely related to geographic position, different from that in big cities in northern China and neighboring Hong Kong.\\n\\nText: Acute respiratory tract infections (ARTIs) are a persistent and pervasive public health problem in both developed and developing countries. They cause a great burden of disease worldwide. Especially in developing countries including China, ARTIs, mainly pneumonia, are the leading cause of death among children under the age of 5 years. 1,2 A great variety of pathogens can cause ARTIs, and viruses have been considered as the predominant pathogens in this children population. 3, 4 The most frequently reported viruses include respiratory syncytial virus (RSV), influenza viruses A and B (IAV, IBV), parainfluenza viruses (PIVs), human rhinovirus (HRV) and adenovirus (ADV), which are responsible for most episodes of ARTIs in children. 1 In the past decade, several new viruses associated with ARTIs such as human metapneumovirus (HMPV), novel strains of coronaviruses (SARS-CoV, HCoV-NL63 and HKUI), human bocavirus (BOV), WU polyomavirus (WUPoyV) and KI polyomavirus (KIPoyV) have been discovered in human respiratory tract specimens. Among them, some have been identified to be causative pathogens of ARTIs. 1, 4, 5 Currently, there are no approved vaccines or medications available for most of the respiratory viruses. 1 A better understanding of the epidemiology of viral respiratory tract infections in children plays a key role for the prevention, control and treatment of ARTIs. Studies showed that many viral respiratory infections exhibited predictable seasonal variations. However, the epidemiological profiles of viral respiratory infections from different climate zones or different countries in the same climate zone may be varied. [6] [7] [8] [9] [10] [11] [12] China is a large country crossing three climate zones, and great differences in climate are found from region to region. A better understanding of the epidemiology of ARTIs in different regions could be helpful to develop effective surveillance, prevention and treatment strategies. Although some studies on the epidemiology of ARTIs have recently been reported in big cities such as Beijing, Shanghai and Hong Kong, [13] [14] [15] [16] the epidemic characteristics of viruses in ARTIs are still not well established all around China, especially in other cities and rural areas.\\n\\nShenzhen is the largest migratory city of China with high population density and population mobility. It is located in southern China at 22°27 0 -22°52 0 N and 113°46 0 -114°37 0 E, immediately north of Hong Kong, with a typical subtropical monsoon climate. The annual average temperature and relative humidity of Shenzhen are about 23°C (12-33°C) and 77%, respectively. The purpose of this study is to investigate the prevalence, seasonality and clinical characteristics of acute viral respiratory infections in hospitalized children in Shenzhen and to provide insights into etiologies of ARTIs in local infants and children.\\n\\nA consecutive 3-year prospective study from July 2007 to June 2010 was conducted in Shenzhen, a coastal city neighboring Hong Kong. Four hospitals including a children's hospital were chosen for the study. Selected patients with ARTIs admitted to the pediatric wards were enrolled. The inclusion criteria were as follows: less than 14 years old, acute fever (T ≥ 38°C), with any one of respiratory symptoms (such as sore throat, cough, wheezing and dyspnoea/ tachypnoea), normal or low leukocyte count, the onset of illness within 3 days before hospitalization. The diagnosis of pneumonia was based on the guideline of the management of childhood community acquired pneumonia (CAP) issued by the Chinese Medical Association in 2006. 17 In the guideline, the clinical symptoms and signs for the diagnosis of childhood CAP include fever, cough, tachypnoea (defined according to different age), difficulty breathing and/or lower chest wall indrawing. X-ray evaluation has been carried out when necessary. The study protocol was approved by the medical ethical committees of the hospitals. Written informed consent was obtained from the parents or legal guardians of the children.\\n\\nNasopharyngeal aspirates (NPA) were obtained by trained personnel following standard operating procedures within 24 hour after admission. The specimens were transported immediately to the laboratory by sterile viral transport media, then divided into aliquots and immediately frozen at À80°C until further processing.\\n\\nTotal viral nucleic acids (DNA and RNA) were extracted from 200 ll of NPA specimen using the AxyPrep Body Fluid DNA/RNA Miniprep Kit (Axygen, Union City, CA, USA), according to the manufacturer's instructions. Purified DNA and RNA were stored at À80°C in aliquots for further PCR analysis.\\n\\nFor each specimen, assays for ten common and newly identified viruses were performed. Briefly, WUPoyV and BOV were tested using monoplex PCRs described previously. 18, 19 Other viruses were tested using the Luminex platform and multiplex xTAG TM respiratory viral panel assay (RVP Assay) according to the manufacturer's instructions. 20 All multiple infection samples were retested. If there was discordance between two tests, the sample was confirmed by monoplex PCR.\\n\\nStatistical Package for the Social Sciences (SPSS) for Windows version 11.0 (SPSS Inc. Chicago, IL, USA) was used. For comparison of categorical data, chi-square or Fisher's exact test was used. All tests were two-tailed, and a P value below 0Á05 was considered statistically significant.\\n\\nA total of 2025 specimens were obtained from 2025 eligible patients ranging from 15 days to 14 years old with a median age of 12 months, in which 89Á6% of patients were < 4 years old. There were 964 (47Á6%) females and 1061 (52Á4%) males included. Of all hospitalized children enrolled in this study, 84Á0% involved lower respiratory infection and 16Á0% had upper respiratory tract infection (Table 1) . Among 971 positive cases, 572 (58Á9%) were diagnosed as pneumonia.\\n\\nAbout 971 of the 2025 cases (48Á0%) were positive for at least one viral pathogen. Among them, IAV, RSV, HRV and PIVs were detected in 348 (35Á8%), 296 (30Á5%), 209 (21Á5%) and 169 (17Á4%) cases, respectively. Single infection was observed in 669 (68Á9%) cases, and multiple infection was found in 302 (31Á1%). Our results also showed that RSV, IAV and HRV were the main pathogens in single viral infection cases ( Table 1) . The monthly positive rates varied from 32Á5% to 75Á0% with a mean of 45Á1% ( Figure 1 ). In the year 2009, when influenza A (H1N1) was pandemic worldwide, the positive rate started to increase in March and the highest positive rate 75Á0% was observed in May.\\n\\nAmong the 971 positive cases, a total of 1335 viral pathogens were detected. The most frequently detected pathogen was IAV (26Á1%, 348/1335), followed by RSV (22Á2%, 296/1335), HRV (15Á7%, 209/1335), PIV1 and PIV3 (12Á7%, 169/1335) ( \\n\\nAbout 302 co-infection cases were identified, accounting for 14Á9% of all 2025 hospitalized children. During the H1N1 outbreak from March to August 2009, co-infection cases and co-infection rate increased significantly ( Figure 2 ). 143 of 302 (47Á4%) co-infection cases were detected during that time. Among them, 121 cases were involved in IAV infection, including 90 dual infection cases. Of all co-infection cases, 247 (81Á8%), 49 (16Á2%) and 5 (1Á7%) were infected with two, three and four potential viral pathogens, respectively. One multiple infection with five viruses was detected in a RSV  IAV  HRV  BOV  PIV3  ADV  HMPV  WUPoyV  PIV1  IBV  Total   A  Total cases  163  172  85  55  51  49  37  26  24  7  669  Bronchitis  12  19  7  7  5  2  2  3  5  0  62  Bronchiolitis  45  19  20  7  13  9  8  2  5  0  128  Pneumonia  93  97  55  28  24  28  25  18  14  5  387  URTI  13  37  3  13  9  10  2  3  0  2 *Case number and percentage in all enrolled children. **Incidence rate in this age group significantly higher than the other age groups.\\n\\n***No significant difference between these three age groups. †No significant differences between these two age groups, but significantly lower than the other age groups. 6-month-old infant. IAV, RSV and HRV were the three most frequently found viruses in co-infection and detected in 176, 133 and 124 cases with co-infection rates of 50Á6%, 44Á9% and 59Á3%, respectively ( Table 2) . Various multiple infection patterns were observed in the study. A total of 152 (50Á3%) co-infection cases involved at least two viruses of RSV, HRV and IAV. Co-infection rate of each individual virus detected varied significantly. The lowest and highest co-infection rates were observed in WUPoyV (33Á3%) and IBV (66Á7%), respectively. 91Á4% (276/302) of co-infection cases were tested in the age group of 4 years old or younger ( Table 2) , but among all age groups, no statistical difference in co-infection rate was found (v 2 = 1Á83, P = 0Á8721). Gender-specific difference in co-infection rate was not observed (v 2 = 2Á17, P = 0Á1404). There was no significant difference in co-infection rates between PICU and non-PICU cases. Similarly, no significant difference in clinical symptoms was observed between co-infection and single cases (data not shown).\\n\\nIn general, respiratory viruses were detected more often in the period of March to May than in other months (55Á4% and 40Á6%, respectively, v 2 = 28Á06, P = 0Á0000), and obvious seasonal peaks were observed during those months with peak strength varying from 1 year to another. A weaker seasonal peak could also be distinguished in some winter months in different years ( Figure 1) .\\n\\nThe seasonality profile of each individual virus detected was diversified. A seasonal distribution of IAV can be observed from late spring to summer (mainly March to May) and sometimes in fall (October, November or December). A wide seasonal peak of IAV infection was detected from March to August 2009 ( Figure 3A ). Although RSV was tested almost a whole year, two yearly peaks were identified. One was found in November and/or December and the other stronger one was found in March to May of the year. The peak duration in 2009 was longer than those in other years. The seasonal trends of HRV and PIVs were similar to that of RSV, but the peaks of these three viruses fluctuated and shifted mildly ( Figure 3B ). Although IBV and ADV had a low detection rate in the study, similar seasonality was observed and their infection peaks were mainly in midwinter. Peaks in spring and summer were also observed in some years ( Figure 3C ).\\n\\nOur investigation did not find regular seasonality in BOV infections. A sudden increase in BOV infection was recorded in April and May 2010. Although the positive rate of HMPV infection was only 4Á8%, regular seasonality was observed from March to May of each year. Of 39 patients with WUPoyV infection, 36 were detected after July 2008. Our data implied that peak months of WUPoyV infection were from March to May ( Figure 3D ).\\n\\nThe positive rates of viral infections in male and female were 52Á5% and 47Á5%, respectively. No significant gender difference was revealed (v 2 = 0Á012, P = 0Á9118). The distribution of viral agents and infection patterns in different age groups are shown in Table 2 .\\n\\nOf all 971 positive children, 890 (91Á7%) were 4 years old or younger. The positive rate in this age group was significantly higher than that in children more than 4 years old (v 2 = 8Á26, P = 0Á0041). Children under 6 months were the most susceptible to respiratory viral pathogens with a positive rate of 14Á8% (Table 2) .\\n\\nVery few long-term prospective studies were performed for viral etiologies of ARTIs among hospitalized children. In this present study, the infection frequency, seasonality, co-infection pattern and clinical features of viral respiratory infections were investigated based on prospective analysis of three consecutive year's data from hospitalized children with ARTIs. Our results provided a distinctive epidemiological profile of viral respiratory infections in hospitalized children with ARTIs in the study areas, which was different from those in the big cities in northern China such as Beijing and Shanghai and also different from that in adjacent Hong Kong.\\n\\nOverall, 48Á0% of our cases were positive for respiratory virus infections, which resembled the latest study in the same city. 21 A similar incidence rate has been obtained in neighboring regions 13, 22 and other cities such as Rome 23 and Milan, 24 but it was different from other studies. [10] [11] [12] In China, the overall positive rate reported varied from 27Á3 to 74Á8% depending on different areas and detection methods. 15, 16, [25] [26] [27] [28] [29] [30] [31] The rate of respiratory viral infections varied worldwide, and many factors such as geographic distribu-tion, study design and detection protocols could lead to these variations. 1, 7, 8, 32 In our study, leukocyte count was used as an indicator of inclusion criteria and it probably affected the positive rate. Viruses not considered in the study, for example coronaviruses, would underestimate the positive rate. Most studies showed that RSV or HRV was the most prevalent viruses in children with viral respiratory tract infection. 1 In this study, IAV was the most frequently detected respiratory virus, followed by RSV and HRV. IAV (H1N1) outbreak in 2009 could explain this shift. Data showed that about 60% of IAV infections were detected during the outbreak period. Studies showed that the H1N1 outbreak could change viral distribution patterns. 24, 29, 33 Regardless of the IAV (H1N1) outbreak, RSV and HRV were the two most common viral pathogens in ARTIs, which was consistent with most previous studies. 1, 10, 15, 16, 22, [25] [26] [27] [28] [29] Our study further confirmed the importance of RSV and HRV in children with ARTIs, especially in children < 4 years of age. 10, 14, 23 Our results also showed that 12Á7% of viral pathogens detected were PIV1 and PIV3, which implied that PIVs played an important role in children with ARTIs. Similar findings were obtained in the studies conducted in Shanghai, 14, 34 Changsha, 26 Harbin, 30 Hong Kong 13 and Rome. 23 The prevalence of PIV3 was twofold higher than that of PIV1, particularly in infants, which was similar with other reports, 25, 26, 30, 35 implying that infants could be more vulnerable to infection with PIV3 than PIV1.\\n\\nHMPV has been proven to be one of the main viral pathogens responsible for ARTIs in children. 5 The positive rate found in the study was consistent with previously published results. 10, 36, 37 In China, the infection rate of HMPV varied from 3Á2 to 10Á6%. 22, 26, 28, 29, 31 The seasonality of HMPV in this study was mainly from March to May, similar to that in Hong Kong, 36 but different from other places. 5, 37 In our study, 4Á9% of cases were positive for BOV, which coincided with 5Á0% in Hong Kong 38 and higher than Guangzhou 39 and eastern Guangdong. 22 Our result suggested that BOV might be present throughout the year with no seasonal distribution. However, seasonal distribution was noted from September to February in Hong Kong 38 and May and June in Guangzhou. 39 The use of multiple PCR made it possible to simultaneously detect a broad spectrum of viruses with excellent sensitivity, at the same time, with increased viral detection rate and co-infection rate for ARTIs. 12,40 Among our positive cases, co-infection rate was 31Á1%, which was similar to 27Á9% reported by Do et al. 10 Co-infection rate reported elsewhere varied widely from 25Á4 to 47Á9%. 40 The relatively lower co-infection rates ranging from 0Á24 to 26Á9% were reported in the studies conducted in various cities of China. 22, [25] [26] [27] [28] [29] [30] [31] In most of these studies, immunofluorescence kits were used to test a lower number of respiratory viruses. It was worth to note that in the study by Peng et al. 32 in Wuhan, China, 69Á5% of co-infection rate was reported with immunofluorescence kit. These variations might be attributed to geographic differences, diagnostic methods for viral agents and study design. 12, 32, 34, 41, 42 Pathogens in those negative patients need to be further investigated as only ten common and newly identified viruses were included in our study, which might underestimate positive rate or coinfection rate. It was notable that the correlation between co-infection rate and positive rate was not observed.\\n\\nOf multiple infections, dual infection was predominant in this study whether or not considering the IAV (H1N1) outbreak in 2009, which was consistent with previous studies. 28, 32, 42, 43 Similar with the studies conducted in the cities of Guangzhou and Wuhan, China, 28, 29 our study showed that IAV, RSV and HRV were the main viruses involved in multiple infections. High co-infection rate between these three viruses could be explained from the overlap of their seasonal distributions. A variety of predominant multiple infection patterns between respiratory viruses were observed in different studies. 12, 32, 42, 43 For example, it was shown in Martin et al.'s study 43 that ADV and coronaviruses were the most common co-infection pattern.\\n\\nOur study showed that RSV and HRV were the two most viruses involved in multiple infection, followed by IAV and PIVs, regardless of IAV infection in the H1N1 outbreak period. It was difficult to explain the variations of coinfection patterns based only on seasonal distribution. A recent study suggested that co-infection patterns were not random and certain pathogens had higher frequency of coinfection. 41 As molecular assays only detect nucleic acid and positive result does not mean the presence of the pathogen, when studying co-infection patterns of respiratory viruses, the ability to differentiate the real causative pathogens needs to be solved first. Viral load detection could provide some clues for solving this issue. 43, 44 Although high co-infection rates have been reported in various studies, the associations among multiple infections, hospitalization rate and severity of ARTIs were still not clear with inconsistent results in different studies. 42, 43, 45 Our data suggested that multiple infection had less association with the severity of disease, consistent with Peng et al.'s study. 32 The relationship between co-infection rates and age group was also investigated in our study, and little correlation was observed. Several previous studies observed that co-infection rates were more frequent in a certain age group, but results were varied. 32, 43 In contrast to temperate region, where most viruses had winter-spring seasonality, the respiratory viral infections in tropical and subtropical regions appeared mainly to be spring-summer seasonality. 9 In this study, due to the high detection rate and similar seasonality of RSV, HRV, IAV, PIV and HMPV, an overall spring-summer seasonality of viral respiratory infections in children was concluded. Studies conducted in Hong Kong showed that a clear seasonal peak was from April to September, 36, 46 with a longer duration than our study. The overall seasonality in this study was also different from the studies conducted in northern or central cities of China, in which the seasonality of most viruses presented in autumn-winter and/or winter-spring. 15, [25] [26] [27] 30 The winter-spring seasonality was also observed in Guangzhou, a city about 150 kilometers north of Shenzhen. 28 Different seasonal onset and duration were observed in various studies conducted in (sub-) tropical regions. In these studies, ambient temperature, humidity and rainfall were widely used to explain these differences in seasonality, but inconsistent results were observed. 9, 46, 47 Although most studies demonstrated that the seasonality of viral respiratory infections was correlated with increased rainfall, effects of climate factors such as humidity and temperature on the seasonality were complex and interactive. 9, 46, 48 The study areas have four indistinct seasons, and the coldest month usually emerges in January (average 12°C). During the period from March to May, the weather featured warm ambient temperature (average 18-25°C), high relative humidity (average 85%-90%) and increasing rainfall. These meteorological conditions were perhaps conducive to viral survival. 9, 48 In addition, intensive temperature fluctuations during seasonal alternation could increase the susceptibility to infections. 49 As reported in other studies in temperate, tropical and subtropical regions, viral infection rates in children population showed an inverse correlation with age, with younger individuals experiencing higher viral infection rates. 3, 4, 6, 9, 24 Our results suggested that children younger than 4 years of age, particularly <6 months, were at higher risk of hospitalization for ARTIs, compared with older children. This was particularly substantiated in RSV infection. Our presumption was supported by other studies. 14,25-28 Of course, this speculation needed to be validated by the population-based study. The findings reported elsewhere suggested that more males than females were affected by ARTIs, which were not observed in our study.\\n\\nNotably, our study occurred over a span of 3 years, which included the IAV (H1N1) outbreak in 2009. The impact of the outbreak on the results should be considered. Data showed that the detection rate of IAV increased significantly and co-infection rate during outbreak months was much higher than average co-infection rate. Unfortunately, we did not type these influenza strains based on the original study design. It was most likely that these strains contributed to the relatively high proportion of IAV. Relatively higher single and multiple infections of RSV, HRV and PIVs were also observed during the outbreak of IAV. Increased susceptible population and awareness, intensive testing and altered patient and physician behavior could lead to these increases. These factors could partly explain the relatively high proportion of pneumonia cases in the study. Furthermore, studies showed that the outbreak of IAV (H1N1) could increase the risk of other viral infections such as RSV and HRV. 24, 33 Other limitations also existed in this study. First, molecular methods allowed the detection of only viral nucleic acid even without virus replication, which complicates the interpretation of positive detection results. Second, the subtype identification of some common respiratory viruses such as IAV and HRV was not performed in our study, particularly during the IAV (H1N1) outbreak in 2009.\\n\\nIn summary, despite those aforementioned limitations, this three consecutive years' surveillance would provide a basic profile of the spectrum, seasonality, age and gender distribution, co-infection patterns as well as clinical association of viral respiratory infections in hospitalized children in the study sites. It could help the prediction, prevention and control of ARTIs in children.\", 'document_id': 1575}]}, {'paragraphs': [{'qas': [{'question': 'Which type of bacteria are implicated in carrying genes of drug resistance?', 'id': 544, 'answers': [{'text': 'Gammaproteobacteria', 'answer_start': 507}], 'is_impossible': False}, {'question': 'What may be a likely cause of sink-to-sink spreading of pathogens in the hospital setting?', 'id': 545, 'answers': [{'text': 'via a common sanitary pipe', 'answer_start': 19969}], 'is_impossible': False}], 'context': 'Spread from the Sink to the Patient: In Situ Study Using Green Fluorescent Protein (GFP)-Expressing Escherichia coli To Model Bacterial Dispersion from Hand-Washing Sink-Trap Reservoirs\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC5377511/\\n\\nSHA: 615071c8c959f24857b1bad521cc432b59719bfb\\n\\nAuthors: Kotay, Shireen; Chai, Weidong; Guilford, William; Barry, Katie; Mathers, Amy J.\\nDate: 2017-03-31\\nDOI: 10.1128/aem.03327-16\\nLicense: cc-by\\n\\nAbstract: There have been an increasing number of reports implicating Gammaproteobacteria as often carrying genes of drug resistance from colonized sink traps to vulnerable hospitalized patients. However, the mechanism of transmission from the wastewater of the sink P-trap to patients remains poorly understood. Herein we report the use of a designated hand-washing sink lab gallery to model dispersion of green fluorescent protein (GFP)-expressing Escherichia coli from sink wastewater to the surrounding environment. We found no dispersion of GFP-expressing E. coli directly from the P-trap to the sink basin or surrounding countertop with coincident water flow from a faucet. However, when the GFP-expressing E. coli cells were allowed to mature in the P-trap under conditions similar to those in a hospital environment, a GFP-expressing E. coli-containing putative biofilm extended upward over 7 days to reach the strainer. This subsequently resulted in droplet dispersion to the surrounding areas (<30 in.) during faucet operation. We also demonstrated that P-trap colonization could occur by retrograde transmission along a common pipe. We postulate that the organisms mobilize up to the strainer from the P-trap, resulting in droplet dispersion rather than dispersion directly from the P-trap. This work helps to further define the mode of transmission of bacteria from a P-trap reservoir to a vulnerable hospitalized patient. IMPORTANCE Many recent reports demonstrate that sink drain pipes become colonized with highly consequential multidrug-resistant bacteria, which then results in hospital-acquired infections. However, the mechanism of dispersal of bacteria from the sink to patients has not been fully elucidated. Through establishment of a unique sink gallery, this work found that a staged mode of transmission involving biofilm growth from the lower pipe to the sink strainer and subsequent splatter to the bowl and surrounding area occurs rather than splatter directly from the water in the lower pipe. We have also demonstrated that bacterial transmission can occur via connections in wastewater plumbing to neighboring sinks. This work helps to more clearly define the mechanism and risk of transmission from a wastewater source to hospitalized patients in a world with increasingly antibiotic-resistant bacteria that can thrive in wastewater environments and cause infections in vulnerable patients.\\n\\nText: D espite early reports (1) (2) (3) (4) (5) , the premise that hand-wash sink traps can act as reservoirs of bacteria that cause nosocomial infections has been frequently overlooked. There has recently been an alarming increase in sink-related outbreaks worldwide, with many reports establishing an observational link (6) (7) (8) (9) (10) (11) (12) (13) . A sink often operates as an open conduit to wastewater in a patient care area that is often in the same room as the patient.\\n\\nHealth care establishments often invest in desperate interventions to deal with nosocomial outbreaks. The preferred method for addressing most of the environmentrelated transmission is to employ enhanced cleaning using chemical and physical agents (14, 15) . Unfortunately, routine approaches are inefficient in completely eliminating drug-resistant Gammaproteobacteria in an inaccessible microbiologically active area such as a sink trap (6, (16) (17) (18) (19) (20) . The wet, humid, and relatively protected environment in a sink trap favors the formation of rich stable microbial communities (16, 21, 22) . These communities will be exposed to liquids and waste that are discarded in a sink and may include antimicrobials, discarded beverages, soap, presumably pathogenic bacteria from health care workers\\' hands, and other items. In short, sink traps could serve as a breeding ground for opportunistic and highly antimicrobial-resistant bacteria that cannot be easily cleaned or removed (23) (24) (25) (26) (27) (28) .\\n\\nThere are many reports of a genetic association between pathogens found in sink traps and those found in patients (29, 30) . However, surprisingly little work has been done to understand the microscale transmission dynamics. It was previously demonstrated using a suspension of fluorescent particles (Glo Germ; Glo Germ Co., Moab, UT) that material injected into the P-trap gets dispersed around a hand-washing sink (6) . This result however has not been replicated hitherto in the follow-up studies. Dispersion has never been investigated with living organisms. Ultimately, many details remain unaddressed surrounding the spread of Enterobacteriaceae in sink-trap wastewater systems: (i) can organisms grow retrograde from the P-trap water to the sink strainer, (ii) can organisms spread from one sink to another along the internal surfaces of pipes with shared drainage systems, and (iii) which portion of a colonized drain pipe results in dispersion into the sink bowl during a hand-washing event? We aim to better understand the dispersion dynamics of Gammaproteobacteria living in the wastewater of a sink strainer and P-trap into an area where patients and health care workers could be exposed. To study this dynamic, we used a surrogate organism that could be easily tracked while remaining in the Enterobacteriaceae family, where some of the most concerning threats in antimicrobial resistance are developing (30) .\\n\\nGrowth and colonization of GFP-expressing E. coli in the P-trap. In the first 14 days following the installation of the P-trap with established green fluorescent protein (GFP)-expressing Escherichia coli and just water running from the faucet, GFPexpressing E. coli was not detected in the tailpipe beyond 1.5 in. above the liquid level in the P-trap. GFP-expressing E. coli, however, was found to be viable in the P-trap without any nutrients added. A nutrient regimen was then instituted to understand the influence of nutrients on mobility and upward growth. The addition of tryptic soy broth (TSB) promoted GFP-expressing E. coli growth as early as day 1, with growth observed in the tailpipe 2 in. above the liquid surface in the P-trap (Table 1) . On day 7, the strainer (ϳ8 in. above the liquid in the P-trap) was found to be colonized with GFP-expressing E. coli. This translates to an average growth rate of 1 in./day along the length of the tailpipe with the addition of nutrients and without faucet operation. GFP-expressing E. coli was not detected in the faucet water.\\n\\nSink-to-sink transmission of bacteria. In these experiments, a flanking sink (sink 5) was the only P-trap inoculated with GFP-expressing E. coli and therefore was the sole source for transmission to the connected sinks. Starting with a lower inoculum concentration (10 3 CFU/ml) in sink 5, on day 7, GFP-expressing E. coli was detected in the sink 2 and sink 3 P-traps (Fig. 1a) . With inoculum concentrations of 10 6 CFU/ml and Ͼ10 10 CFU/ml in sink 5, all of the sink P-traps in the sink gallery with the exception of sink 1 were found to be colonized with GFP-expressing E. coli after 7 days (Fig. 1b and  c) . Faucet water and aerators tested negative for GFP-expressing E. coli. Irrespective of the starting inoculum concentration, on day 7 the highest level of colonization was recorded in the sink 3 P-trap. After day 7, when the nutrient regimen (described previously) was followed for an additional 7 days in each of the sinks in the sink gallery with an inoculum concentration of Ͼ10 10 CFU/ml, GFP-expressing E. coli was detected in the strainers of sinks 2 and 3 on day 14. This finding validated the upward growth and growth rate in the tailpipe when nutrients were added. Nonfluorescent colonies were occasionally observed in the P-trap water samples collected from the sinks, which were subsequently identified to be Pseudomonas sp. or Stenotrophomonas maltophilia, and fluorescent colonies were confirmed to be E. coli.\\n\\nDispersion of microspheres from sinks. In the first dispersion experiment, when fluorescent microspheres were inoculated into the offset drain tailpiece only 4 in. below the strainer, no microspheres were detected on the polyester sheets placed on the counter space.\\n\\nHowever, when the sink bowl was coated with the microspheres, polyester sheets overlaid on the counter space captured the dispersed microspheres caused by the faucet operation. Dispersion was observed on almost all zones of the sink counter space (Fig. 2) . Relatively higher levels of dispersion were observed along the major and minor axes of the elliptical sink bowl (zones 2, 5, 6, 9, 11, and 12) . Anterior corners of the sink counter space (zones 4 and 7), which were most distant from the impact of water in the sink bowl, received the lowest dispersion.\\n\\nDispersion of GFP-expressing E. coli from sinks. Initially the P-trap alone was inoculated with GFP-expressing E. coli and carefully installed, keeping the tailpipe and strainer free of GFP-expressing E. coli before operating the faucets. No fluorescent CFU were observed on the plates placed on the counter or attached to the bowl surface after faucet operation. Similarly, no fluorescent CFU were detected when GFPexpressing E. coli was inoculated into the offset drain tailpiece only 4 in. below the strainer. Interestingly, when there was conspicuous water backup over the strainer as a result of a higher water flow rate from the faucet than the drainage rate from the P-trap, dispersal was detected on the plates attached to the bowl surface.\\n\\nThe dispersion pattern recorded when the sink bowl was coated with GFPexpressing E. coli was comparable to the pattern recorded when the sink bowl was coated with fluorescent microspheres (Fig. 2) . Dispersion was significantly higher along the axes (zones 6, 9, 11, and 12) and lower at the corners of the sink counter space (zones 4, 7, and 10).\\n\\nIn contrast, dispersion of GFP-expressing E. coli caused by the faucet operation was much more extensive when the strainer was allowed to be colonized with GFPexpressing E. coli prior to the dispersion experiment. In addition to the sink counter space, we measured dispersion to the sink bowl, faucet, faucet handles, splatter shields, and the extended counter surface. Dispersion of GFP-expressing E. coli was highest on the plates attached to the sink bowl (Fig. 3b) . Further, dispersion was greater along the \\n\\nϪ\" and \"ϩ\" denote the absence and presence of GFP-expressing E. coli, respectively.\\n\\nminor axis of the sink bowl (Fig. 3b , zones B3, B4, and B10) than along the major axis of the sink bowl, associated with a shorter distance from the strike point of the faucet water to the bowl along this axis. The next highest CFU count from the dispersal was recorded on the counter area near the faucets (Fig. 3a , zones 12 and 11). A similar pattern of higher dispersion near the faucets and lower dispersion at the corners of the counter space (Fig. 3a , zones 4, 7, and 10) was also observed using microspheres. Dispersion was also recorded in other zones of the counter space, on the Plexiglas splatter shields, faucets, faucet handles, and extended surface (Fig. 3c ). There were no GFP-expressing E. coli CFU recorded on plates placed beyond 30 in. from the strainer, demarcating the range of dispersion under these experimental conditions. Table 2 gives a summary of the total distribution loads recorded using fluorescent microspheres and GFP-expressing E. coli across each experiment. The loads of dispersion on the sink counter were comparable when the sink bowl was coated with microspheres or GFP-expressing E. coli before the faucet operation. Although the dispersion load on the sink counter was lower when the sink strainer was colonized, it is interesting to note that the sink bowl received the highest dispersion.\\n\\nTo mimic dispersion in a hospital setting, we first investigated whether GFPexpressing E. coli would establish consistent colonization in a sink trap as many other Gammaproteobacteria implicated in nosocomial outbreaks have done (6, 28) . Many recent reports demonstrate that P-traps become colonized with highly consequential Gammaproteobacteria, which then results in nosocomial transmission (29, 31, 32) . The retained water in a sink P-trap is present to provide a water barrier to prevent off-gassing of sewer smell, but it may inadvertently provide favorable conditions for pathogenic and opportunistic antibiotic-resistant microorganisms to survive and develop resilient biofilms (3, 33) . However, the mechanism of dispersal of the bacteria in the P-trap to patients or the surrounding health care area had not been fully elucidated. We began with the hypothesis that the bacteria originate from the P-trap via droplet creation when the water from the faucet hits the P-trap water, thus contaminating the sink bowl and the surrounding area. The finding supporting this theory had been previously reported using Glo Germ particles (6) . However, in the present study with careful attention to avoid strainer and tail piece contamination, the dispersal directly from the sink P-trap with either microspheres or GFP-expressing E. coli could not be reproduced as previously reported (6) .\\n\\nRather this work demonstrates a different, more staged mode of transmission from a P-trap reservoir to the sink and surrounding environment. GFP-expressing E. coli in the P-trap alone was sustained for 14 days but did not grow or mobilize up the tailpipe to the strainer with just intermittent water exposure. However, when nutrients were subsequently added to the system, the organisms rapidly grew up the tailpipe to the strainer at approximately an inch per day. In a real-world setting, motility of bacteria inside the tailpipe is restricted to relatively sporadic and brief wetting events in which swimming is an opportunity to colonize new surfaces. It is assumed that once established, the biofilm promotes the upward growth of GFP-expressing E. coli in the tailpipe at an accelerated rate. The nutrient regimen that promoted colonization in our model reflects our and others\\' observations of items commonly disposed of in hospital sinks (intravenous fluids, feeding supplements, and leftover beverages) (5, 32) .\\n\\nTransmission of bacteria between sinks via a common pipe was a key finding in this study as this highlights the concept that premise plumbing may be a more continuous system with shared microbiology than a single isolated sink. The sink gallery used in this study provided a unique in situ advantage to investigate sink-to-sink transmission of bacteria through common drains. The two possible mechanisms for P-trap strainers becoming colonized are seeding of organisms from above and retrograde spread of organisms along common pipes in a hospital wastewater infrastructure. Here we demonstrate that it is possible for GFP-expressing E. coli to contaminate adjacent P-traps with just time and water given a standard U.S. code piping rise of 0.25 ft/ft. Sink-to-sink or retrograde transmission may explain the recurrence of pathogen colonization following intervention strategies like disinfection or replacement of plumbing (23) . Sink 3 was lowest on the slope in the drain line (see Fig. 4 ) with arguably the most opportunity for reflux and retrograde wetting. Sink 1, on the other hand, was farthest away from the source (sink 5), and its P-trap had the greatest incline in the drain line connecting the sinks, which could perhaps contribute to the reasons there was no GFP-expressing E. coli colonization detected in it after 7 days. There has been more investigation about microbiologic dynamics of infectious viral particles such as those of severe acute respiratory syndrome (SARS) and Ebola viruses through premise plumbing systems (34) (35) (36) . However, the microbiology, sustainability, and dynamics might be very different, although the backflow and inoculation issues could have some parallels when comparing viruses to bacteria. As Enterobacteriaceae can either multiply or remain viable for long periods of time in biofilms coating the interior of P-traps and the connected plumbing, it may not be sustainable to target any intervention limited to a single isolated sink as a source of a particular pathogen.\\n\\nData from different dispersal experiments suggest that although P-traps can act as the source or the reservoir of pathogens, the physical presence of the organism in the sink bowl or colonization of the strainer is necessary for the dispersal to occur. Colonization of strainers or drains reported in earlier studies (7, 10, 13, 24, 37) was perhaps a result of ascending biofilm growth from the P-trap to the strainer or introduction through contaminated fluids. Many of the studies used swab samples, which likely sampled the strainer rather than P-trap water (17, 20) . Once the strainer was colonized, the water from the faucet resulted in GFP-expressing E. coli dispersion in the bowl and to the surrounding surfaces of up to 30 in. The range of dispersal (6) . Greater dispersal near the faucet may be attributed to the specific designs of the sink bowl and faucet in this study, which determine the contact angle of water impact. This is an important finding since many sinks in hospitals are similar in design, with faucet handles representing a high-touch surface for the sink users (38) . It can also be concluded from the dispersion experiments that secondary and successive dispersals would likely increase the degree and the scope of dispersion. There are several limitations to this work. First the use of similar sink bowls across these sinks only examines the dispersion pattern of this particular sink design. Similarly the sink-to-sink transmission may not be applicable to all wastewater plumbing systems as the fixtures on the pipe are very close together, unlike most layouts in health care settings. However, we speculate that transmission could occur on larger systems over greater time scales, especially if heavy nutrient and contamination loads were also included. GFP-expressing E. coli is a laboratory surrogate, and the putative biofilms established in the short time frame of our experiments are unlikely to be as complex or stable as biofilms developed in a hospital wastewater system over many years. However, to address the monomicrobial dominance of the GFP-expressing E. coli added to the system, we kept the system open, and other environmental organisms were able to cocolonize in an attempt to mimic the hospital system. Another limitation was the need to add nutrients to the drain to ensure rapid and robust colonization. We are not clear how widespread the practice of disposing of dextrose-containing intravenous fluids or leftover beverages in the hand-wash sinks is; however, we have observed this practice, and anecdotally it appears to be relatively common in the United States. We also did not completely characterize the droplet sizes, nor do we demonstrate air sampling to understand if the dispersion is only droplet or if there are also aerosols that contain GFP-expressing E. coli. This would require additional testing and is planned as future work.\\n\\nIn summary, this work for the first time better models the mechanisms of spread of multidrug-resistant pathogens arising from the sink drain and infecting patients. Droplet dispersion from the P-trap does not happen directly. Rather it is a multistage process: dispersal originates from the strainer and/or the bowl after growth of the biofilm up from the microbial reservoir of the P-trap. We also demonstrate sink-to-sink transmission via a common sanitary pipe. This work could have implications for patient safety, infection control, and interventions as well as the design of future hospital plumbing systems to eliminate this mode of transmission to vulnerable hospitalized patients.\\n\\nSink gallery design. A dedicated sink gallery was set up to simulate hospital hand-washing sinks. The gallery was comprised of five sink modules assembled next to each other (Fig. 4) . The five hand-wash sink stations were identical in bowl designs and dimensions and were modeled from the most common intensive care unit hand-washing sink type in the acute care hospital at the University of Virginia Medical Center. Partitions made of 24-in.-high Plexiglas sheet were installed between the sinks to prevent splatter and cross contamination. Each sink module was built with Corian integrated sink/countertops without an overflow and fitted with an 8-in. centerset 2-handle gooseneck faucet (Elkay, Oak Brook, IL). The drain line (Dearborn Brass-Oatey, Cleveland, OH). All of the fixtures were made of brass with chrome plating. Each of the sink P-traps was connected to a 3-in. common cast-iron pipe sloping into a T-joint leading into the building sanitary line located behind sink 3 (Fig. 4) .\\n\\nInoculation, growth, and establishment of GFP-expressing E. coli in sink P-traps. For the GFP-expressing E. coli strain (ATCC 25922GFP), the green fluorescent protein (GFP) gene is contained on a plasmid that also contains an ampicillin resistance gene. A single isolated colony of GFP-expressing E. coli grown from a Ϫ80°C stock was inoculated into 5 ml tryptic soy broth (TSB) (Becton, Dickinson and Company, Sparks, MD) containing 100 g/ml ampicillin (ATCC medium 2855). The inoculum concentration and method varied for each experiment. For establishment of GFP-expressing E. coli in sink P-traps, new autoclaved P-traps were filled with 100 ml 0.1ϫ TSB and inoculated with ϳ10 3 CFU/ml GFPexpressing E. coli. Following inoculation, both ends of the P-traps were covered with perforated Parafilm (Bemis, Inc., Oshkosh, WI) and allowed to incubate at room temperature (22 Ϯ 2°C) for 14 days to facilitate adherent bacterial growth. The medium in the P-trap was decanted and replaced with fresh 0.1ϫ TSB every 48 h. An aliquot of decanted medium and a swab sample from the inner surface of the P-trap were plated on tryptic soy agar (Becton, Dickinson and Company, Sparks, MD) plates containing 100 g/ml ampicillin (TSA) to monitor the growth of GFP-expressing E. coli in the P-traps. TSA plates were incubated overnight at 37°C, and CFU fluorescing under UV light were enumerated. All preparatory culturing of GFP-expressing E. coli took place in a separate room from the sink gallery to avoid unintentional contamination.\\n\\nInstallation of P-traps colonized with GFP-expressing E.coli. After the 14-day incubation, P-traps were fastened into the plumbing of the sinks (Fig. 5a) . The remainder of the drain line was either autoclaved (strainer, tailpipe, and trap arms) prior to installation or surface disinfected (sink bowl, countertop, and faucets) with Caviwipes-1 (Meterx Research, Romulus, MI), maintaining at least 1 min of contact time. After the P-trap was installed, a daily regimen was followed in which 25 ml of TSB followed by 25 ml of 0.9% NaCl solution (saline) were added in the ratio 1:3 via the strainer (Fig. 5b) to mimic the potential nutrient exposure in the hospital.\\n\\nSampling and enumeration of GFP-expressing E. coli. To monitor the growth of GFP-expressing E. coli in the plumbing, sampling ports were drilled along the length of the tailpiece (between the P-trap and the strainer) and the trap arm (between the P-trap and the common line). These holes were fitted with size 00 silicone stoppers (Cole-Parmer, Vernon Hills, IL) (Fig. 5a) . Sterile cotton swabs (Covidien, Mansfield, MA) presoaked in saline were inserted through these sampling ports, and samples were collected by turning the swab in a circular motion on the inner surface (ϳ20 cm 2 ) of the tailpipes. Sample swabs were pulse-vortexed in 3 ml saline, and serial dilutions were plated on TSA. The strainer, faucet aerator, and bowl surface were sampled with presoaked swabs and processed as described earlier.\\n\\nSink-to-sink transmission of bacteria. To investigate sink-to-sink transmission of bacteria, a distal sink (sink 5) (Fig. 4) was fitted with a P-trap inoculated with GFP-expressing E. coli. The effects of different inoculum concentrations of GFP-expressing E. coli-10 3 , 10 6 , and Ͼ10 10 CFU/ml (colonized for 14 days)-were investigated. Identification to the species level of fluorescent and nonfluorescent colonies identified from mixed pipe cultures was performed using a matrix-assisted laser desorption-ionization (MALDI)-time of flight (MALDI-TOF) mass spectrometer (Vitek-MS; bioMérieux, Durham, NC). The wastewater paths of sinks 1 to 4 were either autoclaved (strainer, tailpipe, P-traps, and trap arms) prior to installation or surface disinfected (sink bowl, countertop, and faucets) with Caviwipes-1 (Meterx Research, Romulus, MI). Faucets on each of the five sinks were turned on simultaneously for 1 min, supplying water at a flow rate of 8 liters/min, once every 24 h for 7 days. No additional feed to any of the sinks was added during this period of 7 days. On days 0 and 7, P-traps on each of the five sinks were unfastened, and swab samples from the P-trap were collected and processed as described earlier.\\n\\nDispersion measured using fluorescent microspheres. Fluoresbrite YO carboxylate microspheres (Polysciences, Inc.) which had a 1-m diameter and maximum excitation and emission of 529 nm and 546 nm, respectively, were used as a tracer in the preliminary experiments to understand droplet dispersion from the hand-wash sinks.\\n\\nTo test whether microspheres could be dispersed from below the sink strainer, a 1-ml suspension of microspheres (ϳ10 10 particles) was injected through a strainer attached to a Hert 4.5-in. offset drain tailpiece typically used for wheelchair-accessible sinks (American Standard, model 7723018.002) (Fig. 5c) . The vertical distance between the strainer and microsphere suspension injected into the tailpipe was ϳ4 in. Counter space around the sink bowl was thoroughly wiped with alcohol wipes (Covidien Webcol 6818; Kendall), and polyester sheets precut to appropriate shapes were placed on the counter to cover the entire sink counter and labeled according to position (Fig. 6a) . The faucet was turned on for 5 min at a water flow rate of 1.8 to 3.0 liters/min. Polyester sheets were harvested and immediately analyzed using a ChemiDoc MP system (Bio-Rad Laboratories, Inc.) with an exposure time of 5 s. Fluorescent microspheres were enumerated from the digital micrographs using the Image Lab Software (Bio-Rad Laboratories, Inc.).\\n\\nTo test whether microspheres could be dispersed from the surface of the sink bowl, the sink bowl was evenly coated with a 20-ml microsphere suspension (ϳ10 10 particles/ml) using a disposable swab (Sage Products, Inc., Cary, IL), and the dispersion experiment was repeated following the protocol described above. To ascertain there was no nonspecific background fluorescence in the sink and/or the water from the faucet, a control using the same protocol but without the fluorescent microspheres was performed before each experiment. Dispersion measured using GFP-expressing E.coli. Dispersion using GFP-expressing E. coli was investigated in three experiments. To test whether live organisms in the P-trap could be dispersed by running water, ϳ10 10 CFU/ml GFP-expressing E. coli in saline was added to an autoclaved P-trap and fitted into the drain line that was preautoclaved (strainer, tailpipe, and trap arms). Similarly, to test whether live organisms could be dispersed from the tailpieces of wheelchair-accessible sinks, a suspension of ϳ10 10 CFU/ml GFP-expressing E. coli was added with a syringe through the strainer into the Hert 4.5-ft offset drain tailpiece (Fig. 5c ). Just as in the microsphere dispersion experiment, the vertical distance between the strainer and GFP-expressing E. coli suspension injected into the tailpipe was ϳ4 in.\\n\\nWe next tested whether live organisms from the surface of the sink bowl could be dispersed by running water. The sink bowl surface was evenly coated with an approximately 20-ml suspension of 10 10 CFU/ml GFP-expressing E. coli.\\n\\nFinally, to mimic all of these conditions, a P-trap colonized with GFP-expressing E. coli (for 14 days) was installed, and a nutrient regimen (Fig. 5b) was followed for 14 days to intentionally promote the GFP-expressing E. coli colonization in the attached tailpipe and strainer. On day 15, the dispersion experiment was performed.\\n\\nBefore each of the GFP-expressing E. coli dispersion experiments, the counter space was thoroughly disinfected with Caviwipes-1. TSA plates were then positioned on the sink counter surrounding the bowl and an extension platform (Fig. 6b) . Additional plates were attached to the sink bowl, faucets, Plexiglas partitions, and faucet handles using adhesive tape. TSA plates were also placed 3 m away from the sink as negative controls. The faucet was turned on for 5 min with a water flow rate of 1.8 to 3.0 liters/min. Lids of the TSA plates were removed only during faucet operation. Swab samples from the faucet aerators before and after operation were collected and plated on TSA. Prior to each dispersion experiment, 50 ml water from the faucet was also collected, and aliquots were plated to assess for the presence of GFP-expressing E. coli in source water and ensure cross contamination of GFP-expressing E. coli had not occurred. A control dispersion experiment was also performed using the same protocol prior to GFP-expressing E. coli inoculation in each case. Dispersion per defined area (CFU per square centimeter) was deduced by dividing the CFU counts in the TSA plate by the surface area of the TSA plate.', 'document_id': 2585}]}, {'paragraphs': [{'qas': [{'question': 'What types of acute respiratory infections can be screened and diagnosed with multiplex PCR?', 'id': 327, 'answers': [{'text': 'influenza and parainfluenza viruses, RSV, adenovirus, rhinovirus, human metapneumovirus, enterovirus, bocavirus and coronavirus', 'answer_start': 2525}], 'is_impossible': False}], 'context': \"The impact of rapid molecular diagnostic testing for respiratory viruses on outcomes for emergency department patients\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC6617970/\\n\\nSHA: eea9d5e3d2244b3ecfb5e909515e00a4a3cabaa7\\n\\nAuthors: Wabe, Nasir; Li, Ling; Lindeman, Robert; Yimsung, Ruth; Dahm, Maria R; Clezy, Kate; McLennan, Susan; Westbrook, Johanna; Georgiou, Andrew\\nDate: 2019-03-05\\nDOI: 10.5694/mja2.50049\\nLicense: cc-by\\n\\nAbstract: OBJECTIVE: To determine whether rapid polymerase chain reaction (PCR) testing for influenza and respiratory syncytial viruses (RSV) in emergency departments (EDs) is associated with better patient and laboratory outcomes than standard multiplex PCR testing. DESIGN, SETTING: A before‐and‐after study in four metropolitan EDs in New South Wales. PARTICIPANTS: 1491 consecutive patients tested by standard multiplex PCR during July–December 2016, and 2250 tested by rapid PCR during July–December 2017. MAIN OUTCOME MEASURES: Hospital admissions; ED length of stay (LOS); test turnaround time; patient receiving test result before leaving the ED; ordering of other laboratory tests. RESULTS: Compared with those tested by standard PCR, fewer patients tested by rapid PCR were admitted to hospital (73.3% v 77.7%; P < 0.001) and more received their test results before leaving the ED (67.4% v 1.3%; P < 0.001); the median test turnaround time was also shorter (2.4 h [IQR, 1.6–3.9 h] v 26.7 h [IQR, 21.2–37.8 h]). The proportion of patients admitted to hospital was also lower in the rapid PCR group for both children under 18 (50.6% v 66.6%; P < 0.001) and patients over 60 years of age (84.3% v 91.8%; P < 0.001). Significantly fewer blood culture, blood gas, sputum culture, and respiratory bacterial and viral serology tests were ordered for patients tested by rapid PCR. ED LOS was similar for the rapid (7.4 h; IQR, 5.0–12.9 h) and standard PCR groups (6.5 h; IQR, 4.2–11.9 h; P = 0.27). CONCLUSION: Rapid PCR testing of ED patients for influenza virus and RSV was associated with better outcomes on a range of indicators, suggesting benefits for patients and the health care system. A formal cost–benefit analysis should be undertaken.\\n\\nText: The health and economic burdens associated with acute respiratory infections by influenza and respiratory syncytial viruses (RSV) are significant in Australia and overseas. 1-3 Polymerase chain reaction (PCR) testing is effective for confirming respiratory viral infections. 4 Multiplex PCR can detect numerous respiratory viruses, including influenza and parainfluenza viruses, RSV, adenovirus, rhinovirus, human metapneumovirus, enterovirus, bocavirus and coronavirus with very high sensitivity and specificity. 5 Although the results of standard multiplex PCR are accurate and comprehensive, it has traditionally been performed in a central laboratory with a lengthy turnaround time, which may be inconvenient in settings where test results are urgently required, including emergency departments (EDs).\\n\\nRapid, easy-to-use PCR-based respiratory virus diagnostic tests have been introduced in recent years; 6,7 the GeneXpert system (Cepheid), for instance, was introduced in New South Wales in July 2017. Rapid PCR tests were expected to facilitate timely and appropriate initiation of treatment, improve outbreak prevention and infection control measures, and expedite the assessment of patients in EDs.\\n\\nIn this study, we analysed routinely collected data to determine whether rapid PCR testing for influenza and RSV infections in EDs is associated with improved patient and laboratory outcomes. We compared data for patients tested for influenza A/B viruses and RSV immediately after rapid PCR diagnosis was introduced (July-December 2017) with data for patients tested with a standard multiplex PCR system during July-December 2016.\\n\\nWe undertook a before-and-after study in four metropolitan public hospital EDs in Sydney, NSW: three general hospitals (EDs A, B and C; 76 228, 54 443 and 50 025 annual ED presentations respectively) and one children's hospital (ED D; 36 700 annual ED presentations; all data for January-December 2016 8 ). The four hospitals were served by a single pathology laboratory provider.\\n\\nWe analysed data for all patients tested for influenza virus or RSV. During July-December 2016, patients were tested with the standard PCR system, a central laboratory-based multiplex PCR test for sixteen respiratory viruses (including RSV and influenza viruses A and B), available as a referral test at the central laboratory in Hospital B. During July-December 2017, patients were tested with the rapid PCR system, a hospital laboratory-based test specific for RSV and influenza viruses A and B. Hospitals A, B and D have onsite laboratories that perform rapid PCR testing; Hospital C sends samples to the nearby Hospital A.\\n\\nAll tests were conducted in virology laboratories by trained staff, and test results were entered into laboratory information system datasets. We obtained relevant patient characteristics and The proportion of patients admitted to hospital was also lower in the rapid PCR group for both children under 18 (50.6% v 66.6%; P < 0.001) and patients over 60 years of age (84.3% v 91.8%; P < 0.001). Significantly fewer blood culture, blood gas, sputum culture, and respiratory bacterial and viral serology tests were ordered for patients tested by rapid PCR. ED LOS was similar for the rapid (7.4 h; IQR, 5.0-12.9 h) and standard PCR groups (6.5 h; IQR, 4.2-11.9 h; P = 0.27).\\n\\nConclusion: Rapid PCR testing of ED patients for influenza virus and RSV was associated with better outcomes on a range of indicators, suggesting benefits for patients and the health care system. A formal cost-benefit analysis should be undertaken.\\n\\nThe known: Rapid polymerase chain reaction (PCR) testing for influenza and respiratory syncytial viruses (RSV) was introduced in New South Wales in July 2017. Its impact on outcomes for emergency department (ED) patients has not been investigated.\\n\\nThe new: Compared with standard PCR testing, rapid PCR was associated with significantly fewer hospital admissions, more rapid test turnaround, more patients receiving test results before leaving the ED, and reduced numbers of some other common microbiology tests. It did not significantly affect ED length of stay.\\n\\nThe implications: Rapid PCR testing of ED patients for major respiratory viruses can benefit patients and reduce resource use.\\n\\nMJA 210 (7) ▪ 15 April 2019 317 laboratory data by linking the ED and laboratory information system datasets. Detailed information about the datasets and the linkage process have been described previously. 9\\n\\nThe primary outcomes were admission to hospital, ED length of stay (LOS), test turnaround time, and the patient receiving their test result before leaving the ED. ED LOS was defined as the time from arrival in the ED to discharge or admission to hospital. Turnaround was defined as the time from when the sample was received by the laboratory to when the test result was available in hospital electronic records. As secondary outcomes, we compared ordering of typical biochemistry and haematology tests (full blood count; electrolyte, urea, creatinine levels; liver function test; blood gas analysis; C-reactive protein level) and microbiology tests (blood culture; urine microscopy, culture and sensitivity analysis; sputum culture; respiratory bacteria and virus serology) during the two study periods.\\n\\nAnalyses were conducted in Stata 15 (StataCorp). Descriptive statistics are reported (medians with interquartile ranges [IQRs], means with standard deviations [SDs], numbers with proportions). Baseline characteristics were compared in χ 2 tests (categorical outcomes) or Wilcoxon rank-sum tests (continuous outcomes). Logistic regression analyses of binary outcomes (eg, hospital admission: yes/no) and quantile regression analyses of continuous outcomes (eg, ED LOS) were undertaken. All analyses were adjusted for baseline characteristics.\\n\\nSensitivity analyses limited to data for children (under 18 years of age) or older patients (over 60 years of age) were conducted. P < 0.05 (2-tailed) was deemed statistically significant.\\n\\nEthics approval for the investigation was granted by the Human Research Ethics Committee of the South Eastern Sydney Local Health District (reference, HREC/16/POWH/412).\\n\\nWe analysed data for 3741 patients presenting to the four EDs during two periods: 1491 consecutive patients during July-December 2016 (standard PCR) and 2250 during July-December 2017 (rapid PCR). Baseline characteristics for the two groups were similar in terms of sex, triage category, and arrival day of the week, but differed significantly for age, arrival time, and mode of arrival (Box 1 \\n\\nThe overall numbers of tests per patient were similar in the standard PCR (mean, 7.2 tests; SD, 3.8) and rapid PCR groups (mean, 7.1 tests; SD, 3.4). The mean number of microbiology tests per patient was significantly lower for the rapid PCR group (1.5 tests; SD, 1.8) than for the standard PCR group (2.0 tests; SD, 2.1; P < 0.001 after controlling for baseline characteristics).\\n\\nThe 16 265 biochemistry/haematology and microbiology tests comprised 71.1% of the 22 876 other tests (that is, not including PCR virus testing) ordered for patients in the study. After adjusting for baseline characteristics, the proportions of patients for whom full blood count, electrolyte/urea/creatinine levels, liver function, or C-reactive protein were assessed were similar, as were the proportions for urine microscopy, culture and sensitivity tests. Significantly fewer blood culture, blood gas, sputum culture, and respiratory bacterial and viral serology tests were ordered for patients in the rapid PCR group (Box 4). Information, figure 1 ).\\n\\nED LOS was similar for the standard PCR and rapid PCR groups in both age-based subgroups (Supporting Information, figure 2A ). The differences in test turnaround time identified in the main analysis were also evident for each age-based subgroup (Supporting Information, figure 2B ).\\n\\nIn this before-and-after study, we found that rapid PCR testing of ED patients for major respiratory viruses was associated with significantly fewer admissions to hospital, more rapid delivery of test results, more patients receiving their test results before leaving the ED, and less frequent ordering of some common microbiology tests.\\n\\nOther studies have also reported that hospital admission numbers were significantly lower when rapid influenza virus testing was used in EDs. An analysis of outcomes for more than 300 adults at a tertiary care centre in New York found that early diagnosis of respiratory infections was associated with significantly fewer hospitalisations of influenza-positive patients. 7 In a small Irish study (73 patients), the hospital admission rate for obstetric patients declined from 88% to 45% after on-site rapid influenza PCR testing was introduced. 10 The differences in clinical setting and patient group may explain the smaller decline in our study (from 78% to 67%). Non-PCR-based rapid diagnostic tests for respiratory viruses have also been associated with lower hospital admission rates. 11, 12 The main reason for fewer hospital admissions of patients tested by rapid PCR may be that the earlier availability of results enables clinicians to quickly diagnose or exclude respiratory infections and to make timely and informed decisions about whether to discharge the patient or admit them to hospital. When standard 2 Primary outcomes for 3741 patients at four metropolitan emergency departments (EDs) tested for influenza and respiratory syncytial viruses by standard or rapid polymerase chain reaction (PCR) \\n\\nAfter adjusting for baseline characteristics (Box 1): * P = 0.012; ** P < 0.001. ◆\\n\\nMJA 210 (7) ▪ 15 April 2019 PCR was used, in contrast, our findings suggest that these decisions were made before the test results were available. The possible benefits of not admitting patients to hospital, beyond those for individual patient management, include better infection control and outbreak prevention, as well as reduced demands on hospital resources. 13, 14 The impact of rapid PCR testing on outbreak prevention and infection control measures should be evaluated. Rapid influenza virus testing may also have practical implications for hospital bed management. 10, 15 ED LOS was similar in our study before and after the introduction of rapid PCR methods. This finding was not unexpected, as test turnaround time is not the only rate-limiting factor for decision making in EDs. 16 Before rapid PCR methods were introduced, the long turnaround time of multiplex PCR did not necessarily extend a patient's stay in the ED, as they were usually admitted to hospital or discharged home before the results were available. Consequently, more rapid delivery of test results alone would not reduce ED LOS.\\n\\nReports on the effect of rapid influenza virus testing and LOS have been conflicting. While evidence for an association between rapid testing and shorter overall inpatient LOS has been reported, 6,11 findings regarding ED LOS are inconsistent. 7, 17, 18 For example, a Cochrane review based on three randomised controlled trials did not find a statistically significant association of rapid viral diagnosis with lower mean ED LOS. 18 In a study in children, ED LOS was actually 26 minutes longer with rapid PCR; inpatient LOS did not differ between the two groups, but was significantly shorter when the analysis was limited to patients with positive test results. 6 We found that ordering of some other microbiology tests, including blood culture, sputum culture, and respiratory bacterial and virus serology, was significantly less frequent for patients tested by rapid PCR. The effect of PCR-based rapid testing on the ordering of other laboratory tests has not previously been reported, although some studies of antigen-based pointof-care testing have examined this outcome. 12 Consistent with our finding, several investigators have reported fewer blood culture tests 19, 20 and basic biochemistry and haematology tests, including full blood count, 20,21 C-reactive protein testing, 21 and urinalysis, 20,21 when point-of-care testing was used.\\n\\nThe higher rate of positive results for patients tested by rapid PCR than for those tested by standard PCR may reflect a higher prevalence of influenza during 2017 than in 2016.\\n\\nThe rapid PCR system in our study accurately detects influenza viruses A/B and RSV but, unlike the standard multiplex PCR, cannot detect other clinically relevant respiratory viruses, such as rhinovirus, coronavirus, human metapneumovirus, parainfluenza virus, adenovirus, enterovirus, and bocavirus. If infection with other respiratory viruses is suspected, patients may therefore need further investigations. Standard multiplex PCR can provide broader information, as it can detect multiple respiratory viruses in a single run, although the long turnaround time restricts its suitability for urgent clinical decision making. Improving the turnaround time of multiplex PCR analysis may achieve better outcomes.\\n\\nThe strengths of our study include its relatively large sample size; further, unlike many similar investigations, ours was a multicentre study in four hospital EDs, enhancing the generalisability of our findings. However, our analyses were not adjusted for comorbid conditions, as this information was not available. Because our study was an uncontrolled before-and-after study, our results cannot be interpreted as indicating causal relationships. As with all non-randomised trials, we could not fully account for all potential confounding variables, nor for temporal trends and other unmeasured factors, such as changes in clinician testing practices or differences in the prevalence and severity of disease during the two study periods. 22 For example, a shortage of inpatient beds caused by a high prevalence of influenza could influence decisions in a busy ED about admitting patients to hospital. However, we attempted to reduce seasonal effects by selecting similar time frames for the two study periods, to reduce selection bias by including all ED patients tested for influenza virus and RSV, and to control for differences in baseline patient characteristics by applying multivariate modelling. As medications data were not available to us, we were unable to assess the impact of rapid PCR testing on antibiotic and antiviral drug use. Similarly, the cost-benefit balance of rapid testing was not evaluated because relevant data were not available. The cost per patient of rapid PCR testing is generally higher than for central laboratory testing, but our findings suggest potential savings through lower numbers of hospital admissions and reduced resource use. This question could be evaluated in a further study.\\n\\nRapid PCR testing for influenza virus and RSV infections in patients attending EDs was associated with significant improvements in a range of patient and laboratory outcomes, suggesting potential benefits for both the patients and the health care system. A cost-benefit analysis could examine the impact of rapid PCR testing on bed management and antimicrobial drug prescribing.\", 'document_id': 1556}]}, {'paragraphs': [{'qas': [{'question': 'What is the role of antibodies during infection?', 'id': 465, 'answers': [{'text': 'Antibodies against foreign antigens are a critical component of the overall immune response and can facilitate pathogen clearance during a primary infection and also protect against subsequent infections.', 'answer_start': 283}], 'is_impossible': False}, {'question': 'How can antibodies also create health problems?', 'id': 467, 'answers': [{'text': 'Dysregulation of the antibody response can lead to an autoimmune disease, malignancy, or enhanced infection. ', 'answer_start': 488}], 'is_impossible': False}, {'question': 'Which technology  invention produced antibodies that are clones of a unique parent cell?', 'id': 474, 'answers': [{'text': ' in the 1970s with the development of hybridoma technology to produce monoclonal antibodies ', 'answer_start': 2257}], 'is_impossible': False}, {'question': 'What mechanism is responsible for the creation of diversified repertoire for antibodies?', 'id': 476, 'answers': [{'text': 'somatic rearrangement during B cell differentiation was responsible for antibody diversification ', 'answer_start': 2372}], 'is_impossible': False}, {'question': 'What  developments have been made possible by the study of B-cell repertoire?', 'id': 478, 'answers': [{'text': '(1) vaccine candidates that elicit protective antibodies; (2) antibodies that prevent disease when given prophylactically; and (3) antibodies that can be given as therapy after the onset of disease.', 'answer_start': 2996}], 'is_impossible': False}, {'question': 'What motivates the study of the rare B-cells that produce Broadly Neutralizing Antibodies (bnAb)?', 'id': 485, 'answers': [{'text': 'discovery of broadly neutralizing antibodies (bnAbs) that protect against infection across diverse viral isolates has intensified efforts to understand the developmental pathway of the rare B cells that produce these antibodies', 'answer_start': 3589}], 'is_impossible': False}, {'question': 'How has the study of B-cells helped the treatment for Respiratory syncytial virus (RSV)?', 'id': 489, 'answers': [{'text': \"For RSV, stabilized versions of the fusion (F) protein in the pre-fusion conformation have led to insights in the B cell's response to infection and has generated potentially safer and more efficacious vaccine candidates\", 'answer_start': 4083}], 'is_impossible': False}, {'question': 'How are the studies on B-cells helping the development of a universal influenza vaccine?', 'id': 490, 'answers': [{'text': 'Influenza also performs fusion through the stem region of the hemagglutinin protein, and the identification of B cells that target this relatively conserved site has spurred research on the development of a universal influenza vaccine (', 'answer_start': 4315}], 'is_impossible': False}, {'question': 'What role B-cell  play in malaria infection and prevention?', 'id': 494, 'answers': [{'text': ' Rare memory B cells producing antibodies specific for the EBV fusion machinery have been isolated; these can neutralize both B cell and epithelial cell infection (20). A new paradigm in malaria vaccine development is also emerging with the discovery of IgM+ and IgD+ memory B cells targeting the Merozoite Surface Protein 1, that rapidly respond to malaria re-infection', 'answer_start': 4790}], 'is_impossible': False}, {'question': 'How can the study of B-cells help in the prevention and treatment of autoimmune diseases?', 'id': 497, 'answers': [{'text': 'The study of autoantigen-specific B cells and a detailed analysis of B cell subsets with pathogenic potential in humans could lead to a better understanding of how to prevent and treat autoimmune diseases.', 'answer_start': 6415}], 'is_impossible': False}], 'context': 'Techniques to Study Antigen-Specific B Cell Responses\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC6667631/\\n\\nSHA: ee632fa425607e8ff91fc3730bc0782d43ce9c0c\\n\\nAuthors: Boonyaratanakornkit, Jim; Taylor, Justin J.\\nDate: 2019-07-24\\nDOI: 10.3389/fimmu.2019.01694\\nLicense: cc-by\\n\\nAbstract: Antibodies against foreign antigens are a critical component of the overall immune response and can facilitate pathogen clearance during a primary infection and also protect against subsequent infections. Dysregulation of the antibody response can lead to an autoimmune disease, malignancy, or enhanced infection. Since the experimental delineation of a distinct B cell lineage in 1965, various methods have been developed to understand antigen-specific B cell responses in the context of autoimmune diseases, primary immunodeficiencies, infection, and vaccination. In this review, we summarize the established techniques and discuss new and emerging technologies for probing the B cell response in vitro and in vivo by taking advantage of the specificity of B cell receptor (BCR)-associated and secreted antibodies. These include ELISPOT, flow cytometry, mass cytometry, and fluorescence microscopy to identify and/or isolate primary antigen-specific B cells. We also present our approach to identify rare antigen-specific B cells using magnetic enrichment followed by flow cytometry. Once these cells are isolated, in vitro proliferation assays and adoptive transfer experiments in mice can be used to further characterize antigen-specific B cell activation, function, and fate. Transgenic mouse models of B cells targeting model antigens and of B cell signaling have also significantly advanced our understanding of antigen-specific B cell responses in vivo.\\n\\nText: In his Nobel lecture in 1908, Paul Ehrlich likened the antibody-antigen interaction to a lock and key. He reasoned that antitoxins (antibodies) contained in a solution in the serum of immunized animals must be identical to a cellular receptor \"for a really well-made key will not open different locks at the same time\" (1) . It took almost five decades before immunofluorescence microscopy was used to confirm the cellular origin of antibodies (2) . Major strides in the B cell and antibody field followed in the 1970s with the development of hybridoma technology to produce monoclonal antibodies and the discovery that somatic rearrangement during B cell differentiation was responsible for antibody diversification (3, 4) . The subsequent explosion of available monoclonal antibodies led to revolutionary diagnostic, therapeutic, and research reagents to distinguish different types of immune cells (5) . Together, these discoveries have allowed us to probe humoral immunity at the level of the antigen-specific B cell.\\n\\nMethods to probe the antigen-specific B cell response have advanced our understanding of how to harness the remarkable breadth of the B cell repertoire and the exquisite specificity of the individual B cell in developing (1) vaccine candidates that elicit protective antibodies; (2) antibodies that prevent disease when given prophylactically; and (3) antibodies that can be given as therapy after the onset of disease. Many of the vaccines currently available were originally developed empirically either by inactivating, attenuating, or administering a subunit of the pathogen. However, vaccine development against pathogens that are traditionally difficult to vaccinate against may rely on a deeper investigation of the B cell response to the antigens exposed on the surface of these pathogens.\\n\\nFor HIV-1, the discovery of broadly neutralizing antibodies (bnAbs) that protect against infection across diverse viral isolates has intensified efforts to understand the developmental pathway of the rare B cells that produce these antibodies (6) (7) (8) (9) . Insights into the ontogeny of these rare B cells could allow the design of a step-wise vaccine regimen that stimulates the germ-line precursor to expand and mature to produce circulating bnAbs which could protect against HIV acquisition (10, 11) . For RSV, stabilized versions of the fusion (F) protein in the pre-fusion conformation have led to insights in the B cell\\'s response to infection and has generated potentially safer and more efficacious vaccine candidates (12, 13) . Influenza also performs fusion through the stem region of the hemagglutinin protein, and the identification of B cells that target this relatively conserved site has spurred research on the development of a universal influenza vaccine (14) (15) (16) . Like RSV, HIV, and influenza, the fusion proteins of EBV and CMV exist in a pre-fusion conformation, and stabilization in their pre-fusion states could greatly accelerate vaccine development against these pathogens (17-19). Rare memory B cells producing antibodies specific for the EBV fusion machinery have been isolated; these can neutralize both B cell and epithelial cell infection (20). A new paradigm in malaria vaccine development is also emerging with the discovery of IgM+ and IgD+ memory B cells targeting the Merozoite Surface Protein 1, that rapidly respond to malaria re-infection (21). Further, highly potent neutralizing antibodies targeting a novel and conserved site on the Circumsporozoite Protein have been isolated from B cells (22). Together, these examples demonstrate the importance of studying antigen-specific humoral responses to infectious diseases. The solutions to the crystal structures of surface proteins for a variety of pathogens, the conformational stabilization of these antigens, and the application of the methods summarized in this review, to probe antigen-specific B cell responses, have created new opportunities for systematic and rational vaccine design for HIV, RSV, EBV, malaria, and many other pathogens.\\n\\nThe study of B cell responses has not only informed vaccine design but has also advanced our understanding of antibodymediated autoimmune diseases, such as rheumatoid arthritis and systemic lupus erythematosus (23, 24). Up to 20% of mature, naïve B cells have receptors with the capacity to bind self-antigens (25). Although these cells are potentially pathogenic, the deletion of B cells with high affinity to self-antigen through apoptosis, anergy of B cells with low affinity to self-antigen, and the absence of T cell help combine together to protect against autoimmune disease in mice (26). The study of autoantigen-specific B cells and a detailed analysis of B cell subsets with pathogenic potential in humans could lead to a better understanding of how to prevent and treat autoimmune diseases.\\n\\nAlthough the term antigen-specific B cell is used throughout this mini-review to denote the analysis of B cells based on binding between the B cell receptor (BCR) and a specific antigen used as bait, it is important to keep in mind that BCRs within the polyclonal B cell repertoire exhibit a spectrum of polyreactivity. On one end of the spectrum, a highly polyreactive BCR is able to bind multiple structurally unrelated antigens with physiologically relevant affinities. The frequency of polyreactivity in the normal adult human B cell repertoire has been estimated to be 4% of naïve B cells, 23% of IgG+ memory B cells, and 26% of intestinal IgA+ and IgG+ plasmablasts (27-29). On the other end of the spectrum, a mono reactive BCR is activated only when it encounters a single cognate antigen. Although there are exceptions, the accumulation of somatic hypermutations within the variable regions of the BCR during the process of affinity maturation is generally thought to lead to increased affinity and specificity for the cognate antigen (30, 31).\\n\\nSeveral general techniques are commonly used to identify antigen-specific B cells ( Table 1 ). The B cell enzyme linked immunospot (ELISPOT) technique relies on the principle of capturing the secreted antibody in the vicinity of each cell. In the B cell ELISPOT, antibody secreting B cells (ASCs) present in a sample or differentiated in vitro are added to plates coated with the antigen of interest. Antigen-specific antibodies will bind in close proximity to the location of the individual B cells producing those antibodies. Enzyme or fluorescent labeled secondary antibodies are then used to visualize spots of antibody secretion and binding to plate-bound antigen at the location of the ASCs. Each spot corresponds to antibody produced from a single antigen-specific B cell and therefore the technique is extremely sensitive. Secondary antibodies conjugated to combinatorial colored beads can also be used to detect the antibodies secreted from individual B cells with the advantage of multiplexing the assay (32). One limitation of the assay is its requirement for antibody secretion by B cells thereby limiting the assay to only a subset of B cells in the repertoire, namely ASCs (33). Memory B cells can be stimulated in vitro to differentiate into ASCs prior to addition to the antigen-coated plate (34) . Further, the antigenspecific B cells identified by ELISPOT are generally not available for downstream analysis.\\n\\nLimiting dilution is another technique that has been used to isolate antigen-specific B cells. In this approach, primary cells can be diluted serially until individual B cells are separated in microwell plates (36) . The B cells can then be cultured and expanded ex vivo and/or immortalized using EBV such that each well contains a monoclonal antibody (3, 37, 38) . Antigen-specific B cells can be selected by screening the culture supernatants for monoclonal antibodies that bind an antigen of interest. Although antibodies can be sequenced and cloned, the requirement for an ex vivo culture prior to selection precludes determination of the transcriptional profile of the original B cell in this approach. This technique can potentially be time-consuming and laborious, but the use of microfluidics and robotics has greatly improved the throughput for selecting antigen-specific B cells (39) . Advances in single cell next generation sequencing technology have allowed high throughput transcriptional profiling and sequencing of paired immunoglobulin heavy and light chains (40) . In this approach, antigen specificity can be tested after monoclonal antibodies are cloned and produced using the sequencing data. This method can be useful in identifying antigen-specific B cells that have undergone clonal expansion after vaccination or acute infection (41) . Flow cytometry is the most common method used for single cell analysis and isolation (39) . Flow cytometry-based analysis of antigen-specific B cells is dependent on labeling antigen with a fluorescent tag to allow detection. Fluorochromes can either be attached covalently via chemical conjugation to the antigen, expressed as a recombinant fusion protein, or attached non-covalently by biotinylating the antigen. After biotinylation, fluorochrome-conjugated streptavidin is added to generate a labeled tetramer of the antigen. Biotinylation of the antigen at a ratio ≤1 biotin to 1 antigen is important, since each streptavidin has the potential to bind four biotins. If the ratio of biotin to antigen is >1:1, then clumping and precipitation of the antigen out of solution can occur as soon as streptavidin is added. Alternatively, site directed biotinylation can be accomplished by adding either an AviTag or BioEase tag to the recombinant antigen prior to expression (77, 78) . When site-specific biotinylation is utilized, researchers must keep in mind that the tag may occlude an epitope from recognition by B cells which can be problematic for vaccine antigens. Further, for proteins that oligomerize, multiple tags may be incorporated, possibly resulting in aggregation.\\n\\nAnother important consideration is the potential for confounding by B cells in the repertoire that bind to the fluorochrome, streptavidin, or any linkers rather than to the antigen of interest. Binding between fluorochromes, linkers, or streptavidin and BCRs from humans and mice never exposed to these antigens are generally of low affinity, and these BCRs are generally expressed by naïve and potentially polyreactive B cells (62, 79, 80) . Dual labeling, in which the same antigen is separately labeled with two different fluorochromes, can be used to identify double positive B cells and remove confounding by B cells that bind the fluorochrome (12, 42) . However, even when tetramers are utilized for dual labeling, streptavidin-specific B cells will contaminate the double positive population. To fully remove confounding from the fluorochrome, streptavidin, and linkers, a \"decoy\" tetramer can be used to identify these contaminating B cells (21, 26). In this approach, the same fluorochrome used to identify antigen-specific B cells is conjugated to a different fluorochrome such that the emission spectrum is altered by fluorescence resonance energy transfer (FRET) (26). Decoy-binding B cells can therefore be excluded from the true antigen-specific B cells. Notably, it is critical to use the same source of fluorochrome conjugated streptavidin in the tetramer and decoy reagent, because conjugation methods, recombinant streptavidin, and protein fluorochromes like R-phycoerythrin vary enough from company to company to alter some of the epitopes available for B cells to bind.\\n\\nOne weakness of the flow cytometric approach is the reliance on antigens that can be readily conjugated to a fluorochrome or biotinylated. In addition to recombinant proteins and synthesized peptides, labeled polysaccharides, lipids, haptens, virus-like particles, and pseudo viruses have also been used to identify antigen-specific cells by flow cytometry (33, [43] [44] [45] [46] [47] [48] [49] [50] [51] [52] [53] [54] [55] [56] [57] [58] [59] . Further, epitope-specific B cells have been identified by screening bacteriophage-displays or microarray peptide libraries with polyclonal antibodies targeting the native antigen to select conformational epitopes that can be fused to fluorescent proteins for use in flow cytometry (47, 60) .\\n\\nWith technologic advancements increasing the number of simultaneously measurable parameters, antigen-specific B cells can be further characterized by cell surface markers and intracellular staining. Additionally, the immunoglobulin capture assay is a flow cytometry-based adaptation of the ELISPOT assay in which a streptavidin-conjugated anti-CD45 antibody carrying four biotinylated anti-IgG antibodies is used to simultaneously bind plasmablasts and capture secreted antibody followed by fluorescent-labeled antigen to detect antigenspecific plasmablasts (61) . The mean fluorescence intensity measured by flow cytometry and normalized to the level of BCR expression also provides a measure of the relative amount of antigen binding to a B cell and can be used as a rough surrogate for binding affinity (79, 81, 82) . Preincubation of B cells with increasing concentrations of a monomeric antigen prior to labeling with tetrameric antigen can also be used to further quantify binding affinity. Cells expressing high affinity BCRs will bind monomeric antigen at low concentrations, whereas low affinity BCRs will require higher concentrations of monomeric antigen to compete with and inhibit tetramer binding (26). Individual cells can also be isolated by fluorescence activated cell sorting (FACS) for downstream analysis, including BCR sequencing and cloning, BCR affinity measurement, in vitro proliferation, and transcriptional profiling. \\n\\nMethods have recently been developed to further improve the sensitivity for detecting rare antigen-specific B cells. Magnetic nanoparticles conjugated to antibodies targeting the fluorochrome on the antigen of interest, allow for the enrichment of antigen-specific B cells prior to flow cytometry (20, 26, 80, 83) . This approach is particularly useful for detecting rare antigenspecific naïve B cells, autoreactive B cells, memory B cells, and plasmablasts (21, 26, 47, 50) . The magnetic enrichment strategy allows for the analysis of significantly more cells in a shorter period of time by concentrating the cells of interest prior to flow cytometry (Figure 1) . Notably, as with any method that seeks to identify a population of cells at a very low frequency, the background and noise inherent in the detection system is magnified with respect to the signal of interest, especially when that signal is weak. Therefore, to detect the antigen-specific population of interest, the following considerations are critical: (1) Using decoys to exclude B cells of unwanted specificities;\\n\\n(2) careful design of flow cytometry panels to avoid emission spillover into the channel for the antigen of interest; and (3) choosing the brightest fluorochromes, like R-phycoerythrin or allophycocyanin.\\n\\nIn vivo methods to probe antigen-specific B cell responses in the presence of other antigen-presenting cells and T cell helpers, have increased our mechanistic understanding of the humoral immune response during vaccination, infection, and autoimmunity. Adoptively transferred B cells can be distinguished from recipient lymphocytes by taking advantage of mouse strains with allelic variations in CD45 or mice devoid of B cells. The adoptively transferred B cells can come from wildtype mice or from mice expressing transgenic BCRs ( Table 2) , and antigen-specific B cells can be analyzed using the techniques described above.\\n\\nMicroscopy is another general technique that has been used to identify antigen-specific cells in vivo and offers the advantage of direct visualization. In the first reported application of this technique to demonstrate the cellular origin of antibodies in 1955, fluorescein-conjugated antibodies against ovalbumin and human immunoglobulin were used to stain tissue sections of the spleen from hyperimmune rabbits (2) . Since then, other groups have fluorescently labeled antigens to localize antigen-specific B cells by microscopy (62, 65) . Advances in laser capture dissection microscopy, already used in the T cell field, also provide an opportunity for isolating individual antigen-specific B cells for downstream analysis, including sequencing and cloning of the BCR or transcriptional profiling (66) . However, antigen staining of BCRs in situ can be challenging depending on the binding of antigens from pathogens to other cellular receptors or an alteration of BCR specificity during tissue fixation or processing. Two-photon or multiphoton microscopy has the ability to resolve images at greater depths and with less photobleaching than confocal microscopy (67, 68) . As a result, this technology has allowed real-time imaging in living, intact lymphoid tissues of mice, permitting the direct in vivo observation of immune cell interactions. The dynamic movements and interactions of antigen-specific B cells can be studied in vivo by combining an adoptive transfer of individual B cells (isolated by limiting dilution or FACS) with two-photon microscopy (63, 69, 70) .\\n\\nHumanized mouse models are powerful tools for translating experiments in mice to applications in humans. Transgenic mice that produce humanized cytokines by knock-in replacement can be used to support human hematopoietic stem cells (104) . Transgenic mice with complete humanization of the mouse immunoglobulin loci provide an opportunity for recapitulating the breadth of the human B cell repertoire and serve as a valuable tool for therapeutic antibody discovery (71) . However, one caveat is that the allele frequencies found in the B cell repertoires of these mouse models may not necessarily recapitulate those found in humans (72) . Mass cytometry has the potential to provide further high-dimensional analysis of antigen-specific B cells. In this method, heavy metal ion tags rather than fluorochromes are used to label cells. Since data is collected as time-offlight mass spectrometry, up to 42 unique parameters can be simultaneously measured from a single sample without significant spillover between channels or the need for compensation. Mass cytometry with heavy metal-labeled tetramers can be constructed using streptavidin (73) . Mass cytometry with metal-labeled peptide-MHC tetramers has been used successfully to identify and characterize antigen-specific T cells, but to our knowledge has not yet been applied to antigen-specific B cells (73, 74) . One limitation of this approach is that cells are unavailable for downstream analysis since they are vaporized by a plasma torch to atomize the ion tags. However, by simultaneously detecting many more surface markers and intracellular cytokines, transcription factors, and detecting more signaling molecules from individual cells than previously possible with traditional fluorescent labels, the application of mass cytometry with dimensionality reduction algorithms could help dissect the complexity of the B cell compartment, provide a higher resolution view of B cell development, and reveal novel subsets of antigen-specific B cells involved in mediating autoimmune diseases or protection against infection.\\n\\nOn the horizon, single cell RNA-sequencing (RNA-seq) technologies have the potential to revolutionize the study of antigen-specific immune cells (75, 76) . The ability to generate a library of tetramers with unique barcodes could allow the simultaneous examination of gene expression profiles from a large number of cells with different antigen specificities in a single experiment. Combining barcoded tetramers with oligonucleotide-conjugated antibodies and RNA-seq to simultaneously measure the protein and gene expression of antigen-specific cells could further increase the amount of unbiased multi-omic information about individual antigen-specific cells in normal and disease states and aid the rational design of vaccines and therapeutics (105) (106) (107) .\\n\\nThe ongoing analysis of antigen-specific B cell responses has led to the development of new diagnostic, therapeutic, and research reagents. Methods for studying antigen-specific B cell responses are being increasingly applied to tackle diseases like HIV, RSV, and autoimmune diseases, in which the immune response either fails to protect or clear disease, or where it enhances disease or is responsible for the disease itself. Considerable opportunities exist on the horizon for applying these methods to a myriad of diseases in which B cells play an active role.\\n\\nJB and JT reviewed the literature, generated figures and tables, and wrote the manuscript.', 'document_id': 1569}]}, {'paragraphs': [{'qas': [{'question': 'When was the  Middle East Respiratory Syndrome Coronavirus isolated first?', 'id': 557, 'answers': [{'text': '(MERS-CoV) was first isolated in 2012, in a 60-year-old man who died in Jeddah, KSA due to severe acute pneumonia and multiple organ failure', 'answer_start': 2273}], 'is_impossible': False}, {'question': 'What is the Case fatality rate for MERS Coronavirus?', 'id': 558, 'answers': [{'text': 'Recent fatality rate (CFR) of 21%', 'answer_start': 2686}], 'is_impossible': False}, {'question': 'How does gender influence MERS-COV infection?', 'id': 559, 'answers': [{'text': ' MERS-CoV infects males more than females', 'answer_start': 2869}], 'is_impossible': False}, {'question': 'Which is the source animal for the MERS-COV?', 'id': 560, 'answers': [{'text': 'Dromedary camels are the major animal source of MERS-CoV transmission to humans.', 'answer_start': 3322}], 'is_impossible': False}, {'question': 'What is the median time until death in MERS-COV?', 'id': 562, 'answers': [{'text': 'median time until death is 11-13 days (range 5-27 days) among severely ill patients', 'answer_start': 3925}], 'is_impossible': False}, {'question': 'What is the incubation period for MERS-COV?', 'id': 563, 'answers': [{'text': ' incubation period of 16 days with a mean of 5-6 days [', 'answer_start': 3850}], 'is_impossible': False}, {'question': 'What is the treatment for MERS-COV?', 'id': 564, 'answers': [{'text': 'ere is no specific treatment for MERS-CoV. Like most viral infections, the treatment options are supportive and symptomatic', 'answer_start': 4155}], 'is_impossible': False}, {'question': 'What age group had the most MERS-COV infections?', 'id': 565, 'answers': [{'text': 'majority of confirmed cases of MERS-CoV were reported among people aged 40 and above', 'answer_start': 11825}], 'is_impossible': False}], 'context': \"Demographic Variations of MERS-CoV Infection among Suspected and Confirmed Cases: An Epidemiological Analysis of Laboratory-Based Data from Riyadh Regional Laboratory\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC7049846/\\n\\nSHA: edee452881f826fb72c58ee68a982789b12aa99d\\n\\nAuthors: Altamimi, Asmaa; Abu-Saris, Raghib; El-Metwally, Ashraf; Alaifan, Taghreed; Alamri, Aref\\nDate: 2020-02-19\\nDOI: 10.1155/2020/9629747\\nLicense: cc-by\\n\\nAbstract: Introduction. Middle East respiratory syndrome coronavirus was first recognized in September 2012 in Saudi Arabia. The clinical presentations of MERS and non-MERS SARI are often similar. Therefore, the identification of suspected cases that may have higher chances of being diagnosed as cases of MERS-CoV is essential. However, the real challenge is to flag these patients through some demographic markers. The nature of these markers has not previously been investigated in Saudi Arabia, and hence, this study aims to identify them. METHODS: It was a surveillance system-based study, for which data from a total of 23,646 suspected patients in Riyadh and Al Qassim regions were analyzed from January 2017 until December 2017 to estimate the prevalence of MERS-CoV among suspected cases and to determine potential demographic risk factors related to the confirmation of the diagnosis. RESULTS: Of 23,646 suspected cases, 119 (0.5%) were confirmed by laboratory results. These confirmed cases (67.2% of which were males) had a mean age of 43.23 years (SD ± 22.8). Around 42.2% of the confirmed cases were aged between 41 and 60 years and about 47% of confirmed cases had their suspected specimen tested in the summer. The study identified three significant and independent predictors for confirmation of the disease: an age between 41 and 60 years, male gender, and summer season admission. CONCLUSION: The study provides evidence that the MERS-CoV epidemic in the subject regions has specific characteristics that might help future plans for the prevention and management of such a contagious disease. Future studies should aim to confirm such findings in other regions of Saudi Arabia as well and explore potential preventable risk factors.\\n\\nText: A respiratory viral disease caused by the Middle East Respiratory Syndrome CoronaVirus (MERS-CoV) was first isolated in 2012, in a 60-year-old man who died in Jeddah, KSA due to severe acute pneumonia and multiple organ failure [1] . Since then, 27 countries have reported the presence of this virus, including the 12 countries of the Eastern Mediterranean region. Several outbreaks have occurred in multiple countries including Saudi Arabia, the United Arab Emirates and the Republic of Korea [2] . Recent fatality rate (CFR) of 21% [5, 6] . Very limited evidence is available for exploring the epidemiology of this virus among the pediatric population [7] . e literature shows that MERS-CoV infects males more than females [8, 9] . e casefatality rate of men (52%) is higher than that of women (23%) [10] . Males with a history of serious medical conditions are highly susceptible to this infection. Moreover, the mean age of infection in adults is 60 years [10] . e mode of transmission is not entirely understood yet [2] ; however, human-to-human [11] and zoonotic sources of transmission [12] have been documented in many studies. Dromedary camels are the major animal source of MERS-CoV transmission to humans. Interhuman transmission of the virus did not occur easily, but it is seen mainly in patients' families and healthcare settings [2] . Clinical pictures of this infection varied from asymptomatic to mild respiratory symptoms to severe respiratory distress and death [2] . Severe ailment can often cause respiratory catastrophes that need mechanical ventilation and support in ICUs across different healthcare settings [4] . Studies have suggested an incubation period of 16 days with a mean of 5-6 days [12, 13] , while the median time until death is 11-13 days (range 5-27 days) among severely ill patients [13] . e gold standard test for the detection of this virus is real-time reverse-transcription polymerase chain reaction (rRT-PCR) assays [14] .\\n\\nere is no specific treatment for MERS-CoV. Like most viral infections, the treatment options are supportive and symptomatic [2] . At present, no vaccine exists for preventing the infections of MERS-CoV. e CDC indicated that preventative actions should be taken for any type of respiratory illness [4] . Such actions include washing hands with water and soap for around 20 seconds or using hand sanitizers with alcohol if no water is available. One must cover their nose and mouth during instances of sneezing and coughing with a tissue and avoid touching the mouth, nose, or eyes with their hands until washed properly. Repeatedly touched surfaces, such as door knobs, should be disinfected and cleaned regularly. Intimate personal contact, e.g., kissing, and sharing cups or eating utensils must also be avoided [15] .\\n\\nMany studies have been conducted in recent years in Saudi Arabia to combat this deadly disease. A large multicentre study showed that it is nearly impossible to differentiate between patients of MERS-CoV and non-MERS-CoV just on the basis of clinical presentation [16] . Another cohort study, which was hospital-based (17 cases vs. 82 controls), found that there were statistically significant differences in terms of gender, clinical, and radiographic presentations [17] . Similarly, two more single-centre case control studies reported that the presenting symptoms of MERS-CoV infection were not specific [18, 19] .\\n\\nPhysicians and public health practitioners need to identify suspected cases which have higher chances of diagnosis as confirmed cases prior to laboratory testing (which usually takes between 12 and 24 hours). Identification of a confirmed case is necessary to implement preventive strategies to combat the spread of the disease to family members and hospital healthcare workers [20] . Mild symptomatic cases, which result in a positive PCR, may be isolated at home. Severe to moderate cases should be admitted to and isolated in a hospital until they improve and then be discharged for isolation at home for an extended period. Both mild and severe cases are retested after 7 days, and the test is subsequently repeated after every 3 days until a negative result is obtained [20] .\\n\\nIdentifying suspected cases which may have higher chances of getting diagnosed as a confirmed case and implementing strict procedures on them might offer the best solution.\\n\\ne challenge is to flag these patients by some demographic markers, as the clinical presentation of MERS-CoV infected patients were non-specific. erefore, we aimed to identify some demographic markers specific to confirmed cases of MERS-CoV. e nature of these markers has not been investigated in Saudi Arabia, and hence this study aims to identify them.\\n\\nA cross-sectional study was conducted at the regional laboratory and blood bank, located at Shumaisi Hospital in Riyadh, KSA. e laboratory has received the Central Blood Banks and Reference Laboratories Accreditation Program Saudi Central Board for Accreditation of Healthcare Institution (CBAHI) 2018 [21] .\\n\\nTechnique. Data were collected during the period of January 2017 to December 2017. All patients in Riyadh and Al-Qassim regions who had their samples tested at Riyadh regional lab during the study period were considered as suspected cases.\\n\\ne study had two aims: descriptive and analytical. For the descriptive aim, we estimated the prevalence of MERS-CoV. For the analytical aim, a binary logistic regression model was developed. In this model, we included the risk factors of gender, age, seasons, nationality, healthcare status (yes/no), hospitals, and area of residence. Data were cross-checked with a labcomputerized database. Further data were collected on demographic characteristics (age and sex), underlying nationality, and health care status.\\n\\nWe collected data from 25,400 cases, of which 23,646 suspected cases of MERS-CoV were included in the final analysis. Data were cleaned, entered, stored, and managed with an excel database and IBM SPSS Version 25. e statistical analyses consisted of descriptive counts and percentages. For those continuously scaled items, nonparametric statistics (medians, interquartile ranges, minimum, and maximum) were used to describe the distribution. A logistic regression analysis was used to identify predictors of confirmation of infection within the suspected cases groups. At first, univariate analyses were conducted to estimate the unadjusted contribution and to determine the significant risk factors. is was followed by a multivariate logistic regression analysis to estimate the independent contribution of each covariate. To determine significant factors, a p value below 0.05 and a 95% confidence interval were considered. \\n\\nA confirmed case is defined as a suspected case with laboratory confirmation of MERS-CoV infection [20] . \\n\\nA total of 23,646 of MERS-CoV suspected cases were included in this study, of which 52.3% were males (n � 12376) and 47.7% were females (n � 11270). e age of individuals with suspected cases ranged between 0 to 92 years with a mean age of 43. 23 e adjusted odds of MERS-CoV remained significant among different age groups; the odds of patients aged between 20-40 years increased threefold (A.OR: 3.11, 95% CI: 1.104-8.76, P value � 0.032), whereas in the age group of 41-60 years, it increased further to a risk that was six times higher \\n\\nis cross-sectional study about the epidemiological analysis of MERS-CoV infection laboratory-based data was conducted in Riyadh over a one-year period (2017). A total of 23,646 suspected cases were included in the results. Of the total suspected cases, 119 cases had been confirmed via laboratory results. All the confirmed cases are reported to MOH through HESN (health electronic surveillance networks) and to the World Health Organization (WHO) through the International Health Regulations (IHR), National Focal Point of Saudi Arabia. We found that MERS-CoV infection was found significantly in people aged between 41 and 60 years and was reported most commonly during the summer season. e odds of infection among males were found to be twice as high as that of females with suspected cases.\\n\\nDuring the study period, i.e., the year 2017, only 119 confirmed cases were reported, which means that the number of MERS-CoV infection cases has decreased in Riyadh and Al-Qassim regions in comparison to that of the last three years. From 2015 to 2016, there was a 25.4% decrease, whereas from 2016 to 2017, it decreased by 48.7%, which translates into a 50% decrease between the two periods. is also complements the findings reported by of Da'ar and Ahmed in their paper [23] . e predominance of infection in males was also observed in another study pwefromed in KSA (2015), which reported the percentage of confirmed cases among males to be 66%, compared with 34% among females [24] .\\n\\nIt is worth mentioning that Saudi Arabia defines age categories differently from the WHO (children: 0-14, adult: otherwise) [20] . However, unlike the classification used in Saudi Arabia, we have followed the WHO categorization of age to differentiate between children/adolescents (0 to 19 years) and adults (20 years and older) as indicated in WHO reports for age-standardized population and in infectious diseases [25] . is categorization was also followed by Aly and his collaborators in their recent paper published in 2017 [14] . Adults were further subcategorized into three groups according to the age distribution of the study population using the following two cutoff points (age of 41 and age of 60) [14] .\\n\\nese data agreed with a previous surveillance study, which stated that the majority of confirmed cases of MERS-CoV were reported among people aged 40 and above [24] . In 2016, only 9 of 552 cases (1.6%) of MERS-CoV infection were found among pediatric patients. Moreover, the study which was conducted in King Fahad Medical City in Riyadh (KFMC) between January 2012 and December 2013 did not report any MERS-CoV cases among children [26] . e study which was conducted across the Gulf countries for four years by Mahmoud Aly et al. between 2012 and 2016 suggests that the prevalence and distribution of MERS-CoV were the highest-risk in elderly aged 60 years or above [14] . Similar to our results, this study also reported the highest number of confirmed cases during the summer season [14] .\\n\\nAmong confirmed cases, only 25.2% were healthcare workers, whereas around 75% were non-healthcare workers.\\n\\nis is in agreement with the study done by Ahmad to estimate the survival rate in MERS-CoV globally prior to 26 January 2017; 86.9% were not health-care workers compared with 13.1% confirmed cases of healthcare workers [27] . Similarly, other studies also reported a lower prevalence in healthcare workers [28] [29] [30] .\\n\\nOur data reported a higher prevalence of infection among Saudi nationals as compared with non-Saudi. Another study also showed similar results but with a much higher percentage among Saudis, which may be due to the fact that it included Saudis from all regions [29] . ere is no finding basis for comparison as such, because our study was focused on the Riyadh and Al Qassim regions only.\\n\\nIn our study, we detected a low prevalence (0.5%). e low positive predictive value of our lab results is not related to the low sensitivity and specificity of the lab assay. e estimated analytical sensitivity and specificity of the Real Star kit from Altona was reported to be 100% with no cross reactivity with other respiratory pathogens [31] . Moreover, this low predictive value in the lab results is related to the high burden of false positive cases referred to the lab. In fact, this research is just the starting point to shed the light on more factors that might help in putting more descriptive criteria to lower the financial and human resources burden.\\n\\nTo the best of our knowledge, no one has developed a logistic regression that focuses on demographic risk factors such as sex, age, and seasons prior to our study. However, it is worth mentioning that Ahmed et al. developed a risk prediction model that encompasses risk factors such as chest pain, leukopenia, and elevated aspartate aminotransferase (AST) [21] . However, further investigations are needed to confirm our findings.\\n\\nOne of the major strengths of our study is that it is a comprehensive regional study which included all the suspected cases of MERS-CoV in the Riyadh and Al-Qassim regions. Secondly, the external validity of our study is also expected to be high, as it covers the two regions completely, meaning that the records of all suspected cases in these two main regions in Saudi Arabia were included. irdly, the quality of the data is considered to be high, given that the contagious and life-threatening nature of this disease has led to strict obedience to rules which are enforced in a timely manner, thus ensuring accurate reporting of suspected cases. In addition to this, quality assurance policies are implemented at HESN in order to maintain the highest level of validity and reliability of the data collection process. e variables available for suspected cases were limited to demographics, which limited the scope of our research, but they provided valuable information to form a basis for future studies of a broader scope. Variables such as primary/secondary infections are vital pieces of information, but due the limitation of the data available, we could not determine their effects.\\n\\nAccording to our knowledge, this is one of the few studies that have specifically investigated MERS-CoV risk factors in the Riyadh and Al-Qassim areas (two major regions in KSA). Given that all suspected and confirmed cases were included in this study, we assume that our results are generalizable for both the regions with confidence. It must be noted that the comparative group of this study is different from that of the previous ones, as we compared those with confirmed MERS-CoV with those with suspected MERS-CoV who have passed all stages of screening at the hospital, whereas other studies were hospital but not lab-based with an aim of identifying factors that help in suspecting rather than confirming cases. is might be the reason why we have found some significant demographic factors unlike other reports.\\n\\nIn conclusion, this research is about predictors for the confirmation of diagnosis among suspected cases only, meaning that the factors we found can help in identifying suspected cases that may have a higher chance of testing positive. is will help primary healthcare professionals to develop a better screening tool for suspected cases, as currently only a small minority of suspected cases are confirmed positive via lab results, consequently resulting in a lot of resources being spent to test thousands of samples, just for the identification of a few cases. e three factors we identified are important because, for example, a female, aged 18, presenting in winter will be less likely to be diagnosed than a male, aged 45, presenting in the summer, or, to give another example, a 60-year-old male who is presenting MERS-CoV signs with a negative lab result may need retesting.\\n\\nOur study covered two main regions in Saudi Arabia and provides evidence that the MERS-CoV epidemic in these two regions has specific characteristics that might help future plans for prevention and management of such contagious diseases. Our results showed that only a minority of suspected cases are actually diagnosed with the disease, meaning that the procedures being implemented seemed to be highly sensitive but not highly specific. e majority of confirmed cases were male, aged 41 to 60 years, and presented to healthcare facilities in the summer. Future studies should aim to confirm such findings in other regions in Saudi Arabia, to explore potential preventable risk factors and go deeper to know the underlying factors that make male aged 41-60 more susceptible than others.\\n\\ne laboratory data used to support the findings of this study were provided by Riyadh Regional Laboratory under license and are not freely available. However, access to data will be considered from the corresponding author upon request.\\n\\ne authors declare that they have no competing interests.\", 'document_id': 1551}]}, {'paragraphs': [{'qas': [{'question': 'Why are nucleosides analogs used for chemotheraphy?', 'id': 5232, 'answers': [{'text': 'they inhibit cellular DNA/RNA polymerases', 'answer_start': 1328}], 'is_impossible': False}, {'question': 'What nucleoside analog is the focus of the current study?', 'id': 5233, 'answers': [{'text': 'gemcitabine', 'answer_start': 6040}], 'is_impossible': False}, {'question': 'Gemcitabine has been shown to have antiviral activity against which viruses?', 'id': 5234, 'answers': [{'text': 'Middle East respiratory syndrome coronavirus (MERS-CoV), severe acute respiratory syndrome coronavirus (SARS-CoV), Zika virus (ZIKV), HCV, poliovirus (PV), influenza A virus (IAV), HIV, and enteroviruses (EV)', 'answer_start': 6863}], 'is_impossible': False}, {'question': 'How does gemcitabine disrupt viral activity?', 'id': 5235, 'answers': [{'text': 'by targeting the salvage pathway of pyrimidine biosynthesis', 'answer_start': 14050}], 'is_impossible': False}], 'context': 'Gemcitabine and Nucleos(t)ide Synthesis Inhibitors Are Broad-Spectrum Antiviral Drugs that Activate Innate Immunity\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC5923505/\\n\\nSHA: f1e1e2511e051195c8327a56d5c311a2dd4ab6b3\\n\\nAuthors: Shin, Hye Jin; Kim, Chonsaeng; Cho, Sungchan\\nDate: 2018-04-20\\nDOI: 10.3390/v10040211\\nLicense: cc-by\\n\\nAbstract: Nucleoside analogs have been frequently identified as antiviral agents. In recent years, gemcitabine, a cytidine analog in clinical use for the treatment of many solid tumors, was also shown to have antiviral activity against a broad range of viruses. Nucleoside analogs generally interfere with cellular nucleos(t)ide synthesis pathways, resulting in the depletion or imbalance of (d)NTP pools. Intriguingly, a few recent reports have shown that some nucleoside analogs, including gemcitabine, activated innate immunity, inducing the expression of interferon-stimulated genes, through nucleos(t)ide synthesis inhibition. The precise crosstalk between these two independent processes remains to be determined. Nonetheless, we summarize the current knowledge of nucleos(t)ide synthesis inhibition-related innate immunity and propose it as a newly emerging antiviral mechanism of nucleoside analogs.\\n\\nText: Nucleoside analogs have been historically used for anti-cancer chemotherapy because they inhibit cellular DNA/RNA polymerases [1] . More recently, nucleoside analogs have expanded their therapeutic applications and are being used to develop antiviral drugs against a wide range of serious and life-threatening viruses. Some nucleoside analog drugs targeting specific viral polymerases (acyclovir for herpesviruses, zidovudine for human immunodeficiency virus (HIV), and sofosbuvir for hepatitis C virus (HCV)) have been successful in clinical trials [2] [3] [4] [5] and are currently in use for the treatment of virus-infected patients. Another class of nucleoside analog drugs such as ribavirin, more broadly-acting on various viruses, has been used in conjunction with IFN-α [6] . Importantly, extensive studies on the antiviral action of ribavirin have established the underlying molecular framework of nucleoside analogs.\\n\\nThe primary mechanism to explain the antiviral effect of nucleoside analogs is based on their direct action on viral polymerization. Nucleoside analogs are transported into the cells and phosphorylated by the consecutive action of viral or cellular kinases, eventually generating nucleotide triphosphates. Mature nucleotide analogs, which are similar to physiological nucleotides, can directly incorporate into the growing viral genome during polymerization, resulting in the termination of chain reaction or the accumulation of mutations ( Figure 1 ). Alternatively, nucleotide analogs can bind to the nucleotide-binding region on viral polymerases and block the entry of incoming natural nucleotides. The other mechanism is based on the modulation of cellular nucleos(t)ide synthesis. There have been accumulating reports that nucleoside analogs act as antiviral agents by interfering with host nucleos(t)ide synthesis pathways [7] [8] [9] [10] . By targeting metabolic enzymes(s), nucleoside analogs block the natural flow of nucleos(t)ide synthesis and consequently cause the depletion or imbalance of (d)NTP pools. As viral replication is highly dependent on the availability of host nucleotides, a nucleotide-defective condition decreases the efficiency of viral replication. A more recently proposed mechanism has been based on the observations that a few nucleoside analogs activate innate immunity, especially involving the upregulation of interferon-stimulated genes (ISGs). Importantly, this phenomenon is usually mediated by the inhibition of nucleotide synthesis, suggesting a potential crosstalk between nucleotide biosynthesis and innate immunity. However, the precise mechanism of this crosstalk remains to be elucidated.\\n\\nThere is now an increasing number of nucleoside analogs with antiviral activity toward a wide range of viruses. They have been well-summarized in a previous report [1] . In the present review, we focus more on gemcitabine as a nucleoside analog, which is clinically relevant and whose broad-spectrum antiviral activity has been recently reported by many groups including our group. More importantly, we summarize inhibitors of the purine/pyrimidine biosynthesis pathways that induce innate immunity and propose possible mechanisms of action for these inhibitors. can directly incorporate into the growing viral genome during polymerization, resulting in the termination of chain reaction or the accumulation of mutations ( Figure 1 ). Alternatively, nucleotide analogs can bind to the nucleotide-binding region on viral polymerases and block the entry of incoming natural nucleotides. The other mechanism is based on the modulation of cellular nucleos(t)ide synthesis. There have been accumulating reports that nucleoside analogs act as antiviral agents by interfering with host nucleos(t)ide synthesis pathways [7] [8] [9] [10] . By targeting metabolic enzymes(s), nucleoside analogs block the natural flow of nucleos(t)ide synthesis and consequently cause the depletion or imbalance of (d)NTP pools. As viral replication is highly dependent on the availability of host nucleotides, a nucleotide-defective condition decreases the efficiency of viral replication. A more recently proposed mechanism has been based on the observations that a few nucleoside analogs activate innate immunity, especially involving the upregulation of interferonstimulated genes (ISGs). Importantly, this phenomenon is usually mediated by the inhibition of nucleotide synthesis, suggesting a potential crosstalk between nucleotide biosynthesis and innate immunity. However, the precise mechanism of this crosstalk remains to be elucidated.\\n\\nThere is now an increasing number of nucleoside analogs with antiviral activity toward a wide range of viruses. They have been well-summarized in a previous report [1] . In the present review, we focus more on gemcitabine as a nucleoside analog, which is clinically relevant and whose broadspectrum antiviral activity has been recently reported by many groups including our group. More importantly, we summarize inhibitors of the purine/pyrimidine biosynthesis pathways that induce innate immunity and propose possible mechanisms of action for these inhibitors. Figure 1 . The mechanism of antiviral effect of nucleos(t)ide analogs. Nucleos(t)ide synthesis inhibition-related innate immunity, a newly emerging antiviral mechanism of nucleoside analogs, was highlighted by yellow boxes.\\n\\nGemcitabine is a cytidine analog that has been clinically used for the treatment of various cancers [11, 12] . However, in recent years, the antiviral activity of gemcitabine has also been reported against a broad range of RNA viruses, including Middle East respiratory syndrome coronavirus (MERS-CoV), severe acute respiratory syndrome coronavirus (SARS-CoV), Zika virus (ZIKV), HCV, poliovirus (PV), influenza A virus (IAV), HIV, and enteroviruses (EV) [13] [14] [15] [16] [17] [18] .\\n\\nThe antiviral activities of gemcitabine against the abovementioned viruses are summarized in Table 1 . MERS-CoV and SARS-CoV belong to the family of Coronaviridae and are causative agents of severe viral respiratory illness in humans. To efficiently select appropriate antiviral drug Figure 1 . The mechanism of antiviral effect of nucleos(t)ide analogs. Nucleos(t)ide synthesis inhibition-related innate immunity, a newly emerging antiviral mechanism of nucleoside analogs, was highlighted by yellow boxes.\\n\\nGemcitabine is a cytidine analog that has been clinically used for the treatment of various cancers [11, 12] . However, in recent years, the antiviral activity of gemcitabine has also been reported against a broad range of RNA viruses, including Middle East respiratory syndrome coronavirus (MERS-CoV), severe acute respiratory syndrome coronavirus (SARS-CoV), Zika virus (ZIKV), HCV, poliovirus (PV), influenza A virus (IAV), HIV, and enteroviruses (EV) [13] [14] [15] [16] [17] [18] .\\n\\nThe antiviral activities of gemcitabine against the abovementioned viruses are summarized in Table 1 . MERS-CoV and SARS-CoV belong to the family of Coronaviridae and are causative agents of severe viral respiratory illness in humans. To efficiently select appropriate antiviral drug candidates, Dyall et al. screened 290 FDA-approved drugs in virus-infected Vero E6 cells and identified gemcitabine as one of drugs with antiviral activity against both MERS-CoV and SARS-CoV (EC 50 of 1.2 µM and 4.9 µM, respectively) [13] . More recently, gemcitabine was shown to effectively suppress ZIKV infection and replication in human retinal pigment epithelium (RPE) cells, particularly at non-cytotoxic concentrations (EC 50 of 0.01 µM vs. CC 50 of > 10 µM) [14] . ZIKV, a member of the Flaviviridae family, can infect pregnant women and cause congenital abnormalities such as microcephaly in infants, which has attracted increasing public attention as well as extensive research and development into possible treatments. Effective antiviral activities of gemcitabine were also found for the replication of HCV in Huh-7 cells and the infection of HIV in U373-MAGI-CXCR4 CEM cells, with estimated EC 50 s of 12 nM and 16.3 nM, respectively [17, 19] , which were lower concentrations than those used in cancer therapy [20] . In the case of HIV, the combination of gemcitabine with decitabine, another nucleoside analog in clinical use for cancer therapy, synergistically reduced HIV infectivity by increasing the viral mutation frequency [21] . In a follow up study, Clouser et al. further reported the antiviral effect of gemcitabine against HIV-related retrovirus, murine leukemia virus (MuLV), in vitro (EC 50 of 1.6 nM) and even in murine AIDS model [17] . A significant antiviral effect of gemcitabine on IAVs was also reported for RPE cells by Denisova et al. (EC 50 of 0.068 µM) [16] . They also tested whether gemcitabine had an antiviral effect on several other viruses of different families and found its strong inhibitory effect on Sindbis virus and herpes simplex virus-1 (HSV-1) (>2 log reduction in virus titer) but relatively weak effects on Semliki forest virus and human echovirus 6, and minimal effects on Bunyamwera virus, measles virus (MeV), and vaccinia virus [16] . The antiviral effect of gemcitabine on EVs, initially performed on Coxsackievirus B3 (CVB3), was found from screening FDA-approved drugs in CVB3 replicon-harboring Vero cells by our group (EC 50 of 0.4 µM) [18] . Its broad-spectrum antiviral activity on EVs was further identified by observing a similar inhibitory effect on enterovirus 71 (EV71) and human rhinoviruses (HRVs) (EC 50 s of 1 and 1-5 µM, respectively). In the case of HRV, the antiviral effect of gemcitabine was further confirmed in a virus-infected mouse model [22] . In this study, intranasal administration of gemcitabine significantly lowered the pulmonary viral load and inflammation by decreasing proinflammatory cytokines, including TNF-α and IL-1β, and the number of lung infiltrating lymphocytes. More recently, Zhang et al. also identified gemcitabine as the best anti-PV inhibitor from a screen of FDA-approved drugs in PV replicon-harboring HeLa cells (EC 50 of 0.3 µM) [15] . As previously mentioned, accumulating evidence has definitively demonstrated that gemcitabine is an effective broad-spectrum inhibitor of RNA viruses and has a therapeutic potential for the treatment of various virus-associated diseases. Moreover, it is possible that gemcitabine is effective for other untested RNA viruses. Because gemcitabine is a deoxycytidine analog that interferes with DNA as well as RNA synthesis, DNA viruses may not be the exception. Consistent with this possibility, there has been a report that the infection of HSV-1, which is a representative DNA virus classified into the Herpesviridae family, was strongly affected by gemcitabine [16] . Most of the abovementioned viruses have, at best, limited prophylactic or therapeutic drugs as possible treatments. This is especially true for newly emerging or re-emerged viruses involving serious illnesses, such as MERS-CoV, SARS-CoV, and ZIKV, which are major threats to public health and which urgently need an effective treatment during their early stages of infection. In this regard, repurposing of gemcitabine for the treatment of patients infected with these deadly viruses is a realistic approach. Importantly, it is noteworthy that ZIKV was the most strongly affected by gemcitabine, with a low nanomolar EC 50 , which was lower than that used in cancer therapy [14, 20] . Even for other viruses with a relatively high EC 50 , there is an option to treat patients with a combination of gemcitabine with other antiviral agents. In this manner, an effective antiviral treatment may be achieved by the synergistic action of two antivirals with much lower doses for each drug, which minimizes deleterious side effects when used clinically. As an example, the synergistic antiviral effect of gemcitabine in combination with ribavirin, an antiviral drug currently being used against a few RNA viruses, was reported against EVs such as CVB3 and EV71 [18] . As previously mentioned, the combination of gemcitabine with decitabine synergistically suppressed HIV infectivity both in vitro and in vivo [17, 21] . However, the actual use of gemcitabine in virus-infected patients necessitates prior in vivo animal studies and clinical trials. Even though most antiviral data have originated from in vitro studies, two recent studies have reported the antiviral effects of gemcitabine in murine models [17, 22] . More extensive analyses of gemcitabine in animal models in the near future will accelerate its therapeutic applications in clinical trials.\\n\\nMost studies regarding the antiviral activity of gemcitabine lack experimental evidence of the mode of action. However, our group has recently reported that gemcitabine had an anti-EV effect by targeting the salvage pathway of pyrimidine biosynthesis [23] . Moreover, gemcitabine strongly induced the expression of several ISGs including CXCL10, IRF7, IRF9, IFIT1, and DDX58, which were the major effectors in the innate immunity that defended the host against the virus infection. These results were consistent with a previous report that gemcitabine stimulated the production of IFN-β and IFN-γ in IAV-infected RPE cells [16] . Importantly, the activation of ISGs was well-correlated with the inhibition of pyrimidine biosynthesis, suggesting a link between pyrimidine biosynthesis and innate immunity. Similar phenomena in terms of ISG activation have been previously reported with a few compounds out of several purine or pyrimidine biosynthesis inhibitors that had antiviral activity, as summarized in Table 2 [6, 10, [23] [24] [25] [26] [27] [28] [29] [30] [31] [32] [33] [34] [35] [36] [37] [38] [39] [40] [41] [42] . Regarding purine biosynthesis inhibitors, ribavirin and mycophenolic acid (MPA) are inhibitors of inosine-5 -monophosphate (IMP) dehydrogenase (IMPDH), which is a key enzyme of the purine biosynthesis pathway. These inhibitors have been successfully used as clinical antiviral or immunosuppressant agents for decades. Both have antiviral activities against viruses such as HCV, hepatitis E virus (HEV), MERS-CoV, dengue virus, yellow fever, hepatitis B virus, West Nile virus (WNV), Chikungunya virus (CHIKV), and IAV [24] [25] [26] [27] [28] [29] [30] , majorly through the inhibition of the purine biosynthesis pathway, with the antiviral activity against HCV and HEV shown to involve the stimulation of ISGs [10, 30] . For the antiviral activity of ribavirin against HCV, ribavirin specifically induced the expression of IRF7, IRF9, and ISG15 mRNAs, which are known to be important for anti-HCV immune responses [6] . ISG activation occurred through an undefined mechanism that was different from the classical IFN signaling, intracellular dsRNA sensing pathway, Toll-like receptor and nuclear factor B pathways. More importantly, ribavirin-induced ISG activation and antiviral activity were suppressed using supplemented guanosine, a natural analog of ribavirin, suggesting IMPDH inhibition-mediated ISG activation as an alternative innate immunity pathway. Like ribavirin, MPA remarkably induced the expression of several ISGs, including IRF1, IRF9, ISG15, IFI6, IRF7, CXCL10, IFIT2, and IFITM3 mRNAs in naïve or HEV-infected Huh-7 cells, and the induction of ISGs was at least partially abrogated by the use of supplemented guanosine [10] . Mechanistically, the induction of ISGs by MPA was independent of the classical JAK/STAT system, which is similar to that observed with ribavirin [30] . Similar results were obtained with several IMPDH1 or IMPDH2 inhibitors, with various affinities, that were custom-designed and synthesized [10] .\\n\\nAs shown in Table 2 , most pyrimidine biosynthesis inhibitors target dihydroorotate dehydrogenase (DHODH), an essential enzyme in de novo pyrimidine synthesis. Lucas-Hourani et al. identified DD264 as an interferon-sensitive response element (ISRE)-stimulating compound from high-throughput screening, and further analyses suggested that it was a DHODH inhibitor with a strong antiviral activity against various viruses including MeV, CHIKV, and WNV [37] . DD264 enhanced the expression of several ISGs, which were almost completely suppressed by the addition of supplemented uridine, indicating DHODH inhibition-mediated ISG activation. Moreover, the antiviral activity of and ISG activation by DD264 required the interferon regulatory factor 1 (IRF1) transcription factor, a master regulator of antiviral gene expression [37] , which was consistent with the observation that the anti-HCV activity of MPA was partially mediated by IRF1 [30] . In this study, similar results were shown with brequinar, another well-known DHODH inhibitor. FA-613 is also an antiviral compound, which inhibits the pyrimidine biosynthesis pathway, probably via targeting DHODH and inducing the expression of ISGs such as IFNB1, CXCL10, ISG15, and CCL5 [38] . However, whether ISG activation is mediated by pyrimidine biosynthesis inhibition remains to be determined.\\n\\nThe mechanism of nucleotide synthesis inhibitor-induced ISG activation is still presently unclear. Nevertheless, there has been accumulating evidence showing that nucleotide synthesis inhibitor-induced ISG activation is independent of the classical JAK/STAT-mediated IFN signal [6, 10, 23] . First, Wang et al. clearly showed that ISG activation and anti-HEV activity induced by MPA or brequinar was not mediated by JAK [10] . Second, IRF7 induction by ribavirin was not affected by knockdown of STAT1, while that of IFN-α was strongly affected under the same conditions [6] . Third, our recent study with gemcitabine further confirmed IFN signal-independent ISG activation by parallel studies comparing the effects of gemcitabine and IFN-α. In our study, the phosphorylation of STAT1 at Tyr701, which was dramatically triggered by IFN-α, did not occur when treated with gemcitabine [23] . Moreover, the upregulation of DDX58 mRNAs induced by gemcitabine was not affected by IRF9 knockdown, which was contrary to the result that IFN-α-induced upregulation of DDX58 mRNAs was significantly suppressed under the same conditions. Consistent with above observations, there have been some reports that ISGs was induced in the absence of JAK1 or STAT1 activation [43, 44] .\\n\\nDespite limited data, we speculate the scenario of ISG activation that is independent of JAK/STAT-mediated IFN signal. Purine or pyrimidine biosynthesis inhibitors could interfere with the metabolic pathway through targeting some key enzymes such as IMPDH and DHODH, leading to the depletion or imbalance of the (d)NTP pool. Inactivation of metabolic enzyme(s) itself or consequently altered nucleos(t)ide pools might trigger a signal, which is ultimately delivered to certain cis-acting elements on the promoter of a subset of ISGs, possibly through the relay of kinases and transcription factors. Based on the previously mentioned reports, this signal is less likely to be dependent on STAT1/2-IRF9 (IFN-stimulated gene factor 3; ISGF3), at least for gemcitabine, which is the major transcriptional complex in the IFN-induced JAK/STAT pathway. It should also be considered that Thomas et al. excluded the involvement of an intracellular double-stranded RNA sensing pathway, Toll-like receptor and nuclear factor κB pathways, as well as a classical IFN signal in the activation of ISGs induced by ribavirin [6] . Despite the consensus of ISG activation, each purine/pyrimidine biosynthesis inhibitor seems to induce distinct sets of ISGs, at least with different patterns [10] . Targeting an enzyme in which pathways (purine or pyrimidine synthesis) or steps (early/late and de novo/salvage) produce different levels of intermediates and nucleos(t)ides will consequently result in diverse outcomes of ISG activations. There might be more than one signaling pathway involved. The synergistic antiviral activity of gemcitabine and ribavirin observed in our study might be explained by the possible existence of two separate signaling pathways that mediate each inhibition of nucleotide synthesis toward ISG activation. Systematic analyses of signaling kinases, IRFs, and STATs using siRNA knockdown and/or pharmacological inhibition and metabolic analyses of corresponding intermediates and nucleos(t)ides should therefore clarify the underlying molecular mechanisms of ISG activation by purine/pyrimidine biosynthesis inhibitors.\\n\\nAs newly emerging or re-emerged viruses such as SARS-CoV, MERS-CoV, and ZIKV have become a major threat to public health, the need for broad-spectrum antiviral drug has increased. In this regard, nucleoside analogs that directly target viral RNA-dependent RNA polymerase and present a high barrier to the development of resistant viruses have been considered advantageous. Moreover, recent discovery of a new antiviral mode of nucleoside analogs acting through innate immunity strengthens the molecular basis for their therapeutic application as broad-spectrum antiviral drugs.\\n\\nNucleoside analogs probably induce different subsets of ISGs, at least with a different pattern, leading to various combinations of ISGs and resulting antiviral outcomes. Moreover, according to Schoggins et al., different viruses are affected by distinct subsets of ISGs and some ISGs such as IRF1, MB21D1, HPSE, DDX58, MDA, and IFITM3 act broadly on various viruses [45] . Thus, more systematic analyses on the subsets of ISGs induced by antiviral nucleoside analogs are required for the identification of better antiviral drugs that can be used broadly or specifically. Given the clinical side effects of IFN treatment, nucleotide analogs that differ from IFN in the activation of subsets of ISGs need to be considered as alternatives. Nevertheless, nucleoside analogs interfering with the host nucleotide synthesis pathway suggest possible side effects in their clinical applications. Careful evaluation of clinical safety is required and their application for the urgent measure of patients infected with deadly viruses would be worth being primarily considered.', 'document_id': 1594}]}, {'paragraphs': [{'qas': [{'question': 'Why are cotton rats considered a strong animal model for biomedical research?', 'id': 1631, 'answers': [{'text': 'its well defined immunological and genetic information, costeffectiveness, and abundant inbred strains and research reagents', 'answer_start': 2066}], 'is_impossible': False}, {'question': 'What is the structure of the CD40 Ligand?', 'id': 1632, 'answers': [{'text': 'a sandwiched extracellular structure composed of a β-sheet, α-helix loop, and a β-sheet', 'answer_start': 4149}], 'is_impossible': False}, {'question': 'What is the effect of CD40L on Dendritic Cells?', 'id': 1633, 'answers': [{'text': 'it promotes cytokine production, the induction of cell surface co-stimulatory molecules, and facilitates the cross-presentation of antigen by these cells', 'answer_start': 4774}], 'is_impossible': False}, {'question': 'What is the effect of CD40L on B Cells?', 'id': 1634, 'answers': [{'text': 'it promotes germinal center formation, immunoglobulin (Ig) isotype switching, somatic hypermutation to enhance antigen affinity, and lastly, the formation of long-lived plasma cells and memory B cells', 'answer_start': 5079}], 'is_impossible': False}], 'context': \"Identification and characterisation of the CD40-ligand of Sigmodon hispidus\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC6063397/\\n\\nSHA: edf2997357501734a93c1b7e16d44e86a7d20853\\n\\nAuthors: Russell, Marsha S.; Muralidharan, Abenaya; Larocque, Louise; Cao, Jingxin; Deschambault, Yvon; Varga, Jessie; Thulasi Raman, Sathya N.; Li, Xuguang\\nDate: 2018-07-27\\nDOI: 10.1371/journal.pone.0199067\\nLicense: cc-by\\n\\nAbstract: Cotton rats are an important animal model to study infectious diseases. They have demonstrated higher susceptibility to a wider variety of human pathogens than other rodents and are also the animal model of choice for pre-clinical evaluations of some vaccine candidates. However, the genome of cotton rats remains to be fully sequenced, with much fewer genes cloned and characterised compared to other rodent species. Here we report the cloning and characterization of CD40 ligand, whose human and murine counterparts are known to be expressed on a range of cell types including activated T cells and B cells, dendritic cells, granulocytes, macrophages and platelets and exerts a broad array of immune responses. The cDNA for cotton rat CD40L we isolated is comprised of 1104 nucleotides with an open reading frame (ORF) of 783bp coding for a 260 amino acid protein. The recombinant cotton rat CD40L protein was recognized by an antibody against mouse CD40L. Moreover, it demonstrated functional activities on immature bone marrow dendritic cells by upregulating surface maturation markers (CD40, CD54, CD80, and CD86), and increasing IL-6 gene and protein expression. The availability of CD40L gene identity could greatly facilitate mechanistic research on pathogen-induced-immunopathogenesis and vaccine-elicited immune responses.\\n\\nText: The cotton rat (Sigmodon hispidus) was first used in polio research in the 1930s [1] , and throughout the last century, it has proven to be an excellent model for biomedical research [2, 3, 4] . Historically in biomedical research, the mouse has been exploited as the default animal model. This is in part due to its well defined immunological and genetic information, costeffectiveness, and abundant inbred strains and research reagents. However, the use of mice as models to study infectious diseases has its limitation since mice are not naturally infected by most human pathogens. On the other hand, cotton rat is susceptible to many human pathogens and is the ideal model of choice for measles (paramyxovirus) [5] , herpes simplex (oral a1111111111 a1111111111 a1111111111 a1111111111 a1111111111 and ophthalmic) [6] , influenza (orthomyxovirus) [7, 8] , HIV-1 [9] , RSV (respiratory syncytial virus) [10] , adenovirus [11, 12] , human parainfluenza [13] , and human metapneumovirus [14] . This model has been valuable for adenovirus-based gene replacement therapy research [15, 16] , and was also proven to be indispensable in pre-clinical evaluation of the prophylactic antibodies (RespiGam 1 [17] , and Synagis 1 [18] . Indeed, the cotton rat model was found to be valuable in terms of its biological and immunological relevance, it was deemed unnecessary to test the adenovirus-based gene therapy and the Synagis 1 prophylactic treatment against RSV disease in non-human primate prior to the human trials [19, 20] .\\n\\nA number of methods and reagents have been developed for the analysis of immune responses in cotton rats over the last decade. Up to date, more than 200 genes encoding cytokines, chemokines, cell surface markers and regulatory molecules have been cloned, with various related research reagents being commercially available. As a result, the use of cotton rats in pathogenesis studies addressing mechanistic questions has significantly increased. Nevertheless, the gene encoding CD154 and CD40 ligand (CD40L), remains elusive.\\n\\nCD40L plays a critical role in orchestrating immune responses against pathogens. Depending on the post-translational modification, the murine CD40L is a 32-39 kDa type II membrane glycoprotein that was initially identified as a surface marker exclusive to activated CD4 + T cells [21, 22] . It is a member of the TNF superfamily consisting of a sandwiched extracellular structure composed of a β-sheet, α-helix loop, and a β-sheet, allowing for the trimerization of CD40L, an additional feature of the TNF family of ligands [23] . Since its initial discovery, CD40L has been shown to be not only expressed on CD4+ T cells, but on dendritic cells (DCs) [24] , B cells [25] , and platelets [26] .\\n\\nIt has been shown that upon interacting with its receptor, CD40, CD40L induces profound effects on T cells, DCs, B cells, endothelial cells, as well as many cells of the hematopoietic and non-hematopoietic systems. Moreover, when CD40L engages CD40 on the surface of DCs, it promotes cytokine production, the induction of cell surface co-stimulatory molecules, and facilitates the cross-presentation of antigen by these cells [27] , enabling DCs to mature and effectively induce the activation and differentiation of T cells. When CD40L engages CD40 on the surface of B cells, it promotes germinal center formation, immunoglobulin (Ig) isotype switching, somatic hypermutation to enhance antigen affinity, and lastly, the formation of long-lived plasma cells and memory B cells [28] .Various studies have been conducted to utilize gene delivery of CD40L to DCs and tumor cells for tumor immunotherapy. It was found that expression of CD40L in a small proportion of tumor cells was sufficient to generate a long-lasting systemic anti-tumor immune response in mice that was shown to be dependent on cytotoxic T lymphocytes [29, 30] .\\n\\nHere we report the successful cloning of the gene encoding cotton rat CD40L (crCD40L); we also expressed and purified the CD40L produced in mammalian cells. Further characterisation of the recombinant cotton rat CD40L revealed its functional activities in promoting DC maturation and cytokine production. [6] [7] weeks old cotton rats were obtained from an inbred colony maintained at Envigo (USA). All animal experiments were conducted in accordance with Institutional Care and Use Committee (IACUC) of Health Canada Ottawa Animal Care Committee which approved this study. The rats were housed 3 animals per cage in Allentown NexGen individually ventilated cages with free access to food and water. These cages provided a floor space of 142 in 2 / 916 cm 2 . Body weight and any sign of distress were monitored daily. If anything associated the animal health was observed, a full examination would be conducted. As In this study spleen cells from normal, healthy animals were isolated, we did not observe any adverse reaction. To isolate splenocytes from the animals, isoflourane was used to put the animals to sleep via inhalation with oxygen for euthanasia.\\n\\nThe spleens from three naïve cotton rats were removed aseptically and snap frozen in liquid nitrogen. The spleens were homogenized individually with a TissueLyser II (Qiagen) and total RNA extracted using the RNeasy Mini kit (Qiagen) with on-column DNase digestion according to the user's manual. The 3' RACE system (Life Technologies) was then used with to amplify the 3' portion of the cotton rat CD40L from the total RNA according to the manufacturer's instructions. A schematic of the 3' RACE procedure used is provided in S1 Fig. A gene specific primer (5'-GGACTCTATTATGTCTACACCCAAGTCACCTTCTG -3') was derived from a consensus sequence aligning the rat (Rattus norvegicus UniProt: Q9Z2V2), mouse (Mus musculus UniProt: P27548), and golden hamster (Mesocricetus auratus NCBI Reference Sequence: XM_005084522.3) CD40L sequences obtained from the National Center for Biotechnology Information (NCBI). Following first strand cDNA synthesis, the 3' portion of the cotton rat CD40L mRNA was PCR amplified using the consensus sequence derived gene specific primer and the abridged universal amplification primer with an annealing temperature at 56˚C. The reverse complementary sequence of this primer was then used as a reverse primer with the forward primer (5'-GATAGAAACATACAGCCAACCTTCTCCCAGATC -3') to amplify the 5' portion of the cotton rat CD40L mRNA with an annealing temperature of 57˚C.\\n\\nAll amplified fragments were sequenced with BigDye Terminator v.3.1 Cycle Sequencing kit (ThermoFisher cat # 4336917). Briefly, samples were amplified in a PTC-200 thermal cycle (MJ Research) with the following program: 26 cycles of 1˚C/S to 96˚C, 96˚C for 10 seconds, 1˚C/S to 50˚C, 50˚C for 5 seconds, 1˚C/S to 60˚C, 60˚C for 4 minutes. The samples were cleaned using DyeEx 2.0 Spin kit (Qiagen cat # 63204) and loaded onto a 3130xl Genetic Analyzer (Applied Biosystems). Raw sequencing data was edited by the instrument's software (ThermoFisher 3130xl Genetic Analyzer Data Collection Software v3.0), and then imported into GeneCodes Sequencher v4.6.1 sequencing analysis software for further editing. The final sequenced contigs are then imported to NCBI BLAST (https://blast.ncbi.nlm.nih.gov/Blast. cgi) to confirm the identity.\\n\\nPutative conserved domains, trimer interface, and receptor binding sites were determined by performing a standard protein BLAST (blastp algorithm; https://blast.ncbi.nlm.nih.gov). The sequences producing significant alignments were imported into Geneosis software, (Auckland, New Zealand). Multiple alignment was conducted as previously described [31] , with phylogenetic analysis using Geneosis Pro 5.6.7.\\n\\nOnce the mRNA sequence was confirmed, a construct was designed beginning with a kozak sequence (5'-CACCGCCGCCACC-3'), followed by a secretion signal consisting of 23 amino acid (aa) (MLLAVLYCLLWSFQTSAGHFPRA) from the human tyrosinase signal peptide as previously described [32] . This is followed by six histidine residues to facilitate protein purification. Following this sequence, a 27-aa fragment from the bacteriophage T4 fibritin trimerization motif was added [33] and finally connected to the full length 783bp open reading frame (ORF) of the cotton rat CD40L sequence at the C terminus. This construct was synthesized and cloned into pUC57 (Biobasic, Markham, ON).\\n\\nGeneration of a recombinant vaccinia virus expressing cotton rat CD40L protein construct was achieved using a vaccinia virus E3L and K3L double deletion mutant virus as the parental virus and taterapoxvirus K3L as the positive selection marker (Jingxin Cao, unpublished information). Briefly, the recombination plasmid vector for expression of the CD40L construct gene consists of the homologous flanking vaccinia DNA sequences targeting vaccinia A45R gene (SOD homolog); the CD40L construct gene driven by a modified vaccinia H5 promoter (Vaccine 1996, 14:1451), and taterapoxvirus 037 gene driven by vaccinia K3L promoter as the positive selection marker. The recombination vector was transfected into a HeLa PKR knockout cells infected with a vaccinia virus with both E3L and K3L genes deleted. Selection and purification of the recombinant vaccinia virus expressing the CD40L was done in BHK21 cells.\\n\\nExpression of the CD40L protein was confirmed by Western blotting using His-tag Ab. Cell monolayers were lysed in sample buffer and homogenized using QIAshredder columns (Qiagen). Western blotting was performed using 4 to 15% TGX gel and Tris/Glycine/SDS running buffer (Bio-Rad Laboratories Inc.), and the protein samples were transferred to Immobilon-FL PVDF membranes (Millipore). Protein was detected with Tetra-HIS Ab (Qiagen) and goat anti-mouse IRDye-800CW (LiCor). Membranes were developed using the Odyssey system (LiCor).\\n\\nThe vaccinia virus carrying the crCD40L gene was propagated in BHK21 cells. The cells were collected and washed with PBS once and then lysed with a denaturing buffer (10 mM Tris-HCl, 100 mM sodium phosphate, 6 M guanidine hydrochloride, 10 mM reduced glutathione, pH 8.0) and disrupted by sonication on ice using a Branson sonifier 150 (ThermoFisher, Waltham, MA) at level 1 for two 10sec bursts with 1min rest on ice between. After separation of cell debris, the supernatant was added to a slurry of Ni-NTA resin (Qiagen, Mississauga, ON, Canada) (10 mL resin bed) and stirred at room temperature for 30 min before loading into a column. The column was purified using an AKTA purifier (Amersham Biosciences) with Unicorn 5.3 software (Amersham Biosciences). Refolding was accomplished under oxidative conditions with a gradient of denaturing buffer to buffer B (buffer B: 10 mM Tris-HCl, 100 mM sodium phosphate, pH 7.8) over 10 column volumes (CVs). The column was then washed with three CVs of buffer B + 60 mM imidazole (pH 7.8) to remove unspecific binding. The protein was eluted off the column with buffer B + 250 mM imidazole (pH 7.8). The resulting protein was dialysed against PBS pH 7.5 and then confirmed by western blot.\\n\\n96-well plates were coated with either recombinant mouse CD40L (R&D Systems) or the recombinant crCD40L protein 2ug/ml in 100μl PBS. Plates were washed with wash buffer (PBS-0.1% tween-20) and then blocked with 200μl/well blocking buffer (PBS containing 0.1% Tween 20 and 3%IgG Free BSA) for 1 hour at 37˚C. Plates were washed with wash buffer and incubated at 37˚C for 1 hour with 100μl/well goat anti-mouseCD40L (R&D Systems) 2ug/ml in blocking buffer. Plates were subsequently washed and incubated at 37˚C for 1 hour with 100μl/well with rabbit anti-goat IgG HRP conjugate (Zymed). Plates were washed again and incubated for 10 min in the dark with 100μl/well 3,3'5,5'-tetramethylbenzidine substrate (New England Bio Labs). The reaction was stopped with Stop solution (New England Bio Labs) and absorbance was read at 450nm on a BioTek Synergy 2 plate reader.\\n\\nPrimary bone marrow cells from Balb/c mice (Chicago, IL) were thawed and cultured in dendritic cell medium from manufacture (Cell Biologics M7711) supplemented with GMCSF (Cell Biologics) without IL-4 at 4x10 5 cells/well in a volume of 200μl. The cells were treated with 0.5μg/ml recombinant mouse CD40L (Preprotech, Montreal, QC) or the recombinant crCD40L protein at 0.5μg/ml, 5μg/ml, or 50μg/ml. Forty hours later, flow cytometry was performed on a BD LSRFortessa cell analyser after 2 x 10 5 cells/tube were stained using CD11c-PE-CF594, CD54-FITC, CD40-BV786, CD80-BV421, and CD86-BV711 antibodies. All antibodies were purchased from BD Biosciences. The resulting spectra were analysed using FACS-Diva version 8.0.1 software.\\n\\nTo assess IL-6 mRNA production of immature bone marrow murine DCs in response to targeting by recombinant crCD40L, quantitative real-time PCR was conducted on an ABI Prism 7500 Fast Sequence detection system (Applied Biosystems). TaqMan assay reagent kits (Applied Biosystems) were used that contain pre-standardized primers and TaqMan MGB probes for IL-6 and 18S which were used to normalize the data. Total RNA was isolated from 8x10 5 stimulated bone marrow DCs using the RNeasy Mini Kit (Qiagen) according to manufactures instructions. The isolated RNA was used to make cDNA using the Superscript III First-Strand Synthesis System for RT-PCR (Invitrogen) according to manufacturer's instructions. The cDNA was then subjected to quantitative PCR using the TaqMan Fast Advanced Master Mix (Applied Biosystems) according to manufactures instructions. Samples were run in duplicate and C t values were obtained. Fold change over unstimulated DCs was calculated using the 2 -ΔΔCT method of relative quantification [34] , using 18S as the housekeeping reference gene. To investigate IL-6 secretion by murine bone marrow DCs, supernatant from forty hour stimulated cultures were collected and assayed using the Mouse IL-6 DuoSet ELISA Kit (R & D Systems) following the manufacturer's protocol.\\n\\nThe complete mRNA sequence of CD40L was obtained in two steps (Fig 1) . A sequence corresponding to nucleotides 535 through to the poly-A tail was obtained using the 3' RACE kit and mRNA as starting material, which was isolated from cotton rat splenocytes and a rodent consensus sequence as a primer. This portion of the sequence has the 3' un-translated region of the mRNA as well as the stop codon. The 5' end of the protein was obtained in the next step by PCR amplification of the cDNA obtained in the first step with the 3' RACE kit and the reverse complement of the consensus sequence primer and a second consensus sequence primer designed to bind to the beginning of the CD40L mRNA. The 783bp ORF encodes 260aa followed by a stop codon.\\n\\nComparison of the sequenced CD40L gene revealed that the crCD40L coding sequence shares 93%, 89%, and 83%, identity with golden hamster, rat, and mouse, respectively. At the amino acid (aa) level, the corresponding identities are 91%, 82%, and 82%, Fig 2a. At both the mRNA and aa levels, the crCD40L shared the closest similarity with Peromyscus maniculatus bairdii (or deer mouse) at 93% and 92% respectively. When sequence homology analysis is performed, crCD40L clusters with other members of the Cricetidae family Fig 2b. We next examined the functional domains in crCD40L in comparison with other known CD40L. As shown in Fig 3a, crCD40L has a putative tumor necrosis factor (TNF) superfamily Using EZmol software [35] , we predicted folding of the protein as shown in Fig 3b. The cotton rat CD40L cDNA that we have isolated was a 1104 nucleotide sequence with a poly-A tail containing an ORF of 783bp which coded for a 260 aa protein. The homology of cotton rat CD40L, at both the amino acid and nucleic acid level, is closer to members of the Cricetidae family (hamster and deer mouse) than to those of the Muridae family (rat and mouse) as shown in Fig 2b. As with other known CD40L proteins, there is a putative TNF superfamily domain, a transmembrane domain, trimerization sites, and receptor binding sites [36] .\\n\\nTNF superfamily members include TNF (TNF-alpha), LT (lymphotoxin-alpha, TNF-beta), CD40 ligand, Apo2L (TRAIL), Fas ligand, and osteoprotegerin (OPG) ligand, among others [37] . The TNF superfamily is composed of 19 ligands and 29 receptors, in which each has vastly diversified roles in the body and exhibit pro-inflammatory activity, partly via activation of NF-kB [37] . Members of this family generally have an intracellular N-terminal domain, a short transmembrane segment, an extracellular stalk, and a globular TNF-like extracellular domain of about 150 residues [23] . They initiate apoptosis by binding to related receptors, some of which have intracellular death domains [38] . These proteins typically form homo-or hetero-trimeric complexes and bind one elongated receptor molecule along each of three clefts formed by neighboring monomers of the trimer and ligand trimerization is for receptor binding [23, 39] . All seven known conserved residues that constitute the trimer interface on the conserved TNF domain [23, 40] , were mapped to the putative crCD40L protein sequence. Additionally, all six known conserved receptor binding sites on the conserved TNF domain [23, 40] , were mapped to the crCD40L protein sequence.\\n\\nIn order to further evaluate the crCD40L deduced sequence, the full 783bp ORF of the crCD40L was cloned into a vaccinia virus vector. The crCD40L construct was designed to carry a secretion signal, histidine tag, and a trimerization motif (Fig 4a) . Selection and purification of the recombinant vaccinia virus expressing the CD40L construct was conducted in BHK21 cells. Western blot with anti-histidine antibody (Ab) was used to confirm expression of the CD40L protein construct Fig 4b and S2 Fig. The resulting 36 kDa protein product was found in both the cell lysate and supernatant (faint band-48 hours only). Since the highest expression was found in the cell lysate, it was used for further purification of the protein. It should be noted that the protein was only able to be detected under reducing conditions. Under non-reducing conditions, the protein was unable to be detected by the anti-histidine Ab, even in the cell lysate (data not shown). This indicates that the histidine tag is folded within the trimer and is unavailable in the native form for purification. This is an additional reason for the need to purify the protein from the cell lysate under harsh denaturing conditions followed by protein refolding. The reason we utilized a mammalian expression system to produce the protein rather than a bacterial system is to facilitate its proper folding into its native structure, trimerization, and glycosylation. The aa backbone predicts a protein of 29 kDa, yet initial studies of the CD40L protein suggested a molecular mass of 39 kDa, and on most cell types the molecular mass of CD40L is 32-33kDa, consistent with extensive post-translation modification [36] .\\n\\nThe BHK21 cells expressing the crCD40L construct were collected and lysed with 6 M guanidine hydrochloride with reduced glutathione and sonication. The lysate was loaded on the nickel column and the washed with denaturing buffer as described in materials and methods. The bound proteins were refolded on the column with gradient buffer exchange, to allow slow refold the protein, given that CD40L biological activity is dependent on a homo-trimer configuration [23] . The resulting bound protein was subsequently eluted with imidazole. The resulting fractions that showed a peak were pooled and dialysed against PBS.\\n\\nThe purified protein was confirmed in ELISA. Since the cotton rat CD40L protein sequence shared 82% identity with the mouse CD40L protein sequence, an Ab known to detect mouse CD40L was used to identify the purified crCD40L protein. The purified recombinant crCD40L was used as a coating antigen in a concentration gradient manner, and was detected with an Ab generated against the mouse CD40L at all concentrations ( Fig 5) . Uncoated controls were performed in parallel and were negative for CD40L in ELISA. We measured the overall strength of the antigen-antibody complex in the presence of 6M urea [41] . The avidity of the cotton rat CD40L for the anti-mouse CD40L antibody was decreased in the presence of 6M urea at all concentrations. Clearly, as the antibody used was raised against mouse CD40L, the crCD40L is detected by mouse CD40L. crCD40L was expressed in vaccinia virus and purified from infected BHK21 cell lysate on a nickel column. The purified protein was detected by ELISA using a mouse antibody against CD40L in a concentration gradient dependent manner. The avidity of the mouse CD40L antibody to the cotton rat CD40L protein was evaluated in the presence of 6M urea. The difference between the untreated and 6M urea treated for each group was calculated using students t-test ÃÃÃ p<0.001, ÃÃÃÃ p<0.0001 (n = 2). Data shown is a representative experiment of three separate experiments where two (n = 2) technical replicates are conducted in each experiment. The no-coating and noprimary antibody negative controls gave average OD values of 0.56 and 0.107 respectively.\\n\\nhttps://doi.org/10.1371/journal.pone.0199067.g005 addition of urea treatment would substantially weaken the interaction between the antibody and crCD40L.\\n\\nSince the cotton rat CD40L protein sequence shared 82% identity with the mouse CD40L protein sequence with similar functional domains, we evaluated the biological activity of the recombinant crCD40L on immature murine bone marrow DCs. We conducted experiments based on known functional activities of CD40L in other animal species. Specifically, maturation of immature DCs after exposure to antigen is known to play a crucial role in their immunity-stimulating function [36] , while trimeric recombinant CD40L has been shown to stimulate DC immunomodulating functions [42] . When CD40L engages CD40 on the surface of DCs, it promotes cytokine production, the induction of cell surface co-stimulatory molecules, and facilitates the cross-presentation of antigen by these cells [27] . In addition, CD11c is a DC integrin marker and upon stimulation, is down-regulated [43] . Intracellular adhesion marker CD54, along with co-stimulatory markers CD40, CD80, and CD86 are all upregulated upon stimulation with CD40L [44, 45] . Moreover, mouse I-A d major histocompatibility complex is also up-regulated upon stimulation with CD40L [45] . When our recombinant crCD40L was used to stimulate immature murine bone marrow DCs, we observed similar results to that when murine CD40L is used (Tables 1 and 2 ). CD11c was down regulated in both median flouresence intensity (Table 1 ) and the percentage of positive cells ( Table 2 ). The co-stimulatory molecules CD54, CD40, CD80, and CD86 were all up-regulated in both median fluorescence intensity (Table 1 ) and the percentage of positive cells ( Table 2 ). The Mouse I-A d major histocompatibility complex was upregulated in median fluorescence intensity (Table 1) but not up-regulated in terms of the overall percentage of positive cells (Table 2) . We speculate this to be due to the species incompatibility since we are stimulating mouse bone marrow cells with cotton rat CD40L. Nevertheless, the crCD40L was able to promote up-regulation of key co-stimulatory markers on immature DCs promoting DC maturation. The gating strategy used for the flow cytometry analysis is provided in S3 Fig along with overlapping histograms of the intracellular adhesion marker and co-stimulatory markers. CD40-induced activation of cytokine gene expression in DCs by CD40L is an important process in the initiation of primary immune responses and is critical for DC maturation and the generation of antigen-specific T cell responses [46] . IL-6 is a highly pleiotropic cytokine in that it stimulates the activation, proliferation, and survival of T cells, and furthermore, modifies DC function and survival [47] [48] [49] [50] . We tested if the recombinant crCD40L could induce IL-6 gene expression (Fig 6a) and production of the cytokine (Fig 6b) by immature murine bone marrow DCs. The results indicate that a significant increase in both IL-6 gene expression and cytokine production in immature murine bone marrow DCs was observed forty hours after stimulation with the crCD40L. Collectively, the observation that both the upregulation of immature DC cell surface maturation markers and increased IL-6 gene expression and cytokine production provide strong evidence of the biological activity of crCD40L.\\n\\nIn summary, the cotton rat CD40L cDNA that we isolated was a 1104 nucleotide sequence with a poly-A tail containing an ORF of 783 bp which coded for a 260 aa protein. The recombinant cotton rat CD40L was recognized by an Ab against mouse CD40L in direct ELISA, and showed biological activity by upregulating maturation markers (CD40, CD54, CD80, and CD86) as well as I-A d on immature bone marrow murine DCs and moreover, inducing upregulation of IL-6 gene and cytokine expression in these cells.\\n\\nThe isolation of the cotton rat CD40L sequence and availability of CD40L has the potential to positively impact basic immunological research and vaccine development, given the critical importance of this protein in orchestrating immune responses [51, 52] .\", 'document_id': 1570}]}, {'paragraphs': [{'qas': [{'question': 'What factor may influence viral replication and gene expression?', 'id': 567, 'answers': [{'text': 'the average codon usage frequencies in the host genome', 'answer_start': 2812}], 'is_impossible': False}, {'question': 'What accounts for the variation of codon usage among open reading frameworks?', 'id': 566, 'answers': [{'text': 'mutational pressure and translational selection', 'answer_start': 1742}], 'is_impossible': False}], 'context': \"Bioinformatics analysis of rabbit haemorrhagic disease virus genome\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC3377956/\\n\\nSHA: eff26d8739498efca2d32fe2e66cdbebf0569c50\\n\\nAuthors: Tian, Xiao-ting; Li, Bao-yu; Zhang, Liang; Jiao, Wen-qiang; Liu, Ji-xing\\nDate: 2011-11-01\\nDOI: 10.1186/1743-422x-8-494\\nLicense: cc-by\\n\\nAbstract: BACKGROUND: Rabbit haemorrhagic disease virus (RHDV), as the pathogeny of Rabbit haemorrhagic disease, can cause a highly infectious and often fatal disease only affecting wild and domestic rabbits. Recent researches revealed that it, as one number of the Caliciviridae, has some specialties in its genome, its reproduction and so on. RESULTS: In this report, we firstly analyzed its genome and two open reading frameworks (ORFs) from this aspect of codon usage bias. Our researches indicated that mutation pressure rather than natural is the most important determinant in RHDV with high codon bias, and the codon usage bias is nearly contrary between ORF1 and ORF2, which is maybe one of factors regulating the expression of VP60 (encoding by ORF1) and VP10 (encoding by ORF2). Furthermore, negative selective constraints on the RHDV whole genome implied that VP10 played an important role in RHDV lifecycle. CONCLUSIONS: We conjectured that VP10 might be beneficial for the replication, release or both of virus by inducing infected cell apoptosis initiate by RHDV. According to the results of the principal component analysis for ORF2 of RSCU, we firstly separated 30 RHDV into two genotypes, and the ENC values indicated ORF1 and ORF2 were independent among the evolution of RHDV.\\n\\nText: Synonymous codons are not used randomly [1] . The variation of codon usage among ORFs in different organisms is accounted by mutational pressure and translational selection as two main factors [2, 3] . Levels and causes of codon usage bias are available to understand viral evolution and the interplay between viruses and the immune response [4] . Thus, many organisms such as bacteria, yeast, Drosophila, and mammals, have been studied in great detail up on codon usage bias and nucleotide composition [5] . However, same researches in viruses, especially in animal viruses, have been less studied. It has been observed that codon usage bias in human RNA viruses is related to mutational pressure, G +C content, the segmented nature of the genome and the route of transmission of the virus [6] . For some vertebrate DNA viruses, genome-wide mutational pressure is regarded as the main determinant of codon usage rather than natural selection for specific coding triplets [4] . Analysis of the bovine papillomavirus type 1 (BPV1) late genes has revealed a relationship between codon usage and tRNA availability [7] . In the mammalian papillomaviruses, it has been proposed that differences from the average codon usage frequencies in the host genome strongly influence both viral replication and gene expression [8] . Codon usage may play a key role in regulating latent versus productive infection in Epstein-Barr virus [9] . Recently, it was reported that codon usage is an important driving force in the evolution of astroviruses and small DNA viruses [10, 11] . Clearly, studies of synonymous codon usage in viruses can reveal much about the molecular evolution of viruses or individual genes. Such information would be relevant in understanding the regulation of viral gene expression.\\n\\nUp to now, little codon usage analysis has been performed on Rabbit haemorrhagic disease virus (RHDV), which is the pathogen causing Rabbit haemorrhagic disease (RHD), also known as rabbit calicivirus disease (RCD) or viral haemorrhagic disease (VHD), a highly infectious and often fatal disease that affects wild and domestic rabbits. Although the virus infects only rabbits, RHD continues to cause serious problems in different parts of the world. RHDV is a single positive stranded RNA virus without envelope, which contains two open reading frames (ORFs) separately encoding a predicted polyprotein and a minor structural protein named VP10 [12] . After the hydrolysis of self-coding 3C-like cysteinase, the polyprotein was finally hydrolyzed into 8 cleavage products including 7 nonstructural proteins and 1 structural protein named as VP60 [13, 14] . Studies on the phylogenetic relationship of RHDVs showed only one serotype had been isolated, and no genotyping for RHDV was reported. It reported that the VP10 was translated with an efficiency of 20% of the preceding ORF1 [15] . In order to better understand the characteristics of the RHDV genome and to reveal more information about the viral genome, we have analyzed the codon usage and dinucleotide composition. In this report, we sought to address the following issues concerning codon usage in RHDV: (i) the extent and causes of codon bias in RHDV; (ii) A possible genotyping of RHDV; (iii) Codon usage bias as a factor reducing the expression of VP10 and (iiii) the evolution of the ORFs.\\n\\nThe 30 available complete RNA sequences of RHDV were obtained from GenBank randomly in January 2011. The serial number (SN), collection dates, isolated areas and GenBank accession numbers are listed in Table 1 .\\n\\nTo investigate the characteristics of synonymous codon usage without the influence of amino acid composition, RSCU values of each codon in a ORF of RHDV were calculated according to previous reports (2 Sharp, Tuohy et al. 1986 ) as the followed formula:\\n\\nWhere g ij is the observed number of the ith codon for jth amino acid which has n i type of synonymous codons. The codons with RSCU value higher than 1.0 have positive codon usage bias, while codons with value lower than 1.0 has relative negative codon usage bias. As RSCU values of some codons are nearly equal to 1.0, it means that these codons are chosen equally and randomly.\\n\\nThe index GC3s means the fraction of the nucleotides G+C at the synonymous third codon position, excluding Met, Trp, and the termination codons.\\n\\nThe ENC, as the best estimator of absolute synonymous codon usage bias [16] , was calculated for the quantification of the codon usage bias of each ORF [17] . The predicted values of ENC were calculated as ENC = 2 + s + 29\\n\\nwhere s represents the given (G+C) 3 % value. The values of ENC can also be obtained by EMBOSS CHIPS program [18] . \\n\\nAnalyses were conducted with the Nei-Gojobori model [19] , involving 30 nucleotide sequences. All positions containing gaps and missing data were eliminated. The values of dn, ds and ω (dn/ds) were calculated in MEGA4.0 [20] .\\n\\nMultivariate statistical analysis can be used to explore the relationships between variables and samples. In this study, correspondence analysis was used to investigate the major trend in codon usage variation among ORFs. In this study, the complete coding region of each ORF was represented as a 59 dimensional vector, and each dimension corresponds to the RSCU value of one sense codon (excluding Met, Trp, and the termination codons) [21] .\\n\\nCorrelation analysis was used to identify the relationship between nucleotide composition and synonymous codon usage pattern [22] . This analysis was implemented based on the Spearman's rank correlation analysis way. All statistical processes were carried out by with statistical software SPSS 17.0 for windows.\\n\\nThe values of nucleotide contents in complete coding region of all 30 RHDV genomes were analyzed and listed in Table 2 and Table 3 . Evidently, (C+G)% content of the ORF1 fluctuated from 50.889 to 51.557 with a mean value of 51.14557, and (C+G)% content of the ORF2 were ranged from 35.593 to 40.113 with a mean value of 37.6624, which were indicating that nucleotides A and U were the major elements of ORF2 against ORF1. Comparing the values of A 3 %, U 3 %, C 3 % and G 3 %, it is clear that C 3 % was distinctly high and A 3 % was the lowest of all in ORF1 of RHDV, while U 3 % was distinctly high and C 3 % was the lowest of all in ORF2 of Table 2 Identified nucleotide contents in complete coding region (length > 250 bps) in the ORF1 of RHDV (30 isolates) genome Table 4 . Most preferentially used codons in ORF1 were C-ended or G-ended codons except Ala, Pro and Ser, however, A-ended or G-ended codons were preferred as the content of ORF2. In addition, the dn, ds and ω(dN/dS) values of ORF1 were separately 0.014, 0.338 and 0.041, and the values of ORF2 were 0.034, 0.103 and 0.034, respectively. The ω values of two ORFs in RHDV genome are generally low, indicating that the RHDV whole genome is subject to relatively strong selective constraints.\\n\\nCOA was used to investigate the major trend in codon usage variation between two ORFs of all 30 RHDV selected for this study. After COA for RHDV Genome, one major trend in the first axis (f' 1 ) which accounted for 42.967% of the total variation, and another major trend in the second axis (f' 2 ) which accounted for 3.632% of the total variation. The coordinate of the complete coding region of each ORF was plotted in Figure 1 defining by the first and second principal axes. It is clear that coordinate of each ORF is relatively isolated. Interestingly, we found that relatively isolated spots from ORF2 tend to cluster into two groups: the ordinate value of one group (marked as Group 1) is \\n\\nTo estimate whether the evolution of RHDV genome on codon usage was regulated by mutation pressure or natural selection, the A%, U%, C%, G% and (C+G)% were compared with A 3 %, U 3 %, C 3 %, G 3 % and (C 3 +G 3 )%, respectively (Table 5 ). There is a complex correlation among nucleotide compositions. In detail, A 3 %, U 3 %, C 3 % and G 3 % have a significant negative correlation with G%, C%, U% and A% and positive correlation with A%, U%, C% and G%, respectively. It suggests that nucleotide constraint may influence synonymous codon usage patterns. However, A 3 % has non-correlation with U% and C%, and U 3 % has noncorrelation with A% and G%, respectively, which haven't indicated any peculiarity about synonymous codon usage. Furthermore, C 3 % and G 3 % have non-correlation with A%, G% and U%, C%, respectively, indicating these data don't reflect the true feature of synonymous codon usage as well. Therefore, linear regression analysis was implemented to analyze the correlation between synonymous codon usage bias and nucleotide compositions. Details of correlation analysis between the first two principle axes (f' 1 and f' 2 ) of each RHDV genome in COA and nucleotide contents were listed in Table 6 . In surprise, only f2 values are closely related to base nucleotide A and G content on the third codon position only, suggesting that nucleotide A and G is a factor influencing the synonymous codon usage pattern of RHDV genome. However, f' 1 value has non-correlation with base nucleotide contents on the third codon position; it is observably suggest that codon usage patterns in RHDV were probably influenced by other factors, such as the second structure of viral genome and limits of host. In spite of that, compositional constraint is a factor shaping the pattern of synonymous codon usage in RHDV genome. Figure 1 A plot of value of the first and second axis of RHDV genome in COA. The first axis (f' 1 ) accounts for 42.967% of the total variation, and the second axis (f' 2 ) accounts for 3.632% of the total variation. Table 5 Summary of correlation analysis between the A, U, C, G contents and A 3 , U 3 , C 3 , G 3 contents in all selected samples \\n\\nThere have been more and more features that are unique to RHDV within the family Caliciviridae, including its single host tropism, its genome and its VP10 as a structural protein with unknown function. After we analyzed synonymous codon usage in RHDV (Table 2) , we obtained several conclusions and conjectures as followed.\\n\\n4.1 Mutational bias as a main factor leading to synonymous codon usage variation ENC-plot, as a general strategy, was utilized to investigate patterns of synonymous codon usage. The ENC-plots of ORFs constrained only by a C 3 +G 3 composition will lie on or just below the curve of the predicted values [18] . ENC values of RHDV genomes were plotted against its corresponding (C 3 +G 3 ) %. All of the spots lie below the curve of the predicted values, as shown in Figure 2 , suggesting that the codon usage bias in all these 30 RHDV genomes is principally influenced by the mutational bias.\\n\\nAs we know, the efficiency of gene expression is influenced by regulator sequences or elements and codon usage bias. It reported that the RNA sequence of the 3terminal 84 nucleotides of ORF1were found to be crucial for VP10 expression instead of the encoded peptide. VP10 coding by ORF2 has been reported as a low expressive structural protein against VP60 coding by ORF1 [5] . And its efficiency of translation is only 20% of VP60. According to results showed by Table 4 , it revealed the differences in codon usage patterns of two ORFs, which is a possible factor reducing the expression of VP10.\\n\\nAlthough VP10 encoded by ORF2, as a minor structural protein with unknown functions, has been described by LIU as a nonessential protein for virus infectivity, the ω Figure 2 Effective number of codons used in each ORF plotted against the GC3s. The continuous curve plots the relationship between GC3s and ENC in the absence of selection. All of spots lie below the expected curve.\\n\\nvalue of ORF2 suggests VP10 plays an important role in the certain stage of whole RHDV lifecycle. After combining with low expression and ω value of VP10, we conjectured that VP10 might be beneficial for the replication, release or both of virus by inducing infected cell apoptosis initiate by RHDV. This mechanism has been confirmed in various positive-chain RNA viruses, including coxsackievirus, dengue virus, equine arterivirus, footand-mouth disease virus, hepatitis C virus, poliovirus, rhinovirus, and severe acute respiratory syndrome [23] [24] [25] [26] [27] [28] [29] , although the details remain elusive.\\n\\nAs preceding description, ENC reflects the evolution of codon usage variation and nucleotide composition to some degree. After the correlation analysis of ENC values between ORF1 and ORF2 (Table 7) , the related coefficient of ENC values of two ORFs is 0.230, and p value is 0.222 more than 0.05. These data revealed that no correlation existed in ENC values of two ORFs, indicating that codon usage patterns and evolution of two ORFs are separated each other. Further, this information maybe helps us well understand why RSCU and ENC between two ORFs are quite different.\\n\\nInterestingly, we found that relatively isolated spots from ORF2 tend to cluster into two groups: the ordinate value of one group (marked as Group 1) is positive value and the other one (marked as Group 2) is negative value. And all of those strains isolated before 2000 belonged to Group 2, including Italy-90, RHDV-V351, RHDV-FRG, BS89, RHDV-SD and M67473.1. Although RHDV has been reported as only one type, this may be a reference on dividing into two genotypes.\\n\\nIn this report, we firstly analyzed its genome and two open reading frameworks (ORFs) from this aspect of codon usage bias. Our researches indicated that mutation pressure rather than natural is the most important determinant in RHDV with high codon bias, and the codon usage bias is nearly contrary between ORF1 and ORF2, which is maybe one of factors regulating the expression of VP60 (encoding by ORF1) and VP10\\n\\n(encoding by ORF2). Furthermore, negative selective constraints on the RHDV whole genome implied that VP10 played an important role in RHDV lifecycle. We conjectured that VP10 might be beneficial for the replication, release or both of virus by inducing infected cell apoptosis initiate by RHDV. According to the results of the principal component analysis for ORF2 of RSCU, we firstly separated 30 RHDV into two genotypes, and the ENC values indicated ORF1 and ORF2 were independent among the evolution of RHDV. All the results will guide the next researches on the RHDV as a reference.\", 'document_id': 1567}]}, {'paragraphs': [{'qas': [{'question': 'What age group has the highest rate of severe outcomes?', 'id': 236, 'answers': [{'text': 'people 85 years and older', 'answer_start': 6117}], 'is_impossible': False}, {'question': 'How is COVID-19 spread?', 'id': 225, 'answers': [{'text': 'person-to-person', 'answer_start': 306}], 'is_impossible': False}, {'question': 'How many states in the U.S. have reported cases of COVID-19?', 'id': 226, 'answers': [{'text': '50', 'answer_start': 1277}], 'is_impossible': False}, {'question': 'When did the White House launch the \"15 Days to Slow the Spread\" program?', 'id': 227, 'answers': [{'text': 'March 16', 'answer_start': 1750}], 'is_impossible': False}, {'question': 'What should mildly-ill patients do?', 'id': 230, 'answers': [{'text': 'isolate at home during their illness', 'answer_start': 3493}], 'is_impossible': False}, {'question': 'What type of virus is SARS-CoV-2?', 'id': 231, 'answers': [{'text': 'betacoronavirus', 'answer_start': 4258}], 'is_impossible': False}, {'question': 'What viruses are similar to the COVID-19 coronavirus?', 'id': 232, 'answers': [{'text': 'MERS-CoV and SARS-CoV.', 'answer_start': 4280}], 'is_impossible': False}, {'question': 'What are the phases of a pandemic?', 'id': 233, 'answers': [{'text': 'investigation phase, followed by recognition, initiation, and acceleration phases', 'answer_start': 7253}], 'is_impossible': False}, {'question': 'At which phase does the peak of the pandemic occur?', 'id': 234, 'answers': [{'text': 'at the end of the acceleration phase', 'answer_start': 7365}], 'is_impossible': False}, {'question': 'People with which medical conditions have a higher rate of severe illness?', 'id': 237, 'answers': [{'text': 'People who have serious chronic medical conditions like:\\nHeart disease\\nDiabetes\\nLung disease', 'answer_start': 9476}], 'is_impossible': False}, {'question': 'What kind of test can diagnose COVID-19?', 'id': 238, 'answers': [{'text': 'rRT-PCR test', 'answer_start': 13059}], 'is_impossible': False}, {'question': 'In what species did the COVID-19 virus likely originate?', 'id': 235, 'answers': [{'text': 'bats', 'answer_start': 4352}], 'is_impossible': False}, {'question': 'What risk factors should be considered in addition to clinical symptoms?', 'id': 228, 'answers': [{'text': 'Does the patient have recent travel from an affected area?\\nHas the patient been in close contact with someone with COVID-19 or with patients with pneumonia of unknown cause?\\nDoes the patient reside in an area where there has been community spread of COVID-19?', 'answer_start': 2381}], 'is_impossible': False}], 'context': 'CDC Summary 21 MAR 2020,\\nhttps://www.cdc.gov/coronavirus/2019-ncov/cases-updates/summary.html\\n\\nThis is a rapidly evolving situation and CDC will provide updated information and guidance as it becomes available.\\n\\nUpdated March 21, 2020\\n\\nCDC is responding to a pandemic of respiratory disease spreading from person-to-person caused by a novel (new) coronavirus. The disease has been named “coronavirus disease 2019” (abbreviated “COVID-19”). This situation poses a serious public health risk. The federal government is working closely with state, local, tribal, and territorial partners, as well as public health partners, to respond to this situation. COVID-19 can cause mild to severe illness; most severe illness occurs in older adults.\\n\\nSituation in U.S.\\nDifferent parts of the country are seeing different levels of COVID-19 activity. The United States nationally is in the initiation phase of the pandemic. States in which community spread is occurring are in the acceleration phase. The duration and severity of each pandemic phase can vary depending on the characteristics of the virus and the public health response.\\n\\nCDC and state and local public health laboratories are testing for the virus that causes COVID-19. View CDC’s Public Health Laboratory Testing map.\\nAll 50 states have reported cases of COVID-19 to CDC.\\nU.S. COVID-19 cases include:\\nImported cases in travelers\\nCases among close contacts of a known case\\nCommunity-acquired cases where the source of the infection is unknown.\\nTwenty-seven U.S. states are reporting some community spread of COVID-19.\\nView latest case counts, deaths, and a map of states with reported cases.\\n\\nCDC Recommends\\nEveryone can do their part to help us respond to this emerging public health threat:\\nOn March 16, the White House announced a program called “15 Days to Slow the Spread,”pdf iconexternal icon which is a nationwide effort to slow the spread of COVID-19 through the implementation of social distancing at all levels of society.\\nOlder people and people with severe chronic conditions should take special precautions because they are at higher risk of developing serious COVID-19 illness.\\nIf you are a healthcare provider, use your judgment to determine if a patient has signs and symptoms compatible with COVID-19 and whether the patient should be tested. Factors to consider in addition to clinical symptoms may include:\\nDoes the patient have recent travel from an affected area?\\nHas the patient been in close contact with someone with COVID-19 or with patients with pneumonia of unknown cause?\\nDoes the patient reside in an area where there has been community spread of COVID-19?\\nIf you are a healthcare provider or a public health responder caring for a COVID-19 patient, please take care of yourself and follow recommended infection control procedures.\\nPeople who get a fever or cough should consider whether they might have COVID-19, depending on where they live, their travel history or other exposures. More than half of the U.S. is seeing some level of community spread of COVID-19. Testing for COVID-19 may be accessed through medical providers or public health departments, but there is no treatment for this virus. Most people have mild illness and are able to recover at home without medical care.\\nFor people who are ill with COVID-19, but are not sick enough to be hospitalized, please follow CDC guidance on how to reduce the risk of spreading your illness to others. People who are mildly ill with COVID-19 are able to isolate at home during their illness.\\nIf you have been in China or another affected area or have been exposed to someone sick with COVID-19 in the last 14 days, you will face some limitations on your movement and activity. Please follow instructions during this time. Your cooperation is integral to the ongoing public health response to try to slow spread of this virus.\\nCOVID-19 Emergence\\nCOVID-19 is caused by a coronavirus. Coronaviruses are a large family of viruses that are common in people and many different species of animals, including camels, cattle, cats, and bats. Rarely, animal coronaviruses can infect people and then spread between people such as with MERS-CoV, SARS-CoV, and now with this new virus (named SARS-CoV-2).\\n\\nThe SARS-CoV-2 virus is a betacoronavirus, like MERS-CoV and SARS-CoV. All three of these viruses have their origins in bats. The sequences from U.S. patients are similar to the one that China initially posted, suggesting a likely single, recent emergence of this virus from an animal reservoir.\\n\\nEarly on, many of the patients at the epicenter of the outbreak in Wuhan, Hubei Province, China had some link to a large seafood and live animal market, suggesting animal-to-person spread. Later, a growing number of patients reportedly did not have exposure to animal markets, indicating person-to-person spread. Person-to-person spread was subsequently reported outside Hubei and in countries outside China, including in the United States. Some international destinations now have ongoing community spread with the virus that causes COVID-19, as do some parts of the United States. Community spread means some people have been infected and it is not known how or where they became exposed. Learn more about the spread of this newly emerged coronavirus.\\n\\nSeverity\\nThe complete clinical picture with regard to COVID-19 is not fully known. Reported illnesses have ranged from very mild (including some with no reported symptoms) to severe, including illness resulting in death. While information so far suggests that most COVID-19 illness is mild, a reportexternal icon out of China suggests serious illness occurs in 16% of cases. Older people and people of all ages with severe chronic medical conditions — like heart disease, lung disease and diabetes, for example — seem to be at higher risk of developing serious COVID-19 illness. A CDC Morbidity & Mortality Weekly Report that looked at severity of disease among COVID-19 cases in the United States by age group found that 80% of deaths were among adults 65 years and older with the highest percentage of severe outcomes occurring in people 85 years and older.\\n\\nLearn more about the symptoms associated with COVID-19.\\n\\nCOVID-19 Pandemic\\nA pandemic is a global outbreak of disease. Pandemics happen when a new virus emerges to infect people and can spread between people sustainably. Because there is little to no pre-existing immunity against the new virus, it spreads worldwide.\\n\\nThe virus that causes COVID-19 is infecting people and spreading easily from person-to-person. Cases have been detected in most countries worldwide and community spread is being detected in a growing number of countries. On March 11, the COVID-19 outbreak was characterized as a pandemic by the WHOexternal icon.\\n\\nThis is the first pandemic known to be caused by the emergence of a new coronavirus. In the past century, there have been four pandemics caused by the emergence of novel influenza viruses. As a result, most research and guidance around pandemics is specific to influenza, but the same premises can be applied to the current COVID-19 pandemic. Pandemics of respiratory disease follow a certain progression outlined in a “Pandemic Intervals Framework.” Pandemics begin with an investigation phase, followed by recognition, initiation, and acceleration phases. The peak of illnesses occurs at the end of the acceleration phase, which is followed by a deceleration phase, during which there is a decrease in illnesses. Different countries can be in different phases of the pandemic at any point in time and different parts of the same country can also be in different phases of a pandemic.\\n\\nThere are ongoing investigations to learn more. This is a rapidly evolving situation and information will be updated as it becomes available.\\n\\nRisk Assessment\\nRisk depends on characteristics of the virus, including how well it spreads between people; the severity of resulting illness; and the medical or other measures available to control the impact of the virus (for example, vaccines or medications that can treat the illness) and the relative success of these. In the absence of vaccine or treatment medications, nonpharmaceutical interventions become the most important response strategy. These are community interventions that can reduce the impact of disease.\\n\\nThe risk from COVID-19 to Americans can be broken down into risk of exposure versus risk of serious illness and death.\\n\\nRisk of exposure:\\n\\nThe immediate risk of being exposed to this virus is still low for most Americans, but as the outbreak expands, that risk will increase. Cases of COVID-19 and instances of community spread are being reported in a growing number of states.\\nPeople in places where ongoing community spread of the virus that causes COVID-19 has been reported are at elevated risk of exposure, with the level of risk dependent on the location.\\nHealthcare workers caring for patients with COVID-19 are at elevated risk of exposure.\\nClose contacts of persons with COVID-19 also are at elevated risk of exposure.\\nTravelers returning from affected international locations where community spread is occurring also are at elevated risk of exposure, with level of risk dependent on where they traveled.\\nRisk of Severe Illness:\\n\\nEarly information out of China, where COVID-19 first started, shows that some people are at higher risk of getting very sick from this illness. This includes:\\n\\nOlder adults, with risk increasing by age.\\nPeople who have serious chronic medical conditions like:\\nHeart disease\\nDiabetes\\nLung disease\\nCDC has developed guidance to help in the risk assessment and management of people with potential exposures to COVID-19.\\n\\nWhat May Happen\\nMore cases of COVID-19 are likely to be identified in the United States in the coming days, including more instances of community spread. CDC expects that widespread transmission of COVID-19 in the United States will occur. In the coming months, most of the U.S. population will be exposed to this virus.\\n\\nWidespread transmission of COVID-19 could translate into large numbers of people needing medical care at the same time. Schools, childcare centers, and workplaces, may experience more absenteeism. Mass gatherings may be sparsely attended or postponed. Public health and healthcare systems may become overloaded, with elevated rates of hospitalizations and deaths. Other critical infrastructure, such as law enforcement, emergency medical services, and sectors of the transportation industry may also be affected. Healthcare providers and hospitals may be overwhelmed. At this time, there is no vaccine to protect against COVID-19 and no medications approved to treat it. Nonpharmaceutical interventions will be the most important response strategy to try to delay the spread of the virus and reduce the impact of disease.\\n\\nCDC Response\\nGlobal efforts at this time are focused concurrently on lessening the spread and impact of this virus. The federal government is working closely with state, local, tribal, and territorial partners, as well as public health partners, to respond to this public health threat.\\n\\nHighlights of CDC’s Response\\nCDC established a COVID-19 Incident Management System on January 7, 2020. On January 21, CDC activated its Emergency Operations Center to better provide ongoing support to the COVID-19 response.\\nThe U.S. government has taken unprecedented steps with respect to travel in response to the growing public health threat posed by this new coronavirus:\\nForeign nationals who have been in China, Iran, the United Kingdom, Ireland and any one of the 26 European countries in the Schengen Area within the past 14 days cannot enter the United States.\\nU.S. citizens, residents, and their immediate family members who have been any one of those countries within in the past 14 days can enter the United States, but they are subject to health monitoring and possible quarantine for up to 14 days.\\nPeople at higher risk of serious COVID-19 illness avoid cruise travel and non-essential air travel.\\nCDC has issued additional specific travel guidance related to COVID-19.\\nCDC has issued clinical guidance, including:\\nClinical Guidance for Management of Patients with Confirmed Coronavirus Disease (COVID-19).\\nInfection Prevention and Control Recommendations for Patients, including guidance on the use of personal protective equipment (PPE) during a shortage.\\nCDC also has issued guidance for other settings, including:\\nPreparing for COVID-19: Long-term Care Facilities, Nursing Homes\\nDiscontinuation of Home Isolation for Persons with COVID-19\\nCDC has deployed multidisciplinary teams to support state health departments in case identification, contact tracing, clinical management, and public communications.\\nCDC has worked with federal partners to support the safe return of Americans overseas who have been affected by COVID-19.\\n\\nAn important part of CDC’s role during a public health emergency is to develop a test for the pathogen and equip state and local public health labs with testing capacity.\\nCDC developed an rRT-PCR test to diagnose COVID-19.\\nAs of the evening of March 17, 89 state and local public health labs in 50 states, the District of Columbia, Guam, and Puerto Rico have successfully verified and are currently using CDC COVID-19 diagnostic tests.\\nCommercial manufacturers are now producing their own tests.\\nCDC has grown the COVID-19 virus in cell culture, which is necessary for further studies, including for additional genetic characterization. The cell-grown virus was sent to NIH’s BEI Resources Repositoryexternal icon for use by the broad scientific community.\\nCDC also is developing a serology test for COVID-19.\\nOther Available Resources\\nThe following resources are available with information on COVID-19\\n\\nWorld Health Organization, Coronavirusexternal icon\\n', 'document_id': 185}]}, {'paragraphs': [{'qas': [{'question': 'What is the acronym SARS-CoV-2?', 'id': 244, 'answers': [{'text': 'severe acute respiratory syndrome coronavirus 2 ', 'answer_start': 524}], 'is_impossible': False}, {'question': 'When was SARS-CoV-2 first identified?', 'id': 245, 'answers': [{'text': 'December 2019', 'answer_start': 1428}], 'is_impossible': False}, {'question': 'Where was SARS-CoV-2 first identified?', 'id': 246, 'answers': [{'text': 'Wuhan, capital of Hubei Province, China', 'answer_start': 1457}], 'is_impossible': False}, {'question': 'What factor positively correlates with imported-and-reported cases counts of SARS-CoV-2 infection?', 'id': 247, 'answers': [{'text': 'daily air travel volume', 'answer_start': 11044}], 'is_impossible': False}, {'question': 'What is the doubling time of the COVID-19 pandemic?', 'id': 272, 'answers': [{'text': '≈6 days', 'answer_start': 1747}], 'is_impossible': False}], 'context': 'Identifying Locations with Possible Undetected Imported Severe Acute Respiratory Syndrome Coronavirus 2 Cases by Using Importation Predictions,\\nhttps://wwwnc.cdc.gov/eid/article/26/7/20-0250_article\\nVolume 26, Number 7—July 2020\\nResearch\\n\\nPablo Martinez De Salazar1Comments to Author , René Niehus, Aimee Taylor1, Caroline O’Flaherty Buckee, and Marc LipsitchComments to Author\\nAuthor affiliations: Harvard T.H. Chan School of Public Health, Boston, Massachusetts, USA\\nSuggested citation for this article\\n\\nAbstract\\nCases of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection exported from mainland China could lead to self-sustained outbreaks in other countries. By February 2020, several countries were reporting imported SARS-CoV-2 cases. To contain the virus, early detection of imported SARS-CoV-2 cases is critical. We used air travel volume estimates from Wuhan, China, to international destinations and a generalized linear regression model to identify locations that could have undetected imported cases. Our model can be adjusted to account for exportation of cases from other locations as the virus spreads and more information on importations and transmission becomes available. Early detection and appropriate control measures can reduce the risk for transmission in all locations.\\n\\nA novel coronavirus, later named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), was identified in December 2019 in the city of Wuhan, capital of Hubei Province, China, where cases were first confirmed (1). During December 2019–February 2020, the number of confirmed cases increased drastically. Model estimates suggested that >75,000 persons were infected by January 25, 2020, and the epidemic had a doubling time of ≈6 days (2). By the end of January 2020, travel restrictions were implemented for Wuhan and neighboring cities. Nonetheless, the virus spread from Wuhan to other cities in China and outside the country. By February 4, 2020, a total of 23 locations outside mainland China reported cases, 22 of which reported imported cases; Spain reported a case caused by secondary transmission (3).\\n\\nMost cases imported to other locations have been linked to recent travel history from China (3), suggesting that air travel plays a major role in exportation of cases to locations outside of China. To prevent other cities and countries from becoming epicenters of the SARS-CoV-2 epidemic, substantial targeted public health interventions are required to detect cases and control local spread of the virus. We collected estimates of air travel volume from Wuhan to 194 international destinations. We then identified 49 countries that had a score of >49.2/100 on category 2, Early Detection and Reporting of Epidemics of Potential International Concern, of the Global Health Security (GHS) Index (4). We assumed these locations would be proficient at detecting SARS-CoV-2 and reporting confirmed imported cases, which we refer to as imported-and-reported cases. We ran a generalized linear regression model on this subset; based on the results, we generated predictions for the remainder of the sample. Using these predictions, we identified locations that might not be detecting imported cases.\\n\\nMethods\\nTo identify locations reporting fewer than predicted imported SARS-CoV-2 infected cases, we fit a model to data from 49 locations outside mainland China with high surveillance capacity according to the GHS Index (4). Among these, 17 had high travel connectivity to Wuhan and 32 have low connectivity to Wuhan. We considered locations to be countries without any position on territorial claims. We performed a Poisson regression by using the cumulative number of imported-and-reported SARS-CoV-2 cases in these 49 countries and the estimated number of daily airline passengers from the Wuhan airport. We then compared predictions from this model with imported-and-reported cases across 194 locations from the GHS Index, excluding China as the epicenter of the outbreak.\\n\\nThe model requires data on imported-and-reported cases of SARS-CoV-2 infection, daily air travel volume, and surveillance capacity. We obtained data on imported-and-reported cases aggregated by destination from the World Health Organization technical report issued February 4, 2020 (3). We assumed a case count of 0 for locations not listed. We used February 4 as the cutoff for cumulative imported-and-reported case counts because exported cases from Hubei Province dropped rapidly after this date (3), likely because of travel restrictions for the province implement on January 23. We defined imported-and-reported cases as those with known travel history from China; of those, 83% had a travel history from Hubei Province and 17% traveled from unknown locations in China (3). We excluded reported cases likely caused by transmission outside of China or cases in which the transmission source was still under investigation (3). In addition, we excluded Hong Kong, Macau, and Taiwan from our model because locally transmitted and imported cases were not disaggregated in these locations.\\n\\nWe obtained data on daily air travel from a network-based modeling study (S. Lai et al., unpub. data, https://doi.org/10.1101/2020.02.04.20020479External Link) that reported monthly air travel volume estimates for the 27 locations outside mainland China that are most connected to Wuhan. These estimates were calculated from International Air Travel Association data from February 2018, which includes direct and indirect flight itineraries from Wuhan. For these 27 locations, estimated air travel volumes are >6 passengers/day. We assumed that travel volumes for locations not among the most connected are censored by a detection limit. We used a common method of dealing with censored data from environmental sampling (5), or metabolomics (6), to set the daily air travel volume to half the minimum previously reported. Therefore, we used 3 passengers/day for estimated travel volumes for the 167 locations from the GHS Index not listed by Lai et al. We tested the robustness of our results by using a set of alternative values of 0.1, 1, and 6 passengers/day for the censored data.\\n\\nWe defined high surveillance locations as those with a GHS Index for category 2 above the 75th quantile. We assessed the number of high surveillance locations, those with 0 imported-and-reported cases, and low surveillance locations, those with case counts >1 (Table).\\n\\nFor our model, we assumed that the cumulative imported-and-reported case counts across 49 high surveillance locations follow a Poisson distribution from the beginning of the epidemic until February 4, 2020. Then the expected case count is linearly proportional to the daily air travel volume in the following formula:where i denotes location, Ci denotes the imported-and-reported case count in a location, λi denotes the expected case count in a location, β denotes the regression coefficient, and xi denotes the daily air travel volume of a location. The Poisson model assumes cases are independent and that the variance is equal to the expected case count. Imported-and-reported cases likely meet the independence assumption because the value excludes cases with local transmission. We also checked the robustness of our results by using an over dispersed model with a negative binomial likelihood. We computed the p value of the overdispersion parameter as shown in Gelman and Hill (7).\\n\\nThumbnail of Regression plot of locations with possible undetected imported cases of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by air travel volume from Wuhan, China. Air travel volume measured in number of persons/day. No. cases refers to possible undetected imported SARS-CoV-2 cases. Solid line indicates the expected imported-and-reported case counts for locations. Dashed lines represent 95% prediction interval bounds smoothed for all locations. Purple dots indicate location\\nFigure 1. Regression plot of locations with possible undetected imported cases of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by air travel volume from Wuhan, China. Air travel volume measured in number of...\\n\\nWe used R version 3.6.1 (https://www.r-project.orgExternal Link) to compute , the maximum likelihood estimate of β, and the expected imported-and-reported case count given high surveillance (Figure 1). We also computed the 95% prediction interval (PI) bounds under this model of high surveillance for all 194 values of daily air travel volume (Figure 1). First, we generated a bootstrapped dataset by sampling n locations with replacement among high surveillance locations. Then, we reestimated β by using the bootstrapped dataset. Finally, we simulated imported-and-reported case counts for all 194 locations under our model by using the estimate of β from the bootstrapped dataset. We repeated the 3 steps 50,000 times to generate 50,000 simulated imported-and-reported case counts for each of the locations computed to the lower and upper PI bounds (PI 2.5%–97.5%). We smoothed the 95% PI bounds by using ggplot2 in R (8). We fit the imported-and-reported case counts of the 49 high surveillance locations to the model and plotted these alongside 145 locations with low surveillance capacity (Figure 1). We noted some overlap between high and low surveillance locations (Figure 1).\\n\\nThumbnail of Analyses of imported-and-reported cases and daily air travel volume using a model to predict locations with potentially undetected cases of severe acute respiratory virus 2 (SARS-CoV-2). Air travel volume measured in number of persons/day. No. cases refers to possible undetected imported SARS-CoV-2 cases. Solid line shows the expected imported-and-reported case counts based on our model fitted to high surveillance locations, indicated by purple dots. Dashed lines indicate the 95% pr\\nFigure 2. Analyses of imported-and-reported cases and daily air travel volume using a model to predict locations with potentially undetected cases of severe acute respiratory virus 2 (SARS-CoV-2). Air travel volume measured in...\\n\\nTo assess the robustness of our results we ran 8 additional regression analyses by implementing a series of changes to the analysis. The changes included the following: set the daily air travel volume to 0.1, 1, or 6 passengers/day for locations not listed by Lai et al. (unpub. data, https://doi.org/10.1101/2020.02.04.20020479External Link) (Figure 2, panels A–C); removed all locations not listed by Lai et al. before fitting (Figure 2, panel D); defined high surveillance locations by using a more lenient GHS Index criterion, 50th quantile (Figure 2, panel E), and a more stringent criterion, 95th quantile (Figure 2, panel F); excluded Thailand from the model because it is a high-leverage point (Figure 2, panel G); or used an overdispersed Poisson likelihood with a negative-binomial likelihood (Figure 2, panel H). We provide code for these analyses on GitHub (https://github.com/c2-d2/cov19flightimportExternal Link).\\n\\nTop\\n\\nResults\\nWe found that daily air travel volume positively correlates with imported-and-reported case counts of SARS-CoV-2 infection among high surveillance locations (Figure 1). We noted that increasing flight volume by 31 passengers/day is associated with 1 additional expected imported-and-reported case. In addition, Singapore and India lie above the 95% PI in our model; Singapore had 12 more imported-and-reported cases (95% PI 6–17 cases) than expected and India had 3 (95% PI 1–3 cases) more than expected. Thailand has a relatively high air travel volume compared with other locations, but it lies below the 95% PI, reporting 16 (95% PI 1–40 cases) fewer imported-and-reported cases than expected under the model. Indonesia lies below the PI and has no imported-and-reported cases, but the expected case count is 5 (95% PI 1–10 cases) in our model. Across all 8 robustness regression analyses, we consistently observed that Singapore lies above the 95% PI and Thailand and Indonesia lie below (Figure 2). India remains above the 95% PI in all robustness analyses except when we used the more stringent GHS Index, 95th quantile, for fitting; then India lies on the upper bound of the 95% PI (Figure 2, panel F).\\n\\nTop\\n\\nDiscussion\\nWe aimed to identify locations with likely undetected or underdetected imported cases of SARS-CoV-2 by fitting a model to the case counts in locations with high surveillance capacity and Wuhan-to-location air travel volumes. Our model can be adjusted to account for exportation of cases from locations other than Wuhan as the outbreak develops and more information on importations and self-sustained transmission becomes available. One key advantage of this model is that it does not rely on estimates of incidence or prevalence in the epicenter of the outbreak. Also, we intentionally used a simple generalized linear model. The linearity of the expected case count means that we have only 1 regression coefficient in the model and no extra parameters. The Poisson likelihood then captures the many 0-counts observed for less highly connected locations but also describes the slope between case-count and flight data among more connected locations. We believe this model provides the most parsimonious phenomenologic description of the data.\\n\\nAccording to our model, locations above the 95% PI of imported-and-reported cases could have higher case-detection capacity. Locations below the 95% PI might have undetected cases because of expected imported-and-reported case counts under high surveillance. Underdetection of cases could increase the international spread of the outbreak because the transmission chain could be lost, reducing opportunities to deploy case-based control strategies. We recommend rapid strengthening of outbreak surveillance and control efforts in locations below the 95% PI lower bound, particularly Indonesia, to curb potential local transmission. Early detection of cases and implantation of appropriate control measures can reduce the risk for self-sustained transmission in all locations.\\n\\nTop\\n\\nDr. De Salazar is a research fellow at Harvard T.H. Chan School of Public Health, working on multiscale statistical models of infectious diseases within host, population, and metapopulation models. His research interests include diagnostic laboratory methods and public health response.\\n\\nTop\\n\\nAcknowledgments\\nWe thank Pamela Martinez, Nicholas Jewel, and Stephen Kissler for valuable feedback.\\n\\nThis work was supported by US National Institute of General Medical Sciences (award no. U54GM088558). P.M.D was supported by the Fellowship Foundation Ramon Areces. A.R.T. and C.O.B. were supported by a Maximizing Investigator’s Research Award (no. R35GM124715-02) from the US National Institute of General Medical Sciences.\\n\\nThe authors are solely responsible for this content and it does not necessarily represent the official views of the National Institute of General Medical Sciences or the National Institutes of Health.\\n\\nDeclaration of interests: Marc Lipsitch has received consulting fees from Merck. All other authors declare no competing interests.\\n\\nTop\\n\\nReferences\\nZhou  P, Yang  XL, Wang  XG, Hu  B, Zhang  L, Zhang  W, et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature. 2020;579:270–3.\\nWu  JT, Leung  K, Leung  GM. Nowcasting and forecasting the potential domestic and international spread of the 2019-nCoV outbreak originating in Wuhan, China: a modelling study. Lancet. 2020;395:689–97. DOIExternal LinkPubMedExternal Link\\nWorld Health Organization. Coronavirus disease 2019 (COVID-19) situation report—15, 4 Feb 2020 [cited 2020 Feb 14]. https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200204-sitrep-15-ncov.pdfExternal Link\\nNuclear Threat Initiative and Johns Hopkins Center for Health Security. Global health security index [cited 2020 Feb 14]. https://www.ghsindex.orgExternal Link\\nUS Environmental Protection Agency. Data quality assessment: statistical methods for practitioners EPA QA/G9-S [cited 2020 Feb 14]. Washington: The Agency; 2006. https://www.epa.gov/sites/production/files/2015-08/documents/g9s-final.pdfExternal Link\\nLamichhane  S, Sen  P, Dickens  AM, Hyötyläinen  T, Orešič  M. An overview of metabolomics data analysis: current tools and future perspectives. In: Jaumot J, Bedia C, Tauler R, editors. Comprehensive analytical chemistry. Vol. 82. Amsterdam: Elsevier; 2018. p. 387–413.\\nGelman  A, Hill  J. Analytical methods for social research. In: Data analysis using regression and multilevel/hierarchical models. Cambridge: Cambridge University Press; 2006. p. 235–236.\\nWickham  H. ggplot2: elegant graphics for data analysis. New York: Springer; 2016.\\nTop\\n\\nFigures\\nFigure 1. Regression plot of locations with possible undetected imported cases of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by air travel volume from Wuhan, China. Air travel volume measured in...\\nFigure 2. Analyses of imported-and-reported cases and daily air travel volume using a model to predict locations with potentially undetected cases of severe acute respiratory virus 2 (SARS-CoV-2). Air travel volume...\\nTable\\nTable. Surveillance capacity of locations with and without imported-and-reported cases of severe acute respiratory syndrome coronavirus 2, 2020\\nTop\\n\\nSuggested citation for this article: De Salazar PM, Niehus R, Taylor A, O’Flaherty Buckee C, Lipsitch M. Identifying locations with possible undetected imported severe acute respiratory syndrome coronavirus 2 cases by using importation predictions. Emerg Infect Dis. 2020 Jul [date cited]. https://doi.org/10.3201/eid2607.200250\\n\\nDOI: 10.3201/eid2607.200250\\n\\nOriginal Publication Date: 3/24/2020\\n\\n1These authors contributed equally to this article.\\n\\nTable of Contents – Volume 26, Number 7—July 2020\\n', 'document_id': 186}]}, {'paragraphs': [{'qas': [{'question': 'When did the first known cases of Middle East respiratory syndrome (MERS) occur?', 'id': 4173, 'answers': [{'text': ' in 2012 ', 'answer_start': 413}], 'is_impossible': False}, {'question': 'Where did the first known cases of Middle East respiratory syndrome (MERS) occur?', 'id': 4174, 'answers': [{'text': 'in Jordan', 'answer_start': 422}], 'is_impossible': False}, {'question': 'Where was the the case first to be publicly reported was from ?', 'id': 4175, 'answers': [{'text': 'Jeddah, in the Kingdom of Saudi Arabia (KSA)', 'answer_start': 515}], 'is_impossible': False}, {'question': 'In what animals MERS-CoV sequences have been found ?', 'id': 4176, 'answers': [{'text': 'in a bat and in many dromedary camels (DC).', 'answer_start': 608}], 'is_impossible': False}, {'question': 'Where is MERS-CoV is enzootic in DC?', 'id': 4177, 'answers': [{'text': 'across the Arabian Peninsula and in parts of Africa', 'answer_start': 679}], 'is_impossible': False}, {'question': 'What does MERS-COV cause?', 'id': 4178, 'answers': [{'text': ' mild upper respiratory tract illness in its camel reservoir and sporadic, but relatively rare human infections.', 'answer_start': 739}], 'is_impossible': False}, {'question': 'Precisely how does the virus transmit to humans?', 'id': 4179, 'answers': [{'text': 'remains unknown', 'answer_start': 892}], 'is_impossible': False}, {'question': 'What appears to be a requirement for transmission?', 'id': 4180, 'answers': [{'text': 'close and lengthy exposure', 'answer_start': 912}], 'is_impossible': False}, {'question': 'What is a focal point of MERS?', 'id': 4181, 'answers': [{'text': 'The KSA ', 'answer_start': 968}], 'is_impossible': False}, {'question': 'What is MERS mostly known as?', 'id': 4182, 'answers': [{'text': 'lower respiratory tract (LRT) disease ', 'answer_start': 1075}], 'is_impossible': False}, {'question': 'What does the MERS LRT disease involve?', 'id': 4183, 'answers': [{'text': 'fever, cough, breathing difficulties and pneumonia that may progress to acute respiratory distress syndrome, multiorgan failure and death in 20 % to 40 % of those infected. ', 'answer_start': 1123}], 'is_impossible': False}, {'question': 'Where else MERS-COV has also been detected?', 'id': 4184, 'answers': [{'text': ' in mild and influenza-like illnesses and in those with no signs or symptoms.', 'answer_start': 1336}], 'is_impossible': False}, {'question': 'Who suffers severe diseases from MERS?', 'id': 4185, 'answers': [{'text': 'Older males', 'answer_start': 1414}], 'is_impossible': False}, {'question': ' Compared to severe acute respiratory syndrome (SARS)  and another sometimes- fatal zoonotic coronavirus disease, how does MERS affect the patients?', 'id': 4186, 'answers': [{'text': 'MERS progresses more rapidly to respiratory failure and acute kidney injury (it also has an affinity for growth in kidney cells under laboratory conditions), is more frequently reported in patients with underlying disease and is more often fatal.', 'answer_start': 1643}], 'is_impossible': False}, {'question': 'To what have most human cases of MERS  been linked?', 'id': 4187, 'answers': [{'text': 'to lapses in infection prevention and control (IPC) in healthcare settings', 'answer_start': 1932}], 'is_impossible': False}, {'question': 'Among whom 20% of the virus detection are reported?', 'id': 4188, 'answers': [{'text': ' among healthcare workers (HCWs) and higher exposures in those with occupations that bring them into close contact with camels. ', 'answer_start': 2064}], 'is_impossible': False}, {'question': 'What have sero-surveys of MERS virus found?', 'id': 4189, 'answers': [{'text': 'widespread evidence of past infection in adult camels and limited past exposure among humans.', 'answer_start': 2216}], 'is_impossible': False}, {'question': 'How was the first culture of the new Coronavirus announced?', 'id': 4190, 'answers': [{'text': 'An email from Dr Ali Mohamed Zaki, an Egyptian virologist working at the Dr Soliman Fakeeh Hospital in Jeddah in the Kingdom of Saudi Arabia (KSA)', 'answer_start': 2831}], 'is_impossible': False}, {'question': 'Where was the Email published?', 'id': 4191, 'answers': [{'text': 'on the website of the professional emerging diseases (ProMED) network', 'answer_start': 3065}], 'is_impossible': False}, {'question': 'When was the Email  published?', 'id': 4192, 'answers': [{'text': '20 th September 2012 ', 'answer_start': 3138}], 'is_impossible': False}, {'question': 'Whose was the first reported case?', 'id': 4193, 'answers': [{'text': 'a 60 year old man from Bisha in the KSA', 'answer_start': 3211}], 'is_impossible': False}, {'question': 'How many  viral RNA or virus-specific antibodies been detected?', 'id': 4194, 'answers': [{'text': '1,493', 'answer_start': 3622}], 'is_impossible': False}, {'question': 'What is the death rate from MERS-COV?', 'id': 4195, 'answers': [{'text': 'over a third of the positive people dying (at least 527, 35 %)', 'answer_start': 3788}], 'is_impossible': False}, {'question': 'What did the discovery process over two to three years reveal?', 'id': 4197, 'answers': [{'text': 'a virus that had infected over 90 % of adult dromedary camels (DC; Camelus dromedarius) in the KSA [4] , also DCs across the Arabian Peninsula and parts of Africa that are a source of DC imports for the KSA ', 'answer_start': 3955}], 'is_impossible': False}, {'question': 'What does subsequent transmission of MERS-CoV to other humans require?', 'id': 4198, 'answers': [{'text': ' relatively close and prolonged exposure ', 'answer_start': 4407}], 'is_impossible': False}, {'question': 'What would restrict access to both the virus and to viral diagnostics ?', 'id': 4200, 'answers': [{'text': 'The first viral isolate was patented ', 'answer_start': 4456}], 'is_impossible': False}, {'question': 'How was the virus then made freely available? ', 'id': 4201, 'answers': [{'text': ' sensitive, validated reverse transcriptase real-time polymerase chain reaction (RT-rtPCR)-based diagnostics were quickly described ', 'answer_start': 4612}], 'is_impossible': False}, {'question': \"What has epidemiology and research identified the MERS-CoV's cell receptor is?\", 'id': 4202, 'answers': [{'text': 'as exopeptidase dipeptidyl peptidase 4 (DPP4; also called CD26)', 'answer_start': 4901}], 'is_impossible': False}, {'question': 'How does MERS-CoV compare with SARS-CoV?', 'id': 4203, 'answers': [{'text': 'eliciting a more proinflammatory response than SARS-CoV', 'answer_start': 5044}], 'is_impossible': False}, {'question': 'How does MERS-CoV compare with SARS-CoV?', 'id': 4204, 'answers': [{'text': 'MERS kills its human host more often than SARS did (20-40 % versus 9 % for SARS ', 'answer_start': 5174}], 'is_impossible': False}, {'question': 'How does MERS-CoV spread among people?', 'id': 4205, 'answers': [{'text': 'From intermittent animal-to-human spill-over events', 'answer_start': 5390}], 'is_impossible': False}, {'question': 'Who gets more severe disease from MERS?', 'id': 4206, 'answers': [{'text': 'among older adults, especially males, with pre-existing diseases. ', 'answer_start': 5519}], 'is_impossible': False}, {'question': 'What is the spread of MERS-CoV among humans, associated with?', 'id': 4207, 'answers': [{'text': 'with outbreaks in hospitals, with around 20 % of all cases to date involving healthcare workers (HCWs).', 'answer_start': 5647}], 'is_impossible': False}, {'question': 'What do the DCs suffer with MERS-CoV infection?', 'id': 4208, 'answers': [{'text': \" the equivalent of a 'common cold' from MERS-CoV infection,\", 'answer_start': 5781}], 'is_impossible': False}, {'question': 'What happens to humans infected by MERS-CoV virus?', 'id': 4209, 'answers': [{'text': ' the virus can be a more serious and opportunistic pathogen associated with the death of up to 40 % of reported cases.', 'answer_start': 5851}], 'is_impossible': False}, {'question': 'What is the incubation period of MERS?', 'id': 4210, 'answers': [{'text': ' five to six days, ranging from two to 16 days', 'answer_start': 6208}], 'is_impossible': False}, {'question': 'What is the duration between when illness begins in one person and subsequently spreads to another?', 'id': 4211, 'answers': [{'text': '13 to 14 days', 'answer_start': 6261}], 'is_impossible': False}, {'question': 'What is the median time to death in case of progressive illness?', 'id': 4212, 'answers': [{'text': '11 to 13 days', 'answer_start': 6441}], 'is_impossible': False}, {'question': 'What is the progression  of symptoms to disease?', 'id': 4214, 'answers': [{'text': 'Fever and gastrointestinal symptoms may form a prodrome, after which symptoms decline, only to be followed by a more severe systemic and respiratory syndrome ', 'answer_start': 6496}], 'is_impossible': False}, {'question': 'What did the later WHO definition of MERS clearly state?', 'id': 4216, 'answers': [{'text': 'a confirmed case included any person whose sample was RT-PCR positive for MERS-CoV, or who produced a seroconversion, irrespective of clinical signs and symptoms.', 'answer_start': 7331}], 'is_impossible': False}, {'question': 'What percentage of cases KSA has been a source of?', 'id': 4217, 'answers': [{'text': ' 79 % ', 'answer_start': 8084}], 'is_impossible': False}, {'question': 'What is severe MARS noted for?', 'id': 4218, 'answers': [{'text': 'its impact among older men with comorbid diseases including diabetes mellitus, cirrhosis and various lung, renal and cardiac conditions ', 'answer_start': 8133}], 'is_impossible': False}, {'question': 'What symptoms appear among the  confirmed cases of MERS?', 'id': 4222, 'answers': [{'text': 'fever, cough and upper respiratory tract (URT) signs and symptoms usually occur first, followed within a week by progressive LRT distress and lymphopaenia', 'answer_start': 8418}], 'is_impossible': False}, {'question': 'What do patients often present to a hospital with, in cases of MERS?', 'id': 4223, 'answers': [{'text': ' pneumonia', 'answer_start': 8621}], 'is_impossible': False}, {'question': 'What can  MERS disease progress to?', 'id': 4224, 'answers': [{'text': 'acute respiratory distress syndrome and multiorgan system failure', 'answer_start': 8732}], 'is_impossible': False}, {'question': 'What percentage of all  reported cases has MERS  reportedly killed ?', 'id': 4225, 'answers': [{'text': ' 35 % ', 'answer_start': 8845}], 'is_impossible': False}, {'question': 'What percentage of of all reported cases has MERS  reportedly killed in KSA?', 'id': 4226, 'answers': [{'text': '42 %', 'answer_start': 8874}], 'is_impossible': False}, {'question': 'What was mortality in  South Korea from MERS disease?', 'id': 4227, 'answers': [{'text': 'ranged from 7 % among younger age groups to 40 % among those aged 60 years and above', 'answer_start': 8955}], 'is_impossible': False}, {'question': 'Which are the preferred method for MERS-CoV detection?', 'id': 4228, 'answers': [{'text': ' PCR-based techniques ', 'answer_start': 12007}], 'is_impossible': False}, {'question': 'What  have become a key diagnostic and taxonomic target for CoV species identification?', 'id': 4229, 'answers': [{'text': 'The first open reading frames ', 'answer_start': 12079}], 'is_impossible': False}, {'question': 'Why  it can be concluded that MERS-COV is a novel and distinct virus?', 'id': 4230, 'answers': [{'text': ' less than 80 % identity between the amino acid sequence of MERS ORF 1ab and betacoronavirus relatives, Tylonycteris bat HKU4 and Pipistrellus bat HKU5', 'answer_start': 12220}], 'is_impossible': False}, {'question': 'What  indicates the likely presence of infectious virus?', 'id': 4231, 'answers': [{'text': ' Detection of viral proteins rather than viral RNA', 'answer_start': 13491}], 'is_impossible': False}, {'question': 'What is the sensitivity with which immunochromatographic tool could detect recombinant MERS-CoV nucleocapsid protein?', 'id': 4232, 'answers': [{'text': '94 %', 'answer_start': 13726}], 'is_impossible': False}, {'question': 'What is the specificity with which immunochromatographic tool could detect recombinant MERS-CoV nucleocapsid protein?', 'id': 4233, 'answers': [{'text': '100 % ', 'answer_start': 13747}], 'is_impossible': False}, {'question': 'What is a different approach for the detection?', 'id': 4234, 'answers': [{'text': 'monoclonal antibody-based capture ELISA targeting the MERS-CoV nucleocapsid protein', 'answer_start': 13821}], 'is_impossible': False}, {'question': 'What is usual in serology testing?', 'id': 4235, 'answers': [{'text': ' to detect a viral footprint, in the form of antibodies', 'answer_start': 14326}], 'is_impossible': False}, {'question': 'How have some sero-assays bypassed the risks of working with infectious virus?', 'id': 4238, 'answers': [{'text': ' by creating transfected cells expressing recombinant portions of the MERS-CoV nucleocapsid and spike proteins [', 'answer_start': 17683}], 'is_impossible': False}, {'question': 'How have some sero-assays bypassed the risks of working with infectious virus?', 'id': 4239, 'answers': [{'text': 'using a recombinant lentivirus expressing MERS-CoV spike protein and luciferase', 'answer_start': 17808}], 'is_impossible': False}, {'question': 'What is the  detection of MERS-CoV infection using ELISA or S1 subunit protein microarray [84] is usually followed by?', 'id': 4240, 'answers': [{'text': 'by confirmatory IFA and/ or a plaque-reduction neutralization (PRNT) [69, 70, 85] or MNT test.', 'answer_start': 20337}], 'is_impossible': False}, {'question': 'What does the confirmatory process aim to  ensure?', 'id': 4241, 'answers': [{'text': 'the antibodies detected are able to specifically neutralize the intended virus and are not more broadly reactive to other coronaviruses found in DCs (bovine CoV, BCoV) or humans (HCoV-OC43, HCoV-229E, HCoV-NL63, HCoV-HKU1, SARS-CoV).', 'answer_start': 20485}], 'is_impossible': False}, {'question': 'When does  generally MERS infection does not trigger a detectable immune response?', 'id': 4242, 'answers': [{'text': 'if only mild disease or asymptomatic infection results', 'answer_start': 21425}], 'is_impossible': False}, {'question': 'When does the WHO recommend samlinf from the LRT?', 'id': 4243, 'answers': [{'text': ' when sample collection is delayed by a week or more after onset of symptoms', 'answer_start': 22029}], 'is_impossible': False}, {'question': 'What do the recommended samples include?', 'id': 4244, 'answers': [{'text': 'are recommended when URT sampling is to be conducted', 'answer_start': 22787}], 'is_impossible': False}, {'question': 'What are recommended when URT sampling is to  be conducted?', 'id': 4245, 'answers': [{'text': ' an oropharyngeal and throat swab or a nasopharyngeal aspirate/wash', 'answer_start': 22719}], 'is_impossible': False}, {'question': 'What paired sera are preferable?', 'id': 4246, 'answers': [{'text': ' collected two to three weeks apart', 'answer_start': 22859}], 'is_impossible': False}, {'question': ' When is a single sample  suggested to be sufficient?', 'id': 4247, 'answers': [{'text': 'Human urine and stool have been found to contain MERS-CoV RNA', 'answer_start': 23122}], 'is_impossible': False}, {'question': 'How long human urine and stool have been found to contain MERS-CoV RNA?', 'id': 4248, 'answers': [{'text': '12 to 26 days after symptom onset ', 'answer_start': 23184}], 'is_impossible': False}, {'question': 'What do individual studies report on viral  shedding?', 'id': 4249, 'answers': [{'text': ' long periods of viral shedding, sometimes intermittently and not necessarily linked to the presence of disease symptoms.', 'answer_start': 25113}], 'is_impossible': False}, {'question': 'What fraction of MERS cases shed viral RNA in their LRT specimens (tracheal aspirates and sputum) ?', 'id': 4262, 'answers': [{'text': 'Over three quarters of', 'answer_start': 25434}], 'is_impossible': False}, {'question': 'How long MERS cases shed viral RNA in their LRT specimens (tracheal aspirates and sputum)? ', 'id': 4263, 'answers': [{'text': 'at least 30 days', 'answer_start': 25542}], 'is_impossible': False}, {'question': 'What percentage of contacts were still shedding RNA in their URT specimens?', 'id': 4264, 'answers': [{'text': '30 %', 'answer_start': 25571}], 'is_impossible': False}, {'question': 'What  samples  returned the highest MERS viral load values?', 'id': 4265, 'answers': [{'text': ' the proportion of deaths among those infected with MERS-CoV is much higher than that known for the HCoVs NL63, HKU1, 229E or', 'answer_start': 29725}], 'is_impossible': False}, {'question': 'What is NPA?', 'id': 4266, 'answers': [{'text': ' nasopharyngeal aspirates', 'answer_start': 25728}], 'is_impossible': False}, {'question': 'What is  NPA?', 'id': 4267, 'answers': [{'text': 'URT sample', 'answer_start': 25763}], 'is_impossible': False}, {'question': 'What is BAL?', 'id': 4268, 'answers': [{'text': 'bronchoalveolar lavage', 'answer_start': 22289}], 'is_impossible': False}, {'question': 'What is the the proportion of deaths among those infected with MERS-CoV?', 'id': 4273, 'answers': [{'text': ' is much higher than that known for the HCoVs NL63, HKU1, 229E or OC43', 'answer_start': 29785}], 'is_impossible': False}, {'question': 'What is the the proportion of deaths among those infected with MERS-CoV?', 'id': 4274, 'answers': [{'text': ' above that for SARS-CoV', 'answer_start': 29884}], 'is_impossible': False}, {'question': 'What   has prevented worldwide spread of MERS-COV?', 'id': 4275, 'answers': [{'text': 'the low transmission rate', 'answer_start': 29995}], 'is_impossible': False}, {'question': 'What must be reported to the world organization for animal health as an emerging disease ?', 'id': 4276, 'answers': [{'text': 'animal MERS-CoV infections', 'answer_start': 30299}], 'is_impossible': False}, {'question': 'What are juvenile DCs more often positive for?', 'id': 4277, 'answers': [{'text': 'virus or viral RNA', 'answer_start': 38778}], 'is_impossible': False}, {'question': 'What are older DCs are more likely to be positive for?', 'id': 4278, 'answers': [{'text': 'seropositive and RNA or virus negative', 'answer_start': 38835}], 'is_impossible': False}, {'question': 'When does the camel calving season?', 'id': 4279, 'answers': [{'text': 't', 'answer_start': 41948}], 'is_impossible': False}, {'question': 'Why is there an increased risk to humans of spill-over during calving season?', 'id': 4280, 'answers': [{'text': 'due to new infections among naïve DC populations', 'answer_start': 42006}], 'is_impossible': False}, {'question': 'Which may be an occupational group with significantly higher incidence of seropositivity to MERS-CoV ?', 'id': 4281, 'answers': [{'text': 'slaughterhouse workers who kill both younger and older DCs', 'answer_start': 42443}], 'is_impossible': False}, {'question': 'How long after Infectious MERS-CoV added to DC, goat or cow milk and stored at 4°C could be recovered?', 'id': 4282, 'answers': [{'text': ' up to 48 h', 'answer_start': 43372}], 'is_impossible': False}, {'question': 'How long after Infectious MERS-CoV added to DC, goat or cow milk and stored at 22°C could be recovered?', 'id': 4283, 'answers': [{'text': 'up to 48 h', 'answer_start': 43373}], 'is_impossible': False}, {'question': 'What ablated MERS-COV infection completely?', 'id': 4284, 'answers': [{'text': 'pasteurization', 'answer_start': 43463}], 'is_impossible': False}, {'question': 'How long MERS-CoV remained viable at high ambient temperature (30°C) and low RH (30 %) ?', 'id': 4285, 'answers': [{'text': 'for 24 h', 'answer_start': 43986}], 'is_impossible': False}, {'question': 'How long can pathogenic bacteria remain viable and airborne in a coughed aerosol?', 'id': 4286, 'answers': [{'text': ' for 45 min', 'answer_start': 44465}], 'is_impossible': False}, {'question': 'How far can pathogenic bacteria spread in a coughed aerosol?', 'id': 4287, 'answers': [{'text': '4 m', 'answer_start': 44513}], 'is_impossible': False}, {'question': 'What is considered the mechanism of human-to-human transmission of MERS-COV?', 'id': 4288, 'answers': [{'text': 'Droplet spread between humans', 'answer_start': 45254}], 'is_impossible': False}, {'question': 'What is the transmission of MERS-CoV is defined as?', 'id': 4289, 'answers': [{'text': 'sporadic ', 'answer_start': 46287}], 'is_impossible': False}, {'question': 'What is the transmission of MERS-CoV is defined as?', 'id': 4290, 'answers': [{'text': 'intra-familial,', 'answer_start': 46313}], 'is_impossible': False}, {'question': 'What is the transmission of MERS-CoV is defined as?', 'id': 4291, 'answers': [{'text': 'often healthcare associated', 'answer_start': 46329}], 'is_impossible': False}, {'question': 'What is the transmission of MERS-CoV is defined as?', 'id': 4292, 'answers': [{'text': 'inefficient ', 'answer_start': 46358}], 'is_impossible': False}, {'question': 'What is the transmission of MERS-CoV is defined as?', 'id': 4293, 'answers': [{'text': 'requiring close and prolonged contact', 'answer_start': 46374}], 'is_impossible': False}, {'question': 'What is the  the rate of general transmission, even in outbreaks?', 'id': 4294, 'answers': [{'text': 'readily transmit to more than one other human', 'answer_start': 46742}], 'is_impossible': False}, {'question': 'What do the majority of human cases of MERS-CoV  seem not to do?', 'id': 4296, 'answers': [{'text': 'readily transmit to more than one other human ', 'answer_start': 46742}], 'is_impossible': False}, {'question': 'What is the basic reproduction number (R 0)?', 'id': 4298, 'answers': [{'text': 'the average number of infections caused by one infected individual in a fully susceptible population', 'answer_start': 46950}], 'is_impossible': False}, {'question': 'What is the basic reproduction number (R 0 )  for MERS-COV?', 'id': 4299, 'answers': [{'text': 'close to one', 'answer_start': 47059}], 'is_impossible': False}, {'question': 'Why has MERS had a constant presence in the Arabian Peninsula?', 'id': 4300, 'answers': [{'text': 'due to ongoing, sporadic spill-over events from DCs amplified by poorly controlled hospital outbreaks.\\n', 'answer_start': 47402}], 'is_impossible': False}, {'question': 'What was the first known MERS human-to-human transmission event was one characterized by?', 'id': 4301, 'answers': [{'text': 'acute LRT disease', 'answer_start': 47586}], 'is_impossible': False}, {'question': 'Where was the first known MERS human-to-human transmission event?', 'id': 4302, 'answers': [{'text': 'in a healthcare setting in Jordan', 'answer_start': 47604}], 'is_impossible': False}, {'question': 'Approximately what percentage of MERS cases were fatal in  KSA?', 'id': 4367, 'answers': [{'text': '21 %', 'answer_start': 49531}], 'is_impossible': False}, {'question': 'Approximately what percentage of MERS cases were died outside KSA?', 'id': 4368, 'answers': [{'text': '21 %', 'answer_start': 49531}], 'is_impossible': False}, {'question': 'What percentage of HCWs comprised of MERS cases in the KSA and South Korea?', 'id': 4369, 'answers': [{'text': '16 %', 'answer_start': 50413}], 'is_impossible': False}, {'question': 'How has most of the analysis of MERS-CoV genetics  been performed?', 'id': 4370, 'answers': [{'text': 'using high throughput or \"deep\" sequencing methods for complete genome deduction', 'answer_start': 51231}], 'is_impossible': False}, {'question': 'What does Clade A contain?', 'id': 4372, 'answers': [{'text': 'only human-derived MERS-CoV genomes', 'answer_start': 52035}], 'is_impossible': False}, {'question': 'What does clade B comprise?', 'id': 4373, 'answers': [{'text': 'the majority of human and camel genomes deduced thus far', 'answer_start': 52108}], 'is_impossible': False}, {'question': 'How many clades have become apparent in genome of  MERS-COV from humans and DCs?', 'id': 4371, 'answers': [{'text': 'has had more MERS cases than any other region of the KSA', 'answer_start': 58320}], 'is_impossible': False}, {'question': 'Which city has had has had more MERS cases than any other region of the KSA ?', 'id': 4374, 'answers': [{'text': 'Riyadh', 'answer_start': 58271}], 'is_impossible': False}, {'question': 'Which city harbours a wide range of MERS-CoV variants ', 'id': 4375, 'answers': [{'text': ' Riyadh', 'answer_start': 58270}], 'is_impossible': False}, {'question': 'How has the vast majority of MERS-CoV transmission occurred?', 'id': 4376, 'answers': [{'text': 'from infected to uninfected humans in close and prolonged contact ', 'answer_start': 66077}], 'is_impossible': False}, {'question': 'How were the transmission circumstances created?', 'id': 4377, 'answers': [{'text': 'by poor infection control in health care settings', 'answer_start': 66173}], 'is_impossible': False}, {'question': 'What percentage of humans have died  among all humans reported to be infected?', 'id': 4378, 'answers': [{'text': ' nearly 40 %', 'answer_start': 67310}], 'is_impossible': False}, {'question': 'What would  aid accurate calculation of a case fatality ratio?', 'id': 4379, 'answers': [{'text': 'Global alignment of case definitions', 'answer_start': 67480}], 'is_impossible': False}, {'question': 'How does MARS-COV differ from SARS-COV?', 'id': 4380, 'answers': [{'text': 'MERS-CoV has a broader tropism, grows more rapidly in vitro, more rapidly induces cytopathogenic change, triggers distinct transcriptional responses, makes use of a different receptor, induces a more proinflammatory state and has a delayed innate antiviral response compared to SARS-CoV.\\n', 'answer_start': 68137}], 'is_impossible': False}, {'question': 'Is there any evidence that MERS-CoV is a virus of pandemic concern?', 'id': 4381, 'answers': [{'text': 'There is no evidence that MERS-CoV is a virus of pandemic concern', 'answer_start': 69043}], 'is_impossible': False}, {'question': 'How did the first WHO case definition  define probable cases of MERS?', 'id': 4215, 'answers': [{'text': 'based on the presence of febrile illness, cough and requirement for hospitalization with suspicion of lower respiratory tract (LRT) involvement.', 'answer_start': 6732}], 'is_impossible': False}], 'context': 'MERS coronavirus: diagnostics, epidemiology and transmission\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687373/\\n\\nSHA: f6fcf1a99cbd073c5821d1c4ffa3f2c6daf8ae29\\n\\nAuthors: Mackay, Ian M.; Arden, Katherine E.\\nDate: 2015-12-22\\nDOI: 10.1186/s12985-015-0439-5\\nLicense: cc-by\\n\\nAbstract: The first known cases of Middle East respiratory syndrome (MERS), associated with infection by a novel coronavirus (CoV), occurred in 2012 in Jordan but were reported retrospectively. The case first to be publicly reported was from Jeddah, in the Kingdom of Saudi Arabia (KSA). Since then, MERS-CoV sequences have been found in a bat and in many dromedary camels (DC). MERS-CoV is enzootic in DC across the Arabian Peninsula and in parts of Africa, causing mild upper respiratory tract illness in its camel reservoir and sporadic, but relatively rare human infections. Precisely how virus transmits to humans remains unknown but close and lengthy exposure appears to be a requirement. The KSA is the focal point of MERS, with the majority of human cases. In humans, MERS is mostly known as a lower respiratory tract (LRT) disease involving fever, cough, breathing difficulties and pneumonia that may progress to acute respiratory distress syndrome, multiorgan failure and death in 20 % to 40 % of those infected. However, MERS-CoV has also been detected in mild and influenza-like illnesses and in those with no signs or symptoms. Older males most obviously suffer severe disease and MERS patients often have comorbidities. Compared to severe acute respiratory syndrome (SARS), another sometimes- fatal zoonotic coronavirus disease that has since disappeared, MERS progresses more rapidly to respiratory failure and acute kidney injury (it also has an affinity for growth in kidney cells under laboratory conditions), is more frequently reported in patients with underlying disease and is more often fatal. Most human cases of MERS have been linked to lapses in infection prevention and control (IPC) in healthcare settings, with approximately 20 % of all virus detections reported among healthcare workers (HCWs) and higher exposures in those with occupations that bring them into close contact with camels. Sero-surveys have found widespread evidence of past infection in adult camels and limited past exposure among humans. Sensitive, validated reverse transcriptase real-time polymerase chain reaction (RT-rtPCR)-based diagnostics have been available almost from the start of the emergence of MERS. While the basic virology of MERS-CoV has advanced over the past three years, understanding of the interplay between camel, environment, and human remains limited. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (doi:10.1186/s12985-015-0439-5) contains supplementary material, which is available to authorized users.\\n\\nText: An email from Dr Ali Mohamed Zaki, an Egyptian virologist working at the Dr Soliman Fakeeh Hospital in Jeddah in the Kingdom of Saudi Arabia (KSA) announced the first culture of a new coronavirus to the world. The email was published on the website of the professional emerging diseases (ProMED) network on 20 th September 2012 [1] (Fig. 1) and described the first reported case, a 60 year old man from Bisha in the KSA. This information led to the rapid discovery of a second case of the virus, this time in an ill patient in the United Kingdom, who had been transferred from Qatar for care [2] . The new virus was initially called novel coronavirus (nCoV) and subsequentlty entitled the Middle East respiratoy syndrome coronavirus (MERS-CoV). As of 2 nd of September 2015, there have been 1,493 detections of viral RNA or virus-specific antibodies across 26 countries (Additional file 1: Figure S1 ) confirmed by the World Health Organization (WHO), with over a third of the positive people dying (at least 527, 35 %) [3] .\\n\\nSince that first report, a slow discovery process over the following two to three years revealed a virus that had infected over 90 % of adult dromedary camels (DC; Camelus dromedarius) in the KSA [4] , also DCs across the Arabian Peninsula and parts of Africa that are a source of DC imports for the KSA [5] . To date, MERS-CoV has not been detected in DCs tested in zoos or herds from other parts of the world [6] [7] [8] [9] . Occasionally, virus is transmitted from infected DCs to exposed humans. Subsequent transmission to other humans requires relatively close and prolonged exposure [10] .\\n\\nThe first viral isolate was patented and concerns were raised that this would restrict access to both the virus and to viral diagnostics [11, 12] . However, sensitive, validated reverse transcriptase real-time polymerase chain reaction (RT-rtPCR)-based diagnostics were quickly described and virus was made freely available subject to routine biosafety considerations [13] . Subsequent epidemiology and research has identified the cell receptor as exopeptidase dipeptidyl peptidase 4 (DPP4; also called CD26); that MERS-CoV has a broad tropism, replicating better in some cells lines and eliciting a more proinflammatory response than SARS-CoV; is widespread in DCs; has the potential to infect other animals and that MERS kills its human host more often than SARS did (20-40 % versus 9 % for SARS [14] ) [15] [16] [17] [18] [19] .\\n\\nIn humans, overt disease was given the name Middle East respiratory syndrome, with the acronym MERS. From intermittent animal-to-human spill-over events, the MERS-CoV spreads sporadically among people, causing more severe disease among older adults, especially males, with pre-existing diseases. The spread of MERS-CoV among humans has often been associated with outbreaks in hospitals, with around 20 % of all cases to date involving healthcare workers (HCWs).\\n\\nAlthough DCs appear to suffer the equivalent of a \\'common cold\\' from MERS-CoV infection, in humans, the virus can be a more serious and opportunistic pathogen associated with the death of up to 40 % of reported cases. It has yet to be established whether infections thought to have been acquired from an animal source produce a more severe outcome than those spread between humans [20] . Studies have established that the mean incubation period for MERS is five to six days, ranging from two to 16 days, with 13 to 14 days between when illness begins in one person and subsequently spreads to another [21] [22] [23] [24] . Among those with progressive illness, the median time to death is 11 to 13 days, ranging from five to 27 days [23, 24] . Fever and gastrointestinal symptoms may form a prodrome, after which symptoms decline, only to be followed by a more severe systemic and respiratory syndrome [25, 26] .\\n\\nThe first WHO case definition [27] defined probable cases of MERS based on the presence of febrile illness, cough and requirement for hospitalization with suspicion of lower respiratory tract (LRT) involvement. It also included roles for contact with a probable or confirmed case or for travel or residence within the Arabian Peninsula. If strictly adhered to, only the severe syndrome would be subject to laboratory testing, which was the paradigm early on [21] . From July 2013, the revised WHO case definition included the importance of seeking out and understanding the role of asymptomatic cases and from June 2014, the WHO definition more clearly stated that a confirmed case included any person whose sample was RT-PCR positive for MERS-CoV, or who produced a seroconversion, irrespective of clinical signs and symptoms. [28] [29] [30] Apart from the WHO and the KSA Ministry of Health reports, asymptomatic or subclinical cases of MERS-CoV infection were documented in the scientific literature although not always as often as occurred early on [31, 32] . The KSA definition of a case became more strict on 13 th May 2014, relying on the presence of both clinical features and laboratory confirmation [33] . Testing of asymptomatic people was recommended against from December 2014 [34] , reinforced by a case definition released by the KSA Ministry of Health in June 2015 [35] . The KSA has been the source of 79 % of human cases. Severe MERS is notable for its impact among older men with comorbid diseases including diabetes mellitus, cirrhosis and various lung, renal and cardiac conditions [36] [37] [38] . Interestingly in June 2015, an outbreak in South Korea followed a similar distribution [39, 40] . Among laboratory confirmed cases, fever, cough and upper respiratory tract (URT) signs and symptoms usually occur first, followed within a week by progressive LRT distress and lymphopaenia [37] . Patients often present to a hospital with pneumonia, or worse, and secondary bacterial infections have been reported [37, 41] . Disease can progress to acute respiratory distress syndrome and multiorgan system failure [37] . MERS has reportedly killed approximately 35 % of all reported cases, 42 % of cases in the KSA, yet only 19 % of cases in South Korea, where mortality ranged from 7 % among younger age groups to 40 % among those aged 60 years and above [42] ; all may be inflated values with asymptomatic or mild infections sometimes not sought or not reported [34] . General supportive care is key to managing severe cases [43] . Children under the age of 14 years are rarely reported to be positive for MERS-CoV, comprising only 1.1 % (n = 16) of total reported cases. Between 1 st September 2012 and 2 nd December 2013, a study described the then tally of paediatric cases in the KSA, which stood at 11 (two to 16 years of age; median 13 years); nine were asymptomatic (72 %) and one infant died [44] . In Amman, Jordan, 1,005 samples from hospitalized children under the age of two years with fever and/or respiratory signs and symptoms were tested but none were positive for MERS-CoV RNA, despite being collected at a similar time to the first known outbreak of MERS-CoV in the neighbouring town of Al-Zarqa [45] . A second trimester stillbirth occurred in a pregnant woman during an acute respiratory illness and while not RT-rtPCR positive, the mother did subsequently develop antibodies to MERS-CoV, suggestive of recent infection [46] . Her exposure history to a MERS-CoV RT-rtPCR positive relative and an antibody-reactive husband, her incubation period and her symptom history met the WHO criteria for being a probable MERS-CoV case [46] .\\n\\nDiagnostic methods were published within days of the ProMED email announcing the first MERS case [47] , including several now gold standard in-house RT-rtPCR assays (Fig. 2 ) as well as virus culture in Vero and LLC-MK2 cells [18, 47, 48] . A colorectal adenocarcinoma (Caco-2) epithelial cell line has since been recommended for isolation of infections MERS-CoV [49] . We previously [18] .). Open reading frames are indicated as yellow rectangles bracketed by terminal untranslated regions (UTR; grey rectangles). FS-frame-shift. Predicted regions encompassing recombination break-points are indicated by orange pills. Created using Geneious v8.1 [211] and annotated using Adobe Illustrator. Beneath this is a schematic depicting the location of RT-PCR primers (blue arrows indicate direction) and oligoprobes (green rectangles) used in the earliest RT-rtPCR screening assays and conventional, semi-nested (three primers) RT-PCR confirmatory sequencing assays [47, 48] . Publication order is noted by first [27 th September 2012; red] and second [6 th December 2012; orange] coloured rectangles; both from Corman et al. [47, 48] Those assays recommended by the WHO are highlighted underneath by yellow dots [53] . The NSeq reverse primer has consistently contained one sequence mismatch with some MERS-CoV variants. An altered version of that from Mackay IM, Arden KE. Middle East respiratory syndrome: An emerging coronavirus infection tracked by the crowd. Virus Res 2015 Vol 202:60-88 with permission from Elsevier [5] reviewed the broad tropism of MERS-CoV [5] . However, as is well described, cell culture is a slow, specialised and insensitive method [50] while PCR-based techniques are the preferred method for MERS-CoV detection.\\n\\nThe first open reading frames (ORF 1a and 1b; Fig. 2 ) have become a key diagnostic and taxonomic target for CoV species identification. With less than 80 % identity between the amino acid sequence of MERS ORF 1ab and betacoronavirus relatives, Tylonycteris bat HKU4 and Pipistrellus bat HKU5, it can be concluded that it is a novel and distinct virus. MERS-CoV is predicted to encode ten open reading frames with 5\\' and 3\\' untranslated regions [51] . The structural proteins include the spike (S), envelope (E), membrane (M) and nucleocapsid (N) [52] . The products of ORF1a and ORF1b are predicted to encode nonstructural proteins.\\n\\nThe majority of specimen testing to date has employed validated RT-rtPCR assays shown to be sensitive and specific [47, 48, 53] . The RealStar® kit uses these WHOrecommended assays [54] . The target sequences of these screening assays have not changed among genomes examined until at least mid-2015 (IMM observation). Other RT-rtPCR assays have been developed and validated for use as laboratory-based diagnostic tools [55] [56] [57] . Additionally, loop-mediated [58, 59] or recombinase polymerase [60] isothermal assays have been designed for field deployment.\\n\\nThe detection of MERS-CoV antigen has not been common to date but the combination of short turnaround time from test to result, high throughput and identification of viral proteins makes this an attractive option. Detection of viral proteins rather than viral RNA indicates the likely presence of infectious virus. The first rapid immunochromatographic tool described could detect recombinant MERS-CoV nucleocapsid protein from DC nasal swabs with 94 % sensitivity and 100 % specificity compared to RT-rtPCR [61] . A different approach used a monoclonal antibody-based capture ELISA targeting the MERS-CoV nucleocapsid protein with a sensitivity of 10 3 TCID 50 and 100 % specificity [62] .\\n\\nDemonstration of a seroconversion to a MERS-CoV infection meets the current WHO definition of a case so optimized and thoroughly validated sero-assays employed alongside good clinical histories are useful to both identify prior MERS-CoV infection and help support transmission studies. Because serology testing is, by its nature, retrospective, it is usual to detect a viral footprint, in the form of antibodies, in the absence of any signs or symptoms of disease and often in the absence of any viral RNA [63] .\\n\\nStrategic, widespread sero-surveys of humans using samples collected after 2012 are infrequent. Much of the Arabian Peninsula and all of the Horn of Africa lack baseline data describing the proportion of the community who may have been infected by a MERS-CoV. However, sero-surveys have had widespread use in elucidating the role of DCs as a transmission source for MERS-CoV. Because of the identity shared between DC and human MERS-CoV (see Molecular epidemiology: using genomes to understand outbreaks), serological assays for DC sero-surveys should be transferrable to human screening with minimal re-configuration. Also, no diagnostically relevant variation in neutralization activity have been found from among a range of circulating tested MERS-CoV isolates and sera, so whole virus or specific protein-based sero-assays should perform equivalently in detecting serological responses to the single MERS-CoV serotype [49] . The development of robust serological assays requires reliable panels of wellcharacterized animal or human sera, including those positive for antibodies specific to MERS-CoV, as well as to likely sources of cross-reaction [64] . Obtaining these materials was problematic and slowed the development and commercialization of antibody detection assays for human testing [64] . A number of commercial ELISA kits, immunofluorescent assays (IFA) kits, recombinant proteins and monoclonal antibodies have been released [31, [65] [66] [67] [68] . Initially, conventional IFAs were used for human sero-surveys. These relied on MERS-CoV-infected cell culture as an antigen source, detecting the presence of human anti-MERS-CoV IgG, IgM or neutralizing antibodies in human samples [18, 48, 69] . No sign of MERS-CoV antibodies was found among 2,400 sera from patients visiting Hospital in Jeddah, from 2010 through 2012, prior to the description of MERS-CoV [18] . Nor did IFA methods detect any sign of prior MERS-CoV infection among a small sample of 130 healthy blood donors from another Hospital in Jeddah (collected between Jan and Dec 2012) [70] . Of 226 slaughterhouse workers, only eight (3.5 %) were positive by IFA, and those sera could not be confirmed by virus neutralization (NT) test. The study indicated that HCoV-HKU1 was a likely source of crossreactive antigen in the whole virus IFA [70] . Whole virus MERS-CoV IFA also suffered from some cross-reactivity with convalescent SARS patient sera and this could not be resolved by an NT test which was also cross-reactive [71] . IFA using recombinant proteins instead of whole-virus IFA, has been shown to be a more specific tool [31] . Since asymptomatic zoonoses have been posited [72] , an absence of antibodies to MERS-CoV among some humans who have regular and close contact with camels may reflect the rarity of actively infected animals at butcheries, a limited transmission risk associated with slaughtering DCs [70] , a pre-existing cross-protective immune status or some other factor(s) resulting in a low risk of disease and concurrent seroconversion developing after exposure in this group. IFA using recombinant proteins instead.\\n\\nSome sero-assays have bypassed the risks of working with infectious virus by creating transfected cells expressing recombinant portions of the MERS-CoV nucleocapsid and spike proteins [48, 73] , or using a recombinant lentivirus expressing MERS-CoV spike protein and luciferase [74, 75] . A pseudo particle neutralization (ppNT) assay has seen widespread used in animal studies and was at least as sensitive as the traditional microneutralization (MNT) test. [10, 74, [76] [77] [78] ] Studies using small sample numbers and ppNT found no evidence of MERS-CoV neutralizing antibody in sera from 158 children with LRT infections between May 2010 and May 2011, 110 sera from 19 to 52 year old male blood donors and 300 selfidentified animal workers from the Jazan Region of the KSA during 2012 [79, 80] . Similarly, a study of four herdsmen in contact with an infected DC herd in Al-Ahsa, eight people who had intermittent contact with the herd, 30 veterinary surgeons and support staff who were not exposed to the herd, three unprotected abattoir workers in Al-Ahsa and 146 controls who were not exposed to DCs in any professional role, found none with serological evidence of past MERS-CoV infection using the ppNT assay [10] . A delay in the neutralizing antibody response to MERS-CoV infection was associated with increased disease severity in South Korea cases with most responses detectable by week three of illness while others, even though disease was severe, did not respond for four or more weeks [81] . The implications for our ability to detect any response in mild or asymptomatic cases was not explored but may be a signifcant factor in understanding exposure in the wider community.\\n\\nA Jordanian outbreak of acute LRT disease in a hospital in 2012 was retrospectively found to be associated with MERS-CoV infection, initially using RT-rtPCR, but subsequently, and on a larger scale, through positivity by ELISA and IFA or MNT test. [46, 82, 83] This outbreak predated the first case of MERS in the KSA. The ELISA used a recombinant nucleocapsid protein from the group 2 betacoronavirus bat-CoV HKU5 to identify antibodies against the equivalent crossreactive MERS-CoV protein [71] . It was validated using 545 sera collected from people with prior HCoV-OC43, HCoV-229E, SARS-CoV, HCoV-NL63, HRV, HMPV or influenza A(H1N1) infections but was reportedly less specific than the recombinant IFA discussed above. It was still considered an applicable tool for screening large sample numbers [82] . A protein microarray expressing the S1 protein subunit has also been validated and widely used for DC testing [5, 84] . Detection of MERS-CoV infection using ELISA or S1 subunit protein microarray [84] is usually followed by confirmatory IFA and/ or a plaque-reduction neutralization (PRNT) [69, 70, 85] or MNT test. [74, 85, 86] This confirmatory process aims toensure the antibodies detected are able to specifically neutralize the intended virus and are not more broadly reactive to other coronaviruses found in DCs (bovine CoV, BCoV) or humans (HCoV-OC43, HCoV-229E, HCoV-NL63, HCoV-HKU1, SARS-CoV). In the largest study of human sera, a tiered diagnostic process assigned both recombinant IFA and recombinant ELISA positive sera to \\'stage 1\\' seropositivity. A stage 2 seropositive result additionally required a suitably titred PRNT result [87] . The study found 15 sera collected in 2012 to 2013 from 10,009 (0.2 %) people in 13 KSA provinces contained MERS-CoV antibodies, but significantly higher proportions in occurred in camel shepherds (two of 87; 2.3 %) and slaughterhouse workers (five of 140; 3.6 %) [87] . Contemporary surveys are needed.\\n\\nMERS-CoV does not appear to be easily transmitted from DCs to humans, or perhaps it is [72] , but generally does not trigger a detectable immune response if only mild disease or asymptomatic infection results. Serology assays are in need of further validation in this area so care is required when moving newly developed diagnostic serology algorithms from a research setting to one that informs public health decisions. This was reinforced when a false positive US case, purported to have been infected after a handshake and two face-to-face meetings, did not withstand further confirmatory analysis using a more specific, NT assay and was subsequently retracted [88, 89] .\\n\\nThe WHO recommends sampling from the LRT for MERS-CoV RT-rtPCR testing, especially when sample collection is delayed by a week or more after onset of symptoms. [53] LRT samples are also best for attempting isolation of infectious virus, although the success of culture is reduced when disease persists [49] . Recommended sample types include bronchoalveolar lavage (BAL), tracheal/tracheobronchial aspirate, pleural fluid and sputum [53, 90] . Fresh samples yield better diagnostic results than refrigerated material [69] and if delays in testing of ≥72 h are likely, samples (except for blood) should be frozen at −70°C [90] . If available, lung biopsy or autopsy tissues can also be tested [53] . The URT is a less invasive and more convenient sampling site however, and an oropharyngeal and throat swab or a nasopharyngeal aspirate/wash are recommended when URT sampling is to be conducted [90] . Paired sera, collected two to three weeks apart are preferable for serological testing while a single sample is suggested to be sufficient if collected two weeks after onset of disease or a single serum collected during the first 10-12 days if conducting RT-rtPCR [53, 90] . Human urine and stool have been found to contain MERS-CoV RNA 12 to 26 days after symptom onset [25, 69, 91] and are listed as samples that should be considered [53, 90] . In two cases that arrived in the Netherlands, urine was RT-rtPCR negative but faeces was weakly positive and sera were RT-rtPCR positive for five days or more [25] . The finding of MERS-CoV viral RNA in serum provides an avenue for retrospective PCR-based studies if respiratory samples are unavailable [83] . RNAaemia may also correlate with disease severity; signs of virus were cleared from the serum of a recovered patient, yet lingered until the death of another [92] .\\n\\nClinically suspected MERS cases may return negative results by RT-rtPCR. Data have shown one or more negative URT samples may be contradicted by further URT sampling or the use of LRT samples, which is preferred [2, 43, 93] . Higher viral loads occur in the LRT compared to the URT. [22, 69, 88, 94] This fits with the observation that the majority of disease symptoms are reported to manifest as systemic and LRT disease [21] . However, on occasion, even LRT specimens from MERS cases may initially be negative, only to later become positive by RT-PCR [95] . This may be due to poor sampling when a cough is absent or non-productive or because the viral load is low [95] . Despite this both the largest human MERS-CoV studies [32, [96] [97] [98] and smaller ones [22, 25, 99] , use samples from the URT. It is then noteworthy that one study reported an association between higher loads in the URT and worse clinical outcome including intensive care and death [94] . At writing, no human data exist to define whether the virus replicates solely or preferentially in the LRT or URT, or replicates in other human tissues in vivo although MERS-CoV RNA has been detected from both the URT and LRT in a macaque monkey model [100] .The distribution of DPP4 in the human upper airways is also not well described.\\n\\nIndividual human case studies report long periods of viral shedding, sometimes intermittently and not necessarily linked to the presence of disease symptoms. [25, 69, 99, 101] In one instance, a HCW shed viral RNA for 42 days in the absence of disease [99] . It is an area of high priority to better understand whether such cases are able to infect others. Over three quarters of MERS cases shed viral RNA in their LRT specimens (tracheal aspirates and sputum) for at least 30 days, while only 30 % of contacts were still shedding RNA in their URT specimens [91, 102] .\\n\\nIn the only study to examine the effect of sample type on molecular analysis, 64 nasopharyngeal aspirates (NPA; an URT sample), 30 tracheal aspirates, 13 sputa and three BAL were examined. The tracheal aspirates and BAL returned the highest viral load values followed by NPA and sputum. Unsurprisingly, higher viral loads generally paralleled whole genome sequencing and culture success and, in NPA testing, were significantly correlated with severe disease and death [49, 94, 103] . This study demonstrated the importance of LRT sampling for whole genome sequencing.\\n\\nWhen tested, samples positive for MERS-CoV are often negative for other pathogens [2, 25, 93, 104] . However, many studies make no mention of additional testing for endemic human respiratory viruses [21, 23, 73, 105] . When viruses are sought, they have included human herpesvirus (HHV), rhinoviruses (HRV), enteroviruses (EV), respiratory syncytial virus (RSV), parainfluenzavirus types 1, 2 and 3 (PIVs),influenzaviruses (IFVs), endemic HCoVs, adenoviruses (AdVs) metapneumovirus (MPV) and influenza A\\\\H1N1 virus; co-detections with MERS-CoV have been found on occasion [2, 22, 37, 69, 97] . Bacterial testing is sometimes included (for example, for Legionella and Pneumococcus) but the impact of bacterial co-presence is also unclear [22, [104] [105] [106] . Further testing of the LRT sample from the first MERS case used IFA to screen for some viruses (negative for IFV, PIVs, RSV and AdVs) and RT-PCR for others (negative for AdV, EVs, MPV and HHVs) [18] . RT-PCR also detected MERS-CoV. The WHO strongly recommends testing for other respiratory pathogens [53] but with this recommendation often discounted, there are limited data to address the occurrence and impact of co-infections or alternative viral diagnoses among both MERS cases and their contacts. Little is known of other causes of MERS-like pneumonia in the KSA or of the general burden of disease due to the known classical respiratory viruses.\\n\\nTesting of adult pilgrims performing the Hajj in 2012 to 2014 has not detected any MERS-CoV. In 2012, nasal swabs from 154 pilgrims collected prior to leaving for or departing from the KSA were tested [47] . In 2013, testing was significantly scaled up with 5,235 nasopharyngeal swabs from 3,210 incoming pilgrims and 2,025 swabs from outgoing pilgrims tested [98] . It should be noted that most pilgrims arrived from MERS-free countries. A further 114 swabs were taken from pilgrims with influenza-like illness [96, 107] . In earlier Hajj gatherings, it was found that influenza viruses circulated widely, whilst other viruses, often rhinoviruses, circulated more selectively, interpreted as indicating their importation along with foreign pilgrims. [107] [108] [109] Over time, increased influenza vaccination has been credited for a fall in the prevalence of influenza like illnesses among Hajj pilgrims. [110] A LRT sample is often not collected for these studies [98, 107, 109] , so false negative findings are a possibility although little is known about the initial site of MERS-CoV infection and replication; it may have been assumed it was the LRT because disease was first noticed there but the URT may be the site of the earliest replication.\\n\\nIn Jeddah between March and July 2014 (hereafter called the Jeddah-2014 outbreak; Fig. 3 ), there was a rapid increase in MERS cases, accompanied by intense screening; approximately 5,000 samples from in and around the region were tested in a month yielding around 140 MERS-CoV detections (~3 % prevalence) [111] . Among 5,065 individuals sampled and tested across the KSA between October 2012 and September 2013,108 (2.1 %) detections were made in a hospital-centric population which included hospitalized cases (n = 2,908; 57.4 %), their families (n = 462; 9.1 %) and associated HCWs (n = 1,695; 33.5 %) [32] . Among the detections, 19 (17.8 %) were HCWs and 10 (9.3 %) were family contacts [32] .\\n\\nThe 2-3 % prevalence of active MERS-CoV infections is not dissimilar to the hospital-based prevalence of other human CoVs. [112] However, the proportion of deaths among those infected with MERS-CoV is much higher than that known for the HCoVs NL63, HKU1, 229E or OC43 in other countries, and even above that for SARS-CoV; it is not a virus that could reasonably be described as a \"storm in a teacup\". It is the low transmission rate that has prevented worldwide spread, despite many \"opportunities\".\\n\\nVery early in the MERS outbreak, some animals were highly regarded as either the reservoir or intermediate host(s) of MERS-CoV with three of the first five cases having contact with DCs [73, 113, 114] . Today, animal MERS-CoV infections must be reported to the world organization for animal health as an emerging disease [115] . A summary of the first MERS cases reported by the WHO defined animal contact with humans as being direct and within 10 days prior to symptom onset [20] . This definition made no specific allowance for acquisition from DCs through a droplet-based route, which is very likely route for acquisition of a virus that initially and predominantly causes respiratory disease [23] . Camels are known to produce high levels of MERS-CoV RNA in their URT and lungs [116] . Providing support for a droplet transmission route and perhaps indicating the presence of RNA in smaller, drier droplet nuclei, MERS-CoV RNA was identified in a high volume air sample collected from a barn housing an infected DC [117] . The precise source from which humans acquire MERS-CoV remains poorly studied but it seems likely that animal and human behavioural factors may play roles (Fig. 3) [118] . These factors may prove important for human cases who do not describe any DC contact [119] nor any contact with a confirmed case. Whether the WHO definition of animal contact is sufficient to identify exposure to this respiratory virus remains unclear. Wording focuses on consumption of DC products but does not specifically ascribe risk to a droplet route for acquisition of MERS-CoV from DC [120] . Some MERS patients are listed in WHO disease notices as being in proximity to DCs or farms, but the individuals have not described coming into contact with the animals. No alternative path for acquiring infection is reported in many of these instances. What constitutes a definition of \"contact\" during these interviews has been defined for one study [72] . Despite this lack of clarity, the WHO consider that evidence linking MERS-CoV transmission between DCs to humans is irrefutable (Fig. 4) [120] .\\n\\nThe possibility that bats were an animal host of MERS-CoV was initially widely discussed because of the existing diversity of coronaviruses known to reside among them [121] [122] [123] [124] . Conclusive evidence supporting bats as a source for human infections by MERS-CoV has yet to be found, but bats do appear to host ancestral representatives [53, 125] . However, these are not variants of the same virus nor always within the same phylogenetic lineage as MERS-CoV; they are each a genetically distinct virus. Bat-to-human infection by MERS-CoV is a purely speculative event. The only piece of MERS-CoV-specific evidence pointing to bats originates from amplification of a 190 nt fragment of the RNAdependent RNA polymerase gene of the MERS-CoV genome, identified in a faecal pellet from an insectivorous Emballonuridae bat, Taphozous perforatus found in Bisha, the KSA [121] . While very short, the sequence of the fragment defined it as a diagnostic discovery. Subsequently a link to DCs was reported [85] and that link has matured into a verified association [38, 126] (Fig. 4) .\\n\\n(See figure on previous page.) Fig. 3 Monthly detections of MERS-CoV (blue bars) and of cases who died (red bars) with some dates of interest marked for 2012 to 4 th September 2015. An approximation of when DC calving season [128] and when recently born DCs are weaned is indicated. Spring (green) and summer (orange) in the Arabian Peninsula are also shaded. Note the left-hand y-axis scale for 2014 and 2015 which is greater than for 2012/13. Sources of these public data include the WHO, Ministries of Health and FluTrackers [207] [208] [209] . Earlier and subsequent versions of this chart are maintained on a personal blog [210] . Modified and reprinted from Mackay IM, Arden KE. Middle East respiratory syndrome: An emerging coronavirus infection tracked by the crowd. Virus Res 2015 Vol 202:60-88 with permission from Elsevier [5] DCs, which make up 95 % of all camels, have a central presence in the Arabian Peninsula where human-DC contact ranges from little to close [119] . Contact may be commonplace and could occur in variety of ways (Fig. 4a) . There are several large well-attended festivals, races, sales and parades which feature DCs and DCs are also kept and bred close to populated areas in the KSA [127, 128] . DC milk and meat are widely consumed and the older DC is an animal of ritual significance after the Hajj pilgrimage [129] . However, MERS-CoV infection frequency is reportedly much lower than is the widespread and frequent habit of eating, drinking and preparing DC products. Daily ingestion of fresh unpasteurized DC milk is common among the desert Bedouin and many others in the KSA. DC urine is also consumed or used for supposed health benefits. Despite camel butchery being a local occupation, neither butchers nor other at-risk groups are identifiable among MERS cases; this may simply be a reporting issue rather than an unexplainable absence of MERS. A small case-control study published in 2015 identified direct DC contact, and not ingestion of products, to be associated with onset of MERS [38] .\\n\\nThe first sero-survey of livestock living in the Middle East region was conducted during 2012-2013 [85] . DCs were sampled from a mostly Canary Island-born herd and from Omani DCs (originally imported from the Horn of Africa) [85] . A neutralising antibody assay found only 10 % of strongly seropositive Canary Island [5] . b Camel-to-human infections appear to be infrequent, while human-to-human spread of infection is regularly facilitated by poor IPC in healthcare settings where transmission is amplified, accounting for the bulk of cases. There are human MERS cases that do not fall into either category of source and it is unclear if these acquired infection through some entirely separate route, or from cases that escaped diagnosis. c Hypothetical ways in which subclinical (when infection may not meet a previously defined clinical threshold of signs and/or symptoms) or asymptomatic (no obvious signs or measured, noticed or recalled symptoms of illness) MERS-CoV infection may be implicated in transmission DC sera could neutralise MERS-CoV while all Omani DC sera had high levels of specific MERS-CoV neutralizing antibody [85] . This indicated that DCs had in the past been infected by MERS-CoV, or a very similar virus.\\n\\nSince this study, a host of peer-reviewed reports have looked at both DCs and other animals, and the possibility that they may host MERS-CoV infection. Seropositive DCs have been found throughout the Arabian Peninsula including Oman, the KSA, Qatar, Jordan, the United Arab Emirates (UAE), Kuwait as well as Sudan, Somalia, Egypt, Tunisia, Nigeria, Kenya and Ethiopia in Africa and the Canary Islands [85, [130] [131] [132] [133] [134] . Other animals tested include sheep, cows, pigs, horses, donkeys, mules, birds, water buffalo, goats, Bactrian camels, llamas and guanaco (south American camelids) but none had detectable neutralising antibody against MERS-CoV [4, 74, 78, 85, 86, 135, 136] . No virology or serology studies of human samples from areas in Africa where there are camels with a history of MERS-CoV have been reported to date. However,an absence of unexplained pneumonia that may be attributable to MERS-CoV infection may not signal the absence of virus among humans in each country but simply reflect a lack of expensive epidemiology studies conducted by resource-poor countries. It is thus unclear whether MERS-CoV, or an antigenically related CoV, is an unrecognized pathogen in these regions, perhaps circulating for even longer than it has been known in the Arabian Peninsula [133] .\\n\\nMERS-CoV RNA has also been detected in DC samples, and recovery of infectious virus has also been achieved from DC samples [4, 77, 117, 132, [137] [138] [139] [140] [141] . From some of these, full or majority length genomes of MERS-CoV have been sequenced [77, 137, 138] . DC versions of MERS-CoV were found to be as similar to each other, as were variants detected from different humans over time and across distance.\\n\\nAntibody screening assays have also detected crossreactive antibodies in sera. These were identified as such by screening sera against similar viruses, for example BCoV or HCoV-OC43 (as an antigenic facsimile for BCoV). It is possible that other MERS-CoV-like viruses also reside within DCs, but this does not detract from the definitive finding of MERS-CoV genetic sequences in both DCs and humans [117, 142, 143] .\\n\\nScreening studies have shown that juvenile DCs are more often positive for virus or viral RNA while older DCs are more likely to be seropositive and RNA or virus negative [76, 77, 144] . In adult DCs, MERS-CoV RNA has been detected among animals with pre-existing antibody, suggesting re-infection is possible [77, 144] . Viral loads among positive DCs can be very high [4, 76, 77, 139, 144] and DCs have been found positive both when ill with URT respiratory signs [77, 117, 142, 145] or when apparently healthy [137] . These findings indicate DCs host natural MERS-CoV infections. Furthermore, stored DC sera have revealed signs of MERS-CoV in DCs which date back over three decades (the earliest collected in 1983) [4, 133, 135] . Older sera have not been tested and so precisely how long DCs have been afflicted by MERS-CoV, whether the virus is enzootic among them, introduced to them decades or centuries ago from bats in Africa or the Arabian Peninsula, or they are the subject of regular but short-lived viral incursions from an as yet unknown host, cannot be answered.\\n\\nResearchers sought to determine a direction for infection; were DCs transmitting virus to humans or were humans infecting DCs? At a Qatari site, a farm owner and his employee became ill in mid-October 2013 and tested positive for MERS-CoV RNA in a sputum and throat swab sample, respectively. RT-rtPCRs found MERS-CoV RNA in 11 of 14 positive DC nasal swabs at the farm; six (43 %) positive by two or more assays [138] . The results indicated a recent outbreak had occurred in this herd; the first indication of MERS-CoV RNA found within DCs with a temporal association to human infections. Three positive DC samples were confirmed by sequencing a 358 nt portion of the spike gene; these sequences were identical to each other, again with close homology to other human and DC MERS-CoV sequences [138] . The DCs and human contacts yielded ORF1a and ORF4b sequences differing by only a single nucleotide each, clustering closely with the Hafr-Al-Batin_1_2013 variant [138] . Subsequent case studies found evidence of a concurrent human and DC infection and the direction of that infection was inferred to be from the ill DCs and to their human owners [117, 142, 146] . Partial genome sequences indicated that a human and a MERS-CoV RT-rtPCR positive DC had been infected by a variant of the same virus, harbouring the same distinct pattern of nucleotide polymorphisms. [142] All nine DC in the owner\\'s herd, serially sampled, reacted in a recombinant S1 antigen ELISA, with the two animals that had been RT-rtPCR positive showing a small, verifiable rise in antibody titre [142] . A rise in titre theoretically begins 10 to 21 days after DC infection [142] . The authors suggested that the rise in titre in DC sera which occurred alongside a declining RNA load, while the patient was actively ill and hospitalized, indicated that the DCs were infected first followed by the owner [117, 142] . BCoV antibodies were also present, and rising in one of the two RT-rtPCR positive animals but no animal\\'s antibodies could neutralise BCoV infection [142] .\\n\\nCamel calving season occurs in the winter months (between late October and late February; Fig. 3 ) and this may be a time when there is increased risk to humans of spill-over due to new infections among naïve DC populations [128] . What role maternal camel antibody might play in delaying infection of calves remains unknown [128, 142] . Juvenile DCs appear to host active infection more often than adult DCs and thus the sacrificial slaughter of DCs, which must be five years of age or older (termed a thane), may not be accompanied by significant risk of exposure to infection. In contrast to earlier results, slaughterhouse workers who kill both younger and older DCs, may be an occupational group with significantly higher incidence of seropositivity to MERS-CoV when animals have active MERS-CoV infections [129, 139, [147] [148] [149] . Expanded virological investigations of African DCs may lead to more seropositive animals and geographic areas in which humans may be at risk. It is possible that there are areas where humans already harbour MERS-CoV infections that have not been identified because of an absence of laboratory surveillance. Virological investigations of bats may lead to findings of ancestral viruses and viral \\'missing links\\' and identifying any other animal sources of zoonotic spread is important to inform options for reducing human exposures [56, 76] .\\n\\nInfectious MERS-CoV added to DC, goat or cow milk and stored at 4°C could be recovered at least 72 h later and, if stored at 22°C, recovery was possible for up to 48 h [150] . MERS-CoV titre decreased somewhat when recovered from milk at 22°C but pasteurization completely ablated MERS-CoV infectivity [150] . In a subsequent study, MERS-CoV RNA was identified in the milk, nasal secretion and faeces of DCs from Qatar [151] .\\n\\nA single study has examined the ability of MERS-CoV to survive in the environment [150] . Plastic or steel surfaces were inoculated with 10 6 TCID 50 of MERS-CoV at different temperature and relative humidity (RH) and virus recovery was attempted in cell culture. At high ambient temperature (30°C) and low RH (30 %) MERS-CoV remained viable for 24 h [150] . By comparison, a well known and efficently transmitted respiratory virus, influenza A virus, could not be recovered in culture beyond four hours under any conditions [150] . Aerosol experiments found MERS-CoV viability only decreased 7 % at low RH at 20°C. In comparison, influenza A virus decreased by 95 % [150] . MERS-CoV survival is inferior to that previously demonstrated for SARS-CoV [152] . For context, pathogenic bacteria can remain viable and airborne for 45 min in a coughed aerosol and can spread 4 m. MERS-CoV\\'s ability to remain viable over long time periods gives it the capacity to thoroughly contaminate a room\\'s surfaces when occupied by an infected and symptomatic patient [153] . Whether MERS-CoV can remain adrift and infectious for extended periods (truly airborne) remains unknown. Such findings expand our understanding of the possibilities for droplets to transmit respiratory viruses in many settings, including hospital waiting rooms, emergency departments, treatment rooms, open intensive care facilities and private patient rooms. The nature and quality of air exchange, circulation and filtration are important variables in risk measurement and reduction as is the use of negative pressure rooms to contain known cases. Droplet spread between humans is considered the mechanism of human-to-human transmission and the need for droplet precautions was emphasized after the Al-Ahsa hospital, the KSA and the South Korean outbreaks [21, 23, 154, 155] . By extrapolation, aerosol-generating events involving DCs (urination, defecation, and preparation and consumption of DC products) should be factored into risk measurement and reduction efforts and messaged using appropriate context. The provision of evidence supporting the best formulation of personal protective equipment to be worn by HCWs who receive, manage or conduct procedures on infectious cases remains a priority.\\n\\nMERS-CoV was found and characterized because of its apparent association with severe, and therefore more obvious, illness in humans; we were the canaries in the coal mine. Sero-assays and prospective cohort studies have yet to determine the extent to which milder or asymptomatic cases contribute to MERS-CoV transmission chains. However, transmission of MERS-CoV is defined as sporadic (not sustained), intra-familial, often healthcare associated, inefficient and requiring close and prolonged contact [22, 31, 63, 93, 97, 102, 156] In a household study, 14 of 280 (5 %) contacts of 26 MERS-CoV positive index patients were RNA or antibody positive; the rate of general transmission, even in outbreaks is around 3 % [31] . It seems that the majority of human cases of MERS-CoV, even when numbers appear to increase suddenly, do not readily transmit to more than one other human so to date, the localized epidemic of MERS-CoV has not been self-sustaining [157] [158] [159] [160] [161] . That is to say, the basic reproduction number (R 0 ) -the average number of infections caused by one infected individual in a fully susceptible populationhas been close to one throughout various clusters and outbreaks. If R 0 was greater than 1, a sustained increase in case numbers would be expected. Some R o calculations may be affected by incomplete case contact tracing, limited community testing and how a case is defined. That MERS has had a constant presence in the Arabian Peninsula since 2012 is due to ongoing, sporadic spill-over events from DCs amplified by poorly controlled hospital outbreaks.\\n\\nThe first known MERS human-to-human transmission event was one characterized by acute LRT disease in a healthcare setting in Jordan. In stark contrast, a sero-survey of HCW who were sometimes in close and prolonged contact with the first, fatal MERS-CoV case in 2012 [162] , found none of the HCW had seroconverted four months later, despite an absence of eye protection and variable compliance with required PPE standards [162] .\\n\\nEarly on in the MERS story, samples for testing were mostly collected from patients with severe illness and not those with milder acute respiratory tract infections. Contacts of confirmed MERS cases were often observed for clinical illness, but not tested. These omissions may have confounded our understanding of MERS-CoV transmission and biased early data towards higher numbers of seriously ill and hospitalized patients, inflating the apparent proportion of fatal cases. Case-control studies were not a focus. As testing paradigms changed and contacts were increasingly tested, more asymptomatic and mild infections were recognized [163] .\\n\\nA rise in the cases termed asymptomatic (which enlarge the denominator for calculations of the proportion of fatal cases, defined in [164] ) resulted in a drop in the proportion of fatal cases during the Jeddah-2014 outbreak. Historically, such rises are consistent with changing definitions and laboratory responses and clinical management of a newly discovered virus infection that was first noted only among the severely ill. Upon follow-up, over three-quarters of such MERS-CoV RNA positive people did recall having one or more symptoms at the time, despite being reported as asymptomatic [165] raising some question over the reliability of other reported data.\\n\\nThe proportion of fatal MERS cases within the KSA compared to outside the KSA, as well as the age, and sex distribution change in different ways when comparing MERS outbreaks. Approximately 43 % of MERS cases (549 of 1277) in the KSA were fatal betwen 2012 and December 2015 while 21 % (72 of 330) died among those occurring outside of the KSA. The total number of male cases always outnumber females and the proportion of male deaths is always greater than the proportion of females who die. However the proportion of male deaths from total males with MERS is a similar figure to that for females. In the KSA, there is a greater proportion of younger males among cases and deaths than were observed from the 2015 South Korean or the Jeddah-2014 outbreaks (Additional file 2: Figure S2 ). Why these aspects have differed may be due to differences in the time to presentation and diagnosis, the nature and quality of supportive care, the way a person became infected (habits, exposure to a human or zoonotic source, viral load, route of infection) or the extent to which different populations are burdened by underlying diseases [40] .\\n\\nAs a group, HCWs comprised 16 % of MERS cases in the KSA and South Korea. It is apparent that the weekly proportion of infected HCWs increases alongside each steep rise in overall detections (Fig. 5) . In May 2013, the WHO published guidelines for IPC during care of probable or confirmed cases of MERS-CoV infection in a healthcare setting [166] . This is explainable because to date, each case rise has been intimately associated with healthcare-facility related outbreaks [118] . These rises in MERS-CoV detections can decrease the average age during each event because HCWs are usually younger than inpatients with MERS. Healthcare facilities have been a regular target for suggested improvements aimed at improving infection prevention and control (IPC) procedures [115, 118] .\\n\\nMost of the analysis of MERS-CoV genetics has been performed using high throughput or \"deep\" sequencing methods for complete genome deduction [167] [168] [169] . MERS-CoV was the first subject of such widespread use of deep sequencing to study an emerging viral outbreak with global reach. The technique can produce genomic [207] [208] [209] . Earlier and subsequent versions of this chart are maintained on a personal blog [210] length coverage in a single experiment with highly repetitious measurement of each nucleotide position [52, 140] . Despite assays having been published early on, subgenomic sequencing, once the mainstay of viral outbreak studies, has less often been published during MERS-CoV characterization [48] . As more genomes from both humans and DCs have been characterized, two clades have become apparent; A and B (Fig. 6) . Clade A contains only human-derived MERS-CoV genomes from Jordan, while Clade B comprises the majority of human and camel genomes deduced thus far [168] .\\n\\nTwo studies during 2015, one looking at Jeddah-2014 MERS-CoV variants and another looking at a variant exported from South Korea to China, have now identified signs of genetic recombination among MERS-CoV variants. While human and camel whole genome sequences have retained >99 % identity with each other, members of genetically distinct lineages can and do swap genetic material when suitable conditions and coinfections co-occur [170] [171] [172] . Shared identity implies that the major source for human acquisition is the DC, rather than another animal, although more testing of other animal species is needed to confirm that conclusion. Over a month, a DC virus sequenced on different occasions did not change at all indicating a degree of genomic stability in its host, supporting that DCs are the natural, rather than intermediate, host for the MERS-CoV we know today [77] . To date, recombination has been localised to breakpoints near the boundary between ORF1a and ORF1b regions, within the spike gene [170] and in the ORF1b region (Fig. 2) [172] . It is not unexpected that recombination should occur since it is well known among other CoVs [124] and because the majority of MERS-CoV whole genomes collected from samples spanning three years (2012-2015) and from humans, camels and different countries have shown close genetic identity to each other, with just enough subtle variation to support outbreak investigations so long as whole genome sequencing is applied [52, 77, 135, 138, 168, [173] [174] [175] .\\n\\nChanges in genome sequence may herald alterations to virus transmissibility, replication, persistence, lethality or response to future drugs. If we have prior knowledge of the impact of genetic changes because of thorough characterization studies, we can closely Fig. 6 The genetic relationship between MERS-CoV nucleotide sequences (downloaded from GenBank using the listed accession numbers and from virological.org [212] ). This neighbour joining tree was created in MEGA v6 using an alignment of human and DCderived MERS-CoV sequences (Geneious v8.1 [211] ). Clades are indicated next to dark (Clade A) or pale (Clade B) blue vertical bars. Camel icons denote genomes from DCs. Healthcare or community outbreaks are boxed and labelled using previously described schemes [212, 213] monitor the genomic regions and better understand any changes in transmission or disease patterns as they occur. Genetic mutations noted during the largest of human outbreaks, Jeddah-2014, did not impart any major replicative or immunomodulatory changes when compared to earlier viral variants in vitro [156, 176] . However, we understand very little of the phenotypic outcomes that result from subtle genetic change in MERS-CoV genomes. To date no clinical relevance or obvious in vivo changes to viral replication, shedding or transmission has been reported or attributed to mutations or to new recombinant viruses [156] . But vigilance and larger, more contemporary and in vivo studies are needed.\\n\\nGenome sequence located to a distinct clade were identified from an Egyptian DC that was probably imported from Sudan. This does not fit into either of the current clades [125, 168, 177] . A virus sequenced from a Neoromicia capensis bat was more closely related to MERS-CoV than other large bat-derived sequences had been to that point, but the genome of a variant of a MERS-CoV has yet to be discovered and deduced from any bat [125] .\\n\\nAnalyses of MERS-CoV genomes have shown that most single nucleotide differences among variants were located in the last third of the genome (Fig. 2) , which encodes the spike protein and accessory proteins [168] . At least nine MERS-CoV genomes contained amino acid substitutions in the receptor binding domain (RBD) of the spike protein and codons 158 (N-terminal region), 460 (RBD), 1020 (in heptad repeat 1), 1202 and 1208 bear investigation as markers of adaptive change [140, 169] . The spike protein had not changed in the recombinant MERS-CoV genome identified in China in 2015 but was reported to have varied at a higher rate than that for complete MERS-CoV genomes, among South Korean variants [172, 178] . This highlights that subgenomic regions may not always contain enough genetic diversity to prove useful for differentiating viral variants. Despite this, one assay amplifying a 615 nucleotide fragment of the spike S2 domain gene for Sanger sequencing agreed with the results generated by the sequencing of a some full genomes and was useful to define additional sequence groupings [177] .\\n\\nGenomic sequence can also be used to define the geographic boundaries of a cluster or outbreak and monitor its progress, based on the similarity of the variants found among infected humans and animals when occurring together, or between different sites and times (Fig. 6 ) [169] . This approach was employed when defining the geographically constrained MERS hospital outbreak in Al-Ahsa, which occurred between 1 st April and 23 rd May 2013, as well as clusters in Buraidah and a community outbreak in Hafr Al-Batin, the KSA. Genomic sequencing identified that approximately 12 MERS-CoV detections from a community outbreak in Hafr Al-Batin between June and August 2013 may have been triggered by an index case becoming infected through DC contact [175] . Sequencing MERS-CoV genomes from the 2013 Al-Ahsa hospital outbreak indicated that multiple viral variants contributed to the cases but that most were similar enough to each other to be consistent with human-tohuman transmission. Molecular epidemiology has revealed otherwise hidden links in transmission chains encompassing a period of up to five months [179] . However, most outbreaks have not continued for longer than two to three months and so opportunities for the virus to adapt further to humans through co-infection and sustained serial passage have been rare [169] . In Riyadh-2014, genetic evidence supported the likelihood of multiple external introductions of virus, implicating a range of healthcare facilities in an event that otherwise looked contiguous [23, 168, 179] . Riyadh is a nexus for camel and human travel and has had more MERS cases than any other region of the KSA to date but also harbours a wide range of MERS-CoV variants [128, 167, 179] . However the South Korean outbreak originated from a single infected person, resulting in three to four generations of cases [180, 181] . Studies of this apparently recombinant viral variant did not find an increased evolutionary rate and no sign of virus adaptation thus the outbreak seems to have been driven by circumstance rather than circumstance together with mutation [181] .\\n\\nFor many MERS cases detected outside the Arabian Peninsula, extensive contact tracing has been performed and the results described in detail. Contact tracing is essential to contain the emergence and transmission of a new virus and today it is supported by molecular epidemiology. Although it is an expensive and time consuming process, contact tracing can identify potential new infections and through active or passive monitoring, react more rapidly if disease does develop. Results of contact tracing to date have found that onward transmission among humans is an infrequent event. For example, there were 83 contacts, both symptomatic and asymptomatic, of a case treated in Germany who travelled from the UAE but no sign of virus or antibody were found in any of them [73] . The very first MERS case had made contact with 56 HCWs and 48 others, but none developed any indication of infection [162] . In a study of 123 contacts of a case treated in France, only seven matched the definition for a possible case and were tested; one who had shared a 20 m 2 hospital room while in a bed 1.5 m away from the index case for a prolonged period was positive [26] . None of the contacts of the first two MERS cases imported into the USA in 2014 contained any MERS-CoV footprint [182] and none of the 131 contacts of two travellers returning to the Netherlands developed MERS-CoV antibodies or tested RNA positive [25, 183] . Analyses of public data reveal many likely instances of nosocomial acquisition of infection in the Arabian Peninsula and these data may be accompanied by some details noting contact with a known case or facility. One example identified the likely role of a patient with a subclinical infection, present in a hospital during their admission for other reasons, as the likeliest index case triggering a family cluster [93] . Contact tracing was a significant factor in the termination of a 2015 outbreak involving multiple South Korean hospitals [184] . Such studies demonstrate the necessity of finding and understanding a role for mild and asymptomatic cases, together with restricting close contact or prolonged exposure of infected people to others, especially older family members and friends with underlying disease (Fig. 4c) .\\n\\nThe hospital-associated outbreak in Jeddah in 2014 was the largest and most rapid accumulation of MERS-CoV detections to date. The greatest number of MERS-CoV detections of any month on record occurred in Jeddah in April. The outbreak was mostly (>60 % of cases) associated with human-to-human spread within hospital environments and resulted from a lack of, or breakdown in, infection prevention and control [37, 185, 186] . A rise in fatalities followed the rapid increase in case numbers.\\n\\nIn 2015 two large outbreaks occurred. South Korea was the site of the first large scale outbreak outside the Arabian Peninsula and produced the first cases in both South Korea and China, occurring between May and July 2015. This was closely followed by a distinct outbreak in Ar Riyad province in the KSA which appeared to come under control in early November.\\n\\nAfter staying in Bahrain for two weeks, a 68 year old male (68 M) travelled home to South Korea via Qatar, arriving free of symptoms on the 4 th May 2015 [187] . He developed fever, myalgia and a cough nearly a week later (11 th ). He visited a clinic as an outpatient between the 12 th and 15 th of May and was admitted to Hospital A on the 15 th [188] . He was discharged from Hospital A on the 17 th then visited and was admitted to the emergency department of Hospital B on the 18 th . During this second stay, a sputum sample was taken and tested positive for MERS-CoV on the 20 th [187, 188] , triggering transfer to the designated isolation treatment facility. Over a period of 10 days, the index case was seen at three different hospitals, demonstrating a key feature of \"hospital shopping\" that shaped the South Korean outbreak. Approximately 34 people were infected during this time [187] . In total 186 cases were generated in this outbreak, all linked through a single transmission chain to 68 M; 37 cases died [189] . In South Korea, the national health insurance system provides for relatively low cost medical care, defraying some costs by making family members responsible for a portion of the ministration of the sick, resulting in them sometimes staying for long periods in the rooms that often have more than four beds in them [24] . Other factors thought to have enabled this outbreak included unfamiliarity of local clinicians with MERS, ease with which the public can visit and be treated by tertiary hospitals, the custom of visiting sick friends and relatives in hospitals, the hierarchical nature of Korean society, crowded emergency rooms, poor IPC measures, a lack of negative pressure isolation rooms and poor inter-hospital communication of patient disease histories [24, [190] [191] [192] . All of the reported transmission occurred across three or four generations and apart from one unknown source, were all hospital-acquired [24, 120, 181, [193] [194] [195] . Few clinical details about these cases have been reported to date and detail on transmission and contact tracing is minimal. The hospitals involved were initially not identified, governmental guidance and actions produced confusing messages and there was very limited communication at all early on which resulted in unnecessary concern, distrust and a distinct economic impact [191, [196] [197] [198] . Early in the outbreak, a infected traveller, the son of an identified case in South Korea, passed through Hong Kong on his way to China where he was located, isolated and cared for in China [91, 199, 200] . No contacts became ill. The outbreak was brought under control in late July/ early August [201] after improved IPC measures were employed, strong contact tracing monitoring and quarantine, expanded laboratory testing, hospitals were better secured, specialized personnel were dispatched to manage cases and international cooperation increased [202, 203] . A review of public data showed that, as for MERS in the KSA, older age and the presence of underlying disease were significantly associated with a fatal outcome in South Korea. [40] Even though R 0 is <1, super-spreading events facilitated by circumstances created in healthcare settings and characterized by cluster sizes over 150, such as this one, are not unexpected from MERS-CoV infection [204] . The dynamic of an outbreak depends on the R 0 and an individual\\'s viral shedding patterns, contact type and frequency, hospital procedures and population structure and density [204] .\\n\\nIn the region of Ar Riyad, including the capital city of Riyadh, a hospital based cluster began, within a single hospital, from late June 2015 [205] . By mid-September there had been approximately170 cases reported but the outbreak appeared to been brought under control in November.\\n\\nIt became apparent early on that MERS-CoV spread relatively ineffectively from human-to-human. Despite ongoing and possibly seasonal introduction of virus to the human population via infected DCs and perhaps other animals yet to be identified, the vast majority of MERS-CoV transmission has occurred from infected to uninfected humans in close and prolonged contact through circumstances created by poor infection control in health care settings. This opportunistic virus has had its greatest impact on those with underlying diseases and such vulnerable people, sometimes suffering multiple comorbidities, have been most often associated with hospitals, creating a perfect storm of exposure, transmission and mortality. It remains unclear if this group are uniquely affected by MERS-CoV or if other respiratory virus infections, including those from HCoVs, produce a similarly serious impact. In South Korea, a single imported case created an outbreak of 185 cases and 36 deaths that had a disproportionate impact on economic performance, community behaviour and trust in government and the health care system. Household human-to human transmission occurs but is also limited. Educational programs will be essential tools for combatting the spread of MERS-CoV both within urban and regional communities and for the health care setting.\\n\\nVigilance remains important for containment since MERS-CoV is a virus with a genetic makeup that has been observed for only three years and is not stable. Among all humans reported to be infected, nearly 40 % have died. Continued laboratory testing, sequencing, analysis, timely data sharing and clear communication are essential for such vigilance to be effective. Global alignment of case definitions would further aid accurate calculation of a case fatality ratio by including subclinical case numbers. Whole genome sequencing has been used extensively to study MERS-CoV travel and variation and although it remains a tool for experts, it appears to be the best tool for the job.\\n\\nMERS and SARS have some clinical similarities but they also diverge significantly [206] . Defining characteristics include the higher PFC among MERS cases (above 50 % in 2013 and currently at 30-40 %; well above the 9 % of SARS) and the higher association between fatal MERS and older males with underlying comorbidities. For the viruses, MERS-CoV has a broader tropism, grows more rapidly in vitro, more rapidly induces cytopathogenic change, triggers distinct transcriptional responses, makes use of a different receptor, induces a more proinflammatory state and has a delayed innate antiviral response compared to SARS-CoV.\\n\\nThere appears to be a 2-3 % prevalence of MERS-CoV in the KSA with a 5 % chance of secondary transmission within the household. There is an increased risk of infection through certain occupations at certain times and a much greater chance for spread to other humans during circumstances created by humans, which drives more effective transmission than any R 0 would predict on face value. Nonetheless, despite multiple mass gatherings that have afforded the virus many millions of opportunities to spread, there have remarkably been no reported outbreaks of MERS or MERS-CoV during or immediately after these events. There is no evidence that MERS-CoV is a virus of pandemic concern. Nonetheless, hospital settings continue to describe MERS cases and outbreaks in the Arabian Peninsula. As long as we facilitate the spread of MERS-CoV among our most vulnerable populations, the world must remain on alert for cases which may be exported more frequently when a host country with infected camel reservoirs is experiencing human clusters or outbreaks.\\n\\nThe MERS-CoV appears to be an enzootic virus infecting the DC URT with evidence of recent genetic recombination. It may once have had its origins among bats, but evidence is lacking and the relevance of that to today\\'s ongoing epidemic is academic. Thanks to quick action, the sensitive and rapid molecular diagnostic tools required to achieve rapid and sensitive detection goal have been in place and made widely available since the virus was reported in 2012. RT-PCR testing of LRT samples remains the gold standard for MERS-CoV confirmation. Serological tools continue to emerge but they are in need of further validation using samples from mild and asymptomatic infections and a densely sampled cohort study to follow contacts of new cases may address this need. Similarly, the important question of whether those who do shed MERS-CoV RNA for extended periods are infectious while appearing well, continues to go unanswered. It is even unclear just how many \\'asymptomatic\\' infections have been described and reported correctly which in turn raises questions about the reliability of other clinical data collection to date. While the basic virology of MERS-CoV has advanced over the course of the past three years, understanding what is happening in, and the interplay between, camel, environment and human is still in its infancy.\\n\\nAdditional file 1: Figure S1 . The', 'document_id': 1741}]}, {'paragraphs': [{'qas': [{'question': 'What is PHEIC?', 'id': 1238, 'answers': [{'text': \"Public Health Emergency of International Concern'\", 'answer_start': 2619}], 'is_impossible': False}, {'question': 'How is PHEIC defined?', 'id': 1239, 'answers': [{'text': 'an extraordinary event which is determined to constitute a public health risk to other States through the international spread of disease and to potentially require a coordinated international response', 'answer_start': 2740}], 'is_impossible': False}, {'question': 'What platform was instrumental in rapid sharing of COVID-19 information?', 'id': 1240, 'answers': [{'text': 'Global Initiative on Sharing All Influenza Data (GISAID)', 'answer_start': 3228}], 'is_impossible': False}], 'context': 'On the Coronavirus (COVID-19) Outbreak and the Smart City Network: Universal Data Sharing Standards Coupled with Artificial Intelligence (AI) to Benefit Urban Health Monitoring and Management\\n\\nhttps://doi.org/10.3390/healthcare8010046\\n\\nSHA: 90d04764b497a224a1d969f4e317fc19a5feab35\\n\\nAuthors: Allam, Zaheer; Jones, David S.\\nDate: 2020\\nDOI: 10.3390/healthcare8010046\\nLicense: cc-by\\n\\nAbstract: As the Coronavirus (COVID-19) expands its impact from China, expanding its catchment into surrounding regions and other countries, increased national and international measures are being taken to contain the outbreak. The placing of entire cities in ‘lockdown’ directly affects urban economies on a multi-lateral level, including from social and economic standpoints. This is being emphasised as the outbreak gains ground in other countries, leading towards a global health emergency, and as global collaboration is sought in numerous quarters. However, while effective protocols in regard to the sharing of health data is emphasised, urban data, on the other hand, specifically relating to urban health and safe city concepts, is still viewed from a nationalist perspective as solely benefiting a nation’s economy and its economic and political influence. This perspective paper, written one month after detection and during the outbreak, surveys the virus outbreak from an urban standpoint and advances how smart city networks should work towards enhancing standardization protocols for increased data sharing in the event of outbreaks or disasters, leading to better global understanding and management of the same.\\n\\nText: The novel Coronavirus outbreak, (previously known as the 2019-nCoV and later renamed COVID-19 during the writing of this manuscript) is leading to the closure of entire cities in China, and causing stringent measures to be taken in others. While in distant different continents, far from China where the virus was first reported, places are being placed on high alert. In Wuhan, where the virus broke, schools, roads and markets have been shut down [1] . The same is true in Hong Kong, Beijing and Hubei Province amongst surrounding areas, as precautionary measures are being emphasized to ensure that the spread of the virus is minimized, and complete and accurate information on the virus is being obtained [2] . However, the rate of spread of the virus and the uncertainties surrounding the entire situation has led the World Health Organization (WHO) on 30 January 2019 to declare the Coronavirus outbreak a \\'Global Public Health Emergency\\'. WHO determined, however, not to declare the outbreak a \\'Public Health Emergency of International Concern\\' (PHEIC) which is a higher level of declaration. A PHEIC is defined as \"an extraordinary event which is determined to constitute a public health risk to other States through the international spread of disease and to potentially require a coordinated international response\" whose scope may include: serious, sudden, unusual or unexpected; carries implications for public health beyond the affected State\\'s national border; and may require immediate international action [3] .\\n\\nWith the world having experienced some notable influenza pandemics in the past, a Global Initiative on Sharing All Influenza Data (GISAID) platform [4] was established and was instrumental in the rapid sharing of information by the Chinese scientists regarding the emergence of the COVID-19 virus. Through this platform, scientists from other regions were observed to gain access to information and are, subsequently, able to act in a much faster capacity; like in the case of scientists from the Virus Identification Laboratory based at Doherty Institute, Australia, who managed to grow a similar virus in the laboratory after accessing the data shared by the Chinese scientists [5] .\\n\\nBeyond the aspect of pandemic preparedness and response, the case of COVID-19 virus and its spread provide a fascinating case study for the thematics of urban health. Here, as technological tools and laboratories around the world share data and collectively work to devise tools and cures, similar efforts should be considered between smart city professionals on how collaborative strategies could allow for the maximization of public safety on such and similar scenarios. This is valid as smart cities host a rich array of technological products [6, 7] that can assist in early detection of outbreaks; either through thermal cameras or Internet of Things (IoT) sensors, and early discussions could render efforts towards better management of similar situations in case of future potential outbreaks, and to improve the health fabric of cities generally. While thermal cameras are not sufficient on their own for the detection of pandemics -like the case of the COVID-19, the integration of such products with artificial intelligence (AI) can provide added benefits. The fact that initial screenings of temperature is being pursued for the case of the COVID-19 at airports and in areas of mass convergence is a testament to its potential in an automated fashion. Kamel Boulos et al. [8] supports that data from various technological products can help enrich health databases, provide more accurate, efficient, comprehensive and real-time information on outbreaks and their dispersal, thus aiding in the provision of better urban fabric risk management decisions.\\n\\nThe above improvements in the healthcare sector can only be achieved if different smart city products are fashioned to support standardized protocols that would allow for seamless communication between themselves. Weber and Podnar Žarko [9] suggest that IoT devices in use should support open protocols, and at the same time, the device provider should ensure that those fashioned uphold data integrity and safety during communication and transmission. Unfortunately, this has not been the case and, as Vermesan and Friess [10] explain, most smart city products use proprietary solutions that are only understood by the service providers. This situation often creates unnecessary fragmentation of information rendering only a partial integrated view on the dynamics of the urban realm. With restricted knowledge on emergent trends, urban managers cannot effectively take decisions to contain outbreaks and adequately act without compromising the social and economic integrity of their city. This paper, inspired by the case of the COVID-19 virus, explores how urban resilience can be further achieved, and outlines the importance of seeking standardization of communication across and between smart cities.\\n\\nWith the advent of the digital age and the plethora of Internet of Things (IoT) devices it brings, there has been a substantial rise in the amount of data gathered by these devices in different sectors like transport, environment, entertainment, sport and health sectors, amongst others [11] . To put this into perspective, it is believed that by the end of 2020, over 2314 exabytes (1 exabyte = 1 billion gigabytes) of data will be generated globally [12] from the health sector. Stanford Medicine [12] acknowledges that this increase, especially in the medical field, is witnessing a proportional increase due to the increase in sources of data that are not limited to hospital records. Rather, the increase is being underpinned by drawing upon a myriad and increasing number of IoT smart devices, that are projected to exponentially increase the global healthcare market to a value of more than USD $543.3 billion by 2025 [13] . However, while the potential for the data market is understood, such issues like privacy of information, data protection and sharing, and obligatory requirements of healthcare management and monitoring, among others, are critical. Moreover, in the present case of the Coronavirus outbreak, this ought to be handled with care to avoid jeopardizing efforts already in place to combat the pandemic. On the foremost, since these cut across different countries, which are part of the global community and have their unique laws and regulations concerning issues mentioned above, it is paramount to observe them as per the dictate of their source country\\'s laws and regulations; hence, underlining the importance of working towards not only the promoting of data through its usage but also the need for standardized and universally agreed protocols.\\n\\nWhile the significance of such data in advancing efficiency, productivity and processes in different sectors is being lauded, there are criticisms arising as to the nature of data collection, storage, management and accessibility by only a small group of users. The latter particularly includes select ICT corporations that are also located in specific geographies [6, [14] [15] [16] [17] . These criticisms are justified, as in recent years, big data is seen as the new \\'gold rush\\' of the 21st century and limiting its access means higher economic returns and increased influence and control at various scales to those who control data. These associated benefits with big data are clearly influencing geopolitical standings, in both corporate and conventional governance realms, and there is increased competition between powerful economies to ensure that they have the maximum control of big data. As case in point is the amount of \\'push and pull\\' that has arisen from Huawei\\'s 5G internet planned rollout [18] . Though the latter service offers unprecedented opportunities to increase internet speeds, and thereby influence the handling of big data, countries like the U.S. and some European countries that are key proponents and players in global political, economic and health landscapes, are against this rollout, arguing that it is a deceptive way of gathering private data under the guise of espionage. On this, it has been noted that the issue of data control and handling by a few corporations accords with their principles of nationalism, and that these work for their own wellbeing as well as to benefit the territories they are registered in. Therefore, geopolitical issues are expected on the technological front as most large data-rich corporations are located in powerful countries that have influence both economically, health-wise and politically [19] [20] [21] . Such are deemed prized tokens on the international landscape, and it is expected that these economies will continue to work towards their predominant control as much as possible. On the health sector, the same approach is being upheld where critical information and data are not freely shared between economies as that would be seen to be benefiting other in-competition economies, whereas different economies would cherish the maximization of benefits from such data collections.\\n\\nIn addition to the obvious deep-rooted social issues related to nationalism, other challenges include the increasing movement of people globally that is being enhanced by reduced costs and higher speed. In particular, these challenges are more pronounced when it comes to public health. This is because most of the health-related data collected not only can compromise local nations, but also captures those of travelers. In such cases, in a bid to improve the health status of a nation, it becomes paramount to factor in data from other regions necessitating unhindered sharing of this data.\\n\\nSuch data-sharing truth is emphasized in situations like the recent case of Coronavirus outbreak threatening the global health environment, facilitated by air transportation. The virus was first reported in Wuhan, China, and in a matter of three weeks (by 17th January 2020) over 300 cases were confirmed in that region, and 10 days later (26th January 2020), a total of 2014 cases of Coronavirus have been reported, with 684 of those being confirmed, and with 29 reported outside China. The fatalities from the virus stands at 56 as of 26th January 2020 [22] . The virus had then been confirmed in various countries including Taiwan, South Korea, Japan, Thailand, France, the United States, Singapore and Vietnam [23] .\\n\\nIn the above case, though major cities are known to prepare themselves for potential outbreaks, their health policies and protocols are observed to diverge from one another. Thus, without a global collaborative approach, progress towards working for a cure and universally acceptable policy approach can take longer. Such fears, of a lack of international collaboration, were highlighted by the World Health Organization (WHO) during an emergency meeting in Geneva on 22nd January 2020 to determine whether the virus outbreak had reached a level warranting international emergency concern. However, WHO was satisfied that China was being proactive in this case, unlike in 2002, when China withheld information on the outbreak for far too long, causing delays in addressing the epidemic [3] . As in this instance, it is the opinion in this paper that if there was seamless collaboration and seamless sharing of data between different cities, it would not warrant such a high-level meeting to result in action, and instead, a decision could have been made much earlier. On this, the saddest part is that some global cities are less prepared to handle the challenges posed by this type of outbreak for lack of information on issues like symptoms of the virus, the protective measures to be taken, and the treatment procedures that an infected person should be processed through, amongst other issues.\\n\\nThe timely response by stakeholders in regard to this new outbreak are commendable compared to previous cases. The latter includes the Severe Acute Respiratory Syndrome (SARS) outbreak in 2002 that took substantial time (from November 2002 to April 2003) to identify and be dealt with [24] ; the Ebola outbreak in West Africa in 2013 that took months to determine; and the Zika Virus that was first reported in 2014 before being successfully identified in 2015.\\n\\nWith the Coronavirus (COVID-19) , it took only 17 days (31st December 2019 to 17th January 2020) to be identified. The sharing of data has also been quicker, as immediately after the virus\\' genetic sequence was discovered, Chinese scientists were able to share the information with the WHO, thus helping in its identification and enabling the auctioning of precautionary measures in other countries. Latest technological tools have also allowed for the receipt of information in realtime, in contrast to traditional epidemiological approaches that would have required months to identify the outbreak type [25] . Similarly, though substantial data and information on the disease has been shared, Wetsman [26] acknowledges that there is a lack of some vital information, like the ease of spread of the virus from person-to-person, and this is a key to containing the disease as interactions between people from different parts of the globe are still active. This hindrance can be made further possible as many cities advance in their smart and safe city model implementation towards constructing sufficient soft and hard urban infrastructures equipped with, for example, thermal imagery sensors to allow for early detections. However, while that is the case, data access to many is a challenge because the information is often seen as being sensitive for national security reasons, whilst at the same time, acknowledging that a virus outbreak is an equal threat to both national security and the economy.\\n\\nThe outbreak of any disease has significant impacts on local economies across the globe. For instance, when SARS (Severe Acute Respiratory Syndrome) (SARS-CoV) broke in China in 2002, it was estimated, that the Asian region incurred tremendous negative impacts socially, health-wise and economically, potentially amounting to Asian regional economy losses of between USD $12-18 billion from tourism, travel and retail sales industries alone [27] . The Zika virus outbreak, spread by daytime-active Aedes mosquitoes, is estimated to have cost equator-belt local economies in affected areas between USD $7 and USD $18 billion [28] . The Ebola virus (or Ebola hemorrhagic fever (EHF)) caused an estimated loss of USD $2.2 billion in GDP in three West African economies (Guinea, Liberia and Sierra Leone) in 2015 alone [29] . In regard to the current epidemic of Coronavirus, though it is too early to quantify or project its impacts on the global economy, there are fears that it may take the precedent of other outbreaks where billions of dollars will be lost. The foundations for this escalating loss can be witnessed in the rapid growth of travel bans being enacted by some countries and their international airports, especially specifically restricting people from visiting the affected regions in China and their growth into general non-Chinese travel movements. On this, noting that the outbreak came almost on the eve of the Lunar New Year celebrations, and that it had been estimated that over 400 million people were expected to travel in different parts of the world and China to observe this festivity, the majority have had to reconsider their options as to flights, hotels and entertainment events due to service provider cancellations [30] . Those who had already booked their flights are expected to receive their refunds following the directive by the Civil Aviation Administration of China, however, this move has already affected the share value of Chinese airline companies [30] .\\n\\nThe above impacts demonstrate that the issues of virus outbreaks transcend urban safety and impacts upon all other facets of our urban fabric. Therefore, it becomes paramount to ensure that the measures taken to contain a virus transcend nationalist agendas where data and information sharing is normally restricted, to a more global agenda where humanity and global order are encouraged. With such an approach, it would be easier to share urban health data across geographies to better monitor emerging health threats in order to provide more economic stability, thereby ensuring no disruptions on such sectors like tourism and travel industries, amongst others. This is possible by ensuring collaborative, proactive measures to control outbreak spread and thus, human movements. This would remove fears on travelers, and would have positive impacts upon the tourism industry, that has been seen to bear the economic brunt whenever such outbreaks occur. This can be achieved by ensuring that protocols on data sharing are calibrated to remove all hurdles pertaining to sharing of information. On this, Lawpoolsri et al. [31] posits that such issues, like transparency, timelessness of sharing and access and quality of data, should be upheld so that continuous monitoring and assessment can be pursued.\\n\\nVirus outbreaks in recent years have shown that, in the urban realm, data, including health data, can be sourced from diverse places. Presently, in the case of Coronavirus (COVID-19) outbreak, data is being collected from airports through screening and monitoring, through the use of smart sensors installed in airport infrastructures and from personnel working in those air/seaports. For instance, it has been reported that in the U.S.A., screening is being carried out at 20 different airports to ensure that possible affected people are intercepted for quarantine at the point of entry. Beside airports, as reported by Buckley and May [2] , data is also being collected at bus terminals, market places (in Wuhan), subways, and also in health facilities where patients are taken for further medical attention. Such is prevalent especially in China, and other Asian regions where cases of the virus have been recorded and confirmed.\\n\\nIn addition to these methods, other smart city data sources include the application of terminal tracking systems that are mostly emphasized in Safe City concepts, where, at the point of entry or departure, relevant data is collected and analyzed. Li et al. [32] highlights that sensors installed in such locations have the potential to receive and distribute data in real-time to digital infrastructures within the network, and their interconnectedness in the network renders them extremely efficient in providing real-time updates on different issues. Urban areas are also known to be amassed with numerous Urban Health sensors, some of which are wearable. Though these are not specifically fashioned to track the present case of virus outbreak, they are able to track other related parameters like heartbeat, blood pressure, body temperature and others variables, that when analyzed can offer valuable insights. Loncar-Turukalo et al. [33] hail these devices for their role in transforming the health care sector especially by allowing for Connected Health (CH) care, where data collected from them can be analyzed and provide insightful information on the health scenario in any given area. Vashist et al. [34] further highlight how emerging features such as spatiotemporal mapping, remote monitoring and management, and enhanced cloud computing capabilities can emanate from such endeavours, leading to better urban management potential.\\n\\nWhile it is true that the basic source of medical data is generally sourced from general practitioners or medical laboratories-a fact that has also been affirmed in the case of the current epidemic-this paper explores how data sourced from an urban perspective can contribute to the medical narrative. The conviction to dwell on the urban realm in this manuscript is based on the fact that the current epidemic (COVID-19) is transmitted majorly through human-to-human contact, and in most cases, especially where the spread is reported in a different country, the first point of contact is an urban area, where large groups of people convene, like airports or subway stations. In most cases, such facilities, which are mostly based in urban areas, are observed to have installed surveillance technologies to ensure that anyone showing any symptoms of the disease are identified and quarantined. However, even in such cases, as underlined in the present manuscript, the need for anonymizing medical data is emphasized to ensure that the use of current technologies does not breach data privacy and security requirements, across different geographies. In this case, novel technologies like Blockchain technologies and quantum cryptography can aid in the discussion and be made to integrate with data collecting technologies. This would render an increased wealth of data from both the medical field and smart city operators, while ensuring privacy and security; hence, aiding in providing relevant information for better informed decisions.\\n\\nHowever, despite the indisputable roles that installed devices play in providing relevant health information, their data communication aspect needs to be reviewed. First, communications are seen to be geography-restricted (restricted to a given location), such that they seldom expand or communicate with their like, installed beyond their restricted areas. Secondly, these devices are usually sourced and installed by separate corporations that maintain unique and specific standards for data processing and sharing, and accordingly, tying cities to the sole usage of their product(s). Such strategies are adopted as private corporations try to maximize their economic gains, since the digital solution market is a lucrative one and is expected to continue growing and expanding [6, 7] .\\n\\nFor its current application, the standardization of protocols as elaborated in this manuscript need to be pursued to ensure that there is seamless sharing of information and data. By doing this, it is expected that issues like burdens of collecting data, accuracy and other complexity that are experienced (when systems are fragmented) are reduced or eliminated altogether. The standardization can be achieved by, for example, ensuring that all the devices and systems are linked into a single network, like was done in the U.S., where all the surveillance of healthcare were combined into the National Healthcare Safety Network (NHSH) [35] . The fact that cities are increasingly tuning on the concept of Smart Cities and boasting an increased adoption rate of technological and connected products, existing surveillance networks can be re-calibrated to make use of those new sets of databases. Appropriate protocols however have to be drafted to ensure effective actions while ensuring privacy and security of data and people.\\n\\nWith scenarios like the present Coronavirus (COVID-19) outbreak, that not only impacts upon the economic status of cities, but also affects their social standing, it becomes imperative to emphasize the adoption of universal standards for data sharing. Such a move could have far reaching impact across cities and territories especially in positively combating outbreaks and disasters in a quicker, safer and standardized way, such that when the cure is discovered, the results can be replicated in various parts of the globe. With a collaborated data sharing protocol, it would be possible to have a larger dataset resulting in increased processing capabilities especially with technologies that are powered by artificial intelligence (AI) tools. Through this way, as noted by Jiang et al. [36] and Allam [37] , it would be possible to facilitate early detection, achieve better diagnosis and provide better urban management decisions for increased efficiency for virus containment.\\n\\nAn example of how beneficial collaboration and sharing of data can be occurred during the 2014 Ebola outbreak in West Africa where scientists, health workers and clinicians, amongst other stakeholders from around the world, openly worked together and were able to contain the spread of this pandemic [38] . On this front, Boué et al. [39] highlight that levels of trust and transparency need to be reviewed and enhanced to facilitate unfettered data generation and sharing. Such could lead to an even earlier detection scenario of future virus outbreaks, and in the better curative management of the same, without minimal compromise on urban functions and on an urban economy.\\n\\nFurthermore, in cases of emergencies like the current outbreak of COVID-19 and any other, the need for observance of regulatory practices and international healthcare guidelines are paramount. This would ensure that both healthcare professionals and the general populace are informed, protected and remain within the prescribed rules and regulations. As noted by the WHO [40] , the healthcare guidelines and regulatory practices are advanced to also ensure that the health risk in question is reduced together with its consequences. In the current era of technological advancement, such regulations and guidelines are paramount as they have potential to lead to positive or negative outcomes. The position of this paper is to advance that it now possible to integrate technologies like the use of smart devices through IoT networks and wearable devices, data from mobile apps and others to help users to share information with accredited and certified health professionals, and in this case, improve the outcomes for better cross disciplinary and more resilient protocols and policies.', 'document_id': 2527}]}, {'paragraphs': [{'qas': [{'question': \"What's the recommended procedure to disinfect at CT scanner after a COVID-19 exposure?\", 'id': 239, 'answers': [{'text': 'The equipment in the contaminated area are wiped with\\n2000mg/L chlorine-containing disinfectant. The DR and CT gantry in the contaminated\\narea are wiped with 75% ethanol. The equipment in the buffer area is wiped with\\n500-1000mg/L chlorine-containing disinfectant or alcohol-containing disposable\\ndisinfectant wipes twice a day.', 'answer_start': 18441}], 'is_impossible': False}, {'question': \"What's the recommended method to disinfect floors for COVID-19?\", 'id': 240, 'answers': [{'text': 'The ground is wiped with 1000mg/L chlorine-containing\\ndisinfectant, once /4 hours.', 'answer_start': 19312}], 'is_impossible': False}, {'question': 'What is the role of computed tomography (CT) in COVID-19?', 'id': 241, 'answers': [{'text': 'diagnosing and categorizing\\nCOVID-19 on the basis of case definitions issued by the WHO', 'answer_start': 5745}], 'is_impossible': False}, {'question': 'What kind of masks are recommended to protect healthcare workers from COVID-19 exposure?', 'id': 242, 'answers': [{'text': 'N95 mask', 'answer_start': 12064}], 'is_impossible': False}, {'question': 'What thickness of layers is recommended for CT image reconstruction in COVID-19 assessment?', 'id': 243, 'answers': [{'text': '1 mm-thick layers', 'answer_start': 15491}], 'is_impossible': False}, {'question': 'What must the data gathering include?', 'id': 2458, 'answers': [{'text': 'up-to-date case definitions and information about\\nconfirmatory tests to all staff with direct patient contact to allow appropriate barrier precautions\\nto be taken.', 'answer_start': 22205}], 'is_impossible': False}, {'question': 'What must be done to assist in recognition of the disease on images and to allow accurate reporting\\nof these findings?\\n', 'id': 2459, 'answers': [{'text': 'All typical and atypical imaging features of the disease should be made known to\\nall radiologists', 'answer_start': 22369}], 'is_impossible': False}, {'question': 'What is the major role of chest CT?', 'id': 2460, 'answers': [{'text': 'to understand the\\nextent and dynamic evolution of lung lesions induced by COVID-19', 'answer_start': 24384}], 'is_impossible': False}, {'question': 'What is the reason to adopt low-dose CT?', 'id': 2461, 'answers': [{'text': ' low-dose chest CT has been\\nwidely used in the screening of early lung cancer. It is well known that many early lung cancers\\nare ground-glass opacities (GGO), so we believe that low-dose screening is also applicable for\\nCOVID-19. In addition, considering the rapid development of COVID-19, many CT\\n14\\nexaminations may be conducted in the same individual to monitor disease progress. Low-dose\\nscanning can reduce the radiation damage to patients.', 'answer_start': 24546}], 'is_impossible': False}, {'question': 'What did the EMICT responsibilities include?', 'id': 2448, 'answers': [{'text': ' (1) coordination between the\\nhospital’s management and planning of infection control and radiology departments; (2)\\ncollection of the most up-to-date protection-related information to educate and train staff in the\\ndepartment; (3) reallocation of staff according to the actual situation; (4) establishment of the\\nCT procedures for patients with COVID-19; and (5) establishment of an emergency\\nmanagement plan for the radiology department to ensure that the department would run\\nnormally.', 'answer_start': 10070}], 'is_impossible': False}, {'question': 'How were the radiology department areas divided?', 'id': 2449, 'answers': [{'text': 'contaminated, semicontaminated,\\nbuffer, and clean areas', 'answer_start': 10860}], 'is_impossible': False}, {'question': 'How was the contaminated area connected to the CT room and other facilities?', 'id': 2450, 'answers': [{'text': 'connected to the fever clinic and\\nincludes the fever accessway, the CT examination room, and the DR examination room for\\n6\\nconfirmed and suspected cases. One CT scanner and one DR system closest to the emergency\\ndepartment are designated the fever-CT and fever-DR to examine patients with suspected and\\nconfirmed COVID-19. There is a separate dedicated access between the contaminated area and\\nthe fever screening tents. ', 'answer_start': 10953}], 'is_impossible': False}, {'question': 'What does the clean area include?', 'id': 2453, 'answers': [{'text': 'he clean area\\nincludes the administrative office and the diagnostic room.', 'answer_start': 11603}], 'is_impossible': False}, {'question': 'What does the semicontaminated area include?', 'id': 2451, 'answers': [{'text': ' the fever-CT control room,\\nfever-DR control room, and other patient examination access areas. ', 'answer_start': 11408}], 'is_impossible': False}, {'question': 'What does the buffer area include?', 'id': 2452, 'answers': [{'text': 'access areas for medical personnel and a dressing area for technologists. ', 'answer_start': 11528}], 'is_impossible': False}, {'question': 'How was the wearing and removing of the equipment performed?', 'id': 2454, 'answers': [{'text': 'under the supervision of\\nthe infection control nurse.', 'answer_start': 12203}], 'is_impossible': False}, {'question': 'What can lower the physical and mental stress levels of staff members?', 'id': 2455, 'answers': [{'text': ' periodically taking time off ', 'answer_start': 12348}], 'is_impossible': False}, {'question': 'Who must be assigned to the clean area?', 'id': 2456, 'answers': [{'text': 'Pregnant staff ', 'answer_start': 12636}], 'is_impossible': False}, {'question': 'What responses must EMICT consider once a disease has been identified?', 'id': 2457, 'answers': [{'text': 'data gathering, collaboration, needs assessment, and expert advice ', 'answer_start': 22091}], 'is_impossible': False}, {'question': 'What is the conclusion of the report?', 'id': 2462, 'answers': [{'text': 'Strategic planning and adequate protections can help protect patients and staff\\nagainst a highly infectious disease while maintaining function at a high volume capacity.\\nKeywords: Coronavirus, COVID-19, novel coronavirus pneumonia, infection control', 'answer_start': 4062}], 'is_impossible': False}, {'question': 'What were the number of cases in mainland china as of March 11th?', 'id': 2463, 'answers': [{'text': '80,793 ', 'answer_start': 4738}], 'is_impossible': False}, {'question': 'How many COVID deaths occurred in Chinese mainland as of March 11th?', 'id': 2464, 'answers': [{'text': '3,169 ', 'answer_start': 4765}], 'is_impossible': False}, {'question': 'How many people have come in contact and how many of these are in observation?', 'id': 2465, 'answers': [{'text': '677,243 people have been identified as having had close contact with\\ninfected patients of whom13,701 ', 'answer_start': 4931}], 'is_impossible': False}], 'context': 'The Battle Against Coronavirus Disease 2019 (COVID-19): Emergency Management\\nand Infection Control in a Radiology Department\\nhttps://www.jacr.org/article/S1546-1440(20)30285-4/pdf\\nJournal Pre-proof\\nZixing Huang, Shuang Zhao, Zhenlin Li, Weixia Chen, Lihong Zhao, Lipeng Deng, Bin\\nSong\\nPII: S1546-1440(20)30285-4\\nDOI: https://doi.org/10.1016/j.jacr.2020.03.011\\nReference: JACR 5139\\nTo appear in: Journal of the American College of Radiology\\nReceived Date: 24 February 2020\\nRevised Date: 13 March 2020\\nAccepted Date: 15 March 2020\\nPlease cite this article as: Huang Z, Zhao S, Li Z, Chen W, Zhao L, Deng L, Song B, The Battle Against\\nCoronavirus Disease 2019 (COVID-19): Emergency Management and Infection Control in a Radiology\\nDepartment, Journal of the American College of Radiology (2020), doi: https://doi.org/10.1016/\\nj.jacr.2020.03.011.\\nThis is a PDF file of an article that has undergone enhancements after acceptance, such as the addition\\nof a cover page and metadata, and formatting for readability, but it is not yet the definitive version of\\nrecord. This version will undergo additional copyediting, typesetting and review before it is published\\nin its final form, but we are providing this version to give early visibility of the article. Please note that,\\nduring the production process, errors may be discovered which could affect the content, and all legal\\ndisclaimers that apply to the journal pertain.\\n© 2020 Published by Elsevier Inc. on behalf of American College of Radiology\\nThe Battle Against Coronavirus Disease 2019 (COVID-19): Emergency Management\\nand Infection Control in a Radiology Department\\nZixing Huang*, Shuang Zhao*, Zhenlin Li, Weixia Chen, Lihong Zhao, Lipeng Deng,\\nBin Song\\nDepartment of Radiology, West China Hospital, Sichuan University, Chengdu, China\\n*Zixing Huang and Shuang Zhao contributed equally to this work as co-first author.\\nCorresponding Author: Bin Song, MD\\nAddress: Department of Radiology, West China Hospital, Sichuan University.\\nNo. 37, GUOXUE Alley, Chengdu, 610041, China\\nTel.: (+86)28 85423680, Fax: (+86)28 85582944\\nEmail: songlab_radiology@163.com.\\nAuthors’ contributions\\nZXH: conceived the study and drafted the manuscript.\\nZS: conceived the study and drafted the manuscript.\\nZLL: The member of the emergency management and infection control team (EMICT)\\nand was involved in the formulation of the measures.\\nWXC: The member of the EMICT and was involved in the formulation of the\\nmeasures.\\nLHZ: The member of the EMICT and was involved in the formulation of the\\nmeasures.\\nLPD: The member of the EMICT and was involved in the formulation of the\\nmeasures.\\nBS: Leader of the EMICT, conceived the study and reviewed the manuscript.\\nAll authors read and approved the final manuscript.\\nThe authors declare no conflict of interest.\\nThe authors declare that they had full access to all of the data in this study and the\\nauthors take complete responsibility for the integrity of the data and the accuracy of\\nthe data analysis\\n1\\nThe Battle Against Novel Coronavirus Pneumonia (COVID-19): Emergency\\nManagement and Infection Control in a Radiology Department\\nAbstract\\nObjective: To describe the strategy and the emergency management and infection control\\nprocedure of our radiology department during the COVID-19 outbreak.\\nMethods: We set up emergency management and sensing control teams. The team formulated\\nvarious measures: reconfiguration of the radiology department, personal protection and training\\nof staff, examination procedures for patients suspected of or confirmed with COVID-19 as well\\nas patients without an exposure history or symptoms. Those with suspected or confirmed\\nCOVID-19 infection were scanned in the designated fever-CT unit.\\nResults: From January 21, 2020 to March 9, 2020, 3,083 people suspected of or confirmed with\\nCOVID-19 underwent fever-CT examinations. Including initial examinations and\\nreexaminations, the total number of fever-CT examinations numbered 3,340. As a result of our\\nprecautions, none of the staff of the radiology department were infected with COVID-19.\\nConclusion: Strategic planning and adequate protections can help protect patients and staff\\nagainst a highly infectious disease while maintaining function at a high volume capacity.\\nKeywords: Coronavirus, COVID-19, novel coronavirus pneumonia, infection control\\n\\n2\\nIntroduction\\nThe whole world has been closely focusing on an outbreak of respiratory disease caused by a\\nnovel coronavirus that was first reported in Wuhan, China, on December 31, 2019, and that\\ncontinues to spread. On February 11, 2020, the World Health Organization (WHO) named the\\ndisease “coronavirus disease 2019” (COVID-19).\\nAs of 24:00 on March 11, 2020, the National Health Commission (NHC) had received reports\\nof 80,793 confirmed cases and 3,169 deaths on the Chinese mainland. There remain 14,831\\nconfirmed cases (including 4,257 in serious condition) and 253 suspected cases still\\nhospitalized. To date, 677,243 people have been identified as having had close contact with\\ninfected patients of whom13,701 are under medical observation [1]. Outside China, 44,067\\nlaboratory-confirmed cases and 1,440 deaths have occurred in 117 countries /territories/areas\\naccording to the WHO [2]. COVID-19 poses significant threats to international health. Like the\\nflu, COVID-19 is thought to spread mainly from person-to-person between people who are in\\nclose contact with one another through respiratory droplets produced when an infected person\\ncoughs or sneezes. In light of the infectious nature of this disease, healthcare workers are at\\nhigh risk of infection of COVID-19. In China, healthcare workers account for 1,716 confirmed\\ncases of COVID-19, including six deaths [3].\\n Computed tomography (CT) can play a role in both diagnosing and categorizing\\nCOVID-19 on the basis of case definitions issued by the WHO and the treatment guidelines\\nfrom the NHC [4]. Suspected patients having the virus may undergo chest CT. Isolation and\\nbarrier procedures are necessary to protect both the department staff and other patients in the\\nhospital. Note should be made that due to overlap of imaging findings with other respiratory\\n3\\ndiseases, CT is not helpful as a screening tool. But it can help identify the degree of pulmonary\\ninvolvement and disease course.\\nOur hospital is a national regional medical center with 4,300 beds and a tertiary referral\\ncenter in Sichuan province. The initial response started on January 21, 2020, after transmission\\nof COVID-19 was confirmed to be human-to-human on January 20, 2020. The first suspected\\ncase of COVID-19 in Sichuan province was reported on January 21, 2020. The Sichuan\\nprovincial government immediately launched the first-level response to major public health\\nemergencies. On the same day, our hospital was designated to care for Sichuan province\\npatients with COVID-19.\\nThis article describes the emergency management procedure of our radiology department\\nfor situations involving severe infectious diseases, such as COVID-19, and the\\ninfection-protection experience of the department staff.\\nMethods\\nThe hospital provided personal protective equipment (medical protective clothing,\\nsurgical cap, N95 mask, gloves, face shields, and goggles) to all its healthcare staff, erected\\nthree medical tents (fever tents) for screening of fever cases in the parking lot of the emergency\\ndepartment, planned an examination route and examination area for patients suspected of\\nharboring the virus, and placed confirmed patients in an isolation ward. “Fever” was the\\ncolloquial term used to designate suspected COVID-19 based on symptoms such as a fever or\\nwith an epidemiological history of a potential exposure as well as those with confirmed\\nCOVID-19 referred for treatment. Further, during outbreak, emergency and outpatient patients\\n4\\nwithout fever were asked for information such as epidemiological history and sent to fever tents\\nas long as they met suspected criteria.\\nThe radiology department has 65 diagnostic radiologists and 161 other staff members\\n(trained technologists, nurses, engineers, and support staff). The equipment of the radiology\\ndepartment includes 12 magnetic resonance (MR) scanners, 14 CT scanners, 15 digital\\nsubtraction angiography (DSA) systems, 32 sets of digital radiography (DR) systems\\n(including nine mobile bedside DR sets), and 130 imaging diagnostic workstations for picture\\narchiving and communication systems (PACS). Most of the equipment is distributed among\\nfour buildings at the hospital main campus. 4 CT scanners, 4 MR scanners, 1 DR are located on\\nthe first floor of the first inpatient building, and 9 DR and 8 DSA are located on the second\\nfloor. 1 CT and 1 MR scanner are located in the third inpatient building. 1 CT and 1 MR scanner\\nare located in the sixth inpatient building. 2 CT scanners, 2 MR scanners and 7 DSA are located\\nin the technical building. The rest of the equipment is located in the seventh inpatient building\\nin the branch campus.\\nThe first inpatient building, located next to the emergency department, was reconfigured to\\nhandle cases of COVID-19. Fever tents were set up by the emergency department in the\\nemergency department parking lot to separate normal emergency patients from patients with\\nsymptoms or exposure history suspicious of COVID-19. We established separate means of\\naccess between fever tents and between the fever examination area of the radiology department\\nto avoid cross-contamination.\\nThe emergency management and infection control measures, as described below and\\nimplemented in the radiology department during the outbreak, have been approved by the\\n5\\ninfection control committee of hospital. These measures are in accordance with relevant laws\\nand regulations, in order to protect patients as well as the staff.\\nRadiology Emergency Management and Infection Control Team (EMICT)\\nThe radiology department director chaired the EMICT. Its members include the deputy\\ndirector, chief technologist, head nurse, equipment engineer supervisor, and infection control\\nnurse of the radiology department. Team responsibilities included (1) coordination between the\\nhospital’s management and planning of infection control and radiology departments; (2)\\ncollection of the most up-to-date protection-related information to educate and train staff in the\\ndepartment; (3) reallocation of staff according to the actual situation; (4) establishment of the\\nCT procedures for patients with COVID-19; and (5) establishment of an emergency\\nmanagement plan for the radiology department to ensure that the department would run\\nnormally.\\nSuspected patients\\nThe suspected patients were identified according to the Diagnosis and Treatment Program of\\nthe Novel Coronavirus Pneumonia of the NHC [5], mainly based on epidemiological history.\\nReconfiguration of the radiology department\\nThe radiology department was divided into four areas [6]: contaminated, semicontaminated,\\nbuffer, and clean areas (Figure 1). The contaminated area is connected to the fever clinic and\\nincludes the fever accessway, the CT examination room, and the DR examination room for\\n6\\nconfirmed and suspected cases. One CT scanner and one DR system closest to the emergency\\ndepartment are designated the fever-CT and fever-DR to examine patients with suspected and\\nconfirmed COVID-19. There is a separate dedicated access between the contaminated area and\\nthe fever screening tents. The semicontaminated area includes the fever-CT control room,\\nfever-DR control room, and other patient examination access areas. The buffer zone includes\\naccess areas for medical personnel and a dressing area for technologists. The clean area\\nincludes the administrative office and the diagnostic room.\\nThe contaminated area was isolated from other areas using physical barricades.\\nDirectional signs were newly installed to guide patients and staff.\\nPersonal protection and training of staff\\nFor providing care for patients with confirmed and suspected COVID-19, all hospital staff\\nare required to wear complete personal protective equipment [7]: medical protective clothing,\\nsurgical cap, N95 mask, gloves, face shields, and goggles. Wearing and removing of the\\nequipment must be performed in accordance with the procedures and under the supervision of\\nthe infection control nurse.\\nBecause staff members working in the contaminated area are under much situational\\npressure, periodically taking time off could lower their physical and mental stress levels. The\\ntechnologists on fever-CT duty shifts are provided a break once a week for four hours. In\\naddition, the health of staff in the contaminated area must be monitored closely for the\\nsymptoms of COVID-19. Pregnant staff must be assigned to the clean area.\\n7\\nThe EMICT formulates and continually updates guidelines and educates all staff for West\\nChina Hospital of Sichuan University. The EMICT training for staff is mainly involves\\ndocuments regarding infection control and CT findings of COVID-19 and maintains an EMICT\\nWeChat group for West China Hospital of Sichuan University. WeChat is the most widely used\\nsocial media app in China. The EMICT releases the latest national and hospital-based\\ninformation regarding COVID-19, guidance documents, and other notices from the hospital\\nand radiology department in the WeChat group on a daily basis. Staff can also report to the\\nEMICT in the WeChat group any time. Protocols for each modality and infection control\\ninstructions are posted on the walls in all examination rooms. The EMICT periodically reminds\\nstaff to undertake personal measures to reduce infection, such as wearing masks at all instances\\nin the radiology department and N95 masks if working in the contaminated area; not touching\\nthe mask and the eyes; practicing hand hygiene; facing away from colleagues when eating,\\ndrinking, and talking; and not using personal cell phones while on duty.\\n In addition, the chief thoracic radiologist provided lectures on all radiologists and\\ntechnologists on typical CT findings of COVID-19 infection using materials developed in\\nWuhan, the epicenter of the outbreak in China.\\nCT examination procedures\\nThere are two sets of procedures for CT examination: the fever-CT procedure and routine CT\\nprocedure for those not suspected of COVID-19.\\nThe fever-CT procedure for suspected or confirmed COVID-19 (Figure 2)\\n8\\nBefore the fever-CT technologist operates the equipment, he or she should wear personal\\nprotective equipment according to three-level protection standard [8]. Before the CT\\nexamination of patients with suspected and confirmed COVID-19 begins, the fever tent or\\nisolation ward notifies the radiologist in advance. The fever-CT technologist checks the\\nequipment and prepares to disinfect the imaging equipment immediately after the examination.\\nThe patient enters the fever-CT waiting area through the fever access area. If the patient\\ncan get onto and off the examination table by themselves, the patient is allowed to do so. If the\\npatient cannot get onto or off the examination table independently, the person accompanying\\nthe patient assists the patient, rather than the technologist. The technologist checks the patient\\ninformation and, using an intercom system in the examination room, asks the patient to remove\\nany metal ornaments on the neck and chest. Also, by intercom, the technologist trains the\\npatient to hold his or her breath during the examination.\\nThe technologist uses a low-dose chest CT protocol to scan the patient. After scanning, the\\noriginal images are reconstructed as 1 mm-thick layers. The technologist browses the images to\\nensure that their quality meets the diagnostic requirements and then guides the patient to leave\\nthrough the fever access area. The disposable sheets for patient examination are changed after\\neach patient. The equipment is disinfected according to the procedure below.\\nTo protect themselves, the technologists assigned to the fever-CT wear N95 mask and\\nother personal protection as established by the EMICT.\\nThe CT procedure for regular patients (figure.3)\\n9\\nSome patients with COVID-19 have no symptoms, and they may call at the general clinic for\\nother reasons. The following CT procedure is applicable under these circumstances:\\nWhen the patient makes an appointment for examination, the staff asks the patient about\\ntheir epidemiological history, symptoms, and signs. If suspected criteria are met, the patient\\nwill be sent to the fever tent for further screening. When a patient presents to the radiology\\ndepartment entrance, his/her temperature is measured. If the temperature is higher than 37.2 , ℃\\nthe patient is sent to the fever tent for further investigation.\\nThose with no exposure history, suspicious symptoms or fever are screened in one of the\\nnon-contaminated CT scanners. The technologists assigned to these scanners wear surgical\\nmasks. All patients and the person accompanying them are required to wear surgical masks.\\nAfter the CT examination, the technologist browses the images quickly. If the CT appearance is\\ntypical of lung infection, the technologist immediately reports it to the chest radiologist on duty\\nand asks the patient to wait in the CT examination room. If the chest radiologist does not\\nsuspect COVID-19 infection, the patient can leave the CT examination room. If the chest\\nradiologist does suspect COVID-19 infection, the technologist immediately reports it to the\\nEMICT and sends the patient to the fever tent. The floor and equipment in the CT examination\\nroom are disinfected according to regulations, and air disinfection is conducted for 30 min\\nbefore examining other patients. These CT scanners are considered noncontaminated (not\\nfever-CTs) after these sterilization procedures.\\nFever-DR examination procedure\\n10\\nThe COVID-19 guideline of the NHC does not recommend chest DR because its ability in\\ndiagnosing COVID-19 is limited. At our hospital, we only use mobile DR units to provide\\nbedside examination for critically ill patients. The technologist operating the mobile DR\\nwears personal protective equipment according to the three-level protection standard and\\nsterilizes the mobile DR according to the ward management requirements as described below.\\nEquipment and environment disinfection procedures\\nRoutine disinfection procedure [9]\\n1) Object surface disinfection: Object surface is wiped with 1000mg/L chlorine-containing\\ndisinfectant, wipe twice with 75% ethanol for non-corrosion resistance, once /4 hours.\\n2) Equipment disinfection: The equipment in the contaminated area are wiped with\\n2000mg/L chlorine-containing disinfectant. The DR and CT gantry in the contaminated\\narea are wiped with 75% ethanol. The equipment in the buffer area is wiped with\\n500-1000mg/L chlorine-containing disinfectant or alcohol-containing disposable\\ndisinfectant wipes twice a day.\\n3) Air disinfection: Turning off all central air conditioners to prevent air contamination with\\neach other. Polluted area: open the door for ventilation, each time more than 30 minutes,\\nonce /4 hours; The air sterilizer is continuously sterilized or the ultraviolet ray is\\ncontinuously used in the unmanned state for 60 minutes, four times a day, remembered to\\nclose the inner shielding door when air disinfection. Other ambient air is sprayed with\\n1000mg/L chlorine-containing disinfectant and ventilated twice a day\\n4) Ground disinfection: The ground is wiped with 1000mg/L chlorine-containing\\ndisinfectant, once /4 hours.\\n5) When contaminated, disinfect at any time. In case of visible contamination, disposable\\nabsorbent materials should be used first to completely remove the pollutants, and then a\\ncloth soaked with 2000mg/L chlorine-containing disinfectant should be used for 30\\nminutes before wiping.\\n11\\nFever-CT disinfection procedures after examination\\nIn addition to the above, disinfect the examination bed and ground with chlorinated disinfectant\\ncontaining 2000mg/L [10].\\nNoncontaminated CT disinfection procedures after suspected COVID-19 case examination\\nIn addition to the above routine disinfection procedure, air disinfection is conducted for 30 min\\nbefore examining other patients.\\nResults\\nFrom January 21, 2020 when screening for epidemiological history or symptoms\\nsuspicious for COVID-19, to March 9, 2020, our hospital screened a total of 7,203 individuals\\nand confirmed 24 cases of COVID-19. Of these, 3,083 people underwent fever-CT\\nexaminations. Including the initial examination and reexamination, the total number of fever\\nCT examination numbered 3,340. The fever-CT scanned a patient approximately every 21.5\\nminutes. As a result of our precautions, none of the staff of the radiology department developed\\nsymptoms suspicious for COVID-19. The fever-CT technologist, with the highest probability\\nof exposure, remains PCR negative.\\nDiscussion\\nIt has been 17 years since the severe acute respiratory syndrome (SARS) epidemic, the last\\nnational spread of severe infectious disease, broke out. Currently, the Chinese people are\\npanicking again. The speed and extent by which COVID-19 has spread in 2 months are\\n12\\nunprecedented, beyond those of SARS, and this has been aided by its contagious nature and\\nrapid spread via droplets and contact. The droplet mode of transmission means that a person can\\nbe infected easily by means of casual contact or even fomites on contaminated environmental\\nsurfaces. Another theory has yet to be proved: aerosol propagation.\\nHow radiology departments respond to any infectious disease outbreak is determined\\nprimarily by the estimated risk of cross-infection to the staff and other patients. Appropriate\\nprecautions taken only by staff in direct contact with patients may be adequate when the risk is\\nlow. The strongest measures need to be implemented to limit the spread of the disease when the\\nrisk is high. With severe infectious diseases such as COVID-19, the highest level of infection\\ncontrol measures must be implemented; these include providing adequate standard protective\\nequipment, training staff, and instituting proper emergency plans.\\nOnce a contagious infectious disease has been identified, the EMICT must consider four\\nmain areas of response: data gathering, collaboration, needs assessment, and expert advice [10].\\nData gathering includes dissemination of up-to-date case definitions and information about\\nconfirmatory tests to all staff with direct patient contact to allow appropriate barrier precautions\\nto be taken. All typical and atypical imaging features of the disease should be made known to\\nall radiologists to assist in recognition of the disease on images and to allow accurate reporting\\nof these findings. We have stored images of all probable cases of COVID-19 in the PACS so\\nthat these images were readily available for any radiologist to review, and images from\\nprevious imaging studies are also available for comparison.\\nCollaboration with the radiology departments of other hospitals is very important because\\npatients may initially present to different centers, depending on geographic location and travel\\n13\\ndistance. These patients may be few in number at a single hospital, but if data from patients at\\nseveral hospitals are available, a more accurate overall understanding of both imaging features\\nand epidemiology can be achieved. Dissemination of this information to all healthcare facilities\\nwill also lead to early recognition of the disease, and appropriate isolation measures may be\\ninstituted.\\nThe Internet and social media apps, especially WeChat, have been used for distribution of\\nmedical information, and because the exchange of information regarding infectious disease\\noutbreaks is almost instantaneous, it is an indispensable tool for radiologists. In fact, within a\\nmonth of the outbreak, the hospital that received the most infected patients from the source of\\nthe outbreak made a PowerPoint presentation of the CT manifestations of COVID-19, which\\nwas shared via WeChat and disseminated across the country in a very short time. Subsequently,\\nCOVID-19-teaching PowerPoint presentations from various hospitals appeared and were\\nquickly shared via WeChat.\\nOur diagnostic process is limited as chest CT along is not diagnostic of COVID-19\\nbecause of lack of imaging specificity. But when combined with other epidemiological,\\nclinical, laboratory and virus nucleic acid information, typical chest CT imaging findings are\\nhelpful for making the diagnosis. In our opinion, the major role of chest CT is to understand the\\nextent and dynamic evolution of lung lesions induced by COVID-19. The reasons why we\\nadopted the low-dose chest CT scan protocol are as follows: low-dose chest CT has been\\nwidely used in the screening of early lung cancer. It is well known that many early lung cancers\\nare ground-glass opacities (GGO), so we believe that low-dose screening is also applicable for\\nCOVID-19. In addition, considering the rapid development of COVID-19, many CT\\n14\\nexaminations may be conducted in the same individual to monitor disease progress. Low-dose\\nscanning can reduce the radiation damage to patients.\\nAlthough the processes we established minimized the exposure of hospital staff, ancillary\\npersonnel and other patients, it remains limited as follows. Sichuan province is not the center of\\nthe epidemic. The number of patients with COVID-19 whom we have treated has not been\\nhigh, and most cases are from other provinces of China. However, we believe that our\\nexperience in management, the reconfiguration of our radiology department, and the workflow\\nchanges implemented in the current COVID-19 situation are useful for other radiology\\ndepartments that must prepare for dealing with patients with COVID-19. While no radiology\\npersonnel developed symptoms suspicious for or were confirmed as having COVID-19, there\\nmay be asymptomatic personnel.\\nREFERENCES\\n1. National Health Commission of the People’s Republic of China.(2020). March 12: Daily briefing\\non novel coronavirus cases in China. Retrieved from\\nhttp://en.nhc.gov.cn/2020-03/12/c_77618.htm. Accessed March 11, 2020.\\n2. World Health Organization. (2020). Coronavirus disease 2019 (COVID-19) Situation Report-52.\\nRetrieved from\\nhttps://www.who.int/docs/default-source/coronaviruse/20200312-sitrep-52-covid-19.pdf?sfvrsn=e\\n2bfc9c0_2 9. Accessed March 11, 2020.\\n3. National Health Commission of the People’s Republic of China.(2020). Latest developments in\\nepidemic control on Feb 15. Retrieved from http://en.nhc.gov.cn/2020-02/16/c_76622. Accessed\\nMarch 11, 2020.\\n15\\n4. Health Commission of the People’s Republic of China.(2020). The notification of the trial\\noperation based on the guideline version 6 in the coronavirus disease diagnosis and treatment.\\nRetrieved from\\nhttp://www.nhc.gov.cn/xcs/zhengcwj/202002/8334a8326dd94d329df351d7da8aefc2.shtml.\\nAccessed March 11, 2020\\n5. Health Commission of the People’s Republic of China.(2020). The notification of the trial\\noperation based on the guideline version 6 in the coronavirus disease diagnosis and treatment.\\nRetrieved from\\nhttp://www.nhc.gov.cn/xcs/zhengcwj/202002/8334a8326dd94d329df351d7da8aefc2.shtml.\\nAccessed March 11, 2020.\\n6. Health Commission of the People’s Republic of China.(2009). The guideline for pathogens\\nisolated operations in hospital. Retrieved from\\nhttp://www.nhc.gov.cn/wjw/s9496/200904/40116.shtml. Accessed March 11, 2020.\\n7. Health Commission of the People’s Republic of China.(2017). The guideline for prevention and\\ncontrol of hospital acquired infections of airborne pathogens. Retrieved from\\nhttp://www.nhc.gov.cn/wjw/s9496/201701/7e0e8fc6725843aabba8f841f2f585d2.shtml. Accessed\\nMarch 11, 2020.\\n8. Health Commission of the People’s Republic of China.(2017). The guideline for prevention and\\ncontrol of hospital acquired infections of airborne pathogens. Retrieved from\\nhttp://www.nhc.gov.cn/wjw/s9496/201701/7e0e8fc6725843aabba8f841f2f585d2.shtml. Accessed\\nMarch 11, 2020.\\n9. Health Commission of the People’s Republic of China.(2012). The standardization for\\nsterilization techniques in hospital. Retrieved from\\nhttp://www.nhc.gov.cn/wjw/s9496/201204/54510.shtml. Accessed March 11, 2020.\\n10. Health Commission of the People’s Republic of China.(2012). The standardization for\\nsterilization techniques in hospital. Retrieved from\\nhttp://www.nhc.gov.cn/wjw/s9496/201204/54510.shtml. Accessed March 11, 2020.\\n11. Katona P. Bioterrorism Preparedness: Generic Blueprint for Health Departments, Hospitals, and\\nPhysicians. Infectious Diseases in Clinical Practice. 2002;11(3):115-122. Accessed March 11,\\n2020.\\n16\\nFigure Legends\\nFigure 1. Diagram of the layout of our radiology department was divided into four areas: contaminated\\n(shaded in black), semicontaminated (shaded in dark gray), buffer (shaded in light gray), and clean areas\\n(shaded in white). The contaminated area was separated from other areas by barriers.\\nFigure 2. Diagram shows CT protocol for suspected and confirmed patients with COVID-19.\\nFigure 3. Diagram shows CT protocol for regular patients.\\nAbbreviations:\\nCOVID-19: coronavirus disease 2019\\nCT: computed tomography\\nDR: digital radiography\\nEMICT: emergency management and infection control team\\nNHC: National Health Commission\\nPACS: picture archiving and communication system\\nSARS: severe acute respiratory syndrome\\n\\n\\n\\nSentence Summary\\nWith severe infectious diseases such as COVID-19, the highest level of infection control\\nmeasures must be implemented, collaboration with the radiology departments of other\\nhospitals be needed, and social media be employed.\\nTake-home points\\n1. To response to a community infection emergency, a special emergency management team\\nneeds to be setup at the departmental level to implement infection containment and\\ncontrol procedures that continues to allow the imaging examination and imaging\\ndiagnosis of those with suspected infection, and to prevent intra-departmental spreading\\nof infection (EMICT).\\n2. Infection control measures, such as reconfiguration of department areas, personal\\nprotection and anti-infection training of all staff, standardized procedures including\\ncontact minimization for chest CT and DR examinations, and timely disinfection of CT\\nand DR examination rooms, should be implemented properly.\\n3. If there are more than one scanner in a hospital, only one of them should be assigned to\\nsuspected cases.\\n', 'document_id': 188}]}, {'paragraphs': [{'qas': [{'question': 'When was the novel Coronavirus first reported?', 'id': 1219, 'answers': [{'text': 'December 2019', 'answer_start': 974}], 'is_impossible': False}], 'context': 'Identification of COVID-19 Can be Quicker through Artificial Intelligence framework using a Mobile Phone-Based Survey in the Populations when Cities/Towns Are Under Quarantine\\n\\nhttps://doi.org/10.1017/ice.2020.61\\n\\nSHA: 83c96f2a481be06a5c58552cbad2ca67ce789dc2\\n\\nAuthors: Vazquez, Arni S.R. Srinivasa Rao; Jose A.\\nDate: 2020\\nDOI: 10.1017/ice.2020.61\\nLicense: cc-by\\n\\nAbstract: We are proposing to use machine learning algorithms to be able to improve possible case identifications of COVID-19 more quicker when we use a mobile phone-based web survey. This will also reduce the spread in the susceptible populations.\\n\\nText: Emerging and novel pathogens are a significant problem for global public health and technology can assist in faster identification of possible cases to bring timely interventions. This is especially true for viral diseases that are easily and readily transmissible and have asymptomatic infectivity periods. The novel Coronavirus (SARSCoV2) described in December 2019 has resulted in major quarantines throughout the world, including major cites, villages and public areas throughout China [1] [2] [3] to prevent further spread. As of February 25 th 2020, the World Health Organization\\'s situational data indicates that there were about 77780 confirmed cases, including 2666 deaths due to COVID-19, including cases in 25 countries [4] . The majority of the deaths reported so far have been in China only [5] .\\n\\nOrganization have issued interim guidelines in order to protect the population, and to attempt to prevent the further spread of COVID-19 from infected individuals [6] .\\n\\nSince cities and villages throughout China are unable to accommodate such large numbers of infected individuals, and be able to maintain the quarantined. China has built several new hospitals in an attempt to manage the infected individuals [7] . It is imperative that we evaluate novel models in an attempt to control the rapidly spreading virus [8] .\\n\\nIn order to reduce the time to identification of a person under investigation (PUI) for the COVID-19 infection, and the rapid isolation of this individual, we propose to collect the basic travel history along with the more common manifestations using a phone-based online survey. Such collected data can be used to assist in the preliminary screening and early identification of possible COVID-19 infected individuals. Thousands of data points are able to be collected and processed through an artificial intelligence (AI) framework which can ultimately evaluate individuals that may be infected and stratify them into no-risk, minimal-risk, moderate-risk, and high-risk of being infected with the virus. The identification of the high-risk cases can then be quarantined earlier, thus decreasing the chance of spread. Table 1 is inserted here.\\n\\nSee Appendix I for the details on the steps involved in data collection on all the respondents independent of whether or not they think they are infected. The AI algorithm described in Appendix II is to identify possible case identifications and send alerts to the nearest health clinic as well as to the respondent for an immediate health visit, we call this as an \"alert for health check recommendation for COVID-2019. In case the respondent is unable to commute to the health center, the health department can then send an alert to a mobile health unit so they can then do doorto-door assessments and even testing for the virus. This generates alert for mobile health check recommendation for 2019-nCoV (MHCRC). If a respondent does not have an immediate risk of having symptoms or signs related to the viral infection, then the AI-based health alert will be sent to the respondent to notify them that there is no current risk of COVID-2019. Figure 1 summarizes the outcomes of data collection and identification of possible cases. The data recorded in step 5 of the algorithm using signs and symptoms will be collected prior to both the groups who have received alerts HCRC or MHCRC (for possible identification and assessment) and NCRC (for non-identified respondents).\\n\\nThese are explained in steps (iii) and (iv) in the Appendix II. The extended analysis proposed will help to understand if there is any association with different sociodemographic variables and the manifestations such as fever and signs and lower respiratory infections, including cough and SOB in individuals defined as either with and without possible infection.\\n\\nApplications of AI and deep learning argued to be useful tools in assisting diagnosis and treatment decision making [10] [11] . There were studies which promoted disease detection through AI models [12] [13] [14] [15] . Use of mobile phones [16] [17] [18] [19] and web based portals [20] [21] have been tested successfully in health related data collection. However, one need to apply such techniques in a timely way for faster results. Apart from cost-effectiveness, the proposed modeling will be of great assistance in identifying and controlling when populations are closed due to virus spread. In addition to these, our proposed algorithm can be easily extended to identify individuals who might have any mild symptoms and signs.\\n\\nWe have developed our data collection criteria based on CDC\\'s Flowchart to Identify and Assess 2019 Novel Coronavirus [9] and added additional variables for the extended utility of our efforts in identifying infected and controlling the spread (see Table 1 ).\\n\\nLet be the outputs recorded during the data collection steps 1 (ii) If the set of identifiers, , for is equal to one of the elements of the set then send HCRC or MHCRC to that respondent, else proceed to the test criteria (iv).\\n\\nIf is equal to one of the elements of the set , for then the respondent will be sent an NCRC alert.\\n\\n(iv)\\n\\nIf is equal to one of the elements of the set , then the respondent will be sent an NCRC alert.\\n\\nComparison of test criteria results of (iii) and (iv) with their corresponding geographic and socio-demographic details will yield further investigations of signs and symptoms Suppose we define two events and using the sets and as below:\\n\\n: out of responded cases are identified through the algorithm out of have responded to the survey.\\n\\nThe conditional probability of the event given the event , say, is computed as', 'document_id': 2522}]}, {'paragraphs': [{'qas': [{'question': 'When did China detect the first human case of H7N9 infection?', 'id': 5320, 'answers': [{'text': 'February 2013', 'answer_start': 3243}], 'is_impossible': False}, {'question': 'How many deaths were associated with MERS-CoV as of July 2014?', 'id': 5321, 'answers': [{'text': '288', 'answer_start': 21748}], 'is_impossible': False}], 'context': \"Overview of the 3rd isirv-Antiviral Group Conference – advances in clinical management\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC4280814/\\n\\nSHA: f7bb1f005066cb4930f83cde4cdc1ff3fe411def\\n\\nAuthors: Hurt, Aeron C; Hui, David S; Hay, Alan; Hayden, Frederick G\\nDate: 2014-11-15\\nDOI: 10.1111/irv.12293\\nLicense: cc-by\\n\\nAbstract: This review highlights the main points which emerged from the presentations and discussions at the 3rd isirv-Antiviral Group Conference - advances in clinical management. The conference covered emerging and potentially pandemic influenza viruses and discussed novel/pre-licensure therapeutics and currently approved antivirals and vaccines for the control of influenza. Current data on approved and novel treatments for non-influenza respiratory viruses such as MERS-CoV, respiratory syncytial virus (RSV) and rhinoviruses and the challenges of treating immunocompromised patients with respiratory infections was highlighted.\\n\\nText: Recurrent infections by influenza and other respiratory viruses contribute enormously to the burden of human disease and (emergent) sporadic zoonotic infections, such as by influenza H7N9 and H5N1 and Middle East respiratory syndrome (MERS) coronavirus, pose a constant threat of a new global epidemic. Despite extensive knowledge of the viruses and their interaction with the host, there is little in our armoury of vaccines and therapeutics to combat this perpetual onslaught. The 3rd isirv Antiviral Group conference on Influenza and Other Respiratory Virus Infections: Advances in Clinical Management, convened in Tokyo, Japan on 4-6 June 2014, attracted 188 clinicians, public health specialists and medical scientists from 34 countries to present their recent research and discuss various aspects of the impact of respiratory viruses in different patient groups/settings and in different regions of the world. The programme 1 focused on the latest advances in the mitigation and clinical management of influenza and other respiratory virus disease, and the successful use of antivirals (and vaccines) against seasonal and pandemic influenza, particularly in Japan, as well as the development/assessment of novel antiviral agents. This overview highlights some of the main points which emerged from the presentations (both oral and poster) and associated discussion.\\n\\nIn recent years, an increasing number of cases of novel animal influenza A viruses infecting humans have been reported. These include multiple avian influenza A virus subtypes, in particular H5N1 and H7N9, and swine-origin H3N2v. Reasons for this increase include both social factors, for example, increased human populations living in close proximity to animals and increased surveillance and diagnostic testing. Risk assessment tools have been developed by many authorities around the world (e.g. the European CDC, US CDC, USAID and WHO) to assist in predicting the likelihood that a particular virus will emerge and its associated impact, as well as prioritising the development of candidate human vaccine viruses. 2 These tools allow continual reassessment as new data become available and provide an objective, transparent process with which to make resource allocation and pandemic planning decisions.\\n\\nIn February 2013, China detected the first human cases of H7N9 infection in severely ill patients with pneumonia. 3 As of May 22, 2014 , there have been 446 confirmed H7N9 cases in China resulting in 163 deaths. 4 The cases have occurred mainly during two waves (weeks [8] [9] [10] [11] [12] [13] [14] [15] [16] [17] [18] 2013 and week 40, 2013week 20, 2014) , 4 of which 85% had prior exposure to poultry or contaminated live poultry markets. The median time from poultry exposure to disease onset was 5 days, whereas the median time from illness onset to hospital admission, ARDS development, antiviral therapy and death was 5, 6Á8, 7 and 14 days, respectively. 5 Closure of live poultry markets has markedly reduced the risk of H7N9 infection. Across nine areas in the two most affected provinces in China, modelling analysis estimated that the effectiveness of market closure was 97% (95% CI: 89%, 100%). 6 A retrospective serological study of blood specimens taken in January-May and October-November in 2012 from 1544 subjects who worked in live poultry markets, farms, slaughter houses or kept backyard poultry revealed no evidence of H7N9 infection, 7 indicating widespread population susceptibility and lack of prior circulation of antigenically related viruses. Multiple family clusters have been reported, but no sustained human-to-human transmission, with studies demonstrating a very low detection of virus or specific antibody in close contacts (0Á34% and 0Á2% in the first and second waves, respectively) and healthcare workers of positive cases. 8 In both the first and the second waves, the majority of the patients hospitalised with H7N9 infection were older men (median age, 62 and 58 years, respectively with an overall male/female ratio of 2Á2:1) and the case fatality was similar (32% and 39%, respectively). Pre-existing medical conditions occurred in >60% of these cases. The prominent clinical features on admission were those of a severe influenza syndrome with fever, cough, fatigue and dyspnoea, while the most striking laboratory findings were marked lymphopenia and thrombocytopenia. Elevated cytokine levels have been observed in patients and such excessive cytokine responses may contribute to the clinical severity of H7N9 infection. 9 Originating from reassortment events involving at least three avian influenza viruses, H7N9 viruses with multiple genotypes continue to emerge on a more frequent basis in 2014. Many viruses isolated from humans contain the E627K amino acid substitution in the polymerase PB2 component, associated with mammalian adaptation, and the G186V and Q226L substitutions in the haemagglutinin (HA) that are associated with dual receptor binding to both a2,3 and a2,6-linked sialic acid receptors. 10 All H7N9 viruses from the outbreak to date are antigenically similar to the (original) candidate vaccine strain. Prototype inactivated whole particle H7N9 vaccines have been investigated in macaques and shown to induce good antibody responses that significantly reduced the number of days of virus shedding in experimentally infected animals. Treatment of H7N9-infected patients with neuraminidase inhibitors (NAIs), including intravenous (IV) peramivir or zanamivir, 11 appears to have been beneficial even when therapy was started late, although emergence of oseltamivir resistance has been associated with poor clinical outcomes. 12 All H7N9 viruses are amantadine-resistant due to the S31N substitution in the M2 ion channel protein, while viruses containing the R292K substitution in neuraminidase (NA), which confers resistance to both oseltamivir and peramivir (>1000-fold rise in IC 50 ) and reduced susceptibility to zanamivir and laninamivir (50-and 25-fold rises in IC 50 , respectively), have been reported in six cases. Two of these patients with severe H7N9 infection requiring extracorporeal membrane oxygenation (ECMO) also received systemic corticosteroid treatment leading to treatment failure and a poor clinical outcome. 12 The replication and transmission of H7N9 viruses containing the R292K NA mutation have been shown to be comparable to those of wild-type H7N9 viruses in guinea pigs, 13 and in ferrets following both contact and non-contact exposure. However, the wild-type virus did outgrow the R292K-resistant strain in some ferrets over the course of the infection. 14 Interestingly, the R292K variant appeared to be the dominant virus in ferret lung lobes, while in nasal turbinates, the wild-type virus was predominant. Therefore, it appears that the R292K mutation causes less fitness loss in H7N9 virus than in seasonal H3N2 viruses. 13\\n\\nA new reassortant genotype of H5N1 containing the HA and NA genes from clade 1.1.2 and the internal genes from clade 2.3.2.1 emerged during 2013 and was associated with the highest number of cases (n = 26) and deaths (n = 14) in Cambodia. 15 Globally since 2003, there have been 650 confirmed H5N1 cases and 386 deaths reported in humans, 16 with most infections in the last 2 years being in children. Human-to-human transmission remains extremely rare based on virological and serological data from analysis of close contacts of confirmed cases in Cambodia, Thailand and Vietnam. 17 A study of household transmission patterns in Indonesia has shown that the overall household attack rate was 18Á3% and the secondary attack rate was 5Á5%, independent of household size. 18 Oseltamivir therapy appears to reduce mortality when administered within 8 days of H5N1 illness onset, although earlier treatment is more effective, highlighting the need for early patient diagnosis. 19 Early initiation of oseltamivir was particularly effective in reducing mortality in H5N1 patients without respiratory failure (odds ratio, 0Á17; P = 0Á04), whereas those requiring ventilatory support at the time of oseltamivir initiation were more likely to die. 20 A study of the risk factors for mortality related to H5N1 identified age, country, per capita government health expenditure and delay from symptom onset to hospitalisation as the key parameters, highlighting the importance of early diagnosis, treatment and supportive care. 21 High-dose systemic corticosteroids (SC) are associated with worse outcomes in H5N1 patients. 22 One human case of avian H5N6 was recently detected in China; the virus was a reassortant that contained seven genes from H5N1 and the NA gene from an H6N6 virus circulating in ducks. 23 China has also reported the detection of three human infections, two fatal, with avian H10N8 viruses that contain the internal genes from H9N2, as does H7N9. 24 Like the H7N9 virus, the H10N8 virus has low pathogenicity in poultry and is therefore difficult to detect in birds.\\n\\nBurden in target populations Pregnant women and infants have an increased risk of complications following influenza infection. Globally, significant numbers of pregnant women died during the 2009-2010 pandemic, but no maternal mortality occurred in Japan. 25 Through education campaigns directed at pregnant women and healthcare professionals, 67% of pregnant women were vaccinated against H1N1pdm09 resulting in an infection rate among pregnant women in Japan of 3Á5% compared to the overall infection rate in the population of 12%. 26 Of those pregnant women who were infected with H1N1pdm09 in Japan, 95% were treated with antivirals, and importantly, 88% of those were treated within 2 days of symptom onset. 25 In Mongolia, a prospective cohort study during 2013-2014 found that influenza-like illness (ILI) was detected in 17Á9% of pregnant women, of whom the majority tested positive for influenza A, with substantially lower influenza B and respiratory syncytial virus (RSV) infection. 27 During the same period, ILI was detected in 30Á9% of infants <6 months of age, with an even spread of influenza A, influenza B and RSV. 27 The influenza burden in children in a rural Indian community was found to be substantial with 11Á6% of ILI cases being caused by influenza A or B viruses. 28 These findings underscore the importance of maternal immunisation. 29 Transmission patterns Sequence analysis of influenza viruses isolated from students on a Singapore University Campus provided insights into the chain of transmission, showing that 62% of 32 viruses were highly similar, demonstrating that the majority of transmission was occurring on the university campus rather than from infections outside. 30 The effectiveness of surgical masks, hand hygiene and health education investigated in households in Hong Kong and Bangkok detected no significant difference in attack rate in cohorts using one of these interventions. 31 Further analysis of the data enabled some insights into the relative importance of aerosol, large droplet and contact transmission within the households. For influenza A infections, aerosol transmission appeared to be the most common route, whereas contact transmission caused the highest number of influenza B infections. 31 \\n\\nUse and effectiveness Neuraminidase inhibitors are commonly used for the treatment of influenza in Japan, typically following a positive result from a point-of-care (POC) test. During the 2009-2010 pandemic, over 20 million POC test kits were shipped to hospitals and clinics in Japan to enable rapid diagnosis, and 89% of treated cases were administered NAIs within 48 hours of symptom onset. 32 In Japan during 2013, oseltamivir and laninamivir each represented 40% of NAIs used, while zanamivir (15%) and peramivir (5%) use was considerably less. NAI effectiveness has been assessed in numerous observational studies in Japan. Oseltamivir effectiveness is significantly reduced in patients with delayed treatment, and duration of fever and viral shedding is longer in treated patients with influenza B compared to influenza A virus infections. 33, 34 The reduced effectiveness against influenza B viruses was also observed in zanamivir 33, 35 and laninamivir 35 trials.\\n\\nTo determine whether NAIs reduced mortality during the 2009-2010 pandemic, data were compiled on 29 234 patients hospitalised with confirmed A(H1N1)pdm09 infection. 36 Compared with no treatment, NAI treatment was associated with significantly reduced mortality, with early treatment also showing a reduced risk of mortality compared to late treatment. Although there was no significant clinical effect when comparing late treatment with no treatment in hospitalised patients, there was a significant benefit in treating patients who arrive late into intensive care units. 36 In a household prophylaxis study, inhaled laninamivir given for either 2 or 3 days reduced the illness rate within households to 3Á9% and 3Á7%, respectively, compared to 16Á9% in households given a placebo. 37 A ferret model of oseltamivir prophylaxis has shown that while morbidity was significantly reduced, the prophylaxis regimes did not prevent infection nor significantly reduce virus load. 38 Resistance Although all four NAIs are sialic acid analogues, they have subtle differences in chemical structure and binding properties. Consequently, resistance patterns vary across NAIs. The most commonly detected NA substitution causing NAI resistance in N1-containing influenza viruses is H275Y, which confers resistance to oseltamivir and peramivir, but not to zanamivir and laninamivir. This resistance mutation became fixed in seasonal H1N1 viruses circulating in 2008-2009. 39 A late 2013 cluster of H1N1pdm09 viruses containing the H275Y substitution was detected in 38 (39%) of 97 H1N1pdm09 viruses from community patients not receiving NAIs in Sapporo, Japan, 40 reminiscent of a similar cluster of oseltamivir-resistant H1N1pdm09 viruses in community patients in Australia in 2011. 41, 42 Importantly, both sets of viruses contained permissive NA mutations (V241I and N369K) that have been shown in ferret studies to offset the destabilising and negative effect of the H275Y NA mutation. 43 In hospitalised influenza patients being treated with intravenous zanamivir, next-generation sequencing has been utilised to identify minor resistant virus populations. A total of five NA substitutions were identified in different viruses, including E119K and E119D; however, all apart from E119D were present in such low proportions that they could not be detected by Sanger 'population' sequencing methods. 44 The effects of various mutations in catalytic and framework residues of influenza B NA were investigated using reverse genetics and a range of functional assays. Four substitutions (D198E, I222T, H274Y and N294S) conferred reduced susceptibility to oseltamivir, while three substitutions (E119A, D198Y and R371K) caused highly reduced inhibition by oseltamivir, zanamivir and peramivir. 45 Two of these variants (H274Y, E119A) had in vitro replication fitness comparable to the NAI-susceptible viruses. To date, these substitutions have only been detected on rare occasions in circulating influenza viruses.\\n\\nAn intravenous formulation of zanamivir showed both virological and clinical effectiveness without safety concerns in patients hospitalised with influenza in Japan. 46 A range of new adamantane derivatives have good antiviral activity in vitro and in animal models against H1N1pdm09 and H3N2 viruses that contain the S31N M2 ion channel substitution that confers resistance to amantadine. 47 Favipiravir is a novel pyrazinamide molecule that inhibits replication of various RNA viruses, including influenza types A, B and C (including oseltamivir-resistant strains), and has recently been licensed in Japan for the control of novel or reemerging influenza viruses. Its triphosphate metabolite is an RNA polymerase inhibitor which disrupts virus genome replication; synergy with oseltamivir has been demonstrated in pre-clinical models. 48 A phase II study in the US has shown that a twice-daily regimen decreased the titre and time to cessation of virus shedding, and had a significant benefit in reducing clinical symptoms (NCT01068912; www.clinicaltrials.gov). Subsequent phase III studies are currently ongoing (NCT02008344 and NCT02026349).\\n\\nA neutralising monoclonal antibody (MHAA4549A) which binds to the HA stalk of influenza A viruses in both group 1 and group 2 HA subtypes has been effective when given up to 72 hours post-infection in mice and ferrets infected with H5N1. 49 Phase I and IIa trials in humans showed that the antibody was well tolerated, had a mean half-life of 21Á9 days and was effective as therapy at high doses in experimentally infected volunteers (NCT01877785). Upcoming placebo-controlled phase IIb trials will target hospitalised influenza patients requiring oxygen and compare the combination of the monoclonal antibody with oseltamivir to oseltamivir monotherapy (NCT01980966). Other broadly neutralising antibodies against multiple clades of H5N1 have been generated by glycan masking of key HA antigenic residues to direct antibody responses to the more conserved stem region of the HA. 50 FluPep, a novel peptide that prevents virus entry into cells, has been shown in mouse studies to be effective in reducing virus titres in lungs, inflammatory cytokines and mortality. 51 Fludase (DAS-181) is a host-targeted therapeutic agent that removes sialic acid from cellular receptors in the respiratory tract, thus preventing influenza virus binding. Delivered topically, it is effective in animal models of lethal H5N1 and H7N9 infection, including a NAI-resistant R292K H7N9 variant. 52 In a phase 2 RCT, inhaled DAS181 reduced pharyngeal viral replication in uncomplicated influenza but did not reduce nasal virus loads or improve clinical outcomes. 53 Another receptor-targeted approach is the development of multivalent sialic acid-binding proteins; 54 a single administration 7 days pre-infection resulted in the protection of 80-100% of mice from lethal H7N9 challenge. 55 Apart from blocking sialic acid, the compound appears to stimulate the expression of pro-inflammatory mediators, thereby 'preparing' the immune system for subsequent influenza infection. When delivered 24 hours post-infection, protection was, however, only 20-40%. 55 Although drug resistance is considered less likely to occur with host-directed therapies, escape mutants have developed rapidly following exposure to a host-directed vacuolar ATPase-inhibiting drug. 56 Furthermore, following serial passage of different viruses in the presence of bafilomycin A1, two HA mutations were selected (A19T and S210N) which resulted in reduced drug susceptibility and increased virulence in mice. 56 \\n\\nMultiple influenza vaccine effectiveness (IVE) studies have used the control test negative design approach to estimate IVE during early and late phases of influenza seasons, the 2009 pandemic, and by age or target groups. Typically, IVE estimates range from 40% to 60% each season. 57 Future studies will investigate IVE with respect to the type of influenza vaccine used, whether IVE differs between the start and end of the season and the effect of previous vaccination. In Japan in 2013/14, IVE for influenza A in children aged 1-5 years averaged 72% (95% CI 64-79), dropped to 48% (95% CI 31-61) in children aged 6-12 years and was not apparent against influenza B in any age group (À1%, 95% CI À19 to 14). 58 Influenza vaccine effectiveness is known to be lower in adults over 65 years of age, a group that accounts for >60% of seasonal influenza-related hospitalisations and >90% of influenza-related deaths. In an effort to improve IVE in the elderly, recent RCTs have investigated the use of adjuvants, intradermal injection and higher doses of antigen. While the use of AS03-adjuvanted influenza vaccine was only moderately superior to non-adjuvanted vaccine in the elderly, 59 the use of a high-dose vaccine containing four times the standard level of HA (60 lg per virus) did result in improved effectiveness compared to the standard dose vaccine. 60 For vaccine manufacturers, generating high-growth reassortants of certain circulating viruses can be challenging. A recent study used random mutagenesis of PR8 and selection of high-growth clones in MDCK and Vero cells to derive a high-growth version of PR8. 61 Reassortment of the highgrowth PR8 virus with the HA/NA of either H5N1, H7N9 or seasonal influenza viruses showed that yields significantly exceeded equivalent reassortants that contained the internal genes of the 'normal' PR8 virus. 61 \\n\\nAs of July 2014, the number of confirmed cases of MERS-CoV has exceeded 830, with at least 288 associated deaths. 62 The majority of cases have involved patients with comorbidities (76%) and are predominately males (63%) with a median age of 47. 63, 64 Fewer than 25% of patients have reported contact with animals including dromedary camels, which have been shown to be one likely animal reservoir based on sero-positivity and detection of MERS-CoV. 65 More than 25% of the infections have been in healthcare workers, and the large number of nosocomial infections is likely due to inadequate infection control in hospitals plus enhanced surveillance that has detected a substantial number of mild or asymptomatic infections. 63 Outside hospital, the burden of disease is likely to be larger than has been reported. 66 Serological analysis of several UK patients found a rapid rise in antibodies from day 10, and that titres were maintained for at least 300 days post-infection. Anti-S (spike glycoprotein) antibodies are responsible for virus neutralisation. Importantly for serological analyses, patients who experience only mild disease may mount only a modest serological response. 67 Sequential samples from three cases involved in a chain of transmission were extensively analysed using next-generation sequencing. 68 Various minority variants were detected, of which some were transient while others were transmitted, and there was evidence of variation in frequency of some variants in different body compartments.\\n\\nVarious therapeutic options have been investigated for the treatment of MERS-CoV, but no therapy of proven value currently exists. The use of SC was associated with adverse outcome in SARS 69 and is not recommended for MERS-CoV. Many agents have shown inhibitory effects against MERS-CoV in cell culture including interferon +/À ribavirin, cyclosporine A, mycophenolic acid, chloroquine and lopinavir. 70 Interferons, lopinavir, mycophenolate, possibly alisporivir and combinations are reasonable choices for testing in controlled clinical trials. Exploratory post hoc metaanalysis of studies related to SARS and severe influenza has shown a significant reduction in mortality following convalescent plasma treatment compared to placebo or no therapy (odds ratio 0Á25; 95% CI 0Á14-0Á45). 71 Thus, the early use of virus-specific neutralising antibodies in the form of convalescent plasma and monoclonal or polyclonal neutralising antibodies for treatment of MERS-CoV has the highest likelihood of clinical benefit. 64 Modalities with risks likely to exceed benefits include SC, ribavirin monotherapy and IVIG. 72\\n\\nRespiratory syncytial virus (RSV) Respiratory syncytial virus disproportionately impacts children in low-income countries. 73 Almost all children will have been infected with RSV by their 2nd birthday, and it is the number one cause of hospitalisation of infants in the US, causing 10 times more infant deaths than influenza. 74 In addition, RSV infects 3-10% of adults annually and accounts for 5-15% of community acquired pneumonia (CAP) and 9-10% of hospitalisations, 75 a burden of disease that approaches that caused by influenza. In a study, conducted in Hong Kong, of 607 hospitalised adults with RSV, 40% had pneumonia and 70% required supplementary oxygen; mortality rates and duration of hospital stay were similar to those observed for influenza patients. 75 Approximately, 15% of hospitalised RSV patients had bacterial superinfections. Although corticosteroids were used to treat 38% of patients, treatment had no benefit on clinical outcome, and instead increased bacterial secondary infections and caused a longer duration of illness. 75 RSV replication appears prolonged in patients with comorbidities and LRT complications.\\n\\nPalivizumab prophylaxis of premature infants of <6 months of age has been shown to reduce hospitalisation due to RSV by 55%. 76 Preventing RSV during infancy has been associated with reduction of wheezing later in life. 77 Trials of other monoclonal antibodies have typically shown that they do not achieve superiority compared to palivizumab and therefore do achieve licensure. Furthermore, treatment with neutralising monoclonal antibodies does not appear to reduce virus load or disease severity in hospitalised infants. 78 Alternative options for RSV therapy to be assessed in future clinical trials include inhaled nanobodies, aerosolised peptides, nucleoside analogues and RNA-interference molecules.\\n\\nHuman rhinoviruses (HRV) usually cause mild acute respiratory infections, but on occasions can also cause more severe respiratory infections, including exacerbations of asthma and COPD. Of 115 Japanese children with asthma, a respiratory virus was detected in 86%, of which HRV (n = 36) or RSV (n = 47) were most common. 79 Ex vivo bronchial epithelial cells from people with asthma are more susceptible to HRV infection, due to deficient induction of IFN-b and IFN-lambda. In a study of 147 asthmatics on inhaled corticosteroid therapy, with a history of virusassociated exacerbations, patients were randomised to 14-day treatment with inhaled IFN-b or placebo within 24 hours of developing cold symptoms. Patients who received IFN-b had enhanced morning peak expiratory flow recovery, reduced need for additional treatment and boosted innate immunity as assessed by blood and sputum biomarkers. In an exploratory analysis of a subset of more difficult-to-treat asthma (n = 27 IFN-b; n = 31 placebo), worsening of symptoms increased significantly in the placebo group, but was prevented by IFN-b (P = 0Á004). 80 A picornavirus-specific antiviral, vapendavir, was found to reduce symptom scores, lower bronchodilator puffer use and reduce viral load in asthma patients with an URTI due to HRV. 81 \\n\\nDiagnostics Point-of-care tests that can deliver a result in 15 minutes have been available in many countries for the last decade, but while having good specificity, the sensitivity has typically been poor, ranging from 10% to 80% compared to PCR or culture. Their use in emergency departments of hospitals can result in reduced unnecessary antibiotic use and an increased likelihood of discharge. Newer immunofluorescence-based POC tests with improved sensitivity are being developed. In addition, the Quidel Sofia POC test may be linked via the internet such that results can be reported in real-time to central databases. Other POC tests are using photographic silver amplification immunochromatography technology to increase sensitivity. 82 PCR remains the gold standard for virus diagnostics with an ability to be rapid, sensitive, specific and to identify a wide range of pathogens via different assays. The ability to multiplex multiple pathogen targets allows costs to be reduced in a diagnostic setting. New closed-system technologies which involve only minimal hands-on time (a few minutes) and that conduct both automated nucleic acid extraction and PCR for multiple pathogens are now available, but are currently limited for clinical diagnostic purposes due to low-throughput capabilities. 83 Next-generation sequencing technologies and PCR-based analyses with increasing sensitivity both offer considerable scope in diagnosis, although our current understanding of the clinical impact of pathogens at low levels or the presence of variants as minor virus populations is limited. Providing low-cost, sensitive assays for diagnosing respiratory virus infections in low/middle-income countries is challenging, but has the potential to improve treatment and avoid unnecessary antibiotic use in these regions.\\n\\nRepurposed drugs for respiratory viral infections Nitazoxanide (NTX) is an antiparasitic agent approved for Giardia and Cryptosporidium infections that also inhibits replication in vitro of influenza and other respiratory viruses. 84 Treatment with NTX 600 mg twice daily for 5 days was associated with a reduction in the duration of symptoms in participants with acute uncomplicated influenza. 85 In a subset analysis of 238 patients with no confirmed virus infection, treatment with NTX 600 mg also led to a shorter time to alleviation of symptoms in comparison to placebo (88Á4 versus 105Á7 hours, P = 0Á02). 86 Systemic corticosteroids for respiratory virus infections A review of prospective observational studies has shown that SC increased the risks of mortality and morbidity (e.g. secondary infections, hospital-acquired pneumonia) in severe infection due to influenza A(H1N1)pdm09 especially with delayed antiviral therapy. 87 During SARS infections, a higher risk of avascular necrosis and prolonged virus shedding were observed in patients who had received highdose SC therapy. 88 It is therefore important to avoid the use of high-dose SC in severe respiratory viral infections outside the context of clinical trials. Larger trials are needed to resolve the uncertainty regarding the effect of early SC therapy in ARDS. Low-dose SC is indicated for management of refractory septic shock, 89 and a short course of SC is indicated for acute exacerbations of obstructive airway diseases (asthma, COPD) 90 Current evidence does not support a clinically relevant effect of systemic or inhaled glucocorticoids on admission or length of hospitalisation for acute viral bronchiolitis in infants and young children. 91 \\n\\nRespiratory syncytial virus, influenza viruses, parainfluenza (PIV) viruses and adenoviruses (AdVs) cause the most serious disease in immunocompromised hosts, but other respiratory viruses are becoming increasingly appreciated as a cause of both upper and lower respiratory tract disease. The potential for these viruses to cause lower respiratory tract infections (LRTI) after transplantation varies. Human metapneumovirus infections have similar outcomes to RSV infection in hematopoietic stem cell transplant (HSCT) recipients, including potentially severe and fatal pneumonia. HRV and coronavirus infections are very frequent in transplant recipients, but severe lower respiratory tract disease is uncommon.\\n\\nIn a prospective study of 112 lung transplant recipients, the virus infection rates upon screening, routine and emergency visits were 14%, 15% and 34%, respectively. Picornaviruses were identified most frequently in nasopharyngeal (85/140; 61%) and BAL specimens (20/34; 59%). Asymptomatic virus carriage, mainly of picornaviruses, was found at 10% of screening visits. Infections were associated with transient lung function loss and high calcineurin inhibitor blood levels. The hospitalisation rate was 50% for influenza and PIV and 16Á9% for other viruses. Acute rejection was not associated with virus infection. 92 The risk factors for severe LRTI among transplant recipients include early onset post-transplant (<3 m), steroid boluses, young children (<1 year), chronic GVHD, lymphopenia/lymphodepletion and allogeneic HSCT patients. 92 Influenza Immunocompromised patients with influenza exhibit more complications, longer virus shedding and more antiviral resistance, while often demonstrating milder clinical symptoms and signs on initial clinical assessment. 93 Influenza A (H1N1)pdm09 viruses have the potential for rapid emergence of oseltamivir resistance and causing severe morbidity, particularly in immunocompromised patients with lymphopenia and delayed antiviral therapy. 94 Influenza viraemia may serve as a marker for overall poor outcome with increased risk of progression to LRTI, hypoxaemia, respiratory failure and death. Influenza RNA in blood (viraemia) was detected in nine of 79 (11Á4%) HSCT recipients with influenza. Among patients with LRTI, viraemia was associated with increased hazards of overall as well as influenzaassociated death (hazard ratio 3Á5, 1Á1-12). 95 In 143 HSCT recipients with documented seasonal influenza infection, treatment with high-dose corticosteroids was associated with a trend towards prolonged virus shedding [(OR), 3Á3; 95% CI 1Á0-11; P = 0Á05], whereas antiviral therapy initiated to treat upper respiratory tract infection (URTI) was associated with fewer cases of LRTI (OR, 0Á04; 95% CI, 0-0Á2; P < 0Á01) and fewer hypoxaemia episodes (OR, 0Á3; 95% CI, 0Á1-0Á9; P = 0Á03). 96 In view of the risks of prolonged replication and drug resistance emergence, 97 a longer duration and a higher NAI dose may be beneficial. Early therapy was consistently demonstrated to have improved outcomes. [98] [99] [100] [101] Other treatment options under study are the use of triple combination therapy with amantadine, oseltamivir and ribavirin, 102 or intravenous peramivir 103 or zanamivir. 104 Therapy of influenza in lung transplant recipients is associated with a reduced risk of developing bronchiolitis obliterans syndrome. 105 Parainfluenza DAS181 is inhibitory for PIV and influenza viruses, including those resistant to the amantadine and NAIs 106 and may be effective in treating immunocompromised patients with severe PIV lung disease. 107 In a study of four severely immunocompromised children with PIV disease treatment with DAS181 for 5-10 days, by dry powder inhalation or nebulisation, was well tolerated. Transient increase in serum alkaline phosphatase, liver function and coagulation tests were observed, but nasal wash virus loads were reduced in all patients within 1 week with improved clinical features. 108 \\n\\nIn a RCT of lung transplant recipients with RSV infection, the incidence of new or progressive bronchiolitis obliterans syndrome at day 90 was significantly reduced in 16 patients who received a small interfering RNA against the RSV Ngene (ALN-RSV01) compared with placebo (n = 8) (6Á3% versus 50%, P = 0Á027). 109 In a larger follow-up multicentre phase IIb study, treatment with ALN-RSV01 showed a greater than eightfold reduced risk in developing bronchiolitis obliterans syndrome at day 180. 110\\n\\nAdenovirus is a serious, often fatal infection in immunocompromised patients, especially in HSCT recipients. The control of AdV is mostly T-cell mediated, and therefore, patients who have received T-cell suppressive regimens are at an increased risk for AdV infection. The annual incidence of AdV infections in HSCT recipients ranges from 5% to 50%, and is increasing, likely due to increased use of T-celldepleted allografts and cord blood as source. The mortality rate is up to 80%. 111 Brincidofovir (BCV; formerly CMX-001) is an orally bioavailable lipid-conjugate of cidofovir (CDV) that provides high intracellular concentrations of CDV diphosphate with a long intracellular half-life (up to 4-6Á5 days). BCV is 65-fold more potent against AdV than CDV in vitro with a low risk of myeloidor nephrotoxicity, but gastrointestinal side effects are more common. 112 In a retrospective study of 13 immunocompromised patients given BCV for AdV disease after failing or intolerance to i.v. cidofovire nine patients (69Á2%) demonstrated a virological response (VR), which was defined as a 99% drop from baseline or undetectable AdV DNA in serum by week 8. Patients with VR had longer survival than those without VR (median 196 days versus 54Á5 days; P = 0Á04). 113\", 'document_id': 1714}]}, {'paragraphs': [{'qas': [{'question': 'What illness is caused by the 2019-nCOV Coronavirus?', 'id': 1149, 'answers': [{'text': 'The novel coronavirus (2019-nCoV) infection caused pneumonia. ', 'answer_start': 563}], 'is_impossible': False}, {'question': 'In addition to oral swabs, which tests detected the presence of 2019-nCOV virus?', 'id': 1151, 'answers': [{'text': 'the 2109-nCoV RNA was readily detected in the blood (6 of 57 patients) and the anal swabs (11 of 28 patients). ', 'answer_start': 782}], 'is_impossible': False}, {'question': 'What  is the relationship between the presence of virus in blood and anal swabs and disease severity?', 'id': 1160, 'answers': [{'text': 'all of the 6 patients with detectable viral RNA in the blood cohort progressed to severe symptom stage, indicating a strong correlation of serum viral RNA with the disease severity (p-value = 0.0001). Meanwhile, 8 of the 11 patients with annal swab virus-positive was in severe clinical stage.', 'answer_start': 906}], 'is_impossible': False}, {'question': 'Which patients were classified as severe in Chinese guidelines?', 'id': 1168, 'answers': [{'text': 'Patients who were with at least one of the following symptom should be diagnosed to be severe case, 1) distress of respiratory with respiratory rate > = 30/min; 2) Oxygen saturation < = 93% in the rest state, and 3) arterial oxygen tension (PaO₂) over inspiratory oxygen fraction (FIO₂) of less than 300 mm Hg. In the blood detection cohort (Figure 1 (A)), patients who had at less one serum sample measurement with the PCR method were included. ', 'answer_start': 4705}], 'is_impossible': False}, {'question': 'What is the relationship between the presence of virus in blood sample and disease severity?', 'id': 1169, 'answers': [{'text': ' In the 57, 6 cases were detected to be blood positive, all of them (100%) were severe in symptom requiring special care attention, and the blood of the rest 51 cases was without detectable virus in the blood, only 12 of them (23.5%) were severe cases.', 'answer_start': 5150}], 'is_impossible': False}, {'question': 'What test could give an indication for special care for 2019-nCOV patients?', 'id': 1171, 'answers': [{'text': 'presence of viral RNA outside of the respiratory tract might herald the severity of the disease and alarm the requirement of special care', 'answer_start': 9149}], 'is_impossible': False}, {'question': 'What is the relationship between the presence of virus in anal swabs and disease severity in 2019-nCOV?', 'id': 1170, 'answers': [{'text': 'In the anal swab cohort (Figure 1 (B)), 11 of 28 cases were detected to be anal swab positive, 8 of them (72.7%) were with severe symptoms, which was significantly higher than that 4 (23.5%) of the rest 17 cases without detectable virus in anal were severe cases', 'answer_start': 5505}], 'is_impossible': False}, {'question': 'What could be the implication of 2019-nCOV virus in anal swabs?', 'id': 1172, 'answers': [{'text': 'digestive tract might be one extrapulmonary site for virus replication', 'answer_start': 10737}], 'is_impossible': False}, {'question': 'What  could account for the high transmission rate of the 2019-nCOV virus?', 'id': 1173, 'answers': [{'text': 'Intensive structural analysis of the S protein of 2019-nCoV with the SARS-Coronavirus suggested that several critical residues in the viral spike protein might confer favourable interaction with human ACE2 [7] . Of note, ACE2 is also abundantly present in humans in the epithelia of the small intestine besides the respiratory tract and is ubiquitously present in endothelial cells [8] , which might provide possible routes of transmission, and might account for the high transmission capacity of the new virus. ', 'answer_start': 11258}], 'is_impossible': False}, {'question': 'What could account for the dissemination of the 2019-nCOV virus across the whole body?', 'id': 1174, 'answers': [{'text': 'We propose that rampant coronavirus replication in pulmonary alveolus results in the breakdown of the alveolar vessel and the subsequent virus leakage into the blood flow, through which the virus is disseminated across the whole body. Then the virus succeeds in establishing reinfection in the digestive tract by using the highly expressed ACE2 receptor, which exacerbated the disease vice versa', 'answer_start': 11770}], 'is_impossible': False}], 'context': \"Detectable 2019-nCoV viral RNA in blood is a strong indicator for the further clinical severity\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054964/\\n\\nSHA: 77b0c98d1a2ca46b219ad090074814c387c80d8f\\n\\nAuthors: Chen, Weilie; Lan, Yun; Yuan, Xiaozhen; Deng, Xilong; Li, Yueping; Cai, Xiaoli; Li, Liya; He, Ruiying; Tan, Yizhou; Deng, Xizi; Gao, Ming; Tang, Guofang; Zhao, Lingzhai; Wang, Jinlin; Fan, Qinghong; Wen, Chunyan; Tong, Yuwei; Tang, Yangbo; Hu, Fengyu; Li, Feng; Tang, Xiaoping\\nDate: 2020-02-26\\nDOI: 10.1080/22221751.2020.1732837\\nLicense: cc-by\\n\\nAbstract: The novel coronavirus (2019-nCoV) infection caused pneumonia. we retrospectively analyzed the virus presence in the pharyngeal swab, blood, and the anal swab detected by real-time PCR in the clinical lab. Unexpectedly, the 2109-nCoV RNA was readily detected in the blood (6 of 57 patients) and the anal swabs (11 of 28 patients). Importantly, all of the 6 patients with detectable viral RNA in the blood cohort progressed to severe symptom stage, indicating a strong correlation of serum viral RNA with the disease severity (p-value = 0.0001). Meanwhile, 8 of the 11 patients with annal swab virus-positive was in severe clinical stage. However, the concentration of viral RNA in the anal swab (Ct value = 24 + 39) was higher than in the blood (Ct value = 34 + 39) from patient 2, suggesting that the virus might replicate in the digestive tract. Altogether, our results confirmed the presence of virus RNA in extra-pulmonary sites.\\n\\nText: The 2019 novel coronavirus (2019-nCoV), originally outbreaking from Wuhan China, has transmitted in an extremely short period to 25 countries and infected over 31 000 individuals as of Feb 06, 2020, causing an international alarm. Basic scientific research has achieved significantly in the investigation of viral origination [1, 2] , transmission and evolution [3] , and unprecedented public health control actions in China have been activated and effectively prevented the otherwise dramatic spread. The 2019-nCoV virus seems more infectious in its public transmission capacity compared to the well-known 2003 SARS virus in spite of the unavailability of convincingly scientific evidence. The mechanism of viral transmission is still worthy of further exploration.\\n\\nCurrently, one urgent and critical challenge is to treat infected patients and save their lives. Several studies have roughly described the overall clinical features of 2019-nCoV patients [4, 5] . However, the more specific and classified clinical characteristics of the infected patients still require further investigation, particularly for those with severe symptoms, which is roughly estimated to be approximately 15-20 percent of totally confirmed cases based on the local data in our hospital. Clinically, for those severe patients, the main symptoms of 2019-nCoV pneumonia are fever, decreased white blood cell and lymphocyte count, increased C reaction protein and abnormally expressed cytokines [6] .\\n\\nOne remaining question to be resolved is whether the 2019-nCoV virus can replicate in extra-pulmonary sites, which might account for the deteriorated clinical manifestation. In this study, we investigated whether the patients with severe clinical symptoms exhibited special profiles of virus replication or/and distribution compared to those only with mild symptoms.\\n\\nPatients, who were confirmed to be infected by the 2019-nCoV virus, were firstly enrolled in or transferred to Guangzhou Eighth People's Hospital for treatment purposes. This study followed the guideline of the Ethics Committee of Guangzhou Eighth People's Hospital. All blood, pharyngeal swab, and anal swab samples were collected for diagnostic purposes in the laboratory and our study added no extra burden to patients. Viral RNA was extracted with Nucleic Acid Isolation Kit (Da'an Gene Corporation, Cat: DA0630) on an automatic workstation Smart 32 (Da'an Gene Corporation) following the guidelines. Real-time reverse transcriptional polymerase chain reaction (RT-PCR) reagent (Da'an Gene cooperation, Cat DA0930) was employed for viral detection per the protocol. In brief, two PCR primer and probe sets, which target orf1ab (FAM reporter) and N (VIC reporter) genes separately, were added in the same reaction tube. Positive and negative controls were included for each batch of detection. Samples were considered to be viral positive when either or both set(s) gave a reliable signal(s).\\n\\nAll patients had pneumonia-based diseases but with diversified clinical manifestation. To simplify data analysis, the patients were only classified as either mild or severe clinical symptom groups based on the guideline newly released by Chinese government. Patients who were with at least one of the following symptom should be diagnosed to be severe case, 1) distress of respiratory with respiratory rate > = 30/min; 2) Oxygen saturation < = 93% in the rest state, and 3) arterial oxygen tension (PaO₂) over inspiratory oxygen fraction (FIO₂) of less than 300 mm Hg. In the blood detection cohort (Figure 1 (A)), patients who had at less one serum sample measurement with the PCR method were included. In the 57, 6 cases were detected to be blood positive, all of them (100%) were severe in symptom requiring special care attention, and the blood of the rest 51 cases was without detectable virus in the blood, only 12 of them (23.5%) were severe cases. The ratio of severe symptoms between these two groups was significantly different (p value = 0.0001). In the anal swab cohort (Figure 1 (B)), 11 of 28 cases were detected to be anal swab positive, 8 of them (72.7%) were with severe symptoms, which was significantly higher than that 4 (23.5%) of the rest 17 cases without detectable virus in anal were severe cases.\\n\\nFortunately, two cases with detectable virus both in blood and anal swab cohort were recorded. Patient 1 (Figure 2 (A)) was admitted to ICU after enrollment evaluation and was highly suspected infection with 2019-nCoV because of his recent travelling from Wuhan and of confirmed pneumonia by radiographic diagnosis with 5-day fever and 1-day continuous dry coughing. He was then confirmed to be infected by the 2019-nCoV virus on illness day 6 by CDC. High concentrations of the viral RNA were detected in the pharyngeal swabs on illness days 5 (Ct = 17 + 25), 7, 8 (Ct = 25 + 26), and 11 (Ct = 15 + 25). In the blood, no viral RNA was detected on day 5 but the sample on day 6 gave a weak positive signal (Ct = Neg+39), and then the signal was gone again on day 8. On day 9, a low level of viral RNA (Ct = 36 + 41) was detected again in the blood. On day 12, the blood lost signal again. A high concentration of virus RNA (Ct = 23 + 27) was detected in the anal sample on day 13, on the day the 2019-nCoV virus was not detected in the pharyngeal swab. Unfortunately, he was transferred out to another hospital after an emergency expert consultation.\\n\\nPatient 2 (Figure 2 (B)), who had a clear infection history and started fever 5-day ago and dry coughing 2-day ago, was admitted with clinically highly suspect of 2019-nCoV infection, considering the radiographical diagnosis which indicated clear pneumonia in the bilateral lung lobes. The virus was detected in his blood on illness day 7 (Ct = 34 + 36) and 8 (Ct = 38 + 38). His infection was also informed by the CDC on day 8. Because his disease advanced very fast, he was transferred to the ICU ward for special medical care requirements on day 9, on which day high titers of virus (Ct = 25 + 36) were detected in the pharyngeal sample. Importantly, virus RNA was detected in all pharyngeal (Ct = 23 + 24), blood (Ct = 34 + 39) and anal (Ct = 24 + 29) samples on day 10. He was transferred out to another hospital after an emergency expert consultation.\\n\\nFinally, we described here the four patients with detectable serum viral RNA. Patient 3 (Figure 3(A) ) was transferred to the ICU directly on illness day 11 because of his severe condition, the 2019-nCoV virus was laboratory detected both in pharyngeal (Ct = 30 + 30) and blood samples (Ct = 37 + 39) on day 12, And his infection was confirmed by CDC on day 13. Pharyngeal samples were PCR positive on days 14 and 17 and became negative on day 22. Patient 4 (Figure 3(B) ) was transferred to the ICU ward on the illness day 6 with a CDC confirmation. His disease advanced pretty fast and became severe on day 7 and he was transferred to ICU after his blood sample was detected to be virus-positive (Ct = 32 + 37). On day 9, he was transferred out. Patient 5 (Figure 3(C) ) was admitted on illness day 4 and his blood sample was virus-positive (Ct = 38 + Neg) on day 6. Her disease progressed rapidly to a severe stage within the next 3 days. Patient 6 ( Figure 3 (D)) with a clear history of virus infection was confirmed to be infected on infection day 7. Viral RNA was detected in his blood sample on day 9, one day ahead of his transfer into ICU. As his condition worsens, he was transferred out on day 13.\\n\\nIn this retrospective study, we analyzed the PCR data of virus detection in different tissues in our laboratory. Firstly, our observation indicated that the presence of viral RNA outside of the respiratory tract might herald the severity of the disease and alarm the requirement of special care. In the blood test cohort, all the 6 infected patients were in (or later progressed to) severe disease stage when serum viral RNA became detectable, which showed a significant difference compared to the blood negative group (p = 0.0001). Patient 2 (Figure 2(B) ), 5 (Figure 3 (C)) and 6 ( Figure 3(D) ) all had detectable viral RNA in the serum before they progressed to the clinical severe symptom stage. Unfortunately, we missed the earlier time points of patient 1 (Figure 2(A) ) and 3 (Figure 3(A) ) who were directly admitted to ICU on transfer to our hospital because of severe condition, of patient 4 (Figure 3(B) ) who had serum sample collected one day post the diagnosis of severe illness. We, fortunately, observed high serum viral load in serum within their severe illness stage. In the anal swab cohort, we found that the presence of virus RNA in the anal digestive tract was also positively correlated with disease severity (p = 0.0102). The 3 patients detected with anal virus RNA but in mild stage should be monitored whether they will progress to the severe stage. We have summarized the information of approximately 70 percent of the patients in Guangzhou city, and the study represented nearly the whole picture of this region. However, the virus outbroke in such an emergence, allowing no delay in waiting for more patients to further confirm the findings.\\n\\nSecondly, a high concentration of viral RNA in anal swabs suggested the digestive tract might be one extrapulmonary site for virus replication. For patient 1, a high concentration of viral RNA (Ct = 23 + 27, on day 13) was detected in anal swab but not in pharyngeal (the same day) and blood (1 d ahead). For patient 2, higher concentrations of viral RNAs were detected in anal swab (Ct = 24 + 39) and pharyngeal swab (Ct = 23 + 24) than in the blood (Ct = 34 + 39) on the same day. Angiotensin-converting enzyme 2 (ACE2) still is one of the receptors for 2019-nCoV attachment and entry [2] . Intensive structural analysis of the S protein of 2019-nCoV with the SARS-Coronavirus suggested that several critical residues in the viral spike protein might confer favourable interaction with human ACE2 [7] . Of note, ACE2 is also abundantly present in humans in the epithelia of the small intestine besides the respiratory tract and is ubiquitously present in endothelial cells [8] , which might provide possible routes of transmission, and might account for the high transmission capacity of the new virus. We propose that rampant coronavirus replication in pulmonary alveolus results in the breakdown of the alveolar vessel and the subsequent virus leakage into the blood flow, through which the virus is disseminated across the whole body. Then the virus succeeds in establishing reinfection in the digestive tract by using the highly expressed ACE2 receptor, which exacerbated the disease vice versa. Bat originated coronavirus was found to replicate in the swine digestive tract recently, also suggesting the potential replication possibility in the human digestive tract [9] . Nevertheless, confirmation of virus transmission through the digestive tract warrants further virus isolation from the anal swab in high safety level lab.\\n\\nUnfortunately, in our study, we did not collect stool samples from patients and did not pursue viral RNA in the stool. But we believe the existence of virus RNA in the stool samples from these patients because that a large amount of viral RNA was detected in anal swabs and that viral RNA had also been detected in a case reported from the United States [10] . Also, we didn't collect sputum and bronchoalveolar lavage fluid for virus detection because that the dry coughing characteristic of patients infected with 2019-nCoV prevents producing enough amount of sputum and that bronchoalveolar lavage fluid collection requires a sophisticated operation which increases virus exposure possibility of care providers to high concentrations of virus-containing aerosol.\\n\\nIn summary, we find that the presence of viral RNA in the blood and anal swab is positively correlated with the severe disease stage and that early monitoring of virus RNA in blood and the digestive tract on top of the respiratory tract might benefit the disease prediction.\", 'document_id': 2519}]}, {'paragraphs': [{'qas': [{'question': 'When did the World Health Organization declare the Ebola epidemic in West Africa as a Public Health Emergency of International Concern?', 'id': 2178, 'answers': [{'text': '8 August 2014', 'answer_start': 2494}], 'is_impossible': False}, {'question': 'What is PPE?', 'id': 2181, 'answers': [{'text': 'Personal Protective Equipment', 'answer_start': 6407}], 'is_impossible': False}, {'question': 'Where did the 2014 Ebola epidemic in West Africa spread to?', 'id': 2183, 'answers': [{'text': 'neighbouring sub-Saharan countries, North America, and Europe', 'answer_start': 20518}], 'is_impossible': False}, {'question': 'What are the prerequisites for successful emergency preparedness for an epidemic?', 'id': 2184, 'answers': [{'text': 'connectedness between health professionals, technological devices, and knowledge', 'answer_start': 25212}], 'is_impossible': False}], 'context': '‘Tiny Iceland’ preparing for Ebola in a globalized world\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC6507955/\\n\\nSHA: efd94d1135c5ee11c2af624b344881e079a5ce7a\\n\\nAuthors: Gunnlaugsson, Geir; Hauksdóttir, Íris Eva; Bygbjerg, Ib Christian; Pinkowski Tersbøl, Britt\\nDate: 2019-05-07\\nDOI: 10.1080/16549716.2019.1597451\\nLicense: cc-by\\n\\nAbstract: Background: The Ebola epidemic in West Africa caused global fear and stirred up worldwide preparedness activities in countries sharing borders with those affected, and in geographically far-away countries such as Iceland. Objective: To describe and analyse Ebola preparedness activities within the Icelandic healthcare system, and to explore the perspectives and experiences of managers and frontline health workers. Methods: A qualitative case study, based on semi-structured interviews with 21 staff members in the national Ebola Treatment Team, Emergency Room at Landspitali University Hospital, and managers of the response team. Results: Contextual factors such as culture and demography influenced preparedness, and contributed to the positive state of mind of participants, and ingenuity in using available resources for preparedness. While participants believed they were ready to take on the task of Ebola, they also had doubts about the chances of Ebola ever reaching Iceland. Yet, factors such as fear of Ebola and the perceived stigma associated with caring for a potentially infected Ebola patient, influenced the preparation process and resulted in plans for specific precautions by staff to secure the safety of their families. There were also concerns about the teamwork and lack of commitment by some during training. Being a ‘tiny’ nation was seen as both an asset and a weakness in the preparation process. Honest information sharing and scenario-based training contributed to increased confidence amongst participants in the response plans. Conclusions: Communication and training were important for preparedness of health staff in Iceland, in order to receive, admit, and treat a patient suspected of having Ebola, while doubts prevailed on staff capacity to properly do so. For optimal preparedness, likely scenarios for future global security health threats need to be repeatedly enacted, and areas plagued by poverty and fragile healthcare systems require global support.\\n\\nText: Global health; prevention and control; public policy; qualitative evaluation; emergency responders; communicable diseases; emerging; fear Background On 8 August 2014, the World Health Organization declared the Ebola epidemic in West Africa as a Public Health Emergency of International Concern (PHEIC) under the International Health Regulations (IHR) [1] . All three of the worst affected countries were to address the emerging epidemic challenge without staff, stuff, space and systems [2] [3] [4] . With the epidemic seemingly out of control, and a proportionately high number of doctors, nurses, and midwives succumbing to Ebola [5] , there was a growing fear of transmission beyond the region. In breach of WHO recommendations and guidelines [6] , flights were cancelled and cross-border movement curtailed [7] . The epidemic caused public concern outside West Africa [8] , as fear and racism found fertile ground [9] [10] [11] , and in an effort to stop the international spread of the disease, all states were advised to be prepared to detect, investigate, and manage Ebola cases [1] .\\n\\nPreparedness as part of disaster risk reduction is defined as \\'the knowledge and capacities developed by governments, response and recovery organizations, communities and individuals to effectively anticipate, respond to, and recover from the impacts of likely, imminent or current disasters\\' [12] . Yet, preparedness is also enveloped in and influenced by the socio-cultural dimension at the individual, organizational, and national levels, and measures to manage outbreaks are not always accepted or accommodated by the communities to which they are applied [13] . An analysis of eight European countries\\' preparedness plans since 2009 for countering a future influenza A (H1N1) pandemic revealed that the way plans were framed varied considerably, and \\'[told] us something about how the different countries want pandemics and preparedness to be understood by the public\\' [14] . More research was encouraged into cultural and social structures in the respective countries.\\n\\nIn Iceland, information about the Ebola epidemic in West Africa came from several sources. The Directorate of Health (DH) first reported on the epidemic on 8 April 2014 [15] . In Icelandic media, the rapid progress of the Ebola epidemic in West Africa was increasingly highlighted, and exported Ebola cases to Spain, USA, and elsewhere, were widely covered. Fear of a global epidemic was rife, and in media and online discussions, doubts were raised about the Icelandic health system´s capacity to take care of a patient with Ebola [16] [17] [18] , despite its ranking as one of the best in the world [16] .\\n\\nOn 11 August 2014, three days after WHO declared PHEIC because of Ebola, DH encouraged Icelandic citizens to avoid visits to the area, if possible, and reported that the national epidemic preparedness plan was being activated for Ebola [19] . It was elaborated by a team that involved the Chief Epidemiologist at the DH, Landspitali University Hospital (LSH), the Department of Civil Protection and Emergency Management (DCPEM), and the seven Primary Healthcare Regional Organizations in the country at the time. Key external partners were the European Centre for Disease Prevention and Control (ECDC) and WHO, in addition to Nordic collaborators in epidemic preparedness [20] . At the same time, it was regarded as highly unlikely that Ebola Virus Disease (EVD) would spread in the country [21] . Recognized scenarios included the possible appearance of an infected person in need of treatment, who could be either an Icelandic citizen who had visited or worked in one of the affected West African countries, or a person with signs of EVD on a trans-Atlantic flight in the navigation area controlled by Icelandic authorities [22] [23] [24] [25] . On 3 November 2014, the plan was put to the test when a foreign airline made a non-scheduled landing at Keflavík International Airport due to fear of EVD in one passenger from South Africa. Parked in a closed-off area, a physician in full Personal Protective Equipment (PPE) entered the plane, but quickly ruled out Ebola [26] .\\n\\nIrrespective of good or bad overall performance, health systems are tested in times of crisis, such as epidemics. Here, the aim is to describe and analyse the process of establishing preparedness plans for Ebola in Iceland, with a specific focus on the perspectives and experiences of managers and frontline health workers involved in the process.\\n\\nThis study is part of a larger study on the impact that the global threat of the Ebola epidemic had in Iceland [16, 27] . Qualitative case study methodology was applied, perceiving the preparedness planning and training process as the case with clear boundaries of the initiation, process, and wrap-up of preparedness planning and training. The study was conducted in April-May 2016, and the interviewed participants were administrators and frontline health professionals central to the case, so as to explore their perspectives and experiences concerning Ebola preparedness [28, 29] . Staff in managerial positions were contacted by one of the authors (GG) for permission to interview them based on their role in the preparedness plan. To identify potential interviewees in the Ebola Treatment Team (ETT), the director of the team listed relevant email contacts. Those who responded positively were subsequently invited for an interview, conducted in Icelandic by one of the authors (ÍEH), a physiotherapist. In case interviewees suggested other potential participants, they were invited through email to participate. A similar methodology was applied to identify participants from the Emergency Room (ER). They were included in order to represent frontline health workers who worked in the only ER in Reykjavík, where persons exposed to EVD were most likely to first seek care in case of acute illness.\\n\\nThree separate interview guides were developedone each for managers, ETT, and ER respectively (see supplementary material). The interviews included open questions probing the role of their institution in preparedness, the experience of the training process, challenges encountered or expected, and any dilemmas that they may have experienced in relation to the preparedness plan. The recruitment of participants was concluded when saturation was reached. Each interview was recorded and took about 20 to 60 minutes; they were then transcribed and analysed using thematic analysis. The data material was read through repeatedly, sorted, and categorized, based on the participants\\' priorities in the representation of their views. From this exercise, three broad themes were inductively identified that corresponded to critical perspectives introduced by the participants.\\n\\nPermission to conduct the study was granted by Iceland\\'s National Bioethics Committee (VSN- and Landspitali University Hospital (LSH 13-16, 4 February 2016) . Reporting on the results was guided by the COREC guidelines [30] ; however, to ensure anonymity of the respondents within the small community of staff who took part in the preparedness activities, participant information is not associated to quotations.\\n\\nThe Icelandic Ebola Preparedness Plan included the establishment of an ETT within LSH [31] , and the preparatory activities engaged more than two hundred staff across all of its departments. The ETT consisted of about 50 healthcare professionals who had volunteered to participate, including 11 doctors and 28 nurses, a few laboratory technicians, radiologists, and auxiliary nurses. They attended special training sessions focused on protocols for admission and treatment of a patient with EVD, the donning/doffing of PPE, and personal protective measures during patient care. A new provisory unit was designed to be set up on the ground floor to minimize the risk of infection spreading to other units within the hospital, with two rooms specifically identified for the care of a patient with EVD [31] .\\n\\nManagers\\' accounts of this period elaborated the complexity of preparedness planning in terms of the involved institutions, actors, procedures and requirement of the plan. One manager concluded:\\n\\nYou get no discount. You can never go the shorter way. There was always something that surprised you. We thought this was a lot like a three headed monster, so when you chopped off one of its heads, three other emerged, every solution was followed by more problems.\\n\\nThe health professionals who volunteered to join ETT did so for different reasons. Ebola preparedness was \\'a job that had to be done\\', and \\'someone had to do it\\'. Some referred to ethical or professional obligations: This is just a part of being a nurse, to encounter situations that can be dangerous to you or someone else, but you have made this decision and you deal with it. Some connected their decision to their \\'action gene\\' or \\'addiction to taking risks\\', while others said they had already raised their kids and had years of experience, including work with other epidemics, such as HIV. Yet, the practice of volunteering in the preparation was questioned. One participant said:\\n\\nWe learned that we could not rely on volunteers … when you work in an infectious disease department you cannot choose what infections you want to work with.\\n\\nER staff indicated that for them working in the ER was enough of a risk to take, no reason to expose oneself even more by joining the ETT, and appreciated that others had volunteered.\\n\\nAll participants noted that co-operation and communication had generally functioned well during the preparedness planning, with information flowing both ways. Short communication lines within the healthcare system were perceived as both a strength and a weakness; a strength, insofar as people knew each other, but a weakness because of the uneven burden of workload. Staff of the ETT and in the ER felt they had been well-informed, and that openness and honesty had characterized the planning and diminished their initial fear. Those in managerial positions had listened and taken their opinions into consideration. One said:\\n\\nThey were honest, no one was hiding anything, everything was on the table, no one tried to make things more appealing and say that everything would be OK, they just told us about things as they were.\\n\\nBoth management and participants from the ETT and ER expressed their ambiguity in terms of trust, doubt, and fear. Participants conveyed trust in the health system and their own role as health professionals, while at the same time admitting to facing formidable challenges during the elaboration of the preparedness plan. Facilities for isolation and treatment of patients with Ebola were less than perfect:\\n\\nWe assessed how we could use the department … and change it in just a few hours into some kind of an isolation unit that we could possibly use.\\n\\nSome compared this short-term isolation facility to a \\'camping site\\', as the facilities were too provisional and not comparable to those found elsewhere. There was also doubt about how many Ebola patients LSH would be able to care for: \\'Maybe one or two patients, barely more\\'.\\n\\nRespondents believed that the training and education of the members of the ETT and ER had been satisfactory. They felt that it had been proportionate to the risk, while some were concerned about the lack of staff. Nonetheless, there were contradictions on the division of labour among the professionals, exemplified by different ideas on how to proceed if a patient suspected of having an EVD came in an ambulance to the LSH for treatment. Almost all participants stated that they were ready to do their part in the Ebola response, or \\'as ready as [we] could be\\'.\\n\\nThere were diverse opinions on what it meant to be ready: to treat one confirmed case of Ebola, one suspected case, or more EVD patients? When asked if Ebola was a real threat to the country, participants usually referred to how easy it was to travel the globe: \\'Yeah, why not, the world is getting smaller\\'. Although Ebola was thought of as a real danger by many, some participants expressed difficulty in taking their training seriously, doubting that Ebola would ever reach Iceland. One respondent said:\\n\\nPeople were dedicated in the beginning, but when the news appeared that Ebola was receding, that diminished, and I never felt like this formally ended.\\n\\nParticipants described their relief that nothing really happened, while emphasizing the need to experience a real situation to evaluate the preparedness efforts. One participant said that \\'a little bit more seriousness [would have been] needed in the PPE practices\\'.\\n\\nIt was taken as a manifestation of fear that some of the staff in the communicable disease department of the LSH refused to take part in the ETT. When describing their fears, ETT members frequently connected it to their working conditions. Many of them were afraid that they would not get the best PPE, others that they would not do the donning/doffing correctly and, lastly, they were worried about work performance while in the PPE. One participant said:\\n\\nWhat bothered most of us was how uncomfortable the PPE was and I think that made people nervous: \"How will I manage working in this for hours?\"\\n\\nAnother described the donning/doffing process like a \\'complicated ballroom dance\\'. Moreover, participants were afraid of \\'unknown territories\\', that is, they did not know the hospital ward, they were supposed to work in, and some team members had no recent experience of clinical work. One participant said: I didn\\'t think these [non-clinical] people belonged in the team, because this is a very clinical environment in addition to having to be in this costume [PPE] with the risk of becoming infected by mistake.\\n\\nThose with non-clinical background were, however, aware of their limitations: I realized that I would not be the one in the front, I would not be managing patients directly.\\n\\nThe importance ascribed to teamwork was evident in relation to fear. Participants described fear of working with people they had not worked with before:\\n\\nThe weakest link in the preparation was that even though I knew their faces, I had never worked with them.\\n\\nAnother issue was no-show by some team members in training sessions or in lectures: This is team-work, one does this and the other one does this, [we] help each other. Then you don\\'t want to be working with someone who didn\\'t show up.\\n\\nThere were a lot of doctors who just dropped in, dropped out, and then dropped in again. I asked myself: Are these individuals … ready to take this on?\\n\\nParticipants in the ETT mentioned the precautions they took or intended to take to cope with their feelings of fear, should Ebola emerge in Iceland. A major precaution was planning to avoid contact with the family while working with Ebola patients. One participant said: \\'You thought … about your children at school … parents in the neighbourhood …\\' if they knew (s)he was working with an Ebola patient. For them, it was important they would have access to special accommodation in case of clinical EVD work \\'so I wouldn\\'t be exposing anyone or creating hysteria\\'. ETT members mentioned the extra insurance offered as a prerequisite for taking part in the team. \\'The normal insurance for LHS staff would not cover everything if we were to become sick or even lose our lives.\\' Amongst ER staff, the matter of insurance did seem to be less of an issue compared to the ETT. One respondent said: \\'You are used to being at risk by many disease threats\\'. Furthermore, the issue of higher salaries and risk commission came up in the interviews, but overall did not matter as much to the participants as the insurance, or assurance of accommodation in case of need.\\n\\nCharacteristics associated with Iceland and the Icelandic people were referred to repeatedly by participants. The concept \\'Tiny Iceland\\' was often mentioned and emerged with positive and negative connotations. \\'Tiny Iceland\\' referred to the size of the country and population and its perceived capability to still \\'get the job done\\'. even though compromises had to be made. Comparing how Iceland handled its responsibilities differently from other countries of a larger size was often brought up, both with pride in Iceland as a strong independent nation, and with insecurities about its capacity in comparison to other countries. It was pointed out that since the preparedness process was in the hands of a few people, everyone knew their role. As one administrator said: This little hospital system, as complicated as it might seem every day, gives you the chance to just pick up the phone and call the one in charge.\\n\\nBeing a small population presents challenges regarding resources, infrastructure, and specialized medical training to comply with standards of international actors. Notions of Icelanders as resilient in spite of shortcomings were common; referring to the experience of preparedness planning and training, one health staff said:\\n\\nIt was very much the Icelandic way, we\\'ll manage, we\\'ll work it out, and there was so much ingenuity. This notion of a particular Icelandic approach to coping, in spite of shortcomings, was also detected more generally, as in the statement:\\n\\nWould it have worked? Yes, it would have worked. Would it have been optimal? We cannot say, it would have been optimal; we can say, it would have been sufficient.\\n\\nIn contrast to this, there were concerns about whether Icelandic aid workers falling ill in Ebolaaffected countries should be transferred to Iceland or to hospitals in other Nordic countries with better isolation units. Some of the participants trusted that patients with EVD would not be transferred to Iceland. One participant stated: You heard that Norwegians were criticized for transferring their aid worker from Africa to Norway. We don\\'t know what would have happened if they would have transferred an Icelander into the country.\\n\\nWe don\\'t have good enough isolation unitsyou are not supposed to send patients to a hospital that is less than 100%. I thought there was assurance in that.\\n\\nDuring the devastating Ebola epidemic in West Africa that spread to neighbouring sub-Saharan countries, North America, and Europe [32] , preparedness plans were widely elaborated and later evaluated. Evaluations have, for example, been conducted in 11 African countries close to the epidemic [33] , in the EU region [34, 35] , and the US [36] . Here we present data from a qualitative case study on the process, and experiences with establishing a preparedness plan for Ebola in Iceland in 2014. Interviews with staff who were engaged, either as administrators or frontline healthcare workers, alert us to the manner in which geographic, demographic, cultural, and organizational characteristics shaped the response. The results show that the process of establishing and training for preparedness was permeated by ambiguities of pride and pragmatism, trust, doubts, and fear.\\n\\n\\'Getting the job done\\' (theme 1) refers to the multitude of tasks and considerations that surrounds and feeds into the preparedness plan itself and are necessary for successful planning and implementation. Using the metaphors of \\'hard core\\' and \\'soft periphery\\', Langley and Denis [37] emphasize the importance of relatively \\'peripheral\\' concerns and processes for planning and implementation of new interventions. The hard core represents the actual intervention or goal, e.g. implementation of a preparedness plan. The soft periphery refers to all the contextually important networking, negotiations, and agreements necessary to deliver the hard core. If the soft periphery is neglected, it will cause multiple challenges in the implementation process, and the benefit of the hard core, the intervention itself, may not transpire as anticipated. Due attention to the soft periphery may, however, considerably promote the delivery of an innovation, and secure support from important stakeholders. In our data, one manager speaks of the preparedness process as dealing with a three-headed monster where every solution was followed by new problems. The data indicate that the process of dealing with \\'the three headed monster\\' was given due attention as a means to successfully develop Iceland\\'s preparedness plan. Comprehensive consultations and the involvement of many associated institutions were mentioned. Still ambiguity remained with some staff in terms of division of responsibilities and taskse.g. when transporting a patient potentially infected with Ebola from the airport to the hospital, and other such activities.\\n\\nDuring epidemics, rumours, gossip, and unreliable information on the news and social media spread rapidly, resulting in so-called \\'infodemics\\' [38] . The West African Ebola epidemic was covered widely by media [39] , and the fear of Ebola reached every corner of the world, exemplified by travel bans from affected countries, and trade barriers [40] , in contrast to the ongoing epidemic in the Democratic Republic of Congo [41, 42] . In our second theme, trust, doubt, and fear of health workers were represented. Although all intentions were good, concerns remained about the suitability and safety of the isolation ward, the PPE, and other tools, as well as adequate engagement of colleagues who might potentially work alongside them, in case an Ebola patient came to Iceland. The foreignness of putting on, removing, and working from within a PPE and the trustworthiness of available PPE were mentioned. In preparedness efforts in other countries, scarcity of resources in relation to manpower demand and problems with training and protocols involving PPE were common challenges [35] . Similar problems were encountered in Iceland. Provisory treatment facility had to be designed, called \\'camping site\\' by some, in contrast to facilities found elsewhere [43] . Further, the ETT was established based on voluntary recruitment rather than on the staff\\'s assigned roles within the healthcare system, a procedure that was deemed less than optimal. The members of the ETT pointed out that they had never worked together as a team under circumstances that demanded strict adherence to infectious control procedures. This eroded trust, compounded by the laissez-faire attitude of some of its members during the preparation exercises, possibly due to other competing tasks in a busy hospital and insufficient resources that hampered full participation [44] . Further, it was a constraint that simulation exercises were not an option, found to be an important element in preparation for epidemics [35] . This might have resulted in less than optimal staff protection for those who would have been in direct contact with an infected patient, as reported during the SARS epidemic in Canada [45, 46] .\\n\\nAnthropological work on emergency preparedness emphasizes the connectedness between health professionals, technological devices, and knowledge as a prerequisite for successful preparedness. Wolf and Hall present preparedness efforts as a form of governance that involves human bodies (those of health professionals), clinical architectures (e.g. isolation wards), and technical artefacts (gloves, protective suits, disinfectants, etc.) [47] . During preparedness training and implementation, \\'nursing bodies are transformed into instruments of preparedness\\', and become part of infrastructural arrangements. Health professionals are, here, both vulnerable and powerful tools in the management of contamination. The authors argue that successful planning, training, and implementation of a preparedness plan require such intrinsic connectedness. In the case of Ebola preparedness in Iceland, health professionals draw our attention to dilemmas of connectedness, and their assessment of the fact that these shortcomings might hamper the mobilization of \\'preparedness within the human body\\'that is, the embodied experience, routine, and tacit knowledge which Wolf and Hall state are key to successful implementation. Repeated enactment of receiving and treating a patient with Ebola within experienced and trustful teams would probably enhance such embodiment, provided that there is justified trust in the involved technology. In addition, repetition would also strengthen the \\'soft periphery\\' of preparedness, and divisions of responsibilities would be clearer manifested.\\n\\nIn the third theme, we observe how notions of the \\'Icelandic way\\' help participants make sense of ambiguities about Ebola preparedness. Loftsdóttir explored how people negotiated the imagination of the local and the global during the 2008 economic crisis in Iceland [48] . Notions of the intrinsic character of Iceland, and of being Icelandic, serve to underscore certain points and explain positive and negative experiences with the preparedness plan. Iceland is far away from the continents, but still connected through global needs for policy, risk of contamination, and dependency in terms of collaboration, in emergencies emerging from elsewhere. In our study, participants highlighted the importance of believing in oneself and the \\'Icelandic way of doing things,\\' summed up in the paraphrase \\'þetta reddast\\' (things always have a way of working out in the end). The preparedness plan had to be completed, and adapted to Iceland\\'s particular global situation.\\n\\nIn the 21st century, the world has faced new epidemic threats, such as SARS, and old scourges such as the plague have resurfaced [38] . One of the main findings on Ebola preparedness measures in the EU was that measures taken were based on past preparedness and experience of other epidemics, such as SARS and H1N1 [35] . Further, key stakeholders within each country found their measures to have been adequate for dealing with a single case of Ebola, as was the case in Iceland. A preparedness plan for pandemic influenzae in Iceland was elaborated in 2006activated in response to the H1N1 epidemic in 2009and revised in 2016 [49] . During the elaboration of these plans, communication among the different levels of the healthcare system and supporting agencies, such as the DCPEM, had been clearly defined, and proved to be useful in the preparedness for Ebola. Further, as found important in preparedness activities for pandemic influenzae elsewhere [44] , honesty, transparency in communication, and sharing of information from managers to front-line health professionals, was found to be critical. It gave a feeling of being involved, and mitigated the fear that is so frequently encountered during epidemics [38] .\\n\\nIceland was far away from the epicentre of the Ebola epidemic in West Africa. Yet this case study shows that health professionals felt the strain of possibly having to treat one or more patients with EVD. Their situation stands in sharp contrast to the situation in the three worst affected West African countries that lacked staff, stuff, space, and systems to effectively address the challenge of EVD. Although Icelandic health professionals had trust in the national healthcare system, and in their own capacity, doubt and fear influenced the reflections on preparedness planning of both administrators and healthcare staff. References to national identity and the characteristic of an \\'Icelandic approach\\' to handling challenges assisted participants in coming to terms with the experienced shortcomings of the preparedness plan, and underscored the pride in the ingenuity applied in the process. These references negotiate the role and character of the nation of Iceland, and its role in a globalized world, as both a small and isolated nation on one hand, and a central and capable one, on the other.\\n\\nThe experienced ambiguity needs attention in a health system and among healthcare staff that have to act resolutely and unfailingly, should they be placed in charge of containing contamination. This study points to the necessity of repeatedly re-enacting, as realistically as possible, the likely scenarios of receiving and treating one or more patients infected with Ebola (or other contagious global health threats) as a routine matter. This would assist in the identification of overlooked \\'soft periphery\\' concerns, and promote embodied preparedness among teams of health care staff on the frontline. Geir Gunnlaugsson conceptualized the study, and took part in all necessary steps towards its completion, such as analysis and interpretation of data, and writing the manuscript for submission. Íris Eva Hauksdóttir collected and analysed the data as part of a master thesis work conducted under the supervision of all three co-authors, revised the manuscript, and approved the final version. Ib Bygbjerg took part in the interpretation of data, revision of the manuscript, and approved the final version. Britt Pinkowski Tersbøl took part in designing interview tools and in the thematic analysis of interview data, interpretation, revision of the manuscript, and approved the final version.\\n\\nDr. Gunnlaugsson reports he was the Chief Medical Officer (CMO) for Iceland, Directorate of Health, in the period 2010-2014. Other authors report no conflict of interest.\\n\\nThe study was reported to the Data Protection Authority and approved by the National Bioethics Committee in Iceland (number VSI- ). Subsequently, the study was approved by the University Hospital Ethical Committee on 4 February 2016 (number LSH [13] [14] [15] [16] . Participants signed an informed consent form before taking part in the study.\\n\\nNot applicable.\\n\\nThe manuscript builds on the work of Íris Eva Hauksdóttir towards a MSc in Global Health, Section of Global Health, Department of Public Health, Copenhagen University, Denmark.', 'document_id': 1618}]}, {'paragraphs': [{'qas': [{'question': 'What is the structure of a coronavirus?', 'id': 5314, 'answers': [{'text': 'large, non-segmented, positive sense and single stranded RNA animal viruses', 'answer_start': 1119}], 'is_impossible': False}], 'context': 'Recent Progress in Studies of Arterivirus- and Coronavirus-Host Interactions\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC3397358/\\n\\nSHA: f5e974ef3a8c983ae63ac4f4aa2b6ec0e3678032\\n\\nAuthors: Zhong, Yanxin; Tan, Yong Wah; Liu, Ding Xiang\\nDate: 2012-06-19\\nDOI: 10.3390/v4060980\\nLicense: cc-by\\n\\nAbstract: Animal coronaviruses, such as infectious bronchitis virus (IBV), and arteriviruses, such as porcine reproductive and respiratory syndrome virus (PRRSV), are able to manifest highly contagious infections in their specific native hosts, thereby arising in critical economic damage to animal industries. This review discusses recent progress in studies of virus-host interactions during animal and human coronavirus and arterivirus infections, with emphasis on IBV-host cell interactions. These interactions may be directly involved in viral replication or lead to the alteration of certain signaling pathways, such as cell stress response and innate immunity, to facilitate viral replication and pathogenesis.\\n\\nText: Coronaviruses, together with arteriviruses and toroviruses, belong in the order Nidovirales, a group of large, non-segmented, positive sense and single stranded RNA animal viruses that produce an extensive 3\\'-nested set of subgenomic mRNAs for transcription during infection [1] (Table 1) . Nidoviruses such as avian infectious bronchitis coronavirus (IBV), human coronavirus 229E (HCoV-229E), equine arteritis virus (EAV) and the porcine reproductive and respiratory syndrome arterivirus (PRRSV) are important pathogens of both human and animals [2] [3] [4] , and are commonly associated with mild respiratory and enteric diseases, although they are also known to cause more OPEN ACCESS critical lower respiratory tract illness, such as the severe acute respiratory syndrome coronavirus (SARS-CoV) epidemic that occurred in 2003 [5] .\\n\\nAs these viruses infect livestock, coronaviral and arteriviral infections in farms have resulted in large-scale economic losses in farming nations, and are therefore of exceptional veterinary research value. Coronaviruses in fowls, as exemplified by the highly contagious IBV in chickens, can be highly lethal to young chicks. IBV is the etiological agent of infectious bronchitis, an avian disease that is mainly associated with upper respiratory and urogenital tract infections in adult chickens, and to a lesser extent, nephrogenic infections, or inflammation of the kidneys [6, 7] . The impact of IBV infection is also emphatically increased as a consequence of its enhancement of diseases associated with lethal co-infections by bacteria and mycoplasmas [8, 9] . Domestic animals such as cats and dogs are also susceptible to coronaviruses. Feline coronaviruses, especially feline infectious peritonitis virus (FIPV)-a mutation of Feline Enteric Coronavirus (FECV), may induce lethal diseases in cats [10] , while canine coronavirus (CCoV) infections, which cause canine enteric illness in dogs, are prevalent as well [11] . In larger livestock like pigs and cattle, on the other hand, coronaviruses and arteriviruses typically establish enteric infections. An infection or outbreak can cause severe economic losses from the death of young offspring, lifelong impact on the yield of animal produce such as eggs and milk, weight losses and the general health of the population. Bovine coronavirus (BCV), for example, causes Winter Dysentery (WD) in adult cows and diarrhea in young calves [12] .\\n\\nThe pathogenicity of these viruses is typically species-dependent, as is the severity of infection; they infect mainly their natural hosts and/or species that are closely related. Certain virus infections, however, can cross the species barrier, with the prime example being the zoonotic SARS-CoV, a novel coronavirus that is thought to have originated from bats before it adapted to its intermediate host, civet cats, and finally to humans [13] . Bat colonies, which are scattered worldwide, are widely known to play host to a variety of coronaviral and adenoviral pathogens while acting as natural wildlife reservoirs of these viruses [14] [15] [16] .\\n\\nCoronavirus infections are also generally tissue-specific-the Transmissible Gastroenteritis Coronavirus (TGEV), for example, affects mainly the gastrointestinal tract [17] that may lead to the onset of fatal watery diarrhoea and severe dehydration in pigs [18] , while human coronaviruses mostly cause respiratory infections [4] .\\n\\nWith respect to their significance to the economy, vaccines have also been developed for many of these viruses in a bid to prevent localized infections from progressing into serious outbreaks. This has, however, proven to be a hard battle as the vaccines are unable to provide complete cross-protection among the various serotypes of each virus [19] . \\n\\nDuring infection, the virus replicates in the host cytosol amidst a myriad of host signaling pathways and systems such that interaction between the virus and the host systems is inevitable. Virus infection and the consequent host cell response also involve complicated interaction between various host cellular and viral networks. Virus-host interplay occurs at multiple points during the virus replication cycle, from entry to exit. The nature of such interactions can range from a simple exploitation of existing host machinery to destructive interactions that modulate the host environment to the advantage of the virus while inhibiting host activities. One of the most important interactions between virus and host is the modulation of host cell environment, such that the latter is converted into one in which the virus can replicate successfully. Viruses also regulate the differential expression of host genes, as well as various host antiviral defense mechanisms, for more efficient replication.\\n\\nPrevious studies on the infection of different hosts by nidoviruses, for example, have shown various modifications in host innate immune and stress responses, cell cycle, autophagy and cell death pathways [20] [21] [22] [23] , all of which will be discussed in this review. The significance of host components being used to supplement the gene-poor virus in various processes cannot be dismissed, for although they typically serve as enhancers, they could also become major pathogenicity factors.\\n\\nA number of stimuli can precipitate apoptotic events, including cell homeostatic imbalance such as cell stress, and the binding of ligands to cell surface \"death\" receptors; these in turn trigger the onset of major apoptotic pathways: the extrinsic or intrinsic pathway [24] .\\n\\nThe extrinsic pathway can be induced by several cytokine \"death\" receptors from the tumor necrosis factor (TNF) family, such as Fas (Apo1/CD95) [25] . Upon recruitment of their respective ligands, they form complexes that subsequently bind death effector domain (DED)-containing pro-Cysteine Aspartyl-Specific Proteases (pro-caspases), in particular pro-caspase-8, where the activation and consequent oligomerization of which further serves is a signal for downstream activations, thus pledging the doomed cell towards its own death [26] . The intrinsic pathway, on the other hand, is activated by the release of cytochrome c from the mitochondria into the cytoplasm [27] . In the cytosol, cytochrome c binds the apoptotic protease-activating factor (Apaf1); together, they form an apoptosome that leads to the release of active caspase 9.\\n\\nApoptotic mitochondrial events are also regulated primarily through the activation of pro-survival and pro-apoptotic proteins [28] . The Bcl-2 family of proteins constitutes a critical control point in the regulation of apoptosis. They form three major protein subgroups: the Bcl-2 homology (BH) 3, or BH3-only proteins [e.g., BH3-interacting domain death agonist (Bid), BCL2-associated agonist of cell death (Bad)], Bax-like proteins [e.g., BCL-2-antagonist/killer 1 (Bak), BCL-2-associated X (Bax)] and the Bcl-2-like factors [e.g., Myeloid cell leukemia-1 (Mcl-1), Bcl-extra large (Bcl-X L )] [29] . BH3-only and Bax-like proteins are essential initiators of apoptosis while the Bcl-2-like proteins are pro-survival factors that safeguard the cells against apoptosis.\\n\\nBoth caspase 8, from the extrinsic pathway, and caspase 9, from the intrinsic pathway, have been observed to activate the main effector caspase 3, which in turn activates a caspase cascade to eventually evoke the morphological hallmarks of apoptosis such as DNA fragmentation [30, 31] .\\n\\nA third apoptotic pathway, induced by prolonged endoplasmic reticulum (ER) stress, has also been shown to activate multiple downstream apoptotic targets, including rodent caspase 12, growth arrest and DNA damage-inducible gene 153 (GADD153), also known as the transcription factor C/EBP homologous protein (CHOP) as well as activation of the pro-apoptotic c-Jun NH2-terminal kinase (JNK) [32] . Human pro-inflammatory caspase 4, a nearly identical paralogue of the rodent form of caspase 12, has also been shown to possess comparable roles in ER-stressed apoptosis [33] . JNK activation is mediated by ER transmembrane protein kinases, while CHOP is triggered by ER stress at the transcriptional level [32] . The downstream apoptotic activities of both JNK and CHOP have also been postulated, at least in part, to be connected with the Bcl-2 family of proteins (Bak and Bax) for recruitment to the ER and subsequent initiation of apoptosis in response to stress [32] .\\n\\nAs viruses depend on the host cells they infect in order to reproduce, apoptosis is often employed as an important host antiviral defense mechanism that, as a protective measure, leads to the abortion of virus infection such that viral productivity and persistent infectivity is consequently limited [34] . In many cases, p53 and the Bcl-2 family of proteins have been shown to be the main mediators that induce the beleaguered cell to undergo self-induced death at various stages of the infection cycle [35] .\\n\\nMoreover, host endosomal membranes are forced to undergo conformational changes for the fusion of virus and host cell membranes during virus uncoating; membrane integrity is also antagonized during the process of virus disassembly. As such, these drastic alterations to membranes may elicit downstream pro-death signals that prompt infected cells to commit suicide [36] .\\n\\nHowever, certain viruses have evolved strategies to both counteract and induce apoptosis in order to maximize the production of virus progeny and promote its spread to neighbouring cells. An increasing number of known viruses from different families, including arteriviruses, have been found to induce apoptosis during their infection cycle, which may possibly contribute to the cytotoxicity associated with virus infections, especially during late stages of infection [37] . Membrane-bound cell fragments are also produced as apoptotic bodies to be phagocytosed by surrounding cells. This provides an excellent method for a virus to disperse its progeny without eliciting host immune response [38] .\\n\\nWhile more comprehensive work needs to be done to paint a clearer picture of how coronaviruses and arteriviruses regulate apoptosis during infection, recent reports have suggested the possible activation of more than one apoptotic pathway during infection. EAV, an arterivirus that is prevalent among global horse populations and which may induce abortions in pregnant mares [39] , has been shown to activate apoptosis through the initiation of caspase-8-dependent mechanisms, which is followed by mitochondria-dependent caspase-9 activation mechanisms [40] . PPRSV, a virus that causes respiratory tract problems in young pigs and has a commercially significant impact on swine industries worldwide as a result of reproductive impairment in breeding livestock [41] , has also been implicated to regulate both apoptosis and necrosis during infection [42, 43] . Replication of TGEV in porcine kidney cells, as well as that of canine coronavirus type II (CCoV-II) in canine fibrosarcoma cells, on the other hand, has been reported to induce apoptosis both through Fas/FasL-activation and mitochondrial-dependent pathways [44, 45] . Figure 1 . Viral genes and the activation of apoptosis. Extrinsic signals from receptors (e.g., Fas ligand) culminate in the activation of caspase 8, which activates the effector caspase 3 while intrinsic signaling requires the participation of the mitochondria in releasing cytochrome c (shown as circles labeled \"C\") to activate caspase 9 for the downstream activation of caspase 3. Key proteins in the intrinsic apoptosis signaling pathway are p53, both pro-apoptotic (e.g., Bax, Bak) and anti-apoptotic proteins (e.g., Bcl-2, Bcl-X L) from the Bcl-2 family. Viral genes (see yellow boxes), which act at multiple points along the different signaling pathways, target both the extrinsic and intrinsic apoptosis signaling pathways and enhances the pro-apoptotic effect brought upon by virus infection. Anti-apoptotic proteins (black oval) listed are key anti-apoptotic members from the Bcl-2 family of proteins. Other pathways triggered by coronavirus infections, such as ER stress and DNA damage response, activate apoptotic signaling pathways as well.\\n\\nAlthough IBV is an avian virus, it can acclimate to primate cells and has been demonstrated to conquer the host species barrier and become zoonotic to infect both human and animal cells [46, 47] . IBV also triggers apoptosis during the late stages of its cytolytic infection cycle. Specifically, IBV-induced apoptosis has been shown to involve Bcl-2 family of proteins [48] , through caspase-dependent [49] and p53-independent [50] pathways in cultured mammalian cells. The modulation of Bcl-2 family proteins during IBV infection has also been postulated to be under the regulation of signaling pathways such as ER stress and Mitogen-Activated Protein Kinase/Extracellular signal-Regulated Kinase (MAPK/ERK) pathways [48] . The effects of these pathways on the regulation of other host defenses will be discussed in later paragraphs.\\n\\nWhile viral genes can also manipulate the induction of apoptosis to the benefit of viruses, only two have been reported in the case of coronaviruses. The unique SARS-CoV encoded protein, 7a, was discovered to have caspase-dependent, pro-apoptotic functions and may function as a plausible source of virus-derived apoptotic signal [51] , while TGEV accessory gene 7, present only in coronaviruses classified under genus \\uf0611 [52] , thwarts virulence and host-induced antiviral mechanisms through the negative modulation of downstream caspase-dependent apoptotic pathways [53] (Figure 1 ).\\n\\nThe maintenance of apoptosis is also important in the establishment and governance of immune responses in a cell. A loss in the control of apoptosis leads to an imbalance in cell homeostasis, which ultimately affects immune sensitivity [54] . The presence of apoptotic cells, particularly in existence with infectious agents, may also lead to the mobilization and initiation of innate immune defenses [55] . This crosstalk between apoptosis and innate immunity is therefore of considerable importance during pathogenic infection and can be manipulated by both host and pathogen, either as a form of immune defense or immune evasion, respectively [56] .\\n\\nWhen the host immune system is exposed to viral pathogens, it reacts straightaway by triggering a diverse array of defense mechanisms in order to establish a more efficacious shield. The first line of defense is the mounting of an innate immune response, as characterized by the increased production of type I interferons (IFN-α and IFN-β) and other inflammatory cytokines. These, in turn, choreograph the expression of downstream IFN-stimulated genes (ISGs) and activate several signaling pathways, all of which collaboratively lead to the induction of a protective antiviral state and, subsequently, the inhibition of both viral replication and proliferation [57] .\\n\\nThe cytokine family of interferons is dedicated to the conveyance of the presence of infection, as well as the expedition of numerous connections among the cells that provide protection against, or eradication of, foreign pathogens. Other than interfering with viral progeny production in host cellshence the name \\'interferon\\'-IFNs also induce Natural Killer (NK) cells and macrophages to \\'kill\\' or engulf infected cells, increase antigen presentation to thymus (T) cell lymphocytes for rapid recognition of infected cells and bring about virus resistance to new uninfected cells [58] . IFNs are conventionally classified into three types: Type I (IFN-α, IFN-β and IFN-ω), Type II (IFN-γ) and the more recently identified Type III (IFN-λ1, IFN-λ2 and IFN-λ3) [59, 60] . Over time, mammalian hosts have gradually developed a multitude of cellular sensors for the detection of viral infection, and it is the involvement and operation of these cellular protein receptors that eventually leads, through an intricate network of pathways, to the expression of type 1 IFNs. Major receptor systems that conduct immune surveillance and trigger the production and subsequent release of type I IFNs are known as pattern recognition receptors (PPRs), which detect viral infection through the identification of various pathogen-associated molecular patterns (PAMPs); PPR families include the toll-like receptor (TLR), RIG-like helicase (RLH) and Nucleotide-binding oligomerization domain (NOD)-like receptor (NLRs) families [61] .\\n\\nThe TLR family is mainly made up of transmembrane proteins, which conduct surveillance from the cell surface, as well as cellular compartments such as the endosome or ER, constantly scanning the extracellular environment for PAMPs that can be derived from a wide range of microorganisms, including viruses and bacteria. The expression of TLRs appear to be cell-specific and is mainly found in antigen-presenting cells such as dendritic cells (DCs), monocytes and macrophages, as well as on B cells [62] . TLRs recognize a wide variety of PAMPs, and the recognition of these ligands can be converted into specific intracellular responses through the direct interaction of the TLR toll-interleukin 1 receptor (TIR) domain with one of its cytoplasmic TIR-containing signaling adaptor molecules, such as myeloid differentiation primary response protein 88 (MyD88) and TIR domain-containing adaptorinducing IFNβ (TRIF) [63] . Viral recognition by TLRs has been detailed in several reports [64, 65] . In particular, the activation of TLR-3, -4, -7, -8, and 9 can also culminate in type I IFN production [65, 66] . TLR3, as a general viral sensor, detects mainly through double stranded RNA (dsRNA), a replication intermediate of both DNA and RNA viruses; TLR4 recognizes envelope proteins from viruses such as mouse mammary tumor virus (MMTV); TLR7 and TLR8 have been identified to recognize ssRNA viruses like influenza and vesicular stomatitis virus (VSV); TLR9 detects dsDNA viruses such as herpesviruses [63] .\\n\\nNLRs are stimulated by microbial agonists and collaborate with TLRs to evoke intracellular immune responses through MAPK and caspase signaling cascades upon sensing bacterial components [67] . NLRs are also known to sense both PAMPS from infectious agents and DAMPs (danger-associated molecular patterns) that arise as a result of insult or injury to the cell, as well as those derived from the environment [68] . Although the direct binding of virus-derived PAMPs to\\n\\nNLRs is yet to be reported, structural and functional studies of the C-terminal domain of NLRX1, a mitochondrial member of the NLR family, has highlighted its ability to bind both ssRNA and dsRNA, implying that some NLRs may be capable of binding viral RNA directly as well [69] . As such, the notion of NLR-mediated recognition of coronaviruses is, therefore, probable and could be further investigated.\\n\\nThe RLH family of purely cytoplasmic PRRs is made up of the following: retinoic acid inducible gene-I (RIG-I or DDX58), melanoma differentiation-associated gene-5 (MDA5 or IFIH1), and laboratory of genetics and physiology 2 (LGP2). RIG-I and MDA5 are PRRs with two N-terminal caspase-recruitment domains (CARDs) followed by a DExD/H box RNA helicase domain; LGP2 lacks the signaling caspase recruitment domains but shares a helicase domain of similar homology and is thought to serve as a regulator of the former [70] . RIG-I and MDA-5 both sense cytoplasmic dsRNA, which the host recognizes as \\'non-self\\', via the N-terminal CARDs [71] . However, the two PRRs each sense distinct PAMPs, depending on the length of viral dsRNA, from different RNA viruses. In addition to long dsRNAs (>2 kb) such as the synthetic dsRNA analogue poly-inosinic poly-cytidylic acid [poly(I:C)], MDA5 also recognizes picornaviruses and noroviruses [72, 73] . RIG-I, on the other hand, responds to paramyxoviruses, flaviviruses, orthomyxoviruses and rhabdoviruses [61, 74, 75] . This is through its recognition of a variety of ligands such as relatively short dsRNA (19-mer to 1 kb) and ssRNA (single stranded RNA), both preferably in the presence of a 5\\'-triphosphate end, full-length RNA viral genomes, the presence of secondary structures such as poly-uridine motifs within 5\\'-triphosphate genome termini or longer RNA sequences without 5\\'-triphosphates, such as 3\\' untranslated regions (UTRs) of the genome [76] [77] [78] [79] [80] .\\n\\nWhile the elicitation of TLRs and RLRs by PAMPs trigger their distinct signaling cascades through divergent downstream effectors at varying efficacies, they ultimately cross paths at the juncture of transcriptional activation of interferon regulatory factor 3 (IRF3) [81] , IRF7 [82] and nuclear factor kappa-light-chain-enhancer of activated B cells (NF-κB) [83] [84] [85] [86] , all of which translocate to the nucleus and activate the transcription of both type I interferons (IFNα and IFNβ) and inflammatory cytokines that eventually culminates in the concerted induction and development of adaptive antiviral immune response.\\n\\nThe expression and induction of interferons from the cells occurs in response to viral insults and tumor growth [87] . Type I interferons, the major group of cytokines in innate anti-virus defense, bind a specific cell surface heteromeric receptor, the interferon-α/β receptor (IFNAR), which composed of two subunits, IFNAR1 and IFNAR2 [88] . The best characterized type I IFNs can be classified into two major groups, the immediate-early genes (such as IFNβ) that are triggered by the initial response to virus infection, and the delayed-set (such as IFNα subtypes) that rely on a secondary de novo protein synthesis pathway [89] . IFN expression is regulated by IFN regulatory factors. In particular, IRF3 and IRF7 play vital roles in activating innate immune response through their respective antiviral response [66] .\\n\\nIRF3 has been functionally characterized to consist of a nuclear export signal (NES), a DNA-binding domain (DBD), a C-terminal IRF association domain (IAD), as well as a number of phosphorylation sites as well as two auto-inhibitory domains that prevents a constitutive activation of the NES, DBD and IAD [90] . Normally found in an inactive cytoplasmic form, IRF3 is phosphorylated as a consequence of virus infection. This activation signal exposes the DBD and IAD and results in the dimerization of IRF3, either as a homodimer or as a heterodimer with IRF7, allowing the activated IRF3 to form a complex with the transcriptional co-activator cAMP-response element-binding protein (CREBP) to translocate to the nucleus and bind to DNA to trigger the transcription and expression of immediate-early IFNs, which signals the JAK-STAT (Janus kinase-Signal Transducer and Activator of Transcription) pathway through the binding of the IFNAR [90] . This leads to the formation of a STAT1-STAT2 heterodimer that teams up with the interferon regulatory DNA binding factor IRF9, which together constitute an activated heterotrimeric factor, the interferon-stimulated gene transcription factor ISGF3, that, through the recognition and binding of specific ISREs, induces downstream expression of innate immunity genes for host defense against virus invasion [91] .\\n\\nIn contrast to the constitutive expression of IRF3, IRF7 is involved in the positive feedback regulation of IFN production [89] . Expressed only in minute amounts, the induction of IRF7, via virus-induced activation of ISGF3, results in either homo-dimerization or hetero-dimerization with IRF3 and is subsequently followed by nuclear translocation for the activation of both IFN\\uf061 and IFN\\uf062 genes [92, 93] .\\n\\nCoronaviruses and arteriviruses have evolved multiple strategies to avoid elimination from the host. These tactics range from the prevention of detection to inhibition of antiviral responses mounted by the host immune system. All these activities involve virus-host interactions at different levels ( Figure 2) .\\n\\nThe modulation of SARS-CoV pathogenesis, for example, was reported to be independent of all three types (Types I, II and III) of interferon signaling mechanisms, with SARS-CoV ORF 3b, ORF 6, and nucleocapsid proteins identified to interfere with interferon signaling through various mechanisms [94] . However, STAT1 has been shown by the same group to be crucial in activating innate immune signaling pathways during SARS-CoV infection, with a secondary role in the prevention of uncontrolled cell reproduction [95] . In contrast, PRRSV appears to be receptive to both IFN-α (porcine IFN-α, or Ad5-pIFN-α) and -β (recombinant swine beta interferon, or swIFN-beta) dose-dependent treatment [96, 97] ; however, pigs infected with PRRSV do not evoke significant IFN responses, with little or no IFN-α and IFN-β production [98, 99] . Upstream signaling pathways that may potentially lead to the inhibition of IRF3 activation include the interference of PRRSV with RIG-I signaling events through inactivation of the RIG-I downstream signaling adaptor, MAVS (mitochondrial antiviral signaling protein) [100] . Recent studies have also suggested PRRSV infection inhibits type I IFN production and signaling processes through the impairment of STAT1/STAT2 nuclear translocation [101] .\\n\\nOne of the most indispensible adaptor proteins in the RLH signaling pathway is MAVS. Otherwise known as virus-induced signaling adaptor (VISA), interferon \\uf062 (IFN\\uf062) promoter stimulator-1 (IPS-1) or CARD-adaptor inducing IFN-β (Cardif), it was discovered independently by four different research groups in 2005 [102] [103] [104] [105] , and contains an N-terminal CARD-like domain, which interacts with the CARD domains of RIG-I and MDA5, and a C-terminal transmembrane (TM) region that lodges the protein in the mitochondrial membrane, thus pioneering the connection between mitochondria and innate immunity. A deficiency in, or the cleavage of, MAVS from the mitochondrial membrane results in an arrested antiviral immune response, highlighting the pivotal role MAVS plays in mitochondriamediated innate antiviral immunity [106] . MAVS can also induce apoptosis independently of Type I IFN activation, with SARS-CoV non-structural protein 15 identified as an inhibitor of MAVS-induced apoptosis to escape host antiviral immune responses [107] .\\n\\nCoronaviruses have devised a number of cell type-specific strategies to inhibit type I IFN responses. One such strategy that these viruses have adopted is to avoid detection of its newly synthesized mRNAs in the cytoplasm through the encoding of a 2\\'-O-methylase (non-structural protein nsp16) that creates a 5\\'-cap structure analogous to the cellular mRNAs on their mRNAs, thereby escaping detection by Mda5 [108, 109] . More importantly, this also highlights the significance of mRNA cap alterations, as such 2\\'-O-methylation, as pertinent molecular signals for the discernment between self and non-self mRNA [108] . However, during the course of viral transcription and replication, uncapped double-stranded RNA intermediates are generated and these may serve as ligands for the RIG-I and/or Mda5. Mouse hepatitis virus (MHV) infection was first reported to activate type I IFN responses in various cell types such as macrophages and microglia in the brains of infected animals, and the virus has also been shown in the same report to be recognized specifically by MDA5 in mouse macrophages [110] . In other reports, both RIG-I and Mda5 have been implicated in the detection of mouse hepatitis virus (MHV) infection, particularly in mouse oligodendrocytes [111] , although conflicting evidence has been reported as well, especially since it is not understood how cellular helicases could bind viral replicative/transcriptive intermediates when the latter should have been isolated by the double membrane vesicles [112] . MHV infection can also delay IFNβ-activated ISG induction; however this phenomenon is limited to certain cell types and is observed only when the infection occurs before IFNβ exposure [113] . As such, the general sensing of coronaviruses appears to be cell-type specific, with Type I IFNs derived from plasmacytoid dendritic cells (pDCs), conventional DCs and macrophages being particularly essential in curbing the pathogenesis of mouse coronavirus infection [114] .\\n\\nDetection by the PRRs would activate signaling cascades leading to the production of type I IFNs, resulting in the establishment of an antiviral state. Thus, coronaviruses do not just avoid detection by the host immune system, which appears to be the main strategy of ensuring successful replication [115] ; some viruses also encode proteins that function to disrupt the downstream signaling cascades at various points, preventing the establishment of an effective antiviral state when detection of the viral PAMPs has occurred. Indeed, the N protein has also been shown to interfere with the 2\\', 5\\'-oligoadenylate synthetase/RNaseL (2\\'-5\\'OAS) activation that occurs downstream of IFN induction and which leads to the inhibition of global translation shutdown [116] . This activity is in addition to its suppression of IFNβ induction through the binding of viral RNAs that prevents their detection [117] .\\n\\nIn addition to the processing of polyproteins, the papain-like proteinase 2 (PLP2) domain of nsp3 has been shown to possess de-ubiquitinating activity [118] [119] [120] . PLP2 of nsp3 de-ubiquitinates TANK-binding kinase 1 (TBK1), the activating kinase for IRF3 and IRF3 itself, and sequesters the hypo-phosphorylated TBK1-IRF3 complex in the cytoplasm as well. This prevents IRF3 nuclear translocation [121, 122] , thereby inhibiting the transcription of type I interferons [123] . The activation of TLRs-3 and/or -7 as well as cytoplasmic helicases RIG-I and/or MDA-5 triggers signaling pathways resulting in the synthesis of type I IFNs, inflammatory cytokines and ISGs which acts in concert to establish an antiviral state. The activation of both 2\\'-5\\' OAS and PKR results in global degradation of cellular RNA and inhibition of translation, which may inhibit viral propagation. Coronaviruses encode many proteins (see yellow boxes) that target multiple steps in the innate immune response mounted by the host cells, ensuring its successful replication in the host.\\n\\nIn vitro activation of chicken splenocytes and peripheral blood leukocytes with IBV has also resulted in an increase in chicken interferon gamma (chIFN-γ) production as a form of cell-mediated immune response [124] . This appears to be a polyclonal, non-specific stimulation as chIFN-γ production levels are also elevated in IBV-stimulated chicken splenocytes which lack prior exposure to IBV, when compared to control un-stimulated cells, as well as in cells exposed to inactivated IBV [124] .\\n\\nSuch ineffectual host innate responses could lead to poor cellular responses, which may result in a delay in pathogen clearance and persistent viral infection in compromised pigs, which would present significant advantages to the virus for the subsequent release of viral progeny and spread.\\n\\nThe onset of coronavirus-induced apoptosis is intricately linked with other host antiviral innate defenses. Infections with viruses often result in cell cycle arrests and the activation of unfolded protein response (UPR) due to ER stress, both of which may be accompanied by the parallel activation of apoptosis in the infected cells.\\n\\nDuring virus replication, newly translated viral proteins accumulate in the ER, which may also cause stress that leads to activation of the UPR. MHV S protein has been shown to activate three UPR transducers, inositol-requiring enzyme 1 (IRE1), activating transcription factor 6 (ATF6) and protein kinase RNA-like endoplasmic reticulum kinase (PERK) [125] . In particular, PERK activation leads to the activation of p38 MAPK [126] , which stimulates virus replication [127, 128] . SARS-CoV, on the other hand, has been shown to attenuate the IRE1 signaling pathway, which generally led to the down-regulation of stress responses and UPR, and further decreased apoptotic signaling in infected cells as well, to the likely benefit of the virus for better viral progeny production and release [129] .\\n\\nCoronavirus-induced apoptosis during the late stages of infection is also partially regulated by the p38 MAPK signaling pathway, which in turn up-regulates production of pro-inflammatory cytokines, interleukin (IL)-6 and IL-8, in infected host cells and mount an immune response against virus infection in these cells [130, 131] .\\n\\nHowever, while pro-inflammatory cytokines are up-regulated at the transcriptional level during coronavirus infection, there is minimal to moderate up-regulation at the translational level, leading to the hypothesis that the interaction between IBV spike (S) protein and host eukaryotic initiation factor 3 (eIF3) modulates host gene expression, especially genes involved in innate immunity that are activated during coronavirus infection [132] . This also agrees with the hypothesis that, notwithstanding the presence of virus-induced ER stress responses, a significant decline in host mRNA translation in infected cells impedes the expression of ER stress proteins despite elevated mRNA concentrations of the former [133] . Moreover, the negative modulation of p38 MAPK occurs through an up-regulation of the dual-specificity phosphatases 1 (DUSP1) feedback loop during IBV infection, which reduces cytokine production by dephosphorylation of phosphor-threonine and phosphor-tyrosine residues on activated p38 MAPKs [130] .\\n\\nPRRSV infection of porcine macrophages, too, appears to be under the control of the MAPK/ERK pathway, through various signaling pathways activated by the latter that regulate a variety of cellular processes [134] . In particular, chemical inhibition of the ERK pathway resulted in an attenuation of PRRSV infection during the early stages of infection post-virus attachment, during which a notable reduction in viral sub-genomic mRNA synthesis and translation, as well as in progeny virus release, was observed [134] . An increase in IL-10 production during the early stages of infection has also been observed in pigs infected with PRRSV. The up-regulation of IL-10, an important cytokine in the attenuation of innate and adaptive immune responses, is advantageous for PRRSV to sustain for a longer period of time in the host, and the PRRSV N protein has been suggested as the viral protein responsible for mediating IL-10 induction [135] .\\n\\nLikewise, the expression of pro-inflammatory mediators, IL-1\\uf062, IL-6, IL-8, and TNF-α, are up-regulated during EAV infection of equine endothelial cells and macrophages, with virulent EAV strains activating sufficiently greater amounts of these cytokines, especially TNF-α, than avirulent EAV strains [136] . Moreover, the expression of pro-inflammatory cytokines, IFN-α, TNF-α, IL-1 and IL-6, is also up-regulated during porcine respiratory coronavirus and TGEV infection, and the differential modulation of which are postulated to function as crucial mediators of viral respiratory diseases in pigs [41] . In particular, early type I IFN production in coronavirus-infected pigs effectuated immunomodulatory responses, which suggests the potential ability of IFN inducers as an effective antiviral strategy to control the rate of coronavirus infections in swine populations [137] .\\n\\nRANTES (regulated upon activation, normal T cell expressed and secreted) is a known member of the pro-inflammatory CC chemokines family that function in the modulation of the migration of inflammatory cells such as monocytes and Natural Killer cells to sites of infection, particularly as a form of host antiviral immune response during virus infection [138] . Like coronaviruses, PRRSV infection also triggers the activation of pro-inflammatory cytokines and chemokines, including RANTES [139] , which may be fundamental in initiating the pathological conditions associated with the arterivirus. The activation of RANTES transcription also requires the participation of adaptor molecules from the TLR signaling pathway, such as MyD88, TRIF and TNF receptor-associated factor 6 (TRAF6) [139] .\\n\\nInterestingly, respiratory PRRSV/PRCV viral co-infections that commonly occurs in pigs, on the other hand, frequently leads to severely attenuated innate and adaptive immune responses in order to extend the pathogenicity of PRRSV-thus down-regulating innate immunity-and renders the host more vulnerable to subsequent infections by PRCV, or other respiratory viruses, that instead up-regulates innate immunity, possibly due to further, more severe damage to pulmonary cells and tissue and which may ultimately lead to a more critical form of pneumonia [140] .\\n\\nOther types of host innate defense against virus infection include processes such as autophagy, a vital physiological process in which cells degrade their own organelles in response to a severe lack of nutrients or other cellular stresses, so as to facilitate the removal of damaged cellular components and prevent the build-up of unwanted products in the cells [141] . Autophagy is also known to mount crucial innate immune responses against invading pathogens, with degradation of the latter through autophagy; on the other hand, autophagosomes may instead help promote virus infection by bringing together viral replicase proteins [142] . Multiple host-derived cytokines have emerged to play differential roles in regulating the onset of this essential mechanism as well. Of note are the T-helper cells (Th1) group of cytokines, including IFN-γ, TNF-α, IL-1, IL-2 and IL-6, which have the ability to induce autophagy, while Th2 cytokines such as IL-4, IL-10 and IL-13 have been shown to inhibit autophagy [143] .\\n\\nRecent studies have shown the ability of IBV to induce autophagy during infection independently of cellular stress or nutrient deprivation, and this can be made possible through the functions of IBV nsp 6, as well as in nsp6 orthologs of mammalian coronaviruses such as MHV and arteriviruses such as PRRSV, although a direct link between IBV infection and autophagosome formation could not be established [22] . As such, the induction of autophagy that has previously been reported during coronavirus and arterivirus replication may instead be an example of an innate defense tactic against infection to remove unwanted viral particles, while nsp6-as well as its respective orthologs-may alternatively modify adaptive immune strategies by targeting the breakdown of immunomodulatory proteins synthesized by the ER in autophagosomes [22] .\\n\\nViruses may also exploit the host cell cycle to benefit their own replication [144] (Figure 3 ). Cell cycle regulation typically involves mechanisms critical to cell sustainability, such as the surveillance and correction of genetic damage and the impediment of unrestrained cell division. Cyclins and cyclin-dependent kinases (CDKs) are well-known regulatory proteins, and the activation of which will dictate a cell\\'s progress through the cell cycle. Briefly, activated heterodimers consisting of both cyclins and CDKs as the regulatory and catalytic subunits, respectively, will alter the phosphorylation state of various target proteins for progression into the next stage of the cell cycle. CDK inhibitors include tumor suppressors such as the cip/kip (CDK interacting protein/Kinase inhibitory protein) and the INK4a/ARF (Inhibitor of Kinase 4/Alternative Reading Frame) family of genes that thwart progression to the next stage of the cell cycle [145] .\\n\\nThe inactivation of CDKs can also be a reversible process through the phosphorylation of essential residues Tyr 15 and Thr 14, both of which are located within the CDK ATP-binding loop [146] . Dual-specificity phosphatases of the Cdc25 (cell division cycle 25) family are responsible for the dephosphorylation of Tyr 15 and Thr 14 [147] . These Cdc25 proteins are, in turn, inactivated by Chk1/Chk2-mediated phosphorylation; this inactivation averts Cdk dephosphorylation and impedes its subsequent activation [148] . Chk1/Chk2 activation after DNA damage and/or DNA replication hindrance is dependent on the ataxia-telangiectasia mutated (ATM) and ATM/Rad3-related (ATR) protein kinases in mammalian cells [149, 150] .\\n\\nIBV infection of cultured cells results in cell cycle arrest, at both S and G2/M phases, to boost viral replication and to enhance the production of viral proteins as well. This p53-independent growth inhibitory outcome is catalyzed by regulation of the expression of multiple cell cycle regulatory genes such as corresponding CDK complexes [50] and the down-regulation of down-regulation of G1 phase regulatory cyclins D1 and D2 [151] , as well as through systemic modulation of ATR-dependent cellular DNA damage response [152] . Specifically, the interaction of coronavirus nsp13 with the p125 subunit of DNA polymerase \\uf064 was discovered to trigger DNA replication stress in cells during IBV infection, which eventually led to cell cycle arrest at the S phase [152] .\\n\\nIt has also been observed that the N protein of several coronaviruses can localize in the nucleolus where it may perturb cell cycle activities of the host cell for the benefit of viral mRNA synthesis [153] [154] [155] [156] . IBV N, for example, appears to target CDK2, cyclins A and D1 for proteasomemediated degradation [50, 157] and cause the accumulation of hypophosphorylated retinoblastoma (pRB), resulting in the downregulation of CDK1, cyclins E and B1 [50] .\\n\\nThe regulation of host protein synthesis, especially at the initiation stage, is often a regular viral objective for the extensive reduction of host protein translation so as to construct the most favorable condition for viral replication and progeny assembly. Protein kinase R (PKR) is a prevalent serine/threonine protein kinase that can be induced by interferon in its latent state, and whose activation is dependent by the presence of dsRNA products, is one such viral target. PKR plays an important role in the cellular anti-viral response pathway, and becomes activated via auto-phosphorylation upon binding to virus-derived dsRNA, which then subsequently leads to host translation inhibition through phosphorylation of the alpha subunit of eukaryotic initiation factor 2, eIF2alpha [158] .\\n\\nThe dephosphorylation of eIF2alpha is established by cellular protein phosphatase-1 (PP1), which functions to regular various cellular processes through the physical interaction of its catalytic subunit (PP1c) with regulatory proteins such as GADD34/CHOP that is induced by DNA damage signals [159, 160] .\\n\\nIn the case of IBV, for example, eIF2alpha phosphorylation was significantly suppressed in both human and animal cells during infection. The dephosphorylation of eIF2alpha during IBV infection appears to be induced by the up-regulation of GADD34 expression, which, together with the simultaneous attenuation of PKR auto-phosphorylation following IBV infection in these cells, serves as a viral regulatory tactic in boosting coronavirus replication while managing the delicate balance of de novo protein translation along with the identification of IBV nsp2 as a potential, albeit weak, mediator in the dose-dependent inhibition of PKR activation [161] . TGEV, too, suppresses both cellular RNA degradation and eIF2a phosphorylation during infection through an interaction between TGEV protein 7 and PP1 that regulates host antiviral responses and extends the period of viral progeny dispersal as well [53] .\\n\\nA subunit of the eukaryotic initiation factor 3, eIF3f, has also been discovered to modulate host translation inhibitory effects through physical interaction with coronavirus spike protein [132] . As this inhibition takes place during the late stages of the coronavirus replication cycle, translation of virus-induced transcripts are largely affected, especially pro-inflammatory cytokines and chemokines. This could account for the fact that while IL-6 mRNA expression is up-regulated during the early stages of coronavirus infection, insubstantial increase in IL-6 protein expression was observed [132] . The inhibition of host protein synthesis through the interaction between coronavirus spike protein and eIF3f may therefore have a significant impact on the modulation of coronavirus pathogenicity. \\n\\nHost proteins are also known to play a role in the virus life cycle, especially during viral RNA replication and transcription (Figure 4) . The most well studied host protein that interacts with the coronavirus genome is heterogeneous nuclear ribonucleoprotein A1 (hnRNP A1), a nuclear protein, whose biological function is to regulate alternative splicing of cellular RNAs [162, 163] . The hnRNP A1 has been shown to bind to both negative-sense leader sequence and negative-sense intergenic (IG) sequence of MHV [164] . The formation of a RNP complex among hnRNP A1, negative-sense leader and IG sequences has also been demonstrated [165] . In addition to its ability to interact with the coronavirus RNA, hnRNP A1 was also found to interact and co-localize with N protein [166, 167] , an important player in coronavirus RNA synthesis [168, 169] . It has also been highlighted that hnRNP A1 may be required to recruit other cellular proteins to the replicase complex [170] .\\n\\nTo ascertain the involvement of cellular factors in TGEV RNA synthesis, TGEV 3\\' and 5\\' genome ends were used as baits for RNA affinity protein purification [171] . Of the ten cellular proteins pulled down with either genome end, poly(A)-binding protein (PABP), hnRNP Q, and glutamyl-prolyl-tRNA synthetase (EPRS) were confirmed to enhance TGEV infection through their respective interactions with the TGEV 3\\' end, while glyceraldehyde 3-phosphate dehydrogenase (GAPDH)-originally employed as a control-was discovered, surprisingly, to have a diminishing effect on TGEV infection instead [171] .\\n\\nFor arteriviruses, the common leader sequence of EAV sub-genomic viral RNAs, too, possesses the ability to interact several cellular proteins from the cytoplasmic fractions of Vero cells, likely for the modulation of EAV RNA synthesis [172] .\\n\\nRecent reports, based on a yeast-based three-hybrid system to identify RNA-RNA-binding regulatory protein interactions [173] , and using the 5\\'-UTR (untranslated region) of SARS coronavirus as bait, identified zinc finger CCHC-type and RNA-binding motif 1 (MADP1) as a potential cellular protein that interacts with SARS-CoV, and this was eventually established via an in vitro pull-down assay with the 5\\'-UTR of IBV [174] . MADP1 was also shown by the same authors to play a role in the early stages of the coronavirus replication cycle, with the RNA recognition motif in the N-terminal region of the protein interacting with stem loop 1 of the IBV 5\\'-UTR. Specifically, this protein translocates from the nucleus to the cytoplasm of the cell during IBV infection and co-localizes, in part, with viral replicase/transcriptase complexes (RTCs) in order to enhance viral replication and coronavirus RNA synthesis [174] .\\n\\nInteraction with viral RNA is not the only way by which cellular proteins can take part in viral RNA synthesis; protein-protein interactions with the viral replicase complex can modulate this process as well. This was confirmed in coronaviruses with a yeast two-hybrid screen that was performed using IBV nsp14 as a bait protein, which ultimately led to the discovery of DDX1, an ATP-dependent RNA helicase in the DExD/H helicase family, as an interacting partner that translocates from the nucleus to the cytoplasm and enhances IBV replication in cultured mammalian cells through subcellular colocalization with nsp14, an exonuclease, during infection [175] .\\n\\nAnother yeast two-hybrid screen, with IBV membrane (M) protein as the bait, identified beta-actin as an interacting partner, leading to the suggestion that actin filaments may possess the ability to participate in virion assembly and budding during the coronavirus replication cycle [176] . This interaction may possibly lead to the incorporation of actin into the mature virion. In fact, recent proteomic analysis of purified IBV particles through two-dimensional gel electrophoresis and subsequent mass spectrometry have confirmed, among others, an abundance of actin within the virus particles, further cementing the hypothesis that cytoskeletal elements play crucial roles in IBV replication [177] .\\n\\nRetinoblastoma tumor suppressor proteins are also interacting partners of coronavirus nsp15, an endoribonuclease, which may result in alterations to the cell cycle that impact coronavirus infection and viral progeny release [178] . Similarly, with nsp1\\uf062, the functional proteolytic product of PRRSV nsp1, as a bait protein, cellular poly(C)-binding proteins 1 and 2 (PCBP1 and PCBP2, respectively) have been identified interacting partners of the former, with specific functions in modulating PRRSV replication and RNA synthesis [179] . . Host proteins and the coronavirus life cycle. Virus particle attaches onto the host cell via cellular receptors on the surface and enters. Entry is followed by the uncoating of the ribonucleocapsid to expose the positive-sense genomic RNA which is translated by the host ribosomes to yield the viral replication complex. The viral replication complex continues with viral transcription and genome replication and, with the aid of host proteins such as hnRNPA1, yields a nested set of positive-sense sub-genomic sized mRNAs as well as the full-length virus genome. Sub-genomic sized mRNAs are translated by host ribosomes into viral structural (S, E, M, N) and accessory proteins. The N protein packages the positive-sense genomic RNA into a ribonucleocapsid and is assembled into the virus particles with the help of \\uf062-actin. The newly formed virus particles undergo maturation when passing through the Golgi and exit the host cell via exocytosis.\\n\\nThe relationship between a virus and its host is complex; the virus must evolve evasive strategies to avoid detection and immunological defense mechanisms from the host while the host must develop various lines of defense in order to combat viral invasion.\\n\\nAs highlighted in this review, the diverse virus-host interactions established during animal coronavirus and arterivirus infections may have direct implications on viral replication itself, or they may also lead to the modification of numerous signaling pathways, such as cellular stress and/or host antiviral innate immune response, as a means to expedite viral replication and pathogenesis.\\n\\nThe intricate network between a virus and its host is a complicated affair that involves many players-derived from both virus and host-which are pitted against one another in the battle for ascendancy; certain host processes are readily assimilated and manipulated by the virus in its bid to impede host antiviral responses while the host is itself armored with several lines of defense mechanisms and numerous antiviral factors to combat viral invasion to prevent its spread.\\n\\nWhile much progress has been made in the elucidation of the various regulatory pathways that, under the most propitious of conditions, will allow virus and host to co-exist in an uneasy truce, a clearer understanding of the complex interplay between the virus and its host could give new insights into the role of main players, such as those that govern the onset of apoptosis, or type I interferons and their regulators, and lead to the discovery of novel and/or universal targets for the therapeutic mediation against pathogenic infection.', 'document_id': 1722}]}, {'paragraphs': [{'qas': [{'question': 'What method is developed in this study?', 'id': 2754, 'answers': [{'text': 'a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probably be bats) to the human infection. ', 'answer_start': 840}], 'is_impossible': False}, {'question': 'What is the model simplified to?', 'id': 2755, 'answers': [{'text': 'Reservoir-People (RP) transmission network model', 'answer_start': 1216}], 'is_impossible': False}, {'question': 'What is the estimate of R 0?', 'id': 2756, 'answers': [{'text': 'The value of R 0 was estimated of 2.30 from reservoir to person and 3.58 from person to person which means that the expected number of secondary infections that result from introducing a single infected individual into an otherwise susceptible population was 3.5', 'answer_start': 1439}], 'is_impossible': False}, {'question': 'What is the conclusion of this study?', 'id': 2757, 'answers': [{'text': 'Our model showed that the transmissibility of SARS-CoV-2 was higher than the Middle East respiratory syndrome in the Middle East countries, similar to severe acute respiratory syndrome, but lower than MERS in the Republic of Korea.', 'answer_start': 1716}], 'is_impossible': False}, {'question': 'What was the focus of the study?', 'id': 2758, 'answers': [{'text': 'Huanan Seafood Wholesale Market (reservoir) to people,', 'answer_start': 3803}], 'is_impossible': False}, {'question': 'What were the model assumptions?', 'id': 2759, 'answers': [{'text': 'the virus transmitted among the bats, and then transmitted to unknown hosts (probably some wild animals). The hosts were hunted and sent to the seafood market which was defined as the reservoir of the virus. People exposed to the market got the risks of the infection ', 'answer_start': 5011}], 'is_impossible': False}, {'question': 'What  compartments were the bats divided into?', 'id': 2760, 'answers': [{'text': 'susceptible bats (S B ), exposed bats (E B ), infected bats (I B ), and removed bats (R B ).', 'answer_start': 5425}], 'is_impossible': False}, {'question': 'What compartments were the host animals divided into?', 'id': 2761, 'answers': [{'text': 'The hosts were also divided into four compartments: susceptible hosts (S H ), exposed hosts (E H ), infected hosts (I H ), and removed hosts (R H )', 'answer_start': 5942}], 'is_impossible': False}, {'question': 'What was the SARS-COV-2 reservoir?', 'id': 2762, 'answers': [{'text': 'the seafood market', 'answer_start': 6547}], 'is_impossible': False}, {'question': 'What were the people divided into?', 'id': 2763, 'answers': [{'text': ' into five compartments:\\n\\nsusceptible people (S P ), exposed people (E P ), symptomatic infected people (I P ), asymptomatic infected people (A P ), and removed people (R P ) including recovered and death people.', 'answer_start': 7197}], 'is_impossible': False}, {'question': 'What was the mean incubation period?', 'id': 2764, 'answers': [{'text': '5.2 days (95% confidence interval [CI]: 4.1-7.0) ', 'answer_start': 9998}], 'is_impossible': False}, {'question': 'What was the mean delay from symptom onset to detection/hospitalization of a case?', 'id': 2765, 'answers': [{'text': ' 5-day', 'answer_start': 10193}], 'is_impossible': False}, {'question': 'How long after onset, the cases detected in Thailand and Japan were hospitalized?', 'id': 2766, 'answers': [{'text': ' from 3 to 7 days', 'answer_start': 10323}], 'is_impossible': False}, {'question': 'What was the duration from illness onset to first medical visit ?', 'id': 2767, 'answers': [{'text': 'a mean of 5.8 days (95% CI: 4.3-7.5)', 'answer_start': 10518}], 'is_impossible': False}, {'question': 'What was the assumption of transmissibility of asymptomatic infection?', 'id': 2768, 'answers': [{'text': '0.5 times that of symptomatic infection (κ = 0.5), which was the similar value as influenza ', 'answer_start': 10958}], 'is_impossible': False}, {'question': 'As of January 17, how many people were tested for body temperature?', 'id': 2769, 'answers': [{'text': ' 0.3 million people', 'answer_start': 11379}], 'is_impossible': False}, {'question': 'What is mobile  population in Wuhan?', 'id': 2770, 'answers': [{'text': 'about 2.87 million', 'answer_start': 11463}], 'is_impossible': False}, {'question': 'What was  the R0 of SARS?', 'id': 2771, 'answers': [{'text': '2.7-3.4 or 2-4 in Hong Kong', 'answer_start': 15034}], 'is_impossible': False}, {'question': 'What was the value of R0 in other researches?', 'id': 2772, 'answers': [{'text': 'R 0 of SARS was about 2.1 in Hong Kong, China, 2.7 in Singapore, and 3.8 in Beijing, China', 'answer_start': 15112}], 'is_impossible': False}, {'question': 'What is the reported value of R0 for MERS?', 'id': 2773, 'answers': [{'text': '0.8-1.3 ', 'answer_start': 15455}], 'is_impossible': False}, {'question': 'What was  R0 for  the high transmissibility in  South Korea?', 'id': 2774, 'answers': [{'text': ' 2.5-7.2', 'answer_start': 15674}], 'is_impossible': False}, {'question': 'What is important for containing the transmission?', 'id': 2775, 'answers': [{'text': ' to decrease R 0', 'answer_start': 15929}], 'is_impossible': False}, {'question': 'What did this model show?', 'id': 2776, 'answers': [{'text': ' the transmissibility of SARS-CoV-2 might be higher than MERS in the Middle East countries, similar to SARS, but lower than MERS in the Republic of Korea.', 'answer_start': 17978}], 'is_impossible': False}, {'question': 'What was the objective of the study?', 'id': 2777, 'answers': [{'text': 'provide a mathematical model for calculating the transmissibility of SARS-CoV-2', 'answer_start': 18174}], 'is_impossible': False}], 'context': \"A mathematical model for simulating the phase-based transmissibility of a novel coronavirus\\n\\nhttps://doi.org/10.1186/s40249-020-00640-3\\n\\nSHA: 018269476cd191365d6b8bed046078aea07c8c01\\n\\nAuthors: Yin, Tian-Mu Chen; Jia, Rui; Qiu-Peng, Wang; Ze-Yu, Zhao; Jing-An, Cui; Ling\\nDate: 2020\\nDOI: 10.1186/s40249-020-00640-3\\nLicense: cc-by\\n\\nAbstract: Background As reported by the World Health Organization, a novel coronavirus (2019-nCoV) was identified as the causative virus of Wuhan pneumonia of unknown etiology by Chinese authorities on 7 January, 2020. The virus was named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by International Committee on Taxonomy of Viruses on 11 February, 2020. This study aimed to develop a mathematical model for calculating the transmissibility of the virus. Methods In this study, we developed a Bats-Hosts-Reservoir-People transmission network model for simulating the potential transmission from the infection source (probably be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from Huanan Seafood Wholesale Market (reservoir) to people, we simplified the model as Reservoir-People (RP) transmission network model. The next generation matrix approach was adopted to calculate the basic reproduction number (R 0) from the RP model to assess the transmissibility of the SARS-CoV-2. Results The value of R 0 was estimated of 2.30 from reservoir to person and 3.58 from person to person which means that the expected number of secondary infections that result from introducing a single infected individual into an otherwise susceptible population was 3.58. Conclusions Our model showed that the transmissibility of SARS-CoV-2 was higher than the Middle East respiratory syndrome in the Middle East countries, similar to severe acute respiratory syndrome, but lower than MERS in the Republic of Korea.\\n\\nText: On 31 December 2019, the World Health Organization (WHO) China Country Office was informed of cases of pneumonia of unknown etiology (unknown cause) detected in Wuhan City, Hubei Province of China, and WHO reported that a novel coronavirus (2019-nCoV), which was named as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) by International Committee on Taxonomy of Viruses on 11 February, 2020, was identified as the causative virus by Chinese authorities on 7 January [1] . It is reported that the virus might be bat origin [2] , and the transmission of the virus might related to a seafood market (Huanan Seafood Wholesale Market) exposure [3, 4] . The genetic features and some clinical findings of the infection have been reported recently [4] [5] [6] . Potentials for international spread via commercial air travel had been assessed [7] . Public health concerns are being paid globally on how many people are infected and suspected.\\n\\nTherefore, it is urgent to develop a mathematical model to estimate the transmissibility and dynamic of the transmission of the virus. There were several researches focusing on mathematical modelling [3, 8] . These researches focused on calculating the basic reproduction number (R 0 ) by using the serial intervals and intrinsic growth rate [3, 9, 10] , or using ordinary differential equations and Markov Chain Monte Carlo methods [8] . However, the bat origin and the transmission route form the seafood market to people were not considered in the published models.\\n\\nIn this study, we developed a Bats-Hosts-Reservoir-People (BHRP) transmission network model for simulating the potential transmission from the infection source (probably be bats) to the human infection. Since the Bats-Hosts-Reservoir network was hard to explore clearly and public concerns were focusing on the transmission from Huanan Seafood Wholesale Market (reservoir) to people, we simplified the model as Reservoir-People (RP) transmission network model, and R 0 was calculated based on the RP model to assess the transmissibility of the SARS-CoV-2.\\n\\nThe reported cases of SARS-CoV-2, which have been named as COVID-19, were collected for the modelling study from a published literature [3] . As reported by Li et al. [3] , the onset date of the first case was on 7 December, 2020, and the seafood market was closed on 1 January, 2020 [11] . The epidemic curve from 7 December, 2019 to 1 January, 2020 was collected for our study, and the simulation time step was 1 day. fourth-order Runge-Kutta method, with tolerance set at 0.001, was used to perform curve fitting. While the curve fitting is in progress, Berkeley Madonna displays the root mean square deviation between the data and best run so far. The coefficient of determination (R 2 ) was employed to assess the goodness-of-fit. SPSS 13.0 (IBM Corp., Armonk, NY, USA) was employed to calculate the R 2 .\\n\\nThe Bats-Hosts-Reservoir-People (BHRP) transmission network model\\n\\nThe BHRP transmission network model was posted to bioRxiv on 19 January, 2020 [12] . We assumed that the virus transmitted among the bats, and then transmitted to unknown hosts (probably some wild animals). The hosts were hunted and sent to the seafood market which was defined as the reservoir of the virus. People exposed to the market got the risks of the infection (Fig. 1) . The BHRP transmission network model was based on the following assumptions or facts:\\n\\na) The bats were divided into four compartments: susceptible bats (S B ), exposed bats (E B ), infected bats (I B ), and removed bats (R B ). The birth rate and death rate of bats were defined as n B and m B . In this model, we set Ʌ B = n B × N B as the number of the newborn bats where N B refer to the total number of bats. The incubation period of bat infection was defined as 1/ω B and the infectious period of bat infection was defined as 1/γ B . The S B will be infected through sufficient contact with I B , and the transmission rate was defined as β B . b) The hosts were also divided into four compartments: susceptible hosts (S H ), exposed hosts (E H ), infected hosts (I H ), and removed hosts (R H ). The birth rate and death rate of hosts were defined as n H and m H . In this model, we set Ʌ H = n H × N H where N H refer to the total number of hosts. The incubation period of host infection was defined as 1/ω H and the infectious period of host infection was defined as 1/γ H . The S H will be infected through sufficient contact with I B and I H , and the transmission rates were defined as β BH and β H , respectively. c) The SARS-CoV-2 in reservoir (the seafood market) was denoted as W. We assumed that the retail purchases rate of the hosts in the market was a, and that the prevalence of SARS-CoV-2 in the purchases was I H /N H , therefore, the rate of the SARS-CoV-2 in W imported form the hosts was aWI H /N H where N H was the total number of hosts. We also assumed that symptomatic infected people and asymptomatic infected people could export the virus into W with the rate of μ P and μ' P , although this assumption might occur in a low probability. The virus in W will subsequently leave the W compartment at a rate of εW, where 1/ε is the lifetime of the virus. d) The people were divided into five compartments:\\n\\nsusceptible people (S P ), exposed people (E P ), symptomatic infected people (I P ), asymptomatic infected people (A P ), and removed people (R P ) including recovered and death people. The birth rate and death rate of people were defined as n P and m P . In this model, we set Ʌ P = n P × N P where N P refer to the total number of people. The incubation period and latent period of human infection was defined as 1/ω P and 1/ω' P . The infectious period of I P and A P was defined as 1/γ P and 1/γ' P . The proportion of asymptomatic infection was defined as δ P . The S P will be infected through sufficient contact with W and I P , and the transmission rates were defined as β W and β P , respectively. We also assumed that the transmissibility of A P was κ times that of I P , where 0 ≤ κ ≤ 1.\\n\\nThe parameters of the BHRP model were shown in Table 1 .\\n\\nWe assumed that the SARS-CoV-2 might be imported to the seafood market in a short time. Therefore, we added the further assumptions as follows:\\n\\na) The transmission network of Bats-Host was ignored. b) Based on our previous studies on simulating importation [13, 14] , we set the initial value of W as following impulse function:\\n\\nIn the function, n, t 0 and t i refer to imported volume of the SARS-CoV-2 to the market, start time of the simulation, and the interval of the importation.\\n\\nTherefore, the BHRP model was simplified as RP model and is shown as follows:\\n\\nDuring the outbreak period, the natural birth rate and death rate in the population was in a relative low level. However, people would commonly travel into and out from Wuhan City mainly due to the Chinese New Year holiday. Therefore, n P and m P refer to the rate of people traveling into Wuhan City and traveling out from Wuhan City, respectively.\\n\\nIn the model, people and viruses have different dimensions. Based on our previous research [15] , we therefore used the following sets to perform the normalization:\\n\\nIn the normalization, parameter c refers to the relative shedding coefficient of A P compared to I P . The normalized RP model is changed as follows:\\n\\nThe transmissibility of the SARS-CoV-2 based on the RP model\\n\\nIn this study, we used the R 0 to assess the transmissibility of the SARS-CoV-2. Commonly, R 0 was defined as the expected number of secondary infections that result from introducing a single infected individual into an otherwise susceptible population [13, 16, 17] . If R 0 > 1, the outbreak will occur. If R 0 < 1, the outbreak will toward an end. In this study, R 0 was deduced from the RP model by the next generation matrix approach [18] . The multiple of the transmissibility of A P to that of I P .\\n\\nThe parameters were estimated based on the following facts and assumptions:\\n\\na) The mean incubation period was 5.2 days (95% confidence interval [CI]: 4.1-7.0) [3] . We set the same value (5.2 days) of the incubation period and the latent period in this study. Thus, ω P = ω' P = 0.1923. b) There is a mean 5-day delay from symptom onset to detection/hospitalization of a case (the cases detected in Thailand and Japan were hospitalized from 3 to 7 days after onset, respectively) [19] [20] [21] . The duration from illness onset to first medical visit for the 45 patients with illness onset before January 1 was estimated to have a mean of 5.8 days (95% CI: 4.3-7.5) [3] . In our model, we set the infectious period of the cases as 5.8 days. Therefore, γ P = 0.1724. c) Since there was no data on the proportion of asymptomatic infection of the virus, we simulated the baseline value of proportion of 0.5 (δ P = 0.5). d) Since there was no evidence about the transmissibility of asymptomatic infection, we assumed that the transmissibility of asymptomatic infection was 0.5 times that of symptomatic infection (κ = 0.5), which was the similar value as influenza [22] . We assumed that the relative shedding rate of A P compared to I P was 0.5. Thus, c = 0.5. e) Since 14 January, 2020, Wuhan City has strengthened the body temperature detection of passengers leaving Wuhan at airports, railway stations, long-distance bus stations and passenger terminals. As of January 17, a total of nearly 0.3 million people had been tested for body temperature [23] . In Wuhan, there are about 2.87 million mobile population [24] . We assumed that there was 0.1 million people moving out to Wuhan City per day since January 10, 2020, and we believe that this number would increase (mainly due to the winter vacation and the Chinese New Year holiday) until 24 January, 2020. This means that the 2.87 million would move out from Wuhan City in about 14 days. Therefore, we set the moving volume of 0.2 million per day in our model. Since the population of Wuhan was about 11 million at the end of 2018 [25] , the rate of people traveling out from Wuhan City would be 0.018 (0.2/11) per day. However, we assumed that the normal population mobility before January 1 was 0.1 times as that after January 10. Therefore, we set the rate of people moving into and moving out from Wuhan City as 0.0018 per day (n P = m P = 0.0018).\\n\\nf) The parameters b P and b W were estimated by fitting the model with the collected data. g) At the beginning of the simulation, we assumed that the prevalence of the virus in the market was 1/100000. h) Since the SARS-CoV-2 is an RNA virus, we assumed that it could be died in the environment in a short time, but it could be stay for a longer time (10 days) in the unknown hosts in the market. We set ε = 0.1.\\n\\nIn this study, we assumed that the incubation period (1/ ω P ) was the same as latent period (1/ω' P ) of human infection, thus ω P = ω' P . Based on the equations of RP model, we can get the disease free equilibrium point as: In the matrix:\\n\\nBy the next generation matrix approach, we can get the next generation matrix and R 0 for the RP model: \\n\\nThe R 0 of the normalized RP model is shown as follows:\\n\\nOur modelling results showed that the normalized RP model fitted well to the reported SARS-CoV-2 cases data (R 2 = 0.512, P < 0.001) (Fig. 2) . The value of R 0 was estimated of 2.30 from reservoir to person, and from person to person and 3.58 from person to person which means that the expected number of secondary infections that result from introducing a single infected individual into an otherwise susceptible population was 3.58.\\n\\nIn this study, we developed RP transmission model, which considering the routes from reservoir to person and from person to person of SARS-CoV-2 respectively. We used the models to fit the reported data in Wuhan City, China from published literature [3] . The simulation results showed that the R 0 of SARS-CoV-2 was 3.58 from person to person. There was a research showed that the R 0 of SARS-CoV-2 was 2.68 (95% CI: 2.47-2.86) [8] . Another research showed that the R 0 of SARS-CoV-2 was 2.2 (95% CI: 1.4-3.9) [3] . The different values might be due to the different methods. The methods which Li et al. employed were based on the epidemic growth rate of the epidemic curve and the serial interval [3] . Our previous study showed that several methods could be used to calculate the R 0 based on the epidemic growth rate of the epidemic curve and the serial interval, and different methods might result in different values of R 0 [26] . Our results also showed that the R 0 of SARS-CoV-2 was 2.30 from reservoir to person which was lower than that of person to person. This means that the transmission route was mainly from person to person rather than from reservoir to person in the early stage of the transmission in Wuhan City. However, this result was based on the limited data from a published literature, and it might not show the real situation at the early stage of the transmission.\\n\\nResearches showed that the R 0 of severe acute respiratory syndrome (SARS) was about 2.7-3.4 or 2-4 in Hong Kong, China [27, 28] . Another research found that the R 0 of SARS was about 2.1 in Hong Kong, China, 2.7 in Singapore, and 3.8 in Beijing, China [29] . Therefore, we believe that the commonly acceptable average value of the R 0 of SARS might be 2.9 [30] . The transmissibility of the Middle East respiratory syndrome (MERS) is much lower than SARS. The reported value of the R 0 of MERS was about 0.8-1.3 [31] , with the inter-human transmissibility of the disease was about 0.6 or 0.9 in Middle East countries [32] . However, MERS had a high transmissibility in the outbreak in the Republic of Korea with the R 0 of 2.5-7.2 [33, 34] . Therefore, the transmissibility of SARS-CoV-2 might be higher than MERS in the Middle East countries, similar to SARS, but lower than MERS transmitted in the Republic of Korea.\\n\\nTo contain the transmission of the virus, it is important to decrease R 0 . According to the equation of R 0 deduced from the simplified RP model, R 0 is related to many parameters. The mainly parameters which could be changed were b P , b W , and γ. Interventions such as wearing masks and increasing social distance could decrease the b P , the intervention that close the seafood market could decrease the b W , and shorten the duration form symptoms onset to be diagnosed could decrease 1/γ. All these interventions could decrease the effective reproduction number and finally be helpful to control the transmission.\\n\\nSince there are too many parameters in our model, several limitations exist in this study. Firstly, we did not use the detailed data of the SARS-CoV-2 to perform the estimation instead of using the data from literatures [3] . We simulated the natural history of the infection that the proportion of asymptomatic infection was 50%, and the transmissibility of asymptomatic infection was half of that of symptomatic infection, which were different to those of MERS and SARS. It is known that the proportion of asymptomatic infection of MERS and SARS was lower than 10%. Secondly, the parameters of population mobility were not from an accurate dataset. Thirdly, since there was no data of the initial prevalence of the virus in the seafood market, we assumed the initial value of 1/100 000. This assumption might lead to the simulation been under-or over-estimated. In addition, since we did not consider the changing rate of the individual's activity (such as wearing masks, increasing social distance, and not to travel to Wuhan City), the estimation of importation of the virus might not be correct. All these limitations will lead to the uncertainty of our results. Therefore, the accuracy and the validity of the estimation would be better if the models fit the first-hand data on the population mobility and the data on the natural history, the epidemiological characteristics, and the transmission mechanism of the virus.\\n\\nBy calculating the published data, our model showed that the transmissibility of SARS-CoV-2 might be higher than MERS in the Middle East countries, similar to SARS, but lower than MERS in the Republic of Korea. Since the objective of this study was to provide a mathematical model for calculating the transmissibility of SARS-CoV-2, the R 0 was estimated based on limited data which published in a literature. More data were needed to estimate the transmissibility accurately.\", 'document_id': 2592}]}, {'paragraphs': [{'qas': [{'question': 'What are coronaviruses?', 'id': 2199, 'answers': [{'text': 's are spherical, enveloped, and the largest of positive-strand RNA v', 'answer_start': 1930}], 'is_impossible': False}, {'question': 'What animals can carry coronavirus?', 'id': 2200, 'answers': [{'text': 'uding birds, farm animals, pets, camels, an', 'answer_start': 2039}], 'is_impossible': False}], 'context': 'Potential Maternal and Infant Outcomes from (Wuhan) Coronavirus 2019-nCoV Infecting Pregnant Women: Lessons from SARS, MERS, and Other Human Coronavirus Infections\\n\\nhttps://doi.org/10.3390/v12020194\\n\\nSHA: 779c1b5cb3afe3d50219aa2af791014a22eb355a\\n\\nAuthors: Schwartz, David A.; Graham, Ashley L.\\nDate: 2020\\nDOI: 10.3390/v12020194\\nLicense: cc-by\\n\\nAbstract: In early December 2019 a cluster of cases of pneumonia of unknown cause was identified in Wuhan, a city of 11 million persons in the People&rsquo;s Republic of China. Further investigation revealed these cases to result from infection with a newly identified coronavirus, termed the 2019-nCoV. The infection moved rapidly through China, spread to Thailand and Japan, extended into adjacent countries through infected persons travelling by air, eventually reaching multiple countries and continents. Similar to such other coronaviruses as those causing the Middle East respiratory syndrome (MERS) and severe acute respiratory syndrome (SARS), the new coronavirus was reported to spread via natural aerosols from human-to-human. In the early stages of this epidemic the case fatality rate is estimated to be approximately 2%, with the majority of deaths occurring in special populations. Unfortunately, there is limited experience with coronavirus infections during pregnancy, and it now appears certain that pregnant women have become infected during the present 2019-nCoV epidemic. In order to assess the potential of the Wuhan 2019-nCoV to cause maternal, fetal and neonatal morbidity and other poor obstetrical outcomes, this communication reviews the published data addressing the epidemiological and clinical effects of SARS, MERS, and other coronavirus infections on pregnant women and their infants. Recommendations are also made for the consideration of pregnant women in the design, clinical trials, and implementation of future 2019-nCoV vaccines.\\n\\nText: Coronaviruses are spherical, enveloped, and the largest of positive-strand RNA viruses. They have a wide host range, including birds, farm animals, pets, camels, and bats, in which they primarily cause respiratory and gastrointestinal disease. Belonging to the order Nidovirales, family Coronaviridae, and the subfamily Orthocoronaviridae there are four genera of coronaviruses-Alphacoronavirus, Betacoronavirus, Deltacorona virus, and Gammacoronavirus [1] [2] [3] [4] .\\n\\nIn humans, they are a cause of mild illnesses including the common colds occurring in children and adults, and were believed to be of modest medical importance. However, two zoonotic coronaviruses-including the severe acute respiratory syndrome coronavirus (SARS-CoV) and Middle East respiratory syndrome coronavirus (MERS-CoV)-can produce severe lower respiratory In the beginning of December 2019, a cluster of persons with a pneumonia of unknown cause was identified in Wuhan, the capital of Hubei Province and a large city of approximately 11 million persons located in the central region of the People\\'s Republic of China [7, 8] . Between 8 and 18 December 2019 there were 7 cases of pneumonia identified whose clinical features resembled that of a viral pneumonia. The outbreak was initially believed to be linked to the Wuhan Huanan (South China) Seafood Wholesale Market. This market, termed a \"wet\" market, sells a variety of seafood, cuts of meat, and both live and dead animals in over one thousand stalls in constant close contact; however, whether this market was the origin of the outbreak remains unknown [9] . On 31 December 2019, the Chinese Center for Disease Control and Prevention (China CDC) sent a rapid response team to Hubei to work alongside health personnel from the provincial and Wuhan city health departments to conduct an epidemiologic investigation. As the disease was spreading through secondary and tertiary cases, the World Health Organization (WHO) China Country Office was informed on 31 December 2019 of the occurrence of these cases of pneumonia of unknown etiology. During the period from 31 December 2019 to 3 January 2020, 44 patients with pneumonia of unknown etiology were reported by the Chinese authorities to the WHO. On 7 January 2020 investigators in China identified the etiological agent of the epidemic as a previously unknown coronavirus, and it was given the designation 2019-nCoV (for 2019 novel coronavirus) [8] . Analysis of the clinical features of 41 hospitalized patients with laboratory-confirmed 2019-nCoV infection revealed that 30 were men (73%); less than one-half had underlying co-morbid conditions (13; 32%) which included diabetes (8, 20%) , hypertension (6, 15%), and cardiovascular disease (6; 15%); and the average age was 49.0 years old. The most common symptoms at the beginning of their illness included fever (40, 98%) , cough (31, 76%) , and fatigue or myalgia (18, 44%) , sputum production (11, 28%) , and headache (3, 8%) [10] . Among these 41 initial cases of 2019-nCoV infection there were 12 patients (32%) who developed acute respiratory distress syndrome (ARDS), 13 (32%) required intensive care and 6 (15%) died. During the first weeks of January the infection spread rapidly through China and extended to adjacent countries where cases began to appear-13 January in Thailand, 15 January in Japan, 20 January in the Republic of Korea, and Taiwan and the United States on 21 January [11] . Infected travelers, mostly via commercial air travel, are known to have been responsible for introducing the virus outside of Wuhan. The new coronavirus continued to spread throughout multiple countries and continents, and by 9 February 2020 the WHO reported 37,251 confirmed cases in China that resulted in 812 deaths, surpassing the number of deaths that occurred during the 2002-2003 SARS epidemic. An additional 307 cases of 2019-nCoV infection have occurred among 24 other countries outside of China [12] . (Figure 1 ) At the meeting of the Emergency Committee of the WHO on 30 January, the novel coronavirus 2019 epidemic was declared a Public Health Emergency of International Concern (PHEIC) [11, 13] .\\n\\nViruses 2020, 12, 194 3 of 16 epidemic. An additional 307 cases of 2019-nCoV infection have occurred among 24 other countries outside of China [12] . (Figure 1 ) At the meeting of the Emergency Committee of the WHO on 30 January, the novel coronavirus 2019 epidemic was declared a Public Health Emergency of International Concern (PHEIC) [11, 13] . This newly recognized coronavirus, producing a disease that has been termed COVID-19, is rapidly spreading throughout China, has crossed international borders to infect persons in neighboring countries, and humans infected by the virus are travelling via commercial airlines to other continents. It is certain that 2019-nCoV will infect women who are pregnant, leaving the question open as to whether the novel coronavirus will have a similar or different effect on them compared with SARS-CoV and MERS-CoV. In order to address the potential obstetrical outcomes of infection to both mother and infant, the present communication describes the current state of knowledge regarding the effects of other coronavirus infections in pregnancy.\\n\\nPneumonia arising from any infectious etiology is an important cause of morbidity and mortality among pregnant women. It is the most prevalent non-obstetric infectious condition that occurs during pregnancy [14] [15] [16] . In one study pneumonia was the 3rd most common cause of indirect maternal death [17] . Approximately 25 percent of pregnant women who develop pneumonia will need to be hospitalized in critical care units and require ventilatory support [16] . Although bacterial pneumonia is a serious disease when it occurs in pregnant women, even when the agent(s) are susceptible to antibiotics, viral pneumonia has even higher levels of morbidity and mortality during pregnancy [18] . As with other infectious diseases, the normal maternal physiologic changes that accompany pregnancy-including altered cell-mediated immunity [19] and changes in pulmonary function-have been hypothesized to affect both susceptibility to and clinical severity of pneumonia [20] [21] [22] . This has been evident historically during previous epidemics. The case fatality rate (CFR) for pregnant women infected with influenza during the 1918-1919 pandemic was 27%-even higher when exposure occurred during the 3rd trimester and upwards of 50% if pneumonia supervened [23] . During the 1957-1958 Asian flu epidemic, 10% of all deaths occurred in pregnant women, and their CFR was twice as high as that of infected women who were not pregnant [24] . The most common adverse obstetrical outcomes associated with maternal pneumonias from all causes include This newly recognized coronavirus, producing a disease that has been termed COVID-19, is rapidly spreading throughout China, has crossed international borders to infect persons in neighboring countries, and humans infected by the virus are travelling via commercial airlines to other continents. It is certain that 2019-nCoV will infect women who are pregnant, leaving the question open as to whether the novel coronavirus will have a similar or different effect on them compared with SARS-CoV and MERS-CoV. In order to address the potential obstetrical outcomes of infection to both mother and infant, the present communication describes the current state of knowledge regarding the effects of other coronavirus infections in pregnancy.\\n\\nPneumonia arising from any infectious etiology is an important cause of morbidity and mortality among pregnant women. It is the most prevalent non-obstetric infectious condition that occurs during pregnancy [14] [15] [16] . In one study pneumonia was the 3rd most common cause of indirect maternal death [17] . Approximately 25 percent of pregnant women who develop pneumonia will need to be hospitalized in critical care units and require ventilatory support [16] . Although bacterial pneumonia is a serious disease when it occurs in pregnant women, even when the agent(s) are susceptible to antibiotics, viral pneumonia has even higher levels of morbidity and mortality during pregnancy [18] . As with other infectious diseases, the normal maternal physiologic changes that accompany pregnancy-including altered cell-mediated immunity [19] and changes in pulmonary function-have been hypothesized to affect both susceptibility to and clinical severity of pneumonia [20] [21] [22] . This has been evident historically during previous epidemics. The case fatality rate (CFR) for pregnant women infected with influenza during the 1918-1919 pandemic was 27%-even higher when exposure occurred during the 3rd trimester and upwards of 50% if pneumonia supervened [23] . During the 1957-1958 Asian flu epidemic, 10% of all deaths occurred in pregnant women, and their CFR was twice as high as that of infected women who were not pregnant [24] . The most common adverse obstetrical outcomes associated with maternal pneumonias from all causes include premature rupture of membranes (PROM) and preterm labor (PTL), intrauterine fetal demise (IUFD), intrauterine growth restriction (IUGR), and neonatal death [14] [15] [16] .\\n\\nThe SARS epidemic began quietly at the turn of the 21st century. In November 2002, a cook in Guangdong Province, China, died from an unidentified illness. He had worked at a restaurant in which meat from wild animals was served. On 27 November 2002 Chinese-language media and internet reports were picked up by Canada\\'s Global Public Health Intelligence Network (GPHIN) that indicated a flu-like illness was occurring in China [25, 26] . Unfortunately, the reports were not translated, and China failed to report the occurrence of this illness to the World Health Organization (WHO) until February 2003. The disease spread to other countries where it primarily infected healthcare workers. One of these was Dr. Carlo Urbani, a WHO physician investigating a patient with the new disease in Hanoi. He recognized that the pneumonia was probably caused by a new, highly infectious agent, and rapidly notified the WHO. He contracted the SARS-CoV while there, became febrile and later died after traveling to Thailand to attend a conference. On 12 March 2003, WHO issued a global alert regarding the disease that was occurring primarily among health care workers in Hanoi, Vietnam and Hong Kong. The disease continued to spread, and by 31 July 2003 there were 8422 probable cases, leading to 916 deaths in 29 countries, with the majority of cases occurring in mainland China and Hong Kong. Approximately 30% of infections occurred in healthcare workers. By the termination of the epidemic the global CFR was 11% [27] .\\n\\nAlthough there were relatively few documented cases of SARS occurring during pregnancy, several case reports and small clinical studies have described the clinical effects in pregnant women and their infants. In reviewing these reports describing pregnant women with SARS in China it is possible, and perhaps even probable, that some of the same patients were included in more than one publication. However, even if this is the case, there is no doubt that SARS coronavirus infection was found to be associated with severe maternal illness, maternal death, and spontaneous abortion [19, [28] [29] [30] [31] . Martha Anker, an expert in statistics formerly with the WHO and the University of Massachusetts, estimated that more than 100 cases of SARS-CoV infection occurred in pregnant women, which warrants closer inspection [27] .\\n\\nThe clinical outcomes among pregnant women with SARS in Hong Kong were worse than those occurring in infected women who were not pregnant [32] . Wong et al. [29] evaluated the obstetrical outcomes from a cohort of pregnant women who developed SARS in Hong Kong during the period of 1 February to 31 July 2003. Four of the 7 women (57%) that presented during the 1st trimester sustained spontaneous miscarriages, likely a result of the hypoxia that was caused by SARS-related acute respiratory distress. Among the 5 women who presented after 24 weeks gestation, 4 had preterm deliveries (80%).\\n\\nA case-control study to determine the effects of SARS on pregnancy compared 10 pregnant and 40 non-pregnant women with the infection at the Princess Margaret Hospital in Hong Kong [27, 33] . There were 3 deaths among the pregnant women with SARS (maternal mortality rate of 30%) and no deaths in the non-pregnant group of infected women (P = 0.006). Renal failure (P = 0.006) and disseminated intravascular coagulopathy (P = 0.006) developed more frequently in pregnant SARS patients when compared with the non-pregnant SARS group. Six pregnant women with SARS required admission to the intensive care unit (ICU) (60%) and 4 required endotracheal intubation (40%), compared with a 12.5% intubation rate (P = 0.065) and 17.5% ICU admission rate (P = 0.012) in the non-pregnant group.\\n\\nMaxwell et al. [32] reported 7 pregnant women infected with SARS-CoV who were followed at a designated SARS unit-2 of the 7 died (CFR of 28%), and 4 (57%) required ICU hospitalization and mechanical ventilation. In contrast, the mortality rate was less than 10% and mechanical ventilation rate less than 20% among non-pregnant, age-matched counterparts who were not infected with SARS-CoV. Two women with SARS recovered and maintained their pregnancy but had infants with IUGR. Among the live newborn infants, none had clinical or laboratory evidence for SARS-CoV infection. The new mothers who had developed SARS were advised not to breastfeed to prevent possible vertical transmission of the virus.\\n\\nZhang et al. [34] described SARS-CoV infections in 5 primagravidas from Guangzhou, China at the height of the SARS epidemic. Two of the mothers became infected in the 2nd trimester, and 3 developed infection in the 3rd trimester. Two of the pregnant women had hospital-acquired SARS infections, and the other 3 were community-acquired. All 5 pregnant women had fever and abnormal chest radiographs; 4 had cough; 4 developed hypoalbuminemia; 3 had elevated alanine aminotransferase levels (ALT), 3 had chills or rigor, 2 had decreased lymphocytes, and 2 had decreased platelets. One pregnant woman required intensive care, but all recovered and there were no maternal deaths. The 5 infants were clinically evaluated, and none had evidence of SARS.\\n\\nTwo pregnant women with SARS were reported from the United States. In a detailed case report, Robertson et al. [35] described a 36-year-old pregnant woman with an intermittent cough of approximately 10 days duration and no fever. While travelling in Hong Kong during the 2003 epidemic, she was exposed at her hotel to a person subsequently known to be infected with SARS-CoV. At 19 weeks gestation she developed fever, anorexia, headache, increasing cough, weakness, and shortness of breath. Upon returning to the United States she was hospitalized with pneumonia. Obstetrical ultrasounds revealed a low-lying placenta (placenta previa) but were otherwise normal. Following her discharge home and clinical recovery, she was found to have antibodies to SARS-CoV. She underwent cesarean section at 38 weeks gestation because of the placenta previa and a healthy baby girl was delivered [35, 36] . The placenta was interpreted as being normal. At 130 days post-maternal illness, maternal serum and whole blood, swabs from maternal nasopharynx and rectum, post-delivery placenta, umbilical cord blood, amniotic fluid, and breast milk were collected for analysis-no viral RNA was detected in specimens tested by reverse transcriptase polymerase chain reaction (RT-PCR). Antibodies to SARS-CoV were detected from maternal serum, umbilical cord blood, and breast milk by enzyme immunoassay (EIA) and indirect immunofluorescence assay. No clinical specimens (except for cord blood) were available for testing from the infant. The second case in the USA occurred in a 38-year-old woman who had travelled to Hong Kong at 7 weeks gestation where she was exposed to SARS-CoV in the same hotel as the aforementioned American woman [37] . Following her return to the United States, her husband developed the clinical onset of SARS, and 6 days later she became ill with fever, myalgia, chills, headache, coryza, and a productive cough with shortness of breath and wheezing. Following her hospitalization for SARS she recovered, serum samples taken on days 28 and 64 post-onset of illness were positive for antibodies to SARS-CoV by enzyme immunoassay and immunofluorescent assays. Her pregnancy continued and was unremarkable except for developing elevated glucose levels. A cesarean section that was performed at 36 weeks gestation due to preterm rupture of membranes and fetal distress resulted in a healthy baby boy. At the time of delivery, the mother\\'s serum samples were positive for antibodies to SARS-CoV, but samples taken of umbilical cord blood and placenta were negative. Breast milk sampled 12 and 30 days after delivery were also negative for SARS-CoV antibodies. Specimens evaluated from maternal blood, stool, and nasopharynx samples, as well as umbilical cord blood of the infant, were all negative for coronavirus RNA by RT-PCR. Neonatal stool samples obtained on days-of-life 12 and 30 were also negative for viral RNA.\\n\\nFrom Canada, Yudin et al. [38] reported a 33-year-old pregnant woman who was admitted to the hospital at 31 weeks gestation with a fever, dry cough, and abnormal chest radiograph demonstrating patchy infiltrates. She had acquired SARS from contact with an infected family member. Following a 21-day stay in the hospital, during which she did not require ventilatory support, her convalescent antibody titers were positive for coronavirus infection. She had a normal labor and delivery and her newborn girl had no evidence of infection.\\n\\nIn a study of 5 liveborn neonates who were delivered to women infected with SARS-CoV during the Hong Kong epidemic, results from multiple tests-including serial RT-PCR assays, viral culture, and paired neonatal serological titers-were negative for SARS-CoV [39] . None of the 5 neonates developed any clinical signs or symptoms of respiratory infection or compromise.\\n\\nFortunately, there were no cases of vertical transmission identified among pregnant women infected with SARS-CoV during the 2002-2003 Asian epidemic [27, 30, 31, 39, 40] , and with the exception of a small cluster of cases that recurred in late 2003, no new cases of SARS have occurred.\\n\\nIn the only reported study of the placental pathology of mothers with SARS, Ng et al. [41] reported the findings from 7 pregnant women infected with SARS-CoV. In the case of 2 women who were convalescing from SARS-CoV infection during the 1st trimester of pregnancy, the placentas were found to be normal. Three placentas were delivered from pregnancies in which the mothers had acute SARS-CoV infection-these were abnormal and demonstrated increased subchorionic and intervillous fibrin, a finding that can be associated with abnormal maternal blood flow to the placenta. In the placentas of 2 women who were convalescing from SARS-CoV infection in the 3rd trimester of pregnancy the placentas were highly abnormal. They showed extensive fetal thrombotic vasculopathy with areas of avascular chorionic villi-chronic findings of fetal vascular malperfusion. These 2 pregnancies also were complicated by oligohydramnios and had poor obstetrical outcomes-both infants had developed IUGR. It is interesting that villitis, the microscopic finding of inflammation of the chorionic villi that is the histologic hallmark of many maternal hematogenous infections that are transmitted through the placenta to the fetus, was not identified in any of these placentas.\\n\\nSimilar to other coronavirus infections, SARS-CoV is easily spread from person-to-person via respiratory droplets and secretions as well as through nosocomial contacts [42, 43] . In addition to transmission of SARS-CoV through natural aerosols from infected patients, it was found that in Hong Kong the SARS-CoV could also be transmitted by mechanical aerosols [44] . Environmental factors had an important role when it was discovered that during the Amoy Gardens housing estate outbreak as many as two-thirds of infected persons had diarrhea, SARS-CoV was excreted in their stools, and that aerosols arising from the flushing of toilets could transmit the virus [44] . Healthcare facilities were also an important source of new SARS infections during the 2002-2003 epidemic, and healthcare workers were also at high risk for acquiring the infection.\\n\\nIn order to address the safety issues for the obstetrical management and delivery of pregnant women with SARS, guidelines were prepared by the Canadian Task Force on Preventive Health Care and the Society of Obstetricians and Gynaecologists of Canada [45] . These recommendations include:\\n\\n1.\\n\\n\"All hospitals should have infection control systems in place to ensure that alerts regarding changes in exposure risk factors for SARS or other potentially serious communicable diseases are conveyed promptly to clinical units, including the labour and delivery unit.\\n\\nAt times of SARS outbreaks, all pregnant patients being assessed or admitted to the hospital should be screened for symptoms of and risk factors for SARS.\\n\\nUpon arrival in the labour triage unit, pregnant patients with suspected and probable SARS should be placed in a negative pressure isolation room with at least 6 air exchanges per hour. All labour and delivery units caring for suspected and probable SARS should have available at least one room in which patients can safely labour and deliver while in need of airborne isolation.\\n\\nIf possible, labour and delivery (including operative delivery or Caesarean section) should be managed in a designated negative pressure isolation room, by designated personnel with specialized infection control preparation and protective gear. 5.\\n\\nEither regional or general anaesthesia may be appropriate for delivery of patients with SARS.\\n\\nNeonates of mothers with SARS should be isolated in a designated unit until the infant has been well for 10 days, or until the mother\\'s period of isolation is complete. The mother should not breastfeed during this period. 7.\\n\\nA multidisciplinary team, consisting of obstetricians, nurses, pediatricians, infection control specialists, respiratory therapists, and anaesthesiologists, should be identified in each unit and be responsible for the unit organization and implementation of SARS management protocols. 8.\\n\\nStaff caring for pregnant SARS patients should not care for other pregnant patients. Staff caring for pregnant SARS patients should be actively monitored for fever and other symptoms of SARS. Such individuals should not work in the presence of any SARS symptoms within 10 days of exposure to a SARS patient. 9.\\n\\nAll health care personnel, trainees, and support staff should be trained in infection control management and containment to prevent spread of the SARS virus. 10. Regional health authorities in conjunction with hospital staff should consider designating specific facilities or health care units, including primary, secondary, or tertiary health care centers, to care for patients with SARS or similar illnesses.\"\\n\\nMiddle East respiratory syndrome (MERS) was first reported in September 2012 in Saudi Arabia, following isolation of MERS-CoV from a male patient who died months earlier from severe pneumonia and multiple organ failure [1] . In the 8 years since then, there have been more than 2494 confirmed cases of MERS resulting in upwards of 858 deaths globally [46] . While 27 countries have reported cases of MERS, approximately 80% of confirmed cases originated in Saudi Arabia [47] . To date, all known cases of MERS can be linked to travel or residence in countries along the Arabian Peninsula-that is, Bahrain; Iraq; Iran; Israel, the West Bank, and Gaza; Jordan; Kuwait; Lebanon; Oman; Qatar, Saudi Arabia; Syria; the United Arab Emirates (UAE); and Yemen [48] . The largest documented outbreak outside of this region occurred in 2015 in the Republic of Korea, in which 186 infections occurred, resulting in 38 deaths [49] . The index case in this outbreak reportedly returned from the Arabian Peninsula just prior to onset of illness [50] .\\n\\nMERS-CoV is characterized by sporadic zoonotic transmission events as well as spread between infected patients and close contacts (i.e., intra-familial transmission) [51] . Nosocomial outbreaks in health care settings-the result of poor infection control and prevention-are widely recognized as the hallmark of MERS [1] . Superspreading events have been recorded in healthcare settings in Jordan, Al Hasa, Jeddah, Abu Dhabi and South Korea [47, [52] [53] [54] [55] . Like other coronaviruses, MERS-CoV can be spread through person-to-person contact, likely via infected respiratory secretions [48] . Transmission dynamics, however, are otherwise poorly understood [1] . Bats are believed to be the natural reservoir of MERS-CoV, and dromedary camels can have the virus and have been suggested as possible intermediary hosts as well as a source of infection to humans [2, 56, 57] .\\n\\nThere are no clinical or serological reports of perinatal transmission of MERS, though vertical transmission has been reported for non-coronavirus respiratory viruses including influenza and respiratory syncytial virus (RSV) [58] . Researchers have not yet discovered ongoing transmission of MERS-CoV within communities outside of health care settings.\\n\\nThe clinical presentation of MERS varies from asymptomatic to severe pneumonia with acute respiratory distress syndrome (ARDS), septic shock, and multiple organ failure, often resulting in death. Most patients with MERS develop severe acute respiratory illness accompanied by fever, cough, and shortness of breath [50] . Progression to pneumonia is swift-usually within the first week -and at least one-third of patients also present with gastrointestinal symptoms [1] . MERS progresses much more rapidly to respiratory failure and has a higher case fatality rate than SARS [1] . Unlike SARS, however, infection with MERS-CoV is generally mild in healthy individuals but more severe in immunocompromised patients and people with underlying comorbidities [1] . The overall CFR of MERS is approximately 34.4% [46] . Most fatalities have been associated with pre-existing medical conditions like chronic lung disease, diabetes, and renal failure, as well as weakened immune systems [59] , making such individuals high risk. As a result of the immunological changes that occur during pregnancy, women who are pregnant are included in this high-risk group. Pregnant women may develop severe disease and fatal maternal and/or fetal outcomes as a result of MERS-CoV infection; however, little is known of the pathophysiology of this infection during pregnancy.\\n\\nLimited data exists on the prevalence and clinical features of MERS during pregnancy, birth, and the postnatal period. It is likely, however, that the immunological changes that normally occur in pregnancy may alter susceptibility to the MERS-CoV and the severity of clinical illness [60] . Pregnant women infected with SARS-CoV, a related coronavirus, appear to have increased morbidity and mortality when compared to non-pregnant women, suggesting that MERS-CoV could also lead to severe clinical outcomes in pregnancy. To date, however, very few pregnancy-associated cases (n = 11) have been documented, with 91% having adverse clinical outcomes.\\n\\nBetween November 2012 and February 2016, there were 1308 cases of MERS reported by the Saudi Arabia Ministry of Health (MoH). Of these, 5 patients were pregnant, according to a retrospective study by Assiri et al. [47] , and all resulted in adverse outcomes. Patient ages ranged from 27 to 34 years, with occurrence of exposure in either the 2nd or 3rd trimester. All 5 cases received intensive care. Two women died and there were 2 cases of perinatal death-1 stillbirth and 1 neonatal death shortly after emergency cesarean section. These instances of severe maternal and perinatal outcomes are consistent with other reports of MERS-CoV infection in pregnant women, as well as outcomes associated with SARS-CoV infection. The authors of the retrospectives study concede that unreported cases of MERS in pregnancy are likely due to lack of routine pregnancy testing [47] . They conclude that pregnancy testing for women of reproductive age should be considered for those who test positive for MERS-CoV, to contribute to overall understanding of pathogenesis and epidemiological risk. Additionally, 2 of the 5 patients were healthcare workers, which corresponds with existing knowledge of higher risk of exposure to MERS-CoV in healthcare settings.\\n\\nIn a separate case report of MERS occurring in pregnancy, Alserehi et al. [58] described a 33-year-old critical care nurse who became infected during the 3rd trimester in the midst of a large hospital outbreak. In the days following hospital admission, she developed respiratory failure necessitating mechanical ventilation and administration of dexamethasone as prophylaxis for the fetus. Following an emergency cesarean section at 32 weeks gestation, she was transferred to the intensive care unit (ICU) and later recovered. The preterm but otherwise healthy infant was kept in the neonatal unit for observation and later released along with his mother. In contrast to other reported cases, this patient had a successful outcome, perhaps due to the timing of MERS-CoV exposure, her young age, the use of steroids, and differences in immune response.\\n\\nAlfaraj et al. [61] described 2 cases of maternal infection with MERS-CoV at the Prince Mohammed Bin Abdulaziz Hospital (PMAH) in Saudi Arabia. Maternal infection in both cases was confirmed by nasopharyngeal swab testing by RT-PCR. One patient was a 29-year-old woman at 6 weeks gestation with no underlying medical conditions. The second patient, a 39-year-old at 24 weeks gestation, had several comorbidities, including end stage renal disease, hypertension, and hemodialysis. This woman presented to the hospital after contact with a MERS-CoV-infected person during an active outbreak. Both patients later tested negative for MERS-CoV and were subsequently discharged. The younger patient delivered a healthy, full-term infant. The status of the other delivery is unknown. Neither fetus was tested for MERS-CoV.\\n\\nAccording to Payne et al. [62] , epidemiologic investigation of the 2012 MERS outbreak in Zarqa, Jordan, revealed that a 2nd trimester stillbirth (5 months gestational age) had occurred as a result of maternal exposure to MERS-CoV. The mother experienced fever, fatigue, headache and cough, concurrently with vaginal bleeding and abdominal pain. On the 7th day of symptoms, she had a fetal death. The mother was confirmed to have antibody to MERS-CoV, and she self-reported having had unprotected contact with family members who later tested positive for the virus. This was the first documented occurrence of stillbirth during maternal infection with MERS-CoV.\\n\\nOn 24 November 2013, a 32-year-old pregnant woman in the United Arab Emirates (UAE) developed ARDS following admission to the ICU after suspected community-acquired pneumonia advanced to respiratory failure and hypotension [60] . Later that day, her baby was delivered by caesarean section and subsequent Apgar scores were within healthy range. The next day, RT-PCR evaluation revealed that the mother was positive for MERS-CoV. Despite rigorous intervention, including oral ribavirin-peginterferon-α therapy and ventilator support, the woman continued to deteriorate, developed septic shock, and died. While the outcome for this mother was fatal, Malik et al. noted that virus shedding ceased during therapy with ribavirin and peginterferon-α and radiographic evidence indicated clinical improvement before her death [58] . More research is needed to determine safety, efficacy, and dosage of these therapies in the general population but also in pregnant women. While few data exist on the effects of these treatments in pregnant humans, ribavirin is generally contraindicated during pregnancy [58] .\\n\\nOutside of the Middle East the only confirmed case of MERS in pregnancy occurred in 2015 in South Korea. Jeong et al. [49] reported that a 39-year-old patient was exposed during the 3rd trimester following contact with a patient having MERS. Despite abrupt vaginal bleeding and rupture of membranes, the patient recovered fully and delivered a healthy infant at 37 weeks and 5 days gestation. Subsequent testing of the infant\\'s blood did not detect any IgG, IgM, or IgA antibodies to MERS-CoV.\\n\\nThe mean maternal age of the 11 confirmed maternal SARS cases described above was 33.2 years, with a mean gestational age of 26.3 weeks. The source of infection in 2 of the cases was attributed to contact with family members who tested positive for MERS-CoV, unknown in 3 cases, likely due to animal exposure in 1 case, and 6 were healthcare-associated (2 of these patients were healthcare workers). Six patients required intensive care and 3 died. Of those who died, 2 were exposed to MERS-CoV in the 3rd trimester, and 1 was exposed during the 2nd trimester. The infant death rate for all 11 cases was 27%. Fetal survival did not appear to correlate with the timing of maternal infection and gestational age; however, more data are needed to draw conclusions about this relationship. According to Alfaraj et al. [61] , the CFR for the 11 infected women-also 27%-was not statistically different from the overall CFR of MERS in the general population (35%) (P = 0.75). Only 1 case resulted in both maternal and fetal death.\\n\\nSimilar to SARS in pregnancy, more research is needed to understand the pathogenesis and epidemiology of MERS in pregnancy including the relationship between the timing of maternal infection, gestational age of the fetus, the effects of comorbid factors, and the occurrence of adverse outcomes. Few studies documented the presence of MERS-CoV antibodies in the umbilical cord or neonatal blood, making it difficult to assess perinatal transmission. As such, future studies should involve the collection of samples from relevant specimens including amniotic fluid, placenta, and umbilical cord [49] .\\n\\nMERS prevention should be high priority for high-risk exposures such as healthcare workers, pregnant women and individuals working with camels, camel meat-milk processors and in abattoirs [57] . Since 2013, the Saudi Arabia MoH has recommended that pregnant women postpone travel to Saudi Arabia for the Hajj and Umrah [47] . To further reduce risk of exposure among pregnant women, additional measures such as avoiding contact with camels and sick persons-particularly in healthcare settings-are also recommended. Pregnant women who present with symptoms of pneumonia, influenza-like illness (ILI), or sepsis on the Arabian Peninsula may also benefit from MERS-CoV screening to expedite early diagnosis and improve disease management [60] .\\n\\nWhile multiple agents have been used to treat MERS, none have been tested in large clinical studies. Available data are limited to the use of combination therapies of interferon and other agents in case reports and case series [63] . A prospective or randomized study may prove difficult given the sporadic nature of MERS-CoV outbreaks.\\n\\nDue to a gap in research on the treatment of MERS in pregnancy, there are no therapeutic options currently recommended for pregnant women [58] . Therapies under development and testing may be considered inappropriate for pregnant women due to the unknown potential for teratogenic effects. For example, during the 2003 SARS outbreak, ribavirin was administered to pregnant women with severe cases of the disease, but ribavirin therapy has been documented to increase the risk of teratogenic effects in newborns [58] .\\n\\nThe Alphacoronaviruses HCoV 229E and NL63, as well as the Betacoronaviruses HKU 1 and OC43, can infect humans and cause the common cold. In order to investigate the potential maternal-fetal transmission of human coronaviruses during pregnancy, Gagneur et al. [64, 65] evaluated 3 types of maternal-infant paired specimens that included maternal vaginal and respiratory specimens that were obtained during labor, as well as gastric samples from the newborn infants. These specimens were evaluated for the presence of HCoV 229E, OC-43, NL63 and HKU 1 using RT-PCR methodology. Between the period from July 2003 to August 2005 the authors examined 159 mother-infant dyads. Human coronaviruses were identified in 12 samples (HCoV 229E: 11; HKU 1 : 1) from 7 mother-child pairs. In 3 mother-infant dyads only maternal respiratory samples were positive; in 2 other pairs all 3 of the samples tested positive for human coronavirus; in 1 case only the maternal vaginal and newborn gastric samples were positive; and in another case the maternal vaginal sample alone was positive. There were no signs of clinical infection in any of the 3 neonates that had positive gastric samples for human coronavirus.\\n\\nIt is beyond the scope of this communication to discuss the various technical challenges inherent in developing a safe and efficacious vaccine for coronavirus infections in humans. There are clearly challenges to this endeavor-protective antibodies to coronaviruses are not long-lasting, tissue damage has been reported to occur as a result of exposure to SARS-CoV, development of animal models that closely resemble human infection are limited, and the extensive time and expense necessary to perform clinical trials in humans, to name a few [66] [67] [68] .\\n\\nIt is vitally important that pregnant women be considered in the design, clinical trial, and implementation of vaccine candidates for 2019-nCoV. In examining the history of vaccine design, it is clear that the needs of pregnant women have rarely been prioritized in either the preclinical development or the clinical trial phases of production. Today, pregnant women are usually excluded from experimental trial of drugs and vaccines that do not target obstetric conditions [69] . Excluding pregnant women and their infants from participation in vaccine development and implementation undermines ethical principles of justice-fairness, equity, and maximization of benefit-and potentially places their health at risk during outbreaks and other health emergencies [69] [70] [71] .\\n\\nOn 23 January 2020 the Coalition for Epidemic Preparedness Innovations (CEPI) announced three programs to develop a vaccine against the novel Wuhan coronavirus. The Chief Executive Officer of CEPI, Richard Hatchett, said [72] :\\n\\n\"Given the rapid global spread of the nCoV-2019 virus the world needs to act quickly and in unity to tackle this disease. Our intention with this work is to leverage our work on the MERS coronavirus and rapid response platforms to speed up vaccine development.\"\\n\\nThe novel coronavirus is the first epidemic disease to emerge since the formation of CEPI in Davos in 2017. CEPI was created with the express intent to enable speedy research and development of vaccines against emerging pathogens. In May 2017, WHO released the Target Product Profile (TPP) for MERS-CoV vaccines, following the prioritization of MERS-CoV as one of eight priority pathogens for prevention of epidemics [73] . CEPI and partners aim to use existing platforms-that is, the existing \"backbone\" that can be adapted for use against new pathogens-that are currently in preclinical development for MERS-CoV vaccine candidates. Following the WHO declaration on 30 January that the current 2019-nCoV outbreak is a public health emergency of international concern (PHEIC), global health organizations and researchers will be further mobilized-bolstered by new mechanisms for action and greater resources-to stop the spread of disease.\\n\\nA critical question that must be answered at this stage-with a clear view of the potential deleterious effects of a new coronavirus in pregnancy-is will maternal immunization be a priority in research and development? As of the PHEIC declaration, 12 groups have announced that they are developing new vaccines against 2019-nCoV and seven others announced initiatives to develop new therapies [74] . Safe testing of experimental vaccines in a pregnant population is difficult and, as a result, vaccines are not typically developed with pregnant women in mind. To date, very few clinical trials for vaccines have proactively included pregnant women [75] , and the exclusion of pregnant and lactating women from receiving the rVSV-ZEBOV vaccine through 3 Ebola virus epidemics serves as a recent example [69] [70] [71] . Given the potential severity in pregnancy, as demonstrated by this review of maternal infections of SARS and MERS, women who are pregnant should be considered a priority population in all efforts to prepare for and prevent infection by novel coronaviruses.\\n\\nOn 5 February 2020 it was reported by multiple media outlets that a newborn infant delivered during the epidemic in Wuhan had tested positive for 2019-nCoV at the Wuhan Children\\'s Hospital in Hubei Province 30 hours following its birth. According to the official Xinhua news agency, the infant was delivered on 2 February to a mother who had tested positive for the virus. Reports have stated that the infant had stable vital signs, no fever or cough, but had shortness of breath together with abnormal chest radiographs and abnormalities of liver function [76] [77] [78] . Dr. Zeng Lingkong, Chief Physician at the Neonatal Medicine Department of the hospital, said [78] , \"This reminds us to pay attention to mother-to-child being a possible route of coronavirus transmission\"\\n\\nThe hospital also provided information about a previous case of a baby that had been delivered on 13 January 2020. Following its birth, the infant\\'s nanny was diagnosed with 2019-nCoV, and the mother was diagnosed days later [76] . On 29 January the baby began to develop symptoms. According to Dr. Zeng Lingkong [76] , \"Whether it was the baby\\'s nanny who passed the virus to the mother who passed it to the baby, we cannot be sure at the moment. But we can confirm that the baby was in close contact with patients infected with the new coronavirus, which says newborns can also be infected\"\\n\\nIn considering whether these and future cases of neonatal infection are acquired prior to delivery, it is important to remember that newborn infants can acquire an infection in other ways beyond intrauterine maternal-fetal transmission. In some cases, viral infection can be acquired when the infant passes through the birth canal during a vaginal delivery or through post-partum breast feeding, although these mechanisms would be highly unusual for a respiratory virus. Neonatal infection from respiratory viruses can occur after delivery through such mechanisms as inhalation of the agent through aerosols produced by coughing from the mother, relatives or healthcare workers or other sources in the hospital environment. Based upon past experience with pregnant women who developed MERS and SARS, and realizing that the numbers are limited, there has never been confirmed intrauterine coronavirus transmission from mother to fetus. Discussing the most recent baby to be diagnosed with the 2019-nCoV infection, Dr. Stephen Morse, an epidemiologist at the Mailman School of Public Health at Columbia University stated [77] , \"It\\'s more likely that the baby contracted the virus from the hospital environment, the same way healthcare workers get infected by the patients they treat,\" \"It\\'s quite possible that the baby picked it up very conventionally-by inhaling virus droplets that came from the mother coughing.\"\\n\\nAnd according to Dr. Paul Hunter, Professor of Medicine at the University of East Anglia [79] , \"As far as I am aware there is currently no evidence that the novel coronavirus can be transmitted in the womb. When a baby is born vaginally it is exposed to the mother\\'s gut microbiome, therefore if a baby does get infected with coronavirus a few days after birth we currently cannot tell if the baby was infected in the womb or during birth.\"\\n\\nThere is limited knowledge regarding coronavirus infections that occur during pregnancy-what is known has, for the most part, been the result of epidemics resulting from two different diseases, SARS and MERS. These previous experiences with coronavirus infections in pregnancy indicates that these agents are capable of causing adverse clinical outcomes including life-threatening maternal disease that in some cases requires hospitalization, intensive care and ventilatory support. Both of these coronaviruses can result in maternal death in a small but significant number of cases, but the specific risk factors for a fatal outcome during pregnancy have not been clarified. Coronaviruses can also result in adverse outcomes for the fetus and infant including intrauterine growth restriction, preterm delivery, admission to the ICU, spontaneous abortion and perinatal death. Unlike some viral infections, notably Ebola virus [70] and Zika virus [80] , the likelihood of intrauterine maternal-fetal transmission of coronaviruses is low-there have been no documented cases of vertical transmission occurring with either SARS or MERS. It remains to be seen during the current Wuhan 2019-nCoV epidemic how this newly-emergent coronavirus affects pregnant women and their infants, as well as which factors may modulate obstetrical disease and outcomes including the timing of maternal coronavirus exposure by gestational age, the effects of medications or other treatment regimens, differences in host immune responses, occurrence of coexisting medical and obstetrical conditions, and other covariables. However, pregnant women should be considered to be at high risk for developing severe infection during this current outbreak of 2019-nCoV. Additional clinical research on the treatment of SARS, MERS, and the new coronavirus 2019-nCoV is necessary if we are to understand the potential risks and benefits of novel therapies and new vaccines in pregnancy. This research will be critical in improving the care, and even saving the lives, of pregnant women in the current as well as future outbreaks.', 'document_id': 2551}]}, {'paragraphs': [{'qas': [{'question': 'How many  COVID-19 cases were confirmed on the Diamond Princess cruise ship?', 'id': 1187, 'answers': [{'text': '199', 'answer_start': 451}], 'is_impossible': False}, {'question': 'What was the time period of peak infection of COVID-19 on  the Diamond Princess cruise ship?', 'id': 1188, 'answers': [{'text': '2 to 4 February 2020,', 'answer_start': 710}], 'is_impossible': False}, {'question': 'With the intervention of movement restrictions starting on 5th February 2020, what were the confirmed cases for COVID-19, were limited  to?', 'id': 1190, 'answers': [{'text': '102 and 47 cases, respectively.', 'answer_start': 1194}], 'is_impossible': False}, {'question': 'Who was the first COVID-19 identified case patient on the Diamond Princess cruise ship?', 'id': 1191, 'answers': [{'text': 'case was diagnosed on 1 February, the ship was requested to remain in the ocean near Yokohama from 3 February onwards.', 'answer_start': 1795}], 'is_impossible': False}, {'question': 'When was  the first  passenger patient on the Diamond Princess cruise ship diagnosed with COVID-19?', 'id': 1192, 'answers': [{'text': 'he case was diagnosed on 1 February', 'answer_start': 1792}], 'is_impossible': False}, {'question': 'How many COVID-19 cases were confirmed on the Diamond Princess cruise ship?', 'id': 1193, 'answers': [{'text': 'Out of a total of 3711 persons (consisting of 2666 passengers and 1045 crew members), 199 symptomatic cases have been diagnosed on board as of 24 February, and additional asymptomatic infections and symptomatic cases after disembarkation have also been reported', 'answer_start': 2039}], 'is_impossible': False}, {'question': 'What is the estimated mean incubation period for COVID-19 infection on the Diamond Princess cruise ship?', 'id': 1194, 'answers': [{'text': 'about 5.0 days', 'answer_start': 2739}], 'is_impossible': False}, {'question': 'What was the effect of movement restriction policy on the Diamond Princess cruise ship started on 5th February 2020.', 'id': 1196, 'answers': [{'text': 'highly successful in greatly reducing the number of secondary transmissions on board.', 'answer_start': 7231}], 'is_impossible': False}, {'question': 'What would  have the number of confirmed cases on the Diamond Princess cruise ship,  without a movement restriction starting on the 5th February 2020?', 'id': 1189, 'answers': [{'text': ' the cumulative incidence with and without close contact would have been as large as 1373 (95% CI: 570, 2176) and 766 (95% CI: 587, 946) cases,', 'answer_start': 1008}], 'is_impossible': False}], 'context': 'Backcalculating the Incidence of Infection with COVID-19 on the Diamond Princess\\n\\nhttps://doi.org/10.3390/jcm9030657\\n\\nSHA: 0938d2fb07611897abf38cea727ddbeea77b73d9\\n\\nAuthors: Nishiura, Hiroshi\\nDate: 2020\\nDOI: 10.3390/jcm9030657\\nLicense: cc-by\\n\\nAbstract: To understand the time-dependent risk of infection on a cruise ship, the Diamond Princess, I estimated the incidence of infection with novel coronavirus (COVID-19). The epidemic curve of a total of 199 confirmed cases was drawn, classifying individuals into passengers with and without close contact and crew members. A backcalculation method was employed to estimate the incidence of infection. The peak time of infection was seen for the time period from 2 to 4 February 2020, and the incidence has abruptly declined afterwards. The estimated number of new infections among passengers without close contact was very small from 5 February on which a movement restriction policy was imposed. Without the intervention from 5 February, it was predicted that the cumulative incidence with and without close contact would have been as large as 1373 (95% CI: 570, 2176) and 766 (95% CI: 587, 946) cases, respectively, while these were kept to be 102 and 47 cases, respectively. Based on an analysis of illness onset data on board, the risk of infection among passengers without close contact was considered to be very limited. Movement restriction greatly reduced the number of infections from 5 February onwards.\\n\\nText: An outbreak of novel coronavirus disease (COVID-19) has occurred on a cruise ship, the Diamond Princess [1] . The primary case remains unknown, but the index case, defined as the first identified case, is a passenger who started coughing from 19 January 2020 on board, disembarking the ship in Hong Kong on 25 January. As the case was diagnosed on 1 February, the ship was requested to remain in the ocean near Yokohama from 3 February onwards. Subsequently, the movement of all passengers was restricted on board from 5 February, for a matter of 14 days of quarantine. Out of a total of 3711 persons (consisting of 2666 passengers and 1045 crew members), 199 symptomatic cases have been diagnosed on board as of 24 February, and additional asymptomatic infections and symptomatic cases after disembarkation have also been reported.\\n\\nOne of the critical issues in infectious disease epidemiology is that the time of infection event is seldom directly observable. For this reason, the time of infection needs to be statistically estimated, employing a backcalculation method [2] . Using a sophisticated statistical model with doubly intervalcensored likelihood and right truncation with an exponential growth of cases, the mean incubation period has been estimated to be about 5.0 days [3] . To understand the time-dependent risk of infection throughout the course of outbreak and estimate the effectiveness of the quarantine measure from 5 to 19 February 2020, I aimed to estimate the incidence of infection with COVID-19 and also predict the likely number of infections prevented by the quarantine measure.\\n\\nI analyzed the epidemic curve, ct, on day t, illustrated by the number of confirmed cases by the date of illness onset. The confirmatory diagnosis was made, using the reverse transcriptase polymerase chain reaction (RT-PCR). The date of illness onset was defined as the first date of fever. In addition to the date of illness onset, cases were classified by contact history inside the cabin and also by the type of membership, i.e., crew or passenger. Close contact was defined as having at least one cabinmate who was confirmed by RT-PCR.\\n\\nWe estimate the number of cases by time of infection, it. Using the probability mass function of the incubation period of length s, fs, the incidence of infection is known to satisfy\\n\\nwhere E(.) represents the expected value. As for fs, it is known that the mean and standard deviation are 5.0 and 3.0 days, respectively, best fitted by lognormal distribution [3] . Employing a step function, the incidence of infection was statistically estimated via a maximum likelihood method. The estimation was implemented independently by the history of contact and type of membership.\\n\\nRegarding the real-time forecasting, we employed the so-called Richards model, an analogue to the generalized logistic model [4, 5] :\\n\\nwhere is the cumulative incidence on day t, Z is the cumulative incidence at the end of the outbreak, s is the parameter that governs the flexibility of the logistic curve, a is the early growth rate of cases and ti is the inflection point of the cumulative incidence curve. Assuming that the cumulative incidence is Gaussian distributed, four unknown parameters were estimated. The Richards model was fitted to two different datasets, i.e., (i) the dataset of the entire course of the epidemic and (ii) the dataset by 4 February 2020. The latter dataset corresponds to the time period without any impact of movement restriction that was in place from 5 February onwards. Figure 1 shows the epidemic curve by contact history and type of membership. The highest incidence of illness onset was observed on 7 February. The epidemic curve in a latter half period was dominated by crew members whose movement was not strictly controlled due to the need to continue service on the ship. The second dominating group was passengers with close contact history. The last illness onset date on board of a passenger without close contact was on 14 February. Estimating the incidence of infection, the peak incidence was identified for the period from 2 to 4 February among passengers both with and without close contact (Figure 2 ). The incidence of infection abruptly dropped after 5 February, the date of movement restriction. Among passengers without close contact, the incidence was estimated to be zero, except for 8-10 February 2020, during which 0.98 persons (95% confidence intervals (CI): 0, 7.74) per day were estimated to have been infected. The epidemic peak among crew members was seen for the period from 8 to 10 February 2020. Figure 3 compares the cumulative incidence with and without movement restriction policy from 5 February. In the presence of intervention, the cumulative incidence among passengers with and without close contact and crew members were 102, 47 and 48 cases, respectively, as of 24 February 2020. These were well realized by the Richards model. Without intervention from 5 February onwards, it was predicted that the cumulative incidence with and without close contact would have been 1373 (95% CI: 570, 2176) and 766 (95% CI: 587, 946) cases, respectively. \\n\\nA large outbreak of COVID-19 occurred on a cruise ship. Estimating the incidence, the peak time of infection was shown to have been from 2 to 4 February, and the incidence abruptly declined afterwards. The estimated number of new infections among passengers without close contact was very small from 5 February, on which the movement restriction policy was imposed, and at most there was, on average, one case of infection per day from 8 to 10 February. Other than continued exposure among crew members, the estimated incidence in this study indicates that the movement restriction policy from 5 February 2020 was highly successful in greatly reducing the number of secondary transmissions on board. Based on an analysis of illness onset data on board (and before the disembarkation of a large number of passengers), the risk of infection among passengers without close contact was considered to be very limited Among disembarked passengers, symptomatic cases have started to be reported on the ground in and outside of Japan. In particular, cases arising from passengers without close contact indicate a possible pathway of infection via mechanisms that were not covered by the abovementioned analysis that relied on symptomatic cases. Although the transmission via direct human-to-human contact was prevented by movement restrictions, the role of other modes of transmission, e.g., environmental and asymptomatic transmissions, should be further explored. \\n\\nThe author declares no conflict of interest.', 'document_id': 2555}]}, {'paragraphs': [{'qas': [{'question': 'When was the a cluster of pneumonia cases were first reported ?', 'id': 1235, 'answers': [{'text': '31 December 2019,', 'answer_start': 1124}], 'is_impossible': False}, {'question': 'What is the number of inbound passengers from China?', 'id': 1236, 'answers': [{'text': ' 63.1 million per year in 2017 ', 'answer_start': 3413}], 'is_impossible': False}, {'question': 'What percent of inbound passengers from China were from Wuhan?', 'id': 1237, 'answers': [{'text': '2.1%', 'answer_start': 3466}], 'is_impossible': False}], 'context': 'The Extent of Transmission of Novel Coronavirus in Wuhan, China, 2020\\n\\nhttps://doi.org/10.3390/jcm9020330\\n\\nSHA: 919c524f19f79213e6f81aa38502c70287d273dc\\n\\nAuthors: Nishiura, Hiroshi; Jung, Sung-mok; Linton, Natalie M.; Kinoshita, Ryo; Yang, Yichi; Hayashi, Katsuma; Kobayashi, Tetsuro; Yuan, Baoyin; Akhmetzhanov, Andrei R.\\nDate: 2020\\nDOI: 10.3390/jcm9020330\\nLicense: cc-by\\n\\nAbstract: A cluster of pneumonia cases linked to a novel coronavirus (2019-nCoV) was reported by China in late December 2019. Reported case incidence has now reached the hundreds, but this is likely an underestimate. As of 24 January 2020, with reports of thirteen exportation events, we estimate the cumulative incidence in China at 5502 cases (95% confidence interval: 3027, 9057). The most plausible number of infections is in the order of thousands, rather than hundreds, and there is a strong indication that untraced exposures other than the one in the epidemiologically linked seafood market in Wuhan have occurred.\\n\\nText: Since the announcement of a cluster of pneumonia cases of unknown etiology in Wuhan, Hubei Province, China, was made on 31 December 2019, many rapid virological, clinical, and epidemiological research responses have taken place [1, 2] . The causative agent of the pneumonia is suggested to be a novel coronavirus (2019-nCoV) of the same lineage (but genetically distinct) from the coronavirus causing severe acute respiratory syndrome (SARS) [1] . Cases in the initial cluster reported a common exposure-a seafood market in Wuhan where wild animals were served at a restaurant-indicating that a point-source zoonotic (animal-to-human) route was likely the main mode of transmission for those cases [2] .\\n\\nAlthough early reports from Wuhan [3] stated that (i) there were only tens of cases in the cluster and (ii) no human-to-human transmission was directly observed, the scientific community was alert to the possibility that the novel coronavirus would spread to other geographic locations-including other countries-via direct human-to-human transmission. In early January, the outbreak began to escalate rapidly with hundreds of cases now confirmed along with the presence of a few household clusters [4] [5] [6] [7] .\\n\\nAs of 24 January 2020, the cumulative incidence in China is 830 cases, of which 549 cases were diagnosed in Hubei, 26 in Beijing, 20 in Shanghai, and 53 in Guangdong. Additionally, twenty-six deaths have been linked to the outbreak [6, 8] , and thirteen cases were exported to Japan, Singapore, South Korea, Taiwan, Thailand, Vietnam and the United States as of 22 January 2020. Considering that enhanced surveillance has been underway in these importing countries, case ascertainment has been perhaps better in exported case data.\\n\\nUsing a spatial back-calculation method and analyzing exported cases, we estimate the cumulative incidence of 2019-nCoV cases in China in real time, allowing us to update and discuss the extent of transmission at the source. Table 1 shows the incidence of exported cases by date of hospitalization and report. Due to the initial difficulty of diagnosis in the absence of established primer for polymerase chain reaction testing, the time lag between hospitalization and reporting was longer for early cases compared with that of more recent cases. Among the seven locations reporting importation, the total volume of inbound passengers from China was m = 63.1 million per year in 2017 [9] , of which 100q = 2.1% were from Wuhan [10] , a home of n = 19.0 million people as the catchment population of Wuhan airport. Two other locations with confirmed cases, i.e., Macau and Hong Kong, were excluded from the analysis, because it is commutable by land transporation and the first case in Hong Kong was indeed not via airtravel. As we already know from elsewhere [11] [12] [13] , given the observed cumulative count of c exported cases, we have a balance equation of the cumulative risk of infection:\\n\\nwhere T is the sum of incubation and infectious periods, and here is assumed to be 3.2 and 9.3 days [14] , respectively, assuming that these periods are similar to those of other coronaviruses, and thus, T = 12.5 days. The estimated incidence in China is then given bypn. With an ad-hoc assumption that the data are generated following the binomial sampling process among travelers from Wuhan, the cumulative incidence is then estimated using a maximum likelihood method. Table 1 also shows the estimated incidence in China. The first exportation event in Thailand suggests 423 cases with the upper confidence limit of 1863 cases. The estimated cumulative incidence has grown as additional cases have been reported. As of 24 January 2020, with reports of thirteen exportation events, the cumulative incidence in China is estimated at 5502 cases (95% confidence interval: 3027, 9057).\\n\\nOur latest estimate is comparable to a preliminary report posted by a research group at Imperial College London (ICL) on their own homepage on 22 January 2020 [26] that estimated the incidence based on three importation events at 4000 cases (95% CI: 1000, 9700). Possible reasons for the slight difference include (i) the number of travelers in the previous study was derived from airline passenger data [27] and (ii) the assumed length of T was different. Two other estimates have also been published: a preliminary study by a Northeastern University group estimated 1250 cases (95% CI: 350, 3000) as of 17 January 2020 [28] and a University of Hong Kong group estimated 1343 cases (95% CI: 547, 3446) as of 17 January 2020 [29] . The former study from the United States assumes that the catchment area population is 10 million (we use 11.1 million).\\n\\nThe number of reported 2019-nCoV infections continues to grow as surveillance and detection methods improve. Our estimate and others [26, 28, 29] agree that the actual number of cases is likely in the order of thousands, rather than hundreds, and there is a strong indication that untraced exposures other than that of the originally linked seafood market in Wuhan have occurred. Such exposures are expected to include human-to-human transmission, but the levels of transmissibility have yet to be quantified. It is still plausible that a substantial number of human infections arose from animal-to-human exposures, such as was the case during the first outbreak of highly pathogenic influenza (H7N9) in China, 2013, and the human-to-human transmissibility has yet to be quantified in an explicit manner.\\n\\nDespite initially restricting what information on the outbreak was shared publicly, the Chinese government has begun to respectfully provide updates on the situation on a daily basis. This encourages the real-time release of information by means of regularly updated situation reports, including epidemiological information with dates of exposure, illness onset, and hospitalization among cases.\\n\\nFor researchers to be able to contribute to control efforts by improving situation awareness via an explicit risk assessment, it is crucial that detailed epidemiological data are posted to a public domain in real-time. Such datasets should include not only a deidentified line list of cases but also updates on the infection status of traced contacts. Information on exposure period and illness onset can assist with the estimation of important natural history parameters such as the incubation period. It is critical for the public health community and the public at large to understand more about the process of case ascertainment, including the current case definition and reporting system mechanisms. \\n\\nThe authors declare no conflicts of interest.', 'document_id': 2554}]}, {'paragraphs': [{'qas': [{'question': 'What is the mechanism of action for rupintrivir?', 'id': 314, 'answers': [{'text': 'prevents cleavage of viral proteins required for replication', 'answer_start': 9113}], 'is_impossible': False}, {'question': 'Has rupintrivir been shown to reduce the symptoms of a rhinoviral infection?', 'id': 315, 'answers': [{'text': 'in studies of natural infection, it did not significantly affect viral loads or symptom severity', 'answer_start': 9308}], 'is_impossible': False}, {'question': 'What is the primary etiology of acute respiratory infection?', 'id': 308, 'answers': [{'text': 'viral', 'answer_start': 2812}], 'is_impossible': False}, {'question': 'What is RSV?', 'id': 309, 'answers': [{'text': 'respiratory syncytial virus', 'answer_start': 3119}], 'is_impossible': False}, {'question': 'What virus is most commonly associated with acute respiratory infections?', 'id': 310, 'answers': [{'text': 'human rhinovirus', 'answer_start': 3933}], 'is_impossible': False}, {'question': 'What viruses are most frequently associated with acute respiratory infections?', 'id': 311, 'answers': [{'text': 'HRVs, coronaviruses (CoV), influenza, respiratory syncytial virus (RSV) and parainfluenza viruses (PIV)', 'answer_start': 4174}], 'is_impossible': False}, {'question': 'What are the clinical characteristics of asthma?', 'id': 313, 'answers': [{'text': 'chronic airway inflammation, and a history of respiratory symptoms such as wheeze, shortness of breath, chest tightness and cough', 'answer_start': 6732}], 'is_impossible': False}, {'question': 'What risk factor was associated with hospitalization and death during the 2009 H1N1 influence pandemic?', 'id': 317, 'answers': [{'text': 'morbid obesity', 'answer_start': 11203}], 'is_impossible': False}, {'question': 'How many surface proteins are on the H1N1 influenza virus?', 'id': 319, 'answers': [{'text': 'two', 'answer_start': 12711}], 'is_impossible': False}, {'question': 'What are the 2 surface proteins on the H1N1 influenza virus?', 'id': 320, 'answers': [{'text': 'haemagglutinin (HA) and the neuraminidase (NA) ', 'answer_start': 12737}], 'is_impossible': False}, {'question': 'What pharmaceutical targets the NA glycoprotein of the H1N1 influenza virus?', 'id': 322, 'answers': [{'text': 'oseltamivir', 'answer_start': 14280}], 'is_impossible': False}, {'question': \"What genetic mutation decreases a person's susceptibility to the H1N1 influenza virus?\", 'id': 323, 'answers': [{'text': 'H275Y', 'answer_start': 14705}], 'is_impossible': False}, {'question': 'Why is ribavirin treatment limited?', 'id': 324, 'answers': [{'text': 'difficulty with aerosol delivery, cost and potential harm to healthcare workers', 'answer_start': 16533}], 'is_impossible': False}], 'context': 'The human viral challenge model: accelerating the evaluation of respiratory antivirals, vaccines and novel diagnostics\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC6013893/\\n\\nSHA: f13c88733ea45be9e923a282dfd42f8c277c187c\\n\\nAuthors: Lambkin-Williams, Rob; Noulin, Nicolas; Mann, Alex; Catchpole, Andrew; Gilbert, Anthony S.\\nDate: 2018-06-22\\nDOI: 10.1186/s12931-018-0784-1\\nLicense: cc-by\\n\\nAbstract: The Human Viral Challenge (HVC) model has, for many decades, helped in the understanding of respiratory viruses and their role in disease pathogenesis. In a controlled setting using small numbers of volunteers removed from community exposure to other infections, this experimental model enables proof of concept work to be undertaken on novel therapeutics, including vaccines, immunomodulators and antivirals, as well as new diagnostics. Crucially, unlike conventional phase 1 studies, challenge studies include evaluable efficacy endpoints that then guide decisions on how to optimise subsequent field studies, as recommended by the FDA and thus licensing studies that follow. Such a strategy optimises the benefit of the studies and identifies possible threats early on, minimising the risk to subsequent volunteers but also maximising the benefit of scarce resources available to the research group investing in the research. Inspired by the principles of the 3Rs (Replacement, Reduction and Refinement) now commonly applied in the preclinical phase, HVC studies allow refinement and reduction of the subsequent development phase, accelerating progress towards further statistically powered phase 2b studies. The breadth of data generated from challenge studies allows for exploration of a wide range of variables and endpoints that can then be taken through to pivotal phase 3 studies. We describe the disease burden for acute respiratory viral infections for which current conventional development strategies have failed to produce therapeutics that meet clinical need. The Authors describe the HVC model’s utility in increasing scientific understanding and in progressing promising therapeutics through development. The contribution of the model to the elucidation of the virus-host interaction, both regarding viral pathogenicity and the body’s immunological response is discussed, along with its utility to assist in the development of novel diagnostics. Future applications of the model are also explored. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1186/s12931-018-0784-1) contains supplementary material, which is available to authorized users.\\n\\nText: Acute respiratory infections (ARIs) manifest as Upper (URI) or Lower (LRI) respiratory tract infections and may move between the two compartments; ARIs represent the most common infectious diseases and are predominantly of viral aetiology. The global burden of ARI is substantial with significant morbidity and mortality occurring in children, the elderly and immunocompromised [1] .\\n\\nIn the UK alone during the period 2014-2015, respiratory disease caused an estimated 15,800 excess winter deaths [2] . In the USA, influenza and respiratory syncytial virus (RSV) cause substantial mortality especially among people aged 65 and older [3] .\\n\\nHowever, although deaths in the industrialised world are widely reported, developing countries feel the burden particularly; out of an estimated 1.9 million child deaths from ARIs in 2000, 70% of those deaths occurred in Africa and south-east Asia [4] . The Millennium Summit at the United Nations in 2000 led to the setting up of the Millennium Development Goals.\\n\\nA study reported the progress made in meeting those goals in 40 developing countries; it concluded that the prevalence of ARI was 13%, health expenditure and per capita gross domestic product is directly associated with the prevalence of the disease [5] .\\n\\nViral heterogeneity associated with ARIs is well established [6] . In the past, human rhinovirus (HRV) has been identified as the virus most frequently associated with respiratory illness with 30-50% of infections annually on average, and up to 80% of upper respiratory infections during the autumn outbreaks [7] . After HRVs, coronaviruses (CoV), influenza, respiratory syncytial virus (RSV) and parainfluenza viruses (PIV) are the next most frequent.\\n\\nMore recently an evaluation of illness in 6,266 children under ten years of age in Australia, South East Asia and Latin America emphasised both the viral heterogeneity and the impact of ARI. Of the 2,421 children who experienced 3,717 individual influenza-like Illness (ILI) episodes, rhinovirus/enterovirus was most prevalent (41. 5%). Influenza followed this (15.8%), adenovirus (ADV) (9.8%), PIV and RSV (both 9.7%), CoV (5.6%), human metapneumovirus (HMPV) (5.5%) and human bocavirus (HBoV) (2.0%). The percentage of children missing school or childcare was between 21.4% for HBoV and 52.1% for influenza [8] .\\n\\nWe have compared the data from the two reports one from 2003 [7] and the other in 2017 [8] and found that the reports, despite being separated by 14 years, were similar, with the single exception of HBoV, discovered in 2005 (Table 1) , which we discuss later.\\n\\nFeng et al. [9] described in detail the distribution of ARIs causing hospitalisation by age group: they observed that RSV was predominantly observed in the young and elderly, and influenza although significant in the young was noticeably more predominant in the elderly. Interestingly they observed that co-detection of viruses tended to occur more commonly in the younger age groups, particularly those under the age of five.\\n\\nRhinovirus (the \"common\" cold) HRV infections, often considered trivial can significantly contribute to missed days from work and school, though infections are typically self-limiting [7] . HRV infections throughout the year and in many cases, manifest with symptoms such as nasal congestion, rhinorrhoea, sneezing, sore throat, and cough. HRV is known to be the primary cause of ARI and a severe contributing factor in exacerbations of atopic disease, e.g., asthma as well other conditions such as chronic obstructive pulmonary disease (COPD) [10] [11] [12] [13] .\\n\\nHRV infections are associated with significant economic implications as well as being an important contributor to sinusitis, otitis media, bronchitis and primary pneumonia [14] [15] [16] . HRV is a considerable cause of morbidity in specific at-risk groups such as infants, the elderly, immunocompromised, and, as already mentioned, chronic respiratory diseases such as asthma, COPD and cystic fibrosis. At present, HRV is considered the number one cause of asthma exacerbations [15] [16] [17] [18] [19] .\\n\\nAsthma is a complex disease, characterised by chronic airway inflammation, and a history of respiratory symptoms such as wheeze, shortness of breath, chest tightness and cough. Over time these symptoms can vary in their intensity [20] . Each year over 300 million people worldwide are affected by asthma: approximately 250,000 people die as a result. Many deaths are due to suboptimal long-term medical care and delay in obtaining help during severe exacerbations of the disease [21] . Treatments to prevent worsening of symptoms and other therapies for mild to moderate asthma that avert relapse, i.e., the symptoms worsen again when the treatment stops, are significant unmet medical needs.\\n\\nThe human challenge model has been used to investigate the viral pathogenicity [22] [23] [24] [25] [26] and recent publications on the asthma challenge model have focused on how the asthmatic host responds to HRV infection. Work is ongoing as to susceptibility to viral induced asthma worsening [27, 28] innate immune dysregulation [29] and induction of innate, and type 2 responses in nasal and bronchial epithelial secretions [30] . The pathogenesis of rhinoviral infection, along with other ARIs, in exacerbations of airway disease, has been investigated extensively. Impaired host responses to virus infection, a better understanding of the mechanisms of abnormal immune responses and the potential to develop novel therapeutic targets for virus-induced exacerbations have all used the HVC model [12, [31] [32] [33] [34] .\\n\\nDespite previous research work on multiple small molecule antivirals, such as pleconaril which have been tested using both the experimental challenge model and field studies [35] [36] [37] , there is currently no licensed treatment for HRV infections Other compounds have been tested against HRV, such as Vapendavir (BTA798) which prevented the release of viral RNA into the target cell and demonstrated a reduction in peak viral load in the HVC model [38] . A subsequent study in asthmatics was completed and although not published the compound did have a limited effect [39] .\\n\\nPirodavir an intranasal capsid-binding molecule reached phase 3 clinical trials for HRV prevention and treatment in the 1990s. Although the compound decreased viral replication and shedding, it failed to show a significant reduction in the duration or severity of symptoms [40, 41] .\\n\\nA Protease inhibitor, rupintrivir thats prevents cleavage of viral proteins required for replication was tested in an HRV challenge trial. Rupintrivir was well tolerated and reduced viral loads and respiratory symptoms [36] . However, in studies of natural infection, it did not significantly affect viral loads or symptom severity [42] .\\n\\nTreatments such as zinc-containing products are now widely discredited as demonstrated by the withdrawal of a Cochrane report and JAMA editorial [43] [44] [45] .\\n\\nCurrent treatment of HRV infections primarily consists of over-the-counter (OTC) medicines to manage symptoms. There is also no licensed vaccine, and while there has been some progress on developing multivalent vaccines [46] , development in this area is hampered by the sheer number of serotypes that need to be covered (at present over 160). Despite HRV being associated with up to 50% of adult asthma exacerbations and up to 80% of childhood exacerbations, there are no HRV-specific asthma therapies [34] .\\n\\nAs we better understand the interaction between the virus and the host, new therapies such as the monoclonal antibodies (anti-IgE [omalizumab] and anti-IL-5 [mepolizumab]) along with small molecules carefully targeting specific immune signalling pathways, HRV-specific prophylactic treatment may become practical [47] [48] [49] [50] .\\n\\nIn order to prevent exacerbations, the design of new therapeutics could potentially improve efficacy by both directly acting to inhibit viral replication and alleviate the symptoms of asthma and COPD [51] .\\n\\nInfluenza virus is a well-known human pathogen and can cause severe morbidity and mortality, particularly in older patients, those with co-morbidities and in the immunocompromised. In 2009, the first pandemic virus of the 21 st century hospitalised 195,000 to 403,000 in the US alone resulting in 8,870 to 18,300 deaths by mid-2010 [52] . A World Health Organization (WHO) global pooled analysis of 70,000 laboratory-confirmed hospitalised H1N1 pandemic patients from 19 countries revealed that of the 9,700 patients admitted to intensive care units, 2,500 died, and that morbid obesity might be a risk factor for hospitalisation and/or death [52] . Obesity was confirmed as a factor associated with a higher likelihood of admission to hospital in influenzainfected patients [53] .\\n\\nThe 2009 pandemic was considered mild. However, the classic W shaped age distribution curve of infection for a pandemic virus was observed. That is high mortality in the very young and the old, but an additional spike in death amongst the \"young and healthy\". The pandemic, as did previous outbreaks, occurred in successive waves, but despite national policies favouring the use of antiviral drugs, few patients received these before admission to hospital, and many were given antibiotics [54] . The lack of real, or perceived, \"real world\" efficacy of currently available antivirals leads to the overuse of antibiotics and the subsequent problems that may arise [55] [56] [57] .\\n\\nThe yearly seasonal morbidity and mortality of influenza results in hospitalisation and death mainly among the high-risk groups. Each year epidemics of seasonal influenza are estimated to result in about 3 to 5 million cases of severe illness, and about 290,000 to 650,000 deaths worldwide [58] .\\n\\nIn first world / industrialised countries, most deaths associated with influenza occur among people age 65 or older [59] . Clinics and hospitals, in many countries, can be overwhelmed during peak illness periods, and there can be substantial economic cost [60] .\\n\\nThe virus itself has been well characterised, and the two surface proteins, the haemagglutinin (HA) and the neuraminidase (NA) are important in both vaccine and antiviral development [61] .\\n\\nThe effects of seasonal influenza epidemics in developing countries are not fully known, but research estimates that 99% of deaths in children under five years of age with influenza-related lower respiratory tract infections are found in developing countries [59, 62] .\\n\\nCurrently, vaccines and antivirals exist for the prevention and treatment of influenza, but both have limitations in efficacy due to the rapid evolution of the virus as it mutates on a yearly basis and the sudden unexpected emergence of pandemic influenza strains.\\n\\nThe effectiveness of recent annual influenza vaccines (to date mostly based on the HA, and rarely the NA surface glycoproteins) has languished between 37% and 70% over successive influenza seasons. In particular, the failure of the vaccine across the winter season of 2014-2015, where the overall adjusted effectiveness was 23% [95% confidence interval 14, 31] [63] is memorable. In a mismatched year, the mortality rate is increased in the most at-risk populations [64, 65] . The problem of ensuring that the seasonal vaccine is correctly matched to the upcoming circulating strain highlights the need for rapid development of inter-seasonal/universal vaccines and also the need for a way of testing their efficiency rapidly and accurately before the lengthy and expensive mass production is engaged which takes many months [66, 67] .\\n\\nAntiviral drugs exist of which currently the NA inhibitor oseltamivir is most commonly used. This is active against all known NA subtypes of influenza, and one would, therefore, assume against all influenza strains. They may have decreasing effect with the emergence of resistant influenza strains in which NA protein changes preventing efficient oseltamivir binding and thus its ability to inhibit the essential activity of the viral NA. For example, one genetic mutation known as \\'H275Y\\'a substitution of histidine for tyrosine at NA position 275 -confers an evolutionary advantage to the virus including the 2009 H1N1 influenza [68] . During the 2013-2014 influenza season, 59 (1.2%) of 1,811 influenza A(H1N1) pdm09 virus isolates in 20 of 50 US states had the H275Y oseltamivir resistance substitution. No isolates were resistant to zanamivir [69] . Although animal studies have demonstrated limited transmission of mutant viruses [70, 71] , it is thought that the rise of oseltamivir resistance may be due to community transmission [72, 73] rather than the H275Y mutation becoming fixed in the viral genome.\\n\\nAsystematic systematic review and meta-analysis of published data from 2000 onwards concluded that most RSV-associated child deaths occur particularly in preterm infants and in infants up to 1-year of age [62, 74] . An effective maternal RSV vaccine or monoclonal antibody could have a substantial effect on disease burden in this age group [75] .\\n\\nThe RSV-specific monoclonal antibody palivizumab is approved for prevention of serious LRI caused by RSV in susceptible infants. Economic benefit in a UK health setting has not been shown due to the high cost and lack of benefit on serious outcomes [76] . A single-centre cohort study of 22 infants showed no difference in treatment outcomes for patients receiving palivizumab when compared to patients only receiving \"standard of care\" treatment [77] . Despite the lack of evidence for clinical benefit, post-licensure data supports the use of palivizumab for reducing RSV-associated hospitalisations in premature infants under 33 weeks and in children with chronic lung and heart diseases [78] . Importantly, palivizumab resistant mutant virus has rarely been isolated in clinical specimens [79] .\\n\\nThe RSV treatment ribavirin is limited due to difficulty with aerosol delivery, cost and potential harm to healthcare workers, despite off-label treatment of immunocompromised patients being reasonably successful. In the immunocompromised, therapy with a concomitant immunoglobulin or palivizumab has had mixed results, probably due to the difficulty of knowing when to initiate treatment [80] .\\n\\nDespite the call for the accelerated development of prevention and treatment strategies for an effective RSV vaccine for children [81] , research has stalled for decades since the death in the 1960s of two subjects in a clinical study. These subjects were infected with a communityacquired RSV infection after receiving the US National Institutes for Health (NIH\\'s) formalin-inactivated, alumprecipitated RSV candidate vaccine.\\n\\nIn contrast to influenza for which vaccines to date have shown themselves to be moderately effective but in need of improvement, RSV vaccines require substantially more research. There is currently no licensed vaccine for RSV; the most advanced candidate vaccine recently failed to show efficacy in a field study [82] . Effective treatments are urgently required.\\n\\nRSV is, even amongst healthcare professionals, considered a childhood disease and other confounders have obscured the understanding of the consequences of RSV in adults.\\n\\nRSV is poorly understood as a disease in the elderly [83] , and while the morbidity and mortality in children are of importance, it has been clearly shown that RSV has a comparable health burden to influenza in the elderly [84] .\\n\\nAs an example, a recent study was conducted on adult (≥18 years) patients admitted to an emergency department with suspected ARI during 2013-2015 (N = 3743). Multiplex PCR was used to diagnose the cause of the respiratory infection. Eighty-seven patients were identified with RSV. A comparator group with influenza (n=312) was utilised. Based on a 20-day all-cause mortality endpoint, adult patients were less likely to be diagnosed with RSV than with flu (2.3 vs 8.3%, respectively), also they were older, often diagnosed with pneumonia, COPD, hypoxemia, and bacterial co-infection. RSV infection in the elderly was significantly associated with a greater risk of death than seasonal influenza, adjusted for potential confounders and comorbidities. [85] \\n\\nThe clinical significance of viral/bacterial co-infections has long been a controversial topic. While severe bacterial pneumonia following influenza infection has been well described, associations are less clear among infections caused by viruses common in young children; secondary infections due to other viruses are less well understood and has been reviewed by others [86] . Although assessing the overall contribution of bacteria to disease severity is complicated by the presence of many confounding factors in clinical studies, understanding the role of viral/bacterial co-infections in defining the outcome of paediatric ARI may potentially reveal novel treatment and prevention strategies, improving patient outcomes [33, [86] [87] [88] [89] [90] [91] [92] [93] [94] [95] .\\n\\nA recent (2017) publication considered the role of bacterial colonisation with Streptococcus pneumoniae, Haemophilus influenzae and Moraxella catarrhalis during symptomatic and asymptomatic viral upper respiratory infection in the nasopharynx of 4 to 7-year-old children during URI and when well. Using a multiplex PCR, virus was detected in about 80% of upper respiratory tract infections (URIs) in children and is also detectable in the nasopharynx of 30% of asymptomatic children. All three bacteria \"levels\" were higher during acute URI visits compared to asymptomatic surveillance visits by the children. Of note, however, is that even during asymptomatic follow-up visits, if the virus was present, all bacteria were detected at higher levels [96] .\\n\\nIt is worth noting that the presence of confounding infections, can mask the importance of the primary aetiology. Taylor et al. [8] report the incidence of HBoV following its identification in 2005 from the respiratory tract samples of children, as an important respiratory pathogen in children. However, the role of this virus on its own as a pathogen of significance was initially unclear, co-infection with other viruses or bacteria was common and confounding.\\n\\nMoesker et al. [97] studied whether HBoV alone could cause acute respiratory infections in children. Using Next Generation Sequencing (NGS), they were able to exclude co-infections amongst those admitted to intensive care unit and studied HBoV viral loads. Of the 990 children who tested positive for a respiratory virus by RT-PCR, HBoV and RSV were detected in 178 and 366 of the children respectively. Forty-nine HBoV-positive patients and 72 RSV-positive patients were admitted to the intensive care. Seven HBoV-infected cases with severe ARI had no other co-infection (7/49, 14%). Importantly, these children did not have another detectable virus as determined by highly sensitive NGS. Also, they had much higher HBoV loads than other patients positive for HBoV, i.e., those with a co-infection. Although small, this study provides strong support that HBoV can cause serious ARI in children with no viral and bacterial co-infections.\\n\\nThe history of the human viral challenge model Since Sir Edward Jenner performed the first documented HVC study with smallpox on the 14 th of May 1796 the usefulness of such studies has been apparent [98] . More than a century later, Sir Christopher Andrews returned from the US in 1931 he had observed the use of chimpanzees in the study of influenza. The funding for similar work in the UK was insufficient, and therefore Sir Christopher enrolled students from St Bartholomew\\'s Hospital in London. He explained the next best thing would be a \"Bart\\'s\" student as \"they were cheaper than chimpanzees\". Over 100 students immediately enrolled, but continued their studies and were not isolated in the same way the chimpanzees had been in the USA [99] . Unfortunately the investigators believed that the symptoms observed may not have been due to the challenge virus, but other respiratory infections acquired in the community, thus confounding the studies. A year later the UK\\'s Medical Research Council (MRC) terminated the work.\\n\\nAfter the conclusion of World War II, the withdrawal of the US troops from the UK left the American Red Cross \\'Harvard Hospital\\' Field Unit on Salisbury plain. The hospital became the Common Cold Unit (CCU) led by Dr David Tyrell, from 1946, volunteers were inoculated by instilling small quantities of the virus into their noses [100] . The CCU housed healthy volunteers in relative isolation from other people, thereby reducing the risk of contact with community-acquired sources of infection or from them passing on the virus to members of the public. The unit was eventually closed in 1989; during four decades of research, it attracted 20,000 volunteers. Its research contributed to a better understanding of respiratory viruses, viral lifecycle, possible vaccines [101] as well as the first licensed antiinfluenza compound amantadine [102] .\\n\\nThe use of healthy volunteers in the HVC model provided, and still offers, a unique opportunity to describe the viral lifecycle. Investigators know with certainty the time of infection, nasal virus shedding can be measured, symptoms recorded prospectively, and participants are selected with low pre-existing immunity to the challenge virus to ensure a statistically significant infection rate with a small number of volunteers. Thus, such studies can maximise the safety and efficacy data obtained while minimising the risk to study volunteers and limited research funding.\\n\\nAlthough serum IgG, for influenza virus, was traditionally measured via the HAI assay, as the entry criteria for volunteers into studies, micro neutralisation assays are used for RSV and HRV. Other work does suggest screening for antibodies to the NA influenza surface protein should be considered [103] or T-cell responses to internal proteins [104] should be considered.\\n\\nAfter the closure of the CCU experimental infection studies continued in the USA using small motels and hotels replacing the huts on Salisbury Plain. These studies contributed to the significant development of the new NA inhibitors during the 1990s, including the inhaled drug zanamivir and the orally available drug oseltamivir [105] [106] [107] [108] [109] [110] [111] [112] [113] [114] .\\n\\nStudies however also continued in the UK, specifically the University of Southampton who performed important work in atopic volunteers, demonstrating they had more severe colds when experimentally challenged with rhinovirus, than non-atopic controls [115] .\\n\\nThe experimental A/Texas H1N1 influenza virus that was used successfully during the 1990s was implicated in the development of myocarditis in an experimentally infected subject, although a causal link was never demonstrated [116] . However, this incident halted work in the USA for a substantial period.\\n\\nMost, if not all, challenge viruses are manufactured according to Good Manufacturing Practice (GMP) standard. Although controlled nasal inoculation differs from naturally occurring infectionin which exposure to variable quantities of the virus may occur at various mucosal sites -the developed HVC model used in challenge studies mimics natural disease as far as possible [25, 117, 118] .\\n\\nWe have described the production of a new GMP stock of virus using an HRV-16 isolate from an 18-year-old experimentally infected healthy female volunteer, provided by colleagues from University of Virginia Children\\'s Hospital, USA. Importantly, the clinical sample was provided with the appropriate medical history and consent of the donor. We manufactured this new HRV-16 stock by minimal passage in a WI-38 cell line, to reduce the risk of mutations during the Good Manufacturing Practice process. Having first subjected the stock to rigorous adventitious agent testing and determining the virus suitability for human use, we conducted an initial \"safety and pathogenicity\" clinical study in adult volunteers in a dedicated clinical quarantine facility in London [118] .\\n\\nOur group started HVC studies in the UK in 2001, and since then we have conducted multiple studies with over 2,500 volunteers inoculated with influenza, respiratory syncytial virus (RSV) or human rhinovirus (HRV), and provided numerous proofs of concept [119] [120] [121] .\\n\\nThe human viral challenge model: shortening the drug development pathway for ARIs Influenza, RSV and HRV infection have similar symptomatology, but this differs in severity and predominance of upper, lower or systemic symptoms as has been described by the Center for Disease Control [122] . However, it is not easy to diagnose between the different aetiologies of ARIs, and better diagnostics are needed [123] .\\n\\nSymptoms are common to each infection and manifest on a gradient. Generally, but far from always, influenza infection is more likely to result in a patient feeling so unwell as to take to their bed and have a fever, than RSV, an HRV, CoV or other common cold virus infection, during which daily life is usually less impacted.\\n\\nA variety of animal models exist to research respiratory viruses such as influenza [124] [125] [126] , RSV [127] [128] [129] [130] [131] [132] [133] [134] [135] [136] [137] , HRV [22, [138] [139] [140] . No single animal offers a platform for all respiratory viruses that infect humans, and different animal models exist for the same virus, which can give different, often conflicting results.\\n\\nIn addition, the principles of the 3Rs (Replacement, Reduction and Refinement) were developed over 50 years ago to provide guidance and ensure humane animal research. Over time they have become national and international legislation/regulations. The policies of organisations that fund or conduct animal research include these principles as part of the condition of funding [141] .\\n\\nThe shared symptomatology of respiratory viruses requires a single standard research platform that can be used to evaluate respiratory disease pathogenesis and the efficacy of candidate therapeutics. The use of a dedicated, purpose-built 24 en-suite bedroom isolation facility in which carefully screened volunteers can be safely inoculated with challenge viruses and intensively monitored may help reduce the use of animals while providing a single consistent research platform with standardised evaluable endpoints for respiratory virus research. Also, we have used a standardised diary card across our studies, which allows for comparison of the symptoms that each virus causes and the efficacy of the therapeutic being tested. We have included a copy of the diary card in the Additional file 1.\\n\\nIt is difficult to evaluate the efficacy of a specific antiviral therapeutic \"in the field\" due to the presence of circulating community co-infections of differing microbial aetiology. The HVC model allows the opportunity to study a virus in isolation. HVC studies and field studies are complementary research stratagems necessary for the development of effective ARI therapeutics.\\n\\nIn contemporary HVC trials, (Fig. 1 ) healthy volunteers are administered an investigational therapeutic either before (prophylaxis trials) or after (treatment trials) inoculation with the specific challenge strain of the virus. The viruses used in the HVC model are not attenuated and produce symptoms consistent with clinically observed ARI [25, 117, 118] . Each virus is propagated under GMP conditions, with a minimal number of passages from the isolates to the challenge stocks [118, 142] . The few mutations that occur within the virus are rapidly selected out due to a genetic bottleneck, with the consequence that the virus in the human host is considered wild-type [143] . The similarity between virus recovered from the inoculated host and the originator reference virus strain provides assurance that the model disease process is closely aligned with the reference virus strain and is not altered nor attenuated.\\n\\nThere are limited licensed therapeutic options against respiratory viruses, highlighting a significant unmet medical need. A model such as the HVC allows the rapid evaluation of novel therapeutics. The model shortens both preclinical and early clinical development phases by providing a better understanding of the host and pathogen\\'s initial interaction and has the potential to make the necessary vaccines and medicines more rapidly available than traditional development approaches otherwise might.\\n\\nShortening the traditional development pathway through the early use of a Proof of Concept (PoC) study that incorporates the HVC model (Fig. 2) provides essential evaluable endpoints. Unlike conventional phase 1 studies which rarely include any assessment of efficacy, almost all HVC studies include evaluable efficacy endpoints such as reduction in AUC viral load (mainly recovered from upper respiratory tract samples such as nasal wash or nasopharyngeal swab), volunteer self-reported symptoms, peak symptom score, total symptom score amongst others. Small numbers of subjectsoften in the order of 30-45 per treatment group-are typically included in these rapid to execute short duration studies. The resulting safety and pharmacokinetic (PK) and pharmacodynamic (PD) data in controlled conditions, guide decisions on whether or not to progress to field studies, providing a most valuable set of data immediately after, or even as part of, the conventional phase 1 safety study.\\n\\nThe HVC model also opens a different development route alongside traditional phase 1 allowing rapid progress to statistically powered phase 2b studies that will generate the efficacy data needed to support licensing, while still providing suitable safety data. The FDA guidance on developing influenza therapeutics [144] states that challenge trials cannot take the place of efficacy (phase 2) trials. The guidance states; \"…Challenge trials can provide useful exposure-response and safety information, as well as an opportunity to demonstrate pharmacological antiviral activity in humans under controlled conditions outside the influenza season. Specifically, data from challenge trials can contribute to dose selection for phase 2b and phase 3 trials, and provide the opportunity to explore the effects of different times of drug initiation relative to virus exposure...\".\\n\\nChallenge trial refinements are closing the gap between the experimental infection model and the natural infection setting. The HVC study duration of several weeks is shorter than a field-based phase 2 study that waits for a natural outbreak of the virus and the duration of which can be several months/years. These studies save Fig. 1 The Human Viral Challenge Model. The study typically consists of inputs, such as the volunteers, their selection criteria, isolation in quarantine and exposure to a GMP virus. There are two treatment options; a vaccination/prophylaxis with an antiviral or b treatment with an antiviral. Outputs from the study, summarised on the right, such as virus symptoms, virus shedding etc. X is the number of days before virus exposure vaccination may occur. Y is the number of days post virus exposure that a volunteer may be followed for development time when the transition between phases is fully optimised.\\n\\nImportantly, unlike traditional phase 1b/phase 2 studies, HVC studies are not dependent on a natural outbreak of infection, which can occur at random, and for which the exact time of infection may not be apparent. They provide evaluable endpoints, comparative PD and PK data, along with additional biomarker data on product performance in humans. It must, however, be stated that most often such studies enrol otherwise healthy young adults which imply that the outcome of the infection in the placebo group may be seen as mild to moderate, to some extent. The safety of volunteers has to remain the priority of investigators.\\n\\nThe HRV/HVC model can be a potent tool, not just to study HRV infection and disease, but also to investigate the mechanisms of exacerbation in patients with chronic respiratory disease and to conduct efficacy studies for new therapies.\\n\\nHuman challenge studies with HRV have been shown to produce infection in over 90% of serologically susceptible subjects and result in a clinical syndrome that is comparable to that reported with natural colds [145, 146] . Symptoms usually appear within 24 hours and peak at 48-72 hours after inoculation. Virus shedding follows a pattern similar to that of their symptoms. In recent times, several hundred inoculations of adult subjects have been reported and have established this as a safe and effective method in which to study HRV-related disease in both healthy and asthmatic subjects [145] .\\n\\nThese studies have provided a knowledge base to further develop the HRV experimental model and provide a controlled and useful tool to develop new therapies for the disease areas associated with HRV infection. New treatments for asthma and COPD are urgently needed, and small animal models of asthma are poorly predictive of efficacy. Most drugs that are effective in these animal models are not found to be effective in later stages of development in humans. Models that more closely follow clinical features of human asthma and COPD are needed [32, [147] [148] [149] [150] [151] ].\\n\\nWe have already described current influenza antiviral drugs that can shorten disease and reduce the severity of symptoms if taken early enough after infection, and their prophylactic use can decrease the risk of infection; their utility has been debated however [152] .\\n\\nThe two main classes of currently effective antiinfluenza drugs are the NA inhibitors, such as zanamivir (Relenza™), oseltamivir (Tamiflu™), peramivir (Rapivab™) [153] and M2 inhibitors, although drug resistance makes this class unusable [154] .\\n\\nThe HVC model has recently been used extensively to evaluate new classes of antiviral compounds against influenza, including those such as experimental monoclonal antibodies targeting epitopes within the highly conserved and exposed part of the M2 viral surface Fig. 2 The role of the HVC model in the clinical development pathway. Short duration proof of concept studies, which incorporate the HVC model, typically include small numbers of subjects. The resulting safety and, particularly, efficacy data can more accurately guide decisions on whether to expose a larger number of subjects to promising candidate therapeutics in field studies than conventional phase 1 safety data alone otherwise might protein [155, 156] the conserved stalk of the HA [157] and small molecule antiviral drugs that target the viral polymerase, e.g. favipiravir [158] .\\n\\nThe HVC model allows for the rapid evaluation of novel therapeutic compounds which may be difficult to evaluate in the field, due to the nature of \"at risk\" groups, e.g. paediatrics. Specifically, and given the described historical experience with RSV vaccines, it is important that benefit can first be demonstrated in a healthy population.\\n\\nIn the past, unlike influenza and HRV, the HVC model has not been routinely used with RSV. Recently, however, there are several antiviral therapeutics that have reached an advanced stage of development using the model.\\n\\nWe had for some time wished to restart the HVC/RSV studies at the University of London, the two significant challenges that had stalled antiviral development for RSV presented a considerable research need. In association with the DeVincenzo lab at the University of Tenessee and the biotech company Alnylam, we set about designing possibly the first HVC/RSV study.\\n\\nAlnylam pioneered the use of RNA interference (RNAi) which is a natural mechanism that regulates protein expression and is mediated by small interfering RNAs (siRNA). Working with both groups, we manufactured an RSV Type A virus to GMP standard and titrated it in 35 human volunteers who we divided into five groups, each which was intranasally inoculated with increasing titre (3.0-5.4 log plaque-forming units/person) of the challenge virus. Intranasally. Overall, in this new model, 77% of volunteers consistently shed virus. Infection rate, viral loads, disease severity, and safety were similar between cohorts and were unrelated to the quantity of RSV received.\\n\\nSymptoms began soon after initial viral detection, peaked in severity near when viral load peaked and subsided as viral loads slowly declined. We concluded that regardless of the titre administered once infections were established the viral load drove illness. We saw no adverse events linked to the virus [25] . Using this new model we conducted an HVC clinical study and demonstrated for the first time that an RNAi had significant antiviral activity against human RSV infection -this established the first-ever proof of concept for an RNAi therapeutic in humans adults [159] .\\n\\nAn editorial in the American Journal of Respiratory and Critical Care Medicine, described the utility of the HVC/RSV model saying; \"This model permits the relatively quick and efficient study of new therapeutics in humans and assists in making critical decisions whether to advance a product into costly human trials in populations at highest risk for disease; children, elderly or immunocompromised patients. This constitutes a major and welcome advance in the field of RSV.\" [81] It is notable that two compounds that have distinct modes of action have recently been evaluated using the HVC model.\\n\\nFirst-in-class nucleoside analogue ALS-008176, the efficacy of which was first demonstrated in the HVC model, is currently under evaluation in hospitalised infants [160, 161] . The HVC trial was of randomised, double-blind design, and studied healthy adults inoculated with RSV Memphis 37B [25] . A total of 62 participants received ALS-008176 or placebo for five days after confirmation of RSV infection by PCR (tested twice daily post inoculation). The primary endpoint was the area under the curve (AUC) for viral load post infection. More rapid RSV clearance and a greater reduction in viral load, with accompanying improvements in the severity of clinical disease, were demonstrated in the groups treated with ALS-008176 when compared to the placebo group [160] . Intensive sampling allowed for any potential mutations associated with resistance to be rapidly identified. No such resistant mutations were observed [160] .\\n\\nAn RSV-entry inhibitor, GS-5806, a second molecule, first-in-[its]-class was also evaluated. Among the 54 subjects that received active treatment, lower viral load, lower total mucus weight and a lower AUC symptom score were highly significant when compared to placebo [119] . Based on these challenge study data, this therapeutic is now also progressing into potentially pivotal field studies [162] .\\n\\nAn essential element of design in both studies was the timing of the first administration of therapeutic postexperimental virus inoculation; the timing was dependent on the detection of virus in nasal wash samples post inoculation of challenge virus by a rapid PCR assay [163] , rather than at an arbitrary time point. Subsequently the therapeutic was administered every 12 hours. Careful dose timing, at a clinically relevant point of detection, contributed to the positive outcomes of both studies. It is also believed that by using this \"triggered dosing\" model, it better mimicked what would happen in a clinical setting as symptoms are known to appear soon after the onset of virus shedding.\\n\\nThe HVC model is not limited to novel antiviral compounds but is also important for the evaluation of novel vaccines. Influenza vaccine performance in recent years raises questions about the most appropriate correlates of protection.\\n\\nUnlike field studies, HVC studies are useful tools for assessing the correlates of protection, vital for vaccine development [103, 104, 164] . Specifically, the importance of the humoral and cellular responses has been highlighted along with the pre-existing T-cell immunity for other respiratory viruses [104] .\\n\\nA recent publication describes the use of the HVC model to demonstrate the efficacy of a novel intranasal proteosome-adjuvanted trivalent inactivated influenza vaccine (P-TIV). In two separate studies, selected subjects who were naïve to A/Panama/2007/1999 (H3N2) virus, were dosed via nasal spray with one of three regimens of P-TIV or placebo. Together, the studies evaluated one or two doses, 15 μg or 30 μg, either once only or twice 14 days apart (1 x 30 μg, 2 x 30 μg, 2 x 15 μg) and subjects were challenged with A/Panama/2007/1999 (H3N2) virus. Immune responses to the vaccine antigens were measured by haemagglutination inhibition (HAI) assay and nasal wash secretory IgA (sIgA) antibodies. Vaccine efficacy was observed ranging from 58% to 82%, comparable to traditional vaccines. The studies also demonstrate that protection against illness associated with evidence of influenza infection significantly correlated with pre-challenge HAI (serum IgG) titres (p = 0.0003) and mucosal IgA (p≤0.0001) individually, and HAI (p = 0.028) and sIgA (p = 0.0014) together. HAI and sIgA levels were inversely related to rates of illness. These studies demonstrated the efficacy of this novel intranasal vaccine and answered some important questions concerning true correlates of protection against influenza infection which will help drive future vaccine design. As well as achieving its primary aims, it revealed valuable insights into the correlates of protection and will, we hope, aid future vaccine design [164] .\\n\\nAn inter-seasonal or universal influenza vaccine is desperately needed; it will save many lives, whether in those unexpected years when the recommended composition is not matched, or when a pandemic occurs, as it did in 2009. The significance of the 1918 pandemic [165, 166] makes it very clear; up to 100 million people died. A universal vaccine is one that can be prepared for the unexpected, a virus that occurs due to the reassortment of viral genes from different host species.\\n\\nThe HVC model is possibly the only way to initially test such a universal vaccine.\\n\\nA universal candidate could generate an immune response against the highly conserved virus ion channel protein M2, [167] [168] [169] [170] , although no vaccine has been shown to be effective in this regard; monoclonal antibodies alone have, the HVC model showed their efficacy [156] . Alternatively, a vaccine may target the conserved stalk of the HA protein [104, 171] , or elicit a T-cell response to the internal proteins [172] [173] [174] [175] . All are possibilities that have been and can be explored more efficiently using the HVC model.\\n\\nAlthough HVC studies provide PoC, researchers, as we have shown, have employed regulatory design standards typical of later phase efficacy studies.\\n\\nWith the development of molecular technology, it is now possible to refine the statistical analysis by stratifying the subjects based on their immune profile. For instance, it is now possible to assess whether a subject is carrying other known respiratory pathogens (bacteria, viruses etc.) and if there is a possible impact on the set of results from the volunteer. Subjects often consent for further analysis of their samples, which allows a valuable biobank of samples to be built for further testing. Moving forward, such samples will allow the use of the HVC model to understand further what happens when a virus infects a person.\\n\\nIt is worth noting that the HVC model is not limited to PoC work on potential therapeutic agents; it is also extensively being used for research purposes, upon which improved treatments for respiratory viruses can be built. In recent years it has been used to demonstrate \"gene switching signatures\" that could form part of a diagnostic that would reveal infected individuals before they become symptomatic, in the early stages of infection; this could be vitally important in the event of a pandemic [176, 177] .\\n\\nAlso, the HVC model has been used to allow a comparison of the relative disease dynamics of different respiratory viruses [24] and to provide a better understanding of the interaction of the virus and the human host [26, 178, 179] .\\n\\nThe HVC model has increased our understanding of the viral life cycle and disease pathogenesis in a tightly controlled setting using small numbers of volunteers. Each volunteer is isolated from each other, and the wider community, ensuring that the disease under consideration is the only one of interest.\\n\\nThe applicability of the virus used to challenge volunteers in the HVC model to a virus that an individual might become exposed to in the \"real world\" is significant. Whether challenge trials are feasible is dependent on the availability of adequately safety-tested challenge virus strains that are of know providence.\\n\\nThe HVC model provides certain knowledge of the character of the virus; the exact time point of infection; measurability of nasal virus shedding; prospective recording of symptoms and pre-selection of participants for viral challenge who are sero-suitable. This ensures that a statistically significant rate of infection is achieved with the minimal number of volunteers, thus optimising the risk-benefit ratio that supports the determination of therapeutic efficacy.\\n\\nCrucial to HVC study design is the timing of administration of the first dose of product under investigation to determine optimal effectiveness, not just in the challenge study itself, but in both later stage clinical studies and final clinical use.\\n\\nThe HVC model is an important tool in drug development, in particular with regard to acute respiratory infections. It can accelerate the development of therapeutics that address multiple unmet medical needs. It helps in the understanding of the relationship between a virus and its human host and offers the potential for the development of early-stage diagnostics. It contributes towards identifying new areas for therapeutic intervention. Possibly, and arguably, more importantly, it can ensure that scarce medical resources are directed towards later stage clinical development in an evidence-based manner, and promising therapeutic opportunities are prioritised.\\n\\nA careful and targeted study design process is a crucial step towards the successful outcome of a challenge trial, because almost all parameters, can be either controlled or at least known (either pre-or post-hoc). Furthermore, results from such trials can be used to make commercial decisions and can lead to major publications, expanding the collective understanding of the scientific community.\\n\\nSamples from such experiments are of immense value to researchers for the understanding of host interaction mechanisms and the development and validation of therapeutics. Utilisation of consistently collected historical data from HVC studies informs the accurate design and powering of subsequent studies.\\n\\nHVC studies have been successful in providing proof of concept for DNA vaccines, T-cell vaccines, intranasal vaccines, monoclonal antibodies and small molecules against a range of important respiratory viruses.\\n\\nIt is also encouraging to see that the HVC model is now expanding into further patient populations such as the elderly, asthmatics and those with other conditions such as chronic obstructive pulmonary disease.\\n\\nAn expanding archive of data from preceding studies is an invaluable asset to assist in the selection of volunteers, decide on appropriate endpoints and refine future field study designs.\\n\\nThis allows for safer, statistically sound and more rapidly delivered research. drafted the initial version of this manuscript with author RLW. hVIVO was responsible for overall management of this work and verified the accuracy of the data presented. Other non-author contributors included Ben Murdoch of hVivo who provided figures. hVivo would like to thank the volunteers without whose altruism the human viral challenge studies conducted at hVivo over many years would not have been possible.\\n\\nThe work, including professional medical writing services for preparing this manuscript, was wholly funded by hVivo Services Limited, the employer of all authors.\\n\\nAuthor RLW conceived the strategy for this paper. Author RLW and professional medical writer Samina Hamilton drafted the article (see \\'Acknowledgements\\'). Authors RLW and AG critically reviewed the complete article for important intellectual content. Authors RLW and AG had full authority over the choice of the journal and approved the final article. Author RLW is a guarantor for the paper and takes overall responsibility for this publication. All other authors contributed to the writing and review of this manuscript.\\n\\nEthics approval and consent to participate All clinical studies were described received appropriate Ethical Committee approval, including informed consent of volunteers.\\n\\nAll authors declare that they are employees of hVivo and as such, have provided or do provide ethical professional clinical research services to academic, biotechnology, or pharmaceutical clients. A patent (patent applications 14/366602 (US) 12813946.6 (EP) application is in progress regarding specific utilisation of the HVC model. This does not alter the authors\\' adherence to International Society for Medical Publication Professionals (ISMPP) \\'Good Publication Practice for Communicating Company-Sponsored Medical Research: GPP3\\'.', 'document_id': 1740}]}, {'paragraphs': [{'qas': [{'question': 'To what the lack of exposure report could be attributed?', 'id': 2974, 'answers': [{'text': ' lack of awareness or recall bias', 'answer_start': 4503}], 'is_impossible': False}, {'question': 'Why is the determination of asymptomatic or pre-symptomatic transmission, an urgent priority?', 'id': 2975, 'answers': [{'text': ' it has direct implications for public health and hospital infection control', 'answer_start': 4767}], 'is_impossible': False}, {'question': 'When does the infectivity of SARS-COV peak?', 'id': 2976, 'answers': [{'text': '10 days after illness onset [7] , consistent with the peak in viral load at around that time', 'answer_start': 5028}], 'is_impossible': False}, {'question': 'How can the 2019-nCov spread?', 'id': 2967, 'answers': [{'text': ' from person to person,', 'answer_start': 396}], 'is_impossible': False}, {'question': 'What is the estimate of the basic reproduction number?', 'id': 2968, 'answers': [{'text': '2.2', 'answer_start': 519}], 'is_impossible': False}, {'question': 'What is assumed for the mean serial interval?', 'id': 2969, 'answers': [{'text': ' 7.5 days', 'answer_start': 558}], 'is_impossible': False}, {'question': 'What  would a shorter mean serial interval mean?', 'id': 2970, 'answers': [{'text': 'slightly lower basic reproductive number.', 'answer_start': 698}], 'is_impossible': False}, {'question': 'What should have reduced the basic reproduction number in January?', 'id': 2971, 'answers': [{'text': 'Control measures and changes in population behaviou', 'answer_start': 740}], 'is_impossible': False}, {'question': 'What  are the delays between infection to illness and illness to laboratory confirmatiion?', 'id': 2972, 'answers': [{'text': ' around 5–6 days, with an upper limit of around 11-14 days [2,5], and delays from illness onset to laboratory confirmation added a further 10 days on average', 'answer_start': 1180}], 'is_impossible': False}, {'question': 'What is the estimate of number of infections in Wuhan on 25 January 2020?', 'id': 2973, 'answers': [{'text': '75,000', 'answer_start': 1976}], 'is_impossible': False}, {'question': 'when is viral shedding the highest?', 'id': 2977, 'answers': [{'text': 'on the day of illness onse', 'answer_start': 5283}], 'is_impossible': False}, {'question': 'How does the transmission of the respiratory virus happen?', 'id': 2978, 'answers': [{'text': 'through large respiratory droplets, ', 'answer_start': 5617}], 'is_impossible': False}, {'question': 'How do some respiratory viruses spread?', 'id': 2979, 'answers': [{'text': ' through fine particle aerosols', 'answer_start': 5692}], 'is_impossible': False}, {'question': 'What can also play  a role?', 'id': 2980, 'answers': [{'text': 'indirect transmission via fomites', 'answer_start': 5735}], 'is_impossible': False}, {'question': 'What can play a role  in the infection of gastrointestinal tract?', 'id': 2981, 'answers': [{'text': ' faecal-oral transmission ', 'answer_start': 5868}], 'is_impossible': False}, {'question': 'What was attributed to the spread of SARS-COV at Amoy Gardens?', 'id': 2982, 'answers': [{'text': 'through pressure differentials between contaminated effluent pipes, bathroom floor drains and flushing toilets ', 'answer_start': 6078}], 'is_impossible': False}, {'question': 'How were the first human infections identified?', 'id': 2983, 'answers': [{'text': 'through a surveillance system for pneumonia of unknown aetiology', 'answer_start': 6720}], 'is_impossible': False}, {'question': 'What do mild clinical presentations of 2019-nCOV indicate?', 'id': 2984, 'answers': [{'text': 'that there could be many more mild infections than severe infections', 'answer_start': 7154}], 'is_impossible': False}, {'question': 'Why is important to  determine the spectrum of clinical manifestations of 2019-nCoV infections?', 'id': 2985, 'answers': [{'text': 'because it determines the strength of public health response required.', 'answer_start': 7346}], 'is_impossible': False}, {'question': 'What, beyond the assessment of severity, is important?', 'id': 2986, 'answers': [{'text': ' to determine high risk groups.', 'answer_start': 7896}], 'is_impossible': False}, {'question': 'For whom would the infections be more severe?', 'id': 2987, 'answers': [{'text': 'older adults, obese individuals or those with underlying medical conditions,', 'answer_start': 7970}], 'is_impossible': False}], 'context': \"Epidemiological research priorities for public health control of the ongoing global novel coronavirus (2019-nCoV) outbreak\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC7029449/\\n\\nSHA: 90de2d957e1960b948b8c38c9877f9eca983f9eb\\n\\nAuthors: Cowling, Benjamin J; Leung, Gabriel M\\nDate: 2020-02-13\\nDOI: 10.2807/1560-7917.es.2020.25.6.2000110\\nLicense: cc-by\\n\\nAbstract: Infections with 2019-nCoV can spread from person to person, and in the earliest phase of the outbreak the basic reproductive number was estimated to be around 2.2, assuming a mean serial interval of 7.5 days [2]. The serial interval was not precisely estimated, and a potentially shorter mean serial interval would have corresponded to a slightly lower basic reproductive number. Control measures and changes in population behaviour later in January should have reduced the effective reproductive number. However, it is too early to estimate whether the effective reproductive number has been reduced to below the critical threshold of 1 because cases currently being detected and reported would have mostly been infected in mid- to late-January. Average delays between infection and illness onset have been estimated at around 5–6 days, with an upper limit of around 11-14 days [2,5], and delays from illness onset to laboratory confirmation added a further 10 days on average [2].\\n\\nText: It is now 6 weeks since Chinese health authorities announced the discovery of a novel coronavirus (2019-nCoV) [1] causing a cluster of pneumonia cases in Wuhan, the major transport hub of central China. The earliest human infections had occurred by early December 2019, and a large wet market in central Wuhan was linked to most, but not all, of the initial cases [2] . While evidence from the initial outbreak investigations seemed to suggest that 2019-nCoV could not easily spread between humans [3] , it is now very clear that infections have been spreading from person to person [2] . We recently estimated that more than 75,000 infections may have occurred in Wuhan as at 25 January 2020 [4] , and increasing numbers of infections continue to be detected in other cities in mainland China and around the world. A number of important characteristics of 2019-nCoV infection have already been identified, but in order to calibrate public health responses we need improved information on transmission dynamics, severity of the disease, immunity, and the impact of control and mitigation measures that have been applied to date.\\n\\nInfections with 2019-nCoV can spread from person to person, and in the earliest phase of the outbreak the basic reproductive number was estimated to be around 2.2, assuming a mean serial interval of 7.5 days [2] . The serial interval was not precisely estimated, and a potentially shorter mean serial interval would have corresponded to a slightly lower basic reproductive number. Control measures and changes in population behaviour later in January should have reduced the effective reproductive number. However, it is too early to estimate whether the effective reproductive number has been reduced to below the critical threshold of 1 because cases currently being detected and reported would have mostly been infected in mid-to late-January. Average delays between infection and illness onset have been estimated at around 5-6 days, with an upper limit of around 11-14 days [2, 5] , and delays from illness onset to laboratory confirmation added a further 10 days on average [2] .\\n\\nChains of transmission have now been reported in a number of locations outside of mainland China. Within the coming days or weeks it will become clear whether sustained local transmission has been occurring in other cities outside of Hubei province in China, or in other countries. If sustained transmission does occur in other locations, it would be valuable to determine whether there is variation in transmissibility by location, for example because of different behaviours or control measures, or because of different environmental conditions. To address the latter, virus survival studies can be done in the laboratory to confirm whether there are preferred ranges of temperature or humidity for 2019-nCoV transmission to occur.\\n\\nIn an analysis of the first 425 confirmed cases of infection, 73% of cases with illness onset between 12 and 22 January reported no exposure to either a wet market or another person with symptoms of a respiratory illness [2] . The lack of reported exposure to another ill person could be attributed to lack of awareness or recall bias, but China's health minister publicly warned that pre-symptomatic transmission could be occurring [6] . Determining the extent to which asymptomatic or pre-symptomatic transmission might be occurring is an urgent priority, because it has direct implications for public health and hospital infection control. Data on viral shedding dynamics could help in assessing duration of infectiousness. For severe acute respiratory syndrome-related coronavirus (SARS-CoV), infectivity peaked at around 10 days after illness onset [7] , consistent with the peak in viral load at around that time [8] . This allowed control of the SARS epidemic through prompt detection of cases and strict isolation. For influenza virus infections, virus shedding is highest on the day of illness onset and relatively higher from shortly before symptom onset until a few days after onset [9] . To date, transmission patterns of 2019-nCoV appear more similar to influenza, with contagiousness occurring around the time of symptom onset, rather than SARS.\\n\\nTransmission of respiratory viruses generally happens through large respiratory droplets, but some respiratory viruses can spread through fine particle aerosols [10] , and indirect transmission via fomites can also play a role. Coronaviruses can also infect the human gastrointestinal tract [11, 12] , and faecal-oral transmission might also play a role in this instance. The SARS-CoV superspreading event at Amoy Gardens where more than 300 cases were infected was attributed to faecal-oral, then airborne, spread through pressure differentials between contaminated effluent pipes, bathroom floor drains and flushing toilets [13] . The first large identifiable superspreading event during the present 2019-nCoV outbreak has apparently taken place on the Diamond Princess cruise liner quarantined off the coast of Yokohama, Japan, with at least 130 passengers tested positive for 2019-nCoV as at 10 February 2020 [14] . Identifying which modes are important for 2019-nCoV transmission would inform the importance of personal protective measures such as face masks (and specifically which types) and hand hygiene.\\n\\nThe first human infections were identified through a surveillance system for pneumonia of unknown aetiology, and all of the earliest infections therefore had Modelling studies incorporating healthcare capacity and processes pneumonia. It is well established that some infections can be severe, particularly in older adults with underlying medical conditions [15, 16] , but based on the generally mild clinical presentation of 2019-nCoV cases detected outside China, it appears that there could be many more mild infections than severe infections. Determining the spectrum of clinical manifestations of 2019-nCoV infections is perhaps the most urgent research priority, because it determines the strength of public health response required. If the seriousness of infection is similar to the 1918/19 Spanish influenza, and therefore at the upper end of severity scales in influenza pandemic plans, the same responses would be warranted for 2019-nCoV as for the most severe influenza pandemics. If, however, the seriousness of infection is similar to seasonal influenza, especially during milder seasons, mitigation measures could be tuned accordingly.\\n\\nBeyond a robust assessment of overall severity, it is also important to determine high risk groups. Infections would likely be more severe in older adults, obese individuals or those with underlying medical conditions, but there have not yet been reports of severity of infections in pregnant women, and very few cases have been reported in children [2] .\\n\\nThose under 18 years are a critical group to study in order to tease out the relative roles of susceptibility vs severity as possible underlying causes for the very rare recorded instances of infection in this age group. Are children protected from infection or do they not fall ill after infection? If they are naturally immune, which is unlikely, we should understand why; otherwise, even if they do not show symptoms, it is important to know if they shed the virus. Obviously, the question about virus shedding of those being infected but asymptomatic leads to the crucial question of infectivity. Answers to these questions are especially pertinent as basis for decisions on school closure as a social distancing intervention, which can be hugely disruptive not only for students but also because of its knock-on effect for child care and parental duties. Very few children have been confirmed 2019-nCoV cases so far but that does not necessarily mean that they are less susceptible or that they could not be latent carriers. Serosurveys in affected locations could inform this, in addition to truly assessing the clinical severity spectrum.\\n\\nAnother question on susceptibility is regarding whether 2019-nCoV infection confers neutralising immunity, usually but not always, indicated by the presence of neutralising antibodies in convalescent sera. Some experts already questioned whether the 2019-nCoV may behave similarly to MERS-CoV in cases exhibiting mild symptoms without eliciting neutralising antibodies [17] . A separate question pertains to the possibility of antibody-dependent enhancement of infection or of disease [18, 19] . If either of these were to be relevant, the transmission dynamics could become more complex.\\n\\nA wide range of control measures can be considered to contain or mitigate an emerging infection such as 2019-nCoV. Internationally, the past week has seen an increasing number of countries issue travel advisories or outright entry bans on persons from Hubei province or China as a whole, as well as substantial cuts in flights to and from affected areas out of commercial considerations. Evaluation of these mobility restrictions can confirm their potential effectiveness in delaying local epidemics [20] , and can also inform when as well as how to lift these restrictions.\\n\\nIf and when local transmission begins in a particular location, a variety of community mitigation measures can be implemented by health authorities to reduce transmission and thus reduce the growth rate of an epidemic, reduce the height of the epidemic peak and the peak demand on healthcare services, as well as reduce the total number of infected persons [21] . A number of social distancing measures have already been implemented in Chinese cities in the past few weeks including school and workplace closures. It should now be an urgent priority to quantify the effects of these measures and specifically whether they can reduce the effective reproductive number below 1, because this will guide the response strategies in other locations. During the 1918/19 influenza pandemic, cities in the United States, which implemented the most aggressive and sustained community measures were the most successful ones in mitigating the impact of that pandemic [22] .\\n\\nSimilarly to international travel interventions, local social distancing measures should be assessed for their impact and when they could be safely discontinued, albeit in a coordinated and deliberate manner across China such that recrudescence in the epidemic curve is minimised. Mobile telephony global positioning system (GPS) data and location services data from social media providers such as Baidu and Tencent in China could become the first occasion when these data inform outbreak control in real time.\\n\\nAt the individual level, surgical face masks have often been a particularly visible image from affected cities in China. Face masks are essential components of personal protective equipment in healthcare settings, and should be recommended for ill persons in the community or for those who care for ill persons. However, there is now a shortage of supply of masks in China and elsewhere, and debates are ongoing about their protective value for uninfected persons in the general community.\\n\\nThe Table summarises research gaps to guide the public health response identified.\\n\\nIn conclusion, there are a number of urgent research priorities to inform the public health response to the global spread of 2019-nCoV infections. Establishing robust estimates of the clinical severity of infections is probably the most pressing, because flattening out the surge in hospital admissions would be essential if there is a danger of hospitals becoming overwhelmed with patients who require inpatient care, not only for those infected with 2019-nCoV but also for urgent acute care of patients with other conditions including those scheduled for procedures and operations. In addressing the research gaps identified here, there is a need for strong collaboration of a competent corps of epidemiological scientists and public health workers who have the flexibility to cope with the surge capacity required, as well as support from laboratories that can deliver on the ever rising demand for diagnostic tests for 2019-nCoV and related sequelae. The readiness survey by Reusken et al. in this issue of Eurosurveillance testifies to the rapid response and capabilities of laboratories across Europe should the outbreak originating in Wuhan reach this continent [23] .\\n\\nIn the medium term, we look towards the identification of efficacious pharmaceutical agents to prevent and treat what may likely become an endemic infection globally. Beyond the first year, one interesting possibility in the longer term, perhaps borne of wishful hope, is that after the first few epidemic waves, the subsequent endemic re-infections could be of milder severity. Particularly if children are being infected and are developing immunity hereafter, 2019-nCoV could optimistically become the fifth human coronavirus causing the common cold.\\n\\nNone declared.\", 'document_id': 2526}]}, {'paragraphs': [{'qas': [{'question': 'When is this especially true?', 'id': 3963, 'answers': [{'text': 'when not all exacerbation events occurred during the viral infection but may also occur well after viral clearance (Kim et al., 2008; Stolz et al., 2019) in particular the late onset of a bacterial infection', 'answer_start': 15414}], 'is_impossible': False}, {'question': 'What is the current understanding on viral-induced exacerbations?', 'id': 3855, 'answers': [{'text': 'that viral infection increases airway inflammation which aggravates disease symptoms.', 'answer_start': 749}], 'is_impossible': False}, {'question': 'What have evoked new understandings as to the mechanisms of viral exacerbations?', 'id': 3856, 'answers': [{'text': 'Recent advances in in vitro air-liquid interface 3D cultures, organoid cultures and the use of novel human and animal challenge models', 'answer_start': 835}], 'is_impossible': False}, {'question': 'What is one of  the major sources of exacerbation of chronic airway inflammatory diseases?', 'id': 3854, 'answers': [{'text': 'Respiratory virus infection', 'answer_start': 491}], 'is_impossible': False}, {'question': 'What is the focus of this review?', 'id': 3857, 'answers': [{'text': ' recent novel findings that elucidate how respiratory viral infections alter the epithelial barrier in the airways, the upper airway microbial environment, epigenetic modifications including miRNA modulation, and other changes in immune responses throughout the upper and lower airways.', 'answer_start': 1078}], 'is_impossible': False}, {'question': 'What have the authors reviewed?', 'id': 3858, 'answers': [{'text': 'the prevalence of different respiratory viral infections in causing exacerbations in chronic airway inflammatory diseases.', 'answer_start': 1384}], 'is_impossible': False}, {'question': 'What is summarized?', 'id': 3859, 'answers': [{'text': 'how recent models have expanded our appreciation of the mechanisms of viral-induced exacerbations.', 'answer_start': 1539}], 'is_impossible': False}, {'question': 'What is highlighted by the authors?', 'id': 3860, 'answers': [{'text': 'the importance of the virome within the airway microbiome environment and its impact on subsequent bacterial infection.', 'answer_start': 1661}], 'is_impossible': False}, {'question': 'What is consolidated in this review?', 'id': 3861, 'answers': [{'text': 'the understanding of viral induced exacerbation in chronic airway inflammatory diseases and indicates pathways that may be targeted for more effective management of chronic inflammatory diseases.', 'answer_start': 1806}], 'is_impossible': False}, {'question': 'What is this disease  characterized by ?', 'id': 3862, 'answers': [{'text': ' airway inflammation leading to complications such as coughing, wheezing and shortness of breath. ', 'answer_start': 2230}], 'is_impossible': False}, {'question': 'Where can this disease manifest?', 'id': 3863, 'answers': [{'text': ' in both the upper airway (such as chronic rhinosinusitis, CRS) and lower airway (such as asthma and chronic obstructive pulmonary disease, COPD)', 'answer_start': 2352}], 'is_impossible': False}, {'question': 'Why do treatment and management vary in efficacy?', 'id': 3864, 'answers': [{'text': 'due to the complexity and heterogeneity of the disease.', 'answer_start': 2640}], 'is_impossible': False}, {'question': 'What complicates this  further?', 'id': 3865, 'answers': [{'text': 'the effect of episodic exacerbations of the disease, defined as worsening of disease symptoms including wheeze, cough, breathlessness and chest tightness ', 'answer_start': 2727}], 'is_impossible': False}, {'question': 'What are such exacerbations due to?', 'id': 3866, 'answers': [{'text': 'the effect of enhanced acute airway inflammation impacting upon and worsening the symptoms of the existing disease', 'answer_start': 2949}], 'is_impossible': False}, {'question': 'What do the acute exacerbations cause?', 'id': 3867, 'answers': [{'text': 'morbidity and sometimes mortality in patients, as well as resulting in major economic burdens worldwide', 'answer_start': 3168}], 'is_impossible': False}, {'question': 'What are acute exacerbations  usually due to ?', 'id': 3869, 'answers': [{'text': 'the presence of environmental factors such as allergens, pollutants, smoke, cold or dry air and pathogenic microbes in the airway', 'answer_start': 3501}], 'is_impossible': False}, {'question': 'What does the immune response elicited by these agents lead to?', 'id': 3870, 'answers': [{'text': ' infiltration of activated immune cells that further release inflammatory mediators that cause acute symptoms such as increased mucus production, cough, wheeze and shortness of breath. ', 'answer_start': 3739}], 'is_impossible': False}, {'question': 'Among these agents which is a major driver?', 'id': 3871, 'answers': [{'text': 'viral infection is one of the major drivers of asthma exacerbations accounting for up to 80-90% and 45-80% of exacerbations in children and adults respectively', 'answer_start': 3944}], 'is_impossible': False}, {'question': 'What is the viral involvement in COPD exacerbation? ', 'id': 3872, 'answers': [{'text': '30-80% of acute COPD exacerbations', 'answer_start': 4293}], 'is_impossible': False}, {'question': 'What is the reason for the involvement of respiratory viruses in exacerbation?', 'id': 3873, 'answers': [{'text': 'their ease of transmission and infection', 'answer_start': 4685}], 'is_impossible': False}, {'question': 'What does the involvement of respiratory viruses contribute to?', 'id': 3874, 'answers': [{'text': ' important to identify the exact mechanisms underpinning viral exacerbations in susceptible subjects', 'answer_start': 4957}], 'is_impossible': False}, {'question': 'Why is it  important to identify the exact mechanisms underpinning viral exacerbations in susceptible subjects?', 'id': 3875, 'answers': [{'text': ' to properly manage exacerbations via supplementary treatments that may alleviate the exacerbation symptoms or prevent severe exacerbations.', 'answer_start': 5066}], 'is_impossible': False}, {'question': 'What is the lower airway the site of?', 'id': 3876, 'answers': [{'text': 'dysregulated inflammation in most chronic airway inflammatory diseases', 'answer_start': 5246}], 'is_impossible': False}, {'question': 'Where is the the first point of contact with sources of exacerbation', 'id': 3877, 'answers': [{'text': 'the upper airway', 'answer_start': 5318}], 'is_impossible': False}, {'question': 'What is the focus of this review?', 'id': 3878, 'answers': [{'text': ' recent findings of viral interaction with the upper airway.', 'answer_start': 5683}], 'is_impossible': False}, {'question': 'What is complied by the authors?', 'id': 3879, 'answers': [{'text': 'how viral induced changes to the upper airway may contribute to chronic airway inflammatory disease exacerbations, to provide a unified elucidation of the potential exacerbation mechanisms initiated from predominantly upper airway infections.', 'answer_start': 5756}], 'is_impossible': False}, {'question': 'Before linking respiratory viruses, what was linked to acute exacerbations?', 'id': 3880, 'answers': [{'text': 'Despite being a major cause of exacerbation, reports linking respiratory viruses to acute exacerbations only start to emerge in the late 1950s (Pattemore et al., 1992) ; with bacterial infections previously considered as the likely culprit for acute exacerbation ', 'answer_start': 6000}], 'is_impossible': False}, {'question': 'What did the advent of PCR technology lead to?', 'id': 3881, 'answers': [{'text': ' viruses were recovered during acute exacerbations events and reports implicating their role emerged in the late 1980s', 'answer_start': 6357}], 'is_impossible': False}, {'question': 'What are the predominant viruses linked to airway inflammatory diseases?', 'id': 3882, 'answers': [{'text': 'Rhinovirus (RV) and respiratory syncytial virus (RSV) ', 'answer_start': 6507}], 'is_impossible': False}, {'question': 'What other viruses are implicated in acute exacerbations but to a much lesser extent?', 'id': 3883, 'answers': [{'text': 'parainfluenza virus (PIV), influenza virus (IFV) and adenovirus (AdV)', 'answer_start': 6720}], 'is_impossible': False}, {'question': 'What other viruses have been recently reported as contributing to acute exacerbations?', 'id': 3884, 'answers': [{'text': ' viruses including bocavirus (BoV), human metapneumovirus (HMPV), certain coronavirus (CoV) strains, a specific enterovirus (EV) strain EV-D68, human cytomegalovirus (hCMV) and herpes simplex virus (HSV)', 'answer_start': 6951}], 'is_impossible': False}, {'question': 'What are the common feature of these viruses share?', 'id': 3885, 'answers': [{'text': 'they can infect both the upper and/or lower airway, further increasing the inflammatory conditions in the diseased airway', 'answer_start': 7262}], 'is_impossible': False}, {'question': 'Where do the respiratory viruses primarily infect and replicate?', 'id': 3886, 'answers': [{'text': 'within airway epithelial cells ', 'answer_start': 7487}], 'is_impossible': False}, {'question': 'What happens  during the replication process?', 'id': 3887, 'answers': [{'text': 'the cells release antiviral factors and cytokines that alter local airway inflammation and airway niche ', 'answer_start': 7552}], 'is_impossible': False}, {'question': 'What does the inflammation lead to in healthy airways?', 'id': 3888, 'answers': [{'text': 'leads to type 1 inflammatory responses consisting of activation of an antiviral state and infiltration of antiviral effector cells. This eventually results in the resolution of the inflammatory response and clearance of the viral infection ', 'answer_start': 7726}], 'is_impossible': False}, {'question': 'How may the responses be different in a chronically inflamed airway?', 'id': 3891, 'answers': [{'text': ' the responses against the virus may be impaired or aberrant, causing sustained inflammation and erroneous infiltration, resulting in the exacerbation of their symptoms (Mallia and Johnston, 2006; Dougherty and Fahy, 2009; Busse et al., 2010; Britto et al., 2017; Linden et al., 2019) . This is usually further compounded by the increased susceptibility of chronic airway inflammatory disease patients toward viral respiratory infections, thereby increasing the frequency of exacerbation as a whole', 'answer_start': 8057}], 'is_impossible': False}, {'question': 'What will the review focus on?', 'id': 3892, 'answers': [{'text': 'compiling and collating the current known mechanisms of viral-induced exacerbation of chronic airway inflammatory diseases; as well as linking the different viral infection pathogenesis to elucidate other potential ways the infection can exacerbate the disease', 'answer_start': 8884}], 'is_impossible': False}, {'question': 'How will this review serve?', 'id': 3893, 'answers': [{'text': 'to provide further understanding of viral induced exacerbation to identify potential pathways and pathogenesis mechanisms that may be targeted as supplementary care for management and prevention of exacerbation.', 'answer_start': 9168}], 'is_impossible': False}, {'question': 'Why is this approach significant?', 'id': 3894, 'answers': [{'text': 'due to the current scarcity of antiviral drugs for the management of viral-induced exacerbations.', 'answer_start': 9427}], 'is_impossible': False}, {'question': 'What happens upon infection?', 'id': 3941, 'answers': [{'text': 'viruses evoke an inflammatory response as a means of counteracting the infection.', 'answer_start': 9877}], 'is_impossible': False}, {'question': 'How does the infected airway cell respond?', 'id': 3942, 'answers': [{'text': 'Generally, infected airway epithelial cells release type I (IFNα/β) and type III (IFNλ) interferons, cytokines and chemokines such as IL-6, IL-8, IL-12, RANTES, macrophage inflammatory protein 1α (MIP-1α) and monocyte chemotactic protein 1 (MCP-1) (Wark and Gibson, 2006; Matsukura et al., 2013) .', 'answer_start': 9959}], 'is_impossible': False}, {'question': 'What  does the epithelial proteins cause?', 'id': 3943, 'answers': [{'text': 'enable infiltration of innate immune cells and of professional antigen presenting cells (APCs) that will then in turn release specific mediators to facilitate viral targeting and clearance, including type II interferon (IFNγ), IL-2, IL-4, IL-5, IL-9, and IL-12 (Wark and Gibson, 2006; Singh et al., 2010; Braciale et al., 2012) .', 'answer_start': 10273}], 'is_impossible': False}, {'question': 'What is the effect of these factors?', 'id': 3944, 'answers': [{'text': 'These factors heighten local inflammation and the infiltration of granulocytes, T-cells and B-cells (Wark and Gibson, 2006; Braciale et al., 2012) . The increased inflammation, in turn, worsens the symptoms of airway diseases.', 'answer_start': 10603}], 'is_impossible': False}, {'question': 'What additional effects are caused  in patients with asthma and patients with CRS with nasal polyp ?', 'id': 3945, 'answers': [{'text': 'viral infections such as RV and RSV promote a Type 2-biased immune response (Becker, 2006; Jackson et al., 2014; Jurak et al., 2018) . This amplifies the basal type 2 inflammation resulting in a greater release of IL-4, IL-5, IL-13, RANTES and eotaxin and a further increase in eosinophilia, a key pathological driver of asthma and CRSwNP', 'answer_start': 10918}], 'is_impossible': False}, {'question': 'What is the result of increased eosinophilia?', 'id': 3946, 'answers': [{'text': 'worsens the classical symptoms of disease and may further lead to life-threatening conditions due to breathing difficulties', 'answer_start': 11380}], 'is_impossible': False}, {'question': 'What are the effects for patients with COPD and patients with CRS without nasal polyp (CRSsNP) ?', 'id': 3947, 'answers': [{'text': 'are more neutrophilic in nature due to the expression of neutrophil chemoattractants such as CXCL9, CXCL10, and CXCL11 (Cukic et al., 2012; Brightling and Greening, 2019) . The pathology of these airway diseases is characterized by airway remodeling due to the presence of remodeling factors such as matrix metalloproteinases (MMPs) released from infiltrating neutrophils (Linden et al., 2019) . Viral infections in such conditions will then cause increase neutrophilic activation; worsening the symptoms and airway remodeling in the airway thereby exacerbating COPD, CRSsNP and even CRSwNP in certain cases ', 'answer_start': 11594}], 'is_impossible': False}, {'question': ' Which are the type 2 inflammatory cytokines expressed by the epithelial cells upon injury to the epithelial barrier?', 'id': 3948, 'answers': [{'text': ' IL-25, IL-33 and TSLP ', 'answer_start': 12529}], 'is_impossible': False}, {'question': 'Which cells lacking both B and T cell receptors but play a crucial role in secreting type 2 cytokines to perpetuate type 2 inflammation when activated ?', 'id': 3949, 'answers': [{'text': ' ILC2s are a group of lymphoid cells', 'answer_start': 12704}], 'is_impossible': False}, {'question': 'What is the effect cell death and injury to the epithelial barrier due to infection?', 'id': 3950, 'answers': [{'text': 'induce the expression of IL-25, IL-33 and TSLP, with heighten expression in an inflamed airway', 'answer_start': 13025}], 'is_impossible': False}, {'question': 'What happens when the 3 cytokines are expressed?', 'id': 3951, 'answers': [{'text': 'These 3 cytokines then work in concert to activate ILC2s to further secrete type 2 cytokines IL-4, IL-5, and IL-13 which further aggravate the type 2 inflammation in the airway causing acute exacerbation (', 'answer_start': 13303}], 'is_impossible': False}, {'question': 'What  happens in the case of COPD?', 'id': 3952, 'answers': [{'text': ' increased ILC2 activation, which retain the capability of differentiating to ILC1, may also further augment the neutrophilic response and further aggravate the exacerbation', 'answer_start': 13551}], 'is_impossible': False}, {'question': 'What happens during viral infection of healthy individuals?', 'id': 3954, 'answers': [{'text': ' these factors are not released to any great extent and do not activate an ILC2 response during viral infection in healthy individuals (Yan et al., 2016; Tan et al., 2018a) ; despite augmenting a type 2 exacerbation in chronically inflamed airways (Jurak et al., 2018) . These classical mechanisms of viral induced acute exacerbations are summarized in Figure 1 ', 'answer_start': 13763}], 'is_impossible': False}, {'question': 'What happens upon viral infection in the airway?', 'id': 3960, 'answers': [{'text': 'antiviral state will be activated to clear the invading pathogen from the airway. Immune response and injury factors released from the infected epithelium normally would induce a rapid type 1 immunity that facilitates viral clearance. ', 'answer_start': 14396}], 'is_impossible': False}, {'question': 'What is the effect of the inflammation of the airway?', 'id': 3961, 'answers': [{'text': ' However, in the inflamed airway, the cytokines and chemokines released instead augmented the inflammation present in the chronically inflamed airway, strengthening the neutrophilic infiltration in COPD airway, and eosinophilic infiltration in the asthmatic airway. The effect is also further compounded by the participation of Th1 and ILC1 cells in the COPD airway; and Th2 and ILC2 cells in the asthmatic airway.', 'answer_start': 14630}], 'is_impossible': False}, {'question': 'What increases the severity of exacerbations in the airway?', 'id': 3962, 'answers': [{'text': 'the synergistic effect of viral infection with other sensitizing agents ', 'answer_start': 15260}], 'is_impossible': False}, {'question': 'Why viruses do not need to directly infect the lower airway to cause an acute exacerbation?', 'id': 3964, 'answers': [{'text': 'he nasal epithelium remains the primary site of most infections.', 'answer_start': 15786}], 'is_impossible': False}, {'question': 'what is suggested by the fact that  not all viral infections of the airway lead to acute exacerbations?', 'id': 3965, 'answers': [{'text': 'a more complex interplay between the virus and upper airway epithelium which synergize with the local airway environment in line with the \"united airway\" hypothesis ', 'answer_start': 15945}], 'is_impossible': False}, {'question': 'What is the effect of chronic airway inflammatory disease  in patients?', 'id': 3966, 'answers': [{'text': 'viral infections or their components persist in patients', 'answer_start': 16152}], 'is_impossible': False}, {'question': 'What is the effect of chronic airway inflammatory disease?', 'id': 3967, 'answers': [{'text': 'may further alter the local environment and contribute to current and future exacerbations', 'answer_start': 16333}], 'is_impossible': False}, {'question': 'Why should future studies  be performed using metagenomics in addition to PCR analysis ?', 'id': 3968, 'answers': [{'text': 'to determine the contribution of the microbiome and mycobiome to viral infections.', 'answer_start': 16507}], 'is_impossible': False}, {'question': 'What is highlighted by the authors in this review?', 'id': 3969, 'answers': [{'text': ' recent data regarding viral interactions with the airway epithelium that could also contribute to, or further aggravate, acute exacerbations of chronic airway inflammatory diseases.', 'answer_start': 16618}], 'is_impossible': False}, {'question': 'Who has impaired or reduced ability of viral clearance ?', 'id': 3970, 'answers': [{'text': 'Patients with chronic airway inflammatory diseases ', 'answer_start': 16802}], 'is_impossible': False}, {'question': 'What does their impairment stems from?', 'id': 3971, 'answers': [{'text': 'a type 2-skewed inflammatory response which deprives the airway of important type 1 responsive CD8 cells that are responsible for the complete clearance of virusinfected cells ', 'answer_start': 17072}], 'is_impossible': False}, {'question': 'Where is this  especially evident?', 'id': 3972, 'answers': [{'text': 'in weak type 1 inflammation-inducing viruses such as RV and RSV', 'answer_start': 17315}], 'is_impossible': False}, {'question': 'What are other  effects?', 'id': 3973, 'answers': [{'text': 'there are also evidence of reduced type I (IFNβ) and III (IFNλ) interferon production due to type 2-skewed inflammation, which contributes to imperfect clearance of the virus resulting in persistence of viral components, or the live virus in the airway epithelium ', 'answer_start': 17454}], 'is_impossible': False}, {'question': 'What is the  effect of viral components remaining in the airway?', 'id': 3974, 'answers': [{'text': 'antiviral genes such as type I interferons, inflammasome activating factors and cytokines remained activated resulting in prolong airway inflammation', 'answer_start': 17828}], 'is_impossible': False}, {'question': 'What do these factors do?', 'id': 3975, 'answers': [{'text': 'enhance granulocyte infiltration thus prolonging the exacerbation symptoms. Such persistent inflammation may also be found within DNA viruses such as AdV, hCMV and HSV, whose infections generally persist longer (Imperiale and Jiang, 2015) , further contributing to chronic activation of inflammation when they infect the airway', 'answer_start': 18044}], 'is_impossible': False}, {'question': 'What is also linked with the chronic inflammation that precedes the malignancies?', 'id': 3976, 'answers': [{'text': 'human papilloma virus (HPV), a DNA virus highly associated with head and neck cancers and respiratory papillomatosis,', 'answer_start': 18519}], 'is_impossible': False}, {'question': 'What should be investigated in the future?', 'id': 3977, 'answers': [{'text': ' the role of HPV infection in causing chronic inflammation in the airway and their association to exacerbations of chronic airway inflammatory diseases, which is scarcely explored', 'answer_start': 18818}], 'is_impossible': False}, {'question': 'What further can viral persistence lead to?', 'id': 3978, 'answers': [{'text': 'viral persistence which lead to continuous expression of antiviral genes may also lead to the development of steroid resistance, which is seen with RV, RSV, and PIV infection', 'answer_start': 19050}], 'is_impossible': False}, {'question': 'What  effect the use of steroids to  suppress inflammation can have?', 'id': 3979, 'answers': [{'text': 'may also cause the virus to linger longer in the airway due to the lack of antiviral clearance', 'answer_start': 19332}], 'is_impossible': False}, {'question': 'What should be further focus of research?', 'id': 3980, 'answers': [{'text': 'The concomitant development of steroid resistance together with recurring or prolong viral infection ', 'answer_start': 19542}], 'is_impossible': False}, {'question': 'Which viruses may not cause prolonged inflammation due to strong induction of antiviral clearance?', 'id': 3981, 'answers': [{'text': 'viruses that induce strong type 1 inflammation and cell death such as IFV (Yan et al., 2016; Guibas et al., 2018) and certain CoV (including the recently emerged COVID-19 virus)', 'answer_start': 19859}], 'is_impossible': False}, {'question': 'What do these infections cause?', 'id': 3982, 'answers': [{'text': 'massive damage and cell death to the epithelial barrier, so much so that areas of the epithelium may be completely absent post infection (Yan et al., 2016; Tan et al., 2019) . Factors such as RANTES and CXCL10, which recruit immune cells to induce apoptosis, are strongly induced from IFV infected epithelium ', 'answer_start': 20212}], 'is_impossible': False}, {'question': 'What do the necroptotic factors such as RIP3 do?', 'id': 3983, 'answers': [{'text': ' further compounds the cell deaths in IFV infected epithelium . The massive cell death induced may result in worsening of the acute exacerbation due to the release of their cellular content into the airway, further evoking an inflammatory response in the airway ', 'answer_start': 20610}], 'is_impossible': False}, {'question': 'What  may the destruction of the epithelial barrier  cause?', 'id': 3984, 'answers': [{'text': ' further contact with other pathogens and allergens in the airway which may then prolong exacerbations or results in new exacerbations.', 'answer_start': 20957}], 'is_impossible': False}, {'question': 'What may the epithelial destruction cause?', 'id': 3985, 'answers': [{'text': 'may also promote further epithelial remodeling during its regeneration as viral infection induces the expression of remodeling genes such as MMPs and growth factors . Infections that cause massive destruction of the epithelium, such as IFV, usually result in severe acute exacerbations with non-classical symptoms of chronic airway inflammatory diseases.', 'answer_start': 21116}], 'is_impossible': False}, {'question': 'What is recommended that patients with chronic airway inflammatory disease?', 'id': 3986, 'answers': [{'text': ' receive their annual influenza vaccination as the best means to prevent severe IFV induced exacerbation.', 'answer_start': 21663}], 'is_impossible': False}, {'question': 'What is another mechanism that viral infections use to drive acute exacerbations?', 'id': 3987, 'answers': [{'text': 'the induction of vasodilation or tight junction opening factors which may increase the rate of infiltration. ', 'answer_start': 21850}], 'is_impossible': False}, {'question': 'What does infection of respiratory viruses cause?', 'id': 3988, 'answers': [{'text': 'disruption of tight junctions with the resulting increased rate of viral infiltration. This also increases the chances of allergens coming into contact with airway immune cells.', 'answer_start': 22016}], 'is_impossible': False}, {'question': 'What is an example of this?', 'id': 3989, 'answers': [{'text': ' IFV infection was found to induce oncostatin M (OSM) which causes tight junction opening (Pothoven et al., 2015; Tian et al., 2018) . Similarly, RV and RSV infections usually cause tight junction opening which may also increase the infiltration rate of eosinophils and thus worsening of the classical symptoms of chronic airway inflammatory diseases ', 'answer_start': 22206}], 'is_impossible': False}, {'question': 'What are also associated with viral infections and pneumonia development, which may worsen inflammation in the lower airway?', 'id': 3990, 'answers': [{'text': ' the expression of vasodilating factors and fluid homeostatic factors such as angiopoietin-like 4 (ANGPTL4) and bactericidal/permeabilityincreasing fold-containing family member A1 (BPIFA1)', 'answer_start': 22630}], 'is_impossible': False}, {'question': 'What is another area of interest?', 'id': 3991, 'answers': [{'text': 'the relationship between asthma and COPD exacerbations and their association with the airway microbiome.', 'answer_start': 23156}], 'is_impossible': False}, {'question': 'What is usually linked with the development of chronic airway inflammatory diseases?', 'id': 3992, 'answers': [{'text': 'specific bacterial species in the microbiome which may thrive in the inflamed airway environment ', 'answer_start': 23338}], 'is_impossible': False}, {'question': 'What follows  in the event of a viral infection such as RV infection?', 'id': 3993, 'answers': [{'text': ' the effect induced by the virus may destabilize the equilibrium of the microbiome present (Molyneaux et al., 2013; Kloepfer et al., 2014; Kloepfer et al., 2017; Jubinville et al., 2018; van Rijn et al., 2019) . In addition, viral infection may disrupt biofilm colonies in the upper airway (e.g., Streptococcus pneumoniae) microbiome to be release into the lower airway and worsening the inflammation', 'answer_start': 23513}], 'is_impossible': False}, {'question': 'What does the viral infection alter?', 'id': 3994, 'answers': [{'text': 'the nutrient profile in the airway through release of previously inaccessible nutrients that will alter bacterial growth', 'answer_start': 23999}], 'is_impossible': False}, {'question': 'What is  the destabilization is further compounded by?', 'id': 3995, 'answers': [{'text': ' impaired bacterial immune response, either from direct viral influences, or use of corticosteroids to suppress the exacerbation symptoms', 'answer_start': 24222}], 'is_impossible': False}, {'question': 'What does all this gradually lead to?', 'id': 3996, 'answers': [{'text': 'more far reaching effect when normal flora is replaced with opportunistic pathogens, altering the inflammatory profiles ', 'answer_start': 24491}], 'is_impossible': False}, {'question': 'Why do these changes may  result in more severe and frequent acute exacerbations ?', 'id': 3997, 'answers': [{'text': 'due to the interplay between virus and pathogenic bacteria in exacerbating chronic airway inflammatory diseases ', 'answer_start': 24713}], 'is_impossible': False}, {'question': 'How effective are microbiome based  trial therapies?', 'id': 3998, 'answers': [{'text': ' have shown efficacy in the treatments of irritable bowel syndrome by restoring the intestinal microbiome (', 'answer_start': 24954}], 'is_impossible': False}, {'question': 'What can viral infections cause?', 'id': 3999, 'answers': [{'text': 'the disruption of mucociliary function, an important component of the epithelial barrier. ', 'answer_start': 25257}], 'is_impossible': False}, {'question': 'Which is the primary contact/infection site of most respiratory viruses?', 'id': 4000, 'answers': [{'text': 'The upper airway epithelium', 'answer_start': 25490}], 'is_impossible': False}, {'question': 'What does the destruction of epithelial barrier, mucociliary function and cell death of the epithelial cells do?', 'id': 4001, 'answers': [{'text': 'serves to increase contact between environmental triggers with the lower airway and resident immune cells.', 'answer_start': 25837}], 'is_impossible': False}, {'question': 'What are viral infections are usually accompanied with?', 'id': 4002, 'answers': [{'text': 'oxidative stress which will further increase the local inflammation in the airway. ', 'answer_start': 26111}], 'is_impossible': False}, {'question': 'What is the dysregulation of inflammation can be further compounded by?', 'id': 4003, 'answers': [{'text': 'modulation of miRNAs and epigenetic modification such as DNA methylation and histone modifications that promote dysregulation in inflammation', 'answer_start': 26257}], 'is_impossible': False}, {'question': 'What does  the change in the local airway environment and inflammation promote?', 'id': 4004, 'answers': [{'text': 'growth of pathogenic bacteria that may replace the airway microbiome. ', 'answer_start': 26478}], 'is_impossible': False}, {'question': 'What does the the inflammatory environment dispersal of upper airway commensals into the lower airway cause?', 'id': 4005, 'answers': [{'text': 'inflammation and alteration of the lower airway environment, resulting in prolong exacerbation episodes following viral infection. ', 'answer_start': 26671}], 'is_impossible': False}, {'question': 'Which are the most commonly  studied viruses in chronic airway inflammatory diseases?', 'id': 4006, 'answers': [{'text': 'RV, RSV, and IFV', 'answer_start': 26947}], 'is_impossible': False}, {'question': 'What do the infections such as RSV are shown to do?', 'id': 4007, 'answers': [{'text': ' to directly destroy the cilia of the ciliated cells and almost all respiratory viruses infect the ciliated cells', 'answer_start': 28279}], 'is_impossible': False}, {'question': 'What does mucus overproduction do?', 'id': 4008, 'answers': [{'text': ' disrupt the equilibrium of the mucociliary function following viral infection, resulting in symptoms of acute exacerbation', 'answer_start': 28495}], 'is_impossible': False}, {'question': 'What does the disruption of the ciliary movement during viral infection may cause?', 'id': 4009, 'answers': [{'text': 'MicroRNAs (miRNAs)', 'answer_start': 29025}], 'is_impossible': False}, {'question': 'What are MicroRNAs(miRNA)?', 'id': 4010, 'answers': [{'text': 'short non-coding RNAs involved in post-transcriptional modulation of biological processes, and implicated in a number of diseases ', 'answer_start': 29048}], 'is_impossible': False}, {'question': 'What are  miRNAs found to be induced by?', 'id': 4011, 'answers': [{'text': ' viral infections and may play a role in the modulation of antiviral responses and inflammation ', 'answer_start': 29232}], 'is_impossible': False}, {'question': 'What  were linked to the exacerbation of the airway inflammation disease?', 'id': 4012, 'answers': [{'text': 'circulating miRNA changes', 'answer_start': 29446}], 'is_impossible': False}, {'question': 'Where might such miRNA changes have originated from?', 'id': 4013, 'answers': [{'text': 'from the infected epithelium and responding immune cells, which may serve to further dysregulate airway inflammation leading to exacerbations.', 'answer_start': 29615}], 'is_impossible': False}, {'question': 'What are both IFV and RSV infections shown to do?', 'id': 4014, 'answers': [{'text': ' to increase miR-21 and augmented inflammation in experimental murine asthma models, which is reversed with a combination treatment of anti-miR-21 and corticosteroids', 'answer_start': 29800}], 'is_impossible': False}, {'question': 'What are IFV infection shown to do?', 'id': 4015, 'answers': [{'text': ' increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections ', 'answer_start': 30018}], 'is_impossible': False}, {'question': 'What happens in in asthmatic epithelium in IFV infection?', 'id': 4016, 'answers': [{'text': 'miR-22 was shown to be suppressed in asthmatic epithelium in IFV infection which lead to aberrant epithelial response, contributing to exacerbations ', 'answer_start': 30247}], 'is_impossible': False}, {'question': 'What do non-coding RNAs present as?', 'id': 4017, 'answers': [{'text': 'as targets to modulate viral induced airway changes as a means of managing exacerbation of chronic airway inflammatory diseases.', 'answer_start': 30753}], 'is_impossible': False}, {'question': 'What mechanisms, other than miRNA modulation play a role?', 'id': 4018, 'answers': [{'text': 'epigenetic modification such as DNA methylation ', 'answer_start': 30917}], 'is_impossible': False}, {'question': 'What have  recent epigenetic studies indicated?', 'id': 4019, 'answers': [{'text': 'the association of epigenetic modification and chronic airway inflammatory diseases, and that the nasal methylome was shown to be a sensitive marker for airway inflammatory changes (Cardenas et al., 2019; Gomez, 2019) .', 'answer_start': 31084}], 'is_impossible': False}, {'question': ' What have these studies also shown?', 'id': 4020, 'answers': [{'text': ' viral infections such as RV and RSV alters DNA methylation and histone modifications in the airway epithelium which may alter inflammatory responses, driving chronic airway inflammatory diseases and exacerbations', 'answer_start': 31344}], 'is_impossible': False}, {'question': 'What has Spalluto et.al. have shown?', 'id': 4021, 'answers': [{'text': 'that antiviral factors such as IFNγ epigenetically modifies the viral resistance of epithelial cells.', 'answer_start': 31672}], 'is_impossible': False}, {'question': ' What infections such as RV and RSV that weakly induce antiviral responses may result in?', 'id': 4022, 'answers': [{'text': ' an altered inflammatory state contributing to further viral persistence and exacerbation of chronic airway inflammatory diseases ', 'answer_start': 31886}], 'is_impossible': False}, {'question': 'What can viral infection result in?', 'id': 4023, 'answers': [{'text': ' enhanced production of reactive oxygen species (ROS), oxidative stress and mitochondrial dysfunction in the airway epithelium ', 'answer_start': 32081}], 'is_impossible': False}, {'question': 'What sustains the inflammation  in the airway?', 'id': 4024, 'answers': [{'text': 'state of constant oxidative stress ', 'answer_start': 32365}], 'is_impossible': False}, {'question': 'What may viral infections of the respiratory epithelium by viruses such as IFV, RV, RSV and HSV do?', 'id': 4025, 'answers': [{'text': 'may trigger the further production of ROS as an antiviral mechanism', 'answer_start': 32578}], 'is_impossible': False}, {'question': 'What can happen  in response to the infection such as neutrophils?', 'id': 4026, 'answers': [{'text': ' infiltrating cells in response to the infection such as neutrophils will also trigger respiratory burst as a means of increasing the ROS in the infected region. The increased ROS and oxidative stress in the local environment may serve as a trigger to promote inflammation thereby aggravating the inflammation in the airway ', 'answer_start': 32697}], 'is_impossible': False}, {'question': 'In addition to worsening disease symptoms, what do viral-induced exacerbations do?', 'id': 4027, 'answers': [{'text': ' also may alter the management of the disease or confer resistance toward treatments that worked before. ', 'answer_start': 33515}], 'is_impossible': False}, {'question': 'What may studies in natural exacerbations and in viral-challenge models using RNA-sequencing (RNA-seq) or single cell RNA-seq on a range of time-points  provide?', 'id': 4028, 'answers': [{'text': 'important information regarding viral pathogenesis and changes induced within the airway of chronic airway inflammatory disease patients to identify novel targets and pathway for improved management of the disease.', 'answer_start': 34001}], 'is_impossible': False}, {'question': 'What analysis functions may be useful?', 'id': 4029, 'answers': [{'text': ' epithelial cell models such as the air-liquid interface, in vitro airway epithelial model that has been adapted to studying viral infection and the changes it induced in the airway ', 'answer_start': 34256}], 'is_impossible': False}, {'question': 'For what  purpose animal based models aare developed for?', 'id': 4030, 'answers': [{'text': 'to identify systemic mechanisms of acute exacerbation', 'answer_start': 34551}], 'is_impossible': False}, {'question': 'What can be used unravel the immune profile of a viral infection in healthy and diseased condition?', 'id': 4031, 'answers': [{'text': ' the humanized mouse model that possess human immune cells ', 'answer_start': 34685}], 'is_impossible': False}, {'question': 'For what purpose controlled in vivo human infections can be performed for mild viruses?', 'id': 4032, 'answers': [{'text': 'the best mode of verification of the associations of the virus with the proposed mechanism of viral induced acute exacerbations ', 'answer_start': 34966}], 'is_impossible': False}, {'question': 'Why may the  mechanisms of exacerbation vary considerably?', 'id': 3868, 'answers': [{'text': ' due to the complex interactions between the host and the exacerbation agents', 'answer_start': 3281}], 'is_impossible': False}], 'context': 'Respiratory Viral Infections in Exacerbation of Chronic Airway Inflammatory Diseases: Novel Mechanisms and Insights From the Upper Airway Epithelium\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC7052386/\\n\\nSHA: 45a566c71056ba4faab425b4f7e9edee6320e4a4\\n\\nAuthors: Tan, Kai Sen; Lim, Rachel Liyu; Liu, Jing; Ong, Hsiao Hui; Tan, Vivian Jiayi; Lim, Hui Fang; Chung, Kian Fan; Adcock, Ian M.; Chow, Vincent T.; Wang, De Yun\\nDate: 2020-02-25\\nDOI: 10.3389/fcell.2020.00099\\nLicense: cc-by\\n\\nAbstract: Respiratory virus infection is one of the major sources of exacerbation of chronic airway inflammatory diseases. These exacerbations are associated with high morbidity and even mortality worldwide. The current understanding on viral-induced exacerbations is that viral infection increases airway inflammation which aggravates disease symptoms. Recent advances in in vitro air-liquid interface 3D cultures, organoid cultures and the use of novel human and animal challenge models have evoked new understandings as to the mechanisms of viral exacerbations. In this review, we will focus on recent novel findings that elucidate how respiratory viral infections alter the epithelial barrier in the airways, the upper airway microbial environment, epigenetic modifications including miRNA modulation, and other changes in immune responses throughout the upper and lower airways. First, we reviewed the prevalence of different respiratory viral infections in causing exacerbations in chronic airway inflammatory diseases. Subsequently we also summarized how recent models have expanded our appreciation of the mechanisms of viral-induced exacerbations. Further we highlighted the importance of the virome within the airway microbiome environment and its impact on subsequent bacterial infection. This review consolidates the understanding of viral induced exacerbation in chronic airway inflammatory diseases and indicates pathways that may be targeted for more effective management of chronic inflammatory diseases.\\n\\nText: The prevalence of chronic airway inflammatory disease is increasing worldwide especially in developed nations (GBD 2015 Chronic Respiratory Disease Collaborators, 2017 Guan et al., 2018) . This disease is characterized by airway inflammation leading to complications such as coughing, wheezing and shortness of breath. The disease can manifest in both the upper airway (such as chronic rhinosinusitis, CRS) and lower airway (such as asthma and chronic obstructive pulmonary disease, COPD) which greatly affect the patients\\' quality of life (Calus et al., 2012; Bao et al., 2015) . Treatment and management vary greatly in efficacy due to the complexity and heterogeneity of the disease. This is further complicated by the effect of episodic exacerbations of the disease, defined as worsening of disease symptoms including wheeze, cough, breathlessness and chest tightness (Xepapadaki and Papadopoulos, 2010) . Such exacerbations are due to the effect of enhanced acute airway inflammation impacting upon and worsening the symptoms of the existing disease (Hashimoto et al., 2008; Viniol and Vogelmeier, 2018) . These acute exacerbations are the main cause of morbidity and sometimes mortality in patients, as well as resulting in major economic burdens worldwide. However, due to the complex interactions between the host and the exacerbation agents, the mechanisms of exacerbation may vary considerably in different individuals under various triggers. Acute exacerbations are usually due to the presence of environmental factors such as allergens, pollutants, smoke, cold or dry air and pathogenic microbes in the airway (Gautier and Charpin, 2017; Viniol and Vogelmeier, 2018) . These agents elicit an immune response leading to infiltration of activated immune cells that further release inflammatory mediators that cause acute symptoms such as increased mucus production, cough, wheeze and shortness of breath. Among these agents, viral infection is one of the major drivers of asthma exacerbations accounting for up to 80-90% and 45-80% of exacerbations in children and adults respectively (Grissell et al., 2005; Xepapadaki and Papadopoulos, 2010; Jartti and Gern, 2017; Adeli et al., 2019) . Viral involvement in COPD exacerbation is also equally high, having been detected in 30-80% of acute COPD exacerbations (Kherad et al., 2010; Jafarinejad et al., 2017; Stolz et al., 2019) . Whilst the prevalence of viral exacerbations in CRS is still unclear, its prevalence is likely to be high due to the similar inflammatory nature of these diseases (Rowan et al., 2015; Tan et al., 2017) . One of the reasons for the involvement of respiratory viruses\\' in exacerbations is their ease of transmission and infection (Kutter et al., 2018) . In addition, the high diversity of the respiratory viruses may also contribute to exacerbations of different nature and severity (Busse et al., 2010; Costa et al., 2014; Jartti and Gern, 2017) . Hence, it is important to identify the exact mechanisms underpinning viral exacerbations in susceptible subjects in order to properly manage exacerbations via supplementary treatments that may alleviate the exacerbation symptoms or prevent severe exacerbations.\\n\\nWhile the lower airway is the site of dysregulated inflammation in most chronic airway inflammatory diseases, the upper airway remains the first point of contact with sources of exacerbation. Therefore, their interaction with the exacerbation agents may directly contribute to the subsequent responses in the lower airway, in line with the \"United Airway\" hypothesis. To elucidate the host airway interaction with viruses leading to exacerbations, we thus focus our review on recent findings of viral interaction with the upper airway. We compiled how viral induced changes to the upper airway may contribute to chronic airway inflammatory disease exacerbations, to provide a unified elucidation of the potential exacerbation mechanisms initiated from predominantly upper airway infections.\\n\\nDespite being a major cause of exacerbation, reports linking respiratory viruses to acute exacerbations only start to emerge in the late 1950s (Pattemore et al., 1992) ; with bacterial infections previously considered as the likely culprit for acute exacerbation (Stevens, 1953; Message and Johnston, 2002) . However, with the advent of PCR technology, more viruses were recovered during acute exacerbations events and reports implicating their role emerged in the late 1980s (Message and Johnston, 2002) . Rhinovirus (RV) and respiratory syncytial virus (RSV) are the predominant viruses linked to the development and exacerbation of chronic airway inflammatory diseases (Jartti and Gern, 2017) . Other viruses such as parainfluenza virus (PIV), influenza virus (IFV) and adenovirus (AdV) have also been implicated in acute exacerbations but to a much lesser extent (Johnston et al., 2005; Oliver et al., 2014; Ko et al., 2019) . More recently, other viruses including bocavirus (BoV), human metapneumovirus (HMPV), certain coronavirus (CoV) strains, a specific enterovirus (EV) strain EV-D68, human cytomegalovirus (hCMV) and herpes simplex virus (HSV) have been reported as contributing to acute exacerbations . The common feature these viruses share is that they can infect both the upper and/or lower airway, further increasing the inflammatory conditions in the diseased airway (Mallia and Johnston, 2006; Britto et al., 2017) .\\n\\nRespiratory viruses primarily infect and replicate within airway epithelial cells . During the replication process, the cells release antiviral factors and cytokines that alter local airway inflammation and airway niche (Busse et al., 2010) . In a healthy airway, the inflammation normally leads to type 1 inflammatory responses consisting of activation of an antiviral state and infiltration of antiviral effector cells. This eventually results in the resolution of the inflammatory response and clearance of the viral infection (Vareille et al., 2011; Braciale et al., 2012) . However, in a chronically inflamed airway, the responses against the virus may be impaired or aberrant, causing sustained inflammation and erroneous infiltration, resulting in the exacerbation of their symptoms (Mallia and Johnston, 2006; Dougherty and Fahy, 2009; Busse et al., 2010; Britto et al., 2017; Linden et al., 2019) . This is usually further compounded by the increased susceptibility of chronic airway inflammatory disease patients toward viral respiratory infections, thereby increasing the frequency of exacerbation as a whole (Dougherty and Fahy, 2009; Busse et al., 2010; Linden et al., 2019) . Furthermore, due to the different replication cycles and response against the myriad of respiratory viruses, each respiratory virus may also contribute to exacerbations via different mechanisms that may alter their severity. Hence, this review will focus on compiling and collating the current known mechanisms of viral-induced exacerbation of chronic airway inflammatory diseases; as well as linking the different viral infection pathogenesis to elucidate other potential ways the infection can exacerbate the disease. The review will serve to provide further understanding of viral induced exacerbation to identify potential pathways and pathogenesis mechanisms that may be targeted as supplementary care for management and prevention of exacerbation. Such an approach may be clinically significant due to the current scarcity of antiviral drugs for the management of viral-induced exacerbations. This will improve the quality of life of patients with chronic airway inflammatory diseases.\\n\\nOnce the link between viral infection and acute exacerbations of chronic airway inflammatory disease was established, there have been many reports on the mechanisms underlying the exacerbation induced by respiratory viral infection. Upon infecting the host, viruses evoke an inflammatory response as a means of counteracting the infection. Generally, infected airway epithelial cells release type I (IFNα/β) and type III (IFNλ) interferons, cytokines and chemokines such as IL-6, IL-8, IL-12, RANTES, macrophage inflammatory protein 1α (MIP-1α) and monocyte chemotactic protein 1 (MCP-1) (Wark and Gibson, 2006; Matsukura et al., 2013) . These, in turn, enable infiltration of innate immune cells and of professional antigen presenting cells (APCs) that will then in turn release specific mediators to facilitate viral targeting and clearance, including type II interferon (IFNγ), IL-2, IL-4, IL-5, IL-9, and IL-12 (Wark and Gibson, 2006; Singh et al., 2010; Braciale et al., 2012) . These factors heighten local inflammation and the infiltration of granulocytes, T-cells and B-cells (Wark and Gibson, 2006; Braciale et al., 2012) . The increased inflammation, in turn, worsens the symptoms of airway diseases.\\n\\nAdditionally, in patients with asthma and patients with CRS with nasal polyp (CRSwNP), viral infections such as RV and RSV promote a Type 2-biased immune response (Becker, 2006; Jackson et al., 2014; Jurak et al., 2018) . This amplifies the basal type 2 inflammation resulting in a greater release of IL-4, IL-5, IL-13, RANTES and eotaxin and a further increase in eosinophilia, a key pathological driver of asthma and CRSwNP (Wark and Gibson, 2006; Singh et al., 2010; Chung et al., 2015; Dunican and Fahy, 2015) . Increased eosinophilia, in turn, worsens the classical symptoms of disease and may further lead to life-threatening conditions due to breathing difficulties. On the other hand, patients with COPD and patients with CRS without nasal polyp (CRSsNP) are more neutrophilic in nature due to the expression of neutrophil chemoattractants such as CXCL9, CXCL10, and CXCL11 (Cukic et al., 2012; Brightling and Greening, 2019) . The pathology of these airway diseases is characterized by airway remodeling due to the presence of remodeling factors such as matrix metalloproteinases (MMPs) released from infiltrating neutrophils (Linden et al., 2019) . Viral infections in such conditions will then cause increase neutrophilic activation; worsening the symptoms and airway remodeling in the airway thereby exacerbating COPD, CRSsNP and even CRSwNP in certain cases (Wang et al., 2009; Tacon et al., 2010; Linden et al., 2019) .\\n\\nAn epithelial-centric alarmin pathway around IL-25, IL-33 and thymic stromal lymphopoietin (TSLP), and their interaction with group 2 innate lymphoid cells (ILC2) has also recently been identified (Nagarkar et al., 2012; Hong et al., 2018; Allinne et al., 2019) . IL-25, IL-33 and TSLP are type 2 inflammatory cytokines expressed by the epithelial cells upon injury to the epithelial barrier (Gabryelska et al., 2019; Roan et al., 2019) . ILC2s are a group of lymphoid cells lacking both B and T cell receptors but play a crucial role in secreting type 2 cytokines to perpetuate type 2 inflammation when activated (Scanlon and McKenzie, 2012; Li and Hendriks, 2013) . In the event of viral infection, cell death and injury to the epithelial barrier will also induce the expression of IL-25, IL-33 and TSLP, with heighten expression in an inflamed airway (Allakhverdi et al., 2007; Goldsmith et al., 2012; Byers et al., 2013; Shaw et al., 2013; Beale et al., 2014; Jackson et al., 2014; Uller and Persson, 2018; Ravanetti et al., 2019) . These 3 cytokines then work in concert to activate ILC2s to further secrete type 2 cytokines IL-4, IL-5, and IL-13 which further aggravate the type 2 inflammation in the airway causing acute exacerbation (Camelo et al., 2017) . In the case of COPD, increased ILC2 activation, which retain the capability of differentiating to ILC1, may also further augment the neutrophilic response and further aggravate the exacerbation (Silver et al., 2016) . Interestingly, these factors are not released to any great extent and do not activate an ILC2 response during viral infection in healthy individuals (Yan et al., 2016; Tan et al., 2018a) ; despite augmenting a type 2 exacerbation in chronically inflamed airways (Jurak et al., 2018) . These classical mechanisms of viral induced acute exacerbations are summarized in Figure 1 .\\n\\nAs integration of the virology, microbiology and immunology of viral infection becomes more interlinked, additional factors and FIGURE 1 | Current understanding of viral induced exacerbation of chronic airway inflammatory diseases. Upon virus infection in the airway, antiviral state will be activated to clear the invading pathogen from the airway. Immune response and injury factors released from the infected epithelium normally would induce a rapid type 1 immunity that facilitates viral clearance. However, in the inflamed airway, the cytokines and chemokines released instead augmented the inflammation present in the chronically inflamed airway, strengthening the neutrophilic infiltration in COPD airway, and eosinophilic infiltration in the asthmatic airway. The effect is also further compounded by the participation of Th1 and ILC1 cells in the COPD airway; and Th2 and ILC2 cells in the asthmatic airway.\\n\\nFrontiers in Cell and Developmental Biology | www.frontiersin.org mechanisms have been implicated in acute exacerbations during and after viral infection (Murray et al., 2006) . Murray et al. (2006) has underlined the synergistic effect of viral infection with other sensitizing agents in causing more severe acute exacerbations in the airway. This is especially true when not all exacerbation events occurred during the viral infection but may also occur well after viral clearance (Kim et al., 2008; Stolz et al., 2019) in particular the late onset of a bacterial infection (Singanayagam et al., 2018 (Singanayagam et al., , 2019a . In addition, viruses do not need to directly infect the lower airway to cause an acute exacerbation, as the nasal epithelium remains the primary site of most infections. Moreover, not all viral infections of the airway will lead to acute exacerbations, suggesting a more complex interplay between the virus and upper airway epithelium which synergize with the local airway environment in line with the \"united airway\" hypothesis (Kurai et al., 2013) . On the other hand, viral infections or their components persist in patients with chronic airway inflammatory disease (Kling et al., 2005; Wood et al., 2011; Ravi et al., 2019) . Hence, their presence may further alter the local environment and contribute to current and future exacerbations. Future studies should be performed using metagenomics in addition to PCR analysis to determine the contribution of the microbiome and mycobiome to viral infections. In this review, we highlight recent data regarding viral interactions with the airway epithelium that could also contribute to, or further aggravate, acute exacerbations of chronic airway inflammatory diseases.\\n\\nPatients with chronic airway inflammatory diseases have impaired or reduced ability of viral clearance (Hammond et al., 2015; McKendry et al., 2016; Akbarshahi et al., 2018; Gill et al., 2018; Wang et al., 2018; Singanayagam et al., 2019b) . Their impairment stems from a type 2-skewed inflammatory response which deprives the airway of important type 1 responsive CD8 cells that are responsible for the complete clearance of virusinfected cells (Becker, 2006; McKendry et al., 2016) . This is especially evident in weak type 1 inflammation-inducing viruses such as RV and RSV (Kling et al., 2005; Wood et al., 2011; Ravi et al., 2019) . Additionally, there are also evidence of reduced type I (IFNβ) and III (IFNλ) interferon production due to type 2-skewed inflammation, which contributes to imperfect clearance of the virus resulting in persistence of viral components, or the live virus in the airway epithelium (Contoli et al., 2006; Hwang et al., 2019; Wark, 2019) . Due to the viral components remaining in the airway, antiviral genes such as type I interferons, inflammasome activating factors and cytokines remained activated resulting in prolong airway inflammation (Wood et al., 2011; Essaidi-Laziosi et al., 2018) . These factors enhance granulocyte infiltration thus prolonging the exacerbation symptoms. Such persistent inflammation may also be found within DNA viruses such as AdV, hCMV and HSV, whose infections generally persist longer (Imperiale and Jiang, 2015) , further contributing to chronic activation of inflammation when they infect the airway (Yang et al., 2008; Morimoto et al., 2009; Imperiale and Jiang, 2015; Lan et al., 2016; Tan et al., 2016; Kowalski et al., 2017) . With that note, human papilloma virus (HPV), a DNA virus highly associated with head and neck cancers and respiratory papillomatosis, is also linked with the chronic inflammation that precedes the malignancies (de Visser et al., 2005; Gillison et al., 2012; Bonomi et al., 2014; Fernandes et al., 2015) . Therefore, the role of HPV infection in causing chronic inflammation in the airway and their association to exacerbations of chronic airway inflammatory diseases, which is scarcely explored, should be investigated in the future. Furthermore, viral persistence which lead to continuous expression of antiviral genes may also lead to the development of steroid resistance, which is seen with RV, RSV, and PIV infection (Chi et al., 2011; Ford et al., 2013; Papi et al., 2013) . The use of steroid to suppress the inflammation may also cause the virus to linger longer in the airway due to the lack of antiviral clearance (Kim et al., 2008; Hammond et al., 2015; Hewitt et al., 2016; McKendry et al., 2016; Singanayagam et al., 2019b) . The concomitant development of steroid resistance together with recurring or prolong viral infection thus added considerable burden to the management of acute exacerbation, which should be the future focus of research to resolve the dual complications arising from viral infection.\\n\\nOn the other end of the spectrum, viruses that induce strong type 1 inflammation and cell death such as IFV (Yan et al., 2016; Guibas et al., 2018) and certain CoV (including the recently emerged COVID-19 virus) (Tao et al., 2013; Yue et al., 2018; Zhu et al., 2020) , may not cause prolonged inflammation due to strong induction of antiviral clearance. These infections, however, cause massive damage and cell death to the epithelial barrier, so much so that areas of the epithelium may be completely absent post infection (Yan et al., 2016; Tan et al., 2019) . Factors such as RANTES and CXCL10, which recruit immune cells to induce apoptosis, are strongly induced from IFV infected epithelium (Ampomah et al., 2018; Tan et al., 2019) . Additionally, necroptotic factors such as RIP3 further compounds the cell deaths in IFV infected epithelium . The massive cell death induced may result in worsening of the acute exacerbation due to the release of their cellular content into the airway, further evoking an inflammatory response in the airway (Guibas et al., 2018) . Moreover, the destruction of the epithelial barrier may cause further contact with other pathogens and allergens in the airway which may then prolong exacerbations or results in new exacerbations. Epithelial destruction may also promote further epithelial remodeling during its regeneration as viral infection induces the expression of remodeling genes such as MMPs and growth factors . Infections that cause massive destruction of the epithelium, such as IFV, usually result in severe acute exacerbations with non-classical symptoms of chronic airway inflammatory diseases. Fortunately, annual vaccines are available to prevent IFV infections (Vasileiou et al., 2017; Zheng et al., 2018) ; and it is recommended that patients with chronic airway inflammatory disease receive their annual influenza vaccination as the best means to prevent severe IFV induced exacerbation.\\n\\nAnother mechanism that viral infections may use to drive acute exacerbations is the induction of vasodilation or tight junction opening factors which may increase the rate of infiltration. Infection with a multitude of respiratory viruses causes disruption of tight junctions with the resulting increased rate of viral infiltration. This also increases the chances of allergens coming into contact with airway immune cells. For example, IFV infection was found to induce oncostatin M (OSM) which causes tight junction opening (Pothoven et al., 2015; Tian et al., 2018) . Similarly, RV and RSV infections usually cause tight junction opening which may also increase the infiltration rate of eosinophils and thus worsening of the classical symptoms of chronic airway inflammatory diseases (Sajjan et al., 2008; Kast et al., 2017; Kim et al., 2018) . In addition, the expression of vasodilating factors and fluid homeostatic factors such as angiopoietin-like 4 (ANGPTL4) and bactericidal/permeabilityincreasing fold-containing family member A1 (BPIFA1) are also associated with viral infections and pneumonia development, which may worsen inflammation in the lower airway Akram et al., 2018) . These factors may serve as targets to prevent viral-induced exacerbations during the management of acute exacerbation of chronic airway inflammatory diseases.\\n\\nAnother recent area of interest is the relationship between asthma and COPD exacerbations and their association with the airway microbiome. The development of chronic airway inflammatory diseases is usually linked to specific bacterial species in the microbiome which may thrive in the inflamed airway environment (Diver et al., 2019) . In the event of a viral infection such as RV infection, the effect induced by the virus may destabilize the equilibrium of the microbiome present (Molyneaux et al., 2013; Kloepfer et al., 2014; Kloepfer et al., 2017; Jubinville et al., 2018; van Rijn et al., 2019) . In addition, viral infection may disrupt biofilm colonies in the upper airway (e.g., Streptococcus pneumoniae) microbiome to be release into the lower airway and worsening the inflammation (Marks et al., 2013; Chao et al., 2014) . Moreover, a viral infection may also alter the nutrient profile in the airway through release of previously inaccessible nutrients that will alter bacterial growth (Siegel et al., 2014; Mallia et al., 2018) . Furthermore, the destabilization is further compounded by impaired bacterial immune response, either from direct viral influences, or use of corticosteroids to suppress the exacerbation symptoms (Singanayagam et al., 2018 (Singanayagam et al., , 2019a Wang et al., 2018; Finney et al., 2019) . All these may gradually lead to more far reaching effect when normal flora is replaced with opportunistic pathogens, altering the inflammatory profiles (Teo et al., 2018) . These changes may in turn result in more severe and frequent acute exacerbations due to the interplay between virus and pathogenic bacteria in exacerbating chronic airway inflammatory diseases (Wark et al., 2013; Singanayagam et al., 2018) . To counteract these effects, microbiome-based therapies are in their infancy but have shown efficacy in the treatments of irritable bowel syndrome by restoring the intestinal microbiome (Bakken et al., 2011) . Further research can be done similarly for the airway microbiome to be able to restore the microbiome following disruption by a viral infection.\\n\\nViral infections can cause the disruption of mucociliary function, an important component of the epithelial barrier. Ciliary proteins FIGURE 2 | Changes in the upper airway epithelium contributing to viral exacerbation in chronic airway inflammatory diseases. The upper airway epithelium is the primary contact/infection site of most respiratory viruses. Therefore, its infection by respiratory viruses may have far reaching consequences in augmenting and synergizing current and future acute exacerbations. The destruction of epithelial barrier, mucociliary function and cell death of the epithelial cells serves to increase contact between environmental triggers with the lower airway and resident immune cells. The opening of tight junction increasing the leakiness further augments the inflammation and exacerbations. In addition, viral infections are usually accompanied with oxidative stress which will further increase the local inflammation in the airway. The dysregulation of inflammation can be further compounded by modulation of miRNAs and epigenetic modification such as DNA methylation and histone modifications that promote dysregulation in inflammation. Finally, the change in the local airway environment and inflammation promotes growth of pathogenic bacteria that may replace the airway microbiome. Furthermore, the inflammatory environment may also disperse upper airway commensals into the lower airway, further causing inflammation and alteration of the lower airway environment, resulting in prolong exacerbation episodes following viral infection. \\n\\nViral specific trait contributing to exacerbation mechanism (with literature evidence) Oxidative stress ROS production (RV, RSV, IFV, HSV)\\n\\nAs RV, RSV, and IFV were the most frequently studied viruses in chronic airway inflammatory diseases, most of the viruses listed are predominantly these viruses. However, the mechanisms stated here may also be applicable to other viruses but may not be listed as they were not implicated in the context of chronic airway inflammatory diseases exacerbation (see text for abbreviations).\\n\\nthat aid in the proper function of the motile cilia in the airways are aberrantly expressed in ciliated airway epithelial cells which are the major target for RV infection (Griggs et al., 2017) . Such form of secondary cilia dyskinesia appears to be present with chronic inflammations in the airway, but the exact mechanisms are still unknown (Peng et al., , 2019 Qiu et al., 2018) . Nevertheless, it was found that in viral infection such as IFV, there can be a change in the metabolism of the cells as well as alteration in the ciliary gene expression, mostly in the form of down-regulation of the genes such as dynein axonemal heavy chain 5 (DNAH5) and multiciliate differentiation And DNA synthesis associated cell cycle protein (MCIDAS) (Tan et al., 2018b . The recently emerged Wuhan CoV was also found to reduce ciliary beating in infected airway epithelial cell model (Zhu et al., 2020) . Furthermore, viral infections such as RSV was shown to directly destroy the cilia of the ciliated cells and almost all respiratory viruses infect the ciliated cells (Jumat et al., 2015; Yan et al., 2016; Tan et al., 2018a) . In addition, mucus overproduction may also disrupt the equilibrium of the mucociliary function following viral infection, resulting in symptoms of acute exacerbation (Zhu et al., 2009) . Hence, the disruption of the ciliary movement during viral infection may cause more foreign material and allergen to enter the airway, aggravating the symptoms of acute exacerbation and making it more difficult to manage. The mechanism of the occurrence of secondary cilia dyskinesia can also therefore be explored as a means to limit the effects of viral induced acute exacerbation.\\n\\nMicroRNAs (miRNAs) are short non-coding RNAs involved in post-transcriptional modulation of biological processes, and implicated in a number of diseases (Tan et al., 2014) . miRNAs are found to be induced by viral infections and may play a role in the modulation of antiviral responses and inflammation (Gutierrez et al., 2016; Deng et al., 2017; Feng et al., 2018) . In the case of chronic airway inflammatory diseases, circulating miRNA changes were found to be linked to exacerbation of the diseases (Wardzynska et al., 2020) . Therefore, it is likely that such miRNA changes originated from the infected epithelium and responding immune cells, which may serve to further dysregulate airway inflammation leading to exacerbations. Both IFV and RSV infections has been shown to increase miR-21 and augmented inflammation in experimental murine asthma models, which is reversed with a combination treatment of anti-miR-21 and corticosteroids (Kim et al., 2017) . IFV infection is also shown to increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections (Hsu et al., 2016 (Hsu et al., , 2017 . Conversely, miR-22 was shown to be suppressed in asthmatic epithelium in IFV infection which lead to aberrant epithelial response, contributing to exacerbations (Moheimani et al., 2018) . Other than these direct evidence of miRNA changes in contributing to exacerbations, an increased number of miRNAs and other non-coding RNAs responsible for immune modulation are found to be altered following viral infections (Globinska et al., 2014; Feng et al., 2018; Hasegawa et al., 2018) . Hence non-coding RNAs also presents as targets to modulate viral induced airway changes as a means of managing exacerbation of chronic airway inflammatory diseases. Other than miRNA modulation, other epigenetic modification such as DNA methylation may also play a role in exacerbation of chronic airway inflammatory diseases. Recent epigenetic studies have indicated the association of epigenetic modification and chronic airway inflammatory diseases, and that the nasal methylome was shown to be a sensitive marker for airway inflammatory changes (Cardenas et al., 2019; Gomez, 2019) . At the same time, it was also shown that viral infections such as RV and RSV alters DNA methylation and histone modifications in the airway epithelium which may alter inflammatory responses, driving chronic airway inflammatory diseases and exacerbations (McErlean et al., 2014; Pech et al., 2018; Caixia et al., 2019) . In addition, Spalluto et al. (2017) also showed that antiviral factors such as IFNγ epigenetically modifies the viral resistance of epithelial cells. Hence, this may indicate that infections such as RV and RSV that weakly induce antiviral responses may result in an altered inflammatory state contributing to further viral persistence and exacerbation of chronic airway inflammatory diseases (Spalluto et al., 2017) .\\n\\nFinally, viral infection can result in enhanced production of reactive oxygen species (ROS), oxidative stress and mitochondrial dysfunction in the airway epithelium (Kim et al., 2018; Mishra et al., 2018; Wang et al., 2018) . The airway epithelium of patients with chronic airway inflammatory diseases are usually under a state of constant oxidative stress which sustains the inflammation in the airway (Barnes, 2017; van der Vliet et al., 2018) . Viral infections of the respiratory epithelium by viruses such as IFV, RV, RSV and HSV may trigger the further production of ROS as an antiviral mechanism Aizawa et al., 2018; Wang et al., 2018) . Moreover, infiltrating cells in response to the infection such as neutrophils will also trigger respiratory burst as a means of increasing the ROS in the infected region. The increased ROS and oxidative stress in the local environment may serve as a trigger to promote inflammation thereby aggravating the inflammation in the airway (Tiwari et al., 2002) . A summary of potential exacerbation mechanisms and the associated viruses is shown in Figure 2 and Table 1 .\\n\\nWhile the mechanisms underlying the development and acute exacerbation of chronic airway inflammatory disease is extensively studied for ways to manage and control the disease, a viral infection does more than just causing an acute exacerbation in these patients. A viral-induced acute exacerbation not only induced and worsens the symptoms of the disease, but also may alter the management of the disease or confer resistance toward treatments that worked before. Hence, appreciation of the mechanisms of viral-induced acute exacerbations is of clinical significance to devise strategies to correct viral induce changes that may worsen chronic airway inflammatory disease symptoms. Further studies in natural exacerbations and in viral-challenge models using RNA-sequencing (RNA-seq) or single cell RNA-seq on a range of time-points may provide important information regarding viral pathogenesis and changes induced within the airway of chronic airway inflammatory disease patients to identify novel targets and pathway for improved management of the disease. Subsequent analysis of functions may use epithelial cell models such as the air-liquid interface, in vitro airway epithelial model that has been adapted to studying viral infection and the changes it induced in the airway (Yan et al., 2016; Boda et al., 2018; Tan et al., 2018a) . Animal-based diseased models have also been developed to identify systemic mechanisms of acute exacerbation (Shin, 2016; Gubernatorova et al., 2019; Tanner and Single, 2019) . Furthermore, the humanized mouse model that possess human immune cells may also serves to unravel the immune profile of a viral infection in healthy and diseased condition (Ito et al., 2019; Li and Di Santo, 2019) . For milder viruses, controlled in vivo human infections can be performed for the best mode of verification of the associations of the virus with the proposed mechanism of viral induced acute exacerbations . With the advent of suitable diseased models, the verification of the mechanisms will then provide the necessary continuation of improving the management of viral induced acute exacerbations.\\n\\nIn conclusion, viral-induced acute exacerbation of chronic airway inflammatory disease is a significant health and economic burden that needs to be addressed urgently. In view of the scarcity of antiviral-based preventative measures available for only a few viruses and vaccines that are only available for IFV infections, more alternative measures should be explored to improve the management of the disease. Alternative measures targeting novel viral-induced acute exacerbation mechanisms, especially in the upper airway, can serve as supplementary treatments of the currently available management strategies to augment their efficacy. New models including primary human bronchial or nasal epithelial cell cultures, organoids or precision cut lung slices from patients with airways disease rather than healthy subjects can be utilized to define exacerbation mechanisms. These mechanisms can then be validated in small clinical trials in patients with asthma or COPD. Having multiple means of treatment may also reduce the problems that arise from resistance development toward a specific treatment.', 'document_id': 2504}]}, {'paragraphs': [{'qas': [{'question': 'What clinical condition is caused by Hantaan virus?', 'id': 546, 'answers': [{'text': 'hemorrhagic fever with renal syndrome', 'answer_start': 605}], 'is_impossible': False}, {'question': 'What is the structure of Hantaan virus?', 'id': 547, 'answers': [{'text': 'enveloped, negative-sense RNA virus', 'answer_start': 24937}], 'is_impossible': False}, {'question': 'What the animal vector reservoir for Hantaan virus?', 'id': 548, 'answers': [{'text': 'rodents', 'answer_start': 25187}], 'is_impossible': False}, {'question': 'What diagnostic test is correlated with the severity of HFRS?', 'id': 549, 'answers': [{'text': 'plasma viral load', 'answer_start': 25635}], 'is_impossible': False}, {'question': 'How is Interferon used in practice?', 'id': 551, 'answers': [{'text': 'an antiviral medicine', 'answer_start': 25927}], 'is_impossible': False}, {'question': 'What genotypes are associated with the severity of HFRS?', 'id': 552, 'answers': [{'text': 'rs12252 C allele and CC genotype', 'answer_start': 26357}], 'is_impossible': False}, {'question': 'Which IFITM proteins have been shown to possess antiviral activity?', 'id': 554, 'answers': [{'text': 'IFITM1, 2, and 3', 'answer_start': 27495}], 'is_impossible': False}, {'question': 'What is the hypothesized mechanism by which IFITMs work?', 'id': 555, 'answers': [{'text': 'restrict viral infection at the stage of cellular entry', 'answer_start': 28051}], 'is_impossible': False}, {'question': 'What genotype causes truncation of the IFITM3 protein?', 'id': 556, 'answers': [{'text': 'rs12252 C allele', 'answer_start': 34574}], 'is_impossible': False}], 'context': 'Interferon-Induced Transmembrane Protein 3 Inhibits Hantaan Virus Infection, and Its Single Nucleotide Polymorphism rs12252 Influences the Severity of Hemorrhagic Fever with Renal Syndrome\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC5206578/\\n\\nSHA: 4328e18bdf9b52875c87f3f5ddb1911636a192d2\\n\\nAuthors: Xu-yang, Zheng; Pei-yu, Bian; Chuan-tao, Ye; Wei, Ye; Hong-wei, Ma; Kang, Tang; Chun-mei, Zhang; Ying-feng, Lei; Xin, Wei; Ping-zhong, Wang; Chang-xing, Huang; Xue-fan, Bai; Ying, Zhang; Zhan-sheng, Jia\\nDate: 2017-01-03\\nDOI: 10.3389/fimmu.2016.00535\\nLicense: cc-by\\n\\nAbstract: Hantaan virus (HTNV) causes hemorrhagic fever with renal syndrome (HFRS). Previous studies have identified interferon-induced transmembrane proteins (IFITMs) as an interferon-stimulated gene family. However, the role of IFITMs in HTNV infection is unclear. In this study, we observed that IFITM3 single nucleotide polymorphisms (SNP) rs12252 C allele and CC genotype associated with the disease severity and HTNV load in the plasma of HFRS patients. In vitro experiments showed that the truncated protein produced by the rs12252 C allele exhibited an impaired anti-HTNV activity. We also proved that IFITM3 was able to inhibit HTNV infection in both HUVEC and A549 cells by overexpression and RNAi assays, likely via a mechanism of inhibiting virus entry demonstrated by binding and entry assay. Localization of IFITM3 in late endosomes was also observed. In addition, we demonstrated that the transcription of IFITM3 is negatively regulated by an lncRNA negative regulator of interferon response (NRIR). Taken together, we conclude that IFITM3, negatively regulated by NRIR, inhibits HTNV infection, and its SNP rs12252 correlates with the plasma HTNV load and the disease severity of patients with HFRS.\\n\\nText: associates with the severity of disease, indicating the importance of viremia in the pathogenesis of HFRS (2) . Therefore, further studies of host factors limiting HTNV infection and influencing antiviral response as well as disease progression are clinically significant and timely. The human family of interferon-induced transmembrane proteins (IFITMs) was discovered 25 years ago to consist of interferon-stimulated genes (ISGs) (3) . This family includes five members, namely, IFITM1, 2, 3, 5, and 10, among which IFITM1, 2, and 3 possess antiviral activity (4) . Different IFITM proteins have different antiviral spectrum (5) . For example, IFITM3 has been shown to prevent influenza virus infection in vitro and in mice (6, 7) , and it also inhibits multiple viruses, including filoviruses, rhabdoviruses, flaviviruses, and even Ebola and Zika virus (7) (8) (9) (10) (11) . The antiviral mechanism of IFITM3 is thought to be the restriction of viral entry into cells (4, 12) . Single nucleotide polymorphisms (SNPs) are single nucleotide variations in a genetic sequence that occur at an appreciable frequency in the population. Several SNPs has been identified in IFITM3, among which the rs12252 site with C allele results in a N-terminal truncation of IFITM3 protein, leading to impaired inhibition of influenza virus in vitro (13, 14) . Notably, the frequencies of rs12252 C allele and CC genotype correlate with disease severity in patients infected with influenza virus (13, 15) . HTNV has been shown to induce a type I interferon response (though in later time postinfection) (16, 17) . While overexpression of IFITM1, 2, and 3 in Vero E6 cells has been reported to inhibit HTNV infection (18) , however, the effect of IFITMs on HTNV infection in human cell lines and its role in HFRS still remain unknown.\\n\\nLncRNA comprises a group of non-coding RNAs longer than 200 nt that function as gene regulators. Some lncRNAs have been shown to play a role in innate immunity (19) . Among them, negative regulator of interferon response (NRIR) (lncRNA NRIR, also known as lncRNA-CMPK2) is a non-coding ISG that negatively regulates IFITM1 and Mx1 expression in HCV infection (20) . Notably, IFITM3 is largely homologous to IFITM1, but the role of NRIR in the regulation of IFITM3 in HTNV infection remains unclear.\\n\\nIn the present study, we investigate the effect of IFTTM3 on the replication of HTNV and its role in the development of HFRS in humans. We provide primary evidence suggesting that IFITM3, regulated by NRIR, can inhibit HTNV infection and its SNP rs12252 correlates with the disease severity and viral load in patients with HFRS. This study expands our understanding of the antiviral activity of IFITM3 and enriches our knowledge of innate immune responses to HTNV infection.\\n\\nThis study was conducted in accordance with the recommendations of the biomedical research guidelines involving human participants established by the National Health and Family Planning Commission of China. The Institutional Ethics Committee of Tangdu Hospital approved this study. All subjects gave written informed consent in accordance with the Declaration of Helsinki. Before inclusion, all participants were informed of the study objectives and signed the consent form before blood samples and medical records were obtained.\\n\\nSixty-nine HFRS patients admitted into the Department of Infectious Diseases, Tangdu Hospital between October 2014 and March 2016 were enrolled in this study. All patients were Han Chinese. The diagnosis of HFRS was made based on typical symptoms and signs as well as positive IgM and IgG antibodies against HTNV in the serum assessed by enzyme linked immunosorbent assay (ELISA) in our department. The classification of HFRS severity and the exclusion criteria were described as follows (21) : white blood cells (WBC), platelets (PLT), blood urea nitrogen (BUN), serum creatinine (Scr), and heteromorphic lymphocytes that were tested by the Department of Clinical Laboratory (shown in Table 1 ).\\n\\nAccording to clinical symptoms and signs, such as fever, effusion, hemorrhage, edema, and renal function, the severity of HFRS can be classified as previously described (21): (1) mild patients were identified with mild renal failure without an obvious oliguric stage; (2) moderate patients were those with obvious symptoms of uremia, effusion (bulbar conjunctiva), hemorrhage (skin and mucous membrane), and renal failure with a typical oliguric stage; (3) severe patients had severe uremia, effusion (bulbar conjunctiva and either peritoneum or pleura), hemorrhage (skin and mucous membrane), and renal failure with oliguria (urine output, 50-500 ml/day) for ≤5 days or anuria (urine output, <50 ml/day) for ≤2 days; and (4) critical patients exhibited ≥1 of the following signs during the illness: refractory shock, visceral hemorrhage, heart failure, pulmonary edema, brain edema, severe secondary infection, and severe renal failure with oliguria (urine output, 50-500 ml/day) for >5 days, anuria (urine output, <50 ml/day) for >2 days, or a BUN level of >42.84 mmol/l. Due to the sample quantity required for SNP typing, the mild and moderate patients were assessed together in the mild group, and we combined severe and critical patients as severe group.\\n\\nThe exclusion criteria for this study were patients with: (1) any other kidney disease, (2) diabetes mellitus, (3) autoimmune disease, (4) hematological disease, (5) cardiovascular disease, (6) viral hepatitis (types A, B, C, D, or E), or (7) any other liver disease. In addition, no patients received corticosteroids or other immunomodulatory drugs during the study period (21) .\\n\\nGenomic DNA was extracted from the peripheral blood of patients using the PureGene DNA Isolation kit (Gentra Systems, Minneapolis, MN, USA). The region encompassing the human IFITM3 rs12252 were amplified by PCR (forward primer, 5′-GGAAACTGTTGAGAAACCGAA-3′ and reverse primer, 5′-CATACGCACCTTCACGGAGT-3′). The PCR products were purified and sequenced using an Applied Biosystems 3730xl DNA Analyzer (Thermo Scientific, Waltham, MA, USA). The allele frequencies and genotypes of healthy Han Chinese and other groups were obtained from the 1,000 genomes project (http:// www.1000genomes.org).\\n\\nThe HTNV load in plasma samples (collected during the acute phase) from 24 age-and sex-matched HFRS patients with different genotypes were measured using previously reported methods (2) . Briefly, viral RNA was extracted from the plasma of HFRS patients using Purelink Viral RNA/DNA Kits (Invitrogen, Carlsbad, CA, USA). The SuperScript III Platinum One-Step Quantitative RT-PCR System kit (Invitrogen, Carlsbad, CA, USA) was employed for the real-time RT-PCR assay. The primers and probe (provided by Sangon Biotech, Shanghai, China) were as follows: forward, 5′-TACAGAGGGAAATCAATGCC-3′, reverse, 5′-TGTTCAACTCATCTGGATCCTT-3′, and probe, 5′-(FAM) ATCCCTCACCTTCTGCCTGGCTATC (TAMRA)-3′. The synthetic S segment of the HTNV standard strain 76-118 RNA transcript was used as the quantitative calibrator. The external standard was the culture supernatant of Vero E6 cells infected with HTNV 76-118, which was quantified using synthetic quantitative calibrators. For each experiment, one aliquot of calibrated 76-118 standard was extracted in parallel with the clinical samples and serially 10-fold diluted with concentrations ranging from 10.56 to 2.56 log10 copies/ml. PCR was performed using an iQ5 Cycler (Bio-Rad, Hercules, CA, USA) with following conditions: 42°C for 15 min, 95°C for 2 min, and 50 cycles of 15 s at 95°C, 30 s at 53°C, and 30 s at 72°C. Fluorescence was read during the 72°C step of the final segment of every cycling program.\\n\\nHUVEC cells (ScienCell Research Laboratories, Carlsbad, CA, USA) were grown in ECM BulletKit (ScienCell Research Laboratories, Carlsbad, CA, USA) in a 5% CO2 incubator. A549 cells (ATCC Cat# CRM-CCL-185, RRID:CVCL_0023) were grown in our laboratory in DMEM with 10% FBS (Thermo Scientific, Waltham, MA, USA) in a 5% CO2 incubator. Cells were used within passage 10 after primary culture. HTNV strain 76-118 was cultured in Vero E6 cells (ATCC Cat# CRL-1586, RRID:CVCL_0574) in our laboratory and titrated using an immunofluorescence staining assay for HTNV nucleocapsid protein (NP) as previously described (22) . The TCID50 was 10 5 /ml, which was calculated using the Reed-Muench method.\\n\\nThe recombinant human IFN-α2a was obtained from PBL Interferon Source (Piscataway, NJ, USA) and dissolved in the buffer provided by the manufacturer (composition not disclosed). HUVEC and A549 cells were infected by incubation with HTNV as indicated moi at 37°C for 60 mins. Subsequently, the virus solution was removed and fresh medium added to the cell culture.\\n\\nCells were transfected with lentiviral vectors of c-myc-tagged IFITM1, IFITM2, IFITM3, and IFITM3 NΔ21 (purchased from GENECHEM, Shanghai, China) at a moi of 10. Puromycin (2 μg/ ml for HUVEC and 6 μg/ml for A549 cells) was used to create cell lines stably expressing IFITMs. Cells were transfected with control (scrambled) short interfering RNA (siRNA), IFITM1 siRNA, IFITM2 siRNA, or IFITM3 siRNA (10 nM) using Lipofectamine 3000 transfection reagent (Invitrogen, Carlsbad, CA, USA). SiRNAs were purchased from Origene (Rockville, MD, USA), and the sequences were not disclosed.\\n\\nTotal RNA was extracted using TRIzol reagent (Invitrogen, Carlsbad, CA, USA), and cDNA was synthesized using the K1622 kit (Thermo Scientific, Waltham, MA, USA). Quantitative realtime PCR (qPCR) was performed using SYBR Premix Ex Taq II (Takara Biotechnology Co., Dalian, China) with a Bio-Rad iQ5 cycler (Bio-Rad, Hercules, CA, USA). β-actin was used as the reference gene. The primers (Sangon Biotech, Shanghai, China) were as follows: IFITM1 (forward, 5′-ACTCCGTGAAGTCTAGGGACA-3′ and reverse, 5′-TGTCACAGAGCCGAATACCAG-3′); IFITM2 (forward, 5′-ATCCCGGTAACCCGATCAC-3′ and reverse, 5′-CTTCCTGTCCCTAGACTTCAC-3′); IFITM3 (forward, 5′-GGTCTTCGCTGGACACCAT-3′ and reverse, 5′-TGTCCCTAGACTTCACGGAGTA-3′); IFITM3 pre-mRNA (forward, 5′-CATAGCACGCGGCTCT CAG-3′ and reverse, 5′-CGTCGCCAACCATCTTCCTG-3′); HTNV S segment (forward, 5′-GCCTGGAGACCATCTGA AAG-3′ and reverse, 5′-AGTATCGGGACGACAAAGGA-3′); β-actin (forward, 5′-GCTACGTCGCCCTGGACTTC-3′ and reverse, 5′-GTCATAGTCCGCCTAGAAGC-3′); NRIR (forward, 5′-ATGGTTTTCTGGTGCCTTG-3′ and reverse, 5′-GGAGGTTAGAGGTGTCTGCTG-3′); NRAV (forward, 5′-TCACTACTGCCCCAGGATCA-3′ and reverse, 5′-GGTGGTCACAGGACTCATGG-3′).\\n\\nFor detection of miR-130a, cDNA was synthesized using the TaqMan microRNA reverse transcription kit (Invitrogen, Carlsbad, CA, USA) with a specific primer in gene-specific TaqMan assay kit (000454, Invitrogen, Carlsbad, CA, USA). MiR-130a level was determined using the gene-specific TaqMan assay kit (000454, Invitrogen, Carlsbad, CA, USA). U6 (001973, Invitrogen, Carlsbad, CA, USA) was used as an endogenous control (23) . Because the pre-mRNA levels can represent the initial transcription rate (24) , the primers used to detect the pre-mRNA of IFITM3 were designed targeting the intron of IFITM3 as previously described (25) . IFITM3 has two exons and one intron. For qPCR of IFITM3 pre-mRNA, the forward primers were positioned in the intron, and the reverse primer was positioned at the beginning of the second exon. For qPCR of IFITM3 mRNA, the forward primers were positioned in the first exon, and the reverse primer was positioned at the beginning of the second exon (24) . Because the basal expression of IFITM3 is low in A549 cells, we detected IFITM3 mRNA and pre-mRNA in A549 cells following IFN-α2a treatment (20 IU/ml for 12 h) after the overexpression of NRIR.\\n\\nCell lysates were prepared using Radio Immunoprecipitation Assay (RIPA) buffer (Sigma-Aldrich, St. Louis, MO, USA). Equal amounts of protein (20 μg protein/lane) were electrophoresed on a 10%-SDS-polyacrylamide gel and electrophoretically transferred to a polyvinylidene difluoride membrane (Millipore, Billerica, MA, USA). After blocking with 5% bovine serum albumin in Trisbuffered saline at room temperature for 1 h, the membranes were incubated with antibodies against IFITM1 (Proteintech Group Cat# 60074-1-Ig Lot# RRID:AB_2233405), IFITM2, IFITM3 (Proteintech Group Cat# 66081-1-Ig Lot# RRID:AB_11182821), and β-actin (Proteintech, Wuhan, Hubei, China) or HTNV NP (provided by the Department of Microbiology, The Fourth Military Medical University) overnight at 4°C. The membranes were then washed and incubated with HRP-conjugated IgG antibody (Cell Signaling Technology, Danvers, MA, USA) for 1 h at room temperature. The blots were developed using an enhanced chemiluminescence detection kit (Millipore, Billerica, MA, USA) and visualized using X-ray film. The blot densities were analyzed using the Quantity One software (Bio-Rad, Hercules, CA, USA). In addition, the RIPA buffer contains 50mM Tris (pH = 7.4), 150 mM NaCl, 1% Triton X-100, 1% sodium deoxycholate, 0.1% SDS. Protease inhibitor cocktail (Roche, Basel, Switzerland) was added before use.\\n\\nThe cells were cultured on glass coverslips (Millipore, Billerica, MA, USA) until they were semi-confluence and then incubated with HTNV for 60 min (moi = 1). At the indicated times post-HTNV infection, the cells were fixed with 4% PFA, incubated with 0.3% Triton X-100 (Sigma-Aldrich, St. Louis, MO, USA), and blocked with 5% BSA for 1 h. Following incubation with a mouse monoclonal antibody against c-myc-tag (Sigma-Aldrich, St. Louis, MO, USA, Sigma-Aldrich Cat# M5546), IFITM3, lysosome-associated membrane glycoprotein 1 (LAMP1, Cell Signaling Technology, Danvers, MA, USA), or HTNV NP at 37°C for 2 h, the cells were washed and incubated with anti-rabbit Ig conjugated to Alexa 555 and anti-mouse Ig conjugated to Alexa 488 (Abcam, Cambridge, MA, USA) secondary antibodies at room temperature for 1 h. The nuclei were counterstained with DAPI. An Olympus BX51 fluorescence microscope system and FV1000 confocal microscopy system (Olympus, Tokyo, Japan) were used to capture the images. hTnV binding and entry assay Cells transduced with IFITM3 or the empty vector were detached and washed extensively with cold PBS. The cells and HTNV were pre-chilled on ice for 30 min, mixed at a moi of 1 and incubated at 4°C for 1 h with rotation. Part of cells were washed extensively with ice-cold PBS and harvested for binding assay. Another part of cells were switched to 37°C for 2 h to allow HTNV entry. The HTNV that remained on the cell surface was removed by treatment with proteinase K (0.1 mg/ml, Thermo Scientific, Waltham, MA, USA). To achieve direct entry of HTNV into cells by virus-plasma membrane fusion as a positive control, cells were pre-chilled on ice for 10 min with 20 mM NH4Cl. Adsorption of HTNV (moi = 1) was performed at 4°C for 1 h. The cells were then washed, and fusion of the virus with the plasma membrane was triggered by incubation in low pH medium (20 mM sodium succinate, pH = 5.5) for 10 min at 37°C. Infection was followed by incubation for 2 h at 37°C in the presence of 20 mM NH4Cl (26) . qPCR analysis of the HTNV S segment was conducted to evaluate the influence of IFITMs on HTNV cell adhesion and HTNV entry.\\n\\nAll data were expressed as the mean ± SEM. Statistical analyses were performed using GraphPad Prism 5 (GraphPad Software, La Jolla, CA, USA). For association analysis of the rs12252 allele and genotype, Fisher\\'s exact test was used. Independent samples t-tests were used for normally distributed data. Differences among groups were determined by one-way analysis of variance (ANOVA) with repeated measures, followed by Bonferroni\\'s post hoc test. P < 0.05 was considered statistically significant.\\n\\nThe iFiTM3 snP rs12252 c allele and cc genotype associated with severe hFrs Disease and a higher Plasma hTnV load\\n\\nTo determine the clinical significance of IFITM3 SNP in HTNV infection, the relationship between rs12252 SNP and the severity of HFRS in 69 patients were examined. We sequenced 300 bp of the IFITM3 locus encompassing SNP rs12252 in all enrolled patients. Then, we stratified these patients into mild and severe, based on the clinical assessment as described in Section \"Material and Methods. \" We found a significantly higher frequency of the C allele among severe HFRS patients compared with the healthy Han Chinese in the 1,000 genomes sequence database (68.29 vs. 52.16%, P = 0.0076). The frequency of rs12252 C in severe patients was also higher than those mild patients (68.29 vs. 46.43%, P = 0.013, Figures 1A,B; Table 2 ). These data suggest that harboring rs12252 C allele increases the risk of suffering severe disease in HTNV-infected individuals, with an odds ratio (95% CI) of 2.124 (1.067-4.230). For genotypes, 43.90% of the severe patients carried the CC genotype, a significantly higher frequency than the control Han Chinese per 1,000 genomes sequence database (26.92% CC genotype, P = 0.03) as well as mildly infected patients (14.29%, P = 0.02, Figures 1A,B ; Table 2 ). However, mildly ill individuals did not exhibit a Fisher\\'s exact test was used to test the association between rs12252 allele/genotype and HFRS severity. (c) The plasma HTNV load in CC genotype patients and CT/TT genotype patients, tested by qRCR analysis. Each symbol represents one individual patient. Independent samples t-test was used to test the difference of HTNV load between groups. *P < 0.05, **P < 0.01. significantly different genotype frequencies compared with the Han Chinese population. In addition, we also found that patients with CC genotype had higher plasma viral load in acute phase ( Figure 1C) . These results support the notion that the normal function of IFITM3 plays a critical role in the immune response to HTNV infection in vivo, which has a substantial influence on the clinical manifestation of HFRS.\\n\\nPrevious studies reveal that the truncated IFITM3 protein produced by SNP rs12252 C allele (Figure 2A , the missing part stands for the truncated 21 amino acids from N-terminal of IFITM3, the intramembrane helix, and transmembrane helix was presented as boxes) leads to an impaired anti-influenza activity (14) . To test the functional significance of this polymorphism in HTNV infection, we transfected the majority T or minority C variant IFITM3 alleles that produce full-length or N-terminally truncated (NΔ21) proteins (Figure 2A ) with c-myc-tag to HUVEC and A549 cell using lentivirus vectors ( Figure 2B) . Then, we challenged the cells with HTNV at moi = 1 for 24 h and found that cells with the minority C variant were more susceptible to HTNV infection with higher expression of HTNV S segment ( Figure 2C ) and more positive of HTNV NP ( Figure S3 in Supplementary Material). Indeed, compared with the mock (empty vector)-infected control, the NΔ21 protein almost lost the ability to inhibit HTNV infection in both HUVEC and A549 cells (Figures 2C,D ; Figure S3 in Supplementary Material).\\n\\nTo determine the role of HTNV infection in inducing IFITMs, qPCR as well as Western blot of IFITMs were conducted in HUVEC and A549 cells (Figures 3A,B ; Figure S1 in Supplementary Material). While we observed only a moderate upregulation of IFITM1, 2, and 3 mRNA and protein in HUVECs after more than 24 h postinfection; IFITM1, 2, and 3 mRNA, however, were only transiently upregulated in A549 cells and caused no significant change in protein level.\\n\\nWe knocked down the IFITM1, 2, and 3 expression by transfection of their siRNAs individually. The effect of siRNAs on the expression of target IFITMs was tested by qPCR in HUVECs ( Figure S2 in Supplementary Material), and the effect of the best oligo against each IFITMs (IFITM1C, IFITM2A, IFITM3B) was tested by Western blot in A549 ( Figure 4A ) and HUVEC cells ( Figure 4B) . To assess the role of IFITMs in anti-HTNV effect of IFN-α2a, IFITM1, 2, and 3 were knocked down respectively by transfecting the above-tested oligoes for 12 h, followed by IFN-α2a treatment (20 IU/ml for another 12 h). The cells were then challenged with HTNV (moi = 1) for 24 h. The HTNV S segment and NP levels were significantly suppressed in both HUVEC and A549 cells in response to IFN-α2a treatment.\\n\\nNotably, knockdown of IFITM3 significantly restored the levels of HTNV S segment and NP in HUVEC and A549 cells. Knockdown of IFITM1 also partially restored the HTNV level in A549 cells (Figures 4C,D) . These results demonstrate that \\n\\nTo assess the anti-HTNV effects of IFITMs, we tested the effect of overexpressed IFITM1, 2, and 3 on HTNV infection. c-myc-tagged IFITM1, 2, and 3 were expressed in both HUVEC and A549 cells (Figure 5A) , and the cells were then challenged with HTNV (moi = 1) for 24 h. The HTNV S segment and NP levels were suppressed by IFITM3 overexpression in HUVEC cells (Figures 5B-D) . They were also suppressed by expressing IFITM1 and IFITM3 in A549 cells (Figures 5B-D) .\\n\\nThe inhibitory effect of IFITM3 was further confirmed by immunofluorescence analysis of HTNV NP ( Figure S3 in Supplementary Material). These results were in accordance with the above-described RNAi results. To determine whether IFITM3 inhibited HTNV binding or entry, HUVEC and A549 cells were incubated with HTNV (moi = 1) at 4°C for 1 h, unbound virus was washed away, and HTNV RNA collected at this time point represents HTNV bound to the cell surface. After virus binding, the cells were shifted to 37°C for 2 h to allow HTNV internalization, and HTNV RNA collected at this time point represents cell-internalized virus. As a positive control for inhibition of virus entry, we incubated a parallel group of cells with HTNV at pH = 5.5 as described in Section \"Materials and Methods.\" Expression of IFITM3 did not affect HTNV binding ( Figure 6A ) but significantly suppressed HTNV entry in both HUVEC and A549 cells (Figure 6B ). iFiTM3 Was Partially localized to laMP1 + late endosomes in the host cells\\n\\nTo elucidate the mechanism of IFITM3 function, we investigated the subcellular localization of IFTIM3 in the host cells. IFITM3 was found partially localized to LAMP1 + late endosomes in HUVECs analyzed by confocal microscopy (Figure 6C) . The co-localization of IFITM3 and LAMP1 + late endosomes had also been found in A549 cells (27) . Because the transfer into LAMP1 + late endosomes is a necessary step for HTNV entry (28) , this result provides an evidence for the anti-HTNV mechanism of IFITM3.\\n\\nLncRNA-and microRNA-mediated regulation of IFITM3 has been reported in several studies. We tested the change of previously reported regulators of IFITMs, such as NRAV, NRIR, and miR-130a after HTNV infection, among which NRIR was the only changed one (downregulated) after HTNV infection ( Figure 7A ; Figure S4 in Supplementary Material) in HUVEC. However, the expression of NRIR was unchanged in A549 cells. We overexpressed NRIR in HUVEC and A549 cells using the pcDNA3.1 vector ( Figure 7B) . Importantly, overexpression of NRIR significantly suppressed IFITM3 mRNA and pre-mRNA levels and facilitated HTNV infection in HUVEC and A549 cells (Figures 7C-E) . These data suggest that lncRNA NRIR is a negative regulator of IFITM3 transcription.\\n\\nHantaan virus is an enveloped, negative-sense RNA virus from the genus Hantavirus within the family Bunyaviridae. It causes HFRS, which is an important threat to public health worldwide. It is also a potential weapon for biological terrorism. Reservoir animals, usually rodents, are asymptomatic during persistent infection. Unlike in rodents, Hantavirus infection leads to HFRS and Hantavirus pulmonary syndrome (HPS) in humans (21) . The major clinical characteristics of HFRS include fever, hemorrhage, hypotension, and renal injury (1, 21) , causing severe manifestations and death in some cases. The current standard of care for HFRS relies on symptomatic and supportive treatment. It has been confirmed that the plasma viral load is associated with the severity of HFRS, implicating the importance of viremia in the pathogenesis of HFRS (2). However, no direct antiviral medications are currently available for this illness. Interferon is the key molecule for the antiviral response and has been used as an antiviral medicine in many diseases. It has been reported that HTNV infection induces a late type I interferon response (16) . However, the set of ISGs required for IFN-mediated inhibition of HTNV has not yet been identified. Therefore, identification of ISGs that are effective against HTNV is an attractive strategy to identify novel therapeutic targets.\\n\\nIn this study, we demonstrated a significantly high frequency of the rs12252 C allele and CC genotype among HFRS patients with severe illness compared with mildly infected individuals and the healthy Han Chinese. The rs12252 C allele and CC genotype are also found to be associated with higher plasma viral load in the early stage of HFRS. We also discovered that HTNV infection induces IFITMs, and the truncated IFITM3 produced by rs12252 C allele exhibits significantly decreased anti-HTNV activity. Interestingly, IFITM3 is found to restrict HTNV infection with a mechanism of cellular entry inhibition. Indeed, IFITM3 is localized to the late endosome in the host cells, which is a necessary structure for HTNV entry. In addition, we find that HTNV infection downregulated lncRNA NRIR 48 h post infection, which negatively regulates the transcription of IFITM3. Collectively, these results suggest that IFITM3, regulated by NRIR, inhibits HTNV infection, and its SNP rs12252 correlates with the disease severity and viral load in patients with HFRS.\\n\\nThe antiviral properties of IFITM proteins were identified in 2009 in an RNAi screen for host factors that influence influenza virus replication (29) . IFITM1, 2, and 3 have been demonstrated to possess antiviral activity in several studies. Everitt et al. demonstrated that the severity of influenza virus infection was greatly increased in IFITM3-knockout mice compared with wild-type animals (15) . Different IFITM members have also been confirmed to inhibit the cellular entry of multiple virus families (including filoviruses, rhabdoviruses, and flaviviruses) (7, (9) (10) (11) 30) . For example, HIV-1 and HCV infection are inhibited by IFITM1 (31) (32) (33) (34) . It is commonly believed that IFITMs restrict viral infection at the stage of cellular entry (12) . Recent studies suggested that the cellular location of different IFITMs may influence the range of viruses restricted by each protein (5) . IFITM1 prevents HCV entry because it colocalizes with CD81 on the cell membrane, interrupting the endocytosis of HCV particles (32) , whereas IFITM3 confines influenza virus in acidified endosomal compartments (27) . Notably, retrovirus subvirus particles (ISVPs), which do not require endosomal acidification for entry, are not inhibited by IFITM3 expression, suggesting that IFITM3 may function at the stage of endosomal entry (35) . Studies utilizing cell-cell fusion assays have suggested that IFITM3 blocks the entry of enveloped virus by preventing the fusion of the viral membrane with a limiting membrane of the host cell, either the plasma membrane and/or the endosomal membranes. The results obtained using two-photon laser scanning and fluorescence lifetime imaging (FLIM) suggest that IFITM proteins may reduce membrane fluidity and increase the spontaneous positive curvature in the outer leaflet of membranes (36) . In the present study, we demonstrated that IFN-α2a (20 U/ ml) significantly inhibited HTNV infection, siRNA-mediated depletion of IFITM3 alone significantly mitigated the antiviral effect of IFN-α2a in both HUVEC and A549 cells, whereas depletion of IFITM1 alone alleviated the antiviral effect of IFN-α2a in A549 cells. Overexpression of IFITM3 inhibited HTNV infection to HUVEC and A549 cells. IFITM1 overexpression was also effective in inhibition of HTNV in A549 cells. All these results suggest that IFITM3 is an important control factor under natural infection of HTNV. Our results also demonstrate that the effectiveness of IFITM3 is cell type-independent, which is in accordance with the results from similar viruses, such as RVFV (18) . Binding and entry assays, conducted by controlling the temperature and pH, showed that IFITM3 did not significantly influence HTNV binding but inhibited HTNV entry into HUVEC and A549 cells. Indeed, IFITM3 partially localizes to the late endosome of the host cells, which is a necessary site for the HTNV entry. However, we failed in tracking the transportation of HTNV in infected cells possibly due to the lack of fluorescence-labeled virus. In addition, IFITM1 also suppressed HTNV infection in A549 cells. The mechanism underlying anti-HTNV effect of IFITM1 remains undetermined and deserves to be further explored.\\n\\nAccording to a recent study on the three-dimensional structure of IFITM3, there is a C-terminal transmembrane α-helix and a two-N-terminal intramembrane α-helices (shown in Figure 2A as black boxes) in IFITM3 (14) . There are two splice variants that differ by the presence or absence of the first N-terminal 21 amino acids (deleted part, shown in Figure 2A as red dotted line). Several SNPs including 13 non-synonymous, 13 synonymous, 1 in-frame stop, and 1 splice site acceptoraltering have been reported in the translated IFITM3 sequence (15, 29) . Among them, the rare SNP rs12252C allele of IFITM3 truncates the protein as described above, leading to a reduced inhibition of influenza virus infection in A549 cells (15) . We demonstrated that truncated IFITM3 protein also loses the ability to inhibit HTNV infection in vitro. In Northern European patients hospitalized with seasonal influenza or pandemic influenza A virus, increased homozygosity of the minor C allele of SNP rs12252 in IFITM3 was observed (37) . In Chinese patients infected with influenza A (H1N1) virus, there was also an increased frequency of the C allele and CC genotype of SNP rs12252 (13) . In the present study, we observed an increased frequency of the C allele and CC genotype of SNP rs12252 in severely infected HFRS patients compared with healthy control and mildly affected patients. Patients carrying the CC genotype also had higher plasma viral loads compared with those with the CT/TT genotype. Given the impaired function of the IFITM3 protein produced by the C mutation, and the fact that enrichment of the rs12252 C allele in patients with severe disease and the higher viral load in patients with the CC genotype, this founding suggests that IFITM3 plays a pivotal role in the anti-HTNV response in vivo. We speculate that the much higher level of CC allele at healthy population of Han Chinese compared with Caucasians may place the Chinese at a higher risk for developing severe illness upon HTNV infection, which needs further investigation.\\n\\nLncRNAs are a group of non-coding RNAs longer than 200 nt that function as gene regulators, playing a role in regulating multiple cellular functions, including the innate immunity. For example, lncRNA NEAT1 is reported to be upregulated by influenza virus or PolyI:C stimulation, which promotes IL-8 expression (38) . lncRNA NRAV has been shown to negatively regulate the initial transcription of IFITM3 and Mx1 by affecting the histone modification of these genes (25) . lncRNA NRIR is a non-coding ISG, which has been reported to negatively regulate IFITM1 and Mx1 expression in HCV infection (20) . Mir-130a was also reported as a regulator of IFITM1 (23) . In this analysis, lncRNA NRIR was downregulated in HUVECs after HTNV infection for 48 h, overexpression of NRIR negatively regulates the initial transcription of IFITM3, evidenced by the decreased pre-mRNA as well as mRNA levels. NRIR overexpression also facilitated HTNV infection. These results indicate that the downregulation of NRIR after HTNV infection is possibly involved in the activation of innate immune responses against HTNV infection. We have also evaluated other potential regulators of IFITM3 before we choose NRIR for further study. Another lncRNA that can regulate IFITM3, i.e., NRAV (NR_038854), remained unchanged after HTNV infection ( Figures S4A,B in Supplementary Material). Additionally, miR-130a, which potentially regulate IFITM3, was also unaltered after HTNV infection ( Figures S4C,D in Supplementary Material).\\n\\nIn conclusion, this study revealed a critical role for IFITM3 in HTNV infection. We demonstrated, for the first time to our knowledge, that IFITM3 is a newly identified anti-HTNV ISG; its expression is negatively regulated by NRIR; and its antiviral activity seems via a mechanism of inhibiting virus entry into the host cells. In addition, we discovered that the IFITM3 SNP rs12252 C allele and CC genotype correlates with the plasma HTNV load and the severity of HFRS; and the rs12252 C allele produces a truncated IFITM3 protein (NΔ21) that attenuates its anti-HTNV function. These results provide new insights into the role of IFITM3 in regulating innate immunity against HTNV infection, which is the basis for identifying new targets to develop novel agent against this worldwide infectious disease. aUThOr cOnTribUTiOns ZX-y, BP-y, YC-t, and MH-w performed the experiments; WP-z, BX-f, LY-f, ZY, and JZ-s designed the research; HC-x, YW, and WX analyzed the data; TK and ZC-m provided clinical data; ZX-y and BP-y wrote the paper.', 'document_id': 2565}]}, {'paragraphs': [{'qas': [{'question': 'What can cause a slowing growth in daily reported deaths?', 'id': 806, 'answers': [{'text': 'significant impact\\nof interventions implemented several weeks earlier.', 'answer_start': 3126}], 'is_impossible': False}, {'question': 'When did Italy go into Iockdown?', 'id': 807, 'answers': [{'text': '11th March', 'answer_start': 3314}], 'is_impossible': False}, {'question': 'Approximately how many deaths have been averted in Western Europe with current non-pharmaceutical interventions remaining in place until the end of March?', 'id': 808, 'answers': [{'text': '59,000 deaths', 'answer_start': 3819}], 'is_impossible': False}, {'question': 'What are some non-pharmaceutical interventions?', 'id': 796, 'answers': [{'text': 'case isolation, the closure of schools and\\nuniversities, banning of mass gatherings and/or public events, and most recently, widescale social\\ndistancing including local and national Iockdowns', 'answer_start': 1878}], 'is_impossible': False}, {'question': 'Type of model used to infer the impact non-pharmaceutical interventions?', 'id': 797, 'answers': [{'text': 'semi-mechanistic Bayesian hierarchical model', 'answer_start': 2097}], 'is_impossible': False}, {'question': 'How can a semi-mechanistic Bayesian hierarchical model estimate changes to the reproductive number?', 'id': 801, 'answers': [{'text': 'calculating backwards from the deaths observed over time to estimate transmission that\\noccurred several weeks prior, allowing for the time lag between infection and death.', 'answer_start': 2468}], 'is_impossible': False}, {'question': 'What is a key assumption of a semi-mechanistic Bayesian hierarchical model used for coronavirus?', 'id': 803, 'answers': [{'text': 'each intervention has the same effect on the\\nreproduction number across countries and over time', 'answer_start': 2689}], 'is_impossible': False}, {'question': 'What is the time lag between when transmission changes occur and when their impact can be\\nobserved in trends in mortality?', 'id': 810, 'answers': [{'text': '2-3 weeks', 'answer_start': 4427}], 'is_impossible': False}, {'question': 'What is the key aim of non-pharmaceutical interventions?', 'id': 814, 'answers': [{'text': 'reduce the effective reproduction number', 'answer_start': 6703}], 'is_impossible': False}, {'question': 'What happens if the reproduction number is greater then 1?', 'id': 815, 'answers': [{'text': 'infections will increase (dependent on how much greater than 1 the reproduction number is)', 'answer_start': 7073}], 'is_impossible': False}, {'question': 'When did China introduce strict movement restrictions and other measures including case isolation and quarantine?', 'id': 820, 'answers': [{'text': '23rd January', 'answer_start': 7377}], 'is_impossible': False}, {'question': \"What was the result of China's interventions introduced in January?\", 'id': 823, 'answers': [{'text': 'downward trend in the number of\\nconfirmed new cases during February', 'answer_start': 7408}], 'is_impossible': False}, {'question': 'What are examples of social distancing?', 'id': 811, 'answers': [{'text': 'banning large gatherings and advising individuals not to socialize outside\\ntheir households', 'answer_start': 6147}], 'is_impossible': False}, {'question': \"What was the estimated effect on China's reproduction number in March based on the intervention introduced in January?\", 'id': 827, 'answers': [{'text': 'from\\naround 2-4 during the uncontrolled epidemic down to below 1', 'answer_start': 7632}], 'is_impossible': False}, {'question': 'Why is it challenging to estimate the reproduction number?', 'id': 829, 'answers': [{'text': 'high proportion of\\ninfections not detected by health systems”7 and regular changes in testing policies,', 'answer_start': 8467}], 'is_impossible': False}, {'question': 'What is an alternative way to estimate the course of an epidemic?', 'id': 837, 'answers': [{'text': 'back-calculate infections from\\nobserved deaths', 'answer_start': 9002}], 'is_impossible': False}, {'question': 'What is the estimated attack rate in Italy?', 'id': 840, 'answers': [{'text': '9.8% [3.2%-25%] of the population', 'answer_start': 12476}], 'is_impossible': False}, {'question': \"As of the end of March what is the proportion of Spain's population to be infected?\", 'id': 843, 'answers': [{'text': '15.0% (7.0 [18-19] million people)', 'answer_start': 12685}], 'is_impossible': False}, {'question': 'Which Western European country is estimated to have the lowest attack rate?', 'id': 844, 'answers': [{'text': 'Germany', 'answer_start': 12748}], 'is_impossible': False}, {'question': \"What is Austria's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\", 'id': 847, 'answers': [{'text': '1.1% [0.36%-3.1%]', 'answer_start': 13088}], 'is_impossible': False}, {'question': \"What is Belgium's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\", 'id': 848, 'answers': [{'text': '3.7% [1.3%-9.7%]', 'answer_start': 13114}], 'is_impossible': False}, {'question': \"What is Denmark's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\", 'id': 849, 'answers': [{'text': '1.1% [0.40%-3.1%]', 'answer_start': 13139}], 'is_impossible': False}, {'question': \"What is France's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\", 'id': 850, 'answers': [{'text': '3.0% [1.1%-7.4%]', 'answer_start': 13164}], 'is_impossible': False}, {'question': \"What is Germany's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\", 'id': 851, 'answers': [{'text': '0.72% [0.28%-1.8%]', 'answer_start': 13189}], 'is_impossible': False}, {'question': \"What is Italy's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\", 'id': 852, 'answers': [{'text': '9.8% [3.2%-26%]', 'answer_start': 13214}], 'is_impossible': False}, {'question': \"What is Norway's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\", 'id': 853, 'answers': [{'text': '0.41% [0.09%-1.2%]', 'answer_start': 13237}], 'is_impossible': False}, {'question': \"What is Spain's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\", 'id': 854, 'answers': [{'text': '15% [3.7%-41%]', 'answer_start': 13262}], 'is_impossible': False}, {'question': \"What is Spain's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\", 'id': 855, 'answers': [{'text': '3.1% [0.85%-8.4%]', 'answer_start': 13284}], 'is_impossible': False}, {'question': \"What is Switzerland's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\", 'id': 856, 'answers': [{'text': '3.2% [1.3%-7.6%]', 'answer_start': 13314}], 'is_impossible': False}, {'question': \"What is United Kingdom's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\", 'id': 857, 'answers': [{'text': '2.7% [1.2%-5.4%]', 'answer_start': 13346}], 'is_impossible': False}, {'question': 'What is the estimated averaged initial reproduction number [95% credible interval] for Western Europe as of 28th March?', 'id': 859, 'answers': [{'text': '3.87 [3.01-4.66]', 'answer_start': 13500}], 'is_impossible': False}, {'question': 'Why is there high uncertainty in estimating the impact of interventions against the coronavirus?', 'id': 862, 'answers': [{'text': 'too early to detect substantial\\nintervention impact in many countries at earlier stages of their epidemic', 'answer_start': 15611}], 'is_impossible': False}, {'question': 'Why is it statistically impossible to determine which individual intervention had the greatest effect on reducing the coronavirus reproduction number?', 'id': 869, 'answers': [{'text': 'close spacing of interventions in time', 'answer_start': 16248}], 'is_impossible': False}, {'question': 'What is estimated to drop immediately after an introduction of a non-pharmaceutical intervention?', 'id': 871, 'answers': [{'text': ' number of daily infections', 'answer_start': 16949}], 'is_impossible': False}, {'question': 'One way to understand the impact of interventions?', 'id': 873, 'answers': [{'text': 'fit a counterfactual model without the interventions\\nand compare this to the actual model', 'answer_start': 23274}], 'is_impossible': False}, {'question': 'What is a lockdown?', 'id': 1071, 'answers': [{'text': 'banned gatherings of more than 2 people', 'answer_start': 26840}], 'is_impossible': False}, {'question': 'Why is it hard to know the true incidence of infections number?', 'id': 1074, 'answers': [{'text': 'underreporting as well as systematic and country-specific changes in testing', 'answer_start': 28430}], 'is_impossible': False}, {'question': 'What is an example of a case-based measure against coronavirus?', 'id': 1076, 'answers': [{'text': 'strong recommendations or laws to the general\\npublic and primary care about self—isolation when showing COVID-19-like symptoms', 'answer_start': 29883}], 'is_impossible': False}, {'question': 'What is an example of containment phase intervention?', 'id': 1078, 'answers': [{'text': 'isolation if travelling back from an epidemic country', 'answer_start': 30331}], 'is_impossible': False}, {'question': 'What does a public events ban intervention mean?', 'id': 1079, 'answers': [{'text': 'banning all public events of more than 100 participants', 'answer_start': 30438}], 'is_impossible': False}, {'question': 'An example of social distancing', 'id': 1080, 'answers': [{'text': 'work from home wherever possible', 'answer_start': 30719}], 'is_impossible': False}, {'question': 'Example of social distancing', 'id': 1081, 'answers': [{'text': 'reducing use ofpublictransport', 'answer_start': 30753}], 'is_impossible': False}, {'question': 'What is a lockdown?', 'id': 1083, 'answers': [{'text': 'regulations/legislations regarding strict face-to-face social interaction', 'answer_start': 31133}], 'is_impossible': False}, {'question': 'What is an infection-to-onset-distribution?', 'id': 1084, 'answers': [{'text': 'time from infection to the onset\\nof symptoms', 'answer_start': 32609}], 'is_impossible': False}, {'question': 'What is an onset-to-death distribution?', 'id': 1085, 'answers': [{'text': 'time from the onset of symptoms to death', 'answer_start': 32688}], 'is_impossible': False}, {'question': 'What is the population-averaged infection fatality ratio?', 'id': 1086, 'answers': [{'text': 'adjusted for the age structure and contact patterns of\\neach country', 'answer_start': 32785}], 'is_impossible': False}, {'question': 'What is the time-varying reproduction number a function of?', 'id': 1095, 'answers': [{'text': ' initial reproduction number before interventions and the effect sizes from\\ninterventions', 'answer_start': 33246}], 'is_impossible': False}, {'question': 'What is an incubation period?', 'id': 1097, 'answers': [{'text': 'infection to onset of symptoms', 'answer_start': 37080}], 'is_impossible': False}, {'question': 'What is an incubation period?', 'id': 1098, 'answers': [{'text': 'infection-\\nto-onset', 'answer_start': 37114}], 'is_impossible': False}, {'question': 'What are the descriptive statistics for the onset-to-death for coronavirus?', 'id': 1102, 'answers': [{'text': 'Gamma distributed with a mean of 18.8 days and a\\ncoefficient of va riation 0.45', 'answer_start': 37366}], 'is_impossible': False}, {'question': 'What are the descriptive statistics for the incubation period for coronavirus?', 'id': 1100, 'answers': [{'text': 'distribution is Gamma distributed with mean 5.1 days and coefficient of variation\\n0.86', 'answer_start': 37238}], 'is_impossible': False}, {'question': \"What is the estimated infection-to-death distribution's mean for coronavirus?\", 'id': 1103, 'answers': [{'text': '23.9 days', 'answer_start': 37820}], 'is_impossible': False}, {'question': 'What is a well known approach to model the true number of infected individuals?', 'id': 1104, 'answers': [{'text': 'discrete renewal process', 'answer_start': 38407}], 'is_impossible': False}, {'question': 'What is a way to measure virus transmission?', 'id': 799, 'answers': [{'text': 'reproductive number', 'answer_start': 2265}], 'is_impossible': False}, {'question': 'What term describes when a majority of the population has built an immunity to a virus?', 'id': 819, 'answers': [{'text': 'herd immunity', 'answer_start': 7235}], 'is_impossible': False}, {'question': 'What is the estimated number of people in Italy infected with coronavirus by March 28th?', 'id': 839, 'answers': [{'text': 'cumulatively, 5.9 [1.9-15.2] million people have been\\ninfected as of March 28th', 'answer_start': 12370}], 'is_impossible': False}], 'context': 'Estimating the number of infections and the impact of non-\\npharmaceutical interventions on COVID-19 in 11 European countries\\n\\n30 March 2020 Imperial College COVID-19 Response Team\\n\\nSeth Flaxmani Swapnil Mishra*, Axel Gandy*, H JulietteT Unwin, Helen Coupland, Thomas A Mellan, Harrison\\nZhu, Tresnia Berah, Jeffrey W Eaton, Pablo N P Guzman, Nora Schmit, Lucia Cilloni, Kylie E C Ainslie, Marc\\nBaguelin, Isobel Blake, Adhiratha Boonyasiri, Olivia Boyd, Lorenzo Cattarino, Constanze Ciavarella, Laura Cooper,\\nZulma Cucunuba’, Gina Cuomo—Dannenburg, Amy Dighe, Bimandra Djaafara, Ilaria Dorigatti, Sabine van Elsland,\\nRich FitzJohn, Han Fu, Katy Gaythorpe, Lily Geidelberg, Nicholas Grassly, Wi|| Green, Timothy Hallett, Arran\\nHamlet, Wes Hinsley, Ben Jeffrey, David Jorgensen, Edward Knock, Daniel Laydon, Gemma Nedjati—Gilani, Pierre\\nNouvellet, Kris Parag, Igor Siveroni, Hayley Thompson, Robert Verity, Erik Volz, Caroline Walters, Haowei Wang,\\nYuanrong Wang, Oliver Watson, Peter Winskill, Xiaoyue Xi, Charles Whittaker, Patrick GT Walker, Azra Ghani,\\nChristl A. Donnelly, Steven Riley, Lucy C Okell, Michaela A C Vollmer, NeilM.Ferguson1and Samir Bhatt*1\\n\\nDepartment of Infectious Disease Epidemiology, Imperial College London\\n\\nDepartment of Mathematics, Imperial College London\\n\\nWHO Collaborating Centre for Infectious Disease Modelling\\n\\nMRC Centre for Global Infectious Disease Analysis\\n\\nAbdul LatifJameeI Institute for Disease and Emergency Analytics, Imperial College London\\nDepartment of Statistics, University of Oxford\\n\\n*Contributed equally 1Correspondence: nei|.ferguson@imperial.ac.uk, s.bhatt@imperial.ac.uk\\n\\nSummary\\n\\nFollowing the emergence of a novel coronavirus (SARS-CoV-Z) and its spread outside of China, Europe\\nis now experiencing large epidemics. In response, many European countries have implemented\\nunprecedented non-pharmaceutical interventions including case isolation, the closure of schools and\\nuniversities, banning of mass gatherings and/or public events, and most recently, widescale social\\ndistancing including local and national Iockdowns.\\n\\nIn this report, we use a semi-mechanistic Bayesian hierarchical model to attempt to infer the impact\\nof these interventions across 11 European countries. Our methods assume that changes in the\\nreproductive number— a measure of transmission - are an immediate response to these interventions\\nbeing implemented rather than broader gradual changes in behaviour. Our model estimates these\\nchanges by calculating backwards from the deaths observed over time to estimate transmission that\\noccurred several weeks prior, allowing for the time lag between infection and death.\\n\\nOne of the key assumptions of the model is that each intervention has the same effect on the\\nreproduction number across countries and over time. This allows us to leverage a greater amount of\\ndata across Europe to estimate these effects. It also means that our results are driven strongly by the\\ndata from countries with more advanced epidemics, and earlier interventions, such as Italy and Spain.\\nWe find that the slowing growth in daily reported deaths in Italy is consistent with a significant impact\\nof interventions implemented several weeks earlier. In Italy, we estimate that the effective\\nreproduction number, Rt, dropped to close to 1 around the time of Iockdown (11th March), although\\nwith a high level of uncertainty.\\n\\nOverall, we estimate that countries have managed to reduce their reproduction number. Our\\nestimates have wide credible intervals and contain 1 for countries that have implemented a||\\ninterventions considered in our analysis. This means that the reproduction number may be above or\\nbelow this value. With current interventions remaining in place to at least the end of March, we\\nestimate that interventions across all 11 countries will have averted 59,000 deaths up to 31 March\\n[95% credible interval 21,000-120,000]. Many more deaths will be averted through ensuring that\\ninterventions remain in place until transmission drops to low levels. We estimate that, across all 11\\ncountries between 7 and 43 million individuals have been infected with SARS-CoV-Z up to 28th March,\\nrepresenting between 1.88% and 11.43% ofthe population. The proportion of the population infected\\n\\nto date — the attack rate - is estimated to be highest in Spain followed by Italy and lowest in Germany\\nand Norway, reflecting the relative stages of the epidemics.\\n\\nGiven the lag of 2-3 weeks between when transmission changes occur and when their impact can be\\nobserved in trends in mortality, for most of the countries considered here it remains too early to be\\ncertain that recent interventions have been effective. If interventions in countries at earlier stages of\\ntheir epidemic, such as Germany or the UK, are more or less effective than they were in the countries\\nwith advanced epidemics, on which our estimates are largely based, or if interventions have improved\\nor worsened over time, then our estimates of the reproduction number and deaths averted would\\nchange accordingly. It is therefore critical that the current interventions remain in place and trends in\\ncases and deaths are closely monitored in the coming days and weeks to provide reassurance that\\ntransmission of SARS-Cov-Z is slowing.\\n\\nSUGGESTED CITATION\\n\\nSeth Flaxman, Swapnil Mishra, Axel Gandy et 0/. Estimating the number of infections and the impact of non—\\npharmaceutical interventions on COVID—19 in 11 European countries. Imperial College London (2020), doi:\\nhttps://doi.org/10.25561/77731\\n\\n1 Introduction\\n\\nFollowing the emergence of a novel coronavirus (SARS-CoV-Z) in Wuhan, China in December 2019 and\\nits global spread, large epidemics of the disease, caused by the virus designated COVID-19, have\\nemerged in Europe. In response to the rising numbers of cases and deaths, and to maintain the\\ncapacity of health systems to treat as many severe cases as possible, European countries, like those in\\nother continents, have implemented or are in the process of implementing measures to control their\\nepidemics. These large-scale non-pharmaceutical interventions vary between countries but include\\nsocial distancing (such as banning large gatherings and advising individuals not to socialize outside\\ntheir households), border closures, school closures, measures to isolate symptomatic individuals and\\ntheir contacts, and large-scale lockdowns of populations with all but essential internal travel banned.\\nUnderstanding firstly, whether these interventions are having the desired impact of controlling the\\nepidemic and secondly, which interventions are necessary to maintain control, is critical given their\\nlarge economic and social costs.\\n\\nThe key aim ofthese interventions is to reduce the effective reproduction number, Rt, ofthe infection,\\na fundamental epidemiological quantity representing the average number of infections, at time t, per\\ninfected case over the course of their infection. Ith is maintained at less than 1, the incidence of new\\ninfections decreases, ultimately resulting in control of the epidemic. If Rt is greater than 1, then\\ninfections will increase (dependent on how much greater than 1 the reproduction number is) until the\\nepidemic peaks and eventually declines due to acquisition of herd immunity.\\n\\nIn China, strict movement restrictions and other measures including case isolation and quarantine\\nbegan to be introduced from 23rd January, which achieved a downward trend in the number of\\nconfirmed new cases during February, resulting in zero new confirmed indigenous cases in Wuhan by\\nMarch 19th. Studies have estimated how Rt changed during this time in different areas ofChina from\\naround 2-4 during the uncontrolled epidemic down to below 1, with an estimated 7-9 fold decrease\\nin the number of daily contacts per person.1\\'2 Control measures such as social distancing, intensive\\ntesting, and contact tracing in other countries such as Singapore and South Korea have successfully\\nreduced case incidence in recent weeks, although there is a riskthe virus will spread again once control\\nmeasures are relaxed.3\\'4\\n\\nThe epidemic began slightly laterin Europe, from January or later in different regions.5 Countries have\\nimplemented different combinations of control measures and the level of adherence to government\\nrecommendations on social distancing is likely to vary between countries, in part due to different\\nlevels of enforcement.\\n\\nEstimating reproduction numbers for SARS-CoV-Z presents challenges due to the high proportion of\\ninfections not detected by health systems”7 and regular changes in testing policies, resulting in\\ndifferent proportions of infections being detected over time and between countries. Most countries\\nso far only have the capacity to test a small proportion of suspected cases and tests are reserved for\\nseverely ill patients or for high-risk groups (e.g. contacts of cases). Looking at case data, therefore,\\ngives a systematically biased view of trends.\\n\\nAn alternative way to estimate the course of the epidemic is to back-calculate infections from\\nobserved deaths. Reported deaths are likely to be more reliable, although the early focus of most\\nsurveillance systems on cases with reported travel histories to China may mean that some early deaths\\nwill have been missed. Whilst the recent trends in deaths will therefore be informative, there is a time\\nlag in observing the effect of interventions on deaths since there is a 2-3-week period between\\ninfection, onset of symptoms and outcome.\\n\\nIn this report, we fit a novel Bayesian mechanistic model of the infection cycle to observed deaths in\\n11 European countries, inferring plausible upper and lower bounds (Bayesian credible intervals) of the\\ntotal populations infected (attack rates), case detection probabilities, and the reproduction number\\nover time (Rt). We fit the model jointly to COVID-19 data from all these countries to assess whether\\nthere is evidence that interventions have so far been successful at reducing Rt below 1, with the strong\\nassumption that particular interventions are achieving a similar impact in different countries and that\\nthe efficacy of those interventions remains constant over time. The model is informed more strongly\\nby countries with larger numbers of deaths and which implemented interventions earlier, therefore\\nestimates of recent Rt in countries with more recent interventions are contingent on similar\\nintervention impacts. Data in the coming weeks will enable estimation of country-specific Rt with\\ngreater precision.\\n\\nModel and data details are presented in the appendix, validation and sensitivity are also presented in\\nthe appendix, and general limitations presented below in the conclusions.\\n\\n2 Results\\n\\nThe timing of interventions should be taken in the context of when an individual country’s epidemic\\nstarted to grow along with the speed with which control measures were implemented. Italy was the\\nfirst to begin intervention measures, and other countries followed soon afterwards (Figure 1). Most\\ninterventions began around 12th-14th March. We analyzed data on deaths up to 28th March, giving a\\n2-3-week window over which to estimate the effect of interventions. Currently, most countries in our\\nstudy have implemented all major non-pharmaceutical interventions.\\n\\nFor each country, we model the number of infections, the number of deaths, and Rt, the effective\\nreproduction number over time, with Rt changing only when an intervention is introduced (Figure 2-\\n12). Rt is the average number of secondary infections per infected individual, assuming that the\\ninterventions that are in place at time t stay in place throughout their entire infectious period. Every\\ncountry has its own individual starting reproduction number Rt before interventions take place.\\nSpecific interventions are assumed to have the same relative impact on Rt in each country when they\\nwere introduced there and are informed by mortality data across all countries.\\n\\n\\nFigure l: Intervention timings for the 11 European countries included in the analysis. For further\\ndetails see Appendix 8.6.\\n\\n2.1 Estimated true numbers of infections and current attack rates\\n\\nIn all countries, we estimate there are orders of magnitude fewer infections detected (Figure 2) than\\ntrue infections, mostly likely due to mild and asymptomatic infections as well as limited testing\\ncapacity. In Italy, our results suggest that, cumulatively, 5.9 [1.9-15.2] million people have been\\ninfected as of March 28th, giving an attack rate of 9.8% [3.2%-25%] of the population (Table 1). Spain\\nhas recently seen a large increase in the number of deaths, and given its smaller population, our model\\nestimates that a higher proportion of the population, 15.0% (7.0 [18-19] million people) have been\\ninfected to date. Germany is estimated to have one of the lowest attack rates at 0.7% with 600,000\\n[240,000-1,500,000] people infected.\\n\\nImperial College COVID-19 Response Team\\n\\nTable l: Posterior model estimates of percentage of total population infected as of 28th March 2020.\\n\\nCountry % of total population infected (mean [95% credible intervall)\\nAustria 1.1% [0.36%-3.1%]\\nBelgium 3.7% [1.3%-9.7%]\\nDenmark 1.1% [0.40%-3.1%]\\nFrance 3.0% [1.1%-7.4%]\\nGermany 0.72% [0.28%-1.8%]\\nItaly 9.8% [3.2%-26%]\\nNorway 0.41% [0.09%-1.2%]\\nSpain 15% [3.7%-41%]\\nSweden 3.1% [0.85%-8.4%]\\nSwitzerland 3.2% [1.3%-7.6%]\\nUnited Kingdom 2.7% [1.2%-5.4%]\\n\\n2.2 Reproduction numbers and impact of interventions\\n\\nAveraged across all countries, we estimate initial reproduction numbers of around 3.87 [3.01-4.66],\\nwhich is in line with other estimates.1\\'8 These estimates are informed by our choice of serial interval\\ndistribution and the initial growth rate of observed deaths. A shorter assumed serial interval results in\\nlower starting reproduction numbers (Appendix 8.4.2, Appendix 8.4.6). The initial reproduction\\nnumbers are also uncertain due to (a) importation being the dominant source of new infections early\\nin the epidemic, rather than local transmission (b) possible under-ascertainment in deaths particularly\\nbefore testing became widespread.\\n\\nWe estimate large changes in Rt in response to the combined non-pharmaceutical interventions. Our\\nresults, which are driven largely by countries with advanced epidemics and larger numbers of deaths\\n(e.g. Italy, Spain), suggest that these interventions have together had a substantial impact on\\ntransmission, as measured by changes in the estimated reproduction number Rt. Across all countries\\nwe find current estimates of Rt to range from a posterior mean of 0.97 [0.14-2.14] for Norway to a\\nposterior mean of2.64 [1.40-4.18] for Sweden, with an average of 1.43 across the 11 country posterior\\nmeans, a 64% reduction compared to the pre-intervention values. We note that these estimates are\\ncontingent on intervention impact being the same in different countries and at different times. In all\\ncountries but Sweden, under the same assumptions, we estimate that the current reproduction\\nnumber includes 1 in the uncertainty range. The estimated reproduction number for Sweden is higher,\\nnot because the mortality trends are significantly different from any other country, but as an artefact\\nof our model, which assumes a smaller reduction in Rt because no full lockdown has been ordered so\\nfar. Overall, we cannot yet conclude whether current interventions are sufficient to drive Rt below 1\\n(posterior probability of being less than 1.0 is 44% on average across the countries). We are also\\nunable to conclude whether interventions may be different between countries or over time.\\n\\nThere remains a high level of uncertainty in these estimates. It is too early to detect substantial\\nintervention impact in many countries at earlier stages of their epidemic (e.g. Germany, UK, Norway).\\nMany interventions have occurred only recently, and their effects have not yet been fully observed\\ndue to the time lag between infection and death. This uncertainty will reduce as more data become\\navailable. For all countries, our model fits observed deaths data well (Bayesian goodness of fit tests).\\nWe also found that our model can reliably forecast daily deaths 3 days into the future, by withholding\\nthe latest 3 days of data and comparing model predictions to observed deaths (Appendix 8.3).\\n\\nThe close spacing of interventions in time made it statistically impossible to determine which had the\\ngreatest effect (Figure 1, Figure 4). However, when doing a sensitivity analysis (Appendix 8.4.3) with\\nuninformative prior distributions (where interventions can increase deaths) we find similar impact of\\n\\nImperial College COVID-19 Response Team\\n\\ninterventions, which shows that our choice of prior distribution is not driving the effects we see in the\\n\\nmain analysis.\\n\\n\\nFigure 2: Country-level estimates of infections, deaths and Rt. Left: daily number of infections, brown\\nbars are reported infections, blue bands are predicted infections, dark blue 50% credible interval (CI),\\nlight blue 95% CI. The number of daily infections estimated by our model drops immediately after an\\nintervention, as we assume that all infected people become immediately less infectious through the\\nintervention. Afterwards, if the Rt is above 1, the number of infections will starts growing again.\\nMiddle: daily number of deaths, brown bars are reported deaths, blue bands are predicted deaths, CI\\nas in left plot. Right: time-varying reproduction number Rt, dark green 50% CI, light green 95% CI.\\nIcons are interventions shown at the time they occurred.\\n\\nImperial College COVID-19 Response Team\\n\\nTable 2: Totalforecasted deaths since the beginning of the epidemic up to 31 March in our model\\nand in a counterfactual model (assuming no intervention had taken place). Estimated averted deaths\\nover this time period as a result of the interventions. Numbers in brackets are 95% credible intervals.\\n \\n\\n2.3 Estimated impact of interventions on deaths\\n\\nTable 2 shows total forecasted deaths since the beginning of the epidemic up to and including 31\\nMarch under ourfitted model and under the counterfactual model, which predicts what would have\\nhappened if no interventions were implemented (and R, = R0 i.e. the initial reproduction number\\nestimated before interventions). Again, the assumption in these predictions is that intervention\\nimpact is the same across countries and time. The model without interventions was unable to capture\\nrecent trends in deaths in several countries, where the rate of increase had clearly slowed (Figure 3).\\nTrends were confirmed statistically by Bayesian leave-one-out cross-validation and the widely\\napplicable information criterion assessments —WA|C).\\n\\nBy comparing the deaths predicted under the model with no interventions to the deaths predicted in\\nour intervention model, we calculated the total deaths averted up to the end of March. We find that,\\nacross 11 countries, since the beginning of the epidemic, 59,000 [21,000-120,000] deaths have been\\naverted due to interventions. In Italy and Spain, where the epidemic is advanced, 38,000 [13,000-\\n84,000] and 16,000 [5,400-35,000] deaths have been averted, respectively. Even in the UK, which is\\nmuch earlier in its epidemic, we predict 370 [73-1,000] deaths have been averted.\\n\\nThese numbers give only the deaths averted that would have occurred up to 31 March. lfwe were to\\ninclude the deaths of currently infected individuals in both models, which might happen after 31\\nMarch, then the deaths averted would be substantially higher.\\n\\n\\nFigure 3: Daily number of confirmed deaths, predictions (up to 28 March) and forecasts (after) for (a)\\nItaly and (b) Spain from our model with interventions (blue) and from the no interventions\\ncounterfactual model (pink); credible intervals are shown one week into the future. Other countries\\nare shown in Appendix 8.6.\\n\\n\\n03/0 25% 50% 753% 100%\\n(no effect on transmissibility) (ends transmissibility\\nRelative % reduction in R.\\n\\nFigure 4: Our model includes five covariates for governmental interventions, adjusting for whether\\nthe intervention was the first one undertaken by the government in response to COVID-19 (red) or\\nwas subsequent to other interventions (green). Mean relative percentage reduction in Rt is shown\\nwith 95% posterior credible intervals. If 100% reduction is achieved, Rt = 0 and there is no more\\ntransmission of COVID-19. No effects are significantly different from any others, probably due to the\\nfact that many interventions occurred on the same day or within days of each other as shown in\\nFigure l.\\n\\n3 Discussion\\n\\nDuring this early phase of control measures against the novel coronavirus in Europe, we analyze trends\\nin numbers of deaths to assess the extent to which transmission is being reduced. Representing the\\nCOVlD-19 infection process using a semi-mechanistic, joint, Bayesian hierarchical model, we can\\nreproduce trends observed in the data on deaths and can forecast accurately over short time horizons.\\n\\nWe estimate that there have been many more infections than are currently reported. The high level\\nof under-ascertainment of infections that we estimate here is likely due to the focus on testing in\\nhospital settings rather than in the community. Despite this, only a small minority of individuals in\\neach country have been infected, with an attack rate on average of 4.9% [l.9%-ll%] with considerable\\nvariation between countries (Table 1). Our estimates imply that the populations in Europe are not\\nclose to herd immunity (\"50-75% if R0 is 2-4). Further, with Rt values dropping substantially, the rate\\nof acquisition of herd immunity will slow down rapidly. This implies that the virus will be able to spread\\nrapidly should interventions be lifted. Such estimates of the attack rate to date urgently need to be\\nvalidated by newly developed antibody tests in representative population surveys, once these become\\navailable.\\n\\nWe estimate that major non-pharmaceutical interventions have had a substantial impact on the time-\\nvarying reproduction numbers in countries where there has been time to observe intervention effects\\non trends in deaths (Italy, Spain). lfadherence in those countries has changed since that initial period,\\nthen our forecast of future deaths will be affected accordingly: increasing adherence over time will\\nhave resulted in fewer deaths and decreasing adherence in more deaths. Similarly, our estimates of\\nthe impact ofinterventions in other countries should be viewed with caution if the same interventions\\nhave achieved different levels of adherence than was initially the case in Italy and Spain.\\n\\nDue to the implementation of interventions in rapid succession in many countries, there are not\\nenough data to estimate the individual effect size of each intervention, and we discourage attributing\\n\\nassociations to individual intervention. In some cases, such as Norway, where all interventions were\\nimplemented at once, these individual effects are by definition unidentifiable. Despite this, while\\nindividual impacts cannot be determined, their estimated joint impact is strongly empirically justified\\n(see Appendix 8.4 for sensitivity analysis). While the growth in daily deaths has decreased, due to the\\nlag between infections and deaths, continued rises in daily deaths are to be expected for some time.\\n\\nTo understand the impact of interventions, we fit a counterfactual model without the interventions\\nand compare this to the actual model. Consider Italy and the UK - two countries at very different stages\\nin their epidemics. For the UK, where interventions are very recent, much of the intervention strength\\nis borrowed from countries with older epidemics. The results suggest that interventions will have a\\nlarge impact on infections and deaths despite counts of both rising. For Italy, where far more time has\\npassed since the interventions have been implemented, it is clear that the model without\\ninterventions does not fit well to the data, and cannot explain the sub-linear (on the logarithmic scale)\\nreduction in deaths (see Figure 10).\\n\\nThe counterfactual model for Italy suggests that despite mounting pressure on health systems,\\ninterventions have averted a health care catastrophe where the number of new deaths would have\\nbeen 3.7 times higher (38,000 deaths averted) than currently observed. Even in the UK, much earlier\\nin its epidemic, the recent interventions are forecasted to avert 370 total deaths up to 31 of March.\\n\\n4 Conclusion and Limitations\\n\\nModern understanding of infectious disease with a global publicized response has meant that\\nnationwide interventions could be implemented with widespread adherence and support. Given\\nobserved infection fatality ratios and the epidemiology of COVlD-19, major non-pharmaceutical\\ninterventions have had a substantial impact in reducing transmission in countries with more advanced\\nepidemics. It is too early to be sure whether similar reductions will be seen in countries at earlier\\nstages of their epidemic. While we cannot determine which set of interventions have been most\\nsuccessful, taken together, we can already see changes in the trends of new deaths. When forecasting\\n3 days and looking over the whole epidemic the number of deaths averted is substantial. We note that\\nsubstantial innovation is taking place, and new more effective interventions or refinements of current\\ninterventions, alongside behavioral changes will further contribute to reductions in infections. We\\ncannot say for certain that the current measures have controlled the epidemic in Europe; however, if\\ncurrent trends continue, there is reason for optimism.\\n\\nOur approach is semi-mechanistic. We propose a plausible structure for the infection process and then\\nestimate parameters empirically. However, many parameters had to be given strong prior\\ndistributions or had to be fixed. For these assumptions, we have provided relevant citations to\\nprevious studies. As more data become available and better estimates arise, we will update these in\\nweekly reports. Our choice of serial interval distribution strongly influences the prior distribution for\\nstarting R0. Our infection fatality ratio, and infection-to-onset-to-death distributions strongly\\ninfluence the rate of death and hence the estimated number of true underlying cases.\\n\\nWe also assume that the effect of interventions is the same in all countries, which may not be fully\\nrealistic. This assumption implies that countries with early interventions and more deaths since these\\ninterventions (e.g. Italy, Spain) strongly influence estimates of intervention impact in countries at\\nearlier stages of their epidemic with fewer deaths (e.g. Germany, UK).\\n\\nWe have tried to create consistent definitions of all interventions and document details of this in\\nAppendix 8.6. However, invariably there will be differences from country to country in the strength of\\ntheir intervention — for example, most countries have banned gatherings of more than 2 people when\\nimplementing a lockdown, whereas in Sweden the government only banned gatherings of more than\\n10 people. These differences can skew impacts in countries with very little data. We believe that our\\nuncertainty to some degree can cover these differences, and as more data become available,\\ncoefficients should become more reliable.\\n\\nHowever, despite these strong assumptions, there is sufficient signal in the data to estimate changes\\nin R, (see the sensitivity analysis reported in Appendix 8.4.3) and this signal will stand to increase with\\ntime. In our Bayesian hierarchical framework, we robustly quantify the uncertainty in our parameter\\nestimates and posterior predictions. This can be seen in the very wide credible intervals in more recent\\ndays, where little or no death data are available to inform the estimates. Furthermore, we predict\\nintervention impact at country-level, but different trends may be in place in different parts of each\\ncountry. For example, the epidemic in northern Italy was subject to controls earlier than the rest of\\nthe country.\\n\\n5 Data\\n\\nOur model utilizes daily real-time death data from the ECDC (European Centre of Disease Control),\\nwhere we catalogue case data for 11 European countries currently experiencing the epidemic: Austria,\\nBelgium, Denmark, France, Germany, Italy, Norway, Spain, Sweden, Switzerland and the United\\nKingdom. The ECDC provides information on confirmed cases and deaths attributable to COVID-19.\\nHowever, the case data are highly unrepresentative of the incidence of infections due to\\nunderreporting as well as systematic and country-specific changes in testing.\\n\\nWe, therefore, use only deaths attributable to COVID-19 in our model; we do not use the ECDC case\\nestimates at all. While the observed deaths still have some degree of unreliability, again due to\\nchanges in reporting and testing, we believe the data are ofsufficient fidelity to model. For population\\ncounts, we use UNPOP age-stratified counts.10\\n\\nWe also catalogue data on the nature and type of major non-pharmaceutical interventions. We looked\\nat the government webpages from each country as well as their official public health\\ndivision/information webpages to identify the latest advice/laws being issued by the government and\\npublic health authorities. We collected the following:\\n\\nSchool closure ordered: This intervention refers to nationwide extraordinary school closures which in\\nmost cases refer to both primary and secondary schools closing (for most countries this also includes\\nthe closure of otherforms of higher education or the advice to teach remotely). In the case of Denmark\\nand Sweden, we allowed partial school closures of only secondary schools. The date of the school\\nclosure is taken to be the effective date when the schools started to be closed (ifthis was on a Monday,\\nthe date used was the one of the previous Saturdays as pupils and students effectively stayed at home\\nfrom that date onwards).\\n\\nCase-based measures: This intervention comprises strong recommendations or laws to the general\\npublic and primary care about self—isolation when showing COVID-19-like symptoms. These also\\ninclude nationwide testing programs where individuals can be tested and subsequently self—isolated.\\nOur definition is restricted to nationwide government advice to all individuals (e.g. UK) or to all primary\\ncare and excludes regional only advice. These do not include containment phase interventions such\\nas isolation if travelling back from an epidemic country such as China.\\n\\nPublic events banned: This refers to banning all public events of more than 100 participants such as\\nsports events.\\n\\nSocial distancing encouraged: As one of the first interventions against the spread of the COVID-19\\npandemic, many governments have published advice on social distancing including the\\nrecommendation to work from home wherever possible, reducing use ofpublictransport and all other\\nnon-essential contact. The dates used are those when social distancing has officially been\\nrecommended by the government; the advice may include maintaining a recommended physical\\ndistance from others.\\n\\nLockdown decreed: There are several different scenarios that the media refers to as lockdown. As an\\noverall definition, we consider regulations/legislations regarding strict face-to-face social interaction:\\nincluding the banning of any non-essential public gatherings, closure of educational and\\n\\npublic/cultural institutions, ordering people to stay home apart from exercise and essential tasks. We\\ninclude special cases where these are not explicitly mentioned on government websites but are\\nenforced by the police (e.g. France). The dates used are the effective dates when these legislations\\nhave been implemented. We note that lockdown encompasses other interventions previously\\nimplemented.\\n\\nFirst intervention: As Figure 1 shows, European governments have escalated interventions rapidly,\\nand in some examples (Norway/Denmark) have implemented these interventions all on a single day.\\nTherefore, given the temporal autocorrelation inherent in government intervention, we include a\\nbinary covariate for the first intervention, which can be interpreted as a government decision to take\\nmajor action to control COVID-19.\\n\\nA full list of the timing of these interventions and the sources we have used can be found in Appendix\\n8.6.\\n\\n6 Methods Summary\\n\\nA Visual summary of our model is presented in Figure 5 (details in Appendix 8.1 and 8.2). Replication\\ncode is available at https://github.com/|mperia|CollegeLondon/covid19model/releases/tag/vl.0\\n\\nWe fit our model to observed deaths according to ECDC data from 11 European countries. The\\nmodelled deaths are informed by an infection-to-onset distribution (time from infection to the onset\\nof symptoms), an onset-to-death distribution (time from the onset of symptoms to death), and the\\npopulation-averaged infection fatality ratio (adjusted for the age structure and contact patterns of\\neach country, see Appendix). Given these distributions and ratios, modelled deaths are a function of\\nthe number of infections. The modelled number of infections is informed by the serial interval\\ndistribution (the average time from infection of one person to the time at which they infect another)\\nand the time-varying reproduction number. Finally, the time-varying reproduction number is a\\nfunction of the initial reproduction number before interventions and the effect sizes from\\ninterventions. \\n\\n\\nFigure 5: Summary of model components.\\n\\nFollowing the hierarchy from bottom to top gives us a full framework to see how interventions affect\\ninfections, which can result in deaths. We use Bayesian inference to ensure our modelled deaths can\\nreproduce the observed deaths as closely as possible. From bottom to top in Figure 5, there is an\\nimplicit lag in time that means the effect of very recent interventions manifest weakly in current\\ndeaths (and get stronger as time progresses). To maximise the ability to observe intervention impact\\non deaths, we fit our model jointly for all 11 European countries, which results in a large data set. Our\\nmodel jointly estimates the effect sizes of interventions. We have evaluated the effect ofour Bayesian\\nprior distribution choices and evaluate our Bayesian posterior calibration to ensure our results are\\nstatistically robust (Appendix 8.4).\\n\\n7 Acknowledgements\\n\\nInitial research on covariates in Appendix 8.6 was crowdsourced; we thank a number of people\\nacross the world for help with this. This work was supported by Centre funding from the UK Medical\\nResearch Council under a concordat with the UK Department for International Development, the\\nNIHR Health Protection Research Unit in Modelling Methodology and CommunityJameel.\\n\\n8 Appendix: Model Specifics, Validation and Sensitivity Analysis\\n8.1 Death model\\n\\nWe observe daily deaths Dam for days t E 1, ...,n and countries m E 1, ...,p. These daily deaths are\\nmodelled using a positive real-Valued function dam = E(Dam) that represents the expected number\\nof deaths attributed to COVID-19. Dam is assumed to follow a negative binomial distribution with\\n\\n\\nThe expected number of deaths (1 in a given country on a given day is a function of the number of\\ninfections C occurring in previous days.\\n\\nAt the beginning of the epidemic, the observed deaths in a country can be dominated by deaths that\\nresult from infection that are not locally acquired. To avoid biasing our model by this, we only include\\nobserved deaths from the day after a country has cumulatively observed 10 deaths in our model.\\n\\nTo mechanistically link ourfunction for deaths to infected cases, we use a previously estimated COVID-\\n19 infection-fatality-ratio ifr (probability of death given infection)9 together with a distribution oftimes\\nfrom infection to death TE. The ifr is derived from estimates presented in Verity et al11 which assumed\\nhomogeneous attack rates across age-groups. To better match estimates of attack rates by age\\ngenerated using more detailed information on country and age-specific mixing patterns, we scale\\nthese estimates (the unadjusted ifr, referred to here as ifr’) in the following way as in previous work.4\\nLet Ca be the number of infections generated in age-group a, Na the underlying size of the population\\nin that age group and AR“ 2 Ca/Na the age-group-specific attack rate. The adjusted ifr is then given\\n\\nby: ifra = ﬁfiﬁé, where AR50_59 is the predicted attack-rate in the 50-59 year age-group after\\n\\nincorporating country-specific patterns of contact and mixing. This age-group was chosen as the\\nreference as it had the lowest predicted level of underreporting in previous analyses of data from the\\nChinese epidemic“. We obtained country-specific estimates of attack rate by age, AR“, for the 11\\nEuropean countries in our analysis from a previous study which incorporates information on contact\\nbetween individuals of different ages in countries across Europe.12 We then obtained overall ifr\\nestimates for each country adjusting for both demography and age-specific attack rates.\\n\\nUsing estimated epidemiological information from previous studies,“\\'11 we assume TE to be the sum of\\ntwo independent random times: the incubation period (infection to onset of symptoms or infection-\\nto-onset) distribution and the time between onset of symptoms and death (onset-to-death). The\\ninfection-to-onset distribution is Gamma distributed with mean 5.1 days and coefficient of variation\\n0.86. The onset-to-death distribution is also Gamma distributed with a mean of 18.8 days and a\\ncoefficient of va riation 0.45. ifrm is population averaged over the age structure of a given country. The\\ninfection-to-death distribution is therefore given by:\\n\\num ~ ifrm ~ (Gamma(5.1,0.86) + Gamma(18.8,0.45))\\n\\nFigure 6 shows the infection-to-death distribution and the resulting survival function that integrates\\nto the infection fatality ratio.\\n\\n\\nFigure 6: Left, infection-to-death distribution (mean 23.9 days). Right, survival probability of infected\\nindividuals per day given the infection fatality ratio (1%) and the infection-to-death distribution on\\nthe left.\\n\\nUsing the probability of death distribution, the expected number of deaths dam, on a given day t, for\\ncountry, m, is given by the following discrete sum:\\n\\n\\nThe number of deaths today is the sum of the past infections weighted by their probability of death,\\nwhere the probability of death depends on the number of days since infection.\\n\\n8.2 Infection model\\n\\nThe true number of infected individuals, C, is modelled using a discrete renewal process. This approach\\nhas been used in numerous previous studies13\\'16 and has a strong theoretical basis in stochastic\\nindividual-based counting processes such as Hawkes process and the Bellman-Harris process.”18 The\\nrenewal model is related to the Susceptible-Infected-Recovered model, except the renewal is not\\nexpressed in differential form. To model the number ofinfections over time we need to specify a serial\\ninterval distribution g with density g(T), (the time between when a person gets infected and when\\nthey subsequently infect another other people), which we choose to be Gamma distributed:\\n\\ng ~ Gamma (6.50.62).\\n\\nThe serial interval distribution is shown below in Figure 7 and is assumed to be the same for all\\ncountries.\\n\\n \\n\\nFigure 7: Serial interval distribution g with a mean of 6.5 days.\\n\\nGiven the serial interval distribution, the number of infections Eamon a given day t, and country, m,\\nis given by the following discrete convolution function:\\n\\n_ t—1\\nCam — Ram ZT=0 Cr,mgt—‘r r\\nwhere, similarto the probability ofdeath function, the daily serial interval is discretized by\\n\\nfs+0.5\\n\\n1.5\\ngs = T=s—0.Sg(T)dT fors = 2,3, and 91 = fT=Og(T)dT.\\n\\nInfections today depend on the number of infections in the previous days, weighted by the discretized\\nserial interval distribution. This weighting is then scaled by the country-specific time-Varying\\nreproduction number, Ram, that models the average number of secondary infections at a given time.\\n\\nThe functional form for the time-Varying reproduction number was chosen to be as simple as possible\\nto minimize the impact of strong prior assumptions: we use a piecewise constant function that scales\\nRam from a baseline prior R0,m and is driven by known major non-pharmaceutical interventions\\noccurring in different countries and times. We included 6 interventions, one of which is constructed\\nfrom the other 5 interventions, which are timings of school and university closures (k=l), self—isolating\\nif ill (k=2), banning of public events (k=3), any government intervention in place (k=4), implementing\\na partial or complete lockdown (k=5) and encouraging social distancing and isolation (k=6). We denote\\nthe indicator variable for intervention k E 1,2,3,4,5,6 by IkI’m, which is 1 if intervention k is in place\\nin country m at time t and 0 otherwise. The covariate ”any government intervention” (k=4) indicates\\nif any of the other 5 interventions are in effect,i.e.14’t’m equals 1 at time t if any of the interventions\\nk E 1,2,3,4,5 are in effect in country m at time t and equals 0 otherwise. Covariate 4 has the\\ninterpretation of indicating the onset of major government intervention. The effect of each\\nintervention is assumed to be multiplicative. Ram is therefore a function ofthe intervention indicators\\nIk’t’m in place at time t in country m:\\n\\nRam : R0,m eXp(— 212:1 O(Rheum)-\\n\\nThe exponential form was used to ensure positivity of the reproduction number, with R0,m\\nconstrained to be positive as it appears outside the exponential. The impact of each intervention on\\n\\nRam is characterised by a set of parameters 0(1, ...,OL6, with independent prior distributions chosen\\nto be\\n\\nock ~ Gamma(. 5,1).\\n\\nThe impacts ock are shared between all m countries and therefore they are informed by all available\\ndata. The prior distribution for R0 was chosen to be\\n\\nR0,m ~ Normal(2.4, IKI) with K ~ Normal(0,0.5),\\nOnce again, K is the same among all countries to share information.\\n\\nWe assume that seeding of new infections begins 30 days before the day after a country has\\ncumulatively observed 10 deaths. From this date, we seed our model with 6 sequential days of\\ninfections drawn from cl’m,...,66’m~EXponential(T), where T~Exponential(0.03). These seed\\ninfections are inferred in our Bayesian posterior distribution.\\n\\nWe estimated parameters jointly for all 11 countries in a single hierarchical model. Fitting was done\\nin the probabilistic programming language Stan,19 using an adaptive Hamiltonian Monte Carlo (HMC)\\nsampler. We ran 8 chains for 4000 iterations with 2000 iterations of warmup and a thinning factor 4\\nto obtain 2000 posterior samples. Posterior convergence was assessed using the Rhat statistic and by\\ndiagnosing divergent transitions of the HMC sampler. Prior-posterior calibrations were also performed\\n(see below).\\n\\n8.3 Validation\\n\\nWe validate accuracy of point estimates of our model using cross-Validation. In our cross-validation\\nscheme, we leave out 3 days of known death data (non-cumulative) and fit our model. We forecast\\nwhat the model predicts for these three days. We present the individual forecasts for each day, as\\nwell as the average forecast for those three days. The cross-validation results are shown in the Figure\\n8.\\n\\n\\nFigure 8: Cross-Validation results for 3-day and 3-day aggregatedforecasts\\n\\nFigure 8 provides strong empirical justification for our model specification and mechanism. Our\\naccurate forecast over a three-day time horizon suggests that our fitted estimates for Rt are\\nappropriate and plausible.\\n\\nAlong with from point estimates we all evaluate our posterior credible intervals using the Rhat\\nstatistic. The Rhat statistic measures whether our Markov Chain Monte Carlo (MCMC) chains have\\n\\nconverged to the equilibrium distribution (the correct posterior distribution). Figure 9 shows the Rhat\\nstatistics for all of our parameters\\n\\n\\nFigure 9: Rhat statistics - values close to 1 indicate MCMC convergence.\\n\\nFigure 9 indicates that our MCMC have converged. In fitting we also ensured that the MCMC sampler\\nexperienced no divergent transitions - suggesting non pathological posterior topologies.\\n\\n8.4 SensitivityAnalysis\\n\\n8.4.1 Forecasting on log-linear scale to assess signal in the data\\n\\nAs we have highlighted throughout in this report, the lag between deaths and infections means that\\nit ta kes time for information to propagate backwa rds from deaths to infections, and ultimately to Rt.\\nA conclusion of this report is the prediction of a slowing of Rt in response to major interventions. To\\ngain intuition that this is data driven and not simply a consequence of highly constrained model\\nassumptions, we show death forecasts on a log-linear scale. On this scale a line which curves below a\\nlinear trend is indicative of slowing in the growth of the epidemic. Figure 10 to Figure 12 show these\\nforecasts for Italy, Spain and the UK. They show this slowing down in the daily number of deaths. Our\\nmodel suggests that Italy, a country that has the highest death toll of COVID-19, will see a slowing in\\nthe increase in daily deaths over the coming week compared to the early stages of the epidemic.\\n\\n\\nWe investigated the sensitivity of our estimates of starting and final Rt to our assumed serial interval\\ndistribution. For this we considered several scenarios, in which we changed the serial interval\\ndistribution mean, from a value of 6.5 days, to have values of 5, 6, 7 and 8 days.\\n\\nIn Figure 13, we show our estimates of R0, the starting reproduction number before interventions, for\\neach of these scenarios. The relative ordering of the Rt=0 in the countries is consistent in all settings.\\nHowever, as expected, the scale of Rt=0 is considerably affected by this change — a longer serial\\ninterval results in a higher estimated Rt=0. This is because to reach the currently observed size of the\\nepidemics, a longer assumed serial interval is compensated by a higher estimated R0.\\n\\nAdditionally, in Figure 14, we show our estimates of Rt at the most recent model time point, again for\\neach ofthese scenarios. The serial interval mean can influence Rt substantially, however, the posterior\\ncredible intervals of Rt are broadly overlapping.\\n\\n\\nFigure 13: Initial reproduction number R0 for different serial interval (SI) distributions (means\\nbetween 5 and 8 days). We use 6.5 days in our main analysis.\\n\\n\\nFigure 14: Rt on 28 March 2020 estimated for all countries, with serial interval (SI) distribution means\\nbetween 5 and 8 days. We use 6.5 days in our main analysis.\\n\\n8.4.3 Uninformative prior sensitivity on or\\n\\nWe ran our model using implausible uninformative prior distributions on the intervention effects,\\nallowing the effect of an intervention to increase or decrease Rt. To avoid collinearity, we ran 6\\nseparate models, with effects summarized below (compare with the main analysis in Figure 4). In this\\nseries of univariate analyses, we find (Figure 15) that all effects on their own serve to decrease Rt.\\nThis gives us confidence that our choice of prior distribution is not driving the effects we see in the\\nmain analysis. Lockdown has a very large effect, most likely due to the fact that it occurs after other\\ninterventions in our dataset. The relatively large effect sizes for the other interventions are most likely\\ndue to the coincidence of the interventions in time, such that one intervention is a proxy for a few\\nothers.\\n\\n\\nFigure 15: Effects of different interventions when used as the only covariate in the model.\\n\\n8.4.4\\n\\nTo assess prior assumptions on our piecewise constant functional form for Rt we test using a\\nnonparametric function with a Gaussian process prior distribution. We fit a model with a Gaussian\\nprocess prior distribution to data from Italy where there is the largest signal in death data. We find\\nthat the Gaussian process has a very similartrend to the piecewise constant model and reverts to the\\nmean in regions of no data. The correspondence of a completely nonparametric function and our\\npiecewise constant function suggests a suitable parametric specification of Rt.\\n\\nNonparametric fitting of Rf using a Gaussian process:\\n\\n8.4.5 Leave country out analysis\\n\\nDue to the different lengths of each European countries’ epidemic, some countries, such as Italy have\\nmuch more data than others (such as the UK). To ensure that we are not leveraging too much\\ninformation from any one country we perform a ”leave one country out” sensitivity analysis, where\\nwe rerun the model without a different country each time. Figure 16 and Figure 17 are examples for\\nresults for the UK, leaving out Italy and Spain. In general, for all countries, we observed no significant\\ndependence on any one country.\\n\\nFigure 16: Model results for the UK, when not using data from Italy for fitting the model. See the\\n\\n\\nFigure 17: Model results for the UK, when not using data from Spain for fitting the model. See caption\\nof Figure 2 for an explanation of the plots.\\n\\n8.4.6 Starting reproduction numbers vs theoretical predictions\\n\\nTo validate our starting reproduction numbers, we compare our fitted values to those theoretically\\nexpected from a simpler model assuming exponential growth rate, and a serial interval distribution\\nmean. We fit a linear model with a Poisson likelihood and log link function and extracting the daily\\ngrowth rate r. For well-known theoretical results from the renewal equation, given a serial interval\\ndistribution g(r) with mean m and standard deviation 5, given a = mZ/S2 and b = m/SZ, and\\n\\na\\nsubsequently R0 = (1 + %) .Figure 18 shows theoretically derived R0 along with our fitted\\n\\nestimates of Rt=0 from our Bayesian hierarchical model. As shown in Figure 18 there is large\\ncorrespondence between our estimated starting reproduction number and the basic reproduction\\nnumber implied by the growth rate r.\\n\\nR0 (red) vs R(FO) (black)\\n\\nFigure 18: Our estimated R0 (black) versus theoretically derived Ru(red) from a log-linear\\nregression fit.\\n\\n8.5 Counterfactual analysis — interventions vs no interventions\\n\\n\\nFigure 19: Daily number of confirmed deaths, predictions (up to 28 March) and forecasts (after) for\\nall countries except Italy and Spain from our model with interventions (blue) and from the no\\ninterventions counterfactual model (pink); credible intervals are shown one week into the future.\\n\\nDOI: https://doi.org/10.25561/77731\\n\\nPage 28 of 35\\n\\n30 March 2020 Imperial College COVID-19 Response Team\\n\\n8.6 Data sources and Timeline of Interventions\\n\\nFigure 1 and Table 3 display the interventions by the 11 countries in our study and the dates these\\ninterventions became effective.\\n\\nTable 3: Timeline of Interventions.\\n \\n\\nCountry Type Event Date effective\\nSchool closure\\nordered Nationwide school closures.20 14/3/2020\\nPublic events\\nbanned Banning of gatherings of more than 5 people.21 10/3/2020\\nBanning all access to public spaces and gatherings\\nLockdown of more than 5 people. Advice to maintain 1m\\nordered distance.22 16/3/2020\\nSocial distancing\\nencouraged Recommendation to maintain a distance of 1m.22 16/3/2020\\nCase-based\\nAustria measures Implemented at lockdown.22 16/3/2020\\nSchool closure\\nordered Nationwide school closures.23 14/3/2020\\nPublic events All recreational activities cancelled regardless of\\nbanned size.23 12/3/2020\\nCitizens are required to stay at home except for\\nLockdown work and essential journeys. Going outdoors only\\nordered with household members or 1 friend.24 18/3/2020\\nPublic transport recommended only for essential\\nSocial distancing journeys, work from home encouraged, all public\\nencouraged places e.g. restaurants closed.23 14/3/2020\\nCase-based Everyone should stay at home if experiencing a\\nBelgium measures cough or fever.25 10/3/2020\\nSchool closure Secondary schools shut and universities (primary\\nordered schools also shut on 16th).26 13/3/2020\\nPublic events Bans of events >100 people, closed cultural\\nbanned institutions, leisure facilities etc.27 12/3/2020\\nLockdown Bans of gatherings of >10 people in public and all\\nordered public places were shut.27 18/3/2020\\nLimited use of public transport. All cultural\\nSocial distancing institutions shut and recommend keeping\\nencouraged appropriate distance.28 13/3/2020\\nCase-based Everyone should stay at home if experiencing a\\nDenmark measures cough or fever.29 12/3/2020\\n\\nSchool closure\\nordered Nationwide school closures.30 14/3/2020\\nPublic events\\nbanned Bans of events >100 people.31 13/3/2020\\nLockdown Everybody has to stay at home. Need a self-\\nordered authorisation form to leave home.32 17/3/2020\\nSocial distancing\\nencouraged Advice at the time of lockdown.32 16/3/2020\\nCase-based\\nFrance measures Advice at the time of lockdown.32 16/03/2020\\nSchool closure\\nordered Nationwide school closures.33 14/3/2020\\nPublic events No gatherings of >1000 people. Otherwise\\nbanned regional restrictions only until lockdown.34 22/3/2020\\nLockdown Gatherings of > 2 people banned, 1.5 m\\nordered distance.35 22/3/2020\\nSocial distancing Avoid social interaction wherever possible\\nencouraged recommended by Merkel.36 12/3/2020\\nAdvice for everyone experiencing symptoms to\\nCase-based contact a health care agency to get tested and\\nGermany measures then self—isolate.37 6/3/2020\\nSchool closure\\nordered Nationwide school closures.38 5/3/2020\\nPublic events\\nbanned The government bans all public events.39 9/3/2020\\nLockdown The government closes all public places. People\\nordered have to stay at home except for essential travel.40 11/3/2020\\nA distance of more than 1m has to be kept and\\nSocial distancing any other form of alternative aggregation is to be\\nencouraged excluded.40 9/3/2020\\nCase-based Advice to self—isolate if experiencing symptoms\\nItaly measures and quarantine if tested positive.41 9/3/2020\\nNorwegian Directorate of Health closes all\\nSchool closure educational institutions. Including childcare\\nordered facilities and all schools.42 13/3/2020\\nPublic events The Directorate of Health bans all non-necessary\\nbanned social contact.42 12/3/2020\\nLockdown Only people living together are allowed outside\\nordered together. Everyone has to keep a 2m distance.43 24/3/2020\\nSocial distancing The Directorate of Health advises against all\\nencouraged travelling and non-necessary social contacts.42 16/3/2020\\nCase-based Advice to self—isolate for 7 days if experiencing a\\nNorway measures cough or fever symptoms.44 15/3/2020\\n\\nordered Nationwide school closures.45 13/3/2020\\nPublic events\\nbanned Banning of all public events by lockdown.46 14/3/2020\\nLockdown\\nordered Nationwide lockdown.43 14/3/2020\\nSocial distancing Advice on social distancing and working remotely\\nencouraged from home.47 9/3/2020\\nCase-based Advice to self—isolate for 7 days if experiencing a\\nSpain measures cough or fever symptoms.47 17/3/2020\\nSchool closure\\nordered Colleges and upper secondary schools shut.48 18/3/2020\\nPublic events\\nbanned The government bans events >500 people.49 12/3/2020\\nLockdown\\nordered No lockdown occurred. NA\\nPeople even with mild symptoms are told to limit\\nSocial distancing social contact, encouragement to work from\\nencouraged home.50 16/3/2020\\nCase-based Advice to self—isolate if experiencing a cough or\\nSweden measures fever symptoms.51 10/3/2020\\nSchool closure\\nordered No in person teaching until 4th of April.52 14/3/2020\\nPublic events\\nbanned The government bans events >100 people.52 13/3/2020\\nLockdown\\nordered Gatherings of more than 5 people are banned.53 2020-03-20\\nAdvice on keeping distance. All businesses where\\nSocial distancing this cannot be realised have been closed in all\\nencouraged states (kantons).54 16/3/2020\\nCase-based Advice to self—isolate if experiencing a cough or\\nSwitzerland measures fever symptoms.55 2/3/2020\\nNationwide school closure. Childminders,\\nSchool closure nurseries and sixth forms are told to follow the\\nordered guidance.56 21/3/2020\\nPublic events\\nbanned Implemented with lockdown.57 24/3/2020\\nGatherings of more than 2 people not from the\\nLockdown same household are banned and police\\nordered enforceable.57 24/3/2020\\nSocial distancing Advice to avoid pubs, clubs, theatres and other\\nencouraged public institutions.58 16/3/2020\\nCase-based Advice to self—isolate for 7 days if experiencing a\\nUK measures cough or fever symptoms.59 12/3/2020\\n\\n\\n9 References\\n\\n1. Li, R. et al. Substantial undocumented infection facilitates the rapid dissemination of novel\\ncoronavirus (SARS-CoV2). Science (2020) doi:10.1126/science.abb3221.\\n\\n2. Zhang, J. et al. Patterns of human social contact and contact with animals in Shanghai, China.\\n5cLRep.9,1—11(2019)\\n\\n3. Worldometers.info. Hong Kong: coronavirus cases.\\nhttps://www.wo rldometers.info/co ronavirus/country/china-hong-kong-sar/.\\n\\n4. Ferguson, N. et al. Impact of non-pharmaceutical interventions (NPIs) to reduce COVID-19\\nmortality and healthcare demand (Report 9). https://www.imperial.ac.uk/mrc-global-infectious-\\ndisease-analysis/news--wuhan-coronavirus/.\\n\\n5. Cereda, D. et al. The early phase of the COVID-19 outbreak in Lombardy, Italy. arXiv (2020).\\n\\n6. Zhao, A. J. et al. Title: Antibody responses to SARS-CoV-2 in patients of novel coronavirus\\ndisease 2019 Brief Title : Antibody responses in COVID-19 patients. (2020).\\n\\n7. Jombart, T. et al. Inferring the number of COVID-19 cases from recently reported deaths.\\nmedRXiV 2020.03.10.20033761(2020)doi:10.1101/2020.03.10.20033761.\\n\\n8. Zhang, J. et al. Age profile of susceptibility, mixing, and social distancing shape the dynamics\\nof the novel coronavirus disease 2019 outbreak in China. (2020) doi:10.1101/2020.03.19.20039107.\\n\\n9. Lourenco, J. et al. Fundamental principles of epidemic spread highlight the immediate need\\nfor large-scale serological surveys to assess the stage of the SARS-CoV-2 epidemic.\\ndoi:10.1101/2020.03.24.20042291\\n\\n10. United Nations, Department of Economic and Social Affairs, Population Division. World\\nPopulation Prospects 2019: Data Booket. ST/ESA/SER.A/424. (2019).\\n\\n11. Verity, R. et al. Estimates ofthe severity of COVID-19 disease. Lancet Infect Dis in press, (2020).\\n\\n12. Walker, P. G. T. et al. Report 12: The Global Impact of COVID-19 and Strategies for Mitigation\\nand Suppression.\\n\\n13. Fraser, C. Estimating Individual and Household Reproduction Numbers in an Emerging\\nEpidemic. PL05 ONE 2, e758 (2007).\\n\\n14. Cori, A., Ferguson, N. M., Fraser, C. & Cauchemez, S. A New Framework and Software to\\nEstimate Time-Varying Reproduction Numbers During Epidemics. Am. J. Epidemiol. 178, 1505—1512\\n(20131\\n\\n15. Nouvellet, P. et al. A simple approach to measure transmissibility and forecast incidence.\\nEpidemics 22, 29—35 (2018).\\n\\n16. Cauchemez, 8., Valleron, A. J., Boelle, P. Y., Flahault, A. & Ferguson, N. M. Estimating the\\nimpact of school closure on influenza transmission from Sentinel data. Nature 452, 750—754 (2008).\\n\\n17. Bellman, R. & Harris, T. On Age-Dependent Binary Branching Processes. Ann. Math. 55, 280—\\n295(19521\\n\\n18. Bellman, R. & Harris, T. E. On the Theory of Age-Dependent Stochastic Branching Processes.\\nProc. Natl. Acad. Sci. 34, 601—604 (1948).\\n\\n19. Stan Development Team. 2018. The Stan Core Library, Version 2.18.0. http://mc-stan.org.\\n\\n20. Bundesministerium. Coronavirus (COVID-19): Status quo — Schulen, Hochschulen,\\nUniversitaten und Forschungsinstitutionen.\\nhttps://www.bmbwf.gv.at/Ministerium/Informationspflicht/corona/corona_status.html.\\n\\n21. Henley, J. Coronavirus: EU states enact tough measures to stem spread. The Guardian\\nhttps://www.theguardian.com/world/2020/mar/10/coronavirus-several-eu-states-ban-mass-events-\\nafter-italian-lockdown (2020).\\n\\n22. Bundesministerium. Coronavirus - Aktuelle MaBnahmen.\\nhttps://www.sozialministerium.at/Informationen-zum-Coronavirus/Coronavirus—Aktuelle-\\nMaBnahmen.html (2020).\\n\\n23. Federal Public Service. Coronavirus : Phase 2 maintained, transition to the federal phase and\\nadditional measures. https://www.info-coronavirus.be/en/2020/03/12/phase-2-maintained-\\ntransition-to-the-federal-phase-and-additional-measures/ (2020).\\n\\n24. Belgium.be. Coronavirus: reinforced measures | Belgium.be.\\nhttps://www.belgium.be/en/news/2020/coronavirus_reinforced_measures (2020).\\n\\n25. Federal Public Service. Protect yourself and protect the others. https://www.info-\\ncoronavirus.be/en/2020/03/10/protect-yourself—and-protect-the-others/ (2020).\\n\\n26. Wikipedia. 2020 coronavirus pandemic in Denmark. Wikimedia Foundation\\nhttps://en.wikipedia.org/wiki/2020_coronavirus_pandemic_in_Denmark.\\n\\n27. Stephensen, Emma K|inker; Hansen, T. S. Danmark lukker ned: Her er regeringens nye tiltag.\\nTV2 https://nyheder.tv2.dk/samfund/2020-03-11-danmark-lukker-ned-her-er-regeringens-nye-tiltag\\n(20201\\n\\n28. Politi. Nye tiltag mod covid-19. Politi https://politi.dk/coronavirus-i-danmark/seneste-nyt-fra-\\nmyndighederne/nye-tiltag-mod-covid-19 (2020).\\n\\n29. Styrelsen for Patientsikkerhed. Indberetning om covid-19zlnformation om mulighed for\\np\\\\aabud til enkeltpersoner (coronavirus/covid-19). https://stps.dk/da/ansvar-og-\\nretningslinjer/vejledning/indberetning-om-covid-19/#.\\n\\n30. Wikipedia. 2020 coronavirus pandemic in France. Wikimedia Foundation\\nhttps://en.wikipedia.org/wiki/2020_coronavirus_pandemic_in_France.\\n\\n31. The Local. France bans gatherings of more than 100 people as coronavirus death toll rises -\\nThe Local. The Local https://www.thelocal.fr/20200313/france-bans-gatherings-of—over-100-people-\\nto-fight-coronavirus-pandemic (2020).\\n\\n32. Henley, Jon; Willsher, Kim; Kassam, A. Coronavirus: France imposes lockdown as EU calls for\\n30-day travel ban. The Guardian https://www.theguardian.com/world/2020/mar/16/coronavirus-\\nspain-takes-over-private-healthcare-amid-more-european-lockdowns (2020).\\n\\n33. Wikipedia. 2020 coronavirus pandemic in Germany. Wikimedia Foundation\\nhttps://en.wikipedia.org/wiki/2020_coronavirus_pandemic_in_Germany.\\n\\n34. BMI. Coronavirus: Fragen und Antworten. Bundesministerium des Innern,fur Bau und Heimat\\nhttps://web.archive.org/web/20200317073042/https://www.bmi.bund.de/SharedDocs/faqs/DE/the\\nmen/bevoelkerungsschutz/coronavirus/coronavirus-faqs.htmI#doc13738352bodyText7.\\n\\n35. BBC News. Coronavirus: Germany tightens curbs and bans meetings of more than two. BBC\\nNews https://www.bbc.co.uk/news/world-europe-51999080 (2020).\\n\\n36. Bundesregierung. Kanzlerin trifft Regierungschefs der Lander Sozialkontakte vermeiden,\\nAusbreitung verlangsamen. https://www.bundesregierung.de/breg-de/themen/coronavirus/mpk-\\n1730186(2020)\\n\\n37. Robert Koch Institut. Antworten auf haufig gestellte Fragen zum Coronavirus SARS-CoV-2.\\nRobert Koch Institut\\nhttps://web.archive.org/web/20200312004624/https://www.rki.de/SharedDocs/FAQ/NCOV2019/F\\nAQ_Liste.html (2020).\\n\\n38. Ministero della Salute. Governo annuncia sospensione dell’attivita didattica dal 5 a|15 marzo.\\nMinistero della Salute\\nhttp://www.salute.gov.it/portale/nuovocoronavirus/dettaglioVideoNuovoCoronavirus.jsp?lingua=ita\\nliano&menu=multimedia&p=video&id=2052 (2020).\\n\\n39. CNN. Italy prohibits travel and cancels all public events in its northern region. CNN\\nhttps://edition.cnn.com/2020/03/08/europe/italy-coronavirus-lockdown-europe-intl/index.html\\n(2020).\\n\\n40. Attualita. Coronavirus: stop a pub, cinema, teatro e discoteche anche a Roma. Ecco cosa\\nprevede il nuovo decreto. Roma Today https://www.romatoday.it/attualita/coronavirus-pub-cinema-\\nteatri-locali-chiusi-nuovo-decreto.html (2020).\\n\\n41. Gazzetta Ufficiale. DECRETO DEL PRESIDENTE DEL CONSIGLIO DEl MINISTRI. Gazzetta Ufflclale\\nhttps://www.gazzettaufficiale.it/eli/id/2020/03/08/20A01522/sg (2020).\\n\\n42. Helsedirektoratet. The Norwegian Directorate of Health has issued a decision to close schools\\nand other educational institutions. Helsedirektoratet https://www.helsedirektoratet.no/nyheter/the-\\nnorwegian-directorate-of—health-has-issued-a-decision-to-close-schools-and-other-educationa|-\\ninstitutions (2020).\\n\\n43. Krostensen, Mette; Hellem-Hansen, Viktoria L.; Tandstad, B. Folkehelseinstituttet mener\\n23.000 kan vaere smittet. NRK https://www.nrk.no/norge/folkehelseinstituttet-mener-23.000-kan-\\nvaere-smittet-1.14958149 (2020).\\n\\n44. Norweigen Government. The Government is establishing clear quarantine and isolation rules.\\nregjeringen.no https://www.regjeringen.no/en/aktuelt/the-government-is-establishing-clear-\\nquarantine-and-isolation-rules/id2693647/ (2020).\\n\\n45. Wikipedia. 2020 coronavirus pandemic in Spain. Wikimedia Foundation\\nhttps://en.wikipedia.org/wiki/2020_coronavirus_pandemic_in_Spain.\\n\\n46. Gabinete de Prensa. El Gobierno anuncia nuevas medidas para evitar la extension del nuevo\\ncoronavirus COVID-19. Gobierno de Espana\\nhttps://www.mscbs.gob.es/gabinete/notasPrensa.do?id=4807 (2020).\\n\\n47. Gabinete de Prensa. El Consejo Interterritorial del SNS acuerda medidas concretas para zonas\\ncon transmision comunitaria significativa de coronavirus. Gobierno de Espana\\nhttps://www.mscbs.gob.es/gabinete/notasPrensa.do?id=4806 (2020).\\n\\n48. Folkhalsomyndigheten. Larosaten och gymnasieskolor uppmanas nu att bedriva\\ndistansundervisning. Folkhdlsomyndigheten https://www.folkhalsomyndigheten.se/nyheter-och-\\npress/nyhetsarkiv/2020/mars/larosaten-och-gymnasieskolor-uppmanas-nu-att-bedriva-\\ndistansundervisning(2020).\\n\\n49. The Local. Sweden bans large events to halt coronavirus spread. The Local\\nhttps://www.theloca|.se/20200311/sweden-to-ban-large-public-gatherings-over-coronavirus (2020).\\n\\n50. Radosevich. Stockholmers urged to work from home as COVID-19 community spread\\nconfirmed. Sveriges Radio\\nhttps://sverigesradio.se/sida/artikel.aspx?programid=2054&artikel=7430511(2020).\\n\\n51. Folkhalsomyndigheten. Flera tecken p\\\\aa samhallsspridning av covid-19 i Sverige.\\nFolkhdlsomyndigheten https://www.folkhalsomyndigheten.se/nyheter-och-\\npress/nyhetsarkiv/2020/mars/flera-tecken-pa-samhallsspridning-av-covid-19-i-sverige/ (2020).\\n\\n52. Bundesamt fur Gesendheit BAG. Bundesrat verscharft Massnahmen gegen das Coronavirus\\nzum Schutz der Gesundheit und unterstUtzt betroffene Branchen. Schweizerische Eidgenossenschaft\\nhttps://www.bag.admin.ch/bag/de/home/das-bag/aktuell/medienmitteilungen.msg-id-78437.html\\n(20201\\n\\n53. Bundesamt fur Gesundheit BAG. Coronavirus: Bundesrat verbietet Ansammlungen von mehr\\nals fUnf Personen. Schweizerische Eidgenossenschaft https://www.bag.admin.ch/bag/de/home/das-\\nbag/aktuell/medienmitteilungen.msg-id-78513.html (2020).\\n\\n54. Bundesamt fur Gesundheit BAG. Coronavirus: Bundesrat erklart die «ausserordentliche Lage»\\nund verscharft die Massnahmen. Schweizerische Eidgenossenschaft\\nhttps://www.bag.admin.ch/bag/de/home/das-bag/aktuell/medienmitteilungen.msg-id-78454.html\\n(20201\\n\\n55. Bundesamt fur Gesundheit BAG. Neue Hygiene- und Verhaltensregeln zum Schutz gegen das\\nneue Coronavirus. Schweizerische Eidgenossenschaft https://www.bag.admin.ch/bag/de/home/das-\\nbag/a ktuell/medienmitteilungen.msg-id-78304.html (2020).\\n\\n56. UK Government, D. for E. Schools, colleges and early years settings to close. UK Government\\nhttps://www.gov.uk/government/news/schools-colleges-and-early-years-settings-to-close (2020).\\n\\n57. UK Government. PM address to the nation on coronavirus: 23 March 2020. UK Government\\nhttps://www.gov.uk/government/speeches/pm-address-to-the-nation-on-coronavirus-23-march-\\n2020(20201\\n\\n58. Boycott-Owen, Mason; Bowman, Verity; Kelly-Linden, Jordan; Gartner, A. G. H. S. T.\\nCoronavirus: Boris Johnson puts UK in lockdown as death tolls reaches 55. The Telegraph\\nhttps://www.telegraph.co.uk/global-health/science-and-disease/coronavirus-news-uk-latest-\\nupdate-covid-19-death-toll-cases/ (2020).\\n\\n59. BBC News. Coronavirus: People with fever or ’continuous’ cough told to self—isolate. BBC News\\nhttps://www.bbc.co.uk/news/uk-51857856 (2020).\\n\\n', 'document_id': 2683}]}, {'paragraphs': [{'qas': [{'question': 'In 2009 what was the reported H1N1 vaccination rate in China?', 'id': 523, 'answers': [{'text': '10.8%', 'answer_start': 1433}], 'is_impossible': False}, {'question': 'What is the highest alert level given by the World Health Organization to a pandemic?', 'id': 525, 'answers': [{'text': 'phase 6', 'answer_start': 2938}], 'is_impossible': False}, {'question': 'What does it mean for a pandemic to have a WHO alert level of 6?', 'id': 527, 'answers': [{'text': 'spread in more than two continents', 'answer_start': 2984}], 'is_impossible': False}, {'question': 'What was the estimated economic impact in the U.S. from the 2009 SARS pandemic?', 'id': 528, 'answers': [{'text': 'estimated at $30-$100 billion', 'answer_start': 5423}], 'is_impossible': False}], 'context': \"Knowledge, Attitudes and Practices (KAP) related to the Pandemic (H1N1) 2009 among Chinese General Population: a Telephone Survey\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC3112099/\\n\\nSHA: fe954b75ed45c02d47090ee70d25c726b24b081c\\n\\nAuthors: Lin, Yilan; Huang, Lijuan; Nie, Shaofa; Liu, Zengyan; Yu, Hongjie; Yan, Weirong; Xu, Yihua\\nDate: 2011-05-16\\nDOI: 10.1186/1471-2334-11-128\\nLicense: cc-by\\n\\nAbstract: BACKGROUND: China is at greatest risk of the Pandemic (H1N1) 2009 due to its huge population and high residential density. The unclear comprehension and negative attitudes towards the emerging infectious disease among general population may lead to unnecessary worry and even panic. The objective of this study was to investigate the Chinese public response to H1N1 pandemic and provide baseline data to develop public education campaigns in response to future outbreaks. METHODS: A close-ended questionnaire developed by the Chinese Center for Disease Control and Prevention was applied to assess the knowledge, attitudes and practices (KAP) of pandemic (H1N1) 2009 among 10,669 responders recruited from seven urban and two rural areas of China sampled by using the probability proportional to size (PPS) method. RESULTS: 30.0% respondents were not clear whether food spread H1N1 virusand. 65.7% reported that the pandemic had no impact on their life. The immunization rates of the seasonal flu and H1N1vaccine were 7.5% and 10.8%, respectively. Farmers and those with lower education level were less likely to know the main transmission route (cough or talk face to face). Female and those with college and above education had higher perception of risk and more compliance with preventive behaviors. Relationships between knowledge and risk perception (OR = 1.69; 95%CI 1.54-1.86), and knowledge and practices (OR = 1.57; 95%CI 1.42-1.73) were found among the study subjects. With regard to the behavior of taking up A/H1N1 vaccination, there are several related factors found in the current study population, including the perception of life disturbed (OR = 1.29; 95%CI 1.11-1.50), the safety of A/H1N1 vaccine (OR = 0.07; 95%CI 0.04-0.11), the knowledge of free vaccination policy (OR = 7.20; 95%CI 5.91-8.78), the state's priority vaccination strategy(OR = 1.33; 95%CI 1.08-1.64), and taking up seasonal influenza vaccine behavior (OR = 4.69; 95%CI 3.53-6.23). CONCLUSIONS: This A/H1N1 epidemic has not caused public panic yet, but the knowledge of A/H1N1 in residents is not optimistic. Public education campaign may take the side effects of vaccine and the knowledge about the state's vaccination strategy into account.\\n\\nText: At the end of March 2009, an outbreak of novel influenza A (H1N1) (here after called A/H1N1) infection occurred in Mexico, followed by ongoing spread to all over the world in a short period [1] . On June 11 2009, the World Health Organization raised its pandemic alert level to the highest level, phase 6 [2] , meaning that the A/H1N1 flu had spread in more than two continents and reached pandemic proportions. As of June 13, 2010, it had caused over 18,172 deaths in more than 214 countries and overseas territories or communities [3] . Most illness, especially the severe illness and deaths, had occurred among healthy young adults, which was markedly different from the disease pattern seen during epidemics of seasonal influenza [4, 5] .\\n\\nChina is highly susceptible to A/H1N1 because of its huge population and high residential density, besides the high infectiousness of this novel influenza virus. After the first imported case reported on May 11, 2009 , the confirmed cases were reported in various provinces of China [6] . By the late of October 2009, A/H1N1 cases had increased dramatically, with 44,981 cases and 6 deaths confirmed at the end of October 2009. The A/ H1N1 infection rate peaked in November 2009, when approximately 1500 new cases of A/H1N1 were being confirmed each day. By the end of this month, a total of 92,904 cases and 200 deaths had resulted from A/ H1N1-related causes [7] . The Chinese government has taken a series of preventive measures according to WHO guidelines, including the promotion of public knowledge about flu through mass media, patient isolation, quarantine of close contact person, and free vaccinations to population at high risk (e.g. young children, healthcare workers, and people with chronic disease) [8] . However, there were few public reports on the assessment of the effect of these policies and the level of knowledge, attitude and practice (KAP) associating with A/H1N1 among general population.\\n\\nIt is well-known that confused comprehension and negative attitude towards the emerging communicable disease may lead to unnecessary worry and chaos, even excessive panic which would aggravate the disease epidemic [9] . For instance, during SARS epidemic from 2002 to 2004, the misconceptions and the excessive panic of Chinese public to SARS led the public resistant to comply with the suggested preventive measures such as avoiding public transportation, going to hospital when they were sick, which contributed to the rapid spread of SARS and resulted in a more serious epidemic situation, making China one of the worst affected countries with over 5327 cases and 439 deaths [10, 11] . In addition, the panic of infectious disease outbreak could cause huge economic loss, for example the economic loss of SARS has been estimated at $30-$100 billion in US, though less than 10,000 persons were infected [12] . SARS experience has demonstrated the importance of monitoring the public perception in disease epidemic control, which may affect the compliance of community to the precautionary strategies. Understanding related factors affecting people to undertake precautionary behavior may also help decision-makers take appropriate measures to promote individual or community health. Therefore, it is important to monitor and analyze the public response to the emerging disease.\\n\\nTo investigate community responses to A/H1N1 in China, we conducted this telephone survey to describe the knowledge, attitudes and practices of A/H1N1 among general population in China and put forward policy recommendations to government in case of future similar conditions.\\n\\nThis study was performed in seven urban regions (Beijing, Shanghai, Wuhan, Jingzhou, Xi'an, Zhengzhou, Shenzhen cities) and two rural areas (Jingzhou and Zhengzhou counties) of China with over one million people in each region. Regarding the urban sites, Beijing as the capital of China locates in the northeast; Shanghai is a municipality in the east of China; Wuhan (the provincial capital of Hubei) and Zhengzhou (the provincial capital of Henan province) are both in the centre of China; Xi'an in the northwest of China is the provincial capital of Shanxi province; and Shenzhen of the Guangdong province is in the southeast of China. As for the rural sites, Jingzhou county and Zhengzhou county, from Hubei and Henan provinces, respectively, both locate in the centre of China.\\n\\nThis current study was carried out in three phases during the pandemic peak season of A/H1N1. The first phase was from 30 November 2009 to 27 December 2009, the second from 4 January 2010 to 24 January 2010, and the third from 24 February to 25 March in 2010.\\n\\nA two-stage proportional probability to size (PPS) sampling method was used in each phase. In stage І, about 30% of administrative regions in each study site were selected as primary sample units (PSUs) for cluster sampling. In stage II, telephone numbers were sampled randomly, of which the first four digitals were obtained from each PSU's post office as initial number and the other three or four digitals were obtained from random number generated by Excel 2003. Then each family was chosen as per unit (excluding school, hotel public or cell phone etc.) and at least 400 families in each site at each phase were selected finally. If the family was selected repeatedly or refused to answer the questionnaire, we added one to the last digit of phone number and dial again. If the line was busy or of no response, we would dial three times and then give up this phone number if there was still no respondent.\\n\\nAnonymous telephone interviews were conducted from 6:30 pm to 10:00 pm so as to avoid over-presenting the non-work population by well-trained interviewers with Bachelor degree of Epidemiology. The Questionnaire to Survey the Level of Knowledge, Attitude and Practice in Different Stages of A/H1N1 Pandemic by Telephone was designed by the Chinese Centre for Disease Control and Prevention (China CDC, Beijing). The majority of the questions were closed-ended and variables in the questionnaire were categorical, except age. The inclusion criteria of subjects were: age≥18 and proper communication skills. There were seven questions related to the knowledge of A/H1N1, four referred to the attitude, and five concerning about the practice in this questionnaire (See additional file 1: The Questionnaire to Survey the Level of Knowledge, Attitude and Practice in Different Stages of H1N1 Pandemic by Telephone in China).\\n\\nThis study was approved by the institutional review board of the Tongji Medical College of Huazhong University of Science and Technology. All respondents were informed consent. We respected their wishes whether to accept our survey and promised to protect their secrets.\\n\\nAll data were entered into computer using Epidata V.3.1 and were analyzed in SPSS statistical software V.12. Chi-square test was applied to compare the immunization rates of the seasonal flu and A/H1N1 vaccine. The associations between the socio-demographic factors and the KAP regarding A/H1N1 were firstly investigated by using univariate odds ratios (OR) and then stepwise logistic regression modeling applied. Adjusting for such background variables including gender, age, level of education, occupation, region, and survey wave, stepwise multivariate logistic regression models were applied to investigate the impact factors associated with the risk perception of A/H1N1, A/H1N1 vaccination uptake and the compliance with suggested preventive measures (avoid crowd places/wash hand frequently/keep distance from people with influenza-like symptoms). For the purposes of analysis, the factor knowledge about the main modes of transmission was divided into two groups according to whether the respondents knew both cough and talk faceto-face can spread A/H1N1. Odds ratios and respective 95% confidence intervals (CI) were obtained from the logistic regression analysis. P values lower than 0.05 were judged to be statistically significant.\\n\\nA total of 88541 telephone numbers were dialed. Except 65323 invalid calls (including vacant numbers, fax numbers, busy tone numbers and non-qualified respondents whose age <18 and whose phones were from school, hotel or other public places), 23218 eligible respondents were identified. Among these respondents, 12360 completed the interview. Therefore, the response rate was 46.8%. Excluding missing, and logical erroneous data, 10669 questionnaires in total were eligible for analysis. The baseline characteristics of the respondents were presented in Table1. The mean age of all respondents was 41.47 years (over range: 18-90 year) . Of all respondents, 54.4% were female, and 42.4% had received college or above education (Table 1) .\\n\\nThe overall KAP related to A/H1N1 was reported in Table 2 . As to knowledge, 75.6% of all respondents knew that influenza could be transmitted by coughing and sneezing, and 61.9% thought that talking face-to-face was the transmission route, whereas 30.0% believed the transmission could be through food. Less than one third of respondents knew that virus could be transmitted by handshaking and indirect hand contact (26.8% and 22.3%, respectively). Multiple logistic regression analysis showed that those with middle school (OR = 1.71; 95%CI 1.48-1.98), or having an education level of college and above (OR = 2.16; 95%CI 1.83-2.54) were more likely to know the transmission routes comparing with other people. Comparing with students, teachers (OR = 1.46; 95%CI 1.09-1.96) were more likely to answer the above questions Table  3 and Table 4 ). Regarding the A/H1N1vaccination, 69.9% respondents believed that the occurrence rate of adverse reactions caused by A/H1N1 vaccination was fairly low and they were not afraid of taking up vaccination. Most residents (96.1%) thought that the state's vaccination strategy was reasonable.\\n\\nAbout half of the respondents (42.9%) had avoided going to crowded places during the past two weeks of our survey. In case people nearby held influenza-like symptoms such as fever or cough, 56.9% increased the frequency of hand-washing and 57.4% would stay away from them. Multiple logistic regression analysis indicated compliance with the preventive practices were more likely to be taken by those who were females (OR = 1. Table 3 and Table 4 ). The immunization rates of the seasonal flu and A/ H1N1 in respondents were 7.5% and 10.8% respectively. The multivariate stepwise models further showed that except the health care workers (OR = 1.52; 95%CI 1.09-2.11), residents in other occupations (OR = 0.06-0.67) were less likely to take up the A/H1N1 vaccination comparing with students (in Table 3 ). Adjusting for the background covariates the knowledge about the free vaccination policy (OR = 7.20; 95%CI 5.91-8.78) and the state's initial vaccination strategy(OR = 1.33; 95%CI 1.08-1.64), perception of daily life disturbed (OR = 1.29; 95% CI 1.11-1.50), practice of injecting the seasonal influenza vaccine (OR = 4.69; 95%CI 3.53-6.23) were significantly associated with behavior of taking up the A/H1N1 vaccination positively (in Table 5 ), and the adverse reaction of A/H1N1 vaccine negatively influenced people's practice (OR = 0.07; 95%CI 0.04-0.11).\\n\\nNovel A/H1N1 has caused pandemic in this century. It is important to encourage the public to adopt precautionary behaviors, which is based on the correct knowledge of the epidemic and appropriate response among residents. Many studies have examined the various levels of KAP about infectious disease outbreaks, such as SARS, avian influenza [13] [14] [15] . Some studies have been reported specifically on community responses to A/H1N1 in Australia and Europe [16, 17] . But through literature search, we haven't found any public reports on KAP regarding A/H1N1 among Chinese population until now. Therefore, we conducted this large population-based survey (10669 respondents) to investigate community responses to A/H1N1 and to provide baseline data to government for preventive measures in case of future outbreaks.\\n\\nUnless people have basic knowledge about the modes of transmission, they respond appropriately during an outbreak [16] . It has been proved that influenza is transmitted through person to person via respiratory secretions [18] . Most residents in our survey recognized that OR m : odds ratio obtained from stepwise multivariate logistics regression analysis using univariately significant variables as candidate variables and adjusting for region; NU: not significant in the univariate analysis; *: P < 0.05; †: P < 0.01; ‡: P < 0.0001.\\n\\nthe risk of getting infected would increase when an infected person coughed or sneezed in close distance. This may be due to the previous experience of SARS and avian flu. Multivariate analysis results showed that workers and farmers with lower education level were less likely to have this knowledge, which indicated that the contents and forms of propaganda should be more understandable and acceptable. A large proportion of residents in our survey overlooked the indirect hand contact and hand-shaking transmission route and about one third of public misconceived that A/H1N1 was food borne, which was associated with the previous knowledge of avian flu and the new A/H1N1 flu in the general population. The confusion with avian flu might mislead some residents to believe that the A/H1N1 virus is fatal and cause public panic [19] . Therefore, it is important for the government and health authorities to provide continuously updated information of the emerging disease through televisions, newspapers, radios, and Internet. There are regional differences in the perception of A/H1N1. For example, the public in Hong Kong did not perceive a high likelihood of having a local A/H1N1 outbreak [19] , but Malaysians were particularly anxious about the pandemic [20] . The current study shows that emotional distress was relatively mild in China as few residents worried about being infected (25.1%). This phenomenon may also be related to the previous experience of the SARS epidemic, as well as the open epidemic information. A survey in Korean university showed that women perceived higher illness severity and personal susceptibility to A/ H1N1 infection, which had been reconfirmed in our study [21] . Logistic regression analysis results suggested that women with higher educational level had higher perception of risk. As time went by, the knowledge about the main transmission route increased, but the risk perception of being infected in residents decreased, suggesting the positive effect of government policy regarding A/H1N1 infection prevention, as well as the promotion of the media.\\n\\nThe previous study presented various results of influencing factors on the the compliance with the preventive practices. The study in Saudi showed that older men with better education were more likely to take preventive practices [9] ; female students in Korean washed hands more frequently during the peak pandemic period of A/ H1N1 [21] ; in another pandemic study in USA, younger people was found to have greater uptake of recommended behaviors but not for gender [16] . We found female with higher education took more precautionary behaviors, but office staffs and farmers took less comparing with students. While such differences could result from study population demographics, profound differences may also exist in the knowledge of A/H1N1 and the perceptions of recommended behaviors in those countries. Adjusting for the background factors, the multivariate logistic regression showed the possible relationship between knowledge and risk perception, knowledge and practices (odd ratios were 1.57 and 2.09, respectively), which indicated that good knowledge is important to enable individuals to have better attitudes and practices in influenza risk reduction. Similar findings were observed in other studies performed during A/ H1N1 pandemic in Singapore [22] and during SARS pandemic in Hong Kong [13] . Therefore, it is important to focus on inculcating the correct knowledge to individuals as it will influence both attitudes and practices. Injecting vaccination is an effective measure to prevent infectious disease [23] . In China, the seasonal influenza vaccination is not included in the national immunization program and must be purchased by recipients. Those who are above 60 years old, the pupil and children in kindergarten, and people with chronic diseases are recommended to get inoculation. Data provided by China CDC in 2009 showed that the immunization rate of the seasonal flu in Chinese population was below 2% [24] , which was much lower than 7.5% in our study (P < 0.0001). This phenomenon is partly due to the state's prior vaccination strategy for population at high risk such as students, teachers, healthcare workers and people with chronic disease, as well as the confusion between seasonal flu vaccine and A/H1N1 vaccine in residents. People who couldn't access the A/H1N1 vaccine may take up seasonal flu vaccine as preventive behaviors. The A/ H1N1 vaccine was not available in China until the middle of September 2009. All populations at high risk above three years old were invited for vaccination free of charge [25] . A survey among 868 European travelers showed 14.2% participants were vaccinated against pandemic influenza A/H1N1 [26] , higher than 10.8% in our study (P < 0.01). Our study also showed students and health care workers were more likely to take up, which may be due to the prior vaccination strategy. Multivariate stepwise logistic regression analysis, which allowed us to adjust for background factors, further showed the perceived risk of infection and the knowledge about the main modes of transmission related to A/H1N1 vaccination were insignificantly, similar results seen in Lau's study [8] . Therefore, the vaccination rate of A/H1N1 is not expected to increase even if the virus becomes more prevalent or the knowledge of its transmission mode improved. Additionally, the behavior of taking up A/H1N1 vaccine was associated with perceptions of vaccine's safety and influence on daily life by A/H1N1 as well as the knowledge about the free vaccination policy and the state's initial vaccination strategy. This suggests that improving the safety of vaccine, the acceptability of side effect and the knowledge about the state's strategy related to A/H1N1 vaccination in residents may be helpful to promote A/H1N1 vaccination in the general population. The cross-sectional telephone survey adopted in the study has some limitations. We were unable to interview the people who did not have phones and the depth of the questionnaire was largely limited because questions and pre-existing answers could not be too long and complex. In addition, the telephone response rate was 46.8%, which means more than half of the interviewees rejected or didn't finish the survey. It was impossible to compare the difference between respondents and nonrespondents due to the lack of their basic information.\\n\\nThis A/H1N1 epidemic has not caused public panic yet, but the knowledge of A/H1N1 in residents is not optimistic as most of them confused the transmission route of A/H1N1. There are many factors influencing the KAP related to A/H1N1. Female with higher educational level had higher perceived risk of infection and took more precautionary behaviors. Public education campaign may take the side effects of vaccine and the knowledge about the state's vaccination strategy into account. The data collected in this survey could be used as baseline data to monitor public perceives and behaviors in the event of future outbreak of infectious disease in China.\\n\\nAdditional file 1: Questionnaire. The Questionnaire to Survey the Level of Knowledge, Attitude and Practice in Different Stages of H1N1 Pandemic by Telephone in China.\", 'document_id': 2675}]}, {'paragraphs': [{'qas': [{'question': 'How is 2019-nCOV transmitted?', 'id': 881, 'answers': [{'text': '2019-nCoV was transmitted through respiratory tract and then induced pneumonia,', 'answer_start': 600}], 'is_impossible': False}, {'question': 'What are ways in  which 2019-nCOV is transmitted?', 'id': 882, 'answers': [{'text': 'We found the presence of 2019-nCoV in anal swabs and blood as well, and more anal swab positives than oral swab positives in a later stage of infection, suggesting shedding and thereby transmitted through oral–fecal route.', 'answer_start': 1156}], 'is_impossible': False}, {'question': 'Is oral swab for detecting 2019-nCOV infection, sufficient?', 'id': 883, 'answers': [{'text': 'the current strategy for the detection of viral RNA in oral swabs used for 2019-nCoV diagnosis is not perfect. The virus may be present in anal swabs or blood of patients when oral swabs detection negative.', 'answer_start': 9634}], 'is_impossible': False}, {'question': 'Is oral swab for detecting 2019-nCOV infection, sufficient?', 'id': 884, 'answers': [{'text': ' patients infected with 2019-nCoV may harbour the virus in the intestine at the early or late stage of disease. It is also worth to note none of the patients with viremia blood had positive swabs. These patients would likely be considered as 2019-nCoV negative through routine surveillance, and thus pose a threat to other people.', 'answer_start': 9970}], 'is_impossible': False}, {'question': 'What other tests should be considered for 2019-nCOV epidemiology?', 'id': 885, 'answers': [{'text': 'serology should be considered for 2019-nCoV epidemiology.', 'answer_start': 10373}], 'is_impossible': False}, {'question': 'What tests should  be done before a 2019-nCOV infected patient is discharged?', 'id': 886, 'answers': [{'text': 'we cannot discharge a patient purely based on oral swabs negative, who may still shed the virus by oral-fecal route. Above all, we strongly suggest using viral IgM and IgG serological test to confirm an infection, considering the unreliable results from oral swabs detection', 'answer_start': 10581}], 'is_impossible': False}], 'context': \"Molecular and serological investigation of 2019-nCoV infected patients: implication of multiple shedding routes\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC7048229/\\n\\nSHA: da81f0d3a12ab7faa09148acb6564271474e9e02\\n\\nAuthors: Zhang, Wei; Du, Rong-Hui; Li, Bei; Zheng, Xiao-Shuang; Yang, Xing-Lou; Hu, Ben; Wang, Yan-Yi; Xiao, Geng-Fu; Yan, Bing; Shi, Zheng-Li; Zhou, Peng\\nDate: 2020-02-17\\nDOI: 10.1080/22221751.2020.1729071\\nLicense: cc-by\\n\\nAbstract: In December 2019, a novel coronavirus (2019-nCoV) caused an outbreak in Wuhan, China, and soon spread to other parts of the world. It was believed that 2019-nCoV was transmitted through respiratory tract and then induced pneumonia, thus molecular diagnosis based on oral swabs was used for confirmation of this disease. Likewise, patient will be released upon two times of negative detection from oral swabs. However, many coronaviruses can also be transmitted through oral–fecal route by infecting intestines. Whether 2019-nCoV infected patients also carry virus in other organs like intestine need to be tested. We conducted investigation on patients in a local hospital who were infected with this virus. We found the presence of 2019-nCoV in anal swabs and blood as well, and more anal swab positives than oral swab positives in a later stage of infection, suggesting shedding and thereby transmitted through oral–fecal route. We also showed serology test can improve detection positive rate thus should be used in future epidemiology. Our report provides a cautionary warning that 2019-nCoV may be shed through multiple routes.\\n\\nText: Coronaviruses (CoVs) belong to the subfamily Orthocoronavirinae in the family Coronaviridae and the order Nidovirales. A human coronavirus (SARS-CoV) caused the severe acute respiratory syndrome coronavirus (SARS) outbreak in 2003. Most recently, an SARS-related CoV was implicated as the etiological agent responsible for the outbreak in Wuhan, central China. This outbreak is estimated to have started on 12th December 2019 and 17,332 laboratory confirmed cases with 361 deaths as of 3rd February 2020 in China [1] . The virus has spread to 23 other countries by travellers from Wuhan [1] . Typical symptoms are fever, malaise, shortness of breath and in severe cases, pneumonia [2] [3] [4] . The disease was first called unidentified viral pneumonia.\\n\\nWe quickly identified the etiological agent, termed 2019-nCoV (virus name designated by the World Health Organization). The newly identified virus is an SARS-related virus (SARSr-CoV) but shares only 74.5% genome identity to SARS-CoV [2] . We developed molecular detection tools based on viral spike genes. Our previous studies indicate that qPCR method can be used for the detection of 2019-nCoV in oral swabs or in bronchoalveolar lavage fluid (BALF) [5] .\\n\\nAdditionally, we developed IgM and IgG detection methods using a cross-reactive nucleocapsid protein (NP) from another SARSr-CoV Rp3 [6] , which is 92% identical to 2019-nCoV NP. Using these serological tools, we demonstrate viral antibody titres increase in patients infected with 2019-nCoV [5] .\\n\\nLike SARS-CoV, 2019-nCoV induced pneumonia through respiratory tract by clinical observation. Therefore, the presence of viral antigen in oral swabs was used as detection standard for 2019-nCoV. Similarly, two times of oral swabs negative in a 24-h interval was considered as viral clearance by patients officially.\\n\\nHere we launched an investigation of 2019-nCoV in a Wuhan hospital, aiming to investigate the other possible transmission route of this virus.\\n\\nHuman samples, including oral swabs, anal swabs and blood samples were collected by Wuhan pulmonary hospital with the consent from all patients and approved by the ethics committee of the designated hospital for emerging infectious diseases. Two investigations were performed. In the first investigation, we collected samples from 39 patients, 7 of which were in severe conditions. In the second investigation, we collected samples from 139 patients, yet their clinical records were not available. We only showed patients who were viral nucleotide detection positive. Patients were sampled without gender or age preference unless where indicated. For swabs, 1.5 ml DMEM+2% FBS medium was added in each tube. Supernatant was collected after 2500 rpm, 60 s vortex and 15-30 min standing. Supernatant from swabs were added to lysis buffer for RNA extraction. Serum was separated by centrifugation at 3000 g for 15 min within 24 h of collection, followed by 56°C 30 min inactivation, and then stored at 4°C until use.\\n\\nWhenever commercial kits were used, manufacturer's instructions were followed without modification. RNA was extracted from 200 μl of samples with the High Pure Viral RNA Kit (Roche). RNA was eluted in 50 μl of elution buffer and used as the template for RT-PCR. QPCR detection method based on 2019-nCoV S gene can be found in the previous study [5] . In brief, RNA extracted from above used in qPCR by HiScript® II One Step qRT-PCR SYBR® Green Kit (Vazyme Biotech Co., Ltd). The 20 μl qPCR reaction mix contained 10 μl 2× One Step SYBR Green Mix, 1 μl One Step SYBR Green Enzyme Mix, 0.4 μl 50 × ROX Reference Dye 1, 0.4 μl of each primer (10 μM) and 2 μl template RNA. Amplification was performed as follows: 50°C for 3 min, 95°C for 30 s followed by 40 cycles consisting of 95°C for 10 s, 60°C for 30 s, and a default melting curve step in an ABI 7500 machine.\\n\\nIn-house anti-SARSr-CoV IgG and IgM ELISA kits were developed using SARSr-CoV Rp3 NP as antigen, which shared above 90% amino acid identity to all SARSr-CoVs, as reported previously [5] . For IgG test, MaxiSorp Nunc-immuno 96 well ELISA plates were coated (100 ng/well) overnight with recombinant NP. Human sera were used at 1:20 dilution for 1 h at 37°C. An anti-Human IgG-HRP conjugated monoclonal antibody (Kyab Biotech Co., Ltd, Wuhan, China) was used at a dilution of 1:40,000. The OD value (450-630) was calculated. For IgM test, Maxi-Sorp Nunc-immuno 96 wellELISA plates were coated (500 ng/well) overnight with anti-human IgM (µ chain). Human sera were used at 1:100 dilution for 40 min at 37°C, followed by anti-Rp3 NP-HRP conjugated (Kyab Biotech Co., Ltd, Wuhan, China) at a dilution of 1:4000. The OD value (450-630) was calculated.\\n\\nIn the first investigation, we aimed to test whether viral positive can be found in anal swab and blood as well as oral swabs. We conducted a molecular investigation to patients in Wuhan pulmonary hospital, who were detected as oral swabs positive for 2019-nCoV upon admission. We collected blood, oral swabs and anal swabs for 2019-nCoV qPCR test using previously established method [5] .\\n\\nWe found 15 patients who still carry virus following days of medical treatments. Of these patients, 8 were oral swabs positive (53.3%), 4 were anal swabs positive (26.7%), 6 blood positives (40%) and 3 serum positives (20%). Two patients were positive by both oral swab and anal swab, yet none of the blood positive was also swabs positive. Not surprisingly, all serum positives were also whole serum positive (Table 1 ). In summary, viral nucleotide can be found in anal swab or blood even if it cannot be detected in oral swabs. It should be noted that although swabs may be negative, the patient might still be viremic.\\n\\nWe then did another investigation to find out the dynamic changes of viral presence in two consecutive studies in both oral and anal swabs in another group of patients. The target patients were those who received around 10 days of medical treatments upon admission. We tested for both viral antibody and viral nucleotide levels by previously established method [5] . We showed that both IgM and IgG titres were relatively low or undetectable in day 0 (the day of first sampling). On day 5, an increase of viral antibodies can be seen in nearly all patients, which was normally considered as a transition from earlier to later period of infection ( Figure 1 and supplementary table 1 ). IgM positive rate increased from 50% (8/16) to 81% (13/16), whereas IgG positive rate increased from 81% (13/16) to 100% (16/16). This is in contrast to a relatively low detection positive rate from molecular test (below). For molecular detection, we found 8 oral swabs positive (50%) and 4 anal swabs (25%) in these 16 people on day 0. On day 5, we were only able to find 4 oral swabs positive (25%). In contrast, we found 6 anal swabs positive (37.5%). When counting all swab positives together, we found most of the positives came from oral swab (8/10, 80%) on day 0. However, this trend appears to change on day 5. We found more (6/8, 75%) anal swab positive than oral swab positive (4/8, 50%). Another observation is the reoccurrence of virus in 6 patients who were detected negative on day 0. Of note, 4 of these 6 viral positives were from anal swabs ( Table 2) . These data suggested a shift from more oral positive during early period (as indicated by antibody titres) to more anal positive during later period might happen.\\n\\nWithin 1 month of the 2019-nCoV disease outbreak, we rapidly developed molecular and serological detection tools. This is the first molecular and serological study on this virus after the initial identification of 2019-NCoV from 7 patients diagnosed with unidentified viral pneumonia [5] . We detected the virus in oral swabs, anal swabs and blood, thus infected patients can potentially shed this pathogen through respiratory, fecal-oral or body fluid routes. In addition, we successfully applied serology test a large population and showed which could greatly improved detection positive rate.\\n\\nWe show that the current strategy for the detection of viral RNA in oral swabs used for 2019-nCoV diagnosis is not perfect. The virus may be present in anal swabs or blood of patients when oral swabs detection negative. In SARS-CoV and MERS-CoV infected patients, intestinal infection was observed at later stages of infection [7] [8] [9] . However, patients infected with 2019-nCoV may harbour the virus in the intestine at the early or late stage of disease. It is also worth to note none of the patients with viremia blood had positive swabs. These patients would likely be considered as 2019-nCoV negative through routine surveillance, and thus pose a threat to other people. In contrast, we found viral antibodies in near all patients, indicating serology should be considered for 2019-nCoV epidemiology. A possible shift from oral positive during early infection to anal swab positive during late infection can be observed. This observation implied that we cannot discharge a patient purely based on oral swabs negative, who may still shed the virus by oral-fecal route. Above all, we strongly suggest using viral IgM and IgG serological test to confirm an infection, considering the unreliable results from oral swabs detection.\\n\\nIn summary, we provide a cautionary warning that 2019-nCoV may be transmitted through multiple routes. Both molecular and serological tests are needed to definitively confirm a virus carrier.\", 'document_id': 2653}]}, {'paragraphs': [{'qas': [{'question': 'What is a Hantavirus?', 'id': 4220, 'answers': [{'text': 'zoonotic diseases', 'answer_start': 484}], 'is_impossible': False}, {'question': 'What plays a role in innate immunity to Hantavirus infection?', 'id': 5311, 'answers': [{'text': 'long noncoding RNAs (lncRNAs)', 'answer_start': 645}], 'is_impossible': False}, {'question': 'What evidence suggests that RIG-I and DDX60 collaborate to exert antiviral effects?', 'id': 5312, 'answers': [{'text': 'colocalized after HTNV infection', 'answer_start': 16456}], 'is_impossible': False}, {'question': 'What was the major contribution of this study?', 'id': 5313, 'answers': [{'text': 'the first study to describe the role of NEAT1 in HTNV infection', 'answer_start': 34982}], 'is_impossible': False}], 'context': \"The Long Noncoding RNA NEAT1 Exerts Antihantaviral Effects by Acting as Positive Feedback for RIG-I Signaling\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC5391460/\\n\\nSHA: d18636f47e3c7dd93da309d556ba464d964fd24f\\n\\nAuthors: Ma, Hongwei; Han, Peijun; Ye, Wei; Chen, Hesong; Zheng, Xuyang; Cheng, Linfeng; Zhang, Liang; Yu, Lan; Wu, Xing'an; Xu, Zhikai; Lei, Yingfeng; Zhang, Fanglin\\nDate: 2017-04-13\\nDOI: 10.1128/jvi.02250-16\\nLicense: cc-by\\n\\nAbstract: Hantavirus infection, which causes zoonotic diseases with a high mortality rate in humans, has long been a global public health concern. Over the past decades, accumulating evidence suggests that long noncoding RNAs (lncRNAs) play key regulatory roles in innate immunity. However, the involvement of host lncRNAs in hantaviral control remains uncharacterized. In this study, we identified the lncRNA NEAT1 as a vital antiviral modulator. NEAT1 was dramatically upregulated after Hantaan virus (HTNV) infection, whereas its downregulation in vitro or in vivo delayed host innate immune responses and aggravated HTNV replication. Ectopic expression of NEAT1 enhanced beta interferon (IFN-β) production and suppressed HTNV infection. Further investigation suggested that NEAT1 served as positive feedback for RIG-I signaling. HTNV infection activated NEAT1 transcription through the RIG-I–IRF7 pathway, whereas NEAT1 removed the transcriptional inhibitory effects of the splicing factor proline- and glutamine-rich protein (SFPQ) by relocating SFPQ to paraspeckles, thus promoting the expression of RIG-I and DDX60. RIG-I and DDX60 had synergic effects on IFN production. Taken together, our findings demonstrate that NEAT1 modulates the innate immune response against HTNV infection, providing another layer of information about the role of lncRNAs in controlling viral infections. IMPORTANCE Hantaviruses have attracted worldwide attention as archetypal emerging pathogens. Recently, increasing evidence has highlighted long noncoding RNAs (lncRNAs) as key regulators of innate immunity; however, their roles in hantavirus infection remain unknown. In the present work, a new unexplored function of lncRNA NEAT1 in controlling HTNV replication was found. NEAT1 promoted interferon (IFN) responses by acting as positive feedback for RIG-I signaling. This lncRNA was induced by HTNV through the RIG-I–IRF7 pathway in a time- and dose-dependent manner and promoted HTNV-induced IFN production by facilitating RIG-I and DDX60 expression. Intriguingly, NEAT1 relocated SFPQ and formed paraspeckles after HTNV infection, which might reverse inhibitive effects of SFPQ on the transcription of RIG-I and DDX60. To the best of our knowledge, this is the first study to address the regulatory role of the lncRNA NEAT1 in host innate immunity after HTNV infection. In summary, our findings provide additional insights regarding the role of lncRNAs in controlling viral infections.\\n\\nText: glycoprotein (GP), and viral RNA-dependent polymerase protein (RdRp), respectively. Humans become infected by inhaling contaminated aerosols or by coming into contact with rodent excreta, and they develop two severe acute diseases, namely, hemorrhagic fever with renal syndrome (HFRS) and hantavirus pulmonary syndrome (HPS) (2) . Hantavirus infection affects up to 100,000 to 200,000 humans annually, with fulminant HFRS cases most represented in China (3) . Chinese HFRS cases, mainly caused by Hantaan virus (HTNV) infection, account for approximately 90% of all global cases, with a mortality rate ranging from 0.1 to 15% (4) . Since there is neither an effective therapeutic nor FDA-licensed vaccine, further understanding of host immune responses against hantaviral infection is of great significance for global public health and safety. The innate immune system, characterized by interferon (IFN) responses and immunocyte activation, provides the initial defense against viral invasions. Cellular pathogen recognition receptors (PRRs), including Toll-like receptors (TLRs) and RIG-I like receptors (RLRs), can detect distinct pathogen-associated molecular patterns (PAMPs) and trigger the expression of IFNs and cytokines. RIG-I has been shown to recognize hantaviral invasion, but its regulatory process remains unclear (5) . Long noncoding RNAs (lncRNAs) have emerged as important modulators of gene expression. lncRNA nuclear paraspeckle assembly transcript 1 (NEAT1) is an essential architectural constituent of paraspeckles in the mammalian nucleus, interacting with Drosophila DBHS RNA-binding proteins such as the splicing factor proline-and glutamine-rich protein (SFPQ) and the non-POU domain-containing, octamer-binding protein (NONO/p54) (6) . To date, two isoform transcripts of the NEAT1 gene have been identified, namely, the 3.7-kb NEAT1-1 (MEN) and the 23-kb NEAT1-2 (MEN␤) (Fig. 1A) . A large amount of research has shown that NEAT1 is associated with oncogenesis and tumor progression (7) (8) (9) , promoting cancer formation in mice by dampening oncogene-dependent activation of p53 (10) . Nevertheless, studies assessing the function of NEAT1 in viral infections are scarce.\\n\\nHere, the human umbilical vein endothelial cell (HUVEC) transcriptome was analyzed after HTNV infection by digital gene expression (DGE) profiling, and lncRNA NEAT1 was found to be remarkably upregulated by viral infection. Silencing NEAT1 in vitro or in vivo suppressed host immune responses and aggravated HTNV infection, whereas NEAT1 overexpression in vitro enhanced beta interferon (IFN-␤) production and inhibited HTNV replication. Further investigation showed that NEAT1 promoted RIG-I and DDX60 expression by relocating SFPQ and removing the transcriptional inhibitory effects of SFPQ, which are critical for IFN responses against HTNV infection. We also found that RIG-I signaling, rather than TLR3 and TLR4, accounted for the elevation of HTNV-induced NEAT1. Taken together, our findings provide novel insights into the lncRNA-mediated regulatory mechanism of host innate defense against HTNV infection.\\n\\nTo explore the potential role of long noncoding RNAs in host innate immune responses, DGE analysis of HUVECs for whole-genome profiling was performed at 24 h post-HTNV infection. As shown in Fig. 1B , the NEAT1 level in the HTNV group was higher than that in the mock group (P ϭ 6.86 ϫ 10 Ϫ13 , false discovery rate [FDR] ϭ 9.75 ϫ 10 Ϫ12 ) or the 60 Co-inactivated HTNV group (P ϭ 1.75 ϫ 10 Ϫ14 , FDR ϭ 3.10 ϫ 10 Ϫ13 ); however, the difference between the 60 Co-inactivated HTNV group and the mock group was not significant (P ϭ 0.21034, FDR ϭ 0.58211). To confirm the profiling results, two primer pairs from the published literature (11) , one recognizing both NEAT1-1 and NEAT1-2 and the other specific for NEAT1-2 (Fig. 1A) , were applied to quantify NEAT1 RNA isoforms by quantitative real-time PCR (qRT-PCR). It has been reported that NEAT1-2 rather than NEAT1-1 plays a key regulatory role in paraspeckle formation (11) , and we also found that elevated NEAT1 levels depend on live HTNV infection rather than 60 Co-inactivated HTNV stimulation (Fig. 1C) . Fluorescence in situ hybridization (FISH) with probes specific for NEAT1-2 was performed with HUVECs, and the results confirmed increased NEAT1-2 expression and the aggregation of NEAT1-2 in the nucleus at 24 and 48 h postinfection (hpi) (Fig. 1D) .\\n\\nTo further investigate whether NEAT1 expression was altered in other cell lines, HEK293, HeLa, and A549 cells were used. All these cells could be infected by HTNV ( Fig.  1E and F) and generated hantavirus progeny (Fig. 1G ). Similar to the data obtained from HUVECs, NEAT1 was indeed upregulated by HTNV at a multiplicity of infection (MOI) of 1 beginning at 24 hpi in HUVECs and A549, HEK293, and HeLa cells, and the increasing tendency occurred in a time-dependent manner (Fig. 1H ). Of note, the NEAT1 elevation at 2 hpi might have been unrelated to the virus but resulted in cellular stress responses. Besides, NEAT1 expression increased from an MOI of 0.1 to 1, indicating that the elevation occurred in a viral dose-dependent manner (Fig. 1I) .\\n\\nNEAT1-2 and not NEAT1-1 suppresses HTNV replication in HUVECs. The abovedescribed data showed that HTNV infection increased NEAT1, and we wondered how NEAT1 could reciprocally influence HTNV replication. The small interfering RNA (siRNA) transfection efficiency in HUVECs was confirmed by flow cytometry, and NEAT1 expression was significantly decreased, as assessed by qRT-PCR after RNA interference (RNAi) ( Fig. 2A) . Of note, si-NEAT1 targets both NEAT1-1 and NEAT1-2, whereas the stealth siRNA NEAT1-2 (st-NEAT1-2) is specific for NEAT1-2. Compared with the cells transfected with control siRNA (negative control [NC]), HUVECs with si-NEAT1 could dramatically promote HTNV NP production, and NP expression seemed to be related to the amount of applied si-NEAT1 (Fig. 2B) . Intriguingly, depletion of NEAT1-2 alone could mimic the antiviral effects of simultaneous NEAT1-1 and NEAT1-2 silencing (Fig. 2C) , indicating that NEAT1-2 was critical for the antiviral responses. Consistent with those data, the expressions of HTNV mRNA of S segment (Fig. 2D ) and HTNV titers (Fig. 2E ) were increased after NEAT1 silencing.\\n\\nOn the other hand, plasmids, of which pCMV-NEAT1-1 is transcribed into the 3.7-kb NEAT1-1 (MEN) and pCMV-NEAT1-2 is transcribed into the 2-to 3-kb NEAT1-2 (MEN␤), were applied to directly investigate the role of NEAT1 in HTNV infection (Fig. 2F) . Surprisingly, we found NEAT1-1 overexpression restricted NEAT1-2 transcription (Fig.  2F ). Overexpression of NEAT1 with both pCMV-NEAT1-1 and pCMV-NEAT1-2 could conspicuously repress HTNV NP expression, and NP expression seemed to be associated with the transfected plasmids (Fig. 2G) . Furthermore, overexpression of NEAT1-2 instead of NEAT1-1 could efficiently suppress HTNV replication (Fig. 2H ). NEAT1-1 upregulation even aggravated HTNV infection (Fig. 2H ), which may be the result of downregulation of NEAT1-2. Consistently, through analysis of viral load detected by qRT-PCR and the 50% tissue culture infective dose (TCID 50 ) test by ELISA, we found that expression of HTNV-specific mRNA (Fig. 2I ) and HTNV titers (Fig. 2J ) were limited in HUVECs in which NEAT1-2 was ectopically expressed in comparison to those transfected with control vector or pCMV-NEAT1-1. These data further showed that NEAT1-2 and not NEAT1-1 suppresses HTNV replication in HUVECs.\\n\\nAlteration of NEAT1-2 affects HTNV-induced IFN expression in HUVECs. IFN-␤ production or pretreatment at an early infection stage plays an important role in limiting HTNV infection, while IFN-␤ treatment after 24 hpi exerts little antiviral effect (12, 13) . It has been reported that the GnT of hantaviruses suppressed IFN-␤ expression of host cells at an early stage of infection (14) . Here, we also found that HUVECs could not efficiently produce IFN-␤ until 12 hpi at an MOI of 0.1 or until 24 hpi at an MOI of 1 (Fig. 3A) , which indicated that high doses of HTNV could hamper prompt IFN responses. Notably, enhanced NEAT1-2 transcription appeared at 8 hpi at an MOI of 0.1 or at 20 hpi at an MOI of 1 (Fig. 3B) , suggesting that NEAT1-2 expression increased just before IFN production. We found that expression of endogenous IFN-␤ mRNA was much lower in cells transfected with si-NEAT1-2 at MOIs of both 0.1 (Fig. 3C ) and 1 (Fig.  3D ) than in those transfected with control siRNA (NC). In contrast, overexpression of NEAT1 in HUVECs increased IFN-␤ expression after HTNV infection (MOI ϭ 1) at 24 and 48 hpi (Fig. 3E ). More importantly, HUVECs transfected with pCMV-NEAT1-2 conspicuously increased IFN-␤ gene expression compared with those cells with vector plasmids at 12 hpi (MOI ϭ 1), demonstrating that NEAT1-2 overexpression accelerated robust IFN responses in host cells against HTNV infection. With a dual luciferase reporter system Twenty-four hours after transfection, the cells expressing FAM were calculated by flow cytometry. Right, HUVECs in six-well plates were transfected with NC sequences, si-NEAT1, or the stealth siRNA NEAT1-2 (st-NEAT1-2) (3 g). Twenty-four hours after transfection, the cells were infected with HTNV at an MOI of 1. At 48 hpi, the NEAT1 expression levels were measured by qRT-PCR. Values are means Ϯ SD (n ϭ 3; *, P Ͻ 0.01; **, P Ͻ 0.001; ***, P Ͻ 0.0001; Student's t test, compared with the NC group). NS, nonsignificant. (B) HUVECs in six-well plates were transfected with NC sequences (the amount of Si-NEAT1-2 is considered 0 g) or increasing amounts of si-NEAT1 (0.1, 0.5, 1, and 3 g). Twenty-four hours after transfection, the cells were infected with HTNV at an MOI of 0.1 for 48 h. The expression of HTNV NP was measured by Western blotting. (C) HUVECs were treated as described for panel A, right, but at an MOI of 0.1. In-cell Western (ICW) analysis for HTNV NP was performed at 48 hpi. The ICW for HTNV NP staining is shown on the left, while the relative intensity of fluorescence (NP/␤-actin) was analyzed using Student's t test. (n ϭ 4; *, P Ͻ 0.01; **, P Ͻ 0.001; Student's t test, compared with the NC group). (D) HUVECs were treated as described for panel A, right, but at an MOI of 0.1. The expression of HTNV S segment was measured by qRT-PCR. Values are means Ϯ SD (n ϭ 3; *, P Ͻ 0.01; Student's t test, compared with the NC group). (E) HUVECs were treated as described for panel A, right, but at an MOI of 0.1. The propagated HTNV was acquired at 72 hpi, and viral titers were detected by TCID 50 with ELISA in Vero E6 cells. Values are means Ϯ SD (n ϭ 3; *, P Ͻ 0.01; Student's t test, compared with the NC group). (F) Left, HUVECs in six-well plates were transfected with vectors or pGFP (3 g). Twenty-four hours after transfection, the cells expressing green fluorescent protein (GFP) were calculated by flow cytometry. Right, HUVECs in six-well plates were transfected with control plasmids (vector), pCMV-NEAT1-1, or pCMV-NEAT1-2 (3 g). Twenty-four hours after transfection, the cells were infected with HTNV at an MOI of 1. At (Continued on next page) NEAT1 Promotes Innate Antiviral Responses Journal of Virology maintaining IFN-␤ promoters, we found NEAT1-2 silencing or overexpression could inhibit or increase the promoter activity of the IFN-␤ gene after HTNV infection, respectively, whereas silencing NEAT1-2 or ectopically expressing NEAT1-2 without HTNV infection could not inhibit or enhance IFN-␤ expression (Fig. 3F) . These results showed that NEAT1-2 regulated HTNV-induced IFN-␤ expression.\\n\\nTo explore whether the antihantavirus effects of NEAT1 were caused by IFN-␤ alteration, a series of compensatory experiments was designed. In NEAT1-2 knockdown HUVECs, the addition of IFN-␤ at 12 hpi could efficiently block HTNV NP production (MOI ϭ 0.1), and such phenomena were also determined by the amount of applied IFN-␤ (Fig. 3G ). In addition, in cells with high NEAT1-2 expression, treatment with neutralizing antibodies (NAbs) of IFN-␣ and IFN-␤ could counteract the antiviral effects of NEAT1-2 (MOI ϭ 1), and the compensatory effects were dependent on the magnitude of the NAbs. Together these results demonstrated that NEAT1-2 especially enhanced the host antihantaviral innate immune responses by regulating IFN-␤ signaling.\\n\\nRIG-I and DDX60 regulated by NEAT1-2 facilitate HTNV-induced IFN-␤ production. PRRs maintain a vital role in the promotion of IFN responses, and we conjectured that NEAT1 might amplify IFN responses by modulating these molecules. TLR3, TLR4, and RIG-I have been shown to recognize HTNV infection (5, 15, 16) . DDX60 was recently reported as an important activator of RIG-I, but the antiviral effects of DDX60 remain a subject of debate (17, 18) , Here, we found that multiple Toll-like receptors like TLR1, TLR2, TLR3, and TLR4, as well as MDA5, were increased after HTNV infection, but none of them were influenced by silencing NEAT1-2 ( Fig. 4A ). The upregulated RIG-I and DDX60 were blocked in the cells with low NEAT1-2 expression after HTNV infection (Fig.  4A ). HUVECs with declining NEAT1-2 expression showed gradually decreasing expression of RIG-I and DDX60 (Fig. 4B) , and increasing NEAT1-2 transcription was found to activate RIG-I and DDX60 production accordingly (Fig. 4C ). These data indicated that NEAT1-2 could positively modulate RIG-I and DDX60 expression, while the role of RIG-I and DDX60 upon HTNV infection is obscure.\\n\\nWe then found that RIG-I and DDX60 colocalized after HTNV infection (Fig. 4D ), implying that RIG-I and DDX60 might collaborate with each other to exert antiviral effects. To verify the antiviral role of RIG-I and DDX60, we designed a series of siRNAs targeting RIG-I and DDX60, and we selected the si-RIG-I-2 and siRNA-DDX60-1 with the highest knockdown efficiency by qRT-PCR in HUVECs (data not shown). Simultaneously knocking down RIG-I and DDX60 significantly promoted HTNV NP expression (Fig. 4E ), and knockdown of both of them could greatly affect IFN-␤ expression ( Fig. 4F and G). Ectopic expression of either RIG-I or DDX60 inhibited viral replication, whereas overexpression of both resulted in superior antiviral effects (Fig. 4H ), indicating that efficient anti-HTNV responses might depend on the interactive effects of DDX60 and RIG-I. More importantly, RIG-I or/and DDX60 overexpression enhanced HTNV-induced IFN-␤ expression, and they had synergistic effects on IFN-␤ production ( Fig. 4I and J). Consequently, NEAT1 might regulate IFN-␤ production by upregulating RIG-I and DDX60, and thus we were interested in how NEAT1 regulated RIG-I and DDX60 expression.\\n\\nSFPQ, which is relocated by NEAT1 HTNV infection, regulates the expression of RIG-I and DDX60. NEAT1 was found to interact with SFPQ by RNA immunoprecipitation (RIP) after HTNV infection (Fig. 5A) , indicating that modulatory effects of NEAT1 48 hpi, the NEAT1 expression levels were measured by qRT-PCR. Values are means Ϯ SD (n ϭ 3; *, P Ͻ 0.01; **, P Ͻ 0.001; ***, P Ͻ 0.0001; Student's t test, compared with the vector group). (G) HUVECs in six-well plates were transfected with control plasmids (vector, the amount of pCMV-NEAT1-1 plus pCMV-NEAT1-2 is considered 0 g) or increasing amounts of pCMV-NEAT1-1 plus pCMV-NEAT1-2 (0.05 ϩ 0.05 g, 0.25 ϩ 0.25 g, 0.5 ϩ 0.5 g, 1.5 ϩ 1.5 g, respectively). Twenty-four hours after transfection, the cells were infected with HTNV at an MOI of 1 for 48 h. The expression of HTNV NP was measured by Western blotting. (H) HUVECs were treated as described for panel F, right. In-Cell Western (ICW) analysis for HTNV NP was performed at 48 hpi. The ICW for HTNV NP staining is shown on the left, while the relative intensity of fluorescence (NP/␤-actin) was analyzed using Student's t test. (n ϭ 4; *, P Ͻ 0.01; **, P Ͻ 0.001; Student's t test, compared with the vector group). (H) HUVECs were treated as described for panel F, right. The expression of HTNV S segment was measured by qRT-PCR. Values are means Ϯ SD (n ϭ 3; *, P Ͻ 0.01; **, P Ͻ 0.001; Student's t test, compared with the vector group). (I) HUVECs were treated as described for panel F, right. The propagated HTNV was acquired at 72 hpi, and viral titers were detected by TCID 50 with ELISA in Vero E6 cells. Values are means Ϯ SD (n ϭ 3; *, P Ͻ 0.01; Student's t test, compared with the vector group). might be involved in SFPQ. Interestingly, the protein level of SFPQ, as well as another paraspeckle-forming constituent, NONO, remained unchanged after HTNV infection (Fig. 5B ) or after NEAT1 overexpression and knockdown (Fig. 5C ). However, SFPQ became centralized rather than diffuse in the nucleus after HTNV infection (Fig. 5D) . The enhanced interaction of SFPQ and NONO indicated excess formation of paraspeckles in the nucleus (Fig. 5E ) and relocalization of SFPQ. SFPQ knockdown could inhibit HTNV replication ( Fig. 5F and G), which might have been related to the increase in RIG-I (Fig.  5H ) and DDX60 (Fig. 5I ). SFPQ has been suggested to bind to the promoter region of RIG-I and DDX60 (11), thus preventing the expression of RIG-I and DDX60. Taken together, the above results suggested that NEAT1 might relocate SFPQ from the RIG-I signaling is crucial for NEAT1 expression after HTNV infection. Elevated NEAT1 exerts antiviral effects by modulating the innate immune response, yet it is unclear how HTNV triggers NEAT1 transcription. Interestingly, overexpression of the S or M segment of HTNV in HEK293 cells failed to induce NEAT1 expression, suggesting that NEAT1 transcription was closely related to live viral replication (Fig. 6A) . Of note, the upregulation of NEAT1 by HTNV could not be reversed by applying IFN-I neutralizing antibodies (Fig. 6B) . Meanwhile, NEAT1 expression could not be induced by stimulation with different types of IFNs (Fig. 6C, D, and E) or cytokines (Fig. 6F and G) .\\n\\nWe conjectured that NEAT1 expression was related to the activation of PRRs. By knocking down several PRRs, we found that the RIG-I and TLR4 pathways played important roles in HTNV-induced NEAT1 upregulation (Fig. 7A) . Using RIG-I-and TLR4-deficient cell lines which could be well infected by HTNV (Fig. 7B) , RIG-I was confirmed to be indispensable for NEAT1 induction after HTNV infection (Fig. 7C) . Moreover, using STAT1 as a positive control, we found that the transcription factor IRF7, rather than IRF3 and p65, translocated into the nucleus in HTNV-infected HUVECs at 2 dpi (Fig. 7D) . Furthermore, IRF7 knockdown blocked HTNV-induced NEAT1 upregulation (Fig. 7E) . Therefore, HTNV caused transcriptional activation of the NEAT1 gene, probably via the RIG-I-IRF7 pathway.\\n\\nNEAT1 silencing has profound effects on innate immune responses after HTNV infection in mice. Although cell-based experiments revealed that NEAT1-2 is a crucial regulator of innate antihantaviral responses, its function in vivo has remained unclear. To address this question, we intravenously injected siRNAs targeting mouse NEAT1-2 at 1 day before HTNV infection. NEAT1-2 expression levels in the liver, kidney, and spleen were reduced at 2 dpi (Fig. 8A) . Previous studies have shown that NEAT1 knockout does not affect physiological processes except potentia generandi in mice; hence, we assessed its role under pathological conditions. Body weight loss in NEAT1-2-depleted mice was observed from 2 dpi to 5 dpi, and the IFN production in serum was remarkably decreased in the NEAT1-2 silenced group than those in the NC group at 3 dpi (Fig. 8B) . As expected, NEAT1-2 knockdown mice showed considerably higher HTNV NP levels in the liver, spleen, and kidney at 3 dpi (Fig. 8C) . Moreover, the virus titers in related organs were higher in the NEAT1-2 silenced group than in the NC group (Fig.  8D ). In addition, reduced inflammatory cell filtration but increased tissue injury was found in NEAT1-2 knockdown mice during the early stage of infection (Fig. 8E) . Infiltration of macrophages in the spleen was attenuated (Fig. 8F) , and the activation of macrophages was also suppressed (by flow cytometry; data not shown). Moreover, CD8 ϩ IFN-␥ ϩ T cells were reduced in the spleens of NEAT1-2 knockdown mice in comparison to those in the NC group at 3 dpi (Fig. 8G) . Nevertheless, NEAT1-2 silencing had no effect on the production of neutralizing antibodies at 7 dpi (data not shown). The above-described findings indicated that NEAT1-2 depletion might influence multiple aspects of the innate immune response in HTNV-infected mice.\\n\\nInnate immunity is a phylogenetically ancient and conserved system that counteracts invading microbes, the regulatory mechanism of which is sophisticated and complex. Long noncoding RNAs, which were once considered dark materials in the mammalian genome, have been shown to exert vital modulatory effects on host innate immunity (19) . In this report, we first demonstrated that NEAT1 was induced by HTNV through the RIG-I-IRF7 pathway and served as positive feedback for RIG-I signaling.\\n\\nUsing DGE analysis, we observed upregulated NEAT1 and confirmed its alteration in To further determine the function of NEAT1 after HTNV infection in vivo, mice were injected intravenously with si-NEAT1-2 (1 g/g) or nontarget control siRNA (NC) (1 g/g); 1 day later, they were infected with HTNV (100 LD 50 ) by intramuscular injection. (A) To maintain high knockdown efficiency, siRNAs were injected intravenously every other day. (A) The knockdown efficiency was assessed by qRT-PCR in kidney, liver, and spleen samples at 2 dpi (n ϭ 6 in each group). (B) The effects of NEAT1 on HTNV virulence in mice were determined by body weight loss from 0 to 10 dpi (left panel, n ϭ 10 in each group). The IFN-␤ in sera of different groups was measured by ELISA at 3dpi (right panel, n ϭ 8 in each group). Values are means Ϯ SD (*, P Ͻ 0.01; **, P Ͻ 0.001; Student's t test, compared with the NCϩHTNV group). (C) Mice were sacrificed at 3 dpi, and livers, spleens, and kidneys were collected for ELISA detection of HTNV NP titers (upper panels, n ϭ 8 in each group) and qRT-PCR to assess HTNV S segment levels (bottom panels, n ϭ 8 in each group) at 3 dpi. Values are means Ϯ SD (*, P Ͻ 0.01; **, P Ͻ 0.001; Student's t test, compared with the NCϩHTNV group). (D) NEAT1 effects on HTNV infection kinetics at 3 dpi were determined by testing the HTNV titers in livers, spleens, and kidneys. Values are means Ϯ SD (n ϭ 8; *, P Ͻ 0.01; **, P Ͻ 0.001; Student's t test, compared with the NCϩHTNV group). (E) Hematoxylin and eosin (H&E) staining for mouse liver, spleen, or kidney specimens was performed (3 dpi, n ϭ 8 in each group). After HTNV infection, livers in the NC group showed inflammatory cell infiltration in certain regions, while those in the si-NEAT1-2 group showed slight acute viral hepatitis. Spleens in the NC group showed lymph node hyperplasia, while those in the si-NEAT1-2 group were severely congestive. Kidneys in the NC group also showed inflammatory cell infiltration, while those in the si-NEAT1-2 group had moderate interstitial congestion. (F) Macrophage infiltration in spleens was analyzed by detecting CD11b and F4/80 by flow cytometry at 3 dpi, and the results obtained for four mice in each group are presented. (G) CD3 ϩ CD8 ϩ IFN-␥ ϩ T cells were analyzed by flow cytometry at 3 dpi, and the results obtained for three mice in each group are presented. different cell lines. To assess its effects on HTNV replication, NEAT1 was silenced both in vitro and in vivo, which resulted in increased HTNV infection and suppressed innate immune responses. Further analysis indicated that NEAT1 might interact with SFPQ and regulate DDX60 and RIG-I expression. By virtue of RNAi, the RIG-I-IRF7 pathway was confirmed to be necessary for HTNV-triggered NEAT1 elevation.\\n\\nRecently, large-scale transcriptomic studies identified numerous noncoding transcripts in the mammalian genome, which were speculated to influence diverse biological processes. Among these noncoding RNAs (ncRNAs), long noncoding RNAs (lncRNAs) emerged as important regulators of gene expression and are closely related to the activation of the host innate immune system. TLR2 controls lncRNA-COX2 expression in a MyD88-and NF-B-dependent manner, whereas lncRNA-COX either promotes interleukin 6 (IL-6) secretion or represses ISG15 and CCL5 expression (20) . TLR2 activation or tumor necrosis factor alpha (TNF-␣) stimulation induces transcription of the lncRNA THRIL, the downregulation of which impairs TNF-␣ and IL-6 secretion (21) . TLR4 signaling in response to lipopolysaccharide (LPS) induces lncRNA IL-1␤-eRNA and IL-1␤-RBT46, the knockdown of which attenuates IL-1␤ and CXCL8 release (22) . The lncRNA Lethe, triggered by TNF-␣ and IL-1␤, acts as a negative feedback regulator of NF-B signaling (23) .\\n\\nThe roles of lncRNAs in host-virus interactions have been progressively unveiled. Various viruses, such as influenza virus (IAV), coronavirus, enterovirus, human immunodeficiency virus (HIV), hepatitis B virus (HBV), hepatitis C virus (HCV), Japanese encephalitis virus (JEV), and rabies virus, have been reported to activate the transcription of different lncRNAs in host cells (11, (24) (25) (26) . Importantly, multiple lncRNAs have been shown to affect the IFN response in recent years and have gradually become hot spots in the field of antiviral research. NeST was shown to enhance IFN-␥ production, controlling the susceptibility of mice to persistent Theiler's virus infection as well as resistance to Salmonella enterica serovar Typhimurium infection (27) . Both CMPK2 and NRAV were identified as negative regulators of IFN immune reactions. CMPK2, induced by IFN-␣ or HCV infection, suppresses various ISGs, the knockdown of which dramatically blocks HCV replication (26) . NRAV inhibits some critical ISGs, such as IFITM3 and Mx1, the depletion of which suppresses IAV replication both in vitro and in vivo (25) . Numerous lncRNAs, including lnc-ISG15 and ISR2, respond to IFNs such as ISGs, although their actual function requires further investigation (28) . Considering the poor evolutionary conservation but rapid divergence of lncRNAs, their functions may be highly species and virus specific. Though considerable progress has been achieved to demonstrate the antiviral effects of lncRNAs on model viruses, there are no published reports assessing the role of lncRNAs in hantaviral infection. NEAT1 has been reported to interact with Drosophila DBHS RNA-binding proteins (e.g., SFPQ, NONO-p54nrb, and PSPC1), recruiting them to paraspeckles, a nuclear substructure found in all cultured and primary cells except embryonic stem cells (24) . The versatile function of NEAT1 is rapidly progressing in multiple areas of biology. NEAT1 has been reported to be involved in the pathogenesis of multiple types of cancer (7) (8) (9) . NEAT1 also participates in neurodegenerative diseases such as Huntington's disease (29) and seems to potentially contribute to the elevated production of a number of cytokines and chemokines in patients with systemic lupus erythematosus (SLE) (30) . Furthermore, poly I·C can activate NEAT1 transcription through the TLR3 pathway, whereas NEAT1 positively regulates IL-8 transcription and potentially affects the expression of multiple ISGs after poly I·C stimulation (11) . In addition, NEAT1 has been reported to suppress the export of Rev-dependent instability element (INS)containing HIV-1 mRNAs from the nucleus to the cytoplasm, thus inhibiting HIV replication (24) . However, the role of NEAT1 in hantaviral infection remains unclear.\\n\\nIn this report, NEAT1 has been identified as an important regulator of the host innate immune system against HTNV infection. Elevated NEAT1 promotes IFN secretion, most likely by enhancing RIG-I and DDX60 expression. DDX60, a DEXD/H box RNA helicase similar to Saccharomyces cerevisiae Ski2, is induced after viral infection (31) . DDX60 recognizes viral RNA and activates endogenous RIG-I, thereby promoting the RIG-I signaling-related IFN response. However, the antiviral effects of DDX60 seem to vary among viruses (17) . We found that NEAT1-regulated DDX60 was involved in IFN production in response to HTNV infection. In HTNV-infected cells, double-stranded RNA (dsRNA) could not be detected, and it is unclear how host PRRs, especially RIG-I, recognize HTNV invasion (5) . Here, considering the interaction of RIG-I and DDX60 and the effect of DDX60 on IFN-␤ production, we hypothesize that DDX60 might mediate RIG-I signaling activation upon HTNV infection, which requires further investigation.\\n\\nOf note, we applied multiple cell lines to explore the role of NEAT1 during HTNV infection. HTNV primarily targets vascular endothelial cells in vivo and contributes to the increased vascular permeability and coagulation disorders in HFRS; hence, HUVECs are the most common in vitro cell model to study host innate immunity against HTNV infection or viral pathogenesis (32) . EVC-304 cells are also endovascular cells, whereas EVC-304 TLR4 Ϫ cells are TLR4-deleted cells, both of which have been used for HTNV infection related studies (15, 33) . A549 cells were once used to isolate HTNV, and they were confirmed to be a mature model of infection (34) (35) (36) (37) . Additionally, Huh 7.0 and Huh 7.5 (RIG-I Ϫ ) cells used in our study have been reported to be infected by HTNV by Lee et al. (5) and can be used as a cell model to study immune responses against HTNV replication (38, 39) . Additionally, HEK293 (40) and HeLa (41) cells have been reported to be infected by HTNV. Using qRT-PCR, Western blotting, and immunofluorescence assays, we have also shown that both HEK293 and HeLa cells can be infected by HTNV.\\n\\nTo study the molecular mechanism underlying the effect of NEAT1 on IFN expression and HTNV infection, it may be suitable to use HEK293 and HeLa cells as a cell model, especially under conditions in which HTNV NP can be detected using Western blot or immunofluorescence analyses.\\n\\nIn experiments to assess the effect of NEAT1 on the control of hantaviruses, In-Cell Western (ICW) analysis was applied to qualify HTNV NP production. Alterations in the relative fluorescence intensity of NP after silencing or overexpressing NEAT1-2 did not seem to be as remarkable as qRT-PCR or Western blot analysis results. The NP spotted and exhibited in the ICW results forms obvious stains that mimic PFU. However, the specific values scanned and analyzed by the ICW assay reflect only the fluorescence intensity of the integral well instead of the number of spots. As a consequence, the intensity represented the quantity of NP production but could not directly indicate the virulence, which was better shown by plaque-forming assays. The RNAi studies in vivo are encouraging (Fig. 8) , but the NC used by our group was not mutated si-NEAT1-2 (i.e., same sense strand, but with a point mutation in the targeting strand). The results would be more compelling if the control mice had been treated with the mutated si-NEAT1-2.\\n\\nOne major finding of our study is that the lncRNA NEAT1 serves as positive feedback for RIG-I signaling. After observing that NEAT1 can regulate IFN expression by HTNV infection, we were interested in the function of NEAT1. We noticed that silencing NEAT1-2 or ectopically expressing NEAT1-2 could not inhibit or enhance IFN expression without HTNV infection (Fig. 3F) , which indicated that NEAT1-2 could not directly affect IFN-␤ expression. This finding excludes the possibility that NEAT1-2 directly promoted IFN-␤ and that IFN-␤ promoted the expression of PRRs such as RIG-I. Thereafter, NEAT1 was found to modulate HTNV-induced RIG-I and DDX60 expression. Recent findings have shown that RIG-I signaling is essential for an efficient polyfunctional T cell response during IAV infection (42) . Indeed, we found that the function of T cells was suppressed after NEAT1-2 depletion in our animal experiments (Fig. 8G) , which might be due to the disrupted RIG-I signaling in NEAT1-2 silenced T cells.\\n\\nIn conclusion, this is the first study to describe the role of NEAT1 in HTNV infection. HTNV infection induced NEAT1 expression through the RIG-I-IRF7 pathway, while NEAT1 displayed positive feedback for RIG-I signaling. NEAT1 relocated SFPQ from the potential promoter region of several antiviral genes to the paraspeckles, removing the transcriptional inhibitory effects of SFPQ. This phenomenon would facilitate the expression of DDX60 and RIG-I, thus promoting IFN responses and suppressing HTNV infection (Fig. 9) . nontarget control (i.e., negative control [NC]) or targeted RIG-I and DDX60 were designed by Gene-Pharma as follows: NC, 5=- UUCUUCGAACGUGUCACGUTT-3=; si-RIG-I-1, 5=-GCCCAUUUAAACCAAGAAATT-3=, si-RIG-I-2, 5=-GGUGGAGGAUAUUUGAACUTT-3=, and si-RIG-I-3, 5=-CCCAACGAUAUCAUUUCUTT-3=; si-DDX60-1, 5=-GUCCAGGUGUCAGUUUGAUTT-3=, si-DDX60-2, 5=-CCGAAGUGAAGAAGGUAAATT-3=, and si-DDX60-3, 5=-GAUGGAUGCUAGGAAAUAUTT-3=. The pCMV-NEAT1-1 and pCMV-NEAT1-2 plasmids, which transcribe NEAT1-1 and NEAT1-2, respectively, were provided by Nakagawa Shinichi (10) . The Flag-RIG-I and pUNO-DDX60 plasmids were purchased from Invitrogen.\\n\\nReagents. Mouse monoclonal antibody (MAb) 1A8 for the HTNV nucleocapsid protein (NP) was produced as previously described (43) . Abs against RIG-I, IRF3, IRF7, p65, and STAT1, as well as the neutralizing antibodies against IFN-␣ and IFN-␤, were purchased from Abcam (Cambridge, MA, USA). Phorbol myristate acetate (PMA) and Ab against DDX60 were purchased from Sigma-Aldrich, Inc. (St. Louis, MO, USA). Abs against SFPQ, NONO, GAPDH (glyceraldehyde-3-phosphate dehydrogenase), and ␤-actin were purchased from Protein Tech, Inc. (Wuhan, China). The Abs targeting CD11b, F4/80, CD3, CD8, IFN-␥, inducible nitric oxide synthase (iNOS), and CD206 for flow cytometry were purchased from BD Biosciences (San Jose, CA, USA). IFN-␣, -␤, and -␥, TNF-␣, and IL-1␤ were from PeperoTech (Rocky Hill, NJ). ELISA kits for IFN-␤ detection were manufactured by R&D Systems, Inc. (Minneapolis, MN, USA) .\\n\\nRNA extraction and quantitative real-time PCR (qRT-PCR) analysis. Total cellular RNAs were extracted with RNAiso (TaKaRa, Dalian, China), the concentration of which was measured using a NanoDrop 1000 spectrophotometer. Reverse transcription (RT) was then performed with PrimeScript RT master mix (TaKaRa) according to the instructions provided by the manufacturer. Each cDNA was denatured at 95°C for 5 min and amplified for 40 cycles of 15 s at 98°C, 30 s at 58°C, and 30 s at 72°C using a LightCycler 96 (Roche, Basel, Switzerland). The mRNA expression level of each target gene was normalized to the respective ␤-actin and analyzed. The qRT-PCR primer sequences for NEAT1, NEAT1-2, IFN-␤, HTNV S segment, RIG-I, DDX60, ␤-actin, and GAPDH were obtained from previous reports (24, 45) . The methods used to quantify HTNV RNA load have been described by our group previously (46) . DGE analysis and lncRNA sequencing. HUVECs with a confluence of 80% in 6 wells were mock infected or infected with live or 60 Co-inactivated HTNV at an MOI of 1. RNAs were extracted as previously described at 24 hpi, and the quality was analyzed using FastQC software by the Beijing Genomics Institute (BGI, Shenzhen, China). Digital gene expression (DGE) tags were annotated to the human transcriptome (Ensembl version 58) by mapping the reads to the sequence flanking NlaIII restriction sites on both coding and noncoding strands. Tags matching more than one gene region were discarded. Tag counts were normalized to TPM (transcripts per million) by dividing the raw tag count by the total number of tags from each library and multiplying by 1 million. To avoid the possible noise signal from high-throughput sequencing, the genes with average TPM of less than 1 in these three states were excluded. In this study, an absolute fold change of no less than 1.5 and a false discovery rate (FDR) of less than 0.001 were used to define the differentially expressed genes. Genes were selected as differentially expressed using a P value threshold of 0.01. Genes were selected as differentially expressed using a P value threshold of 0.01.\\n\\nFISH and immunofluorescence assays (IFA). Fluorescence in situ hybridization (FISH) was performed with a FISH kit (Ribobio Co.) according to the manufacturer's instructions. In brief, cells were fixed with 4% paraformaldehyde (PFA) for 10 min at room temperature and permeabilized with 0.5% Triton X-100 for 15 min at room temperature. Prehybridization was performed with lncRNA FISH probe mix at 37°C for 30 min, and then hybridization was performed by adding NEAT1-2 FISH probe mix and incubating the mixture at 37°C overnight. After washing with 4ϫ, 2ϫ, and 1ϫ SSC, the cell nuclei were stained with DAPI (4=,6-diamidino-2-phenylindole). Finally, the samples were observed using a BX60 fluorescence microscope (Olympus, Tokyo, Japan).\\n\\nIFA was performed after FISH or independently. The cells were fixed with 4% PFA for 10 min and permeabilized with 0.1% Triton X-100 for 15 min. Primary Abs were added and incubated at 37°C for 2 h. After five washes with DPBS, secondary Cy3-or fluorescein isothiocyanate (FITC)-conjugated goat anti-rabbit or goat anti-mouse IgG (Sangon, Shanghai, China) was added and incubated at 37°C for 2 h. Cell nuclei were stained with DAPI. Finally, the samples were observed using a BX60 fluorescence microscope (Olympus).\\n\\nCells were washed twice with ice-cold DPBS and lysed with 1ϫ SDS protein loading buffer (50 mM Tris, 2% SDS, 10% glycerol, 2% 2-mercaptoethanol, and 0.1% bromophenol blue). The samples were then boiled at 95°C for 10 min. The lysates were resolved by 10%, 12%, or 15% SDS-PAGE and transferred to polyvinylidene fluoride (PVDF) membranes (Millipore). The membranes were incubated with the primary antibodies, followed by secondary antibodies labeled with infrared dyes (Li-Cor Biosciences, Lincoln, NE, USA). The signals on the PVDF membrane were visualized using an Odyssey infrared imaging system (Li-Cor Biosciences, Lincoln, NE, USA).\\n\\nICW assay. The In-Cell Western (ICW) assay was performed using an Odyssey imaging system (Li-Cor) according to the manufacturer's instructions. HUVECs were either mock transfected or transfected with NC sequences, si-NEAT1, st-NEAT1-2, vector plasmids, pCMV-NEAT1-1, or pCMV-NEAT1-2 and grown in 96-well plates (2 ϫ 10 4 cells/well). Twenty-four hours posttransfection, the cells were either infected or mock infected with HTNV at an MOI of 1. At 48 hpi, HUVECs were washed twice with ice-cold DPBS, fixed with 4% PFA for 10 min, and permeabilized with 1.0% Triton X-100 for 15 min. Cells were added with Li-Cor Odyssey blocking solution at room temperature for 30 min and incubated at 4°C overnight with mouse IgG MAb 1A8 against HTNV NP together with rabbit IgG antibody against ␤-actin, both of which were diluted in PBS containing 3% bovine serum albumin (BSA; HyClone). Subsequently, the cells were washed and stained with goat anti-mouse IgG IRDye 800 antibody (1:5,000; Li-Cor) and goat anti-rabbit\", 'document_id': 2652}]}, {'paragraphs': [{'qas': [{'question': 'What symptoms were reported?', 'id': 3820, 'answers': [{'text': 'Of the remaining 29, 20 reported fever, 14 reported cough and eight reported weakness. Additional symptoms reported included headaches (6 cases), sore throat (2), rhinorrhoea (2), shortness of breath (2), myalgia (1), diarrhoea (1) and nausea (1)', 'answer_start': 7275}], 'is_impossible': False}, {'question': 'What did the collected data include?', 'id': 3806, 'answers': [{'text': 'demographics, history of recent travel to affected areas, close contact with a probable or confirmed COVID-19 case, underlying conditions, signs and symptoms of disease at onset, type of specimens from which the virus was detected, and clinical outcome. ', 'answer_start': 3802}], 'is_impossible': False}, {'question': 'When was COVID  surveillance implemented in European region?', 'id': 3793, 'answers': [{'text': '27 January 2020', 'answer_start': 1324}], 'is_impossible': False}, {'question': 'As of 21 February, how many cases were reported?', 'id': 3794, 'answers': [{'text': '47', 'answer_start': 1891}], 'is_impossible': False}, {'question': 'Where were the cases that were studied?', 'id': 3795, 'answers': [{'text': '21 were linked to two clusters in Germany and France, 14 were infected in China', 'answer_start': 1925}], 'is_impossible': False}, {'question': 'What was the median case age?', 'id': 3796, 'answers': [{'text': '42 years', 'answer_start': 2026}], 'is_impossible': False}, {'question': 'How many were male?', 'id': 3797, 'answers': [{'text': ' 25 ', 'answer_start': 2035}], 'is_impossible': False}, {'question': 'How many cases were there on 5 March?', 'id': 3798, 'answers': [{'text': '4,250 ', 'answer_start': 2162}], 'is_impossible': False}, {'question': 'When did the Chinese authorities share the sequence of a novel coronavirus ?', 'id': 3799, 'answers': [{'text': '12 January 2020', 'answer_start': 2275}], 'is_impossible': False}, {'question': 'What is the name of the disease caused buy SARS-COV-2?', 'id': 3800, 'answers': [{'text': 'coronavirus disease 2019 (COVID -19)', 'answer_start': 2524}], 'is_impossible': False}, {'question': 'What country does this study exclude?', 'id': 3801, 'answers': [{'text': 'United Kingdom (UK)', 'answer_start': 2861}], 'is_impossible': False}, {'question': 'What does the study include?', 'id': 3802, 'answers': [{'text': 'a comparison between cases detected among travellers from China and cases whose infection was acquired due to subsequent local transmission.', 'answer_start': 2925}], 'is_impossible': False}, {'question': 'What did the ECDC and WHO regional office ask the countries?', 'id': 3803, 'answers': [{'text': ' to complete a WHO standard COVID-19 case report form for all confirmed and probable cases according to WHO criteria', 'answer_start': 3203}], 'is_impossible': False}, {'question': 'What was the overall aim of the surveillance?', 'id': 3804, 'answers': [{'text': ' to support the global strategy of containment of COVID-19 with rapid identification and follow-up of cases linked to affected countries in order to minimise onward transmission.', 'answer_start': 3382}], 'is_impossible': False}, {'question': 'What were the surveillance objectives?', 'id': 3805, 'answers': [{'text': 'to: describe the key epidemiological and clinical characteristics of COVID-19 cases detected in Europe; inform country preparedness; and improve further case detection and management. ', 'answer_start': 3594}], 'is_impossible': False}, {'question': 'What is the adopted WHO case definition?', 'id': 3807, 'answers': [{'text': 'a confirmed case was a person with laboratory confirmation of SARS-CoV-2 infection (ECDC recommended two separate SARS-CoV-2 RT-PCR tests), irrespective of clinical signs and symptoms, whereas a probable case was a suspect case for whom testing for SARS-CoV-2 was inconclusive or positive using a pan-coronavirus assay', 'answer_start': 4110}], 'is_impossible': False}, {'question': 'When was the first reported death in France?', 'id': 3808, 'answers': [{'text': '15 February', 'answer_start': 5324}], 'is_impossible': False}, {'question': 'What is the presumed incubation period?', 'id': 3809, 'answers': [{'text': ' up to 14 days [', 'answer_start': 5644}], 'is_impossible': False}, {'question': 'What were the places of infection?', 'id': 3810, 'answers': [{'text': ' 35 cases (missing for three cases), of whom 14 were infected in China (Hubei province: 10 cases; Shandong province: one case; province not reported for three cases). The remaining 21 cases were infected in Europe. ', 'answer_start': 5770}], 'is_impossible': False}, {'question': 'What places were linked to these?', 'id': 3811, 'answers': [{'text': '14 were linked to a cluster in Bavaria, Germany, and seven to a cluster in Haute-Savoie, France ', 'answer_start': 5995}], 'is_impossible': False}, {'question': 'How many cases were hospitalised?', 'id': 3812, 'answers': [{'text': 'All but two cases were hospitalised (35 of 37 where information on hospitalisation was reported)', 'answer_start': 6242}], 'is_impossible': False}, {'question': 'Why were they hospitalised?', 'id': 3813, 'answers': [{'text': ' it is likely that most were hospitalised to isolate the person rather than because of severe disease. ', 'answer_start': 6348}], 'is_impossible': False}, {'question': 'What was time from onset to hospitalisation?', 'id': 3814, 'answers': [{'text': 'ranged between 0 and 10 days with a mean of 3.7 days ', 'answer_start': 6518}], 'is_impossible': False}, {'question': 'What was the duration of hospitalisation?', 'id': 3815, 'answers': [{'text': 'The mean number of days to hospitalisation was 2.5 days for cases imported from China, but 4.6 days for those infected in Europe.', 'answer_start': 6596}], 'is_impossible': False}, {'question': 'Why was this?', 'id': 3816, 'answers': [{'text': 'This was mostly a result of delays in identifying the index cases of the two clusters in France and Germany. In the German cluster, for example, the first three cases detected locally were hospitalised in a mean of 5.7 days, whereas the following six took only a mean of 2 days to be hospitalised.', 'answer_start': 6726}], 'is_impossible': False}, {'question': 'How many cases reported symptoms at this point?\\n', 'id': 3817, 'answers': [{'text': ' 31 cases', 'answer_start': 7077}], 'is_impossible': False}, {'question': 'How many cases were asymptomatic?', 'id': 3818, 'answers': [{'text': 'Two cases ', 'answer_start': 7088}], 'is_impossible': False}, {'question': 'What were the asymptomatic cases tested as?', 'id': 3819, 'answers': [{'text': 'as part of screening following repatriation and during contact tracing respectively. ', 'answer_start': 7190}], 'is_impossible': False}, {'question': ' For how many cases Fever reported as the sole symptom?', 'id': 3821, 'answers': [{'text': ' nine cases', 'answer_start': 7565}], 'is_impossible': False}, {'question': 'In how many cases the symptoms at diagnosis were consistent with the case definition for acute respiratory infection?', 'id': 3822, 'answers': [{'text': 'In 16 of 29 symptomatic cases', 'answer_start': 7578}], 'is_impossible': False}, {'question': 'How many cases had data on preexisting conditions?', 'id': 3823, 'answers': [{'text': 'seven cases', 'answer_start': 7876}], 'is_impossible': False}, {'question': 'How many cases had no pre-existing conditions?', 'id': 3824, 'answers': [{'text': 'five', 'answer_start': 7889}], 'is_impossible': False}, {'question': 'What other data on pre-existing conditions were reported?', 'id': 3825, 'answers': [{'text': 'one was reported to be obese and one had pre-existing cardiac disease. No data on clinical signs e.g. dyspnea etc. were reported for any of the 38 cases.', 'answer_start': 7931}], 'is_impossible': False}, {'question': 'How many reported viral pneumonia?', 'id': 3826, 'answers': [{'text': 'two reported in Italy and two reported in France', 'answer_start': 8154}], 'is_impossible': False}, {'question': 'What was the clinical evolution of the hospitalised cases?', 'id': 3828, 'answers': [{'text': 'All hospitalised cases had a benign clinical evolution except four,', 'answer_start': 8086}], 'is_impossible': False}, {'question': 'What happened to three cases who were aged 65 years or over?', 'id': 3829, 'answers': [{'text': 'were admitted to intensive care and required respiratory support and one French case died.', 'answer_start': 8290}], 'is_impossible': False}, {'question': 'What happened to the case who died?', 'id': 3830, 'answers': [{'text': 'was hospitalised for 21 days and required intensive care and mechanical ventilation for 19 days', 'answer_start': 8399}], 'is_impossible': False}, {'question': 'What was the duration of hospitalisation reported for 16 cases ?', 'id': 3831, 'answers': [{'text': 'a median of 13 days (range: 8-23 days)', 'answer_start': 8559}], 'is_impossible': False}, {'question': 'How were the assays confirmed?', 'id': 3832, 'answers': [{'text': 'according to specific assays targeting at least two separate genes (envelope (E) gene as a screening test and RNA-dependent RNA polymerase (RdRp) gene or nucleoprotein (N) gene for confirmation) ', 'answer_start': 8685}], 'is_impossible': False}, {'question': 'What were the specimen types for 21 cases?', 'id': 3833, 'answers': [{'text': '15 had positive nasopharyngeal swabs, nine had positive throat swabs, three cases had positive sputum, two had a positive nasal swab, one case had a positive nasopharyngeal aspirate and one a positive endotracheal aspirate.', 'answer_start': 8944}], 'is_impossible': False}, {'question': 'As of 5 March 2020, what  are the cases in the WHO European region?', 'id': 3834, 'answers': [{'text': 'there are 4,250 cases including 113 deaths reported among 38 countries', 'answer_start': 9465}], 'is_impossible': False}, {'question': 'What were the two contexts for transmission?', 'id': 3835, 'answers': [{'text': 'sporadic cases among travellers from China (14 cases) and cases who acquired infection due to subsequent local transmission in Europe (21 cases).', 'answer_start': 9651}], 'is_impossible': False}, {'question': 'What  does the analysis show on the difference between locally acquired cases vs imported cases?', 'id': 3836, 'answers': [{'text': 'that the time from symptom onset to hospitalisation/case isolation was about 3 days longer for locally acquired cases than for imported cases.', 'answer_start': 9816}], 'is_impossible': False}, {'question': 'What is required for locally acquired cases?', 'id': 3837, 'answers': [{'text': \"significant resources for contact tracing and quarantine, and countries should be prepared to allocate considerable public health resources during the containment phase, should local clusters emerge in their population. In addition, prompt sharing of information on cases and contacts through international notification systems such as the International Health Regulations (IHR) mechanism and the European Commission's European Early Warning and Response System is essential to contain international spread of infection.\\n\", 'answer_start': 10729}], 'is_impossible': False}, {'question': 'What was common to all imported cases?', 'id': 3838, 'answers': [{'text': 'had a history of travel to China', 'answer_start': 11277}], 'is_impossible': False}, {'question': 'What testing and detection are needed?', 'id': 3839, 'answers': [{'text': 'Testing of suspected cases based on geographical risk of importation needs to be complemented with additional approaches to ensure early detection of local circulation of COVID-19, including through testing of severe acute respiratory infections in hospitals irrespectively of travel history as recommended in the WHO case definition updated on 27 February 2020 ', 'answer_start': 11773}], 'is_impossible': False}, {'question': 'What did the finding prompt ECDC to do?', 'id': 3840, 'answers': [{'text': 'include fever among several clinical signs or symptoms indicative for the suspected case definition.', 'answer_start': 12486}], 'is_impossible': False}, {'question': 'Why is understanding the infection-severity  critical ?', 'id': 3841, 'answers': [{'text': 'to help plan for the impact on the healthcare system and the wider population.', 'answer_start': 13802}], 'is_impossible': False}, {'question': 'Why are serological tests vital?', 'id': 3842, 'answers': [{'text': 'to understand the proportion of cases who are asymptomatic.', 'answer_start': 13911}], 'is_impossible': False}, {'question': 'How can hospital based surveillance help?', 'id': 3843, 'answers': [{'text': 'help estimate the incidence of severe cases and identify risk factors for severity and death', 'answer_start': 14005}], 'is_impossible': False}, {'question': 'How can present systems of surveillance be used?', 'id': 3844, 'answers': [{'text': 'Established hospital surveillance systems that are in place for influenza and other diseases in Europe may be expanded for this purpose. In addition, a number of countries in Europe are adapting and, in some cases, already using existing sentinel primary care based surveillance systems for influenza to detect community transmission of SARS-CoV-2', 'answer_start': 14099}], 'is_impossible': False}, {'question': 'How will  this approach used?', 'id': 3845, 'answers': [{'text': 'will be used globally to help identify evidence of widespread community transmission and, should the virus spread and containment no longer be deemed feasible, to monitor intensity of disease transmission, trends and its geographical spread.', 'answer_start': 14462}], 'is_impossible': False}, {'question': 'Why is additional research needed?', 'id': 3846, 'answers': [{'text': ' to complement surveillance data to build knowledge on the infectious period, modes of transmission, basic and effective reproduction numbers, and effectiveness of prevention and case management options also in settings outside of China.', 'answer_start': 14734}], 'is_impossible': False}], 'context': \"First cases of coronavirus disease 2019 (COVID-19) in the WHO European Region, 24 January to 21 February 2020\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC7068164/\\n\\nSHA: ce358c18aac69fc83c7b2e9a7dca4a43b0f60e2e\\n\\nAuthors: Spiteri, Gianfranco; Fielding, James; Diercke, Michaela; Campese, Christine; Enouf, Vincent; Gaymard, Alexandre; Bella, Antonino; Sognamiglio, Paola; Sierra Moros, Maria José; Riutort, Antonio Nicolau; Demina, Yulia V.; Mahieu, Romain; Broas, Markku; Bengnér, Malin; Buda, Silke; Schilling, Julia; Filleul, Laurent; Lepoutre, Agnès; Saura, Christine; Mailles, Alexandra; Levy-Bruhl, Daniel; Coignard, Bruno; Bernard-Stoecklin, Sibylle; Behillil, Sylvie; van der Werf, Sylvie; Valette, Martine; Lina, Bruno; Riccardo, Flavia; Nicastri, Emanuele; Casas, Inmaculada; Larrauri, Amparo; Salom Castell, Magdalena; Pozo, Francisco; Maksyutov, Rinat A.; Martin, Charlotte; Van Ranst, Marc; Bossuyt, Nathalie; Siira, Lotta; Sane, Jussi; Tegmark-Wisell, Karin; Palmérus, Maria; Broberg, Eeva K.; Beauté, Julien; Jorgensen, Pernille; Bundle, Nick; Pereyaslov, Dmitriy; Adlhoch, Cornelia; Pukkila, Jukka; Pebody, Richard; Olsen, Sonja; Ciancio, Bruno Christian\\nDate: 2020-03-05\\nDOI: 10.2807/1560-7917.es.2020.25.9.2000178\\nLicense: cc-by\\n\\nAbstract: In the WHO European Region, COVID-19 surveillance was implemented 27 January 2020. We detail the first European cases. As at 21 February, nine European countries reported 47 cases. Among 38 cases studied, 21 were linked to two clusters in Germany and France, 14 were infected in China. Median case age was 42 years; 25 were male. Late detection of the clusters’ index cases delayed isolation of further local cases. As at 5 March, there were 4,250 cases.\\n\\nText: In the WHO European Region, COVID-19 surveillance was implemented 27 January 2020. We detail the first European cases. As at 21 February, nine European countries reported 47 cases. Among 38 cases studied, 21 were linked to two clusters in Germany and France, 14 were infected in China. Median case age was 42 years; 25 were male. Late detection of the clusters' index cases delayed isolation of further local cases. As at 5 March, there were 4,250 cases.\\n\\nA cluster of pneumonia of unknown origin was identified in Wuhan, China, in December 2019 [1] . On 12 January 2020, Chinese authorities shared the sequence of a novel coronavirus termed severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) isolated from some clustered cases [2] . Since then, the disease caused by SARS-CoV-2 has been named coronavirus disease 2019 (COVID -19) . As at 21 February 2020, the virus had spread rapidly mostly within China but also to 28 other countries, including in the World Health Organization (WHO) European Region [3] [4] [5] .\\n\\nHere we describe the epidemiology of the first cases of COVID-19 in this region, excluding cases reported in the United Kingdom (UK), as at 21 February 2020. The study includes a comparison between cases detected among travellers from China and cases whose infection was acquired due to subsequent local transmission.\\n\\nOn 27 January 2020, the European Centre for Disease Prevention and Control (ECDC) and the WHO Regional Office for Europe asked countries to complete a WHO standard COVID-19 case report form for all confirmed and probable cases according to WHO criteria [6] [7] [8] . The overall aim of surveillance at this time was to support the global strategy of containment of COVID-19 with rapid identification and follow-up of cases linked to affected countries in order to minimise onward transmission. The surveillance objectives were to: describe the key epidemiological and clinical characteristics of COVID-19 cases detected in Europe; inform country preparedness; and improve further case detection and management. Data collected included demographics, history of recent travel to affected areas, close contact with a probable or confirmed COVID-19 case, underlying conditions, signs and symptoms of disease at onset, type of specimens from which the virus was detected, and clinical outcome. The WHO case definition was adopted for surveillance: a confirmed case was a person with laboratory confirmation of SARS-CoV-2 infection (ECDC recommended two separate SARS-CoV-2 RT-PCR tests), irrespective of clinical signs and symptoms, whereas a probable case was a suspect case for whom testing for SARS-CoV-2 was inconclusive or positive using a pan-coronavirus assay [8] . By 31 January 2020, 47 laboratories in 31 countries, including 38 laboratories in 24 European Union and European Economic Area (EU/EEA) countries, had diagnostic capability for SARS-CoV-2 available (close to 60% of countries in the WHO European Region), with cross-border shipment arrangements in place for many of those lacking domestic testing capacity. The remaining six EU/EEA countries were expected to have diagnostic testing available by mid-February [9] .\\n\\nAs at 09:00 on 21 February 2020, 47 confirmed cases of COVID-19 were reported in the WHO European Region and one of these cases had died [4] . Data on 38 of these cases (i.e. all except the nine reported in the UK) are included in this analysis.\\n\\nThe first three cases detected were reported in France on 24 January 2020 and had onset of symptoms on 17, 19 and 23 January respectively [10] . The first death was reported on 15 February in France. As at 21 February, nine countries had reported cases ( Figure) : Belgium (1), Finland (1), France (12), Germany (16), Italy (3), Russia (2), Spain (2), Sweden (1) and the UK (9 -not included further).\\n\\nThe place of infection (assessed at national level based on an incubation period presumed to be up to 14 days [11] , travel history and contact with probable or confirmed cases as per the case definition) was reported for 35 cases (missing for three cases), of whom 14 were infected in China (Hubei province: 10 cases; Shandong province: one case; province not reported for three cases). The remaining 21 cases were infected in Europe. Of these, 14 were linked to a cluster in Bavaria, Germany, and seven to a cluster in Haute-Savoie, France [12, 13] . Cases from the Bavarian cluster were reported from Germany and Spain, whereas cases from the Haute-Savoie cluster were reported from France All but two cases were hospitalised (35 of 37 where information on hospitalisation was reported), although it is likely that most were hospitalised to isolate the person rather than because of severe disease. The time from onset of symptoms to hospitalisation (and isolation) ranged between 0 and 10 days with a mean of 3.7 days (reported for 29 cases). The mean number of days to hospitalisation was 2.5 days for cases imported from China, but 4.6 days for those infected in Europe. This was mostly a result of delays in identifying the index cases of the two clusters in France and Germany. In the German cluster, for example, the first three cases detected locally were hospitalised in a mean of 5.7 days, whereas the following six took only a mean of 2 days to be hospitalised.\\n\\nSymptoms at the point of diagnosis were reported for 31 cases. Two cases were asymptomatic and remained so until tested negative. The asymptomatic cases were tested as part of screening following repatriation and during contact tracing respectively. Of the remaining 29, 20 reported fever, 14 reported cough and eight reported weakness. Additional symptoms reported included headaches (6 cases), sore throat (2), rhinorrhoea (2), shortness of breath (2), myalgia (1), diarrhoea (1) and nausea (1). Fever was reported as the sole symptom for nine cases. In 16 of 29 symptomatic cases, the symptoms at diagnosis were consistent with the case definition for acute respiratory infection [16] , although it is possible that cases presented additional symptoms after diagnosis and these were not reported.\\n\\nData on pre-existing conditions were reported for seven cases; five had no pre-existing conditions while one was reported to be obese and one had pre-existing cardiac disease. No data on clinical signs e.g. dyspnea etc. were reported for any of the 38 cases.\\n\\nAll hospitalised cases had a benign clinical evolution except four, two reported in Italy and two reported in France, all of whom developed viral pneumonia. All three cases who were aged 65 years or over were admitted to intensive care and required respiratory support and one French case died. The case who died was hospitalised for 21 days and required intensive care and mechanical ventilation for 19 days. The duration of hospitalisation was reported for 16 cases with a median of 13 days (range: 8-23 days). As at 21 February 2020, four cases were still hospitalised.\\n\\nAll cases were confirmed according to specific assays targeting at least two separate genes (envelope (E) gene as a screening test and RNA-dependent RNA polymerase (RdRp) gene or nucleoprotein (N) gene for confirmation) [8, 17] . The specimen types tested were reported for 27 cases: 15 had positive nasopharyngeal swabs, nine had positive throat swabs, three cases had positive sputum, two had a positive nasal swab, one case had a positive nasopharyngeal aspirate and one a positive endotracheal aspirate.\\n\\nAs at 09:00 on 21 February, few COVID-19 cases had been detected in Europe compared with Asia. However the situation is rapidly developing, with a large outbreak recently identified in northern Italy, with transmission in several municipalities and at least two deaths [18] . As at 5 March 2020, there are 4,250 cases including 113 deaths reported among 38 countries in the WHO European region [19] .\\n\\nIn our analysis of early cases, we observed transmission in two broad contexts: sporadic cases among travellers from China (14 cases) and cases who acquired infection due to subsequent local transmission in Europe (21 cases). Our analysis shows that the time from symptom onset to hospitalisation/case isolation was about 3 days longer for locally acquired cases than for imported cases. People returning from affected areas are likely to have a low threshold to seek care and be tested when symptomatic, however delays in identifying the index cases of the two clusters in France and Germany meant that locally acquired cases took longer to be detected and isolated. Once the exposure is determined and contacts identified and quarantined (171 contacts in France and 200 in Germany for the clusters in Haute-Savoie and Bavaria, respectively), further cases are likely to be rapidly detected and isolated when they develop symptoms [15, 20] . In the German cluster, for example, the first three cases detected locally were hospitalised in a mean of 5.7 days, whereas the following six were hospitalised after a mean of 2 days. Locally acquired cases require significant resources for contact tracing and quarantine, and countries should be prepared to allocate considerable public health resources during the containment phase, should local clusters emerge in their population. In addition, prompt sharing of information on cases and contacts through international notification systems such as the International Health Regulations (IHR) mechanism and the European Commission's European Early Warning and Response System is essential to contain international spread of infection.\\n\\nAll of the imported cases had a history of travel to China. This was consistent with the epidemiological situation in Asia, and supported the recommendation for testing of suspected cases with travel history to China and potentially other areas of presumed ongoing community transmission. The situation has evolved rapidly since then, however, and the number of countries reporting COVID-19 transmission increased rapidly, notably with a large outbreak in northern Italy with 3,089 cases reported as at 5 March [18, 19] . Testing of suspected cases based on geographical risk of importation needs to be complemented with additional approaches to ensure early detection of local circulation of COVID-19, including through testing of severe acute respiratory infections in hospitals irrespectively of travel history as recommended in the WHO case definition updated on 27 February 2020 [21] .\\n\\nThe clinical presentation observed in the cases in Europe is that of an acute respiratory infection. However, of the 31 cases with information on symptoms, 20 cases presented with fever and nine cases presented only with fever and no other symptoms. These findings, which are consistent with other published case series, have prompted ECDC to include fever among several clinical signs or symptoms indicative for the suspected case definition.\\n\\nThree cases were aged 65 years or over. All required admission to intensive care and were tourists (imported cases). These findings could reflect the average older age of the tourist population compared with the local contacts exposed to infection in Europe and do not allow us to draw any conclusion on the proportion of severe cases that we could expect in the general population of Europe. Despite this, the finding of older individuals being at higher risk of a severe clinical course is consistent with the evidence from Chinese case series published so far although the majority of infections in China have been mild [22, 23] .\\n\\nThis preliminary analysis is based on the first reported cases of COVID-19 cases in the WHO European Region. Given the small sample size, and limited completeness for some variables, all the results presented should be interpreted with caution.\\n\\nWith increasing numbers of cases in Europe, data from surveillance and investigations in the region can build on the evidence from countries in Asia experiencing more widespread transmission particularly on disease spectrum and the proportion of infections with severe outcome [22] . Understanding the infection-severity is critical to help plan for the impact on the healthcare system and the wider population. Serological studies are vital to understand the proportion of cases who are asymptomatic. Hospital-based surveillance could help estimate the incidence of severe cases and identify risk factors for severity and death. Established hospital surveillance systems that are in place for influenza and other diseases in Europe may be expanded for this purpose. In addition, a number of countries in Europe are adapting and, in some cases, already using existing sentinel primary care based surveillance systems for influenza to detect community transmission of SARS-CoV-2. This approach will be used globally to help identify evidence of widespread community transmission and, should the virus spread and containment no longer be deemed feasible, to monitor intensity of disease transmission, trends and its geographical spread.\\n\\nAdditional research is needed to complement surveillance data to build knowledge on the infectious period, modes of transmission, basic and effective reproduction numbers, and effectiveness of prevention and case management options also in settings outside of China. Such special studies are being conducted globally, including a cohort study on citizens repatriated from China to Europe, with the aim to extrapolate disease incidence and risk factors for infection in areas with community transmission. Countries together with ECDC and WHO, should use all opportunities to address these questions in a coordinated fashion at the European and global level.\\n\\nprovided input to the outline, multiple versions of the manuscript and gave approval to the final draft.\", 'document_id': 2642}]}, {'paragraphs': [{'qas': [{'question': 'What growing dysjunction has been witnessed?', 'id': 4123, 'answers': [{'text': \"a 'Valley of Death' 1,2 no less, between broadening strides in fundamental biomedical research and their incommensurate reach into the clinic. \", 'answer_start': 292}], 'is_impossible': False}, {'question': 'What is aiming to incorporate pathways to translation at the earliest stages?', 'id': 4124, 'answers': [{'text': ' recent changes in the structure of government funding, 2 new public and private joint ventures and specialist undergraduate and postgraduate courses', 'answer_start': 502}], 'is_impossible': False}, {'question': \"How much  have  the number of biomedical research publications targeting 'translational' concepts has increased ?\", 'id': 4125, 'answers': [{'text': 'exponentially, up 1800%', 'answer_start': 842}], 'is_impossible': False}, {'question': 'What ways to  solve the issues are outlined?', 'id': 4126, 'answers': [{'text': \"by creatively leveraging the so-called 'strengths' of viruses. Viral RNA polymerisation and reverse transcription enable resistance to treatment by conferring extraordinary genetic diversity. \", 'answer_start': 2042}], 'is_impossible': False}, {'question': 'How do these exact processes ultimately restrict viral infectivity?', 'id': 4127, 'answers': [{'text': ' by strongly limiting virus genome sizes and their incorporation of new information. ', 'answer_start': 2302}], 'is_impossible': False}, {'question': 'What does the author coin this evolutionary dilemma as?', 'id': 4128, 'answers': [{'text': \"'information economy paradox'.\", 'answer_start': 2424}], 'is_impossible': False}, {'question': 'How do many viruses resolve this ?', 'id': 4129, 'answers': [{'text': 'by manipulating multifunctional or multitasking host cell proteins (MMHPs), thereby maximising host subversion and viral infectivity at minimal informational cost.', 'answer_start': 2492}], 'is_impossible': False}, {'question': 'How may this \"Achilles Heel\" be safely targeted?', 'id': 4130, 'answers': [{'text': ' via host-oriented therapies to impose devastating informational and fitness barriers on escape mutant selection.', 'answer_start': 2725}], 'is_impossible': False}, {'question': 'Why  may MMHP-targeting therapies exhibit both robust and broadspectrum antiviral efficacy?', 'id': 4131, 'answers': [{'text': 'since MMHPs are often conserved targets within and between virus families,', 'answer_start': 2852}], 'is_impossible': False}, {'question': 'What will  achieving this through drug repurposing do?', 'id': 4132, 'answers': [{'text': ' break the vicious cycle of escalating therapeutic development costs and trivial escape mutant selection, both quickly and in multiple places. ', 'answer_start': 3058}], 'is_impossible': False}, {'question': 'What are also discussed by the author?', 'id': 4133, 'answers': [{'text': ' alternative posttranslational and RNA-based antiviral approaches, designer vaccines, immunotherapy and the emerging field of neo-virology.', 'answer_start': 3215}], 'is_impossible': False}, {'question': 'What does  the author anticipate international efforts will do?', 'id': 4134, 'answers': [{'text': ' will enable the tapping of useful new biological functions and processes, methods for controlling infection, and the deployment of symbiotic or subclinical viruses in new therapies and biotechnologies that are so crucially needed.', 'answer_start': 3429}], 'is_impossible': False}, {'question': 'What do pathogens do upon infection?', 'id': 4138, 'answers': [{'text': 'stimulate expression of numerous host inflammatory factors that support recruitment and activation of immune cells. ', 'answer_start': 3688}], 'is_impossible': False}, {'question': 'What is the flip side ?', 'id': 4139, 'answers': [{'text': 'this same process also causes immunopathology when prolonged or deregulated.', 'answer_start': 3822}], 'is_impossible': False}, {'question': 'What do RBPs do?', 'id': 4140, 'answers': [{'text': 'post-transcriptionally control expression of crucial inflammatory factors in various tissues and their potential therapeutic applications', 'answer_start': 4022}], 'is_impossible': False}, {'question': 'What is included in RBPs?', 'id': 4141, 'answers': [{'text': 'tristetraprolin and AUF1', 'answer_start': 4182}], 'is_impossible': False}, {'question': 'What  do  tristetraprolin and AUF1, do?', 'id': 4142, 'answers': [{'text': 'promote degradation of AU-rich element (ARE)-containing mRNA', 'answer_start': 4214}], 'is_impossible': False}, {'question': 'What do RBPs include?', 'id': 4143, 'answers': [{'text': 'members of the Roquin and Regnase families', 'answer_start': 4276}], 'is_impossible': False}, {'question': 'What domembers of the Roquin and Regnase families do?', 'id': 4144, 'answers': [{'text': 'promote or effect degradation of mRNAs harbouring stem-loop structures', 'answer_start': 4339}], 'is_impossible': False}, {'question': 'What do the RBPs include?', 'id': 4145, 'answers': [{'text': 'RNA methylation machinery', 'answer_start': 4453}], 'is_impossible': False}, {'question': 'What is the increasingly apparent role of RNA methylation machinery ?', 'id': 4146, 'answers': [{'text': 'in controlling inflammatory mRNA stability.', 'answer_start': 4479}], 'is_impossible': False}, {'question': 'Where do these activities take place?', 'id': 4147, 'answers': [{'text': ' in various subcellular compartments ', 'answer_start': 4550}], 'is_impossible': False}, {'question': 'What happens to these activities during infection?', 'id': 4148, 'answers': [{'text': 'are differentially regulated', 'answer_start': 4591}], 'is_impossible': False}, {'question': ' In this way, what do the mRNA-destabilising RBPs constitute ?', 'id': 4149, 'answers': [{'text': \"a 'brake' on the immune system\", 'answer_start': 4686}], 'is_impossible': False}, {'question': \"What can be  done with the  'brake' on the immune system?\", 'id': 4150, 'answers': [{'text': 'may ultimately be toggled therapeutically', 'answer_start': 4724}], 'is_impossible': False}, {'question': 'What does the author anticipate that continued efforts will lead to?', 'id': 4151, 'answers': [{'text': 'Another mRNA under post-transcriptional regulation by Regnase-1 and Roquin', 'answer_start': 5011}], 'is_impossible': False}, {'question': 'What is another mRNA under post-transcriptional regulation by Regnase-1 and Roquin?', 'id': 4152, 'answers': [{'text': ' Furin', 'answer_start': 5088}], 'is_impossible': False}, {'question': 'What does Furin encode?', 'id': 4153, 'answers': [{'text': 'a conserved proprotein convertase crucial in human health and disease.', 'answer_start': 5110}], 'is_impossible': False}, {'question': 'What are Furin, along with other PCSK family members implicated in?', 'id': 4154, 'answers': [{'text': ' in immune regulation, cancer and the entry, maturation or release of a broad array of evolutionarily diverse viruses including human papillomavirus (HPV), influenza (IAV), Ebola (EboV), dengue (DenV) and human immunodeficiency virus (HIV)', 'answer_start': 5246}], 'is_impossible': False}, {'question': 'What do Braun and Sauter review?', 'id': 4155, 'answers': [{'text': ' the roles of furin in these processes, as well as the history and future of furin-targeting therapeutics.', 'answer_start': 5516}], 'is_impossible': False}, {'question': 'What dis their recent work reveal?', 'id': 4156, 'answers': [{'text': 'how two IFN-cinducible factors exhibit broad-spectrum inhibition of IAV, measles (MV), zika (ZikV) and HIV by suppressing furin activity.', 'answer_start': 5671}], 'is_impossible': False}, {'question': 'What has the increasing abundance of affordable, sensitive, high-throughput genome sequencing technologies  led to?', 'id': 4157, 'answers': [{'text': ' a recent boom in metagenomics and the cataloguing of the microbiome of our world.', 'answer_start': 6119}], 'is_impossible': False}, {'question': 'What was this system used for the first time for?', 'id': 4160, 'answers': [{'text': ' to directly sequence an RNA virus genome (IAV)', 'answer_start': 6678}], 'is_impossible': False}, {'question': 'What have decades of basic immunology research  provided ?', 'id': 4161, 'answers': [{'text': 'a near-complete picture of the main armaments in the human antiviral arsenal. ', 'answer_start': 7152}], 'is_impossible': False}, {'question': 'What has this focus on mammalian defences and pathologies sidelined?', 'id': 4162, 'answers': [{'text': 'examination of the types and roles of viruses and antiviral defences that exist throughout our biosphere.', 'answer_start': 7307}], 'is_impossible': False}, {'question': 'What has CRISPR/Cas antiviral immune system of prokaryotes been repurposed as?', 'id': 4163, 'answers': [{'text': 'as a revolutionary gene-editing biotechnology in plants and animals.', 'answer_start': 7513}], 'is_impossible': False}, {'question': 'What is another  case in point?', 'id': 4164, 'answers': [{'text': 'the ancient lineage of nucleocytosolic large DNA viruses (NCLDVs)', 'answer_start': 7596}], 'is_impossible': False}, {'question': 'What are the ancient lineage of NCLDVs?\\n', 'id': 4165, 'answers': [{'text': 'emerging human pathogens that possess enormous genomes of up to several megabases in size encoding hundreds of proteins with unique and unknown functions.', 'answer_start': 7673}], 'is_impossible': False}, {'question': 'What do the recent efforts indicate regarding hundreds of human and avian infectious viruses?', 'id': 4166, 'answers': [{'text': ' the true number may be in the millions and many harbour zoonotic potential. ', 'answer_start': 7949}], 'is_impossible': False}, {'question': 'What is neo-virology?', 'id': 4167, 'answers': [{'text': ' an emerging field engaged in cataloguing and characterising this biodiversity through a global consortium.', 'answer_start': 8235}], 'is_impossible': False}, {'question': 'What is predicted these efforts on neo-virology will unlock?', 'id': 4168, 'answers': [{'text': 'a vast wealth of currently unexplored biodiversity, leading to biotechnologies and treatments that leverage the host-virus interactions developed throughout evolution.', 'answer_start': 8382}], 'is_impossible': False}, {'question': 'What are the two of the four pillars of the National Innovation and Science Agenda?', 'id': 4169, 'answers': [{'text': 'Supporting industry-academia collaboration and nurturing talent and skills in the Indo-Pacific region', 'answer_start': 8965}], 'is_impossible': False}, {'question': \"What do Australia's Medical Research and Innovation Priorities include?\", 'id': 4170, 'answers': [{'text': 'antimicrobial resistance, global health and health security, drug repurposing and translational research infrastructure,', 'answer_start': 9226}], 'is_impossible': False}, {'question': 'What is essential for these priority outcomes?', 'id': 4171, 'answers': [{'text': 'Establishing durable international relationships that integrate diverse expertise', 'answer_start': 9414}], 'is_impossible': False}, {'question': 'What is the Japan AMED tasked with?', 'id': 4172, 'answers': [{'text': 'translating the biomedical research output of that country.', 'answer_start': 9774}], 'is_impossible': False}], 'context': \"Frontiers in antiviral therapy and immunotherapy\\n\\nhttps://doi.org/10.1002/cti2.1115\\n\\nSHA: facbfdfa7189ca9ff83dc30e5d241ab22e962dbf\\n\\nAuthors: Heaton, Steven M\\nDate: 2020\\nDOI: 10.1002/cti2.1115\\nLicense: cc-by\\n\\nAbstract: nan\\n\\nText: Globally, recent decades have witnessed a growing disjunction, a 'Valley of Death' 1,2 no less, between broadening strides in fundamental biomedical research and their incommensurate reach into the clinic. Plumbing work on research funding and development pipelines through recent changes in the structure of government funding, 2 new public and private joint ventures and specialist undergraduate and postgraduate courses now aim to incorporate pathways to translation at the earliest stages. Reflecting this shift, the number of biomedical research publications targeting 'translational' concepts has increased exponentially, up 1800% between 2003 and 2014 3 and continuing to rise rapidly up to the present day. Fuelled by the availability of new research technologies, as well as changing disease, cost and other pressing issues of our time, further growth in this exciting space will undoubtedly continue. Despite recent advances in the therapeutic control of immune function and viral infection, current therapies are often challenging to develop, expensive to deploy and readily select for resistance-conferring mutants. Shaped by the hostvirus immunological 'arms race' and tempered in the forge of deep time, the biodiversity of our world is increasingly being harnessed for new biotechnologies and therapeutics. Simultaneously, a shift towards host-oriented antiviral therapies is currently underway. In this Clinical & Translational Immunology Special Feature, I illustrate a strategic vision integrating these themes to create new, effective, economical and robust antiviral therapies and immunotherapies, with both the realities and the opportunities afforded to researchers working in our changing world squarely in mind.\\n\\nOpening this CTI Special Feature, I outline ways these issues may be solved by creatively leveraging the so-called 'strengths' of viruses. Viral RNA polymerisation and reverse transcription enable resistance to treatment by conferring extraordinary genetic diversity. However, these exact processes ultimately restrict viral infectivity by strongly limiting virus genome sizes and their incorporation of new information. I coin this evolutionary dilemma the 'information economy paradox'. Many viruses attempt to resolve this by manipulating multifunctional or multitasking host cell proteins (MMHPs), thereby maximising host subversion and viral infectivity at minimal informational cost. 4 I argue this exposes an 'Achilles Heel' that may be safely targeted via host-oriented therapies to impose devastating informational and fitness barriers on escape mutant selection. Furthermore, since MMHPs are often conserved targets within and between virus families, MMHP-targeting therapies may exhibit both robust and broadspectrum antiviral efficacy. Achieving this through drug repurposing will break the vicious cycle of escalating therapeutic development costs and trivial escape mutant selection, both quickly and in multiple places. I also discuss alternative posttranslational and RNA-based antiviral approaches, designer vaccines, immunotherapy and the emerging field of neo-virology. 4 I anticipate international efforts in these areas over the coming decade will enable the tapping of useful new biological functions and processes, methods for controlling infection, and the deployment of symbiotic or subclinical viruses in new therapies and biotechnologies that are so crucially needed.\\n\\nUpon infection, pathogens stimulate expression of numerous host inflammatory factors that support recruitment and activation of immune cells. On the flip side, this same process also causes immunopathology when prolonged or deregulated. 5 In their contribution to this Special Feature, Yoshinaga and Takeuchi review endogenous RNA-binding proteins (RBPs) that post-transcriptionally control expression of crucial inflammatory factors in various tissues and their potential therapeutic applications. 6 These RBPs include tristetraprolin and AUF1, which promote degradation of AU-rich element (ARE)-containing mRNA; members of the Roquin and Regnase families, which respectively promote or effect degradation of mRNAs harbouring stem-loop structures; and the increasingly apparent role of the RNA methylation machinery in controlling inflammatory mRNA stability. These activities take place in various subcellular compartments and are differentially regulated during infection. In this way, mRNA-destabilising RBPs constitute a 'brake' on the immune system, which may ultimately be toggled therapeutically. I anticipate continued efforts in this area will lead to new methods of regaining control over inflammation in autoimmunity, selectively enhancing immunity in immunotherapy, and modulating RNA synthesis and virus replication during infection.\\n\\nAnother mRNA under post-transcriptional regulation by Regnase-1 and Roquin is Furin, which encodes a conserved proprotein convertase crucial in human health and disease. Furin, along with other PCSK family members, is widely implicated in immune regulation, cancer and the entry, maturation or release of a broad array of evolutionarily diverse viruses including human papillomavirus (HPV), influenza (IAV), Ebola (EboV), dengue (DenV) and human immunodeficiency virus (HIV). Here, Braun and Sauter review the roles of furin in these processes, as well as the history and future of furin-targeting therapeutics. 7 They also discuss their recent work revealing how two IFN-cinducible factors exhibit broad-spectrum inhibition of IAV, measles (MV), zika (ZikV) and HIV by suppressing furin activity. 8 Over the coming decade, I expect to see an ever-finer spatiotemporal resolution of host-oriented therapies to achieve safe, effective and broad-spectrum yet costeffective therapies for clinical use.\\n\\nThe increasing abundance of affordable, sensitive, high-throughput genome sequencing technologies has led to a recent boom in metagenomics and the cataloguing of the microbiome of our world. The MinION nanopore sequencer is one of the latest innovations in this space, enabling direct sequencing in a miniature form factor with only minimal sample preparation and a consumer-grade laptop computer. Nakagawa and colleagues here report on their latest experiments using this system, further improving its performance for use in resource-poor contexts for meningitis diagnoses. 9 While direct sequencing of viral genomic RNA is challenging, this system was recently used to directly sequence an RNA virus genome (IAV) for the first time. 10 I anticipate further improvements in the performance of such devices over the coming decade will transform virus surveillance efforts, the importance of which was underscored by the recent EboV and novel coronavirus (nCoV / COVID-19) outbreaks, enabling rapid deployment of antiviral treatments that take resistance-conferring mutations into account.\\n\\nDecades of basic immunology research have provided a near-complete picture of the main armaments in the human antiviral arsenal. Nevertheless, this focus on mammalian defences and pathologies has sidelined examination of the types and roles of viruses and antiviral defences that exist throughout our biosphere. One case in point is the CRISPR/Cas antiviral immune system of prokaryotes, which is now repurposed as a revolutionary gene-editing biotechnology in plants and animals. 11 Another is the ancient lineage of nucleocytosolic large DNA viruses (NCLDVs), which are emerging human pathogens that possess enormous genomes of up to several megabases in size encoding hundreds of proteins with unique and unknown functions. 12 Moreover, hundreds of human-and avian-infective viruses such as IAV strain H5N1 are known, but recent efforts indicate the true number may be in the millions and many harbour zoonotic potential. 13 It is increasingly clear that host-virus interactions have generated truly vast yet poorly understood and untapped biodiversity. Closing this Special Feature, Watanabe and Kawaoka elaborate on neo-virology, an emerging field engaged in cataloguing and characterising this biodiversity through a global consortium. 14 I predict these efforts will unlock a vast wealth of currently unexplored biodiversity, leading to biotechnologies and treatments that leverage the host-virus interactions developed throughout evolution.\\n\\nWhen biomedical innovations fall into the 'Valley of Death', patients who are therefore not reached all too often fall with them. Being entrusted with the resources and expectation to conceive, deliver and communicate dividends to society is both cherished and eagerly pursued at every stage of our careers. Nevertheless, the road to research translation is winding and is built on a foundation of basic research. Supporting industry-academia collaboration and nurturing talent and skills in the Indo-Pacific region are two of the four pillars of the National Innovation and Science Agenda. 2 These frame Australia's Medical Research and Innovation Priorities, which include antimicrobial resistance, global health and health security, drug repurposing and translational research infrastructure, 15 capturing many of the key elements of this CTI Special Feature. Establishing durable international relationships that integrate diverse expertise is essential to delivering these outcomes. To this end, NHMRC has recently taken steps under the International Engagement Strategy 16 to increase cooperation with its counterparts overseas. These include the Japan Agency for Medical Research and Development (AMED), tasked with translating the biomedical research output of that country. Given the reciprocal efforts at accelerating bilateral engagement currently underway, 17 the prospects for new areas of international cooperation and mobility have never been more exciting nor urgent. With the above in mind, all contributions to this CTI Special Feature I have selected from research presented by fellow invitees to the 2018 Awaji International Forum on Infection and Immunity (AIFII) and 2017 Consortium of Biological Sciences (ConBio) conferences in Japan. Both Australia and Japan have strong traditions in immunology and related disciplines, and I predict that the quantity, quality and importance of our bilateral cooperation will accelerate rapidly over the short to medium term. By expanding and cooperatively leveraging our respective research strengths, our efforts may yet solve the many pressing disease, cost and other sustainability issues of our time.\", 'document_id': 2669}]}, {'paragraphs': [{'qas': [{'question': 'What serious question was raised?', 'id': 3847, 'answers': [{'text': 'as to whether or not our seasonal or pandemic flu might have another reservoir host.', 'answer_start': 464}], 'is_impossible': False}, {'question': 'What is a recent discovery?', 'id': 3848, 'answers': [{'text': 'Bat is believed to harbor many more viruses than we ever thought as a reservoir host or even a susceptible host ', 'answer_start': 1315}], 'is_impossible': False}, {'question': 'Which bat virus have been found to be linked with diseases?', 'id': 3849, 'answers': [{'text': 'potential human infecting HKU-1, 4, 5 and 9 [17, 18] . Recent MERS-CoV infection is another example for severe disease caused by used-to-be-less pathogenic coronaviruses. Shi and colleagues [19] by using NGS have discovered many unknown animal viruses from bat, especially some important paramyxoviruses and reoviruses. Filovirus has also been identified in bat with potential severe outcomes. Lyssaviruses (with many genotypes, including rabies virus) in the Rhabdoviridae family have been linked with severe fatal human cases, even in the developed countries, including Australia, with the bites of bats in the city [20, 21] .', 'answer_start': 1562}], 'is_impossible': False}, {'question': 'What assay played an important role?', 'id': 3850, 'answers': [{'text': 'reverse transcription polymerase chain reaction (RT-PCR)', 'answer_start': 2606}], 'is_impossible': False}], 'context': \"Haunted with and hunting for viruses\\n\\nhttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC7089303/\\n\\nSHA: c51c4f6146d0c636bc4dc3839c16b9e3ef52849a\\n\\nAuthors: Gao, George Fu; Wu, Ying\\nDate: 2013-08-07\\nDOI: 10.1007/s11427-013-4525-x\\nLicense: cc-by\\n\\nAbstract: nan\\n\\nText: pecially with next-generation sequencing (NGS) for new virus genome discovery, e.g., Ruben Donis et al. [10] sequenced a bat-derived influenza virus genome by using NGS in 2012, raising a serious question as to whether or not our seasonal or pandemic flu might have another reservoir host. Chen and colleagues [11] confirmed the SFTSV independently by using NGS. Indeed, metagenomics analysis has yielded a great deal of new viruses, especially from the environment. Our actively hunting for new viruses has made some significant contributions for our understanding of virus ecology, pathogenesis and interspecies transmission.\\n\\nScience China Life Sciences has focused on this hot topic in the event of the H7N9 outbreak after a comprehensive overview of the topic addressing HPAIV H5N1 in 2009 in the journal [12] [13] [14] . In this issue, six groups have been invited to present their recent findings on the emerging viruses, in addition to a previous report on H7N9 [3] .\\n\\nShi [15] reviewed recent discoveries of new viruses or virus genomes from bat. Bat is believed to harbor many more viruses than we ever thought as a reservoir host or even a susceptible host [16] . After the SARS-CoV virus, we have been actively seeking for new coronaviruses from bat and have yielded many of them, including potential human infecting HKU-1, 4, 5 and 9 [17, 18] . Recent MERS-CoV infection is another example for severe disease caused by used-to-be-less pathogenic coronaviruses. Shi and colleagues [19] by using NGS have discovered many unknown animal viruses from bat, especially some important paramyxoviruses and reoviruses. Filovirus has also been identified in bat with potential severe outcomes. Lyssaviruses (with many genotypes, including rabies virus) in the Rhabdoviridae family have been linked with severe fatal human cases, even in the developed countries, including Australia, with the bites of bats in the city [20, 21] . The potential roles of these viruses in bats for interspecies transmission are yet to be elucidated.\\n\\nTan and colleagues [22] specifically focused on the newly-emerged MERS-CoV. The virus was identified in 2012 in the Middle East with some exported cases to Europe. In 2013 the virus has been re-emerging and expanding its borders to more European countries. In the initial diagnosis, the pan-coronavirus real-time reverse transcription polymerase chain reaction (RT-PCR) assay played a very important role for the identification of the causative agents. By using this method, scientists detected an expected-size PCR fragment for the corresponding conserved region of ORF1b of the replicase gene of a coronavirus. This is another example that molecular biology methods played for the discovery of new pathogens. Soon the receptor used by MERS-CoV to enter the host cells was identified [23] and the molecular basis of the receptor binding to the virus was also elucidated recently [8] .\\n\\nEnterovirus has been known as serious human pathogens for a long time but their significance to the public health has been emphasized by the emergence of enterovirus 71 in 1998 as a serious pathogenic agents for children in Taiwan [24] and re-emerged in mainland China in 2008 [25] . In this issue, Duan and colleagues [26] summarized the findings of new enteroviruses by using NGS. Because of the application of new NGS technology they also challenged the Koch's postulates. A new model of Koch's postulates, named the metagenomic Koch's postulates, has provided guidance for the study of the pathogenicity of novel viruses. The review also provided a detailed description of the NGS and related molecular methods for the virus discovery followed by a list of new enteroviruses found in human feces. These include viruses in the family of Piconaviridae, Parvoviridae, Circoviridae, Astroviridae and Polyomaviridae.\\n\\nYu Xue-Jie and colleagues [27] reviewed the new bunyavirus, SFTSV, identified in China. As the virus discoverers, they have overviewed the whole process of the discovery, which is helpful and meaningful for the new virus discoveries in the future. The disease caused by SFTSV, with a CFR of 12%, had been in China for a couple of years before the causative agent was finally identified. There are still a lot of questions remained unknown for this new virus and vigorous studies are in great need. The transmission route of the virus has not been clarified but tick as vector is suspected. Domestic and wild animals, e.g., goats, boars, cattle and dogs, are believed to be the virus-amplifying hosts. Therefore the effective control measures are still under evaluation. Vaccines protecting the SFTSV infection are under its way in Chinese Center for Disease Control and Prevention. Recently a similar virus has been identified in both Japan and USA (a new name of Heartland virus was proposed for the US virus) [9] .\\n\\nIn addition to new viruses infecting human beings, some new viruses infecting animals but their public health significance needing to be further evaluated, have also been discovered. The new flavivirus, duck egg-drop syndrome virus (DEDSV), is a good example. Su and colleagues [28] reviewed the characterization of the DEDSV and its disease form in this issue. The virus was found closely-related to a long-time-known virus, Tembusu virus [29, 30] . Initially, the disease was only found in egg-raising ducks but soon it was found in pigeons, chickens and geese [31, 32] . Yet the transmission vector, though mosquitoes are suspected, has not been identified. Due to the public health concerns of its related viruses, potential human infection of DEDSV should be evaluated.\\n\\nResearch on insect viruses is reviving in recent years. In this issue, Zhou and colleagues [33] reviewed the newly-identified insect viruses in China. Insects are the largest group of animals on the Earth therefore they also carry many more viruses. Studies on these viruses can provide useful knowledge for our understanding about animal or human infecting viruses. More importantly, modification and application of insect-infecting viruses can be used as effective biologicals for the control of insect pest. The new viruses identified include Wuhan nodavirus (WhNV), a member of family Nodaviridae; Dendrolimus punctatus tetravirus (DpTV), a new member of the genus Omegatetravirus of the family Alphatetravirida; Ectropis obliqua picorna-like virus (EoV), a positive-strand RNA virus causing a lethal granulosis infection in the larvae of the tea looper (Ectropis obliqua), the virus a member of the Flaviridae family.\\n\\nWhile we are enjoying ourselves with the civilization of modern societies, the ecology has ever been changing. Human beings encounter more ecology-climate-changing problems, including the zoonotic pathogens. We have to face some unknown pathogenic agents passively. To get ourselves well prepared we also ought to actively hunt for unknown pathogens. Prediction and pre-warning can only be realized by knowing more about the unknown. This is especially true for infectious agents.\", 'document_id': 2628}]}, {'paragraphs': [{'qas': [{'question': 'What was the death toll in the 1918-1919 Spanish Influenza epidemic?', 'id': 1057, 'answers': [{'text': '50 million deaths worldwide', 'answer_start': 157}], 'is_impossible': False}, {'question': 'How many people were infected during the 1918 Spanish Influenza epidemic?', 'id': 1058, 'answers': [{'text': 'An estimated one third of the world’s population (or\\nz500 million persons) were infected and had clinical-\\nly apparent illnesses', 'answer_start': 985}], 'is_impossible': False}, {'question': 'What was the case fatality rate in the 1918 Spanish Influenza epidemic?', 'id': 1059, 'answers': [{'text': 'Case-\\nfatality rates were >2.5%, compared to <0.1% in other\\ninﬂuenza pandemics', 'answer_start': 1198}], 'is_impossible': False}, {'question': 'What was the death toll in the 1918-1919 Spanish Influenza epidemic?', 'id': 1060, 'answers': [{'text': 'Total deaths were estimated at\\nz50 million (577) and were arguably as high as 100 mil-\\nlion ', 'answer_start': 1284}], 'is_impossible': False}, {'question': 'Are the modern day Influenza viruses related to the 1918 Spanish Influenza virus?', 'id': 1062, 'answers': [{'text': ' All inﬂuenza A pandemics since that time, and\\nindeed almost all cases of inﬂuenza A worldwide (except-\\ning human infections from avian Viruses such as H5N1 and\\nH7N7), have been caused by descendants of the 1918\\nVirus, including “drifted” H1N1 Viruses and reassorted\\nH2N2 and H3N2 Viruses.', 'answer_start': 1439}], 'is_impossible': False}, {'question': 'Why is the Spanish Influenza virus the Mother of the modern influenza viruses?', 'id': 1063, 'answers': [{'text': 'The latter are composed of key\\ngenes from the 1918 Virus, updated by subsequently-incor—\\nporated avian inﬂuenza genes that code for novel surface\\n\\n \\n\\n*Armed Forces Institute of Pathology, Rockville, Maryland, USA;\\nand TNational Institutes of Health, Bethesda, Maryland, USA\\n\\nproteins, making the 1918 Virus indeed the “mother” of all\\npandemics.', 'answer_start': 1729}], 'is_impossible': False}, {'question': 'When was it  determined that the 1918  pandemic was caused by the H1N1 Influenza virus?', 'id': 1064, 'answers': [{'text': 'That question did not begin to be resolved until the 1930s,\\nwhen closely related inﬂuenza Viruses (now known to be\\nH1N1 Viruses) were isolated, ﬁrst from pigs and shortly\\nthereafter from humans. Seroepidemiologic studies soon\\nlinked both of these viruses to the 1918 pandemic', 'answer_start': 2355}], 'is_impossible': False}, {'question': 'Did the Spanish Influenza or Swine flu or the H1N1 virus disappear in humans for some time?', 'id': 1065, 'answers': [{'text': ' descendants of the 1918\\nVirus still persists enzootically in pigs. They probably also\\ncirculated continuously in humans, undergoing gradual\\nantigenic drift and causing annual epidemics, until the\\n1950s. With the appearance of a new H2N2 pandemic\\nstrain in 1957 (“Asian ﬂu”), the direct H1N1 Viral descen-\\ndants 0f the 1918 pandemic strain disappeared from human\\ncirculation entirely, although the related lineage persisted\\nenzootically in pigs.', 'answer_start': 2670}], 'is_impossible': False}, {'question': 'When did the Swine Flu (Spanish Influenza) virus reappear in humans?', 'id': 1066, 'answers': [{'text': ' But in 1977, human H1N1 Viruses\\nsuddenly “reemerged” from a laboratory freezer (9). They\\ncontinue to circulate endemically and epidemically.', 'answer_start': 3115}], 'is_impossible': False}, {'question': 'What descendant  lineages of the swine flu (Spanish Influenza) virus were identified in 2006?', 'id': 1068, 'answers': [{'text': ' 2 major descendant lineages of the 1918\\nH1N1 Virus, as well as 2 additional reassortant lineages,\\npersist naturally: a human epidemic/endemic H1N1 line-\\nage, a porcine enzootic H1N1 lineage (so-called classic\\nswine ﬂu), and the reassorted human H3N2 Virus lineage,\\nwhich like the human H1N1 Virus, has led to a porcine\\nH3N2 lineage.', 'answer_start': 3271}], 'is_impossible': False}, {'question': 'Are the modern descendant influenza viruses as dangerous as the 1918 parent swine flu (Spanish Influenza) H1N1 virus?', 'id': 1070, 'answers': [{'text': 'None of these Viral descendants, however,\\napproaches the pathogenicity of the 1918 parent Virus.', 'answer_start': 3605}], 'is_impossible': False}, {'question': 'How dangerous are the modern H1N1 (swine flu) and the H3N2 (Influenza A) viruses compared  to the 1918 H1N1 (swine flu Spanish Influenza)  viruses?', 'id': 1072, 'answers': [{'text': 'the human H1N1 and H3N2 lin-\\neages have both been associated with substantially lower\\nrates ofillness and death than the virus of 1918. In fact, cur-\\nrent H1N1 death rates are even lower than those for H3N2\\nlineage strains (prevalent from 1968 until the present).', 'answer_start': 3781}], 'is_impossible': False}, {'question': 'Are the descendant H1N1 strains of the 1918 H1N1 swine flu (Spanish Influenza) virus, still prevalent?', 'id': 1073, 'answers': [{'text': 'H1N1 Viruses descended from the 1918 strain, as well as \\nH3N2 Viruses, have now been cocirculating worldwide for\\n29 years and show little evidence of imminent extinction.', 'answer_start': 4045}], 'is_impossible': False}, {'question': 'Is the origin and epidemiology of the 1918 swine flu (Spanish Influenza) known?', 'id': 1075, 'answers': [{'text': 'ongoing studies to map Virulence\\nfactors are yielding interesting results. The 1918 sequence\\ndata, however, leave unanswered questions about the ori-\\ngin of the Virus (19) and about the epidemiology of the\\npandemic.', 'answer_start': 5741}], 'is_impossible': False}, {'question': 'What is the geographical origin of the H1N1 swine flu ?', 'id': 1082, 'answers': [{'text': 'Historical and epidemiologic data are inade-\\nquate to identify the geographic origin of the Virus (21),\\nand recent phylogenetic analysis of the 1918 Viral genome\\ndoes not place the Virus in any geographic context', 'answer_start': 6409}], 'is_impossible': False}, {'question': 'Is the geographical origin of the 1918 H1N1 swine flu known?', 'id': 1087, 'answers': [{'text': 'Confounding deﬁnite assignment of a geographic\\npoint of origin, the 1918 pandemic spread more or less\\nsimultaneously in 3 distinct waves during an z12-month\\nperiod in 191871919, in Europe, Asia, and North America\\n(the ﬁrst wave was best described in the United States in\\nMarch 1918)', 'answer_start': 6125}], 'is_impossible': False}, {'question': 'What is an unique feature of the 1918 swine flu?', 'id': 1088, 'answers': [{'text': 'the simultaneous (or nearly simultaneous) infection\\nof humans and swin', 'answer_start': 7949}], 'is_impossible': False}, {'question': 'What season or  time of the year do the new strains of influenza emerge?', 'id': 1089, 'answers': [{'text': 'Historical records since the 16th century suggest that\\nnew inﬂuenza pandemics may appear at any time of year,\\nnot necessarily in the familiar annual winter patterns of\\ninterpandemic years,', 'answer_start': 9120}], 'is_impossible': False}, {'question': 'Once appeared, when do the influenza like diseases occur in subsequent years?', 'id': 1090, 'answers': [{'text': 'confronted by the selection pressures of population immu-\\nnity, these pandemic Viruses begin to drift genetically and\\neventually settle into a pattern of annual epidemic recur-\\nrences caused by the drifted Virus variants.', 'answer_start': 9458}], 'is_impossible': False}, {'question': 'When did the first wave of the H1N1 swine flu (Spanish Influenza) occur?', 'id': 1091, 'answers': [{'text': ' a ﬁrst or spring wave\\nbegan in March 1918 and spread unevenly through the\\nUnited States, Europe, and possibly Asia over the next 6\\nmonths', 'answer_start': 9822}], 'is_impossible': False}, {'question': 'What was the death rate in the first wave of the 1918 swine flu pandemic?', 'id': 1092, 'answers': [{'text': ' Illness rates were high, but death rates\\nin most locales were not appreciably above normal.', 'answer_start': 9972}], 'is_impossible': False}, {'question': 'When were the second and the third wave of the 1918-1919 swine flu pandemic?', 'id': 1093, 'answers': [{'text': ' A sec-\\nond or fall wave spread globally from September to\\nNovember 1918 and was highly fatal. In many nations, a\\nthird wave occurred in early 1919 ', 'answer_start': 10064}], 'is_impossible': False}, {'question': 'What was the primary difference between the first wave and the 2nd and 3rd wave of the 1918-1919 swine flu pandemic?', 'id': 1094, 'answers': [{'text': 'the much higher fre-\\nquency of complicated, severe, and fatal cases in the last 2\\nwaves.', 'answer_start': 10785}], 'is_impossible': False}, {'question': 'Why the human influenza viruses do not disappear after herd immunity is developed?', 'id': 1096, 'answers': [{'text': 'The occurrence, and to some extent the severity, of recur-\\nrent annual outbreaks, are driven by Viral antigenic drift,\\nwith an antigenic variant Virus emerging to become domi-\\nnant', 'answer_start': 11043}], 'is_impossible': False}, {'question': 'What are the circumstances that promote the spread of influenza virus?', 'id': 1099, 'answers': [{'text': ' lower environ-\\nmental temperatures and human nasal temperatures (bene-\\nﬁcial to thermolabile Viruses such as inﬂuenza), optimal\\nhumidity, increased crowding indoors, and imperfect ven-\\ntilation due to closed windows and suboptimal airﬂow', 'answer_start': 11707}], 'is_impossible': False}, {'question': 'Do seasonal temperatures and humidity explain  the appearance of the three waves of the 1918 swine flu?', 'id': 1101, 'answers': [{'text': 'such factors cannot explain the 3 pandemic\\nwaves of 1918-1919, which occurred in the spring-sum-\\nmer, summer—fall, and winter (of the Northern\\nHemisphere), respectively. The ﬁrst 2 waves occurred at a\\ntime of year normally unfavorable to inﬂuenza Virus\\nspread. The second wave caused simultaneous outbreaks\\nin the Northern and Southern Hemispheres from\\nSeptember to November. ', 'answer_start': 11957}], 'is_impossible': False}, {'question': 'Which virus samples  from the 1918 swine flu pandemic have been identified?', 'id': 1105, 'answers': [{'text': 'pandemic Virus samples we have\\nyet identiﬁed are from second-wave patients', 'answer_start': 14082}], 'is_impossible': False}, {'question': 'Are viruses in the first and third waves of the 1918 swine flu pandemic same or derived from the virus from the second wave of the swine flu?', 'id': 1106, 'answers': [{'text': 'nothing\\ncan yet be said about whether the ﬁrst (spring) wave, or for\\nthat matter, the third wave, represented circulation of the\\nsame Virus or variants of it', 'answer_start': 14163}], 'is_impossible': False}, {'question': 'Was the 1918 swine flu virus novel to humans are was it derived from older viruses?', 'id': 1107, 'answers': [{'text': 'Viral sequence data now suggest that the entire 1918\\nVirus was novel to humans in, or shortly before, 1918, and\\nthat it thus was not a reassortant Virus produced from old\\nexisting strains that acquired 1 or more new genes', 'answer_start': 14934}], 'is_impossible': False}, {'question': 'Do avian flu viruses change over long periods?', 'id': 1108, 'answers': [{'text': 'Inﬂuenza Virus gene\\nsequences from a number ofﬁxed specimens ofwild birds\\ncollected circa 1918 show little difference from avian\\nViruses isolated today, indicating that avian Viruses likely\\nundergo little antigenic change in their natural hosts even\\nover long periods', 'answer_start': 15425}], 'is_impossible': False}, {'question': 'What  is the typical age profile of mortality in Influenza diseases?', 'id': 1109, 'answers': [{'text': 'The curve of inﬂuenza deaths by age at death has histor-\\nically, for at least 150 years, been U-shaped (Figure 2),\\nexhibiting mortality peaks in the very young and the very\\nold, with a comparatively low frequency of deaths at all\\nages in between', 'answer_start': 21665}], 'is_impossible': False}, {'question': 'What was the age profile of mortality in the 1918 swine flu?', 'id': 1110, 'answers': [{'text': 'age-speciﬁc death rates in the\\n1918 pandemic exhibited a distinct pattern that has not been\\ndocumented before or since: a “W—shaped” curve, similar to\\nthe familiar U-shaped curve but with the addition of a third\\n(middle) distinct peak of deaths in young adults z20410\\nyears of age', 'answer_start': 21925}], 'is_impossible': False}, {'question': 'Which age group was most susceptible to die during the 1918 swine flu pandemic?', 'id': 1111, 'answers': [{'text': 'Persons 65 years of age in 1918 had a dispro-\\nportionately high inﬂuenza incidence', 'answer_start': 23095}], 'is_impossible': False}, {'question': 'What was the death rate among children during the 1918 swine flu pandemic?', 'id': 1112, 'answers': [{'text': 'those 5 to 14\\nyears of age accounted for a disproportionate number of\\ninﬂuenza cases, but had a much lower death rate from\\ninﬂuenza and pneumonia than other age groups. ', 'answer_start': 23541}], 'is_impossible': False}, {'question': 'What theory provides partial explanation for the age-specific profile of the death rate in the 1918 swine flu pandemic?', 'id': 1113, 'answers': [{'text': ' the 1918 Virus had an intrinsically high Virulence, tem-\\npered only in those patients who had been born before\\n1889, e.g., because of exposure to a then-circulating Virus\\ncapable of providing partial immunoprotection against the\\n1918 Virus strain only in persons old enough (>35 years) to\\nhave been infected during that prior era ', 'answer_start': 24099}], 'is_impossible': False}, {'question': 'Is there a difference in the pathologic feature and course of disease between modern influenza pandemics and the 1918 swine flu pandemic?', 'id': 1114, 'answers': [{'text': ' the 1918\\npandemic was different in degree, but not in kind, from\\nprevious and subsequent pandemics. Despite the extraordi-\\nnary number of global deaths, most inﬂuenza cases in\\n1918 (>95% in most locales in industrialized nations) were\\nmild and essentially indistinguishable from inﬂuenza cases\\ntoday. ', 'answer_start': 26867}], 'is_impossible': False}, {'question': 'Could the 1918 swine flu virus been controlled  by modern day drugs or vaccines?', 'id': 1117, 'answers': [{'text': 'the 1918 and 1918-like Viruses would be\\nas sensitive as other typical Virus strains to the Food and\\nDrug Administrationiapproved antiinﬂuenza drugs riman-\\ntadine and oseltamivir.', 'answer_start': 27291}], 'is_impossible': False}, {'question': 'Why was there such a high death rate in the 19118 swine flu pandemic?', 'id': 1120, 'answers': [{'text': 'Clinically and pathologically, these\\nhigh death rates appear to be the result of several factors,\\nincluding a higher proportion of severe and complicated\\ninfections of the respiratory tract, rather than involvement\\nof organ systems outside the normal range of the inﬂuenza\\nVirus. Also, the deaths were concentrated in an unusually\\nyoung age group', 'answer_start': 27603}], 'is_impossible': False}, {'question': 'Is the molecular basis of human adaptation of a virus understood?', 'id': 1123, 'answers': [{'text': 'While data bearing\\non inﬂuenza Virus human cell adaptation (e.g., receptor\\nbinding) are beginning to be understood at the molecular\\nlevel, the basis for Viral adaptation to efﬁcient human-to-\\nhuman spread, the chief prerequisite for pandemic emer-\\ngence, is unknown for any inﬂuenza Virus.', 'answer_start': 29201}], 'is_impossible': False}], 'context': ' \\n\\n1918 Influenza: the Mother of All Pandemics\\n\\nJeffery K. Taubenberger\" and David M. Morens1-\\n\\nThe “Spanish\" influenza pandemic of 1918—1919,\\nwhich caused :50 million deaths worldwide, remains an\\nominous warning to public health. Many questions about its\\norigins, its unusual epidemiologic features, and the basis of\\nits pathogenicity remain unanswered. The public health\\nimplications of the pandemic therefore remain in doubt\\neven as we now grapple with the feared emergence of a\\npandemic caused by H5N1 or other virus. However, new\\ninformation about the 1918 virus is emerging, for example,\\nsequencing of the entire genome from archival autopsy tis-\\nsues. But, the viral genome alone is unlikely to provide\\nanswers to some critical questions. Understanding the\\n1918 pandemic and its implications for future pandemics\\nrequires careful experimentation and in-depth historical\\nanalysis.\\n\\n \\n\\n”Curiouser and curiouser/ ” criedAlice\\nLewis Carroll, Alice’s Adventures in Wonderland, 1865\\n\\nAn estimated one third of the world’s population (or\\nz500 million persons) were infected and had clinical-\\nly apparent illnesses (1,2) during the 191871919 inﬂuenza\\npandemic. The disease was exceptionally severe. Case-\\nfatality rates were >2.5%, compared to <0.1% in other\\ninﬂuenza pandemics (3,4). Total deaths were estimated at\\nz50 million (577) and were arguably as high as 100 mil-\\nlion (7).\\n\\nThe impact of this pandemic was not limited to\\n191871919. All inﬂuenza A pandemics since that time, and\\nindeed almost all cases of inﬂuenza A worldwide (except-\\ning human infections from avian Viruses such as H5N1 and\\nH7N7), have been caused by descendants of the 1918\\nVirus, including “drifted” H1N1 Viruses and reassorted\\nH2N2 and H3N2 Viruses. The latter are composed of key\\ngenes from the 1918 Virus, updated by subsequently-incor—\\nporated avian inﬂuenza genes that code for novel surface\\n\\n \\n\\n*Armed Forces Institute of Pathology, Rockville, Maryland, USA;\\nand TNational Institutes of Health, Bethesda, Maryland, USA\\n\\nproteins, making the 1918 Virus indeed the “mother” of all\\npandemics.\\n\\nIn 1918, the cause of human inﬂuenza and its links to\\navian and swine inﬂuenza were unknown. Despite clinical\\nand epidemiologic similarities to inﬂuenza pandemics of\\n1889, 1847, and even earlier, many questioned whether\\nsuch an explosively fatal disease could be inﬂuenza at all.\\nThat question did not begin to be resolved until the 1930s,\\nwhen closely related inﬂuenza Viruses (now known to be\\nH1N1 Viruses) were isolated, ﬁrst from pigs and shortly\\nthereafter from humans. Seroepidemiologic studies soon\\nlinked both of these viruses to the 1918 pandemic (8).\\nSubsequent research indicates that descendants of the 1918\\nVirus still persists enzootically in pigs. They probably also\\ncirculated continuously in humans, undergoing gradual\\nantigenic drift and causing annual epidemics, until the\\n1950s. With the appearance of a new H2N2 pandemic\\nstrain in 1957 (“Asian ﬂu”), the direct H1N1 Viral descen-\\ndants 0f the 1918 pandemic strain disappeared from human\\ncirculation entirely, although the related lineage persisted\\nenzootically in pigs. But in 1977, human H1N1 Viruses\\nsuddenly “reemerged” from a laboratory freezer (9). They\\ncontinue to circulate endemically and epidemically.\\n\\nThus in 2006, 2 major descendant lineages of the 1918\\nH1N1 Virus, as well as 2 additional reassortant lineages,\\npersist naturally: a human epidemic/endemic H1N1 line-\\nage, a porcine enzootic H1N1 lineage (so-called classic\\nswine ﬂu), and the reassorted human H3N2 Virus lineage,\\nwhich like the human H1N1 Virus, has led to a porcine\\nH3N2 lineage. None of these Viral descendants, however,\\napproaches the pathogenicity of the 1918 parent Virus.\\nApparently, the porcine H1N1 and H3N2 lineages uncom-\\nmonly infect humans, and the human H1N1 and H3N2 lin-\\neages have both been associated with substantially lower\\nrates ofillness and death than the virus of 1918. In fact, cur-\\nrent H1N1 death rates are even lower than those for H3N2\\nlineage strains (prevalent from 1968 until the present).\\nH1N1 Viruses descended from the 1918 strain, as well as \\nH3N2 Viruses, have now been cocirculating worldwide for\\n29 years and show little evidence of imminent extinction.\\n\\nTrying To Understand What Happened\\n\\nBy the early 1990s, 75 years of research had failed to\\nanswer a most basic question about the 1918 pandemic:\\nwhy was it so fatal? No Virus from 1918 had been isolated,\\nbut all of its apparent descendants caused substantially\\nmilder human disease. Moreover, examination of mortality\\ndata from the 1920s suggests that within a few years after\\n1918, inﬂuenza epidemics had settled into a pattern of\\nannual epidemicity associated with strain drifting and sub-\\nstantially lowered death rates. Did some critical Viral genet-\\nic event produce a 1918 Virus of remarkable pathogenicity\\nand then another critical genetic event occur soon after the\\n1918 pandemic to produce an attenuated H1N1 Virus?\\n\\nIn 1995, a scientiﬁc team identiﬁed archival inﬂuenza\\nautopsy materials collected in the autumn of 1918 and\\nbegan the slow process of sequencing small Viral RNA\\nfragments to determine the genomic structure of the\\ncausative inﬂuenza Virus (10). These efforts have now\\ndetermined the complete genomic sequence of 1 Virus and\\npartial sequences from 4 others. The primary data from the\\nabove studies (11717) and a number of reviews covering\\ndifferent aspects of the 1918 pandemic have recently been\\npublished ([8720) and conﬁrm that the 1918 Virus is the\\nlikely ancestor of all 4 of the human and swine H1N1 and\\nH3N2 lineages, as well as the “extinct” H2N2 lineage. No\\nknown mutations correlated with high pathogenicity in\\nother human or animal inﬂuenza Viruses have been found\\nin the 1918 genome, but ongoing studies to map Virulence\\nfactors are yielding interesting results. The 1918 sequence\\ndata, however, leave unanswered questions about the ori-\\ngin of the Virus (19) and about the epidemiology of the\\npandemic.\\n\\nWhen and Where Did the 1918 Inﬂuenza\\nPandemic Arise?\\n\\nBefore and after 1918, most inﬂuenza pandemics\\ndeveloped in Asia and spread from there to the rest of the\\nworld. Confounding deﬁnite assignment of a geographic\\npoint of origin, the 1918 pandemic spread more or less\\nsimultaneously in 3 distinct waves during an z12-month\\nperiod in 191871919, in Europe, Asia, and North America\\n(the ﬁrst wave was best described in the United States in\\nMarch 1918). Historical and epidemiologic data are inade-\\nquate to identify the geographic origin of the Virus (21),\\nand recent phylogenetic analysis of the 1918 Viral genome\\ndoes not place the Virus in any geographic context ([9).\\n\\nAlthough in 1918 inﬂuenza was not a nationally\\nreportable disease and diagnostic criteria for inﬂuenza and\\npneumonia were vague, death rates from inﬂuenza and\\npneumonia in the United States had risen sharply in 1915\\nand 1916 because of a major respiratory disease epidemic\\nbeginning in December 1915 (22). Death rates then dipped\\nslightly in 1917. The ﬁrst pandemic inﬂuenza wave\\nappeared in the spring of 1918, followed in rapid succes-\\nsion by much more fatal second and third waves in the fall\\nand winter of 191871919, respectively (Figure 1). Is it pos-\\nsible that a poorly-adapted H1N1 Virus was already begin-\\nning to spread in 1915, causing some serious illnesses but\\nnot yet sufﬁciently ﬁt to initiate a pandemic? Data consis-\\ntent with this possibility were reported at the time from\\nEuropean military camps (23), but a counter argument is\\nthat if a strain with a new hemagglutinin (HA) was caus-\\ning enough illness to affect the US national death rates\\nfrom pneumonia and inﬂuenza, it should have caused a\\npandemic sooner, and when it eventually did, in 1918,\\nmany people should have been immune or at least partial-\\nly immunoprotected. “Herald” events in 1915, 1916, and\\npossibly even in early 1918, if they occurred, would be dif-\\nﬁcult to identify.\\n\\nThe 1918 inﬂuenza pandemic had another unique fea-\\nture, the simultaneous (or nearly simultaneous) infection\\nof humans and swine. The Virus of the 1918 pandemic like-\\nly expressed an antigenically novel subtype to which most\\nhumans and swine were immunologically naive in 1918\\n(12,20). Recently published sequence and phylogenetic\\nanalyses suggest that the genes encoding the HA and neu-\\nraminidase (NA) surface proteins of the 1918 Virus were\\nderived from an avianlike inﬂuenza Virus shortly before\\nthe start of the pandemic and that the precursor Virus had\\nnot circulated widely in humans or swine in the few\\ndecades before (12,15, 24). More recent analyses of the\\nother gene segments of the Virus also support this conclu-\\nsion. Regression analyses of human and swine inﬂuenza\\nsequences obtained from 1930 to the present place the ini-\\ntial circulation of the 1918 precursor Virus in humans at\\napproximately 191571918 (20). Thus, the precursor was\\nprobably not circulating widely in humans until shortly\\nbefore 1918, nor did it appear to have jumped directly\\nfrom any species of bird studied to date (19). In summary,\\nits origin remains puzzling.\\n\\nWere the 3 Waves in 1918—1 919 Caused\\nby the Same Virus? If So, How and Why?\\nHistorical records since the 16th century suggest that\\nnew inﬂuenza pandemics may appear at any time of year,\\nnot necessarily in the familiar annual winter patterns of\\ninterpandemic years, presumably because newly shifted\\ninﬂuenza Viruses behave differently when they ﬁnd a uni-\\nversal or highly susceptible human population. Thereafter,\\nconfronted by the selection pressures of population immu-\\nnity, these pandemic Viruses begin to drift genetically and\\neventually settle into a pattern of annual epidemic recur-\\nrences caused by the drifted Virus variants.\\n\\nFigure 1. Three pandemic waves: weekly combined inﬂuenza and\\npneumonia mortality, United Kingdom, 1918—1919 (21).\\n\\nIn the 1918-1919 pandemic, a ﬁrst or spring wave\\nbegan in March 1918 and spread unevenly through the\\nUnited States, Europe, and possibly Asia over the next 6\\nmonths (Figure 1). Illness rates were high, but death rates\\nin most locales were not appreciably above normal. A sec-\\nond or fall wave spread globally from September to\\nNovember 1918 and was highly fatal. In many nations, a\\nthird wave occurred in early 1919 (21). Clinical similari-\\nties led contemporary observers to conclude initially that\\nthey were observing the same disease in the successive\\nwaves. The milder forms of illness in all 3 waves were\\nidentical and typical of inﬂuenza seen in the 1889 pandem-\\nic and in prior interpandemic years. In retrospect, even the\\nrapid progressions from uncomplicated inﬂuenza infec-\\ntions to fatal pneumonia, a hallmark of the 191871919 fall\\nand winter waves, had been noted in the relatively few\\nsevere spring wave cases. The differences between the\\nwaves thus seemed to be primarily in the much higher fre-\\nquency of complicated, severe, and fatal cases in the last 2\\nwaves.\\n\\nBut 3 extensive pandemic waves of inﬂuenza within 1\\nyear, occurring in rapid succession, with only the briefest\\nof quiescent intervals between them, was unprecedented.\\nThe occurrence, and to some extent the severity, of recur-\\nrent annual outbreaks, are driven by Viral antigenic drift,\\nwith an antigenic variant Virus emerging to become domi-\\nnant approximately every 2 to 3 years. Without such drift,\\ncirculating human inﬂuenza Viruses would presumably\\ndisappear once herd immunity had reached a critical\\nthreshold at which further Virus spread was sufﬁciently\\nlimited. The timing and spacing of inﬂuenza epidemics in\\ninterpandemic years have been subjects of speculation for\\ndecades. Factors believed to be responsible include partial\\nherd immunity limiting Virus spread in all but the most\\nfavorable circumstances, which include lower environ-\\nmental temperatures and human nasal temperatures (bene-\\nﬁcial to thermolabile Viruses such as inﬂuenza), optimal\\nhumidity, increased crowding indoors, and imperfect ven-\\ntilation due to closed windows and suboptimal airﬂow.\\n\\nHowever, such factors cannot explain the 3 pandemic\\nwaves of 1918-1919, which occurred in the spring-sum-\\nmer, summer—fall, and winter (of the Northern\\nHemisphere), respectively. The ﬁrst 2 waves occurred at a\\ntime of year normally unfavorable to inﬂuenza Virus\\nspread. The second wave caused simultaneous outbreaks\\nin the Northern and Southern Hemispheres from\\nSeptember to November. Furthermore, the interwave peri-\\nods were so brief as to be almost undetectable in some\\nlocales. Reconciling epidemiologically the steep drop in\\ncases in the ﬁrst and second waves with the sharp rises in\\ncases of the second and third waves is difﬁcult. Assuming\\neven transient postinfection immunity, how could suscep-\\ntible persons be too few to sustain transmission at 1 point,\\nand yet enough to start a new explosive pandemic wave a\\nfew weeks later? Could the Virus have mutated profoundly\\nand almost simultaneously around the world, in the short\\nperiods between the successive waves? Acquiring Viral\\ndrift sufﬁcient to produce new inﬂuenza strains capable of\\nescaping population immunity is believed to take years of\\nglobal circulation, not weeks of local circulation. And hav-\\ning occurred, such mutated Viruses normally take months\\nto spread around the world.\\n\\nAt the beginning of other “off season” inﬂuenza pan-\\ndemics, successive distinct waves within a year have not\\nbeen reported. The 1889 pandemic, for example, began in\\nthe late spring of 1889 and took several months to spread\\nthroughout the world, peaking in northern Europe and the\\nUnited States late in 1889 or early in 1890. The second\\nrecurrence peaked in late spring 1891 (more than a year\\nafter the ﬁrst pandemic appearance) and the third in early\\n1892 (21 ). As was true for the 1918 pandemic, the second\\n1891 recurrence produced of the most deaths. The 3 recur-\\nrences in 1889-1892, however, were spread over >3 years,\\nin contrast to 191871919, when the sequential waves seen\\nin individual countries were typically compressed into\\nz879 months.\\n\\nWhat gave the 1918 Virus the unprecedented ability to\\ngenerate rapidly successive pandemic waves is unclear.\\nBecause the only 1918 pandemic Virus samples we have\\nyet identiﬁed are from second-wave patients ([6), nothing\\ncan yet be said about whether the ﬁrst (spring) wave, or for\\nthat matter, the third wave, represented circulation of the\\nsame Virus or variants of it. Data from 1918 suggest that\\npersons infected in the second wave may have been pro-\\ntected from inﬂuenza in the third wave. But the few data\\nbearing on protection during the second and third waves\\nafter infection in the ﬁrst wave are inconclusive and do lit-\\ntle to resolve the question of whether the ﬁrst wave was\\ncaused by the same Virus or whether major genetic evolu-\\ntionary events were occurring even as the pandemic\\nexploded and progressed. Only inﬂuenza RNAipositive\\nhuman samples from before 1918, and from all 3 waves,\\ncan answer this question.\\n\\nWhat Was the Animal Host\\nOrigin of the Pandemic Virus?\\n\\nViral sequence data now suggest that the entire 1918\\nVirus was novel to humans in, or shortly before, 1918, and\\nthat it thus was not a reassortant Virus produced from old\\nexisting strains that acquired 1 or more new genes, such as\\nthose causing the 1957 and 1968 pandemics. On the con-\\ntrary, the 1918 Virus appears to be an avianlike inﬂuenza\\nVirus derived in toto from an unknown source (17,19), as\\nits 8 genome segments are substantially different from\\ncontemporary avian inﬂuenza genes. Inﬂuenza Virus gene\\nsequences from a number ofﬁxed specimens ofwild birds\\ncollected circa 1918 show little difference from avian\\nViruses isolated today, indicating that avian Viruses likely\\nundergo little antigenic change in their natural hosts even\\nover long periods (24,25).\\n\\nFor example, the 1918 nucleoprotein (NP) gene\\nsequence is similar to that ofviruses found in wild birds at\\nthe amino acid level but very divergent at the nucleotide\\nlevel, which suggests considerable evolutionary distance\\nbetween the sources of the 1918 NP and of currently\\nsequenced NP genes in wild bird strains (13,19). One way\\nof looking at the evolutionary distance of genes is to com-\\npare ratios of synonymous to nonsynonymous nucleotide\\nsubstitutions. A synonymous substitution represents a\\nsilent change, a nucleotide change in a codon that does not\\nresult in an amino acid replacement. A nonsynonymous\\nsubstitution is a nucleotide change in a codon that results\\nin an amino acid replacement. Generally, a Viral gene sub-\\njected to immunologic drift pressure or adapting to a new\\nhost exhibits a greater percentage of nonsynonymous\\nmutations, while a Virus under little selective pressure\\naccumulates mainly synonymous changes. Since little or\\nno selection pressure is exerted on synonymous changes,\\nthey are thought to reﬂect evolutionary distance.\\n\\nBecause the 1918 gene segments have more synony-\\nmous changes from known sequences of wild bird strains\\nthan expected, they are unlikely to have emerged directly\\nfrom an avian inﬂuenza Virus similar to those that have\\nbeen sequenced so far. This is especially apparent when\\none examines the differences at 4-fold degenerate codons,\\nthe subset of synonymous changes in which, at the third\\ncodon position, any of the 4 possible nucleotides can be\\nsubstituted without changing the resulting amino acid. At\\nthe same time, the 1918 sequences have too few amino acid\\ndiﬁerences from those of wild-bird strains to have spent\\nmany years adapting only in a human or swine intermedi-\\nate host. One possible explanation is that these unusual\\ngene segments were acquired from a reservoir of inﬂuenza\\nVirus that has not yet been identiﬁed or sampled. All of\\nthese ﬁndings beg the question: where did the 1918 Virus\\ncome from?\\n\\nIn contrast to the genetic makeup of the 1918 pandem-\\nic Virus, the novel gene segments of the reassorted 1957\\nand 1968 pandemic Viruses all originated in Eurasian avian\\nViruses (26); both human Viruses arose by the same mech-\\nanismireassortment of a Eurasian wild waterfowl strain\\nwith the previously circulating human H1N1 strain.\\nProving the hypothesis that the Virus responsible for the\\n1918 pandemic had a markedly different origin requires\\nsamples of human inﬂuenza strains circulating before\\n1918 and samples of inﬂuenza strains in the wild that more\\nclosely resemble the 1918 sequences.\\n\\nWhat Was the Biological Basis for\\n1918 Pandemic Virus Pathogenicity?\\n\\nSequence analysis alone does not oﬁer clues to the\\npathogenicity of the 1918 Virus. A series of experiments\\nare under way to model Virulence in Vitro and in animal\\nmodels by using Viral constructs containing 1918 genes\\nproduced by reverse genetics.\\n\\nInﬂuenza Virus infection requires binding of the HA\\nprotein to sialic acid receptors on host cell surface. The HA\\nreceptor-binding site conﬁguration is different for those\\ninﬂuenza Viruses adapted to infect birds and those adapted\\nto infect humans. Inﬂuenza Virus strains adapted to birds\\npreferentially bind sialic acid receptors with 01 (273) linked\\nsugars (27729). Human-adapted inﬂuenza Viruses are\\nthought to preferentially bind receptors with 01 (2%) link-\\nages. The switch from this avian receptor conﬁguration\\nrequires of the Virus only 1 amino acid change (30), and\\nthe HAs of all 5 sequenced 1918 Viruses have this change,\\nwhich suggests that it could be a critical step in human host\\nadaptation. A second change that greatly augments Virus\\nbinding to the human receptor may also occur, but only 3\\nof5 1918 HA sequences have it (16).\\n\\nThis means that at least 2 H1N1 receptor-binding vari-\\nants cocirculated in 1918: 1 with high—afﬁnity binding to\\nthe human receptor and 1 with mixed-afﬁnity binding to\\nboth avian and human receptors. No geographic or chrono-\\nlogic indication eXists to suggest that one of these variants\\nwas the precursor of the other, nor are there consistent dif-\\nferences between the case histories or histopathologic fea-\\ntures of the 5 patients infected with them. Whether the\\nViruses were equally transmissible in 1918, whether they\\nhad identical patterns of replication in the respiratory tree,\\nand whether one or both also circulated in the ﬁrst and\\nthird pandemic waves, are unknown.\\nIn a series of in Vivo experiments, recombinant inﬂuen-\\nza Viruses containing between 1 and 5 gene segments of\\nthe 1918 Virus have been produced. Those constructs\\nbearing the 1918 HA and NA are all highly pathogenic in\\n\\nmice (31). Furthermore, expression microarray analysis\\nperformed on whole lung tissue of mice infected with the\\n1918 HA/NA recombinant showed increased upregulation\\nof genes involved in apoptosis, tissue injury, and oxidative\\ndamage (32). These ﬁndings are unexpected because the\\nViruses with the 1918 genes had not been adapted to mice;\\ncontrol experiments in which mice were infected with\\nmodern human Viruses showed little disease and limited\\nViral replication. The lungs of animals infected with the\\n1918 HA/NA construct showed bronchial and alveolar\\nepithelial necrosis and a marked inﬂammatory inﬁltrate,\\nwhich suggests that the 1918 HA (and possibly the NA)\\ncontain Virulence factors for mice. The Viral genotypic\\nbasis of this pathogenicity is not yet mapped. Whether\\npathogenicity in mice effectively models pathogenicity in\\nhumans is unclear. The potential role of the other 1918 pro-\\nteins, singularly and in combination, is also unknown.\\nExperiments to map further the genetic basis of Virulence\\nof the 1918 Virus in various animal models are planned.\\nThese experiments may help deﬁne the Viral component to\\nthe unusual pathogenicity of the 1918 Virus but cannot\\naddress whether speciﬁc host factors in 1918 accounted for\\nunique inﬂuenza mortality patterns.\\n\\nWhy Did the 1918 Virus Kill So Many Healthy\\nYoung Ad ults?\\n\\nThe curve of inﬂuenza deaths by age at death has histor-\\nically, for at least 150 years, been U-shaped (Figure 2),\\nexhibiting mortality peaks in the very young and the very\\nold, with a comparatively low frequency of deaths at all\\nages in between. In contrast, age-speciﬁc death rates in the\\n1918 pandemic exhibited a distinct pattern that has not been\\ndocumented before or since: a “W—shaped” curve, similar to\\nthe familiar U-shaped curve but with the addition of a third\\n(middle) distinct peak of deaths in young adults z20410\\nyears of age. Inﬂuenza and pneumonia death rates for those\\n1534 years of age in 191871919, for example, were\\n20 times higher than in previous years (35). Overall, near-\\nly half of the inﬂuenza—related deaths in the 1918 pandem-\\nic were in young adults 20410 years of age, a phenomenon\\nunique to that pandemic year. The 1918 pandemic is also\\nunique among inﬂuenza pandemics in that absolute risk of\\ninﬂuenza death was higher in those <65 years of age than in\\nthose >65; persons <65 years of age accounted for >99% of\\nall excess inﬂuenza—related deaths in 191871919. In com-\\nparison, the <65-year age group accounted for 36% of all\\nexcess inﬂuenza—related deaths in the 1957 H2N2 pandem-\\nic and 48% in the 1968 H3N2 pandemic (33).\\nA sharper perspective emerges when 1918 age-speciﬁc\\ninﬂuenza morbidity rates (21) are used to adj ust the W-\\nshaped mortality curve (Figure 3, panels, A, B, and C\\n[35,37]). Persons 65 years of age in 1918 had a dispro-\\nportionately high inﬂuenza incidence (Figure 3, panel A).\\n\\nBut even after adjusting age-speciﬁc deaths by age-specif—\\nic clinical attack rates (Figure 3, panel B), a W—shaped\\ncurve with a case-fatality peak in young adults remains and\\nis signiﬁcantly different from U-shaped age-speciﬁc case-\\nfatality curves typically seen in other inﬂuenza years, e.g.,\\n192871929 (Figure 3, panel C). Also, in 1918 those 5 to 14\\nyears of age accounted for a disproportionate number of\\ninﬂuenza cases, but had a much lower death rate from\\ninﬂuenza and pneumonia than other age groups. To explain\\nthis pattern, we must look beyond properties of the Virus to\\nhost and environmental factors, possibly including\\nimmunopathology (e.g., antibody-dependent infection\\nenhancement associated with prior Virus exposures [38])\\nand exposure to risk cofactors such as coinfecting agents,\\nmedications, and environmental agents.\\n\\nOne theory that may partially explain these ﬁndings is\\nthat the 1918 Virus had an intrinsically high Virulence, tem-\\npered only in those patients who had been born before\\n1889, e.g., because of exposure to a then-circulating Virus\\ncapable of providing partial immunoprotection against the\\n1918 Virus strain only in persons old enough (>35 years) to\\nhave been infected during that prior era (35). But this the-\\nory would present an additional paradox: an obscure pre-\\ncursor Virus that left no detectable trace today would have\\nhad to have appeared and disappeared before 1889 and\\nthen reappeared more than 3 decades later.\\n\\nEpidemiologic data on rates of clinical inﬂuenza by\\nage, collected between 1900 and 1918, provide good evi-\\ndence for the emergence of an antigenically novel inﬂuen-\\nza Virus in 1918 (21). Jordan showed that from 1900 to\\n1917, the 5- to 15-year age group accounted for 11% of\\ntotal inﬂuenza cases, while the >65-year age group\\naccounted for 6 % of inﬂuenza cases. But in 1918, cases in\\n\\nFigure 2. “U-” and “W—” shaped combined inﬂuenza and pneumo-\\nnia mortality, by age at death, per 100,000 persons in each age\\ngroup, United States, 1911—1918. Influenza- and pneumonia-\\nspeciﬁc death rates are plotted for the interpandemic years\\n1911—1917 (dashed line) and for the pandemic year 1918 (solid\\nline) (33,34).\\n\\nIncidence male per 1 .nao persunslage group\\nMortality per 1.000 persunslige group\\n\\n+ Case—fataiity rale 1918—1919 \\n\\nCase fatalily par 100 persons ill wilh P&I pel age group\\n\\nFigure 3. Influenza plus pneumonia (P&l) (combined) age-specific\\nincidence rates per 1,000 persons per age group (panel A), death\\nrates per 1,000 persons, ill and well combined (panel B), and\\ncase-fatality rates (panel C, solid line), US Public Health Service\\nhouse-to-house surveys, 8 states, 1918 (36). A more typical curve\\nof age-specific influenza case-fatality (panel C, dotted line) is\\ntaken from US Public Health Service surveys during 1928—1929\\n(37).\\n\\nthe 5 to 15-year-old group jumped to 25% of inﬂuenza\\ncases (compatible with exposure to an antigenically novel\\nVirus strain), while the >65-year age group only accounted\\nfor 0.6% of the inﬂuenza cases, ﬁndings consistent with\\npreviously acquired protective immunity caused by an\\nidentical or closely related Viral protein to which older per-\\nsons had once been exposed. Mortality data are in accord.\\nIn 1918, persons >75 years had lower inﬂuenza and\\n\\npneumonia case-fatality rates than they had during the\\nprepandemic period of 191171917. At the other end of the\\nage spectrum (Figure 2), a high proportion of deaths in\\ninfancy and early childhood in 1918 mimics the age pat-\\ntern, if not the mortality rate, of other inﬂuenza pandemics.\\n\\nCould a 1918-like Pandemic Appear Again?\\nIf So, What Could We Do About It?\\n\\nIn its disease course and pathologic features, the 1918\\npandemic was different in degree, but not in kind, from\\nprevious and subsequent pandemics. Despite the extraordi-\\nnary number of global deaths, most inﬂuenza cases in\\n1918 (>95% in most locales in industrialized nations) were\\nmild and essentially indistinguishable from inﬂuenza cases\\ntoday. Furthermore, laboratory experiments with recombi-\\nnant inﬂuenza Viruses containing genes from the 1918\\nVirus suggest that the 1918 and 1918-like Viruses would be\\nas sensitive as other typical Virus strains to the Food and\\nDrug Administrationiapproved antiinﬂuenza drugs riman-\\ntadine and oseltamivir.\\n\\nHowever, some characteristics of the 1918 pandemic\\nappear unique: most notably, death rates were 5 7 20 times\\nhigher than expected. Clinically and pathologically, these\\nhigh death rates appear to be the result of several factors,\\nincluding a higher proportion of severe and complicated\\ninfections of the respiratory tract, rather than involvement\\nof organ systems outside the normal range of the inﬂuenza\\nVirus. Also, the deaths were concentrated in an unusually\\nyoung age group. Finally, in 1918, 3 separate recurrences\\nof inﬂuenza followed each other with unusual rapidity,\\nresulting in 3 explosive pandemic waves within a year’s\\ntime (Figure 1). Each of these unique characteristics may\\nreﬂect genetic features of the 1918 Virus, but understand-\\ning them will also require examination of host and envi-\\nronmental factors.\\n\\nUntil we can ascertain which of these factors gave rise\\nto the mortality patterns observed and learn more about the\\nformation of the pandemic, predictions are only educated\\nguesses. We can only conclude that since it happened once,\\nanalogous conditions could lead to an equally devastating\\npandemic.\\n\\nLike the 1918 Virus, H5N1 is an avian Virus (39),\\nthough a distantly related one. The evolutionary path that\\nled to pandemic emergence in 1918 is entirely unknown,\\nbut it appears to be different in many respects from the cur-\\nrent situation with H5N1. There are no historical data,\\neither in 1918 or in any other pandemic, for establishing\\nthat a pandemic “precursor” Virus caused a highly patho-\\ngenic outbreak in domestic poultry, and no highly patho-\\ngenic avian inﬂuenza (HPAI) Virus, including H5N1 and a\\nnumber of others, has ever been known to cause a major\\nhuman epidemic, let alone a pandemic. While data bearing\\non inﬂuenza Virus human cell adaptation (e.g., receptor\\nbinding) are beginning to be understood at the molecular\\nlevel, the basis for Viral adaptation to efﬁcient human-to-\\nhuman spread, the chief prerequisite for pandemic emer-\\ngence, is unknown for any inﬂuenza Virus. The 1918 Virus\\nacquired this trait, but we do not know how, and we cur-\\nrently have no way of knowing whether H5N1 Viruses are\\nnow in a parallel process of acquiring human-to-human\\ntransmissibility. Despite an explosion of data on the 1918\\nVirus during the past decade, we are not much closer to\\nunderstanding pandemic emergence in 2006 than we were\\nin understanding the risk of H1N1 “swine ﬂu” emergence\\nin 1976.\\n\\nEven with modern antiviral and antibacterial drugs,\\nvaccines, and prevention knowledge, the return of a pan-\\ndemic Virus equivalent in pathogenicity to the Virus of\\n1918 would likely kill >100 million people worldwide. A\\npandemic Virus with the (alleged) pathogenic potential of\\nsome recent H5N1 outbreaks could cause substantially\\nmore deaths.\\n\\nWhether because of Viral, host or environmental fac-\\ntors, the 1918 Virus causing the ﬁrst or ‘spring’ wave was\\nnot associated with the exceptional pathogenicity of the\\nsecond (fall) and third (winter) waves. Identiﬁcation of an\\ninﬂuenza RNA-positive case from the ﬁrst wave could\\npoint to a genetic basis for Virulence by allowing differ-\\nences in Viral sequences to be highlighted. Identiﬁcation of\\npre-1918 human inﬂuenza RNA samples would help us\\nunderstand the timing of emergence of the 1918 Virus.\\nSurveillance and genomic sequencing of large numbers of\\nanimal inﬂuenza Viruses will help us understand the genet-\\nic basis of host adaptation and the extent of the natural\\nreservoir of inﬂuenza Viruses. Understanding inﬂuenza\\npandemics in general requires understanding the 1918 pan-\\ndemic in all its historical, epidemiologic, and biologic\\naspects.\\n\\nDr Taubenberger is chair of the Department of Molecular\\nPathology at the Armed Forces Institute of Pathology, Rockville,\\nMaryland. His research interests include the molecular patho-\\nphysiology and evolution of inﬂuenza Viruses.\\n\\nDr Morens is an epidemiologist with a long-standing inter-\\nest in emerging infectious diseases, Virology, tropical medicine,\\nand medical history. Since 1999, he has worked at the National\\nInstitute of Allergy and Infectious Diseases.\\n\\nReferences\\n\\n1. Frost WH. Statistics of inﬂuenza morbidity. Public Health Rep.\\n19203558497.\\n2. Bumet F, Clark E. Inﬂuenza: a survey ofthe last 50 years in the light\\nof modern work on the Virus of epidemic inﬂuenza. Melbourne:\\nMacMillan; 1942.\\n3. Marks G, Beatty WK. Epidemics. New York: Scribners, 1976.\\n4. Rosenau MJ, Last JM. Maxcy-Rosenau preventative medicine and\\npublic health. New York: Appleton-Century-Crofts; 1980.\\n5. Crosby A. America’s forgotten pandemic. Cambridge (UK):\\nCambridge University Press;1989.\\n6. Patterson KD, Pyle GF. The geography and mortality of the 1918\\ninfluenza pandemic. Bull Hist Med. 1991;65:4–21.\\n7. Johnson NPAS, Mueller J. Updating the accounts: global mortality of\\nthe 1918–1920 “Spanish” influenza pandemic. Bull Hist Med\\n2002;76:105–15.\\n8. Shope RE. The incidence of neutralizing antibodies for swine\\ninfluenza virus in the sera of human beings of different ages. J Exp\\nMed. 1936;63:669–84.\\n9. Kendal AP, Noble GR, Skehel JJ, Dowdle WR. Antigenic similarity\\nof influenza A (H1N1) viruses from epidemics in 1977–1978 to\\n“Scandinavian” strains isolated in epidemics of 1950–1951. Virology.\\n1978;89:632–6.\\n10. Taubenberger JK, Reid AH, Krafft AE, Bijwaard KE, Fanning TG.\\nInitial genetic characterization of the 1918 “Spanish” influenza virus.\\nScience. 1997;275:1793–6.\\n11. Basler CF, Reid AH, Dybing JK, Janczewski TA, Fanning TG, Zheng\\nH, et al. Sequence of the 1918 pandemic influenza virus nonstructural gene (NS) segment and characterization of recombinant viruses\\nbearing the 1918 NS genes. Proc Natl Acad Sci U S A\\n2001;98:2746–51.\\n12. Reid AH, Fanning TG, Hultin JV, Taubenberger JK. Origin and evolution of the 1918 “Spanish” influenza virus hemagglutinin gene.\\nProc Natl Acad Sci U S A 1999;96:1651–6.\\n13. Reid AH, Fanning TG, Janczewski TA, Lourens RM, and\\nTaubenberger JK. Novel origin of the 1918 pandemic influenza virus\\nnucleoprotein gene segment. J Virol. 2004;78:12462–70.\\n14. Reid AH, Fanning TG, Janczewski TA, McCall S, Taubenberger JK.\\nCharacterization of the 1918 “Spanish” influenza virus matrix gene\\nsegment. J Virol. 2002;76:10717–23.\\n15. Reid AH, Fanning TG, Janczewski TA, Taubenberger JK.\\nCharacterization of the 1918 “Spanish” influenza virus neuraminidase gene. Proc Natl Acad Sci U S A 2000;97:6785–90.\\n16. Reid AH, Janczewski TA, Lourens RM, Elliot AJ, Daniels RS, Berry\\nCL, et al. 1918 influenza pandemic caused by highly conserved viruses with two receptor-binding variants. Emerg Infect Dis.\\n2003;9:1249–53.\\n17. Taubenberger JK, Reid AH, Lourens RM, Wang R, Jin G, Fanning\\nTG. Characterization of the 1918 influenza virus polymerase genes.\\nNature. 2005;437:889–93.\\n18. Reid AH, Taubenberger JK. The 1918 flu and other influenza pandemics: “over there” and back again. Lab Invest. 1999;79:95–101.\\n19. Reid AH, Taubenberger JK, Fanning TG. Evidence of an absence: the\\ngenetic origins of the 1918 pandemic influenza virus. Nat Rev\\nMicrobiol. 2004;2:909–14.\\n20. Taubenberger JK, Reid AH, Fanning TG. The 1918 influenza virus: a\\nkiller comes into view. Virology. 2000;274:241–5.\\n21. Jordan E. Epidemic influenza: a survey. Chicago: American Medical\\nAssociation, 1927.\\n22. Capps J, Moody A. The recent epidemic of grip. JAMA.\\n1916;67:1349–50.\\n33. Oxford JS, Sefton A, Jackson R, Innes W, Daniels RS, Johnson NP.\\nWorld War I may have allowed the emergence of “Spanish” influenza. Lancet Infect Dis. 2002;2:111–4.\\n24. Fanning TG, Slemons RD, Reid AH, Janczewski TA, Dean J,\\nTaubenberger JK. 1917 avian influenza virus sequences suggest that\\nthe 1918 pandemic virus did not acquire its hemagglutinin directly\\nfrom birds. J Virol. 2002;76:7860–2.\\n25. Reid AH, Fanning TG, Slemons RD, Janczewski TA, Dean J,\\nTaubenberger JK. Relationship of pre-1918 avian influenza HA and\\nNP sequences to subsequent avian influenza strains. Avian Dis.\\n2003;47:921–5.\\n26. Bean W, Schell M, Katz J, Kawaoka Y, Naeve C, Gorman O, et al.\\nEvolution of the H3 influenza virus hemagglutinin from human and\\nnonhuman hosts. J Virol. 1992;66:1129–38.\\n27. Weis W, Brown JH, Cusack S, Paulson JC, Skehel JJ, Wiley DC.\\nStructure of the influenza virus haemagglutinin complexed with its\\nreceptor, sialic acid. Nature. 1988;333:426–31.\\n28. Gambaryan AS, Tuzikov AB, Piskarev VE, Yamnikova SS, Lvov DK,\\nRobertson JS, et al. Specification of receptor-binding phenotypes of\\ninfluenza virus isolates from different hosts using synthetic sialylglycopolymers: non-egg-adapted human H1 and H3 influenza A and\\ninfluenza B viruses share a common high binding affinity for 6′-sialyl(N-acetyllactosamine). Virology. 1997;232: 345–50.\\n29. Matrosovich M, Gambaryan A, Teneberg S, Piskarev VE, Yamnikova\\nSS, Lvov DK, et al. Avian influenza A viruses differ from human\\nviruses by recognition of sialyloigosaccharides and gangliosides and\\nby a higher conservation of the HA receptor-binding site. Virology.\\n1997;233:224–34.\\n30. Glaser L, Stevens J, Zamarin D, Wilson IA, Garcia-Sastre A, Tumpey\\nTM, et al. A single amino acid substitution in the 1918 influenza virus\\nhemagglutinin changes the receptor binding specificity. J Virol.\\n2005;79:11533–6.\\n31. Kobasa D, Takada A, Shinya K, Hatta M, Halfmann P, Theriault S, et\\nal. Enhanced virulence of influenza A viruses with the haemagglutinin of the 1918 pandemic virus. Nature. 2004;431:703–7.\\n32. Kash JC, Basler CF, Garcia-Sastre A, Carter V, Billharz R, Swayne\\nDE, et al. Global host immune response: pathogenesis and transcriptional profiling of type A influenza viruses expressing the hemagglutinin and neuraminidase genes from the 1918 pandemic virus. J Virol.\\n2004;78:9499–511.\\n33. Grove RD, Hetzel AM. Vital statistics rates in the United States:\\n1940–1960. Washington: US Government Printing Office, 1968.\\n34. Linder FE, Grove RD. Vital statistics rates in the United States:\\n1900–1940. Washington: US Government Printing Office, 1943.\\n35. Simonsen L, Clarke MJ, Schonberger LB, Arden NH, Cox NJ,\\nFukuda K. Pandemic versus epidemic influenza mortality: a pattern\\nof changing age distribution. J Infect Dis 1998;178:53–60.\\n36. Frost WH. The epidemiology of influenza. Public Health Rep.\\n1919;34:1823–61.\\n37. Collins SD. Age and sex incidence of influenza and pneumonia morbidity and mortality in the epidemic of 1928-1929 with comparative\\ndata for the epidemic of 1918–1919. Public Health Rep.\\n1931;46:1909–37.\\n38. Majde JA. Influenza: Learn from the past. ASM News. 1996;62:514.\\n39. Peiris JS, Yu WC, Leung CW, Cheung CY, Ng WF, Nicholls JM, et al.\\nRe-emergence of fatal human influenza A subtype H5N1 disease.\\nLancet. 2004;363:617–9.\\n\\nAddress for correspondence: Jeffery K. Taubenberger, Department of\\nMolecular Pathology, Armed Forces Institute of Pathology, 1413\\nResearch Blvd, Bldg 101, Rm 1057, Rockville, MD 20850-3125, USA;\\nfax. 301-295-9507; email: taubenberger@afip.osd.mil\\n\\nThe opinions expressed by authors contributing to this journal do\\nnot necessarily reflect the opinions of the Centers for Disease\\nControl and Prevention or the institutions with which the authors\\nare affiliated.', 'document_id': 2684}]}, {'paragraphs': [{'qas': [{'question': 'What  was the initial growth phase pattern?', 'id': 1872, 'answers': [{'text': 'exponential growth pattern', 'answer_start': 1435}], 'is_impossible': False}, {'question': 'What was the result of under-reporting?', 'id': 1875, 'answers': [{'text': '469 (95% CI: 403&minus;540) unreported cases from 1 to 15 Januar', 'answer_start': 1514}], 'is_impossible': False}, {'question': 'What is R0?', 'id': 1877, 'answers': [{'text': 'basic reproduction number,', 'answer_start': 836}], 'is_impossible': False}, {'question': 'What  is likely increase of the reporting rate after the 17th January 2020?', 'id': 1879, 'answers': [{'text': 'reased 21-fold (95% CI: 18', 'answer_start': 1649}], 'is_impossible': False}, {'question': 'What is the estimated value of R0?', 'id': 1880, 'answers': [{'text': '019-nCoV at 2.56 (95% CI', 'answer_start': 1779}], 'is_impossible': False}, {'question': 'What is the likely period of under-reporting?', 'id': 1881, 'answers': [{'text': ' to have occurred during the first ha', 'answer_start': 1865}], 'is_impossible': False}, {'question': 'Where and when was 2019-nCOV first identified?', 'id': 1882, 'answers': [{'text': 's first identified in Wuhan, China', 'answer_start': 2064}], 'is_impossible': False}, {'question': 'What  are some of the symptoms caused by the virus?', 'id': 1884, 'answers': [{'text': 'ymptoms including fever, cough, and sh', 'answer_start': 2151}], 'is_impossible': False}, {'question': 'What was the cumulative number of reported cases by 1 January 2020?', 'id': 1886, 'answers': [{'text': 'sed', 'answer_start': 2266}], 'is_impossible': False}, {'question': 'As of 26 January 2020, what had the outbreak resulted in?', 'id': 1889, 'answers': [{'text': 'k had resulted in 2066 (618 of them are in Wuhan) confirmed cases and 56 (45 of them were in Wuhan) deaths in ', 'answer_start': 2405}], 'is_impossible': False}, {'question': 'As of 26 January 2020, what countries had sporadic cases?', 'id': 1891, 'answers': [{'text': ' were reported in Thailand, Japan, Republic of Korea, Hong Kong, Taiwan, Australia, and', 'answer_start': 2574}], 'is_impossible': False}, {'question': 'What was the result  of the Imperial College estimation?', 'id': 1892, 'answers': [{'text': 'at there had been 4000 (95%CI: 1000-9700) cases in Wuhan with symptoms onset by 18 January 2020, and the basic reproduction number (R 0 ) was estimated at 2.6 (95', 'answer_start': 2927}], 'is_impossible': False}, {'question': 'Who release the time series data from 10th  to 20th January 2020?', 'id': 1893, 'answers': [{'text': 'y released by the Wuhan Municipal ', 'answer_start': 3981}], 'is_impossible': False}, {'question': 'Who released the time series data from after 21st January 2020?', 'id': 1894, 'answers': [{'text': ', and later by the National Health C', 'answer_start': 4064}], 'is_impossible': False}, {'question': 'How was the epidemic curve modelled?', 'id': 1895, 'answers': [{'text': ', the C i series, as an exponential growi', 'answer_start': 5150}], 'is_impossible': False}, {'question': 'How was the epidemic curve modeled?', 'id': 1907, 'answers': [{'text': ', the C i series, as an exponential growin', 'answer_start': 5150}], 'is_impossible': False}], 'context': \"Estimating the Unreported Number of Novel Coronavirus (2019-nCoV) Cases in China in the First Half of January 2020: A Data-Driven Modelling Analysis of the Early Outbreak\\n\\nhttps://doi.org/10.3390/jcm9020388\\n\\nSHA: bf20dda99538a594eafc258553634fd9195104cb\\n\\nAuthors: Zhao, Shi; Musa, Salihu S.; Lin, Qianying; Ran, Jinjun; Yang, Guangpu; Wang, Weiming; Lou, Yijun; Yang, Lin; Gao, Daozhou; He, Daihai; Wang, Maggie H.\\nDate: 2020\\nDOI: 10.3390/jcm9020388\\nLicense: cc-by\\n\\nAbstract: Background: In December 2019, an outbreak of respiratory illness caused by a novel coronavirus (2019-nCoV) emerged in Wuhan, China and has swiftly spread to other parts of China and a number of foreign countries. The 2019-nCoV cases might have been under-reported roughly from 1 to 15 January 2020, and thus we estimated the number of unreported cases and the basic reproduction number, R0, of 2019-nCoV. Methods: We modelled the epidemic curve of 2019-nCoV cases, in mainland China from 1 December 2019 to 24 January 2020 through the exponential growth. The number of unreported cases was determined by the maximum likelihood estimation. We used the serial intervals (SI) of infection caused by two other well-known coronaviruses (CoV), Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS) CoVs, as approximations of the unknown SI for 2019-nCoV to estimate R0. Results: We confirmed that the initial growth phase followed an exponential growth pattern. The under-reporting was likely to have resulted in 469 (95% CI: 403&minus;540) unreported cases from 1 to 15 January 2020. The reporting rate after 17 January 2020 was likely to have increased 21-fold (95% CI: 18&minus;25) in comparison to the situation from 1 to 17 January 2020 on average. We estimated the R0 of 2019-nCoV at 2.56 (95% CI: 2.49&minus;2.63). Conclusion: The under-reporting was likely to have occurred during the first half of January 2020 and should be considered in future investigation.\\n\\nText: A novel coronavirus (2019-nCoV) infected pneumonia infection, which is deadly [1] , was first identified in Wuhan, China in December 2019 [2] . The virus causes a range of symptoms including fever, cough, and shortness of breath [3] . The cumulative number of reported cases slowly increased to cumulative 41 cases by 1 January 2020, and rapidly increased after 16 January 2020. As of 26 January 2020, the still ongoing outbreak had resulted in 2066 (618 of them are in Wuhan) confirmed cases and 56 (45 of them were in Wuhan) deaths in mainland China [4] , and sporadic cases exported from Wuhan were reported in Thailand, Japan, Republic of Korea, Hong Kong, Taiwan, Australia, and the United States, please see the World Health Organization (WHO) news release via https://www.who.int/csr/don/en/ from 14 to 21 January 2020. Using the number of cases exported from Wuhan to other countries, a research group at Imperial College London estimated that there had been 4000 (95%CI: 1000-9700) cases in Wuhan with symptoms onset by 18 January 2020, and the basic reproduction number (R 0 ) was estimated at 2.6 (95%CI: 1.5-3.5) [5] . Leung et al. drew a similar conclusion and estimated the number of cases exported from Wuhan to other major cities in China [6] , and the potentials of travel related risks of disease spreading was also indicated by [7] .\\n\\nDue to an unknown reason, the cumulative number of cases remained at 41 from 1 to 15 January 2020 according to the official report, i.e., no new case was reported during these 15 days, which appears inconsistent with the following rapid growth of the epidemic curve since 16 January 2020. We suspect that the 2019-nCoV cases were under-reported roughly from 1 to 15 January 2020. In this study, we estimated the number of unreported cases and the basic reproduction number, R 0 , of 2019-nCoV in Wuhan from 1 to 15 January 2020 based on the limited data in the early outbreak.\\n\\nThe time series data of 2019-nCoV cases in mainland China were initially released by the Wuhan Municipal Health Commission from 10 to 20 January 2020 [8] , and later by the National Health Commission of China after 21 January 2020 [9] . The case time series data in December 2019 were obtained from a published study [3] . All cases were laboratory confirmed following the case definition by the national health commission of China [10] . We chose the data up to 24 January 2020 instead of to the present study completion date. Given the lag between timings of case confirmation and news release of new cases, the data of the most recent few days were most likely to be tentative, and thus they were excluded from the analysis to be consistent.\\n\\nWe suspected that there was a number of cases, denoted by ξ, under-reported from 1 to 15 January 2020. The cumulative total number of cases, denoted by C i , of the i-th day since 1 December 2019 is the summation of the cumulative reported, c i , and cumulative unreported cases, Ξ i . We have C i = c i + Ξ i , where c i is observed from the data, and Ξ i is 0 for i before 1 January and ξ for i after 15 January 2020. Following previous studies [11, 12] , we modelled the epidemic curve, i.e., the C i series, as an exponential growing Poisson process. Since the data from 1 to 15 January 2020 appeared constant due to unclear reason(s), we removed these data from the fitting of exponential growth. The ξ and the intrinsic growth rate (γ) of the exponential growth were to be estimated based on the log-likelihood, denoted by , from the Poisson priors. The 95% confidence interval (95% CI) of ξ was estimated by the profile likelihood estimation framework with cutoff threshold determined by a Chi-square quantile [13] , χ 2 pr = 0.95, df = 1 . With γ estimated, the basic reproduction number could be obtained by R 0 = 1/M(−γ) with 100% susceptibility for 2019-nCoV presumed at this early stage. Here, the function M(·) was the Laplace transform, i.e., the moment generating function, of the probability distribution for the serial interval (SI) of the disease [11, 14] , denoted by h(k) and k is the mean SI. Since the transmission chain of 2019-nCoV remained unclear, we adopted the SI information from Severe Acute Respiratory Syndrome (SARS) and Middle East Respiratory Syndrome (MERS), which share the similar pathogen as 2019-nCoV [15] [16] [17] . We modelled h(k) as Gamma distributions with mean of 8.0 days and standard deviation (SD) of 3.6 days by averaging the SI mean and SD of SARS, mean of 7.6 days and SD of 3.4 days [18] , and MERS, mean of 8.4 days and SD of 3.8 days [19] .\\n\\nWe were also interested in inferring the patterns of the daily number of cases, denoted by ε i for the i-th day, and thus it is obviously that C i = C i−1 + ε i . A simulation framework was developed for the iterative Poisson process such that E[\\n\\ndenoted the expectation. The simulation was implemented starting from 1 January 2020 with a cumulative number of cases seed of 40, the same as reported on 31 December 2019. We conducted 1000 samples and calculated the median and 95% CI.\\n\\nThe number of 2019-nCoV unreported cases was estimated at 469 (95% CI: 403-540), see Figure 1a , which was significantly larger than 0. This finding implied the occurrence of under-reporting between 1 and 15 January 2020. After accounting for the effect of under-reporting, the R 0 was estimated at 2.56 (95% CI: 2.49-2.63), see Figure 1b , which is consistent with many existing online preprints with range from 2 to 4 [5, [20] [21] [22] . With the R 0 of 2.56 and ξ of 469, the exponential growing framework fitted the cumulative total number of cases (C i ) remarkably well, see Figure 1c iterative Poisson process such that\\n\\ndenoted the expectation. The simulation was implemented starting from 1 January 2020 with a cumulative number of cases seed of 40, the same as reported on 31 December 2019. We conducted 1000 samples and calculated the median and 95% CI.\\n\\nThe number of 2019-nCoV unreported cases was estimated at 469 (95% CI: 403−540), see Figure  1a , which was significantly larger than 0. This finding implied the occurrence of under-reporting between 1 and 15 January 2020. After accounting for the effect of under-reporting, the R0 was estimated at 2.56 (95% CI: 2.49−2.63), see Figure 1b , which is consistent with many existing online preprints with range from 2 to 4 [5, [20] [21] [22] . With the R0 of 2.56 and ξ of 469, the exponential growing framework fitted the cumulative total number of cases (Ci) remarkably well, see Figure 1c , referring to McFadden's pseudo-R-squared of 0.99. show the exponential growth fitting results of the cumulative number of cases (Ci) and the daily number of cases (εi) respectively. In panels (c) and (d), the gold squares are the reported cases, the blue bold curve represents the median of the fitting results, the dashed blue curves are the 95% CI of the fitting results, and the purple shading area represents the time window from 1 to 15 January 2020. In panel (c), the blue dots are the cumulative total, i.e., reported and unreported, number of cases. In panel (d), the grey curves are the 1000 simulation samples.\\n\\nOur estimation of R0 rely on the SI of 2019-nCoV, which remains unknown as of 26 January 2020. In this work, we employed the SIs of SARS and MERS as approximations to that of 2019-nCoV. The determination of SI requires the knowledge of the chain of disease transmission that needs a sufficient number of patient samples and periods of time for follow-up [23] , and thus this is unlikely to be achieved shortly. However, using SIs of SARS and MERS as approximation could provide an panels (a,b) , the green shading area represents the 95% CI (on the horizontal axis), and the vertical green line represents the maximum likelihood estimate (MLE) of the number of unreported cases. With the MLE of R 0 at 2.56, panels (c,d) show the exponential growth fitting results of the cumulative number of cases (C i ) and the daily number of cases (ε i ) respectively. In panels (c,d), the gold squares are the reported cases, the blue bold curve represents the median of the fitting results, the dashed blue curves are the 95% CI of the fitting results, and the purple shading area represents the time window from 1 to 15 January 2020. In panel (c), the blue dots are the cumulative total, i.e., reported and unreported, number of cases. In panel (d), the grey curves are the 1000 simulation samples.\\n\\nOur estimation of R 0 rely on the SI of 2019-nCoV, which remains unknown as of 26 January 2020. In this work, we employed the SIs of SARS and MERS as approximations to that of 2019-nCoV.\\n\\nThe determination of SI requires the knowledge of the chain of disease transmission that needs a sufficient number of patient samples and periods of time for follow-up [23] , and thus this is unlikely to be achieved shortly. However, using SIs of SARS and MERS as approximation could provide an insight into the transmission potential of 2019-nCoV at the early outbreak. We note that slightly varying the mean and SD of SI would not affect our main conclusions. The R 0 of 2019-nCoV was estimated at 2.56 (95% CI: 2.49-2.63), and it is generally in line with those of SARS, i.e., 2-5 [19, 24, 25] , and MERS, i.e., 2.7-3.9 [26] .\\n\\nFor the simulated daily number of cases (ε i ), see Figure 1d , we found that ε i matched the observed daily number after 17 January 2020, but was significantly larger than the observations from 1 to 17 January 2020. This finding implied that under-reporting was likely to have occurred in the first half of January 2020. We estimated that the reporting rate after 17 January 2020 increased 21-fold (95% CI: [18] [19] [20] [21] [22] [23] [24] [25] compared to the situation from 1 to 17 January 2020 on average. One of the possible reasons was that the official diagnostic protocol was released by WHO on 17 January 2020 [27] , and the diagnosis and reporting efforts of 2019-nCoV infections probably increased. Thereafter, the daily number of newly reported cases started increasing rapidly after 17 January 2020, see Figure 1d . We conducted additional sensitivity analysis by varying the starting date of the under-reporting time window, e.g., 1 January 2020 in the main results, from 2 December 2019 to 3 January 2020, and we report our estimates largely hold. The exact value of the reporting rate was difficult to determine due to lack of serological surveillance data. The reporting rate can be determined if serological surveillance data are available for a population; we would know who was infected (seropositive) and who was not (seronegative), with high confidence. The reporting rate is the ratio of reported cases over the number of seropositive individuals. It was statistically evident that increasing in reporting was likely, and thus it should be considered in the future investigation of this outbreak.\\n\\nPrevious preprint suggested cumulative cases of 1723 (95% CI: 427-4471) as of 12 January 2020, and 4000 (95% CI: 1000-9700) as of 18 January 2020 based on the aggregated international export cases [5] . Our analysis yielded cumulative cases of 280 (95% CI: 128-613) as of 12 January 2020, and 609 (95% CI: 278-1333) as of 18 January 2020 based on the exponential growing mechanistic in the early outbreak. Although our estimate case number appeared to have a lower mean than those estimated by Imai et al. [5] , they are not statistically different. This study applied a different screening effort to detect the 2019-nCoV cases from that in Imai et al. [5] . Imai et al. assumed the average screening effort at overseas airports that covered travelers arriving from Wuhan. Whereas we assumed a constant screening effort applied in Wuhan at the same point of time, and then a number of cases (i.e., ξ) should have been reported yet failed to be reported in the first half of January 2020 due to all sorts of reasons. It is not surprising that different assumptions yielded different results, and this difference in screening effort also partly explained why the detected cases out of China mainly presented mild symptoms. Thus, it was reasonable that our estimates appeared lower than those estimated by Imai et al. [5] . It must be emphasized that such a gap in the knowledge would be resolved by serological survey study (for a large population to approximate the actual positive rate) or an explicit estimation of the actual reporting rate.\\n\\nUnder-reporting was likely to have occurred and resulted in 469 (95% CI: 403-540) unreported cases from 1 to 15 January 2020. The reporting rate after 17 January 2020 was likely to have increased 21-fold (95% CI: 18-25) compared with the situation from 1 to 17 January 2020 on average, and it should be considered in future investigation. We estimated the R 0 at 2019-nCoV to be 2.56 (95% CI: 2.49-2.63).\\n\\nAuthor Contributions: All authors conceived the study, carried out the analysis, discussed the results, drafted the first manuscript. All authors have read and agreed to the published version of the manuscript.\", 'document_id': 2620}]}]}\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>answer</th>\n",
              "      <th>answer_text</th>\n",
              "      <th>answer_start</th>\n",
              "      <th>question</th>\n",
              "      <th>context</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>{'text': 'Mother-to-child transmission (MTCT) ...</td>\n",
              "      <td>Mother-to-child transmission (MTCT) is the mai...</td>\n",
              "      <td>370</td>\n",
              "      <td>What is the main cause of HIV-1 infection in c...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>{'text': 'DC-SIGNR plays a crucial role in MTC...</td>\n",
              "      <td>DC-SIGNR plays a crucial role in MTCT of HIV-1...</td>\n",
              "      <td>2003</td>\n",
              "      <td>What plays the crucial role in the Mother to C...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>{'text': 'more than 400,000 children were infe...</td>\n",
              "      <td>more than 400,000 children were infected world...</td>\n",
              "      <td>2291</td>\n",
              "      <td>How many children were infected by HIV-1 in 20...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>{'text': 'High copy numbers of CCL3L1, a poten...</td>\n",
              "      <td>High copy numbers of CCL3L1, a potent HIV-1 su...</td>\n",
              "      <td>28143</td>\n",
              "      <td>What is the role of C-C Motif Chemokine Ligand...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>{'text': 'Dendritic cell-specific ICAM-grabbin...</td>\n",
              "      <td>Dendritic cell-specific ICAM-grabbing non-inte...</td>\n",
              "      <td>3207</td>\n",
              "      <td>What is DC-GENR and where is  it expressed?</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>5</th>\n",
              "      <td>{'text': 'the presence of DC-SIGNR at the plac...</td>\n",
              "      <td>the presence of DC-SIGNR at the placental endo...</td>\n",
              "      <td>28910</td>\n",
              "      <td>How does the presence of DC-SIGNR affect the M...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>6</th>\n",
              "      <td>{'text': 'in placenta containing low levels of...</td>\n",
              "      <td>in placenta containing low levels of DC-SIGNR,...</td>\n",
              "      <td>29090</td>\n",
              "      <td>Why do low levels of DC-SIGNR enhance Mother t...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>7</th>\n",
              "      <td>{'text': 'Without specific interventions, the ...</td>\n",
              "      <td>Without specific interventions, the rate of HI...</td>\n",
              "      <td>2137</td>\n",
              "      <td>What is the percentage of Mother to Child Tran...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>8</th>\n",
              "      <td>{'text': 'Genetic variants in CCR5 have been s...</td>\n",
              "      <td>Genetic variants in CCR5 have been shown to in...</td>\n",
              "      <td>27719</td>\n",
              "      <td>Does C-C chemokine receptor type 5 (CCR5) affe...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>9</th>\n",
              "      <td>{'text': 'Mannose-binding lectin (MBL) is an i...</td>\n",
              "      <td>Mannose-binding lectin (MBL) is an innate immu...</td>\n",
              "      <td>28335</td>\n",
              "      <td>How does Mannanose Binding Lectin (MBL) affect...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "                                              answer  ...                                            context\n",
              "0  {'text': 'Mother-to-child transmission (MTCT) ...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "1  {'text': 'DC-SIGNR plays a crucial role in MTC...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "2  {'text': 'more than 400,000 children were infe...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "3  {'text': 'High copy numbers of CCL3L1, a poten...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "4  {'text': 'Dendritic cell-specific ICAM-grabbin...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "5  {'text': 'the presence of DC-SIGNR at the plac...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "6  {'text': 'in placenta containing low levels of...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "7  {'text': 'Without specific interventions, the ...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "8  {'text': 'Genetic variants in CCR5 have been s...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "9  {'text': 'Mannose-binding lectin (MBL) is an i...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "\n",
              "[10 rows x 5 columns]"
            ]
          },
          "metadata": {},
          "execution_count": 4
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "TzPgf_0kR7jM",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 581
        },
        "outputId": "3ffd2ca8-bee9-47c3-9fd6-c07faff966c2"
      },
      "source": [
        "res = json_to_df(data)\n",
        "res.to_csv(\"input.csv\")\n",
        "res"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>answer</th>\n",
              "      <th>answer_text</th>\n",
              "      <th>answer_start</th>\n",
              "      <th>question</th>\n",
              "      <th>context</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>{'text': 'Mother-to-child transmission (MTCT) ...</td>\n",
              "      <td>Mother-to-child transmission (MTCT) is the mai...</td>\n",
              "      <td>370</td>\n",
              "      <td>What is the main cause of HIV-1 infection in c...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>{'text': 'DC-SIGNR plays a crucial role in MTC...</td>\n",
              "      <td>DC-SIGNR plays a crucial role in MTCT of HIV-1...</td>\n",
              "      <td>2003</td>\n",
              "      <td>What plays the crucial role in the Mother to C...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>{'text': 'more than 400,000 children were infe...</td>\n",
              "      <td>more than 400,000 children were infected world...</td>\n",
              "      <td>2291</td>\n",
              "      <td>How many children were infected by HIV-1 in 20...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>{'text': 'High copy numbers of CCL3L1, a poten...</td>\n",
              "      <td>High copy numbers of CCL3L1, a potent HIV-1 su...</td>\n",
              "      <td>28143</td>\n",
              "      <td>What is the role of C-C Motif Chemokine Ligand...</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>{'text': 'Dendritic cell-specific ICAM-grabbin...</td>\n",
              "      <td>Dendritic cell-specific ICAM-grabbing non-inte...</td>\n",
              "      <td>3207</td>\n",
              "      <td>What is DC-GENR and where is  it expressed?</td>\n",
              "      <td>Functional Genetic Variants in DC-SIGNR Are As...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>783</th>\n",
              "      <td>{'text': 'at there had been 4000 (95%CI: 1000-...</td>\n",
              "      <td>at there had been 4000 (95%CI: 1000-9700) case...</td>\n",
              "      <td>2927</td>\n",
              "      <td>What was the result  of the Imperial College e...</td>\n",
              "      <td>Estimating the Unreported Number of Novel Coro...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>784</th>\n",
              "      <td>{'text': 'y released by the Wuhan Municipal ',...</td>\n",
              "      <td>y released by the Wuhan Municipal</td>\n",
              "      <td>3981</td>\n",
              "      <td>Who release the time series data from 10th  to...</td>\n",
              "      <td>Estimating the Unreported Number of Novel Coro...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>785</th>\n",
              "      <td>{'text': ', and later by the National Health C...</td>\n",
              "      <td>, and later by the National Health C</td>\n",
              "      <td>4064</td>\n",
              "      <td>Who released the time series data from after 2...</td>\n",
              "      <td>Estimating the Unreported Number of Novel Coro...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>786</th>\n",
              "      <td>{'text': ', the C i series, as an exponential ...</td>\n",
              "      <td>, the C i series, as an exponential growi</td>\n",
              "      <td>5150</td>\n",
              "      <td>How was the epidemic curve modelled?</td>\n",
              "      <td>Estimating the Unreported Number of Novel Coro...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>787</th>\n",
              "      <td>{'text': ', the C i series, as an exponential ...</td>\n",
              "      <td>, the C i series, as an exponential growin</td>\n",
              "      <td>5150</td>\n",
              "      <td>How was the epidemic curve modeled?</td>\n",
              "      <td>Estimating the Unreported Number of Novel Coro...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>788 rows × 5 columns</p>\n",
              "</div>"
            ],
            "text/plain": [
              "                                                answer  ...                                            context\n",
              "0    {'text': 'Mother-to-child transmission (MTCT) ...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "1    {'text': 'DC-SIGNR plays a crucial role in MTC...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "2    {'text': 'more than 400,000 children were infe...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "3    {'text': 'High copy numbers of CCL3L1, a poten...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "4    {'text': 'Dendritic cell-specific ICAM-grabbin...  ...  Functional Genetic Variants in DC-SIGNR Are As...\n",
              "..                                                 ...  ...                                                ...\n",
              "783  {'text': 'at there had been 4000 (95%CI: 1000-...  ...  Estimating the Unreported Number of Novel Coro...\n",
              "784  {'text': 'y released by the Wuhan Municipal ',...  ...  Estimating the Unreported Number of Novel Coro...\n",
              "785  {'text': ', and later by the National Health C...  ...  Estimating the Unreported Number of Novel Coro...\n",
              "786  {'text': ', the C i series, as an exponential ...  ...  Estimating the Unreported Number of Novel Coro...\n",
              "787  {'text': ', the C i series, as an exponential ...  ...  Estimating the Unreported Number of Novel Coro...\n",
              "\n",
              "[788 rows x 5 columns]"
            ]
          },
          "metadata": {},
          "execution_count": 5
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "G_ov8VUp2LKM"
      },
      "source": [
        "def df_to_bert_json(json_file):\n",
        "  bert_json = []\n",
        "  for current_sub in json_file['data']:\n",
        "    for current_cntxt in current_sub['paragraphs']:\n",
        "      data = {}\n",
        "      data[\"context\"] = current_cntxt['context']\n",
        "      for current_ques in current_cntxt['qas']:\n",
        "          ques = current_ques['question']\n",
        "          id = current_ques[\"id\"]\n",
        "          if(len(ques)>2):\n",
        "              is_impossible = current_ques['is_impossible']\n",
        "              if(is_impossible == False):\n",
        "                  for ans in current_ques['answers']:\n",
        "                      answer_text = ans['text']\n",
        "                      answer_start = ans['answer_start']\n",
        "                      data[\"qas\"] = [{\"answers\": [{\"answer_start\": answer_start, \"text\": answer_text}],\n",
        "                                      \"id\": id,\n",
        "                                      \"is_possible\": False,\n",
        "                                      \"question\": ques}]\n",
        "      bert_json.append(data)\n",
        "  return bert_json"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "t7wNKTIlEjPj"
      },
      "source": [
        "with open('data_bert.json', 'w') as f:\n",
        "    json.dump(df_to_bert_json(data), f)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "iSw6MeRNIeH5"
      },
      "source": [
        "**Preprocessing**"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "JIhgD1SMIoVS"
      },
      "source": [
        "*Text Lower Case* "
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "hkU_qCg7Ini0"
      },
      "source": [
        "def text_lowercase(text):\n",
        "    return text.lower()"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "amyUNhG3I0gd"
      },
      "source": [
        "*Remove numbers*"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "KL8cxMOvImzK"
      },
      "source": [
        "def remove_numbers(text):\n",
        "    result = re.sub(r'\\d+', '', text)\n",
        "    return result"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "SPsG4Hi9I9sk"
      },
      "source": [
        "*We can also convert the numbers into words. This can be done by using the inflect library.*"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ivpA5gbSI53v"
      },
      "source": [
        "p = inflect.engine()\n",
        "  \n",
        "# convert number into words\n",
        "def convert_number(text):\n",
        "    # split string into list of words\n",
        "    temp_str = text.split()\n",
        "    # initialise empty list\n",
        "    new_string = []\n",
        "  \n",
        "    for word in temp_str:\n",
        "        # if word is a digit, convert the digit\n",
        "        # to numbers and append into the new_string list\n",
        "        if word.isdigit():\n",
        "            temp = p.number_to_words(word)\n",
        "            new_string.append(temp)\n",
        "  \n",
        "        # append the word as it is\n",
        "        else:\n",
        "            new_string.append(word)\n",
        "  \n",
        "    # join the words of new_string to form a string\n",
        "    temp_str = ' '.join(new_string)\n",
        "    return temp_str\n"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "8blZvmfgJCTd"
      },
      "source": [
        "*Remove punctuation*"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "emy6vKhFJGqi"
      },
      "source": [
        "# remove punctuation\n",
        "def remove_punctuation(text):\n",
        "    translator = str.maketrans('', '', string.punctuation)\n",
        "    return text.translate(translator)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "D7zMYAXnJKmg"
      },
      "source": [
        "*Remove whitespaces*"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "1Om9N59JJI5m"
      },
      "source": [
        "# remove whitespace from text\n",
        "def remove_whitespace(text):\n",
        "    return  \" \".join(text.split())"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "4lJ0wFR7JOMz"
      },
      "source": [
        "*Remove default stopwords*"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "DDpzcjWDJRh8"
      },
      "source": [
        "# remove stopwords function\n",
        "def remove_stopwords(text):\n",
        "    stop_words = set(stopwords.words(\"english\"))\n",
        "    word_tokens = word_tokenize(text)\n",
        "    filtered_text = [word for word in word_tokens if word not in stop_words]\n",
        "    return filtered_text"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "pgdgCYJKJWTR"
      },
      "source": [
        "*Stemming:*\n",
        "Stemming is the process of getting the root form of a word. Stem or root is the part to which inflectional affixes (-ed, -ize, -de, -s, etc.) are added. The stem of a word is created by removing the prefix or suffix of a word. So, stemming a word may not result in actual words."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "0ly1Y7GUJUAb"
      },
      "source": [
        "stemmer = PorterStemmer()\n",
        "  \n",
        "# stem words in the list of tokenised words\n",
        "def stem_words(text):\n",
        "    word_tokens = word_tokenize(text)\n",
        "    stems = [stemmer.stem(word) for word in word_tokens]\n",
        "    return stems\n",
        "  "
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "j3XF35m7Jhq7"
      },
      "source": [
        "*Lemmatization*:\n",
        "Like stemming, lemmatization also converts a word to its root form. The only difference is that lemmatization ensures that the root word belongs to the language. We will get valid words if we use lemmatization. In NLTK, we use the WordNetLemmatizer to get the lemmas of words. We also need to provide a context for the lemmatization. "
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "-Cnmt4DxJhUu"
      },
      "source": [
        "lemmatizer = WordNetLemmatizer()\n",
        "# lemmatize string\n",
        "def lemmatize_word(word_tokens):\n",
        "    # provide context i.e. part-of-speech\n",
        "    lemmas = [lemmatizer.lemmatize(word, pos ='v') for word in word_tokens]\n",
        "    return lemmas"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "FEqRARsiJoLH"
      },
      "source": [
        "# Query Analysis\n",
        "The question analysis phase consists of three steps: \n",
        "1. Question classification. \n",
        "2. Tokenization \n",
        "3. Identification of Question Focus. "
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "6--lSxKcQ2he",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "e879caae-c169-4e09-e887-f36faa3b9a95"
      },
      "source": [
        "input = \"How is COVID-19 spread?\"\n",
        "output = text_lowercase(input)\n",
        "print(output)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "how is covid-19 spread?\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "WIVERZa8SyPk",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "032b425a-de73-46b8-b5d1-dbe95ac473ad"
      },
      "source": [
        "output = remove_punctuation(output)\n",
        "print(output)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "how is covid19 spread\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "xS4sGgoUTJj3",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4b48c126-2de6-408c-e7ed-e8580ed5f581"
      },
      "source": [
        "output = remove_whitespace(output)\n",
        "print(output)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "how is covid19 spread\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "nnPRoiB_TPoP",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "94c087a0-dbeb-4a16-f8e0-8d84820982be"
      },
      "source": [
        "nltk.download('punkt')\n",
        "output =  remove_stopwords(output)\n",
        "output"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[nltk_data] Downloading package punkt to /root/nltk_data...\n",
            "[nltk_data]   Package punkt is already up-to-date!\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "['covid19', 'spread']"
            ]
          },
          "metadata": {},
          "execution_count": 19
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "WKgUY3IcT5ld",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "cfe186ce-43e4-40b6-92bb-d1720ef9142b"
      },
      "source": [
        "nltk.download('wordnet')\n",
        "output = lemmatize_word(output)\n",
        "output"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[nltk_data] Downloading package wordnet to /root/nltk_data...\n",
            "[nltk_data]   Package wordnet is already up-to-date!\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "['covid19', 'spread']"
            ]
          },
          "metadata": {},
          "execution_count": 20
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "XZpD6lKjRkKR"
      },
      "source": [
        "**POS  Tagging**"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "d_wXCAhcVjcm"
      },
      "source": [
        "*NN (Noun), VB (Verb), JJ (Adjective), RB(Adverb), IN (Preposition), and CC (Conjunction) and so on.*"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "zVSdJeTuE_cR",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "1aa5351d-9b8c-43e1-8ac8-abad26d8ff18"
      },
      "source": [
        "stop_words = set(stopwords.words('english'))\n",
        "tokenized = sent_tokenize(input)\n",
        "print(tokenized)\n",
        "for i in tokenized:\n",
        "      \n",
        "    # Word tokenizers is used to find the words \n",
        "    # and punctuation in a string\n",
        "    wordsList = nltk.word_tokenize(i)\n",
        "  \n",
        "    # removing stop words from wordList\n",
        "    wordsList = [w for w in wordsList if not w in stop_words] \n",
        "  \n",
        "    #  Using a Tagger. Which is part-of-speech \n",
        "    # tagger or POS-tagger. \n",
        "    tagged = nltk.pos_tag(wordsList)\n",
        "  \n",
        "    print(tagged)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['How is COVID-19 spread?']\n",
            "[('How', 'WRB'), ('COVID-19', 'JJ'), ('spread', 'NN'), ('?', '.')]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "OmrUF9h0PMuM"
      },
      "source": [
        "**Spelling Correction**"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Z75yTzQHPsMa"
      },
      "source": [
        "**Lemmatization**"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "cWRK6pvlUxnG"
      },
      "source": [
        "nlp = pipeline('question-answering',model = 'bert-large-cased-whole-word-masking-finetuned-squad')"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "8I6VKLQ1U1OE",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "1fa6964d-39f5-4a00-8e84-20992cea6263"
      },
      "source": [
        "query_sample = \"How to prevent Corona ?\"\n",
        "relevant_sentence = \"When asked why they were wearing masks, several students answered that they were preventing corona\"\n",
        "nlp(question = query_sample, context = relevant_sentence)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'answer': 'wearing masks',\n",
              " 'end': 38,\n",
              " 'score': 0.6434423923492432,\n",
              " 'start': 25}"
            ]
          },
          "metadata": {},
          "execution_count": 23
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "X9C28L3QZc0N"
      },
      "source": [
        "# Python3 program to Check if given words \n",
        "# appear together in a list of sentence\n",
        "  \n",
        "def check(sentence, words):\n",
        "    res = [all([k in s for k in words]) for s in sentence]\n",
        "    return [sentence[i] for i in range(0, len(res)) if res[i]]"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "wwPwaL9NZxJv",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "723368f3-fe18-4102-f0f2-a49e167cc8c5"
      },
      "source": [
        "questions =[]\n",
        "for q in res.question.unique():\n",
        "    questions.append(q)\n",
        "questions"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "['What is the main cause of HIV-1 infection in children?',\n",
              " 'What plays the crucial role in the Mother to Child Transmission of HIV-1 and what increases the risk',\n",
              " 'How many children were infected by HIV-1 in 2008-2009, worldwide?',\n",
              " 'What is the role of C-C Motif Chemokine Ligand 3 Like 1 (CCL3L1) in mother to child transmission of HIV-1?',\n",
              " 'What is DC-GENR and where is  it expressed?',\n",
              " 'How does the presence of DC-SIGNR affect the MTCT of HIV-1?',\n",
              " 'Why do low levels of DC-SIGNR enhance Mother to Child Transmission of HIV-1?',\n",
              " 'What is the percentage of Mother to Child Transmission of HIV-1, when there is no intervention?',\n",
              " 'Does C-C chemokine receptor type 5 (CCR5) affect the transmission of HIV-1?',\n",
              " 'How does Mannanose Binding Lectin (MBL) affect elimination of HIV-1 pathogen?',\n",
              " \"How can CCR5's effect in HIV-1 transmission be reduced?\",\n",
              " 'What is IFITM?',\n",
              " 'How many cysteine residues are contained in the first transmembrane domain of IFITM3?',\n",
              " 'What inhibits S-palmitoylation?',\n",
              " 'What interaction is inhibited by the presence of 2-bromopalmitic acid (2BP)?',\n",
              " 'What is a function associated with IFITM5?',\n",
              " 'What regulates the antiviral activity of IFITM3?',\n",
              " 'What is another name for IFITM5?',\n",
              " 'Why is the expression of IFITM5 not promoted by interferons?',\n",
              " 'What is the amino acid similarity between IFITM5 and the other IFITM proteins?',\n",
              " 'What is the amino acid similarity between IFITM 1, IFITM 2, and IFITM 3?',\n",
              " 'What amino acid might be involved in calcium binding in the C-terminal region of a protein?',\n",
              " 'What is the size of bovine coronavirus?',\n",
              " 'What is the molecular structure of bovine coronavirus?',\n",
              " 'How many nucleotides does bovine coronavirus contain?',\n",
              " 'What is the size of the orf1ab gene in bovine coronavirus?',\n",
              " \"Is the orf1ab gene at the 3' or 5' end of the bovine coronavirus genome?\",\n",
              " 'What is a significant cause of Influenze like illness among healthy adolescents and adults presenting for medical evaluation?',\n",
              " 'What is the most common species of Human Coronavirus among adults?\\n',\n",
              " 'Which Human Coronavirus showed species specific clinical characteristics of its infection?',\n",
              " 'What causes the outbreak of SARS and MERS.',\n",
              " 'What is the case fatality rate of SARS and MERS?',\n",
              " 'What were the common HCOV strains in the 5 year USA study?',\n",
              " 'Which species are more prevalent but less severe?',\n",
              " 'What is required for a Hepatitis B infection in cells?',\n",
              " 'What regulates the broad, but less specific, virus-cell interaction in a hepatitis B infection?',\n",
              " 'Which protein domain of the Hepatitis B envelope is necessary for infection?',\n",
              " 'Where is NTCP located in the body?',\n",
              " 'What does the NTCP protein mediate?',\n",
              " 'Is NTCP sufficient to allow HBV infection?',\n",
              " 'Why is NTCP thought to not be sufficient for HBV infection?',\n",
              " 'What kinds of viruses are Japanese encephalitis virus(JEV), tick-borne encephalitis virus(TBEV), eastern equine encephalitis virus (EEEV), sindbis virus(SV), and dengue virus(DV)?',\n",
              " 'What are the current clinically-available methods to detect encephalitis viral antigens?',\n",
              " 'What methods exist for detecting multiple antigens simultaneously in a one-sample, laboratory test?',\n",
              " \"How many antigens could be detected by Liew's multiplex ELISA test?\",\n",
              " 'What kind of antibodies were used in the ELISA-array assay?',\n",
              " 'How was the ELISA assay validated?',\n",
              " 'What capture antibodies were used in the study?',\n",
              " 'What was the spotting concentration range for the capture antibodies?',\n",
              " 'How was the proper spotting concentration determined?',\n",
              " 'How was cross reaction detection determined?',\n",
              " 'How was the ELISA-array assay validated?',\n",
              " 'In 2010, how many cases of tuberculosis were estimated in China?',\n",
              " 'What is the population of Shandong province?',\n",
              " 'What was the purpose of this study?',\n",
              " 'What was the age range for the people surveyed?',\n",
              " 'How was the survey designed?',\n",
              " 'Was was the sample size?',\n",
              " 'How were the clusters selected?',\n",
              " 'How many people were in a community cluster?',\n",
              " 'Who was excluded from the study?',\n",
              " 'When was the study conducted?',\n",
              " 'Who conducted the study?',\n",
              " 'What medium was used to collect the sputum samples?',\n",
              " 'What was the response rate for the study?',\n",
              " 'What was the average age of a study participant?',\n",
              " 'What was the prevalence rate in Shandong in 2010 for sputum positive cases of tuberculosis?',\n",
              " 'What was the most striking finding of the study regarding tuberculosis patients?',\n",
              " 'How many cases of sputum positive tuberculosis patients had no persistent cough?',\n",
              " 'How many tuberculosis patients in Shandong were over 65 years old?',\n",
              " 'What enzymes have been reported to be linked with severity of infection and various pathological conditions caused by microorganisms?',\n",
              " 'At what temperatures was the assay completed?',\n",
              " 'What criteria sets the guideline for drug-like properties?',\n",
              " 'What could be novel candidates as potent inhibitors of papain like cysteine proteases in resistant microorganisms?',\n",
              " 'What method is useful in administering small molecules for systemic delivery to the body?',\n",
              " 'Why is the nasal mucosa useful in the delivery of small molecules into the body?',\n",
              " 'What are the most common methods of inhaled delivery of medications?',\n",
              " 'What medications have shown good promise to in vivo delivery via dry powder inhalers?',\n",
              " 'How are siRNAs typically delivered for systemic effect?',\n",
              " 'What structures form the human airway?',\n",
              " 'What size of particle has been shown to be most effective in the delivery to the lower airway?',\n",
              " 'What are the essential conditions in siRNA delivery to effectively produce gene silencing in the lungs?',\n",
              " 'How long is the SAIBK gene?',\n",
              " 'How many open reading frames are in the SAIBK gene?',\n",
              " 'What virus has the closest genetic identity with the SAIBK gene?',\n",
              " 'How many surgical masks or respirators have past studies projected will be required for a pandemic in the United States?',\n",
              " 'What is the acronym MERS-CoV?',\n",
              " 'What are the critical factors that determine the effect of an epidemic?',\n",
              " 'When did the World Health Organization (WHO) officially declare the 2019-nCoV epidemic as a Public Health Emergency of International Concern?',\n",
              " 'What influenza virus was identified in China in 2013?',\n",
              " 'What past research has been done on severe, single-wave pandemics?',\n",
              " 'What is a clinical attack rate?',\n",
              " 'What was the clinical attack rate in the 2009 H1N1 pandemic?',\n",
              " 'What is the estimated R0 of COVID-19?',\n",
              " 'How many ventilators have past studies projected will be required for a pandemic in the United States?\\n',\n",
              " 'How is exhaled breath condensate used in viral research?',\n",
              " 'How many patients were i this study?',\n",
              " 'What was the conclusion of this study?',\n",
              " 'How long did the patient breath into the RTube?',\n",
              " 'What followed the reverse transcription step in the analysis?',\n",
              " 'What was the last step in the analysis?',\n",
              " 'What percentage of the patients were between 20 and 30 years old in this study?',\n",
              " 'Why is EBC an attractive method for screening?',\n",
              " 'What is the leading cause of death among children after the age of 1 month?',\n",
              " 'How has the number  of  childhood pneumonia been reduced?',\n",
              " 'What percentage of childhood deaths are due to  pneumonia?',\n",
              " 'How has the childhood population grown in the last two  decades?',\n",
              " 'What is the reduction in the number of childhood pneumonia cases?',\n",
              " 'How much  is the reduction in the childhood pneumonia deaths?',\n",
              " 'Childhood pneumonia rate for  high income countries vs low and middle income countries.',\n",
              " 'What  percentage of childhood pneumonia deaths occur outside hospital in low and middle  income countries?',\n",
              " 'Case Fatality Rates for Childhood Pneumonia in high income vs low and middle income countries.',\n",
              " 'How can childhood pneumonia affect the subsequent health of a person?',\n",
              " 'What is the increase in the risk of respiratory disease after having childhood pneumonia.',\n",
              " 'Which is the best method to identify pneumonia in a person?',\n",
              " 'What is responsible for the reduction of radiologic pneumonia?',\n",
              " 'What is the percentage reduction in pneumonia cases due to  vaccination?',\n",
              " 'What is the revised WHO definition of Bacterial Pneumonia?',\n",
              " 'What is the reduction in bacterial pneumonia under the revised WHO  definition of bacterial pneumonia?',\n",
              " 'What caused the increase in the  incidence of empyema in children in the recent past?',\n",
              " 'How have the incidence Empyema been reduced?',\n",
              " 'What pneumonia related or chest conditions indicate the need for child radiography?',\n",
              " 'What chest diseases and pneumonia were identified as leading causes prior to  the availability of vaccines?',\n",
              " 'Why has pertussis immunity in infants has decreased in infants?',\n",
              " 'What is the effect of childhood tuberculosis in childhood pneumonia?',\n",
              " 'What are the risk factors in childhood pneumonia?',\n",
              " 'How does air pollution affect the incidence of childhood pneumonia?',\n",
              " 'What is the strongest risk factor for childhood pneumonia?',\n",
              " 'What is the global coverage of influenza and pneumonia vaccines?',\n",
              " 'Is influenza vaccination during pregnancy safe? How long does it protect the child?',\n",
              " 'What is emphyema?',\n",
              " 'How is the term end point consolidation described  with regard to pneumonia diagnosis?',\n",
              " 'What factors make H5N1 a worldwide threat to public health?',\n",
              " 'What are the symptoms of H5N1 infection in humans?',\n",
              " 'Name some medications used to treat influenza.',\n",
              " 'Why have antiretrovirals medications had limited benefit in treating influenza?',\n",
              " 'What is the focus of the current study?',\n",
              " 'What is the result of the current study?',\n",
              " 'How do the polysaccharides in plants effect the immune response?',\n",
              " 'What does this study demonstrate?',\n",
              " 'In this study, how did treatment of EAP after infection affect survival?',\n",
              " 'What viruses have been responsible for most common childhood acute respiratory track infections (ARTI)?',\n",
              " 'Are there any vaccines against to protect against respiratory viral infections?',\n",
              " 'Which type of bacteria are implicated in carrying genes of drug resistance?',\n",
              " 'What may be a likely cause of sink-to-sink spreading of pathogens in the hospital setting?',\n",
              " 'What types of acute respiratory infections can be screened and diagnosed with multiplex PCR?',\n",
              " 'What is the role of antibodies during infection?',\n",
              " 'How can antibodies also create health problems?',\n",
              " 'Which technology  invention produced antibodies that are clones of a unique parent cell?',\n",
              " 'What mechanism is responsible for the creation of diversified repertoire for antibodies?',\n",
              " 'What  developments have been made possible by the study of B-cell repertoire?',\n",
              " 'What motivates the study of the rare B-cells that produce Broadly Neutralizing Antibodies (bnAb)?',\n",
              " 'How has the study of B-cells helped the treatment for Respiratory syncytial virus (RSV)?',\n",
              " 'How are the studies on B-cells helping the development of a universal influenza vaccine?',\n",
              " 'What role B-cell  play in malaria infection and prevention?',\n",
              " 'How can the study of B-cells help in the prevention and treatment of autoimmune diseases?',\n",
              " 'When was the  Middle East Respiratory Syndrome Coronavirus isolated first?',\n",
              " 'What is the Case fatality rate for MERS Coronavirus?',\n",
              " 'How does gender influence MERS-COV infection?',\n",
              " 'Which is the source animal for the MERS-COV?',\n",
              " 'What is the median time until death in MERS-COV?',\n",
              " 'What is the incubation period for MERS-COV?',\n",
              " 'What is the treatment for MERS-COV?',\n",
              " 'What age group had the most MERS-COV infections?',\n",
              " 'Why are nucleosides analogs used for chemotheraphy?',\n",
              " 'What nucleoside analog is the focus of the current study?',\n",
              " 'Gemcitabine has been shown to have antiviral activity against which viruses?',\n",
              " 'How does gemcitabine disrupt viral activity?',\n",
              " 'Why are cotton rats considered a strong animal model for biomedical research?',\n",
              " 'What is the structure of the CD40 Ligand?',\n",
              " 'What is the effect of CD40L on Dendritic Cells?',\n",
              " 'What is the effect of CD40L on B Cells?',\n",
              " 'What factor may influence viral replication and gene expression?',\n",
              " 'What accounts for the variation of codon usage among open reading frameworks?',\n",
              " 'What age group has the highest rate of severe outcomes?',\n",
              " 'How is COVID-19 spread?',\n",
              " 'How many states in the U.S. have reported cases of COVID-19?',\n",
              " 'When did the White House launch the \"15 Days to Slow the Spread\" program?',\n",
              " 'What should mildly-ill patients do?',\n",
              " 'What type of virus is SARS-CoV-2?',\n",
              " 'What viruses are similar to the COVID-19 coronavirus?',\n",
              " 'What are the phases of a pandemic?',\n",
              " 'At which phase does the peak of the pandemic occur?',\n",
              " 'People with which medical conditions have a higher rate of severe illness?',\n",
              " 'What kind of test can diagnose COVID-19?',\n",
              " 'In what species did the COVID-19 virus likely originate?',\n",
              " 'What risk factors should be considered in addition to clinical symptoms?',\n",
              " 'What is the acronym SARS-CoV-2?',\n",
              " 'When was SARS-CoV-2 first identified?',\n",
              " 'Where was SARS-CoV-2 first identified?',\n",
              " 'What factor positively correlates with imported-and-reported cases counts of SARS-CoV-2 infection?',\n",
              " 'What is the doubling time of the COVID-19 pandemic?',\n",
              " 'When did the first known cases of Middle East respiratory syndrome (MERS) occur?',\n",
              " 'Where did the first known cases of Middle East respiratory syndrome (MERS) occur?',\n",
              " 'Where was the the case first to be publicly reported was from ?',\n",
              " 'In what animals MERS-CoV sequences have been found ?',\n",
              " 'Where is MERS-CoV is enzootic in DC?',\n",
              " 'What does MERS-COV cause?',\n",
              " 'Precisely how does the virus transmit to humans?',\n",
              " 'What appears to be a requirement for transmission?',\n",
              " 'What is a focal point of MERS?',\n",
              " 'What is MERS mostly known as?',\n",
              " 'What does the MERS LRT disease involve?',\n",
              " 'Where else MERS-COV has also been detected?',\n",
              " 'Who suffers severe diseases from MERS?',\n",
              " ' Compared to severe acute respiratory syndrome (SARS)  and another sometimes- fatal zoonotic coronavirus disease, how does MERS affect the patients?',\n",
              " 'To what have most human cases of MERS  been linked?',\n",
              " 'Among whom 20% of the virus detection are reported?',\n",
              " 'What have sero-surveys of MERS virus found?',\n",
              " 'How was the first culture of the new Coronavirus announced?',\n",
              " 'Where was the Email published?',\n",
              " 'When was the Email  published?',\n",
              " 'Whose was the first reported case?',\n",
              " 'How many  viral RNA or virus-specific antibodies been detected?',\n",
              " 'What is the death rate from MERS-COV?',\n",
              " 'What did the discovery process over two to three years reveal?',\n",
              " 'What does subsequent transmission of MERS-CoV to other humans require?',\n",
              " 'What would restrict access to both the virus and to viral diagnostics ?',\n",
              " 'How was the virus then made freely available? ',\n",
              " \"What has epidemiology and research identified the MERS-CoV's cell receptor is?\",\n",
              " 'How does MERS-CoV compare with SARS-CoV?',\n",
              " 'How does MERS-CoV spread among people?',\n",
              " 'Who gets more severe disease from MERS?',\n",
              " 'What is the spread of MERS-CoV among humans, associated with?',\n",
              " 'What do the DCs suffer with MERS-CoV infection?',\n",
              " 'What happens to humans infected by MERS-CoV virus?',\n",
              " 'What is the incubation period of MERS?',\n",
              " 'What is the duration between when illness begins in one person and subsequently spreads to another?',\n",
              " 'What is the median time to death in case of progressive illness?',\n",
              " 'What is the progression  of symptoms to disease?',\n",
              " 'What did the later WHO definition of MERS clearly state?',\n",
              " 'What percentage of cases KSA has been a source of?',\n",
              " 'What is severe MARS noted for?',\n",
              " 'What symptoms appear among the  confirmed cases of MERS?',\n",
              " 'What do patients often present to a hospital with, in cases of MERS?',\n",
              " 'What can  MERS disease progress to?',\n",
              " 'What percentage of all  reported cases has MERS  reportedly killed ?',\n",
              " 'What percentage of of all reported cases has MERS  reportedly killed in KSA?',\n",
              " 'What was mortality in  South Korea from MERS disease?',\n",
              " 'Which are the preferred method for MERS-CoV detection?',\n",
              " 'What  have become a key diagnostic and taxonomic target for CoV species identification?',\n",
              " 'Why  it can be concluded that MERS-COV is a novel and distinct virus?',\n",
              " 'What  indicates the likely presence of infectious virus?',\n",
              " 'What is the sensitivity with which immunochromatographic tool could detect recombinant MERS-CoV nucleocapsid protein?',\n",
              " 'What is the specificity with which immunochromatographic tool could detect recombinant MERS-CoV nucleocapsid protein?',\n",
              " 'What is a different approach for the detection?',\n",
              " 'What is usual in serology testing?',\n",
              " 'How have some sero-assays bypassed the risks of working with infectious virus?',\n",
              " 'What is the  detection of MERS-CoV infection using ELISA or S1 subunit protein microarray [84] is usually followed by?',\n",
              " 'What does the confirmatory process aim to  ensure?',\n",
              " 'When does  generally MERS infection does not trigger a detectable immune response?',\n",
              " 'When does the WHO recommend samlinf from the LRT?',\n",
              " 'What do the recommended samples include?',\n",
              " 'What are recommended when URT sampling is to  be conducted?',\n",
              " 'What paired sera are preferable?',\n",
              " ' When is a single sample  suggested to be sufficient?',\n",
              " 'How long human urine and stool have been found to contain MERS-CoV RNA?',\n",
              " 'What do individual studies report on viral  shedding?',\n",
              " 'What fraction of MERS cases shed viral RNA in their LRT specimens (tracheal aspirates and sputum) ?',\n",
              " 'How long MERS cases shed viral RNA in their LRT specimens (tracheal aspirates and sputum)? ',\n",
              " 'What percentage of contacts were still shedding RNA in their URT specimens?',\n",
              " 'What  samples  returned the highest MERS viral load values?',\n",
              " 'What is NPA?',\n",
              " 'What is  NPA?',\n",
              " 'What is BAL?',\n",
              " 'What is the the proportion of deaths among those infected with MERS-CoV?',\n",
              " 'What   has prevented worldwide spread of MERS-COV?',\n",
              " 'What must be reported to the world organization for animal health as an emerging disease ?',\n",
              " 'What are juvenile DCs more often positive for?',\n",
              " 'What are older DCs are more likely to be positive for?',\n",
              " 'When does the camel calving season?',\n",
              " 'Why is there an increased risk to humans of spill-over during calving season?',\n",
              " 'Which may be an occupational group with significantly higher incidence of seropositivity to MERS-CoV ?',\n",
              " 'How long after Infectious MERS-CoV added to DC, goat or cow milk and stored at 4°C could be recovered?',\n",
              " 'How long after Infectious MERS-CoV added to DC, goat or cow milk and stored at 22°C could be recovered?',\n",
              " 'What ablated MERS-COV infection completely?',\n",
              " 'How long MERS-CoV remained viable at high ambient temperature (30°C) and low RH (30 %) ?',\n",
              " 'How long can pathogenic bacteria remain viable and airborne in a coughed aerosol?',\n",
              " 'How far can pathogenic bacteria spread in a coughed aerosol?',\n",
              " 'What is considered the mechanism of human-to-human transmission of MERS-COV?',\n",
              " 'What is the transmission of MERS-CoV is defined as?',\n",
              " 'What is the  the rate of general transmission, even in outbreaks?',\n",
              " 'What do the majority of human cases of MERS-CoV  seem not to do?',\n",
              " 'What is the basic reproduction number (R 0)?',\n",
              " 'What is the basic reproduction number (R 0 )  for MERS-COV?',\n",
              " 'Why has MERS had a constant presence in the Arabian Peninsula?',\n",
              " 'What was the first known MERS human-to-human transmission event was one characterized by?',\n",
              " 'Where was the first known MERS human-to-human transmission event?',\n",
              " 'Approximately what percentage of MERS cases were fatal in  KSA?',\n",
              " 'Approximately what percentage of MERS cases were died outside KSA?',\n",
              " 'What percentage of HCWs comprised of MERS cases in the KSA and South Korea?',\n",
              " 'How has most of the analysis of MERS-CoV genetics  been performed?',\n",
              " 'What does Clade A contain?',\n",
              " 'What does clade B comprise?',\n",
              " 'How many clades have become apparent in genome of  MERS-COV from humans and DCs?',\n",
              " 'Which city has had has had more MERS cases than any other region of the KSA ?',\n",
              " 'Which city harbours a wide range of MERS-CoV variants ',\n",
              " 'How has the vast majority of MERS-CoV transmission occurred?',\n",
              " 'How were the transmission circumstances created?',\n",
              " 'What percentage of humans have died  among all humans reported to be infected?',\n",
              " 'What would  aid accurate calculation of a case fatality ratio?',\n",
              " 'How does MARS-COV differ from SARS-COV?',\n",
              " 'Is there any evidence that MERS-CoV is a virus of pandemic concern?',\n",
              " 'How did the first WHO case definition  define probable cases of MERS?',\n",
              " 'What is PHEIC?',\n",
              " 'How is PHEIC defined?',\n",
              " 'What platform was instrumental in rapid sharing of COVID-19 information?',\n",
              " \"What's the recommended procedure to disinfect at CT scanner after a COVID-19 exposure?\",\n",
              " \"What's the recommended method to disinfect floors for COVID-19?\",\n",
              " 'What is the role of computed tomography (CT) in COVID-19?',\n",
              " 'What kind of masks are recommended to protect healthcare workers from COVID-19 exposure?',\n",
              " 'What thickness of layers is recommended for CT image reconstruction in COVID-19 assessment?',\n",
              " 'What must the data gathering include?',\n",
              " 'What must be done to assist in recognition of the disease on images and to allow accurate reporting\\nof these findings?\\n',\n",
              " 'What is the major role of chest CT?',\n",
              " 'What is the reason to adopt low-dose CT?',\n",
              " 'What did the EMICT responsibilities include?',\n",
              " 'How were the radiology department areas divided?',\n",
              " 'How was the contaminated area connected to the CT room and other facilities?',\n",
              " 'What does the clean area include?',\n",
              " 'What does the semicontaminated area include?',\n",
              " 'What does the buffer area include?',\n",
              " 'How was the wearing and removing of the equipment performed?',\n",
              " 'What can lower the physical and mental stress levels of staff members?',\n",
              " 'Who must be assigned to the clean area?',\n",
              " 'What responses must EMICT consider once a disease has been identified?',\n",
              " 'What is the conclusion of the report?',\n",
              " 'What were the number of cases in mainland china as of March 11th?',\n",
              " 'How many COVID deaths occurred in Chinese mainland as of March 11th?',\n",
              " 'How many people have come in contact and how many of these are in observation?',\n",
              " 'When was the novel Coronavirus first reported?',\n",
              " 'When did China detect the first human case of H7N9 infection?',\n",
              " 'How many deaths were associated with MERS-CoV as of July 2014?',\n",
              " 'What illness is caused by the 2019-nCOV Coronavirus?',\n",
              " 'In addition to oral swabs, which tests detected the presence of 2019-nCOV virus?',\n",
              " 'What  is the relationship between the presence of virus in blood and anal swabs and disease severity?',\n",
              " 'Which patients were classified as severe in Chinese guidelines?',\n",
              " 'What is the relationship between the presence of virus in blood sample and disease severity?',\n",
              " 'What test could give an indication for special care for 2019-nCOV patients?',\n",
              " 'What is the relationship between the presence of virus in anal swabs and disease severity in 2019-nCOV?',\n",
              " 'What could be the implication of 2019-nCOV virus in anal swabs?',\n",
              " 'What  could account for the high transmission rate of the 2019-nCOV virus?',\n",
              " 'What could account for the dissemination of the 2019-nCOV virus across the whole body?',\n",
              " 'When did the World Health Organization declare the Ebola epidemic in West Africa as a Public Health Emergency of International Concern?',\n",
              " 'What is PPE?',\n",
              " 'Where did the 2014 Ebola epidemic in West Africa spread to?',\n",
              " 'What are the prerequisites for successful emergency preparedness for an epidemic?',\n",
              " 'What is the structure of a coronavirus?',\n",
              " 'What method is developed in this study?',\n",
              " 'What is the model simplified to?',\n",
              " 'What is the estimate of R 0?',\n",
              " 'What is the conclusion of this study?',\n",
              " 'What was the focus of the study?',\n",
              " 'What were the model assumptions?',\n",
              " 'What  compartments were the bats divided into?',\n",
              " 'What compartments were the host animals divided into?',\n",
              " 'What was the SARS-COV-2 reservoir?',\n",
              " 'What were the people divided into?',\n",
              " 'What was the mean incubation period?',\n",
              " 'What was the mean delay from symptom onset to detection/hospitalization of a case?',\n",
              " 'How long after onset, the cases detected in Thailand and Japan were hospitalized?',\n",
              " 'What was the duration from illness onset to first medical visit ?',\n",
              " 'What was the assumption of transmissibility of asymptomatic infection?',\n",
              " 'As of January 17, how many people were tested for body temperature?',\n",
              " 'What is mobile  population in Wuhan?',\n",
              " 'What was  the R0 of SARS?',\n",
              " 'What was the value of R0 in other researches?',\n",
              " 'What is the reported value of R0 for MERS?',\n",
              " 'What was  R0 for  the high transmissibility in  South Korea?',\n",
              " 'What is important for containing the transmission?',\n",
              " 'What did this model show?',\n",
              " 'What was the objective of the study?',\n",
              " 'What are coronaviruses?',\n",
              " 'What animals can carry coronavirus?',\n",
              " 'How many  COVID-19 cases were confirmed on the Diamond Princess cruise ship?',\n",
              " 'What was the time period of peak infection of COVID-19 on  the Diamond Princess cruise ship?',\n",
              " 'With the intervention of movement restrictions starting on 5th February 2020, what were the confirmed cases for COVID-19, were limited  to?',\n",
              " 'Who was the first COVID-19 identified case patient on the Diamond Princess cruise ship?',\n",
              " 'When was  the first  passenger patient on the Diamond Princess cruise ship diagnosed with COVID-19?',\n",
              " 'How many COVID-19 cases were confirmed on the Diamond Princess cruise ship?',\n",
              " 'What is the estimated mean incubation period for COVID-19 infection on the Diamond Princess cruise ship?',\n",
              " 'What was the effect of movement restriction policy on the Diamond Princess cruise ship started on 5th February 2020.',\n",
              " 'What would  have the number of confirmed cases on the Diamond Princess cruise ship,  without a movement restriction starting on the 5th February 2020?',\n",
              " 'When was the a cluster of pneumonia cases were first reported ?',\n",
              " 'What is the number of inbound passengers from China?',\n",
              " 'What percent of inbound passengers from China were from Wuhan?',\n",
              " 'What is the mechanism of action for rupintrivir?',\n",
              " 'Has rupintrivir been shown to reduce the symptoms of a rhinoviral infection?',\n",
              " 'What is the primary etiology of acute respiratory infection?',\n",
              " 'What is RSV?',\n",
              " 'What virus is most commonly associated with acute respiratory infections?',\n",
              " 'What viruses are most frequently associated with acute respiratory infections?',\n",
              " 'What are the clinical characteristics of asthma?',\n",
              " 'What risk factor was associated with hospitalization and death during the 2009 H1N1 influence pandemic?',\n",
              " 'How many surface proteins are on the H1N1 influenza virus?',\n",
              " 'What are the 2 surface proteins on the H1N1 influenza virus?',\n",
              " 'What pharmaceutical targets the NA glycoprotein of the H1N1 influenza virus?',\n",
              " \"What genetic mutation decreases a person's susceptibility to the H1N1 influenza virus?\",\n",
              " 'Why is ribavirin treatment limited?',\n",
              " 'To what the lack of exposure report could be attributed?',\n",
              " 'Why is the determination of asymptomatic or pre-symptomatic transmission, an urgent priority?',\n",
              " 'When does the infectivity of SARS-COV peak?',\n",
              " 'How can the 2019-nCov spread?',\n",
              " 'What is the estimate of the basic reproduction number?',\n",
              " 'What is assumed for the mean serial interval?',\n",
              " 'What  would a shorter mean serial interval mean?',\n",
              " 'What should have reduced the basic reproduction number in January?',\n",
              " 'What  are the delays between infection to illness and illness to laboratory confirmatiion?',\n",
              " 'What is the estimate of number of infections in Wuhan on 25 January 2020?',\n",
              " 'when is viral shedding the highest?',\n",
              " 'How does the transmission of the respiratory virus happen?',\n",
              " 'How do some respiratory viruses spread?',\n",
              " 'What can also play  a role?',\n",
              " 'What can play a role  in the infection of gastrointestinal tract?',\n",
              " 'What was attributed to the spread of SARS-COV at Amoy Gardens?',\n",
              " 'How were the first human infections identified?',\n",
              " 'What do mild clinical presentations of 2019-nCOV indicate?',\n",
              " 'Why is important to  determine the spectrum of clinical manifestations of 2019-nCoV infections?',\n",
              " 'What, beyond the assessment of severity, is important?',\n",
              " 'For whom would the infections be more severe?',\n",
              " 'When is this especially true?',\n",
              " 'What is the current understanding on viral-induced exacerbations?',\n",
              " 'What have evoked new understandings as to the mechanisms of viral exacerbations?',\n",
              " 'What is one of  the major sources of exacerbation of chronic airway inflammatory diseases?',\n",
              " 'What is the focus of this review?',\n",
              " 'What have the authors reviewed?',\n",
              " 'What is summarized?',\n",
              " 'What is highlighted by the authors?',\n",
              " 'What is consolidated in this review?',\n",
              " 'What is this disease  characterized by ?',\n",
              " 'Where can this disease manifest?',\n",
              " 'Why do treatment and management vary in efficacy?',\n",
              " 'What complicates this  further?',\n",
              " 'What are such exacerbations due to?',\n",
              " 'What do the acute exacerbations cause?',\n",
              " 'What are acute exacerbations  usually due to ?',\n",
              " 'What does the immune response elicited by these agents lead to?',\n",
              " 'Among these agents which is a major driver?',\n",
              " 'What is the viral involvement in COPD exacerbation? ',\n",
              " 'What is the reason for the involvement of respiratory viruses in exacerbation?',\n",
              " 'What does the involvement of respiratory viruses contribute to?',\n",
              " 'Why is it  important to identify the exact mechanisms underpinning viral exacerbations in susceptible subjects?',\n",
              " 'What is the lower airway the site of?',\n",
              " 'Where is the the first point of contact with sources of exacerbation',\n",
              " 'What is complied by the authors?',\n",
              " 'Before linking respiratory viruses, what was linked to acute exacerbations?',\n",
              " 'What did the advent of PCR technology lead to?',\n",
              " 'What are the predominant viruses linked to airway inflammatory diseases?',\n",
              " 'What other viruses are implicated in acute exacerbations but to a much lesser extent?',\n",
              " 'What other viruses have been recently reported as contributing to acute exacerbations?',\n",
              " 'What are the common feature of these viruses share?',\n",
              " 'Where do the respiratory viruses primarily infect and replicate?',\n",
              " 'What happens  during the replication process?',\n",
              " 'What does the inflammation lead to in healthy airways?',\n",
              " 'How may the responses be different in a chronically inflamed airway?',\n",
              " 'What will the review focus on?',\n",
              " 'How will this review serve?',\n",
              " 'Why is this approach significant?',\n",
              " 'What happens upon infection?',\n",
              " 'How does the infected airway cell respond?',\n",
              " 'What  does the epithelial proteins cause?',\n",
              " 'What is the effect of these factors?',\n",
              " 'What additional effects are caused  in patients with asthma and patients with CRS with nasal polyp ?',\n",
              " 'What is the result of increased eosinophilia?',\n",
              " 'What are the effects for patients with COPD and patients with CRS without nasal polyp (CRSsNP) ?',\n",
              " ' Which are the type 2 inflammatory cytokines expressed by the epithelial cells upon injury to the epithelial barrier?',\n",
              " 'Which cells lacking both B and T cell receptors but play a crucial role in secreting type 2 cytokines to perpetuate type 2 inflammation when activated ?',\n",
              " 'What is the effect cell death and injury to the epithelial barrier due to infection?',\n",
              " 'What happens when the 3 cytokines are expressed?',\n",
              " 'What  happens in the case of COPD?',\n",
              " 'What happens during viral infection of healthy individuals?',\n",
              " 'What happens upon viral infection in the airway?',\n",
              " 'What is the effect of the inflammation of the airway?',\n",
              " 'What increases the severity of exacerbations in the airway?',\n",
              " 'Why viruses do not need to directly infect the lower airway to cause an acute exacerbation?',\n",
              " 'what is suggested by the fact that  not all viral infections of the airway lead to acute exacerbations?',\n",
              " 'What is the effect of chronic airway inflammatory disease  in patients?',\n",
              " 'What is the effect of chronic airway inflammatory disease?',\n",
              " 'Why should future studies  be performed using metagenomics in addition to PCR analysis ?',\n",
              " 'What is highlighted by the authors in this review?',\n",
              " 'Who has impaired or reduced ability of viral clearance ?',\n",
              " 'What does their impairment stems from?',\n",
              " 'Where is this  especially evident?',\n",
              " 'What are other  effects?',\n",
              " 'What is the  effect of viral components remaining in the airway?',\n",
              " 'What do these factors do?',\n",
              " 'What is also linked with the chronic inflammation that precedes the malignancies?',\n",
              " 'What should be investigated in the future?',\n",
              " 'What further can viral persistence lead to?',\n",
              " 'What  effect the use of steroids to  suppress inflammation can have?',\n",
              " 'What should be further focus of research?',\n",
              " 'Which viruses may not cause prolonged inflammation due to strong induction of antiviral clearance?',\n",
              " 'What do these infections cause?',\n",
              " 'What do the necroptotic factors such as RIP3 do?',\n",
              " 'What  may the destruction of the epithelial barrier  cause?',\n",
              " 'What may the epithelial destruction cause?',\n",
              " 'What is recommended that patients with chronic airway inflammatory disease?',\n",
              " 'What is another mechanism that viral infections use to drive acute exacerbations?',\n",
              " 'What does infection of respiratory viruses cause?',\n",
              " 'What is an example of this?',\n",
              " 'What are also associated with viral infections and pneumonia development, which may worsen inflammation in the lower airway?',\n",
              " 'What is another area of interest?',\n",
              " 'What is usually linked with the development of chronic airway inflammatory diseases?',\n",
              " 'What follows  in the event of a viral infection such as RV infection?',\n",
              " 'What does the viral infection alter?',\n",
              " 'What is  the destabilization is further compounded by?',\n",
              " 'What does all this gradually lead to?',\n",
              " 'Why do these changes may  result in more severe and frequent acute exacerbations ?',\n",
              " 'How effective are microbiome based  trial therapies?',\n",
              " 'What can viral infections cause?',\n",
              " 'Which is the primary contact/infection site of most respiratory viruses?',\n",
              " 'What does the destruction of epithelial barrier, mucociliary function and cell death of the epithelial cells do?',\n",
              " 'What are viral infections are usually accompanied with?',\n",
              " 'What is the dysregulation of inflammation can be further compounded by?',\n",
              " 'What does  the change in the local airway environment and inflammation promote?',\n",
              " 'What does the the inflammatory environment dispersal of upper airway commensals into the lower airway cause?',\n",
              " 'Which are the most commonly  studied viruses in chronic airway inflammatory diseases?',\n",
              " 'What do the infections such as RSV are shown to do?',\n",
              " 'What does mucus overproduction do?',\n",
              " 'What does the disruption of the ciliary movement during viral infection may cause?',\n",
              " 'What are MicroRNAs(miRNA)?',\n",
              " 'What are  miRNAs found to be induced by?',\n",
              " 'What  were linked to the exacerbation of the airway inflammation disease?',\n",
              " 'Where might such miRNA changes have originated from?',\n",
              " 'What are both IFV and RSV infections shown to do?',\n",
              " 'What are IFV infection shown to do?',\n",
              " 'What happens in in asthmatic epithelium in IFV infection?',\n",
              " 'What do non-coding RNAs present as?',\n",
              " 'What mechanisms, other than miRNA modulation play a role?',\n",
              " 'What have  recent epigenetic studies indicated?',\n",
              " ' What have these studies also shown?',\n",
              " 'What has Spalluto et.al. have shown?',\n",
              " ' What infections such as RV and RSV that weakly induce antiviral responses may result in?',\n",
              " 'What can viral infection result in?',\n",
              " 'What sustains the inflammation  in the airway?',\n",
              " 'What may viral infections of the respiratory epithelium by viruses such as IFV, RV, RSV and HSV do?',\n",
              " 'What can happen  in response to the infection such as neutrophils?',\n",
              " 'In addition to worsening disease symptoms, what do viral-induced exacerbations do?',\n",
              " 'What may studies in natural exacerbations and in viral-challenge models using RNA-sequencing (RNA-seq) or single cell RNA-seq on a range of time-points  provide?',\n",
              " 'What analysis functions may be useful?',\n",
              " 'For what  purpose animal based models aare developed for?',\n",
              " 'What can be used unravel the immune profile of a viral infection in healthy and diseased condition?',\n",
              " 'For what purpose controlled in vivo human infections can be performed for mild viruses?',\n",
              " 'Why may the  mechanisms of exacerbation vary considerably?',\n",
              " 'What clinical condition is caused by Hantaan virus?',\n",
              " 'What is the structure of Hantaan virus?',\n",
              " 'What the animal vector reservoir for Hantaan virus?',\n",
              " 'What diagnostic test is correlated with the severity of HFRS?',\n",
              " 'How is Interferon used in practice?',\n",
              " 'What genotypes are associated with the severity of HFRS?',\n",
              " 'Which IFITM proteins have been shown to possess antiviral activity?',\n",
              " 'What is the hypothesized mechanism by which IFITMs work?',\n",
              " 'What genotype causes truncation of the IFITM3 protein?',\n",
              " 'What can cause a slowing growth in daily reported deaths?',\n",
              " 'When did Italy go into Iockdown?',\n",
              " 'Approximately how many deaths have been averted in Western Europe with current non-pharmaceutical interventions remaining in place until the end of March?',\n",
              " 'What are some non-pharmaceutical interventions?',\n",
              " 'Type of model used to infer the impact non-pharmaceutical interventions?',\n",
              " 'How can a semi-mechanistic Bayesian hierarchical model estimate changes to the reproductive number?',\n",
              " 'What is a key assumption of a semi-mechanistic Bayesian hierarchical model used for coronavirus?',\n",
              " 'What is the time lag between when transmission changes occur and when their impact can be\\nobserved in trends in mortality?',\n",
              " 'What is the key aim of non-pharmaceutical interventions?',\n",
              " 'What happens if the reproduction number is greater then 1?',\n",
              " 'When did China introduce strict movement restrictions and other measures including case isolation and quarantine?',\n",
              " \"What was the result of China's interventions introduced in January?\",\n",
              " 'What are examples of social distancing?',\n",
              " \"What was the estimated effect on China's reproduction number in March based on the intervention introduced in January?\",\n",
              " 'Why is it challenging to estimate the reproduction number?',\n",
              " 'What is an alternative way to estimate the course of an epidemic?',\n",
              " 'What is the estimated attack rate in Italy?',\n",
              " \"As of the end of March what is the proportion of Spain's population to be infected?\",\n",
              " 'Which Western European country is estimated to have the lowest attack rate?',\n",
              " \"What is Austria's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\",\n",
              " \"What is Belgium's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\",\n",
              " \"What is Denmark's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\",\n",
              " \"What is France's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\",\n",
              " \"What is Germany's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\",\n",
              " \"What is Italy's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\",\n",
              " \"What is Norway's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\",\n",
              " \"What is Spain's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\",\n",
              " \"What is Switzerland's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\",\n",
              " \"What is United Kingdom's estimated mean percentage [95% credible interval] of total population infected as of 28th March?\",\n",
              " 'What is the estimated averaged initial reproduction number [95% credible interval] for Western Europe as of 28th March?',\n",
              " 'Why is there high uncertainty in estimating the impact of interventions against the coronavirus?',\n",
              " 'Why is it statistically impossible to determine which individual intervention had the greatest effect on reducing the coronavirus reproduction number?',\n",
              " 'What is estimated to drop immediately after an introduction of a non-pharmaceutical intervention?',\n",
              " 'One way to understand the impact of interventions?',\n",
              " 'What is a lockdown?',\n",
              " 'Why is it hard to know the true incidence of infections number?',\n",
              " 'What is an example of a case-based measure against coronavirus?',\n",
              " 'What is an example of containment phase intervention?',\n",
              " 'What does a public events ban intervention mean?',\n",
              " 'An example of social distancing',\n",
              " 'Example of social distancing',\n",
              " 'What is an infection-to-onset-distribution?',\n",
              " 'What is an onset-to-death distribution?',\n",
              " 'What is the population-averaged infection fatality ratio?',\n",
              " 'What is the time-varying reproduction number a function of?',\n",
              " 'What is an incubation period?',\n",
              " 'What are the descriptive statistics for the onset-to-death for coronavirus?',\n",
              " 'What are the descriptive statistics for the incubation period for coronavirus?',\n",
              " \"What is the estimated infection-to-death distribution's mean for coronavirus?\",\n",
              " 'What is a well known approach to model the true number of infected individuals?',\n",
              " 'What is a way to measure virus transmission?',\n",
              " 'What term describes when a majority of the population has built an immunity to a virus?',\n",
              " 'What is the estimated number of people in Italy infected with coronavirus by March 28th?',\n",
              " 'In 2009 what was the reported H1N1 vaccination rate in China?',\n",
              " 'What is the highest alert level given by the World Health Organization to a pandemic?',\n",
              " 'What does it mean for a pandemic to have a WHO alert level of 6?',\n",
              " 'What was the estimated economic impact in the U.S. from the 2009 SARS pandemic?',\n",
              " 'How is 2019-nCOV transmitted?',\n",
              " 'What are ways in  which 2019-nCOV is transmitted?',\n",
              " 'Is oral swab for detecting 2019-nCOV infection, sufficient?',\n",
              " 'What other tests should be considered for 2019-nCOV epidemiology?',\n",
              " 'What tests should  be done before a 2019-nCOV infected patient is discharged?',\n",
              " 'What is a Hantavirus?',\n",
              " 'What plays a role in innate immunity to Hantavirus infection?',\n",
              " 'What evidence suggests that RIG-I and DDX60 collaborate to exert antiviral effects?',\n",
              " 'What was the major contribution of this study?',\n",
              " 'What symptoms were reported?',\n",
              " 'What did the collected data include?',\n",
              " 'When was COVID  surveillance implemented in European region?',\n",
              " 'As of 21 February, how many cases were reported?',\n",
              " 'Where were the cases that were studied?',\n",
              " 'What was the median case age?',\n",
              " 'How many were male?',\n",
              " 'How many cases were there on 5 March?',\n",
              " 'When did the Chinese authorities share the sequence of a novel coronavirus ?',\n",
              " 'What is the name of the disease caused buy SARS-COV-2?',\n",
              " 'What country does this study exclude?',\n",
              " 'What does the study include?',\n",
              " 'What did the ECDC and WHO regional office ask the countries?',\n",
              " 'What was the overall aim of the surveillance?',\n",
              " 'What were the surveillance objectives?',\n",
              " 'What is the adopted WHO case definition?',\n",
              " 'When was the first reported death in France?',\n",
              " 'What is the presumed incubation period?',\n",
              " 'What were the places of infection?',\n",
              " 'What places were linked to these?',\n",
              " 'How many cases were hospitalised?',\n",
              " 'Why were they hospitalised?',\n",
              " 'What was time from onset to hospitalisation?',\n",
              " 'What was the duration of hospitalisation?',\n",
              " 'Why was this?',\n",
              " 'How many cases reported symptoms at this point?\\n',\n",
              " 'How many cases were asymptomatic?',\n",
              " 'What were the asymptomatic cases tested as?',\n",
              " ' For how many cases Fever reported as the sole symptom?',\n",
              " 'In how many cases the symptoms at diagnosis were consistent with the case definition for acute respiratory infection?',\n",
              " 'How many cases had data on preexisting conditions?',\n",
              " 'How many cases had no pre-existing conditions?',\n",
              " 'What other data on pre-existing conditions were reported?',\n",
              " 'How many reported viral pneumonia?',\n",
              " 'What was the clinical evolution of the hospitalised cases?',\n",
              " 'What happened to three cases who were aged 65 years or over?',\n",
              " 'What happened to the case who died?',\n",
              " 'What was the duration of hospitalisation reported for 16 cases ?',\n",
              " 'How were the assays confirmed?',\n",
              " 'What were the specimen types for 21 cases?',\n",
              " 'As of 5 March 2020, what  are the cases in the WHO European region?',\n",
              " 'What were the two contexts for transmission?',\n",
              " 'What  does the analysis show on the difference between locally acquired cases vs imported cases?',\n",
              " 'What is required for locally acquired cases?',\n",
              " 'What was common to all imported cases?',\n",
              " 'What testing and detection are needed?',\n",
              " 'What did the finding prompt ECDC to do?',\n",
              " 'Why is understanding the infection-severity  critical ?',\n",
              " 'Why are serological tests vital?',\n",
              " 'How can hospital based surveillance help?',\n",
              " 'How can present systems of surveillance be used?',\n",
              " 'How will  this approach used?',\n",
              " 'Why is additional research needed?',\n",
              " 'What growing dysjunction has been witnessed?',\n",
              " 'What is aiming to incorporate pathways to translation at the earliest stages?',\n",
              " \"How much  have  the number of biomedical research publications targeting 'translational' concepts has increased ?\",\n",
              " 'What ways to  solve the issues are outlined?',\n",
              " 'How do these exact processes ultimately restrict viral infectivity?',\n",
              " 'What does the author coin this evolutionary dilemma as?',\n",
              " 'How do many viruses resolve this ?',\n",
              " 'How may this \"Achilles Heel\" be safely targeted?',\n",
              " 'Why  may MMHP-targeting therapies exhibit both robust and broadspectrum antiviral efficacy?',\n",
              " 'What will  achieving this through drug repurposing do?',\n",
              " 'What are also discussed by the author?',\n",
              " 'What does  the author anticipate international efforts will do?',\n",
              " 'What do pathogens do upon infection?',\n",
              " 'What is the flip side ?',\n",
              " 'What do RBPs do?',\n",
              " 'What is included in RBPs?',\n",
              " 'What  do  tristetraprolin and AUF1, do?',\n",
              " 'What do RBPs include?',\n",
              " 'What domembers of the Roquin and Regnase families do?',\n",
              " 'What do the RBPs include?',\n",
              " 'What is the increasingly apparent role of RNA methylation machinery ?',\n",
              " 'Where do these activities take place?',\n",
              " 'What happens to these activities during infection?',\n",
              " ' In this way, what do the mRNA-destabilising RBPs constitute ?',\n",
              " \"What can be  done with the  'brake' on the immune system?\",\n",
              " 'What does the author anticipate that continued efforts will lead to?',\n",
              " 'What is another mRNA under post-transcriptional regulation by Regnase-1 and Roquin?',\n",
              " 'What does Furin encode?',\n",
              " 'What are Furin, along with other PCSK family members implicated in?',\n",
              " 'What do Braun and Sauter review?',\n",
              " 'What dis their recent work reveal?',\n",
              " 'What has the increasing abundance of affordable, sensitive, high-throughput genome sequencing technologies  led to?',\n",
              " 'What was this system used for the first time for?',\n",
              " 'What have decades of basic immunology research  provided ?',\n",
              " 'What has this focus on mammalian defences and pathologies sidelined?',\n",
              " 'What has CRISPR/Cas antiviral immune system of prokaryotes been repurposed as?',\n",
              " 'What is another  case in point?',\n",
              " 'What are the ancient lineage of NCLDVs?\\n',\n",
              " 'What do the recent efforts indicate regarding hundreds of human and avian infectious viruses?',\n",
              " 'What is neo-virology?',\n",
              " 'What is predicted these efforts on neo-virology will unlock?',\n",
              " 'What are the two of the four pillars of the National Innovation and Science Agenda?',\n",
              " \"What do Australia's Medical Research and Innovation Priorities include?\",\n",
              " 'What is essential for these priority outcomes?',\n",
              " 'What is the Japan AMED tasked with?',\n",
              " 'What serious question was raised?',\n",
              " 'What is a recent discovery?',\n",
              " 'Which bat virus have been found to be linked with diseases?',\n",
              " 'What assay played an important role?',\n",
              " 'What was the death toll in the 1918-1919 Spanish Influenza epidemic?',\n",
              " 'How many people were infected during the 1918 Spanish Influenza epidemic?',\n",
              " 'What was the case fatality rate in the 1918 Spanish Influenza epidemic?',\n",
              " 'Are the modern day Influenza viruses related to the 1918 Spanish Influenza virus?',\n",
              " 'Why is the Spanish Influenza virus the Mother of the modern influenza viruses?',\n",
              " 'When was it  determined that the 1918  pandemic was caused by the H1N1 Influenza virus?',\n",
              " 'Did the Spanish Influenza or Swine flu or the H1N1 virus disappear in humans for some time?',\n",
              " 'When did the Swine Flu (Spanish Influenza) virus reappear in humans?',\n",
              " 'What descendant  lineages of the swine flu (Spanish Influenza) virus were identified in 2006?',\n",
              " 'Are the modern descendant influenza viruses as dangerous as the 1918 parent swine flu (Spanish Influenza) H1N1 virus?',\n",
              " 'How dangerous are the modern H1N1 (swine flu) and the H3N2 (Influenza A) viruses compared  to the 1918 H1N1 (swine flu Spanish Influenza)  viruses?',\n",
              " 'Are the descendant H1N1 strains of the 1918 H1N1 swine flu (Spanish Influenza) virus, still prevalent?',\n",
              " 'Is the origin and epidemiology of the 1918 swine flu (Spanish Influenza) known?',\n",
              " 'What is the geographical origin of the H1N1 swine flu ?',\n",
              " 'Is the geographical origin of the 1918 H1N1 swine flu known?',\n",
              " 'What is an unique feature of the 1918 swine flu?',\n",
              " 'What season or  time of the year do the new strains of influenza emerge?',\n",
              " 'Once appeared, when do the influenza like diseases occur in subsequent years?',\n",
              " 'When did the first wave of the H1N1 swine flu (Spanish Influenza) occur?',\n",
              " 'What was the death rate in the first wave of the 1918 swine flu pandemic?',\n",
              " 'When were the second and the third wave of the 1918-1919 swine flu pandemic?',\n",
              " 'What was the primary difference between the first wave and the 2nd and 3rd wave of the 1918-1919 swine flu pandemic?',\n",
              " 'Why the human influenza viruses do not disappear after herd immunity is developed?',\n",
              " 'What are the circumstances that promote the spread of influenza virus?',\n",
              " 'Do seasonal temperatures and humidity explain  the appearance of the three waves of the 1918 swine flu?',\n",
              " 'Which virus samples  from the 1918 swine flu pandemic have been identified?',\n",
              " 'Are viruses in the first and third waves of the 1918 swine flu pandemic same or derived from the virus from the second wave of the swine flu?',\n",
              " 'Was the 1918 swine flu virus novel to humans are was it derived from older viruses?',\n",
              " 'Do avian flu viruses change over long periods?',\n",
              " 'What  is the typical age profile of mortality in Influenza diseases?',\n",
              " 'What was the age profile of mortality in the 1918 swine flu?',\n",
              " 'Which age group was most susceptible to die during the 1918 swine flu pandemic?',\n",
              " 'What was the death rate among children during the 1918 swine flu pandemic?',\n",
              " 'What theory provides partial explanation for the age-specific profile of the death rate in the 1918 swine flu pandemic?',\n",
              " 'Is there a difference in the pathologic feature and course of disease between modern influenza pandemics and the 1918 swine flu pandemic?',\n",
              " 'Could the 1918 swine flu virus been controlled  by modern day drugs or vaccines?',\n",
              " 'Why was there such a high death rate in the 19118 swine flu pandemic?',\n",
              " 'Is the molecular basis of human adaptation of a virus understood?',\n",
              " 'What  was the initial growth phase pattern?',\n",
              " 'What was the result of under-reporting?',\n",
              " 'What is R0?',\n",
              " 'What  is likely increase of the reporting rate after the 17th January 2020?',\n",
              " 'What is the estimated value of R0?',\n",
              " 'What is the likely period of under-reporting?',\n",
              " 'Where and when was 2019-nCOV first identified?',\n",
              " 'What  are some of the symptoms caused by the virus?',\n",
              " 'What was the cumulative number of reported cases by 1 January 2020?',\n",
              " 'As of 26 January 2020, what had the outbreak resulted in?',\n",
              " 'As of 26 January 2020, what countries had sporadic cases?',\n",
              " 'What was the result  of the Imperial College estimation?',\n",
              " 'Who release the time series data from 10th  to 20th January 2020?',\n",
              " 'Who released the time series data from after 21st January 2020?',\n",
              " 'How was the epidemic curve modelled?',\n",
              " 'How was the epidemic curve modeled?']"
            ]
          },
          "metadata": {},
          "execution_count": 25
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "3gK-OYrChGox",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "3bf98f00-5ce8-4c97-9978-2d3c76c942b5"
      },
      "source": [
        "res.shape"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(788, 5)"
            ]
          },
          "metadata": {},
          "execution_count": 26
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "A8i-iZ5-hKWy",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "b46236bd-d606-4e6d-eb87-4c1d363ea388"
      },
      "source": [
        "len(res.question.unique())"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "775"
            ]
          },
          "metadata": {},
          "execution_count": 27
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "1cKlnb8dbyxX",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "7b060e55-4645-49a0-ca9a-4dd4af95cc53"
      },
      "source": [
        "sample_question = res.iloc[400]\n",
        "sample_question"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "answer          {'text': 'morbid obesity', 'answer_start': 11203}\n",
              "answer_text                                        morbid obesity\n",
              "answer_start                                                11203\n",
              "question        What risk factor was associated with hospitali...\n",
              "context         The human viral challenge model: accelerating ...\n",
              "Name: 400, dtype: object"
            ]
          },
          "metadata": {},
          "execution_count": 28
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "uTl0RTUqhFl5"
      },
      "source": [
        "def color_answer(question):\n",
        "    answer_start = question[\"answer_start\"]\n",
        "    answer_text = question[\"answer_text\"]\n",
        "    context = question[\"context\"]\n",
        "    return colored(answer_text,\"green\")"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "xY7juXNUiirM",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4ab0f980-a04d-43c7-ba77-2d0bf8899da9"
      },
      "source": [
        "print(sample_question[\"question\"])\n",
        "print()\n",
        "print(\"Answer:\")\n",
        "print(color_answer(sample_question))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "What risk factor was associated with hospitalization and death during the 2009 H1N1 influence pandemic?\n",
            "\n",
            "Answer:\n",
            "\u001b[32mmorbid obesity\u001b[0m\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "DBksYhJeF7eF"
      },
      "source": [
        "# Train Model\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "lp_3u25UGA68"
      },
      "source": [
        "**We train our word embedding models on the context(the texts to which the question relate)**"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ekgvnrSa1tOF"
      },
      "source": [
        "# Define and initialize Bert model to be used\n",
        "modelname = 'deepset/bert-base-cased-squad2'\n",
        "\n",
        "model = BertForQuestionAnswering.from_pretrained(modelname)\n",
        "tokenizer = AutoTokenizer.from_pretrained(modelname)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Pq3CTiYu6g5x"
      },
      "source": [
        "# Initialize the pipeline with the appropriate tokenizer and model\n",
        "nlp = pipeline('question-answering', model=model, tokenizer=tokenizer)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "VMm7-MsQTZgK"
      },
      "source": [
        "## Test the pretrained model"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "j7b57pgC7_tN",
        "outputId": "49bdcc02-8649-4159-ac6b-5f08f2e1019d"
      },
      "source": [
        "print(\"Context: \", res.context[0])"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Context:  Functional Genetic Variants in DC-SIGNR Are Associated with Mother-to-Child Transmission of HIV-1\n",
            "\n",
            "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752805/\n",
            "\n",
            "Boily-Larouche, Geneviève; Iscache, Anne-Laure; Zijenah, Lynn S.; Humphrey, Jean H.; Mouland, Andrew J.; Ward, Brian J.; Roger, Michel\n",
            "2009-10-07\n",
            "DOI:10.1371/journal.pone.0007211\n",
            "License:cc-by\n",
            "\n",
            "Abstract: BACKGROUND: Mother-to-child transmission (MTCT) is the main cause of HIV-1 infection in children worldwide. Given that the C-type lectin receptor, dendritic cell-specific ICAM-grabbing non-integrin-related (DC-SIGNR, also known as CD209L or liver/lymph node–specific ICAM-grabbing non-integrin (L-SIGN)), can interact with pathogens including HIV-1 and is expressed at the maternal-fetal interface, we hypothesized that it could influence MTCT of HIV-1. METHODS AND FINDINGS: To investigate the potential role of DC-SIGNR in MTCT of HIV-1, we carried out a genetic association study of DC-SIGNR in a well-characterized cohort of 197 HIV-infected mothers and their infants recruited in Harare, Zimbabwe. Infants harbouring two copies of DC-SIGNR H1 and/or H3 haplotypes (H1-H1, H1-H3, H3-H3) had a 3.6-fold increased risk of in utero (IU) (P = 0.013) HIV-1 infection and a 5.7-fold increased risk of intrapartum (IP) (P = 0.025) HIV-1 infection after adjusting for a number of maternal factors. The implicated H1 and H3 haplotypes share two single nucleotide polymorphisms (SNPs) in promoter region (p-198A) and intron 2 (int2-180A) that were associated with increased risk of both IU (P = 0.045 and P = 0.003, respectively) and IP (P = 0.025, for int2-180A) HIV-1 infection. The promoter variant reduced transcriptional activity in vitro. In homozygous H1 infants bearing both the p-198A and int2-180A mutations, we observed a 4-fold decrease in the level of placental DC-SIGNR transcripts, disproportionately affecting the expression of membrane-bound isoforms compared to infant noncarriers (P = 0.011). CONCLUSION: These results suggest that DC-SIGNR plays a crucial role in MTCT of HIV-1 and that impaired placental DC-SIGNR expression increases risk of transmission.\n",
            "\n",
            "Text: Without specific interventions, the rate of HIV-1 mother-tochild transmission (MTCT) is approximately 15-45% [1] . UNAIDS estimates that last year alone, more than 400,000 children were infected worldwide, mostly through MTCT and 90% of them lived in sub-Saharan Africa. In the most heavilyaffected countries, such as Zimbabwe, HIV-1 is responsible for one third of all deaths among children under the age of five. MTCT of HIV-1 can occur during pregnancy (in utero, IU), delivery (intrapartum, IP) or breastfeeding (postpartum, PP). High maternal viral load, low CD4 cells count, vaginal delivery, low gestational age have all been identified as independent factors associated with MTCT of HIV-1 [1] . Although antiretrovirals can reduce MTCT to 2%, limited access to timely diagnostics and drugs in many developing world countries limits the potential impact of this strategy. A better understanding of the mechanisms acting at the maternal-fetal interface is crucial for the design of alternative interventions to antiretroviral therapy for transmission prevention.\n",
            "\n",
            "Dendritic cell-specific ICAM-grabbing non-integrin-related (DC-SIGNR, also known as CD209L or liver/lymph node-specific ICAM-grabbing non-integrin (L-SIGN)) can interact with a plethora of pathogens including HIV-1 and is expressed in placental capillary endothelial cells [2] . DC-SIGNR is organized in three distinct domains, an N-terminal cytoplasmic tail, a repeat region containing seven repeat of 23 amino acids and a C-terminal domain implicated in pathogen binding. Alternative splicing of DC-SIGNR gene leads to the production of a highly diversify isoforms repertoire which includes membrane-bound and soluble isoforms [3] . It has been proposed that interaction between DC-SIGNR and HIV-1 might enhance viral transfer to other susceptible cell types [2] but DC-SIGNR can also internalize and mediate proteasome-dependant degradation of viruses [4] that may differently affect the outcome of infection.\n",
            "\n",
            "Given the presence of DC-SIGNR at the maternal-fetal interface and its interaction with HIV-1, we hypothesized that it could influence MTCT of HIV-1. To investigate the potential role of DC-SIGNR in MTCT of HIV-1, we carried out a genetic association study of DC-SIGNR in a well-characterized cohort of HIV-infected mothers and their infants recruited in Zimbabwe, and identified specific DC-SIGNR variants associated with increased risks of HIV transmission. We further characterized the functional impact of these genetic variants on DC-SIGNR expression and show that they affect both the level and type of DC-SIGNR transcripts produced in the placenta.\n",
            "\n",
            "Samples consisted of stored DNA extracts obtained from 197 mother-child pairs co-enrolled immediately postpartum in the ZVITAMBO Vitamin A supplementation trial (Harare, Zimbabwe) and followed at 6 weeks, and 3-monthly intervals up to 24 months. The ZVITAMBO project was a randomized placebocontrolled clinical trial that enrolled 14,110 mother-child pairs, between November 1997 and January 2000, with the main objective of investigating the impact of immediate postpartum vitamin A supplementation on MTCT of HIV-1. The samples used in the present study were from mother-child pairs randomly assigned to the placebo group of the ZVITAMBO project. Antiretroviral prophylaxis for HIV-1-positive antenatal women was not available in the Harare public-sector during ZVITAMBO patient recruitment. The samples were consecutively drawn from two groups: 97 HIV-1-positive mother/HIV-1-positive child pairs and 100 HIV-1-positive mother/HIV-negative child pairs. Mother's serological status was determined by ELISA and confirmed by Western Blot. Infants were considered to be infected if they were HIV-1 seropositive at 18 months or older and had two or more positive HIV-1-DNA polymerase chain reaction (PCR) results at earlier ages. 100 infants were considered to be uninfected as they were ELISA negative at 18 months or older and had two DNA PCR negative results from samples collected at a younger age. Of the 97 HIV-1-infected infants, 57 were infected IU, 11 were infected IP, and 17 were infected PP as determined by PCR analyses of blood samples collected at birth, 6 weeks, 3 and 6 months of age and according to the following definitions adapted from Bryson and colleagues [5] . Briefly, infants who were DNA PCR positive at birth were infected IU. Infants with negative PCR results from sample obtained at birth but who become positive by 6 weeks of age were infected IP. Infants with negative PCR results at birth and 6 weeks of age but who subsequently became DNA PCR positive were considered to be infected during the PP period. In the analysis comparing the 3 different modes of MTCT, 12 HIV-1-infected infants were excluded because the PCR results were not available at 6 weeks of age. Full methods for recruitment, baseline characteristics collection, laboratory procedures have been described elsewhere [6] .\n",
            "\n",
            "The nucleotide sequence variation of the entire promoter, coding and part of 39-UTR regions of DC-SIGNR gene in the study population was determined previously [7] . Haplotype reconstruction was performed using Bayesian statistical method implemented in PHASE [8] , version 2.1.1, using single nucleotide polymorphism (SNP) with a minimum allele frequency (MAF) of 2%. We applied the algorithm five times, using different randomly generated seeds, and consistent results were obtained across runs ( Figure 1 ). Fifteen haplotype-tagged SNPs (htSNPs) were identified by the HaploBlockFinder software [9] with a MAF $5%. These htSNPs were genotyped in the 197 infants by direct PCR sequencing analysis as we have described previously [7] . The DC-SIGNR exon 4 repeat region genotype was determined by PCR amplification followed by migration in 1.5% agarose gels [10] . DNA sequences in the promoter region were analysed with the TESS interface (http//:www.cbil.upenn.edu/tess) for putative transcription factors binding sites using the TRANSFAC database.\n",
            "\n",
            "Luciferase reporter assays using pGL2-Basic vector were performed in order to investigate the functional effect of mutations on DC-SIGNR promoter activity. Genomic DNA from subjects homozygous for the promoter variants and WT was amplified from nucleotide position 2715 to 21 and cloned between the BglII and HindIII multiple cloning sites in the pGL2-Basic vector which harbours a reporter firefly luciferase gene downstream (Invitrogen Canada inc, Burlington, Canada). All recombinants clones were verified by DNA sequencing. The firefly luciferase test reporter vector was co-transfected at a ratio of 10:1 with the constitutive expressor of Renilla luciferase, phRL-CMV (Promega, Madison, WI, USA). We cultured HeLa cells in 6 wells plates (2610 5 cells) and transfected them the following day using lipofectamine (Invitrogen) according to the manufacturer. Cells were lysed and luciferase assays were performed using 20 mg of protein extract according to the manufacturer (Promega) at 44 h post-transfection. Firefly luciferase activity was normalized to Renilla luciferase activity. 0 mg, 0,5 mg or 1 mg CMV-Tat vector was transfected with LTR-Luc as a positive control in these experiments. We carried out lucierase assays in triplicate in three independent experiments. Results are expressed as mean6 standard error of the mean (S.E.M).\n",
            "\n",
            "First-term placental tissues were obtained from abortions following voluntary interruption of pregnancy at CHUM Hôpital Saint-Luc (Montreal, Canada). Tissues from 3 H1 (associated with MTCT of HIV-1) and 3 H15 (wild-type) homozygous haplotypes were used to analyse possible differences in isoform expression. Total placental RNAs were extracted by MasterPure DNA and RNA Extraction Kit (Epicentre Biotechnologies, Madison, WI, USA) according to the manufacturer. Fragments corresponding to the DC-SIGNR coding region were reversed transcribed (RT) and then amplified by nested PCR with the following primers; RT primers RR, first PCR RF and RR and second PCR RcF and RcR according to Liu and colleagues [11] . 1 mg of total RNA was reverse transcribed with Expand RT (Roche Applied Science, Indianapolis, IN, USA) according to the manufacturer and were PCR-amplified with DNA Platinum Taq Polymerase (Invitrogen). Major PCR products from the second PCR reaction were gel extracted with the Qiagen Gel Extraction Kit (Qiagen Canada inc, Mississauga, ON, Canada) and cloned using the TOPO TA Cloning Kit for sequencing (Invitrogen). For each placenta, 15 different clones were randomly selected and amplified with M13 primers and sequenced with ABI PRISM 3100 capillary automated sequencer (Applied Biosystems, Foster City, CA, USA). Sequences were analysed and aligned with GeneBank reference sequence NM_014257 using Lasergene software (DNA Stars, Madison, WI, USA).\n",
            "\n",
            "Quantitative expression of DC-SIGNR isoforms 1,5 mg of placental RNA was reverse transcribed using 2.5 mM of Oligo dT 20 and Expand RT in 20 ml volume according to the manufacturer (Roche Applied Science). 15 ng of total cDNA in a final volume of 20 ml was used to perform quantitative real-time PCR using Universal Express SYBR GreenER qPCR Supermix (Invitrogen) on a Rotor Gene Realtime Rotary Analyser (Corbett Life Science, Sydney, Australia). Samples from 2 subjects in each group were used because RNA quality of others was not suitable for a qRT-PCR analysis. Amplification of all DC-SIGNR isoforms was performed using an exon 5 specific primer pair (Table S1 ). Membrane-bound isoforms were amplified using primers specific for exon 3, corresponding to the common trans-membrane domain of DC-SIGNR. Primers were targeted to the exon-exon junction and RNA extracts were treated with DNase (Fermantas International inc, Burlington, ON, Canada) to avoid amplification of contaminant DNA. Standard curves (50-500 000 copies per reaction) were generated using serial dilution of a full-length DC-SIGNR or commercial GAPDH (Invitrogen) plasmid DNA. All qPCR reactions had efficiencies ranging from 99% to 100%, even in the presence of 20 ng of non-specific nucleic acids, and therefore could be compared. The copy number of unknown samples was estimated by placing the measured PCR cycle number (crossing threshold) on the standard curve. To correct for differences in both RNA quality and quantity between samples, the expression levels of transcripts were normalised to the reference GAPDH gene transcripts. GAPDH primer sequences were kindly provided by A. Mes-Masson at the CHUM. The results are presented as target gene copy number per 10 5 copies of GAPDH. The ratio of membrane-bound isoforms was calculated as E3/E5. Soluble isoforms were calculated by subtracting membrane-bound from total isoforms. We carried out qPCR assays in triplicate in three independent experiments. Results are expressed as mean6S.E.M.\n",
            "\n",
            "Statistical analysis was performed using the GraphPad PRISM 5.0 for Windows (GraphPad Software inc, San Diego, CA, USA). Differences in baseline characteristics and genotypic frequencies of haplotypes or htSNPs were compared between groups using the x 2 analysis or Fisher's exact test. Logistic regression analysis was used to estimate odds ratios (OR) for each genotype and baseline risk factors. Multiple logistic regression was used to define independent predictors identified as significant in the crude analysis. ORs and 95% confidence interval were calculated with the exact method. Comparisons of continuous variables between groups were assessed with the unpaired two-tailed Student's t test when variables were normally distributed and with the Mann-Whitney U test when otherwise. Differences were considered significant at P,0.05.\n",
            "\n",
            "Written informed consent was obtained from all mothers who participated in the study and the ZVITAMBO trial and the investigation reported in this paper were approved by The \n",
            "\n",
            "We carried out an association study of DC-SIGNR polymorphism in 197 infants born to untreated HIV-1-infected mothers recruited in Harare, Zimbabwe. Among them, 97 infants were HIV-1-infected and 100 infants remained uninfected. Of the 97 HIV-1-infected infants, 57 were infected IU, 11 were infected IP, and 17 were infected PP. Timing of infection was not determined for 12 HIV-1-infected infants. Baseline characteristics of mothers and infants are presented in Table 1 . Maternal age and CD4 cell count, child sex, mode of delivery, duration of membrane rupture and gestational age were similar among all groups. However, maternal viral load .29 000 copies/ml was associated with increased risk in both IU and PP with odds ratios (OR) of 3.64 (95% CI = 1.82-7.31, P = 0.0002) and 4.45 (95% CI = 1.50-13.2, P = 0.0045) for HIV-1 transmission, respectively.\n",
            "\n",
            "Fifteen haplotype-tagged SNPs (htSNPs) corresponding to the 15 major DC-SIGNR haplotypes ( Figure 1 ) described among Zimbabweans [7] were genotyped in our study samples (Tables S2  and S3 ). H1 (31%) and H3 (11%) were the most frequent haplotypes observed (Figure 1 ). Being homozygous for the H1 haplotype was associated with increased risk of both IU (OR: 4.42, P = 0.022) and PP (OR: 7.31, P = 0.016) HIV-1 transmission ( Table 2) . Infants harbouring two copy combinations of H1 and/ or H3 haplotypes (H1-H1, H1-H3 or H3-H3) had increased risk of IU (OR: 3.42, P = 0.007) and IP (OR: 5.71, P = 0.025) but not PP (P = 0.098) HIV-1 infection compared to infant noncarriers ( Table 2 ). The latter associations remained significant after adjustment was made for the maternal viral load for both IU (OR: 3.57, 95% CI = 1.30-9.82, P = 0.013) and IP (OR: 5.71, 95% CI = 1.40-23.3, P = 0.025) HIV-1 transmission. The H1 and H3 haplotypes share a cluster of mutations (p-198A, int2-391C, int2-180A, ex4RPT, int5+7C) ( Figure 1 ). Of these, the p-198A and int2-180A variants were significantly associated with MTCT of HIV-1 (Table S2 ). In the unadjusted regression analysis, homozygous infants for the p-198A and int2-180A variants had increased risk of IU (OR: 2.07 P = 0.045, OR: 3.78, P = 0.003, respectively) and IP (OR: 2.47, P = 0.17, O.R: 5.71, P = 0.025, respectively) HIV-1 infection compared to heterozygote infants or noncarriers (Table 3) . When adjustment was made for maternal factors, only the association with the int2-180A variant remained significant for IU (OR: 3.83, 95% CI = 1.42-10.4, P = 0.008) and IP (O.R: 5.71, 95% CI = 1.40-23.3, P = 0.025) HIV-1 transmission. Thus, infants homozygous for DC-SIGNR variant int2-180A contained in H1 and H3 haplotypes were 4-fold to 6-fold more likely to be infected by HIV-1 during pregnancy or at delivery, respectively.\n",
            "\n",
            "Alternative splicing of the DC-SIGNR gene in the placenta produces both membrane-bound and soluble isoform repertoires [3] . The relative proportion of membrane bound and soluble DC-SIGNR could plausibly influence the susceptibility to HIV-1 infection [11] . We therefore hypothesized that the DC-SIGNR mutations associated with MTCT of HIV-1 would have an impact on both the level of DC-SIGNR expression and in the isoform repertoire produced. We investigated DC-SIGNR transcript expression in first-term placentas obtained after elective abortion.\n",
            "\n",
            "We cloned DC-SIGNR from placental tissues by RT-PCR from 3 homozygous H1 samples containing both the DC-SIGNR p-198AA and int2-180AA variants associated with HIV-1 transmission and 3 homozygous wild-type (WT) (p-198CC, int2-180GG) samples. Fifteen clones per sample were randomly selected for sequencing. As expected, we found an extensive repertoire of DC-SIGNR transcripts in all samples with 9 to 16 different isoforms per individual. A total of 65 distinct transcripts were identified ( Figure S1 ), of which 3 were full-length transcripts. 64 of the sequenced clones contained a total of 69 amino acid substitutions with 3 new C termini and 2 premature stop codons. However, the diversity was mostly attributable to the entire deletion of exon 2 or exon 3 or to variations in the length of the neck region (exon 4) of DC-SIGNR. The deletion of exon 3 eliminates the trans-membrane domain of the protein and leads to the expression of soluble DC-SIGNR isoforms [3] . Interestingly, the abundance of membrane-bound isoforms in placental tissues of the H1 homozygotes appears to be lower than that observed in samples from WT individuals ( Figure S1 ). The deletion of exon 3 was confirmed by sequencing and we hypothesize that the skipping of exon 3, could be due to the presence of the int2-180A mutation observed in infants with the H1 haplotype. In fact, this intron mutation is located 180 bp downstream from exon 3 and potentially modifies splicing events (Figure 2A ). We confirmed that the variation in transcript proportions seen between the two groups was also reflected at the level of mRNA expression in the placenta. To quantify membrane-bound vs soluble isoforms in placental samples from homozygous H1 and WT infants, we amplified the exon 5 (E5) sequence present in all DC-SIGNR isoforms (total transcripts). We then amplified exon 3 (E3) which is deleted in the soluble forms and then calculated the E3:E5 ratio. We found that placental tissues from homozygous H1 infants express a significantly lower proportion of membrane-bound DC-SIGNR (18%) compared to that in WT individuals (36%) (P = 0.004) ( Figure 2B ) suggesting that exon 3 skipping happens more frequently in presence of the DC-SIGNR int2-180A variant associated with MTCT of HIV-1.\n",
            "\n",
            "The DC-SIGNR int2-180A variant is always transmitted with the promoter mutation p-198A (Figure 1 ). In the unadjusted regression analysis, the p-198A variant was significantly associated with IU but not with IP and PP HIV-1 transmission (Table 3) . Computational transcription factor binding site analysis predicts Table 1 . Baseline characteristics of mother and infants risk factors for intrauterine (IU), intrapartum (IP) and postpartum (PP) mother-to-child HIV-1 transmission. Figure 3A ). The luciferase activity of the p-198A variant construct was significantly lower than that of the WT p-198C promoter construct (p-198C/A ratio = 2, P = 0.006) ( Figure 3B ) suggesting that DC-SIGNR p-198A affects promoter activity. The other promoter mutants (p-577C and p-323A) observed in the Zimbabwean population did not affect DC-SIGNR transcription in this assay ( Figure S2 ). To determine the net impact of the DC-SIGNR p-198A mutation on DC-SIGNR expression in the placenta, we quantitated the absolute number of total and membrane-bound DC-SIGNR transcripts in the H1 homozygote and wild-type placental samples as described earlier. The total number of DC-SIGNR transcripts was determined to be 6856213 (DC-SIGNR copies6S.E.M per 10 5 GAPDH copies) in the placental samples from homozygous H1 infants and was 4-fold lower compared to that found in placentas from WT individuals (27816638, P = 0.011) ( Figure 3C ). As suggested earlier, the int2-180A mutation might induce exon 3 skipping leading to a lower production of membrane-bound DC-SIGNR. Although, the decrease in the total number of DC-SIGNR transcripts in H1 homozygous placental samples containing both the p-198AA and int2-180AA variants affected the proportion of membrane-bound and soluble isoforms, the effect of these mutations was more pronounced on the membrane-bound isoforms with an 8-fold decrease (H1 = 117636.2 vs WT = 9906220.6, P = 0.003) compared to a 3-fold decrease in total soluble isoforms (H1 = 5686181.9 vs WT = 19256495.3, P = 0.03) ( Figure 3C ). Therefore, DC-SIGNR p-198A and int2-180A mutations associated with MTCT of HIV-1 significantly decreased the level of total placental DC-SIGNR transcripts, disproportionately affecting the membrane-bound isoform production. Table 3 . Associations between infant DC-SIGNR promoter p-198 and intron 2 (int2)-180 variants and intrauterine (IU), intrapartum (IP) and postpartum (PP) mother-to-child HIV-1 transmission. \n",
            "\n",
            "Our genetic results, supported by expression assay in placenta, suggest the involvement of DC-SIGNR in MTCT of HIV-1. Homozygosity for the haplotype H1 was associated with IU transmission in the unadjusted regression analysis. However, the association disappeared after adjustment was made for the maternal factors presumably because of the small number of H1 homozygote infants analysed in each groups. H1 and H3 were the most frequent haplotypes observed in the study population and they share a cluster of mutations (Figure 1 ). Grouping haplotypes H1 and H3 increased the power of the study and permitted the identification of specific DC-SIGNR mutations associated with MTCT of HIV-1. Indeed, two mutations shared by haplotypes H1 and H3 were associated with vertical transmission of HIV-1. The int2-180A was associated with a 4-fold increased risk of IU and 6fold increased risk of IP after adjustment for the maternal factors. Although the p-198A variant was associated with IU transmission, the association disappeared after adjustment was made for the maternal viral load. Nevertheless, we showed that this mutation reduces DC-SIGNR transcriptional activity in vitro and produces lower level of DC-SIGNR transcripts in placental tissues in combination with the int2-180A variant. Since int2-180A is always transmitted with p-198A on the MTCT associated combined haplotypes H1/H3, whereas p-198A is carried on other nonassociated haplotypes (Figure 1) , we can speculate that the p-198A mutation alone may have a minor effect in vivo whereas in combination with the int2-180A variant, they both act to reduce the level of placental DC-SIGNR expression resulting in an increased risk of MTCT of HIV-1.\n",
            "\n",
            "The majority of IU transmission occurs during the last trimester of pregnancy (reviewed in [12] ). Full-term placenta samples were not available for the current study and the expression assays were performed on first-term placental tissues. A previous study looking at DC-SIGNR placental isoforms repertoire in full-term placenta samples demonstrated similar diversity of DC-SIGNR transcripts as in the first-term placental tissues studied herein [3] . However, since levels of DC-SIGNR expression have never been compared between the different terms of pregnancy, it is not known whether DC-SIGNR expression varies during the course of pregnancy. Nevertheless, it is reasonable to assume that the inter-individual differences in both DC-SIGNR isoform repertoire and transcript levels observed between the H1 and WT homozygous infants would be reflected throughout the pregnancy. To date, most studies have focused on the potential role of DC-SIGNR in trans infection of HIV-1 in vitro [2, 10] . However, the multiple mechanisms involved in trans infection and redundancy among C-type lectin functions make it difficult to determine the actual participation of DC-SIGNR in this mode of infection in vivo [13, 14] . The strong correlation we observed between MTCT of HIV-1 and DC-SIGNR genetic variants producing low levels of DC-SIGNR in the placenta suggested that mechanisms other than DC-SIGNR-mediated trans infection might operate during vertical transmission of HIV-1. For example, DC-SIGNR has also been shown to function as a HIV-1 antigen-capturing receptor [15] . Chan and colleagues recently demonstrated that DC-SIGNR transfected CHO cells diminish SARS-CoV titers by enhanced capture and degradation of the virus in a proteasome-dependent manner [4] . Since endothelial cells express MHC-I and II, degraded viral antigens could then be presented to immune cells to elicit an adaptive immune response [16, 17] . The HIV-1 coreceptor CCR5, but not CD4, is co-expressed with DC-SIGNR on placental and blood-brain barrier (BBB) endothelial cells [18, 19] . HIV-1 gp120 binding to CCR5 receptor on endothelial cells compromises BBB integrity and enhances monocytes adhesion and transmigration across the BBB [20, 21] . It is thus possible that reduced expression of DC-SIGNR, particularly the membranebound isoforms, on placental capillary endothelial cells might favour HIV-1 binding to CCR5 receptor, instead of DC-SIGNR receptor, facilitating the migration of maternal HIV-1-infected cells across the placental barrier resulting in IU transmission of HIV-1.\n",
            "\n",
            "The int2-180A variant contained in the H1 and H3 haplotypes was associated with IP transmission suggesting that DC-SIGNR also affect transmission of HIV-1 during delivery. Little is known about the mechanisms underlying transmission of HIV-1 during delivery. Passage through the birth canal could potentially expose infants through a mucosal portal entry (presumably ophthalmic, skin, or gastrointestinal), whereas placental insult during delivery (physical or inflammatory) may enhance transplacental passage of maternal HIV-1-infected cells into foetal circulation [22, 23] . Such process called microtransfusion has been proposed in regards to the results obtain in a Malawian cohort. Kweik and colleagues found a significant association between levels of maternal DNA in umbilical cord blood and IP transmission of HIV-1 suggesting that passage of maternal infected cells through the placenta is likely to occur during delivery [22] . Thus, in a similar fashion as suggested earlier for IU transmission, the relatively lower level of DC-SIGNR in the placenta of homozygous infants harbouring the int2-180A variant could promote HIV-1 binding to CCR5 receptor on endothelial cells affecting the placental barrier integrity and facilitating the passage of maternal infected cells in foetal circulation during delivery.\n",
            "\n",
            "Beside DC-SIGNR, other HIV-1 receptors are known to influence MTCT of HIV-1 (reviewed in [24] ). Genetic variants in CCR5 have been shown to influence vertical transmission of HIV-1. CCR5 promoter variants resulting in higher expression of the receptor were associated with increased risk of MTCT of HIV-1 among sub-Saharan Africans [25, 26] . The 32-pb deletion polymorphism in CCR5 has be shown to protect from vertical transmission of HIV-1 [27] , but this variant is virtually absent among African populations [28] . High copy numbers of CCL3L1, a potent HIV-1 suppressive ligand for CCR5, are associated with higher chemokine production and lower risk of MTCT of HIV-1 among South African infants [29, 30] . Mannose-binding lectin (MBL) is an innate immune receptor synthesised in the liver and secreted in the bloodstream in response to inflammation signal. MBL promotes pathogen elimination by opsonization and phagocytosis, and reduced expression of MBL resulting from polymorphism in coding and non-coding regions has been associated with an increased risk of MTCT of HIV-1 [31, 32] .\n",
            "\n",
            "In this study, we demonstrate for the first time, the potential functional impact of DC-SIGNR mutations on its expression in the placenta and in vertical transmission of HIV-1. We believe that the presence of DC-SIGNR at the placental endothelial cell surface may protect infants from HIV-1 infection by capturing virus and promoting its degradation/presentation. However, in placenta containing low levels of DC-SIGNR, HIV-1 would preferentially binds CCR5 on endothelial cells resulting in a loss of placental barrier integrity and enhanced passage of maternal HIV-1-infected cells in foetal circulation leading to MTCT of HIV-1. This mechanism may also apply to other vertically-transmitted pathogens known to interact with DC-SIGNR such as HIV-2, hepatitis C and dengue viruses and warrant further investigation. \n",
            "\n",
            "Associations between child DC-SIGNR exon 4 repeated region genotypes and mother-to-child HIV-1 transmission.CI, Confidence interval; N, number; NA; not applicable; OR, odds ratio a P-value as determined by the Chi-square test. b Comparison between genotype and all others. Found at: doi:10.1371/journal.pone.0007211.s003 (0.05 MB DOC) Figure S1 DC-SIGNR transcripts repertoire in placenta. Major RT-PCR products from RNA extract from 3 homozygous H1 and 3 homozygous WT placenta samples were purified, cloned and sequenced. Sequenced were analysed according to NCBI reference sequence NM_014257. CT; cytoplasmic tail, TM; trans-membrane domain; WT; wild-type Found at: doi:10.1371/journal.pone.0007211.s004 (0.11 MB DOC) Figure S2 Effect of DC-SIGNR promoter variant on transcriptional activity in luciferase reporter assay in vitro in transfected HeLa cells. Relative luciferase expression from pGL2-Basic, parental vector without promoter. Expression DC-SIGNR promoter constructs, spanning p-577C variant or p-323A variant were calculated relatively to this value. Data are presented in mean values6S.E.M of three independent experiments performed in triplicate. One-way ANOVA test followed by the Dunnett test for multiple comparison was used to compare the relative luciferase expression of the p-557C and p-323A variant reporters against the wild-type (WT) construct (not significant). 0 mg, 0,5 mg or 1 mg CMV-Tat vector was transfected with LTR-Luc as a positive control in these experiments.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "iFEY9fQv7_fi",
        "outputId": "89f3f68c-ed27-4225-ec7d-acd6b3af0811"
      },
      "source": [
        "print(\"Question: \", res.question[100])"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Question:  What was the last step in the analysis?\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "q85maKOK7YwB",
        "outputId": "09a9f319-8d2d-40ef-f6a9-55572483a271"
      },
      "source": [
        "print(\"Answer: \", res.answer_text[100])"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Answer:  amplification step\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "N0f7f9u16vYQ",
        "outputId": "2ae5d959-4043-4614-e729-d52945f46851"
      },
      "source": [
        "# Test the pretrained model\n",
        "context = res.context[100]\n",
        "nlp({\n",
        "    'question': res.question[100],\n",
        "    'context': context\n",
        "})"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.7/dist-packages/numpy/core/_asarray.py:83: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray\n",
            "  return array(a, dtype, copy=False, order=order)\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'answer': 'analysis of the EBC demonstrated an InfB infection',\n",
              " 'end': 11830,\n",
              " 'score': 0.16873717308044434,\n",
              " 'start': 11780}"
            ]
          },
          "metadata": {},
          "execution_count": 36
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "bRoFeuUNTfaK"
      },
      "source": [
        "## Creating a dataset"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "YQfpr5OAEppY"
      },
      "source": [
        "contexts = list(res.context)\n",
        "questions = list(res.question)\n",
        "answers = list(res.answer)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "jBAP_b4bFBzg",
        "outputId": "a8576a18-67dd-4ac2-ff56-e6253689bd48"
      },
      "source": [
        "print(len(contexts), len(questions), len(answers))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "788 788 788\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Wezlk2xjFErJ",
        "outputId": "0d961f52-182c-4ae0-c527-a6262b329eff"
      },
      "source": [
        "# Split the dataset to train set 80%\n",
        "train_contexts, val_contexts, train_questions, val_questions, train_answers, val_answers = train_test_split(contexts, questions, answers, test_size=.3)\n",
        "\n",
        "print(len(train_contexts), len(train_questions), len(train_answers))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "551 551 551\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ja0gmqA2GBjA"
      },
      "source": [
        "# Add answer_end to the train and validation answers\n",
        "def add_answer_end(answers, contexts):\n",
        "  for answer, context in zip(answers, contexts):\n",
        "    gold_text = answer[\"text\"]\n",
        "    start_idx = answer[\"answer_start\"]\n",
        "    end_idx = start_idx + len(gold_text)\n",
        "\n",
        "    if context[start_idx:end_idx] == gold_text:\n",
        "      answer[\"answer_end\"] = end_idx\n",
        "\n",
        "    else:\n",
        "      for n in [1,2]:\n",
        "        if context[start_idx-n:end_idx-n] == gold_text:\n",
        "          answer[\"answer_start\"] = start_idx - n\n",
        "          answer[\"answer_end\"] = end_idx - n\n",
        "\n",
        "add_answer_end(train_answers, train_contexts)\n",
        "add_answer_end(val_answers, val_contexts)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ioPLh_TnGHJR",
        "outputId": "b8d19c27-ee2a-4714-9830-1c4745fdc419"
      },
      "source": [
        "train_answers"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[{'answer_end': 10300,\n",
              "  'answer_start': 9970,\n",
              "  'text': ' patients infected with 2019-nCoV may harbour the virus in the intestine at the early or late stage of disease. It is also worth to note none of the patients with viremia blood had positive swabs. These patients would likely be considered as 2019-nCoV negative through routine surveillance, and thus pose a threat to other people.'},\n",
              " {'answer_end': 7053,\n",
              "  'answer_start': 6995,\n",
              "  'text': 'respiratory bronchioles, alveolar ducts, and alveolar sacs'},\n",
              " {'answer_end': 1339, 'answer_start': 1324, 'text': '27 January 2020'},\n",
              " {'answer_end': 19426,\n",
              "  'answer_start': 19332,\n",
              "  'text': 'may also cause the virus to linger longer in the airway due to the lack of antiviral clearance'},\n",
              " {'answer_end': 9379,\n",
              "  'answer_start': 9168,\n",
              "  'text': 'to provide further understanding of viral induced exacerbation to identify potential pathways and pathogenesis mechanisms that may be targeted as supplementary care for management and prevention of exacerbation.'},\n",
              " {'answer_end': 8202,\n",
              "  'answer_start': 8154,\n",
              "  'text': 'two reported in Italy and two reported in France'},\n",
              " {'answer_end': 1766,\n",
              "  'answer_start': 1731,\n",
              "  'text': 'rapid spread and high pathogenicity'},\n",
              " {'answer_end': 8380,\n",
              "  'answer_start': 8290,\n",
              "  'text': 'were admitted to intensive care and required respiratory support and one French case died.'},\n",
              " {'answer_end': 7412,\n",
              "  'answer_start': 7307,\n",
              "  'text': 'examination of the types and roles of viruses and antiviral defences that exist throughout our biosphere.'},\n",
              " {'answer_end': 2039, 'answer_start': 2035, 'text': ' 25 '},\n",
              " {'answer_end': 19995,\n",
              "  'answer_start': 19969,\n",
              "  'text': 'via a common sanitary pipe'},\n",
              " {'answer_end': 12333,\n",
              "  'answer_start': 11957,\n",
              "  'text': 'such factors cannot explain the 3 pandemic\\nwaves of 1918-1919, which occurred in the spring-sum-\\nmer, summer—fall, and winter (of the Northern\\nHemisphere), respectively. The ﬁrst 2 waves occurred at a\\ntime of year normally unfavorable to inﬂuenza Virus\\nspread. The second wave caused simultaneous outbreaks\\nin the Northern and Southern Hemispheres from\\nSeptember to November. '},\n",
              " {'answer_end': 10915,\n",
              "  'answer_start': 10860,\n",
              "  'text': 'contaminated, semicontaminated,\\nbuffer, and clean areas'},\n",
              " {'answer_end': 13119,\n",
              "  'answer_start': 13025,\n",
              "  'text': 'induce the expression of IL-25, IL-33 and TSLP, with heighten expression in an inflamed airway'},\n",
              " {'answer_end': 4536,\n",
              "  'answer_start': 4503,\n",
              "  'text': ' lack of awareness or recall bias'},\n",
              " {'answer_end': 3686,\n",
              "  'answer_start': 3618,\n",
              "  'text': 'using cultured viruses and inoculated chicken eggs with patient sera'},\n",
              " {'answer_end': 13724,\n",
              "  'answer_start': 13551,\n",
              "  'text': ' increased ILC2 activation, which retain the capability of differentiating to ILC1, may also further augment the neutrophilic response and further aggravate the exacerbation'},\n",
              " {'answer_end': 12095,\n",
              "  'answer_start': 11677,\n",
              "  'text': 'prior to availability of new conjugate vaccines confirmed S. pneumoniae and H. influenzae type B as the most important bacterial causes of pneumonia, with Staphylococcus aureus and Klebsiella pneumoniae associated with some severe cases. Respiratory syncytial virus was the leading viral cause, identified in 15-40% of pneumonia cases, followed by influenza A and B, parainfluenza, human metapneumovirus and adenovirus'},\n",
              " {'answer_end': 6089,\n",
              "  'answer_start': 5942,\n",
              "  'text': 'The hosts were also divided into four compartments: susceptible hosts (S H ), exposed hosts (E H ), infected hosts (I H ), and removed hosts (R H )'},\n",
              " {'answer_end': 1337,\n",
              "  'answer_start': 1180,\n",
              "  'text': ' around 5–6 days, with an upper limit of around 11-14 days [2,5], and delays from illness onset to laboratory confirmation added a further 10 days on average'},\n",
              " {'answer_end': 1141, 'answer_start': 1124, 'text': '31 December 2019,'},\n",
              " {'answer_end': 21751, 'answer_start': 21748, 'text': '288'},\n",
              " {'answer_end': 15682, 'answer_start': 15674, 'text': ' 2.5-7.2'},\n",
              " {'answer_end': 14703,\n",
              "  'answer_start': 14462,\n",
              "  'text': 'will be used globally to help identify evidence of widespread community transmission and, should the virus spread and containment no longer be deemed feasible, to monitor intensity of disease transmission, trends and its geographical spread.'},\n",
              " {'answer_end': 25773, 'answer_start': 25763, 'text': 'URT sample'},\n",
              " {'answer_end': 22466,\n",
              "  'answer_start': 22369,\n",
              "  'text': 'All typical and atypical imaging features of the disease should be made known to\\nall radiologists'},\n",
              " {'answer_end': 14658, 'answer_start': 14655, 'text': '45%'},\n",
              " {'answer_end': 4715,\n",
              "  'answer_start': 4621,\n",
              "  'text': 'the anti-H5N1 effects of EAP offer an alternative strategy for developing antiinfluenza agents'},\n",
              " {'answer_end': 12109,\n",
              "  'answer_start': 12079,\n",
              "  'text': 'The first open reading frames '},\n",
              " {'answer_end': 3047, 'answer_start': 3026, 'text': '15 years old or above'},\n",
              " {'answer_end': 12687,\n",
              "  'answer_start': 12629,\n",
              "  'text': 'easy to perform and can be conducted in a home environment'},\n",
              " {'answer_end': 2038, 'answer_start': 2034, 'text': '20kb'},\n",
              " {'answer_end': 4303,\n",
              "  'answer_start': 4083,\n",
              "  'text': \"For RSV, stabilized versions of the fusion (F) protein in the pre-fusion conformation have led to insights in the B cell's response to infection and has generated potentially safer and more efficacious vaccine candidates\"},\n",
              " {'answer_end': 4493,\n",
              "  'answer_start': 4447,\n",
              "  'text': 'effect of EAP against H5N1 influenza infection'},\n",
              " {'answer_end': 2719,\n",
              "  'answer_start': 2686,\n",
              "  'text': 'Recent fatality rate (CFR) of 21%'},\n",
              " {'answer_end': 9958,\n",
              "  'answer_start': 9816,\n",
              "  'text': 'that the time from symptom onset to hospitalisation/case isolation was about 3 days longer for locally acquired cases than for imported cases.'},\n",
              " {'answer_end': 30493,\n",
              "  'answer_start': 30438,\n",
              "  'text': 'banning all public events of more than 100 participants'},\n",
              " {'answer_end': 2189,\n",
              "  'answer_start': 2151,\n",
              "  'text': 'ymptoms including fever, cough, and sh'},\n",
              " {'answer_end': 4103,\n",
              "  'answer_start': 3944,\n",
              "  'text': 'viral infection is one of the major drivers of asthma exacerbations accounting for up to 80-90% and 45-80% of exacerbations in children and adults respectively'},\n",
              " {'answer_end': 12509,\n",
              "  'answer_start': 12476,\n",
              "  'text': '9.8% [3.2%-25%] of the population'},\n",
              " {'answer_end': 29347,\n",
              "  'answer_start': 29090,\n",
              "  'text': 'in placenta containing low levels of DC-SIGNR, HIV-1 would preferentially binds CCR5 on endothelial cells resulting in a loss of placental barrier integrity and enhanced passage of maternal HIV-1-infected cells in foetal circulation leading to MTCT of HIV-1'},\n",
              " {'answer_end': 20036,\n",
              "  'answer_start': 19859,\n",
              "  'text': 'viruses that induce strong type 1 inflammation and cell death such as IFV (Yan et al., 2016; Guibas et al., 2018) and certain CoV (including the recently emerged COVID-19 virus)'},\n",
              " {'answer_end': 11398, 'answer_start': 11379, 'text': ' 0.3 million people'},\n",
              " {'answer_end': 3324, 'answer_start': 3314, 'text': '11th March'},\n",
              " {'answer_end': 572,\n",
              "  'answer_start': 524,\n",
              "  'text': 'severe acute respiratory syndrome coronavirus 2 '},\n",
              " {'answer_end': 5057,\n",
              "  'answer_start': 4957,\n",
              "  'text': ' important to identify the exact mechanisms underpinning viral exacerbations in susceptible subjects'},\n",
              " {'answer_end': 4015,\n",
              "  'answer_start': 3981,\n",
              "  'text': 'y released by the Wuhan Municipal '},\n",
              " {'answer_end': 7563, 'answer_start': 7557, 'text': '1-5 μm'},\n",
              " {'answer_end': 15463, 'answer_start': 15455, 'text': '0.8-1.3 '},\n",
              " {'answer_end': 3529,\n",
              "  'answer_start': 3493,\n",
              "  'text': 'isolate at home during their illness'},\n",
              " {'answer_end': 13156, 'answer_start': 13139, 'text': '1.1% [0.40%-3.1%]'},\n",
              " {'answer_end': 3248, 'answer_start': 3238, 'text': 'Intranasal'},\n",
              " {'answer_end': 15621,\n",
              "  'answer_start': 15414,\n",
              "  'text': 'when not all exacerbation events occurred during the viral infection but may also occur well after viral clearance (Kim et al., 2008; Stolz et al., 2019) in particular the late onset of a bacterial infection'},\n",
              " {'answer_end': 5180,\n",
              "  'answer_start': 5110,\n",
              "  'text': 'a conserved proprotein convertase crucial in human health and disease.'},\n",
              " {'answer_end': 17795,\n",
              "  'answer_start': 17683,\n",
              "  'text': ' by creating transfected cells expressing recombinant portions of the MERS-CoV nucleocapsid and spike proteins ['},\n",
              " {'answer_end': 987, 'answer_start': 974, 'text': 'December 2019'},\n",
              " {'answer_end': 7966,\n",
              "  'answer_start': 7726,\n",
              "  'text': 'leads to type 1 inflammatory responses consisting of activation of an antiviral state and infiltration of antiviral effector cells. This eventually results in the resolution of the inflammatory response and clearance of the viral infection '},\n",
              " {'answer_end': 5485,\n",
              "  'answer_start': 5246,\n",
              "  'text': ' in immune regulation, cancer and the entry, maturation or release of a broad array of evolutionarily diverse viruses including human papillomavirus (HPV), influenza (IAV), Ebola (EboV), dengue (DenV) and human immunodeficiency virus (HIV)'},\n",
              " {'answer_end': 9524,\n",
              "  'answer_start': 9427,\n",
              "  'text': 'due to the current scarcity of antiviral drugs for the management of viral-induced exacerbations.'},\n",
              " {'answer_end': 25234,\n",
              "  'answer_start': 25113,\n",
              "  'text': ' long periods of viral shedding, sometimes intermittently and not necessarily linked to the presence of disease symptoms.'},\n",
              " {'answer_end': 34590, 'answer_start': 34574, 'text': 'rs12252 C allele'},\n",
              " {'answer_end': 9568,\n",
              "  'answer_start': 9476,\n",
              "  'text': 'People who have serious chronic medical conditions like:\\nHeart disease\\nDiabetes\\nLung disease'},\n",
              " {'answer_end': 1151,\n",
              "  'answer_start': 1008,\n",
              "  'text': ' the cumulative incidence with and without close contact would have been as large as 1373 (95% CI: 570, 2176) and 766 (95% CI: 587, 946) cases,'},\n",
              " {'answer_end': 5585,\n",
              "  'answer_start': 5519,\n",
              "  'text': 'among older adults, especially males, with pre-existing diseases. '},\n",
              " {'answer_end': 17977,\n",
              "  'answer_start': 17828,\n",
              "  'text': 'antiviral genes such as type I interferons, inflammasome activating factors and cytokines remained activated resulting in prolong airway inflammation'},\n",
              " {'answer_end': 4008,\n",
              "  'answer_start': 3925,\n",
              "  'text': 'median time until death is 11-13 days (range 5-27 days) among severely ill patients'},\n",
              " {'answer_end': 851,\n",
              "  'answer_start': 739,\n",
              "  'text': ' mild upper respiratory tract illness in its camel reservoir and sporadic, but relatively rare human infections.'},\n",
              " {'answer_end': 2753, 'answer_start': 2739, 'text': 'about 5.0 days'},\n",
              " {'answer_end': 2842,\n",
              "  'answer_start': 2772,\n",
              "  'text': 'Pneumonia deaths decreased from 1.8 million in 2000 to 900,000 in 2013'},\n",
              " {'answer_end': 9167,\n",
              "  'answer_start': 8944,\n",
              "  'text': '15 had positive nasopharyngeal swabs, nine had positive throat swabs, three cases had positive sputum, two had a positive nasal swab, one case had a positive nasopharyngeal aspirate and one a positive endotracheal aspirate.'},\n",
              " {'answer_end': 2875,\n",
              "  'answer_start': 2837,\n",
              "  'text': 'estimate the TB prevalence in Shandong'},\n",
              " {'answer_end': 13130, 'answer_start': 13114, 'text': '3.7% [1.3%-9.7%]'},\n",
              " {'answer_end': 3256,\n",
              "  'answer_start': 3115,\n",
              "  'text': ' But in 1977, human H1N1 Viruses\\nsuddenly “reemerged” from a laboratory freezer (9). They\\ncontinue to circulate endemically and epidemically.'},\n",
              " {'answer_end': 3630,\n",
              "  'answer_start': 3501,\n",
              "  'text': 'the presence of environmental factors such as allergens, pollutants, smoke, cold or dry air and pathogenic microbes in the airway'},\n",
              " {'answer_end': 8269,\n",
              "  'answer_start': 8133,\n",
              "  'text': 'its impact among older men with comorbid diseases including diabetes mellitus, cirrhosis and various lung, renal and cardiac conditions '},\n",
              " {'answer_end': 14379,\n",
              "  'answer_start': 14313,\n",
              "  'text': 'a large proportion of TB patients did not present consistent cough'},\n",
              " {'answer_end': 5723,\n",
              "  'answer_start': 5692,\n",
              "  'text': ' through fine particle aerosols'},\n",
              " {'answer_end': 6621,\n",
              "  'answer_start': 6409,\n",
              "  'text': 'Historical and epidemiologic data are inade-\\nquate to identify the geographic origin of the Virus (21),\\nand recent phylogenetic analysis of the 1918 Viral genome\\ndoes not place the Virus in any geographic context'},\n",
              " {'answer_end': 43383, 'answer_start': 43372, 'text': ' up to 48 h'},\n",
              " {'answer_end': 10664,\n",
              "  'answer_start': 10334,\n",
              "  'text': 'chest radiography should not be routinely performed in children with ambulatory pneumonia [31] [32] [33] . Indications for chest radiography include hospitalization, severe hypoxemia or respiratory distress, failed initial antibiotic therapy, or suspicion for other diseases (tuberculosis, inhaled foreign body) or complications. '},\n",
              " {'answer_end': 23218,\n",
              "  'answer_start': 23184,\n",
              "  'text': '12 to 26 days after symptom onset '},\n",
              " {'answer_end': 14156,\n",
              "  'answer_start': 14082,\n",
              "  'text': 'pandemic Virus samples we have\\nyet identiﬁed are from second-wave patients'},\n",
              " {'answer_end': 58277, 'answer_start': 58271, 'text': 'Riyadh'},\n",
              " {'answer_end': 3537, 'answer_start': 3532, 'text': '52500'},\n",
              " {'answer_end': 184,\n",
              "  'answer_start': 157,\n",
              "  'text': '50 million deaths worldwide'},\n",
              " {'answer_end': 22819,\n",
              "  'answer_start': 22630,\n",
              "  'text': ' the expression of vasodilating factors and fluid homeostatic factors such as angiopoietin-like 4 (ANGPTL4) and bactericidal/permeabilityincreasing fold-containing family member A1 (BPIFA1)'},\n",
              " {'answer_end': 6051, 'answer_start': 6040, 'text': 'gemcitabine'},\n",
              " {'answer_end': 3816,\n",
              "  'answer_start': 3589,\n",
              "  'text': 'discovery of broadly neutralizing antibodies (bnAbs) that protect against infection across diverse viral isolates has intensified efforts to understand the developmental pathway of the rare B cells that produce these antibodies'},\n",
              " {'answer_end': 23017,\n",
              "  'answer_start': 22962,\n",
              "  'text': '1-substituted pyridylimidazo[1,5-a]pyridine derivatives'},\n",
              " {'answer_end': 29081,\n",
              "  'answer_start': 28910,\n",
              "  'text': 'the presence of DC-SIGNR at the placental endothelial cell surface may protect infants from HIV-1 infection by capturing virus and promoting its degradation/presentation. '},\n",
              " {'answer_end': 25753,\n",
              "  'answer_start': 25728,\n",
              "  'text': ' nasopharyngeal aspirates'},\n",
              " {'answer_end': 13984,\n",
              "  'answer_start': 13760,\n",
              "  'text': 'Because pertussis immunity after acellular pertussis vaccination is less long-lasting than immunity after wild-type infection or whole-cell vaccination, many women of child-bearing age have waning pertussis antibody levels. '},\n",
              " {'answer_end': 5441,\n",
              "  'answer_start': 5390,\n",
              "  'text': 'From intermittent animal-to-human spill-over events'},\n",
              " {'answer_end': 518,\n",
              "  'answer_start': 491,\n",
              "  'text': 'Respiratory virus infection'},\n",
              " {'answer_end': 7696,\n",
              "  'answer_start': 7632,\n",
              "  'text': 'from\\naround 2-4 during the uncontrolled epidemic down to below 1'},\n",
              " {'answer_end': 10340, 'answer_start': 10323, 'text': ' from 3 to 7 days'},\n",
              " {'answer_end': 2519,\n",
              "  'answer_start': 2472,\n",
              "  'text': 'drug-resistance and frequent antigenic mutation'},\n",
              " {'answer_end': 3686,\n",
              "  'answer_start': 3618,\n",
              "  'text': 'using cultured viruses and inoculated chicken eggs with patient sera'},\n",
              " {'answer_end': 5517,\n",
              "  'answer_start': 5425,\n",
              "  'text': 'susceptible bats (S B ), exposed bats (E B ), infected bats (I B ), and removed bats (R B ).'},\n",
              " {'answer_end': 6091,\n",
              "  'answer_start': 5995,\n",
              "  'text': '14 were linked to a cluster in Bavaria, Germany, and seven to a cluster in Haute-Savoie, France '},\n",
              " {'answer_end': 3402,\n",
              "  'answer_start': 3322,\n",
              "  'text': 'Dromedary camels are the major animal source of MERS-CoV transmission to humans.'},\n",
              " {'answer_end': 1441, 'answer_start': 1428, 'text': 'December 2019'},\n",
              " {'answer_end': 3560,\n",
              "  'answer_start': 3382,\n",
              "  'text': ' to support the global strategy of containment of COVID-19 with rapid identification and follow-up of cases linked to affected countries in order to minimise onward transmission.'},\n",
              " {'answer_end': 25948, 'answer_start': 25927, 'text': 'an antiviral medicine'},\n",
              " {'answer_end': 13276, 'answer_start': 13262, 'text': '15% [3.7%-41%]'},\n",
              " {'answer_end': 2941,\n",
              "  'answer_start': 2740,\n",
              "  'text': 'an extraordinary event which is determined to constitute a public health risk to other States through the international spread of disease and to potentially require a coordinated international response'},\n",
              " {'answer_end': 15807,\n",
              "  'answer_start': 15402,\n",
              "  'text': 'A recent systematic review of tuberculosis as a comorbidity of childhood pneumonia reported culture-confirmed disease in about 8% of cases [50] . Because intrathoracic tuberculosis disease is only culture-confirmed in a minority of cases, the true burden could be even higher; tuberculosis could therefore be an important contributor to childhood pneumonia incidence and mortality in high-prevalence areas'},\n",
              " {'answer_end': 5985,\n",
              "  'answer_start': 5770,\n",
              "  'text': ' 35 cases (missing for three cases), of whom 14 were infected in China (Hubei province: 10 cases; Shandong province: one case; province not reported for three cases). The remaining 21 cases were infected in Europe. '},\n",
              " {'answer_end': 834,\n",
              "  'answer_start': 749,\n",
              "  'text': 'that viral infection increases airway inflammation which aggravates disease symptoms.'},\n",
              " {'answer_end': 23913,\n",
              "  'answer_start': 23513,\n",
              "  'text': ' the effect induced by the virus may destabilize the equilibrium of the microbiome present (Molyneaux et al., 2013; Kloepfer et al., 2014; Kloepfer et al., 2017; Jubinville et al., 2018; van Rijn et al., 2019) . In addition, viral infection may disrupt biofilm colonies in the upper airway (e.g., Streptococcus pneumoniae) microbiome to be release into the lower airway and worsening the inflammation'},\n",
              " {'answer_end': 5402,\n",
              "  'answer_start': 5150,\n",
              "  'text': ' In the 57, 6 cases were detected to be blood positive, all of them (100%) were severe in symptom requiring special care attention, and the blood of the rest 51 cases was without detectable virus in the blood, only 12 of them (23.5%) were severe cases.'},\n",
              " {'answer_end': 2350,\n",
              "  'answer_start': 2274,\n",
              "  'text': 'global childhood population from 605 million in 2000 to 664 million in 2015 '},\n",
              " {'answer_end': 522, 'answer_start': 519, 'text': '2.2'},\n",
              " {'answer_end': 8797,\n",
              "  'answer_start': 8732,\n",
              "  'text': 'acute respiratory distress syndrome and multiorgan system failure'},\n",
              " {'answer_end': 2866,\n",
              "  'answer_start': 2812,\n",
              "  'text': 'the average codon usage frequencies in the host genome'},\n",
              " {'answer_end': 30881,\n",
              "  'answer_start': 30753,\n",
              "  'text': 'as targets to modulate viral induced airway changes as a means of managing exacerbation of chronic airway inflammatory diseases.'},\n",
              " {'answer_end': 5622,\n",
              "  'answer_start': 5516,\n",
              "  'text': ' the roles of furin in these processes, as well as the history and future of furin-targeting therapeutics.'},\n",
              " {'answer_end': 1768,\n",
              "  'answer_start': 1727,\n",
              "  'text': 'heparan sulfates in the membrane proteins'},\n",
              " {'answer_end': 29757,\n",
              "  'answer_start': 29615,\n",
              "  'text': 'from the infected epithelium and responding immune cells, which may serve to further dysregulate airway inflammation leading to exacerbations.'},\n",
              " {'answer_end': 9679,\n",
              "  'answer_start': 9458,\n",
              "  'text': 'confronted by the selection pressures of population immu-\\nnity, these pandemic Viruses begin to drift genetically and\\neventually settle into a pattern of annual epidemic recur-\\nrences caused by the drifted Virus variants.'},\n",
              " {'answer_end': 8572,\n",
              "  'answer_start': 8418,\n",
              "  'text': 'fever, cough and upper respiratory tract (URT) signs and symptoms usually occur first, followed within a week by progressive LRT distress and lymphopaenia'},\n",
              " {'answer_end': 12072, 'answer_start': 12064, 'text': 'N95 mask'},\n",
              " {'answer_end': 4100,\n",
              "  'answer_start': 4064,\n",
              "  'text': ', and later by the National Health C'},\n",
              " {'answer_end': 4448,\n",
              "  'answer_start': 4407,\n",
              "  'text': ' relatively close and prolonged exposure '},\n",
              " {'answer_end': 965, 'answer_start': 956, 'text': 'HCoV‐HKU1'},\n",
              " {'answer_end': 2269, 'answer_start': 2266, 'text': 'sed'},\n",
              " {'answer_end': 29328,\n",
              "  'answer_start': 29232,\n",
              "  'text': ' viral infections and may play a role in the modulation of antiviral responses and inflammation '},\n",
              " {'answer_end': 5206,\n",
              "  'answer_start': 5066,\n",
              "  'text': ' to properly manage exacerbations via supplementary treatments that may alleviate the exacerbation symptoms or prevent severe exacerbations.'},\n",
              " {'answer_end': 46370, 'answer_start': 46358, 'text': 'inefficient '},\n",
              " {'answer_end': 18806,\n",
              "  'answer_start': 18325,\n",
              "  'text': ' By the end of 2015, Haemophilus influenzae type B conjugate vaccination had been introduced in 73 countries, with global coverage estimated at 68%. However, inequities are still apparent among regions: in the Americas coverage is estimated at 90%, while in the Western Pacific it is only 25%. By 2015, pneumococcal conjugate vaccination had been introduced into 54 countries, with global coverage of 35% for three doses of pneumococcal conjugate vaccination for infant populations'},\n",
              " {'answer_end': 6142,\n",
              "  'answer_start': 6117,\n",
              "  'text': 'people 85 years and older'},\n",
              " {'answer_end': 5988, 'answer_start': 5985, 'text': '20%'},\n",
              " {'answer_end': 2668,\n",
              "  'answer_start': 2619,\n",
              "  'text': \"Public Health Emergency of International Concern'\"},\n",
              " {'answer_end': 5254,\n",
              "  'answer_start': 5174,\n",
              "  'text': 'MERS kills its human host more often than SARS did (20-40 % versus 9 % for SARS '},\n",
              " {'answer_end': 26879,\n",
              "  'answer_start': 26840,\n",
              "  'text': 'banned gatherings of more than 2 people'},\n",
              " {'answer_end': 15850,\n",
              "  'answer_start': 15786,\n",
              "  'text': 'he nasal epithelium remains the primary site of most infections.'},\n",
              " {'answer_end': 7163,\n",
              "  'answer_start': 7073,\n",
              "  'text': 'infections will increase (dependent on how much greater than 1 the reproduction number is)'},\n",
              " {'answer_end': 10064,\n",
              "  'answer_start': 9972,\n",
              "  'text': ' Illness rates were high, but death rates\\nin most locales were not appreciably above normal.'},\n",
              " {'answer_end': 6866,\n",
              "  'answer_start': 6343,\n",
              "  'text': 'Haemophilus influenzae type B conjugate vaccination in high-burden communities, the vaccination was associated with an 18% decrease in radiologic pneumonia [13] . Introduction of pneumococcal conjugate vaccination was associated with a 26% decrease in radiologic pneumonia in California between 1995 and 1998 [14] . In vaccine efficacy trials in low-and middle-income countries, pneumococcal conjugate vaccination reduced radiologic pneumonia by 37% in the Gambia [15] , 25% in South Africa [16] and 26% in the Philippines '},\n",
              " {'answer_end': 4964,\n",
              "  'answer_start': 4901,\n",
              "  'text': 'as exopeptidase dipeptidyl peptidase 4 (DPP4; also called CD26)'},\n",
              " {'answer_end': 1637,\n",
              "  'answer_start': 1539,\n",
              "  'text': 'how recent models have expanded our appreciation of the mechanisms of viral-induced exacerbations.'},\n",
              " {'answer_end': 5120,\n",
              "  'answer_start': 5028,\n",
              "  'text': '10 days after illness onset [7] , consistent with the peak in viral load at around that time'},\n",
              " {'answer_end': 435,\n",
              "  'answer_start': 292,\n",
              "  'text': \"a 'Valley of Death' 1,2 no less, between broadening strides in fundamental biomedical research and their incommensurate reach into the clinic. \"},\n",
              " {'answer_end': 11050,\n",
              "  'answer_start': 10958,\n",
              "  'text': '0.5 times that of symptomatic infection (κ = 0.5), which was the similar value as influenza '},\n",
              " {'answer_end': 7518,\n",
              "  'answer_start': 7487,\n",
              "  'text': 'within airway epithelial cells '},\n",
              " {'answer_end': 6709, 'answer_start': 6693, 'text': '~ 65% similarity'},\n",
              " {'answer_end': 2408,\n",
              "  'answer_start': 2291,\n",
              "  'text': 'more than 400,000 children were infected worldwide, mostly through MTCT and 90% of them lived in sub-Saharan Africa. '},\n",
              " {'answer_end': 30009,\n",
              "  'answer_start': 29883,\n",
              "  'text': 'strong recommendations or laws to the general\\npublic and primary care about self—isolation when showing COVID-19-like symptoms'},\n",
              " {'answer_end': 1113,\n",
              "  'answer_start': 985,\n",
              "  'text': 'An estimated one third of the world’s population (or\\nz500 million persons) were infected and had clinical-\\nly apparent illnesses'},\n",
              " {'answer_end': 4056,\n",
              "  'answer_start': 3802,\n",
              "  'text': 'demographics, history of recent travel to affected areas, close contact with a probable or confirmed COVID-19 case, underlying conditions, signs and symptoms of disease at onset, type of specimens from which the virus was detected, and clinical outcome. '},\n",
              " {'answer_end': 10212,\n",
              "  'answer_start': 10064,\n",
              "  'text': ' A sec-\\nond or fall wave spread globally from September to\\nNovember 1918 and was highly fatal. In many nations, a\\nthird wave occurred in early 1919 '},\n",
              " {'answer_end': 2300,\n",
              "  'answer_start': 2039,\n",
              "  'text': 'Out of a total of 3711 persons (consisting of 2666 passengers and 1045 crew members), 199 symptomatic cases have been diagnosed on board as of 24 February, and additional asymptomatic infections and symptomatic cases after disembarkation have also been reported'},\n",
              " {'answer_end': 6274, 'answer_start': 6261, 'text': '13 to 14 days'},\n",
              " {'answer_end': 8597,\n",
              "  'answer_start': 8559,\n",
              "  'text': 'a median of 13 days (range: 8-23 days)'},\n",
              " {'answer_end': 5192,\n",
              "  'answer_start': 5150,\n",
              "  'text': ', the C i series, as an exponential growin'},\n",
              " {'answer_end': 1225,\n",
              "  'answer_start': 1194,\n",
              "  'text': '102 and 47 cases, respectively.'},\n",
              " {'answer_end': 9066,\n",
              "  'answer_start': 8965,\n",
              "  'text': 'Supporting industry-academia collaboration and nurturing talent and skills in the Indo-Pacific region'},\n",
              " {'answer_end': 7576, 'answer_start': 7565, 'text': ' nine cases'},\n",
              " {'answer_end': 7023,\n",
              "  'answer_start': 6726,\n",
              "  'text': 'This was mostly a result of delays in identifying the index cases of the two clusters in France and Germany. In the German cluster, for example, the first three cases detected locally were hospitalised in a mean of 5.7 days, whereas the following six took only a mean of 2 days to be hospitalised.'},\n",
              " {'answer_end': 4725,\n",
              "  'answer_start': 4685,\n",
              "  'text': 'their ease of transmission and infection'},\n",
              " {'answer_end': 23177,\n",
              "  'answer_start': 23095,\n",
              "  'text': 'Persons 65 years of age in 1918 had a dispro-\\nportionately high inﬂuenza incidence'},\n",
              " {'answer_end': 20521,\n",
              "  'answer_start': 20212,\n",
              "  'text': 'massive damage and cell death to the epithelial barrier, so much so that areas of the epithelium may be completely absent post infection (Yan et al., 2016; Tan et al., 2019) . Factors such as RANTES and CXCL10, which recruit immune cells to induce apoptosis, are strongly induced from IFV infected epithelium '},\n",
              " {'answer_end': 422, 'answer_start': 413, 'text': ' in 2012 '},\n",
              " {'answer_end': 9833,\n",
              "  'answer_start': 9774,\n",
              "  'text': 'translating the biomedical research output of that country.'},\n",
              " {'answer_end': 7316,\n",
              "  'answer_start': 7231,\n",
              "  'text': 'highly successful in greatly reducing the number of secondary transmissions on board.'},\n",
              " {'answer_end': 4162,\n",
              "  'answer_start': 3955,\n",
              "  'text': 'a virus that had infected over 90 % of adult dromedary camels (DC; Camelus dromedarius) in the KSA [4] , also DCs across the Arabian Peninsula and parts of Africa that are a source of DC imports for the KSA '},\n",
              " {'answer_end': 29471,\n",
              "  'answer_start': 29446,\n",
              "  'text': 'circulating miRNA changes'},\n",
              " {'answer_end': 6454, 'answer_start': 6441, 'text': '11 to 13 days'},\n",
              " {'answer_end': 4273, 'answer_start': 4258, 'text': 'betacoronavirus'},\n",
              " {'answer_end': 741,\n",
              "  'answer_start': 566,\n",
              "  'text': 'New conjugate vaccines against Haemophilus influenzae type b and Streptococcus pneumoniae have contributed to decreases in radiologic, clinical and complicated pneumonia cases'},\n",
              " {'answer_end': 42054,\n",
              "  'answer_start': 42006,\n",
              "  'text': 'due to new infections among naïve DC populations'},\n",
              " {'answer_end': 19808,\n",
              "  'answer_start': 19437,\n",
              "  'text': 'Influenza vaccination during pregnancy is safe, provides reasonable maternal protection against influenza, and also protects infants for a limited period from confirmed influenza infection (vaccine efficacy 63% in Bangladesh [63] and 50.4% in South Africa [64] ). However as antibody levels drop sharply after birth, infant protection does not persist much beyond 8 weeks'},\n",
              " {'answer_end': 2945, 'answer_start': 2938, 'text': 'phase 6'},\n",
              " {'answer_end': 2217, 'answer_start': 2195, 'text': 'bone formation factor.'},\n",
              " {'answer_end': 67516,\n",
              "  'answer_start': 67480,\n",
              "  'text': 'Global alignment of case definitions'},\n",
              " {'answer_end': 4493,\n",
              "  'answer_start': 4456,\n",
              "  'text': 'The first viral isolate was patented '},\n",
              " {'answer_end': 8718,\n",
              "  'answer_start': 8257,\n",
              "  'text': ' Using the revised definition, the 10-valent pneumococcal conjugate vaccine (pneumococcal conjugate vaccination-10), had a vaccine efficacy of 22% in preventing presumed bacterial pneumonia in young children in South America [22] , and pneumococcal conjugate vaccination-13 had a vaccine efficacy of 39% in preventing presumed bacterial pneumonia in children older than 16 weeks who were not infected with human immunodeficiency virus (HIV) in South Africa [21]'},\n",
              " {'answer_end': 45283,\n",
              "  'answer_start': 45254,\n",
              "  'text': 'Droplet spread between humans'},\n",
              " {'answer_end': 612, 'answer_start': 609, 'text': '102'},\n",
              " {'answer_end': 5073, 'answer_start': 5069, 'text': 'HCoV'},\n",
              " {'answer_end': 1307,\n",
              "  'answer_start': 1276,\n",
              "  'text': 'Highly virulent species of HCoV'},\n",
              " {'answer_end': 1865, 'answer_start': 1847, 'text': '27,534 nucleotides'},\n",
              " {'answer_end': 1113,\n",
              "  'answer_start': 1075,\n",
              "  'text': 'lower respiratory tract (LRT) disease '},\n",
              " {'answer_end': 16589,\n",
              "  'answer_start': 16507,\n",
              "  'text': 'to determine the contribution of the microbiome and mycobiome to viral infections.'},\n",
              " {'answer_end': 7827,\n",
              "  'answer_start': 7673,\n",
              "  'text': 'emerging human pathogens that possess enormous genomes of up to several megabases in size encoding hundreds of proteins with unique and unknown functions.'},\n",
              " {'answer_end': 5832,\n",
              "  'answer_start': 5745,\n",
              "  'text': 'diagnosing and categorizing\\nCOVID-19 on the basis of case definitions issued by the WHO'},\n",
              " {'answer_end': 14097,\n",
              "  'answer_start': 14005,\n",
              "  'text': 'help estimate the incidence of severe cases and identify risk factors for severity and death'},\n",
              " {'answer_end': 2560,\n",
              "  'answer_start': 2524,\n",
              "  'text': 'coronavirus disease 2019 (COVID -19)'},\n",
              " {'answer_end': 7521,\n",
              "  'answer_start': 7275,\n",
              "  'text': 'Of the remaining 29, 20 reported fever, 14 reported cough and eight reported weakness. Additional symptoms reported included headaches (6 cases), sore throat (2), rhinorrhoea (2), shortness of breath (2), myalgia (1), diarrhoea (1) and nausea (1)'},\n",
              " {'answer_end': 16423,\n",
              "  'answer_start': 16333,\n",
              "  'text': 'may further alter the local environment and contribute to current and future exacerbations'},\n",
              " {'answer_end': 4117, 'answer_start': 4107, 'text': '10 minutes'},\n",
              " {'answer_end': 1789,\n",
              "  'answer_start': 1742,\n",
              "  'text': 'mutational pressure and translational selection'},\n",
              " {'answer_end': 27169,\n",
              "  'answer_start': 26867,\n",
              "  'text': ' the 1918\\npandemic was different in degree, but not in kind, from\\nprevious and subsequent pandemics. Despite the extraordi-\\nnary number of global deaths, most inﬂuenza cases in\\n1918 (>95% in most locales in industrialized nations) were\\nmild and essentially indistinguishable from inﬂuenza cases\\ntoday. '},\n",
              " {'answer_end': 3250,\n",
              "  'answer_start': 3211,\n",
              "  'text': 'a 60 year old man from Bisha in the KSA'},\n",
              " {'answer_end': 13730, 'answer_start': 13726, 'text': '94 %'},\n",
              " {'answer_end': 11909,\n",
              "  'answer_start': 11825,\n",
              "  'text': 'majority of confirmed cases of MERS-CoV were reported among people aged 40 and above'},\n",
              " {'answer_end': 12605, 'answer_start': 12601, 'text': '22.1'},\n",
              " {'answer_end': 28392,\n",
              "  'answer_start': 28279,\n",
              "  'text': ' to directly destroy the cilia of the ciliated cells and almost all respiratory viruses infect the ciliated cells'},\n",
              " {'answer_end': 6573,\n",
              "  'answer_start': 6549,\n",
              "  'text': 'Löwenstein-Jensen medium'},\n",
              " {'answer_end': 862,\n",
              "  'answer_start': 836,\n",
              "  'text': 'basic reproduction number,'},\n",
              " {'answer_end': 29178,\n",
              "  'answer_start': 29048,\n",
              "  'text': 'short non-coding RNAs involved in post-transcriptional modulation of biological processes, and implicated in a number of diseases '},\n",
              " {'answer_end': 8020,\n",
              "  'answer_start': 7735,\n",
              "  'text': ' A revised case definition of \"presumed bacterial pneumonia\" has been introduced, and this definition includes pneumonia cases with WHO-defined alveolar consolidation, as well as those with other abnormal chest radiograph infiltrates and a serum C-reactive protein of at least 40 mg/L '},\n",
              " {'answer_end': 9688, 'answer_start': 9670, 'text': 'amplification step'},\n",
              " {'answer_end': 6725,\n",
              "  'answer_start': 6596,\n",
              "  'text': 'The mean number of days to hospitalisation was 2.5 days for cases imported from China, but 4.6 days for those infected in Europe.'},\n",
              " {'answer_end': 8555,\n",
              "  'answer_start': 8057,\n",
              "  'text': ' the responses against the virus may be impaired or aberrant, causing sustained inflammation and erroneous infiltration, resulting in the exacerbation of their symptoms (Mallia and Johnston, 2006; Dougherty and Fahy, 2009; Busse et al., 2010; Britto et al., 2017; Linden et al., 2019) . This is usually further compounded by the increased susceptibility of chronic airway inflammatory disease patients toward viral respiratory infections, thereby increasing the frequency of exacerbation as a whole'},\n",
              " {'answer_end': 2129,\n",
              "  'answer_start': 2003,\n",
              "  'text': 'DC-SIGNR plays a crucial role in MTCT of HIV-1 and that impaired placental DC-SIGNR expression increases risk of transmission.'},\n",
              " {'answer_end': 7927,\n",
              "  'answer_start': 7896,\n",
              "  'text': ' to determine high risk groups.'},\n",
              " {'answer_end': 2001,\n",
              "  'answer_start': 1806,\n",
              "  'text': 'the understanding of viral induced exacerbation in chronic airway inflammatory diseases and indicates pathways that may be targeted for more effective management of chronic inflammatory diseases.'},\n",
              " {'answer_end': 4006, 'answer_start': 3997, 'text': 'HCoV-OC43'},\n",
              " {'answer_end': 6347,\n",
              "  'answer_start': 6331,\n",
              "  'text': '35\\u202f000 to 60\\u202f000'},\n",
              " {'answer_end': 21959,\n",
              "  'answer_start': 21850,\n",
              "  'text': 'the induction of vasodilation or tight junction opening factors which may increase the rate of infiltration. '},\n",
              " {'answer_end': 23363,\n",
              "  'answer_start': 23274,\n",
              "  'text': 'fit a counterfactual model without the interventions\\nand compare this to the actual model'},\n",
              " {'answer_end': 10009,\n",
              "  'answer_start': 9813,\n",
              "  'text': 'These trends have been reversed since the introduction of pneumococcal conjugate vaccination-13. Data from the United States suggest that empyema decreased by 50% in children younger than 5 years '},\n",
              " {'answer_end': 6407,\n",
              "  'answer_start': 6125,\n",
              "  'text': 'Confounding deﬁnite assignment of a geographic\\npoint of origin, the 1918 pandemic spread more or less\\nsimultaneously in 3 distinct waves during an z12-month\\nperiod in 191871919, in Europe, Asia, and North America\\n(the ﬁrst wave was best described in the United States in\\nMarch 1918)'},\n",
              " {'answer_end': 11223,\n",
              "  'answer_start': 11043,\n",
              "  'text': 'The occurrence, and to some extent the severity, of recur-\\nrent annual outbreaks, are driven by Viral antigenic drift,\\nwith an antigenic variant Virus emerging to become domi-\\nnant'},\n",
              " {'answer_end': 4827,\n",
              "  'answer_start': 4662,\n",
              "  'text': 'The risk of developing at least one of the major sequelae was estimated as 6% after an ambulatory pneumonia event and 14% after an episode of hospitalized pneumonia.'},\n",
              " {'answer_end': 22786,\n",
              "  'answer_start': 22719,\n",
              "  'text': ' an oropharyngeal and throat swab or a nasopharyngeal aspirate/wash'},\n",
              " {'answer_end': 6620,\n",
              "  'answer_start': 6415,\n",
              "  'text': 'The study of autoantigen-specific B cells and a detailed analysis of B cell subsets with pathogenic potential in humans could lead to a better understanding of how to prevent and treat autoimmune diseases.'},\n",
              " {'answer_end': 3660,\n",
              "  'answer_start': 3429,\n",
              "  'text': ' will enable the tapping of useful new biological functions and processes, methods for controlling infection, and the deployment of symbiotic or subclinical viruses in new therapies and biotechnologies that are so crucially needed.'},\n",
              " {'answer_end': 2910,\n",
              "  'answer_start': 2869,\n",
              "  'text': ' MERS-CoV infects males more than females'},\n",
              " {'answer_end': 12552,\n",
              "  'answer_start': 12529,\n",
              "  'text': ' IL-25, IL-33 and TSLP '},\n",
              " {'answer_end': 1803,\n",
              "  'answer_start': 1779,\n",
              "  'text': '019-nCoV at 2.56 (95% CI'},\n",
              " {'answer_end': 9625, 'answer_start': 9615, 'text': '95% to 97%'},\n",
              " {'answer_end': 13255, 'answer_start': 13237, 'text': '0.41% [0.09%-1.2%]'},\n",
              " {'answer_end': 24430,\n",
              "  'answer_start': 24099,\n",
              "  'text': ' the 1918 Virus had an intrinsically high Virulence, tem-\\npered only in those patients who had been born before\\n1889, e.g., because of exposure to a then-circulating Virus\\ncapable of providing partial immunoprotection against the\\n1918 Virus strain only in persons old enough (>35 years) to\\nhave been infected during that prior era '},\n",
              " {'answer_end': 10829,\n",
              "  'answer_start': 10603,\n",
              "  'text': 'These factors heighten local inflammation and the infiltration of granulocytes, T-cells and B-cells (Wark and Gibson, 2006; Braciale et al., 2012) . The increased inflammation, in turn, worsens the symptoms of airway diseases.'},\n",
              " {'answer_end': 12449,\n",
              "  'answer_start': 12370,\n",
              "  'text': 'cumulatively, 5.9 [1.9-15.2] million people have been\\ninfected as of March 28th'},\n",
              " {'answer_end': 11067,\n",
              "  'answer_start': 11044,\n",
              "  'text': 'daily air travel volume'},\n",
              " {'answer_end': 454, 'answer_start': 451, 'text': '199'},\n",
              " {'answer_end': 28106,\n",
              "  'answer_start': 28051,\n",
              "  'text': 'restrict viral infection at the stage of cellular entry'},\n",
              " {'answer_end': 18371,\n",
              "  'answer_start': 18044,\n",
              "  'text': 'enhance granulocyte infiltration thus prolonging the exacerbation symptoms. Such persistent inflammation may also be found within DNA viruses such as AdV, hCMV and HSV, whose infections generally persist longer (Imperiale and Jiang, 2015) , further contributing to chronic activation of inflammation when they infect the airway'},\n",
              " {'answer_end': 16208,\n",
              "  'answer_start': 16152,\n",
              "  'text': 'viral infections or their components persist in patients'},\n",
              " {'answer_end': 22205,\n",
              "  'answer_start': 21925,\n",
              "  'text': 'age-speciﬁc death rates in the\\n1918 pandemic exhibited a distinct pattern that has not been\\ndocumented before or since: a “W—shaped” curve, similar to\\nthe familiar U-shaped curve but with the addition of a third\\n(middle) distinct peak of deaths in young adults z20410\\nyears of age'},\n",
              " {'answer_end': 2328,\n",
              "  'answer_start': 2230,\n",
              "  'text': ' airway inflammation leading to complications such as coughing, wheezing and shortness of breath. '},\n",
              " {'answer_end': 27954,\n",
              "  'answer_start': 27719,\n",
              "  'text': 'Genetic variants in CCR5 have been shown to influence vertical transmission of HIV-1. CCR5 promoter variants resulting in higher expression of the receptor were associated with increased risk of MTCT of HIV-1 among sub-Saharan Africans'},\n",
              " {'answer_end': 1825, 'answer_start': 1807, 'text': 'IFITM5 with FKBP11'},\n",
              " {'answer_end': 2290, 'answer_start': 2275, 'text': '12 January 2020'},\n",
              " {'answer_end': 3850,\n",
              "  'answer_start': 3788,\n",
              "  'text': 'over a third of the positive people dying (at least 527, 35 %)'},\n",
              " {'answer_end': 26194,\n",
              "  'answer_start': 26111,\n",
              "  'text': 'oxidative stress which will further increase the local inflammation in the airway. '},\n",
              " {'answer_end': 8878, 'answer_start': 8874, 'text': '42 %'},\n",
              " {'answer_end': 9173,\n",
              "  'answer_start': 9113,\n",
              "  'text': 'prevents cleavage of viral proteins required for replication'},\n",
              " {'answer_end': 3627, 'answer_start': 3622, 'text': '1,493'},\n",
              " {'answer_end': 5631,\n",
              "  'answer_start': 5539,\n",
              "  'text': 'as a dense or fluffy opacity that occupies a portion or whole of a lobe, or the entire lung.'},\n",
              " {'answer_end': 4398,\n",
              "  'answer_start': 4365,\n",
              "  'text': '4D5, 2B5, 1F1, 2B8, 4F9, and 4E11'},\n",
              " {'answer_end': 1947,\n",
              "  'answer_start': 1716,\n",
              "  'text': 'Our model showed that the transmissibility of SARS-CoV-2 was higher than the Middle East respiratory syndrome in the Middle East countries, similar to severe acute respiratory syndrome, but lower than MERS in the Republic of Korea.'},\n",
              " {'answer_end': 19643,\n",
              "  'answer_start': 19542,\n",
              "  'text': 'The concomitant development of steroid resistance together with recurring or prolong viral infection '},\n",
              " {'answer_end': 6789,\n",
              "  'answer_start': 6720,\n",
              "  'text': 'parainfluenza virus (PIV), influenza virus (IFV) and adenovirus (AdV)'},\n",
              " {'answer_end': 4311,\n",
              "  'answer_start': 4062,\n",
              "  'text': 'Strategic planning and adequate protections can help protect patients and staff\\nagainst a highly infectious disease while maintaining function at a high volume capacity.\\nKeywords: Coronavirus, COVID-19, novel coronavirus pneumonia, infection control'},\n",
              " {'answer_end': 6565, 'answer_start': 6547, 'text': 'the seafood market'},\n",
              " {'answer_end': 4522,\n",
              "  'answer_start': 4479,\n",
              "  'text': 'in controlling inflammatory mRNA stability.'},\n",
              " {'answer_end': 2190,\n",
              "  'answer_start': 1562,\n",
              "  'text': 'potential human infecting HKU-1, 4, 5 and 9 [17, 18] . Recent MERS-CoV infection is another example for severe disease caused by used-to-be-less pathogenic coronaviruses. Shi and colleagues [19] by using NGS have discovered many unknown animal viruses from bat, especially some important paramyxoviruses and reoviruses. Filovirus has also been identified in bat with potential severe outcomes. Lyssaviruses (with many genotypes, including rabies virus) in the Rhabdoviridae family have been linked with severe fatal human cases, even in the developed countries, including Australia, with the bites of bats in the city [20, 21] .'},\n",
              " {'answer_end': 34215,\n",
              "  'answer_start': 34001,\n",
              "  'text': 'important information regarding viral pathogenesis and changes induced within the airway of chronic airway inflammatory disease patients to identify novel targets and pathway for improved management of the disease.'},\n",
              " {'answer_end': 10873,\n",
              "  'answer_start': 10785,\n",
              "  'text': 'the much higher fre-\\nquency of complicated, severe, and fatal cases in the last 2\\nwaves.'},\n",
              " {'answer_end': 6908,\n",
              "  'answer_start': 6881,\n",
              "  'text': \"Lipinski's ''Rule of five''\"},\n",
              " {'answer_end': 1760,\n",
              "  'answer_start': 1673,\n",
              "  'text': 'EBC collection using the RTube™ is not reliable for diagnosis of respiratory infections'},\n",
              " {'answer_end': 1496,\n",
              "  'answer_start': 1457,\n",
              "  'text': 'Wuhan, capital of Hubei Province, China'},\n",
              " {'answer_end': 5335, 'answer_start': 5324, 'text': '15 February'},\n",
              " {'answer_end': 4587,\n",
              "  'answer_start': 4550,\n",
              "  'text': ' in various subcellular compartments '},\n",
              " {'answer_end': 4478,\n",
              "  'answer_start': 4453,\n",
              "  'text': 'RNA methylation machinery'},\n",
              " {'answer_end': 35094,\n",
              "  'answer_start': 34966,\n",
              "  'text': 'the best mode of verification of the associations of the virus with the proposed mechanism of viral induced acute exacerbations '},\n",
              " {'answer_end': 2817, 'answer_start': 2812, 'text': 'viral'},\n",
              " {'answer_end': 7887, 'answer_start': 7876, 'text': 'seven cases'},\n",
              " {'answer_end': 6263,\n",
              "  'answer_start': 6000,\n",
              "  'text': 'Despite being a major cause of exacerbation, reports linking respiratory viruses to acute exacerbations only start to emerge in the late 1950s (Pattemore et al., 1992) ; with bacterial infections previously considered as the likely culprit for acute exacerbation '},\n",
              " {'answer_end': 49535, 'answer_start': 49531, 'text': '21 %'},\n",
              " {'answer_end': 6861,\n",
              "  'answer_start': 6732,\n",
              "  'text': 'chronic airway inflammation, and a history of respiratory symptoms such as wheeze, shortness of breath, chest tightness and cough'},\n",
              " {'answer_end': 14920,\n",
              "  'answer_start': 14834,\n",
              "  'text': 'enhance the secretion of cytokines and chemokines, such as TNF-, IL-6, IL-8, and IL-12'},\n",
              " {'answer_end': 8153,\n",
              "  'answer_start': 8086,\n",
              "  'text': 'All hospitalised cases had a benign clinical evolution except four,'},\n",
              " {'answer_end': 17772,\n",
              "  'answer_start': 17514,\n",
              "  'text': 'The single strongest risk factor for pneumonia is HIV infection, which is especially prevalent in children in sub-Saharan Africa. HIV-infected children have 6 times increased odds of developing severe pneumonia or of death compared to HIV-uninfected children'},\n",
              " {'answer_end': 6189,\n",
              "  'answer_start': 6078,\n",
              "  'text': 'through pressure differentials between contaminated effluent pipes, bathroom floor drains and flushing toilets '},\n",
              " {'answer_end': 1461,\n",
              "  'answer_start': 1435,\n",
              "  'text': 'exponential growth pattern'},\n",
              " {'answer_end': 46788,\n",
              "  'answer_start': 46742,\n",
              "  'text': 'readily transmit to more than one other human '},\n",
              " {'answer_end': 13541,\n",
              "  'answer_start': 13491,\n",
              "  'text': ' Detection of viral proteins rather than viral RNA'},\n",
              " {'answer_end': 7098, 'answer_start': 7088, 'text': 'Two cases '},\n",
              " {'answer_end': 47603, 'answer_start': 47586, 'text': 'acute LRT disease'},\n",
              " {'answer_end': 12784,\n",
              "  'answer_start': 12737,\n",
              "  'text': 'haemagglutinin (HA) and the neuraminidase (NA) '},\n",
              " {'answer_end': 3159, 'answer_start': 3138, 'text': '20 th September 2012 '},\n",
              " {'answer_end': 24119,\n",
              "  'answer_start': 23999,\n",
              "  'text': 'the nutrient profile in the airway through release of previously inaccessible nutrients that will alter bacterial growth'},\n",
              " {'answer_end': 2652,\n",
              "  'answer_start': 2525,\n",
              "  'text': 'influenza and parainfluenza viruses, RSV, adenovirus, rhinovirus, human metapneumovirus, enterovirus, bocavirus and coronavirus'},\n",
              " {'answer_end': 1728,\n",
              "  'answer_start': 1439,\n",
              "  'text': ' All inﬂuenza A pandemics since that time, and\\nindeed almost all cases of inﬂuenza A worldwide (except-\\ning human infections from avian Viruses such as H5N1 and\\nH7N7), have been caused by descendants of the 1918\\nVirus, including “drifted” H1N1 Viruses and reassorted\\nH2N2 and H3N2 Viruses.'},\n",
              " {'answer_end': 23435,\n",
              "  'answer_start': 23338,\n",
              "  'text': 'specific bacterial species in the microbiome which may thrive in the inflamed airway environment '},\n",
              " {'answer_end': 1913,\n",
              "  'answer_start': 1795,\n",
              "  'text': 'case was diagnosed on 1 February, the ship was requested to remain in the ocean near Yokohama from 3 February onwards.'},\n",
              " {'answer_end': 15332,\n",
              "  'answer_start': 15260,\n",
              "  'text': 'the synergistic effect of viral infection with other sensitizing agents '},\n",
              " {'answer_end': 28506,\n",
              "  'answer_start': 28430,\n",
              "  'text': 'underreporting as well as systematic and country-specific changes in testing'},\n",
              " {'answer_end': 31303,\n",
              "  'answer_start': 31084,\n",
              "  'text': 'the association of epigenetic modification and chronic airway inflammatory diseases, and that the nasal methylome was shown to be a sensitive marker for airway inflammatory changes (Cardenas et al., 2019; Gomez, 2019) .'},\n",
              " {'answer_end': 11374,\n",
              "  'answer_start': 10953,\n",
              "  'text': 'connected to the fever clinic and\\nincludes the fever accessway, the CT examination room, and the DR examination room for\\n6\\nconfirmed and suspected cases. One CT scanner and one DR system closest to the emergency\\ndepartment are designated the fever-CT and fever-DR to examine patients with suspected and\\nconfirmed COVID-19. There is a separate dedicated access between the contaminated area and\\nthe fever screening tents. '},\n",
              " {'answer_end': 3949, 'answer_start': 3933, 'text': 'human rhinovirus'},\n",
              " {'answer_end': 4436, 'answer_start': 4427, 'text': '2-3 weeks'},\n",
              " {'answer_end': 18788, 'answer_start': 18779, 'text': 'over half'},\n",
              " {'answer_end': 6571,\n",
              "  'answer_start': 6518,\n",
              "  'text': 'ranged between 0 and 10 days with a mean of 3.7 days '},\n",
              " {'answer_end': 2234,\n",
              "  'answer_start': 2042,\n",
              "  'text': \"by creatively leveraging the so-called 'strengths' of viruses. Viral RNA polymerisation and reverse transcription enable resistance to treatment by conferring extraordinary genetic diversity. \"},\n",
              " {'answer_end': 32208,\n",
              "  'answer_start': 32081,\n",
              "  'text': ' enhanced production of reactive oxygen species (ROS), oxidative stress and mitochondrial dysfunction in the airway epithelium '},\n",
              " {'answer_end': 6451,\n",
              "  'answer_start': 6348,\n",
              "  'text': ' it is likely that most were hospitalised to isolate the person rather than because of severe disease. '},\n",
              " {'answer_end': 5452,\n",
              "  'answer_start': 5423,\n",
              "  'text': 'estimated at $30-$100 billion'},\n",
              " {'answer_end': 26963, 'answer_start': 26947, 'text': 'RV, RSV, and IFV'},\n",
              " {'answer_end': 11676,\n",
              "  'answer_start': 11603,\n",
              "  'text': 'he clean area\\nincludes the administrative office and the diagnostic room.'},\n",
              " {'answer_end': 501, 'answer_start': 484, 'text': 'zoonotic diseases'},\n",
              " {'answer_end': 46356,\n",
              "  'answer_start': 46329,\n",
              "  'text': 'often healthcare associated'},\n",
              " {'answer_end': 597,\n",
              "  'answer_start': 488,\n",
              "  'text': 'Dysregulation of the antibody response can lead to an autoimmune disease, malignancy, or enhanced infection. '},\n",
              " {'answer_end': 9048,\n",
              "  'answer_start': 9002,\n",
              "  'text': 'back-calculate infections from\\nobserved deaths'},\n",
              " {'answer_end': 30384,\n",
              "  'answer_start': 30331,\n",
              "  'text': 'isolation if travelling back from an epidemic country'},\n",
              " {'answer_end': 6338,\n",
              "  'answer_start': 6242,\n",
              "  'text': 'All but two cases were hospitalised (35 of 37 where information on hospitalisation was reported)'},\n",
              " {'answer_end': 5309,\n",
              "  'answer_start': 5283,\n",
              "  'text': 'on the day of illness onse'},\n",
              " {'answer_end': 7086, 'answer_start': 7077, 'text': ' 31 cases'},\n",
              " {'answer_end': 34438,\n",
              "  'answer_start': 34256,\n",
              "  'text': ' epithelial cell models such as the air-liquid interface, in vitro airway epithelial model that has been adapted to studying viral infection and the changes it induced in the airway '},\n",
              " {'answer_end': 3018,\n",
              "  'answer_start': 2984,\n",
              "  'text': 'spread in more than two continents'},\n",
              " {'answer_end': 11481, 'answer_start': 11463, 'text': 'about 2.87 million'},\n",
              " {'answer_end': 5134,\n",
              "  'answer_start': 5040,\n",
              "  'text': 'Chest radiologic changes have been considered the gold standard for defining a pneumonia event'},\n",
              " {'answer_end': 22311,\n",
              "  'answer_start': 22289,\n",
              "  'text': 'bronchoalveolar lavage'},\n",
              " {'answer_end': 2784,\n",
              "  'answer_start': 2689,\n",
              "  'text': 'each intervention has the same effect on the\\nreproduction number across countries and over time'},\n",
              " {'answer_end': 9303,\n",
              "  'answer_start': 9031,\n",
              "  'text': ' An increased incidence of empyema in children was noted in some high-income countries following pneumococcal conjugate vaccination-7 introduction, and this was attributed to pneumococcal serotypes not included in pneumococcal conjugate vaccination-7, especially 3 and 19A'},\n",
              " {'answer_end': 32852,\n",
              "  'answer_start': 32785,\n",
              "  'text': 'adjusted for the age structure and contact patterns of\\neach country'},\n",
              " {'answer_end': 46296, 'answer_start': 46287, 'text': 'sporadic '},\n",
              " {'answer_end': 2309,\n",
              "  'answer_start': 2216,\n",
              "  'text': 'widespread evidence of past infection in adult camels and limited past exposure among humans.'},\n",
              " {'answer_end': 6201,\n",
              "  'answer_start': 6119,\n",
              "  'text': ' a recent boom in metagenomics and the cataloguing of the microbiome of our world.'},\n",
              " {'answer_end': 1893, 'answer_start': 1891, 'text': '47'},\n",
              " {'answer_end': 19224,\n",
              "  'answer_start': 19050,\n",
              "  'text': 'viral persistence which lead to continuous expression of antiviral genes may also lead to the development of steroid resistance, which is seen with RV, RSV, and PIV infection'},\n",
              " {'answer_end': 51311,\n",
              "  'answer_start': 51231,\n",
              "  'text': 'using high throughput or \"deep\" sequencing methods for complete genome deduction'},\n",
              " {'answer_end': 22894,\n",
              "  'answer_start': 22859,\n",
              "  'text': ' collected two to three weeks apart'},\n",
              " {'answer_end': 4745, 'answer_start': 4738, 'text': '80,793 '},\n",
              " {'answer_end': 3604,\n",
              "  'answer_start': 3271,\n",
              "  'text': ' 2 major descendant lineages of the 1918\\nH1N1 Virus, as well as 2 additional reassortant lineages,\\npersist naturally: a human epidemic/endemic H1N1 line-\\nage, a porcine enzootic H1N1 lineage (so-called classic\\nswine ﬂu), and the reassorted human H3N2 Virus lineage,\\nwhich like the human H1N1 Virus, has led to a porcine\\nH3N2 lineage.'},\n",
              " {'answer_end': 24359,\n",
              "  'answer_start': 24222,\n",
              "  'text': ' impaired bacterial immune response, either from direct viral influences, or use of corticosteroids to suppress the exacerbation symptoms'},\n",
              " {'answer_end': 6147,\n",
              "  'answer_start': 6107,\n",
              "  'text': 'bonerestricted IFITM-like (BRIL) protein'},\n",
              " {'answer_end': 66143,\n",
              "  'answer_start': 66077,\n",
              "  'text': 'from infected to uninfected humans in close and prolonged contact '},\n",
              " {'answer_end': 37110,\n",
              "  'answer_start': 37080,\n",
              "  'text': 'infection to onset of symptoms'},\n",
              " {'answer_end': 907, 'answer_start': 892, 'text': 'remains unknown'},\n",
              " {'answer_end': 30751,\n",
              "  'answer_start': 30719,\n",
              "  'text': 'work from home wherever possible'},\n",
              " {'answer_end': 5085,\n",
              "  'answer_start': 5011,\n",
              "  'text': 'Another mRNA under post-transcriptional regulation by Regnase-1 and Roquin'},\n",
              " {'answer_end': 2401,\n",
              "  'answer_start': 2356,\n",
              "  'text': 'zanamivir (Relenza) and oseltamivir (Tamiflu)'},\n",
              " {'answer_end': 21092,\n",
              "  'answer_start': 20957,\n",
              "  'text': ' further contact with other pathogens and allergens in the airway which may then prolong exacerbations or results in new exacerbations.'},\n",
              " {'answer_end': 2387,\n",
              "  'answer_start': 2302,\n",
              "  'text': ' by strongly limiting virus genome sizes and their incorporation of new information. '},\n",
              " {'answer_end': 41949, 'answer_start': 41948, 'text': 't'},\n",
              " {'answer_end': 3490, 'answer_start': 3480, 'text': 'monoclonal'},\n",
              " {'answer_end': 10807,\n",
              "  'answer_start': 10737,\n",
              "  'text': 'digestive tract might be one extrapulmonary site for virus replication'},\n",
              " {'answer_end': 24972,\n",
              "  'answer_start': 24937,\n",
              "  'text': 'enveloped, negative-sense RNA virus'},\n",
              " {'answer_end': 969,\n",
              "  'answer_start': 835,\n",
              "  'text': 'Recent advances in in vitro air-liquid interface 3D cultures, organoid cultures and the use of novel human and animal challenge models'},\n",
              " {'answer_end': 15155,\n",
              "  'answer_start': 14934,\n",
              "  'text': 'Viral sequence data now suggest that the entire 1918\\nVirus was novel to humans in, or shortly before, 1918, and\\nthat it thus was not a reassortant Virus produced from old\\nexisting strains that acquired 1 or more new genes'},\n",
              " {'answer_end': 5808,\n",
              "  'answer_start': 5671,\n",
              "  'text': 'how two IFN-cinducible factors exhibit broad-spectrum inhibition of IAV, measles (MV), zika (ZikV) and HIV by suppressing furin activity.'},\n",
              " {'answer_end': 3284,\n",
              "  'answer_start': 3228,\n",
              "  'text': 'Global Initiative on Sharing All Influenza Data (GISAID)'},\n",
              " {'answer_end': 26389,\n",
              "  'answer_start': 26357,\n",
              "  'text': 'rs12252 C allele and CC genotype'},\n",
              " {'answer_end': 4403, 'answer_start': 4383, 'text': 'bile acid transport '},\n",
              " {'answer_end': 10554,\n",
              "  'answer_start': 10518,\n",
              "  'text': 'a mean of 5.8 days (95% CI: 4.3-7.5)'},\n",
              " {'answer_end': 3089,\n",
              "  'answer_start': 2927,\n",
              "  'text': 'at there had been 4000 (95%CI: 1000-9700) cases in Wuhan with symptoms onset by 18 January 2020, and the basic reproduction number (R 0 ) was estimated at 2.6 (95'},\n",
              " {'answer_end': 7334,\n",
              "  'answer_start': 7253,\n",
              "  'text': 'investigation phase, followed by recognition, initiation, and acceleration phases'},\n",
              " {'answer_end': 18253,\n",
              "  'answer_start': 18174,\n",
              "  'text': 'provide a mathematical model for calculating the transmissibility of SARS-CoV-2'},\n",
              " {'answer_end': 5660, 'answer_start': 5644, 'text': ' up to 14 days ['},\n",
              " {'answer_end': 12029,\n",
              "  'answer_start': 12007,\n",
              "  'text': ' PCR-based techniques '},\n",
              " {'answer_end': 865, 'answer_start': 842, 'text': 'exponentially, up 1800%'},\n",
              " {'answer_end': 2245,\n",
              "  'answer_start': 2137,\n",
              "  'text': 'Without specific interventions, the rate of HIV-1 mother-tochild transmission (MTCT) is approximately 15-45%'},\n",
              " {'answer_end': 9891, 'answer_start': 9888, 'text': '52%'},\n",
              " {'answer_end': 2639,\n",
              "  'answer_start': 2468,\n",
              "  'text': 'calculating backwards from the deaths observed over time to estimate transmission that\\noccurred several weeks prior, allowing for the time lag between infection and death.'},\n",
              " {'answer_end': 12714, 'answer_start': 12711, 'text': 'two'},\n",
              " {'answer_end': 4716,\n",
              "  'answer_start': 4686,\n",
              "  'text': \"a 'brake' on the immune system\"},\n",
              " {'answer_end': 1563,\n",
              "  'answer_start': 1517,\n",
              "  'text': 'HCoV-HKU1, HCoV-OC43, HCoV-NL63, and HCoV-229E'},\n",
              " {'answer_end': 6261,\n",
              "  'answer_start': 6100,\n",
              "  'text': 'Widespread use of pneumococcal conjugate vaccination and Haemophilus influenzae type B conjugate vaccination has decreased the incidence of radiologic pneumonia.'},\n",
              " {'answer_end': 15202,\n",
              "  'answer_start': 15112,\n",
              "  'text': 'R 0 of SARS was about 2.1 in Hong Kong, China, 2.7 in Singapore, and 3.8 in Beijing, China'},\n",
              " {'answer_end': 4236,\n",
              "  'answer_start': 4149,\n",
              "  'text': 'a sandwiched extracellular structure composed of a β-sheet, α-helix loop, and a β-sheet'},\n",
              " {'answer_end': 10855,\n",
              "  'answer_start': 10581,\n",
              "  'text': 'we cannot discharge a patient purely based on oral swabs negative, who may still shed the virus by oral-fecal route. Above all, we strongly suggest using viral IgM and IgG serological test to confirm an infection, considering the unreliable results from oral swabs detection'},\n",
              " {'answer_end': 4302, 'answer_start': 4280, 'text': 'MERS-CoV and SARS-CoV.'},\n",
              " {'answer_end': 3134,\n",
              "  'answer_start': 3065,\n",
              "  'text': 'on the website of the professional emerging diseases (ProMED) network'},\n",
              " {'answer_end': 7071,\n",
              "  'answer_start': 6863,\n",
              "  'text': 'Middle East respiratory syndrome coronavirus (MERS-CoV), severe acute respiratory syndrome coronavirus (SARS-CoV), Zika virus (ZIKV), HCV, poliovirus (PV), influenza A virus (IAV), HIV, and enteroviruses (EV)'},\n",
              " {'answer_end': 20431,\n",
              "  'answer_start': 20337,\n",
              "  'text': 'by confirmatory IFA and/ or a plaque-reduction neutralization (PRNT) [69, 70, 85] or MNT test.'},\n",
              " {'answer_end': 2413,\n",
              "  'answer_start': 2273,\n",
              "  'text': '(MERS-CoV) was first isolated in 2012, in a 60-year-old man who died in Jeddah, KSA due to severe acute pneumonia and multiple organ failure'},\n",
              " {'answer_end': 380,\n",
              "  'answer_start': 295,\n",
              "  'text': 'Pneumonia remains the leading cause of death in children outside the neonatal period,'},\n",
              " {'answer_end': 7607,\n",
              "  'answer_start': 7578,\n",
              "  'text': 'In 16 of 29 symptomatic cases'},\n",
              " {'answer_end': 1902,\n",
              "  'answer_start': 1865,\n",
              "  'text': ' to have occurred during the first ha'},\n",
              " {'answer_end': 2284, 'answer_start': 2265, 'text': 'reproductive number'},\n",
              " {'answer_end': 46411,\n",
              "  'answer_start': 46374,\n",
              "  'text': 'requiring close and prolonged contact'},\n",
              " {'answer_end': 4366,\n",
              "  'answer_start': 4322,\n",
              "  'text': 'lateral surface (canalicular) of hepatocytes'},\n",
              " {'answer_end': 9115,\n",
              "  'answer_start': 9091,\n",
              "  'text': 'from 0.2 to 0.0125 mg/ml'},\n",
              " {'answer_end': 7230,\n",
              "  'answer_start': 7152,\n",
              "  'text': 'a near-complete picture of the main armaments in the human antiviral arsenal. '},\n",
              " {'answer_end': 29966,\n",
              "  'answer_start': 29800,\n",
              "  'text': ' to increase miR-21 and augmented inflammation in experimental murine asthma models, which is reversed with a combination treatment of anti-miR-21 and corticosteroids'},\n",
              " {'answer_end': 25575, 'answer_start': 25571, 'text': '30 %'},\n",
              " {'answer_end': 3196,\n",
              "  'answer_start': 3126,\n",
              "  'text': 'significant impact\\nof interventions implemented several weeks earlier.'},\n",
              " {'answer_end': 6725,\n",
              "  'answer_start': 6678,\n",
              "  'text': ' to directly sequence an RNA virus genome (IAV)'},\n",
              " {'answer_end': 3431,\n",
              "  'answer_start': 3393,\n",
              "  'text': 'in accordance with WHO recommendations'},\n",
              " {'answer_end': 6826, 'answer_start': 6817, 'text': 'aspartate'},\n",
              " {'answer_end': 38431,\n",
              "  'answer_start': 38407,\n",
              "  'text': 'discrete renewal process'},\n",
              " {'answer_end': 13904,\n",
              "  'answer_start': 13821,\n",
              "  'text': 'monoclonal antibody-based capture ELISA targeting the MERS-CoV nucleocapsid protein'},\n",
              " {'answer_end': 5768,\n",
              "  'answer_start': 5562,\n",
              "  'text': 'After a new influenza virus (H7N9) was identified in China in 2013, a series of modeling articles described the effect of, and level of preparedness for, a severe, single-wave pandemic in the United States.'},\n",
              " {'answer_end': 66222,\n",
              "  'answer_start': 66173,\n",
              "  'text': 'by poor infection control in health care settings'},\n",
              " {'answer_end': 30396,\n",
              "  'answer_start': 30247,\n",
              "  'text': 'miR-22 was shown to be suppressed in asthmatic epithelium in IFV infection which lead to aberrant epithelial response, contributing to exacerbations '},\n",
              " {'answer_end': 11503,\n",
              "  'answer_start': 11408,\n",
              "  'text': ' the fever-CT control room,\\nfever-DR control room, and other patient examination access areas. '},\n",
              " {'answer_end': 2515,\n",
              "  'answer_start': 2405,\n",
              "  'text': 'k had resulted in 2066 (618 of them are in Wuhan) confirmed cases and 56 (45 of them were in Wuhan) deaths in '},\n",
              " {'answer_end': 20579,\n",
              "  'answer_start': 20518,\n",
              "  'text': 'neighbouring sub-Saharan countries, North America, and Europe'},\n",
              " {'answer_end': 1376,\n",
              "  'answer_start': 1284,\n",
              "  'text': 'Total deaths were estimated at\\nz50 million (577) and were arguably as high as 100 mil-\\nlion '},\n",
              " {'answer_end': 15716,\n",
              "  'answer_start': 15611,\n",
              "  'text': 'too early to detect substantial\\nintervention impact in many countries at earlier stages of their epidemic'},\n",
              " {'answer_end': 4862,\n",
              "  'answer_start': 4780,\n",
              "  'text': 'clinicians, public health doctors, radiologists, laboratory technicians and nurses'},\n",
              " {'answer_end': 2977,\n",
              "  'answer_start': 2831,\n",
              "  'text': 'An email from Dr Ali Mohamed Zaki, an Egyptian virologist working at the Dr Soliman Fakeeh Hospital in Jeddah in the Kingdom of Saudi Arabia (KSA)'},\n",
              " {'answer_end': 7401,\n",
              "  'answer_start': 7365,\n",
              "  'text': 'at the end of the acceleration phase'},\n",
              " {'answer_end': 13180, 'answer_start': 13164, 'text': '3.0% [1.1%-7.4%]'},\n",
              " {'answer_end': 47071, 'answer_start': 47059, 'text': 'close to one'},\n",
              " {'answer_end': 559,\n",
              "  'answer_start': 515,\n",
              "  'text': 'Jeddah, in the Kingdom of Saudi Arabia (KSA)'},\n",
              " {'answer_end': 2168, 'answer_start': 2162, 'text': '4,250 '},\n",
              " {'answer_end': 2494, 'answer_start': 2484, 'text': '94 million'},\n",
              " {'answer_end': 5956,\n",
              "  'answer_start': 5741,\n",
              "  'text': 'ongoing studies to map Virulence\\nfactors are yielding interesting results. The 1918 sequence\\ndata, however, leave unanswered questions about the ori-\\ngin of the Virus (19) and about the epidemiology of the\\npandemic.'},\n",
              " {'answer_end': 15692,\n",
              "  'answer_start': 15425,\n",
              "  'text': 'Inﬂuenza Virus gene\\nsequences from a number ofﬁxed specimens ofwild birds\\ncollected circa 1918 show little difference from avian\\nViruses isolated today, indicating that avian Viruses likely\\nundergo little antigenic change in their natural hosts even\\nover long periods'},\n",
              " {'answer_end': 503,\n",
              "  'answer_start': 474,\n",
              "  'text': 'Transmissibility and severity'},\n",
              " {'answer_end': 3778,\n",
              "  'answer_start': 3594,\n",
              "  'text': 'to: describe the key epidemiological and clinical characteristics of COVID-19 cases detected in Europe; inform country preparedness; and improve further case detection and management. '},\n",
              " {'answer_end': 2098,\n",
              "  'answer_start': 2064,\n",
              "  'text': 's first identified in Wuhan, China'},\n",
              " {'answer_end': 4318,\n",
              "  'answer_start': 4276,\n",
              "  'text': 'members of the Roquin and Regnase families'},\n",
              " {'answer_end': 47637,\n",
              "  'answer_start': 47604,\n",
              "  'text': 'in a healthcare setting in Jordan'},\n",
              " {'answer_end': 3274,\n",
              "  'answer_start': 3172,\n",
              "  'text': 'Currently, there are no approved vaccines or medications available for most of the respiratory viruses'},\n",
              " {'answer_end': 4683,\n",
              "  'answer_start': 4624,\n",
              "  'text': 'Metered dose inhalers (MDIs) and dry powder inhalers (DPIs)'},\n",
              " {'answer_end': 32016,\n",
              "  'answer_start': 31886,\n",
              "  'text': ' an altered inflammatory state contributing to further viral persistence and exacerbation of chronic airway inflammatory diseases '},\n",
              " {'answer_end': 4124,\n",
              "  'answer_start': 4100,\n",
              "  'text': ' HCoV-OC43 and HCoV-229E'},\n",
              " {'answer_end': 17718,\n",
              "  'answer_start': 17454,\n",
              "  'text': 'there are also evidence of reduced type I (IFNβ) and III (IFNλ) interferon production due to type 2-skewed inflammation, which contributes to imperfect clearance of the virus resulting in persistence of viral components, or the live virus in the airway epithelium '},\n",
              " {'answer_end': 12755, 'answer_start': 12748, 'text': 'Germany'},\n",
              " {'answer_end': 25558, 'answer_start': 25542, 'text': 'at least 30 days'},\n",
              " {'answer_end': 12371,\n",
              "  'answer_start': 12220,\n",
              "  'text': ' less than 80 % identity between the amino acid sequence of MERS ORF 1ab and betacoronavirus relatives, Tylonycteris bat HKU4 and Pipistrellus bat HKU5'},\n",
              " {'answer_end': 1413,\n",
              "  'answer_start': 1336,\n",
              "  'text': ' in mild and influenza-like illnesses and in those with no signs or symptoms.'},\n",
              " {'answer_end': 11309,\n",
              "  'answer_start': 11277,\n",
              "  'text': 'had a history of travel to China'},\n",
              " {'answer_end': 29908,\n",
              "  'answer_start': 29884,\n",
              "  'text': ' above that for SARS-CoV'},\n",
              " {'answer_end': 9784, 'answer_start': 9776, 'text': '46 years'},\n",
              " {'answer_end': 1279, 'answer_start': 1277, 'text': '50'},\n",
              " {'answer_end': 4215,\n",
              "  'answer_start': 4045,\n",
              "  'text': 'H1N1 Viruses descended from the 1918 strain, as well as \\nH3N2 Viruses, have now been cocirculating worldwide for\\n29 years and show little evidence of imminent extinction.'},\n",
              " {'answer_end': 32400,\n",
              "  'answer_start': 32365,\n",
              "  'text': 'state of constant oxidative stress '},\n",
              " {'answer_end': 5969,\n",
              "  'answer_start': 5851,\n",
              "  'text': ' the virus can be a more serious and opportunistic pathogen associated with the death of up to 40 % of reported cases.'},\n",
              " {'answer_end': 18997,\n",
              "  'answer_start': 18818,\n",
              "  'text': ' the role of HPV infection in causing chronic inflammation in the airway and their association to exacerbations of chronic airway inflammatory diseases, which is scarcely explored'},\n",
              " {'answer_end': 5750,\n",
              "  'answer_start': 5647,\n",
              "  'text': 'with outbreaks in hospitals, with around 20 % of all cases to date involving healthcare workers (HCWs).'},\n",
              " {'answer_end': 30020,\n",
              "  'answer_start': 29995,\n",
              "  'text': 'the low transmission rate'},\n",
              " {'answer_end': 11256,\n",
              "  'answer_start': 10918,\n",
              "  'text': 'viral infections such as RV and RSV promote a Type 2-biased immune response (Becker, 2006; Jackson et al., 2014; Jurak et al., 2018) . This amplifies the basal type 2 inflammation resulting in a greater release of IL-4, IL-5, IL-13, RANTES and eotaxin and a further increase in eosinophilia, a key pathological driver of asthma and CRSwNP'},\n",
              " {'answer_end': 15508, 'answer_start': 15491, 'text': '1 mm-thick layers'},\n",
              " {'answer_end': 23260,\n",
              "  'answer_start': 23156,\n",
              "  'text': 'the relationship between asthma and COPD exacerbations and their association with the airway microbiome.'},\n",
              " {'answer_end': 18132,\n",
              "  'answer_start': 17978,\n",
              "  'text': ' the transmissibility of SARS-CoV-2 might be higher than MERS in the Middle East countries, similar to SARS, but lower than MERS in the Republic of Korea.'},\n",
              " {'answer_end': 23183,\n",
              "  'answer_start': 23122,\n",
              "  'text': 'Human urine and stool have been found to contain MERS-CoV RNA'},\n",
              " {'answer_end': 22368,\n",
              "  'answer_start': 22205,\n",
              "  'text': 'up-to-date case definitions and information about\\nconfirmatory tests to all staff with direct patient contact to allow appropriate barrier precautions\\nto be taken.'},\n",
              " {'answer_end': 27511, 'answer_start': 27495, 'text': 'IFITM1, 2, and 3'},\n",
              " {'answer_end': 3354,\n",
              "  'answer_start': 3215,\n",
              "  'text': ' alternative posttranslational and RNA-based antiviral approaches, designer vaccines, immunotherapy and the emerging field of neo-virology.'},\n",
              " {'answer_end': 15044,\n",
              "  'answer_start': 14630,\n",
              "  'text': ' However, in the inflamed airway, the cytokines and chemokines released instead augmented the inflammation present in the chronically inflamed airway, strengthening the neutrophilic infiltration in COPD airway, and eosinophilic infiltration in the asthmatic airway. The effect is also further compounded by the participation of Th1 and ILC1 cells in the COPD airway; and Th2 and ILC2 cells in the asthmatic airway.'},\n",
              " {'answer_end': 7248, 'answer_start': 7235, 'text': 'herd immunity'},\n",
              " {'answer_end': 16286,\n",
              "  'answer_start': 16248,\n",
              "  'text': 'close spacing of interventions in time'},\n",
              " {'answer_end': 24825,\n",
              "  'answer_start': 24713,\n",
              "  'text': 'due to the interplay between virus and pathogenic bacteria in exacerbating chronic airway inflammatory diseases '},\n",
              " {'answer_end': 1199,\n",
              "  'answer_start': 906,\n",
              "  'text': 'all of the 6 patients with detectable viral RNA in the blood cohort progressed to severe symptom stage, indicating a strong correlation of serum viral RNA with the disease severity (p-value = 0.0001). Meanwhile, 8 of the 11 patients with annal swab virus-positive was in severe clinical stage.'},\n",
              " {'answer_end': 3605,\n",
              "  'answer_start': 3574,\n",
              "  'text': 'S-palmitoylation on the protein'},\n",
              " {'answer_end': 16976,\n",
              "  'answer_start': 16949,\n",
              "  'text': ' number of daily infections'},\n",
              " {'answer_end': 6758, 'answer_start': 6742, 'text': '~ 85% similarity'},\n",
              " {'answer_end': 1675,\n",
              "  'answer_start': 1649,\n",
              "  'text': 'reased 21-fold (95% CI: 18'},\n",
              " {'answer_end': 5767,\n",
              "  'answer_start': 5505,\n",
              "  'text': 'In the anal swab cohort (Figure 1 (B)), 11 of 28 cases were detected to be anal swab positive, 8 of them (72.7%) were with severe symptoms, which was significantly higher than that 4 (23.5%) of the rest 17 cases without detectable virus in anal were severe cases'},\n",
              " {'answer_end': 22078,\n",
              "  'answer_start': 21976,\n",
              "  'text': 'EAP leaf extract is a prophylactic and immune enhancement agent against H5N1 influenza virus infection'},\n",
              " {'answer_end': 12719,\n",
              "  'answer_start': 12685,\n",
              "  'text': '15.0% (7.0 [18-19] million people)'},\n",
              " {'answer_end': 526, 'answer_start': 507, 'text': 'Gammaproteobacteria'},\n",
              " {'answer_end': 5151,\n",
              "  'answer_start': 4705,\n",
              "  'text': 'Patients who were with at least one of the following symptom should be diagnosed to be severe case, 1) distress of respiratory with respiratory rate > = 30/min; 2) Oxygen saturation < = 93% in the rest state, and 3) arterial oxygen tension (PaO₂) over inspiratory oxygen fraction (FIO₂) of less than 300 mm Hg. In the blood detection cohort (Figure 1 (A)), patients who had at less one serum sample measurement with the PCR method were included. '},\n",
              " {'answer_end': 13229, 'answer_start': 13214, 'text': '9.8% [3.2%-26%]'},\n",
              " {'answer_end': 3526,\n",
              "  'answer_start': 3446,\n",
              "  'text': 'the surface area can result in rapid absorption of the medication into the blood'},\n",
              " {'answer_end': 21768,\n",
              "  'answer_start': 21663,\n",
              "  'text': ' receive their annual influenza vaccination as the best means to prevent severe IFV induced exacerbation.'},\n",
              " {'answer_end': 7275,\n",
              "  'answer_start': 7190,\n",
              "  'text': 'as part of screening following repatriation and during contact tracing respectively. '},\n",
              " {'answer_end': 4159,\n",
              "  'answer_start': 4022,\n",
              "  'text': 'post-transcriptionally control expression of crucial inflammatory factors in various tissues and their potential therapeutic applications'},\n",
              " {'answer_end': 69108,\n",
              "  'answer_start': 69043,\n",
              "  'text': 'There is no evidence that MERS-CoV is a virus of pandemic concern'},\n",
              " {'answer_end': 5768,\n",
              "  'answer_start': 5735,\n",
              "  'text': 'indirect transmission via fomites'},\n",
              " {'answer_end': 8494,\n",
              "  'answer_start': 8399,\n",
              "  'text': 'was hospitalised for 21 days and required intensive care and mechanical ventilation for 19 days'},\n",
              " {'answer_end': 3201,\n",
              "  'answer_start': 3058,\n",
              "  'text': ' break the vicious cycle of escalating therapeutic development costs and trivial escape mutant selection, both quickly and in multiple places. '},\n",
              " {'answer_end': 730,\n",
              "  'answer_start': 679,\n",
              "  'text': 'across the Arabian Peninsula and in parts of Africa'},\n",
              " {'answer_end': 5595, 'answer_start': 5591, 'text': 'H7N9'},\n",
              " {'answer_end': 3924,\n",
              "  'answer_start': 3739,\n",
              "  'text': ' infiltration of activated immune cells that further release inflammatory mediators that cause acute symptoms such as increased mucus production, cough, wheeze and shortness of breath. '},\n",
              " {'answer_end': 679,\n",
              "  'answer_start': 600,\n",
              "  'text': '2019-nCoV was transmitted through respiratory tract and then induced pneumonia,'},\n",
              " {'answer_end': 5094, 'answer_start': 5088, 'text': ' Furin'},\n",
              " {'answer_end': 2640,\n",
              "  'answer_start': 2381,\n",
              "  'text': 'Does the patient have recent travel from an affected area?\\nHas the patient been in close contact with someone with COVID-19 or with patients with pneumonia of unknown cause?\\nDoes the patient reside in an area where there has been community spread of COVID-19?'},\n",
              " {'answer_end': 3701,\n",
              "  'answer_start': 3605,\n",
              "  'text': 'None of these Viral descendants, however,\\napproaches the pathogenicity of the 1918 parent Virus.'},\n",
              " {'answer_end': 9535,\n",
              "  'answer_start': 9465,\n",
              "  'text': 'there are 4,250 cases including 113 deaths reported among 38 countries'},\n",
              " {'answer_end': 30195,\n",
              "  'answer_start': 30018,\n",
              "  'text': ' increase miR-125a and b, and miR-132 in COPD epithelium which inhibits A20 and MAVS; and p300 and IRF3, respectively, resulting in increased susceptibility to viral infections '},\n",
              " {'answer_end': 20718,\n",
              "  'answer_start': 20485,\n",
              "  'text': 'the antibodies detected are able to specifically neutralize the intended virus and are not more broadly reactive to other coronaviruses found in DCs (bovine CoV, BCoV) or humans (HCoV-OC43, HCoV-229E, HCoV-NL63, HCoV-HKU1, SARS-CoV).'},\n",
              " {'answer_end': 5191,\n",
              "  'answer_start': 5150,\n",
              "  'text': ', the C i series, as an exponential growi'},\n",
              " {'answer_end': 1264,\n",
              "  'answer_start': 1216,\n",
              "  'text': 'Reservoir-People (RP) transmission network model'},\n",
              " {'answer_end': 13105, 'answer_start': 13088, 'text': '1.1% [0.36%-3.1%]'},\n",
              " {'answer_end': 1884, 'answer_start': 1869, 'text': '108 per 100,000'},\n",
              " {'answer_end': 1296,\n",
              "  'answer_start': 1123,\n",
              "  'text': 'fever, cough, breathing difficulties and pneumonia that may progress to acute respiratory distress syndrome, multiorgan failure and death in 20 % to 40 % of those infected. '},\n",
              " {'answer_end': 7475,\n",
              "  'answer_start': 7408,\n",
              "  'text': 'downward trend in the number of\\nconfirmed new cases during February'},\n",
              " {'answer_end': 1701,\n",
              "  'answer_start': 1439,\n",
              "  'text': 'The value of R 0 was estimated of 2.30 from reservoir to person and 3.58 from person to person which means that the expected number of secondary infections that result from introducing a single infected individual into an otherwise susceptible population was 3.5'},\n",
              " {'answer_end': 3804,\n",
              "  'answer_start': 3688,\n",
              "  'text': 'stimulate expression of numerous host inflammatory factors that support recruitment and activation of immune cells. '},\n",
              " {'answer_end': 2082,\n",
              "  'answer_start': 2039,\n",
              "  'text': 'uding birds, farm animals, pets, camels, an'},\n",
              " {'answer_end': 791,\n",
              "  'answer_start': 740,\n",
              "  'text': 'Control measures and changes in population behaviou'},\n",
              " {'answer_end': 7656,\n",
              "  'answer_start': 7552,\n",
              "  'text': 'the cells release antiviral factors and cytokines that alter local airway inflammation and airway niche '},\n",
              " {'answer_end': 5334, 'answer_start': 5318, 'text': 'the upper airway'},\n",
              " {'answer_end': 4428,\n",
              "  'answer_start': 4110,\n",
              "  'text': 'a confirmed case was a person with laboratory confirmation of SARS-CoV-2 infection (ECDC recommended two separate SARS-CoV-2 RT-PCR tests), irrespective of clinical signs and symptoms, whereas a probable case was a suspect case for whom testing for SARS-CoV-2 was inconclusive or positive using a pan-coronavirus assay'},\n",
              " {'answer_end': 13508,\n",
              "  'answer_start': 13303,\n",
              "  'text': 'These 3 cytokines then work in concert to activate ILC2s to further secrete type 2 cytokines IL-4, IL-5, and IL-13 which further aggravate the type 2 inflammation in the airway causing acute exacerbation ('},\n",
              " {'answer_end': 13753, 'answer_start': 13747, 'text': '100 % '},\n",
              " {'answer_end': 58277, 'answer_start': 58270, 'text': ' Riyadh'},\n",
              " {'answer_end': 1913,\n",
              "  'answer_start': 1837,\n",
              "  'text': 'fever, encephalitis, pneumonia, and severe acute respiratory syndrome (SARS)'},\n",
              " {'answer_end': 5132,\n",
              "  'answer_start': 5054,\n",
              "  'text': 'the majority of HepaRG cells were found to express NPCT but not to be infected'},\n",
              " {'answer_end': 8019,\n",
              "  'answer_start': 7949,\n",
              "  'text': 'the simultaneous (or nearly simultaneous) infection\\nof humans and swin'},\n",
              " {'answer_end': 8026,\n",
              "  'answer_start': 7949,\n",
              "  'text': ' the true number may be in the millions and many harbour zoonotic potential. '},\n",
              " {'answer_end': 18769,\n",
              "  'answer_start': 18441,\n",
              "  'text': 'The equipment in the contaminated area are wiped with\\n2000mg/L chlorine-containing disinfectant. The DR and CT gantry in the contaminated\\narea are wiped with 75% ethanol. The equipment in the buffer area is wiped with\\n500-1000mg/L chlorine-containing disinfectant or alcohol-containing disposable\\ndisinfectant wipes twice a day.'},\n",
              " {'answer_end': 24466,\n",
              "  'answer_start': 24384,\n",
              "  'text': 'to understand the\\nextent and dynamic evolution of lung lesions induced by COVID-19'},\n",
              " {'answer_end': 17191,\n",
              "  'answer_start': 16987,\n",
              "  'text': ' Indoor air pollution from use of solid or biomass fuels increases odds of pneumonia by 1.6 times; lack of measles vaccination by the end of the first year of age increases odds of pneumonia by 1.8 times '},\n",
              " {'answer_end': 7416,\n",
              "  'answer_start': 7346,\n",
              "  'text': 'because it determines the strength of public health response required.'},\n",
              " {'answer_end': 34604,\n",
              "  'answer_start': 34551,\n",
              "  'text': 'to identify systemic mechanisms of acute exacerbation'},\n",
              " {'answer_end': 24991,\n",
              "  'answer_start': 24546,\n",
              "  'text': ' low-dose chest CT has been\\nwidely used in the screening of early lung cancer. It is well known that many early lung cancers\\nare ground-glass opacities (GGO), so we believe that low-dose screening is also applicable for\\nCOVID-19. In addition, considering the rapid development of COVID-19, many CT\\n14\\nexaminations may be conducted in the same individual to monitor disease progress. Low-dose\\nscanning can reduce the radiation damage to patients.'},\n",
              " {'answer_end': 385,\n",
              "  'answer_start': 353,\n",
              "  'text': 'interferon-induced transmembrane'},\n",
              " {'answer_end': 3630,\n",
              "  'answer_start': 3310,\n",
              "  'text': 'The incidence in high-income countries is estimated at 0.015 episodes per child year, compared to 0.22 episodes per child year in low-and middle-income countries [3] . On average, 1 in 66 children in high-income countries is affected by pneumonia per year, compared to 1 in 5 children in low-and middle-income countries.'},\n",
              " {'answer_end': 9769, 'answer_start': 9753, 'text': '32uC, 37uC, 42uC'},\n",
              " {'answer_end': 9840,\n",
              "  'answer_start': 9634,\n",
              "  'text': 'the current strategy for the detection of viral RNA in oral swabs used for 2019-nCoV diagnosis is not perfect. The virus may be present in anal swabs or blood of patients when oral swabs detection negative.'},\n",
              " {'answer_end': 1578,\n",
              "  'answer_start': 1514,\n",
              "  'text': '469 (95% CI: 403&minus;540) unreported cases from 1 to 15 Januar'},\n",
              " {'answer_end': 2727,\n",
              "  'answer_start': 2534,\n",
              "  'text': 'The most frequently reported viruses include respiratory syncytial virus (RSV), influenza viruses A and B (IAV, IBV), parainfluenza viruses (PIVs), human rhinovirus (HRV) and adenovirus (ADV), '},\n",
              " {'answer_end': 706, 'answer_start': 701, 'text': 'three'},\n",
              " {'answer_end': 50417, 'answer_start': 50413, 'text': '16 %'},\n",
              " {'answer_end': 4656,\n",
              "  'answer_start': 4627,\n",
              "  'text': 'Military barracks and prisons'},\n",
              " {'answer_end': 4277,\n",
              "  'answer_start': 4174,\n",
              "  'text': 'HRVs, coronaviruses (CoV), influenza, respiratory syncytial virus (RSV) and parainfluenza viruses (PIV)'},\n",
              " {'answer_end': 2034, 'answer_start': 2026, 'text': '42 years'},\n",
              " {'answer_end': 33021,\n",
              "  'answer_start': 32697,\n",
              "  'text': ' infiltrating cells in response to the infection such as neutrophils will also trigger respiratory burst as a means of increasing the ROS in the infected region. The increased ROS and oxidative stress in the local environment may serve as a trigger to promote inflammation thereby aggravating the inflammation in the airway '},\n",
              " {'answer_end': 46328, 'answer_start': 46313, 'text': 'intra-familial,'},\n",
              " {'answer_end': 6407,\n",
              "  'answer_start': 6383,\n",
              "  'text': 'an estimated 7.3 billion'},\n",
              " {'answer_end': 14971,\n",
              "  'answer_start': 14734,\n",
              "  'text': ' to complement surveillance data to build knowledge on the infectious period, modes of transmission, basic and effective reproduction numbers, and effectiveness of prevention and case management options also in settings outside of China.'},\n",
              " {'answer_end': 14109,\n",
              "  'answer_start': 14050,\n",
              "  'text': 'by targeting the salvage pathway of pyrimidine biosynthesis'},\n",
              " {'answer_end': 17378,\n",
              "  'answer_start': 17315,\n",
              "  'text': 'in weak type 1 inflammation-inducing viruses such as RV and RSV'},\n",
              " {'answer_end': 7775, 'answer_start': 7759, 'text': 'January 30, 2020'},\n",
              " {'answer_end': 11602,\n",
              "  'answer_start': 11528,\n",
              "  'text': 'access areas for medical personnel and a dressing area for technologists. '},\n",
              " {'answer_end': 2655,\n",
              "  'answer_start': 2492,\n",
              "  'text': 'by manipulating multifunctional or multitasking host cell proteins (MMHPs), thereby maximising host subversion and viral infectivity at minimal informational cost.'},\n",
              " {'answer_end': 1889,\n",
              "  'answer_start': 1643,\n",
              "  'text': 'MERS progresses more rapidly to respiratory failure and acute kidney injury (it also has an affinity for growth in kidney cells under laboratory conditions), is more frequently reported in patients with underlying disease and is more often fatal.'},\n",
              " {'answer_end': 9637, 'answer_start': 9620, 'text': 'denaturation step'},\n",
              " {'answer_end': 3194,\n",
              "  'answer_start': 2996,\n",
              "  'text': '(1) vaccine candidates that elicit protective antibodies; (2) antibodies that prevent disease when given prophylactically; and (3) antibodies that can be given as therapy after the onset of disease.'},\n",
              " {'answer_end': 22193,\n",
              "  'answer_start': 22016,\n",
              "  'text': 'disruption of tight junctions with the resulting increased rate of viral infiltration. This also increases the chances of allergens coming into contact with airway immune cells.'},\n",
              " {'answer_end': 3319,\n",
              "  'answer_start': 3203,\n",
              "  'text': ' to complete a WHO standard COVID-19 case report form for all confirmed and probable cases according to WHO criteria'},\n",
              " {'answer_end': 584,\n",
              "  'answer_start': 548,\n",
              "  'text': 'the isolation of respiratory viruses'},\n",
              " {'answer_end': 11250,\n",
              "  'answer_start': 10729,\n",
              "  'text': \"significant resources for contact tracing and quarantine, and countries should be prepared to allocate considerable public health resources during the containment phase, should local clusters emerge in their population. In addition, prompt sharing of information on cases and contacts through international notification systems such as the International Health Regulations (IHR) mechanism and the European Commission's European Early Warning and Response System is essential to contain international spread of infection.\\n\"},\n",
              " {'answer_end': 22158,\n",
              "  'answer_start': 22091,\n",
              "  'text': 'data gathering, collaboration, needs assessment, and expert advice '},\n",
              " {'answer_end': 29043, 'answer_start': 29025, 'text': 'MicroRNAs (miRNAs)'},\n",
              " {'answer_end': 2497,\n",
              "  'answer_start': 2352,\n",
              "  'text': ' in both the upper airway (such as chronic rhinosinusitis, CRS) and lower airway (such as asthma and chronic obstructive pulmonary disease, COPD)'},\n",
              " {'answer_end': 38796, 'answer_start': 38778, 'text': 'virus or viral RNA'},\n",
              " {'answer_end': 1535,\n",
              "  'answer_start': 1509,\n",
              "  'text': '2-bromopalmitic acid (2BP)'},\n",
              " {'answer_end': 6561,\n",
              "  'answer_start': 6507,\n",
              "  'text': 'Rhinovirus (RV) and respiratory syncytial virus (RSV) '},\n",
              " {'answer_end': 9958,\n",
              "  'answer_start': 9877,\n",
              "  'text': 'viruses evoke an inflammatory response as a means of counteracting the infection.'},\n",
              " {'answer_end': 1827,\n",
              "  'answer_start': 1792,\n",
              "  'text': 'he case was diagnosed on 1 February'},\n",
              " {'answer_end': 12202,\n",
              "  'answer_start': 11594,\n",
              "  'text': 'are more neutrophilic in nature due to the expression of neutrophil chemoattractants such as CXCL9, CXCL10, and CXCL11 (Cukic et al., 2012; Brightling and Greening, 2019) . The pathology of these airway diseases is characterized by airway remodeling due to the presence of remodeling factors such as matrix metalloproteinases (MMPs) released from infiltrating neutrophils (Linden et al., 2019) . Viral infections in such conditions will then cause increase neutrophilic activation; worsening the symptoms and airway remodeling in the airway thereby exacerbating COPD, CRSsNP and even CRSwNP in certain cases '},\n",
              " {'answer_end': 4927,\n",
              "  'answer_start': 4774,\n",
              "  'text': 'it promotes cytokine production, the induction of cell surface co-stimulatory molecules, and facilitates the cross-presentation of antigen by these cells'},\n",
              " {'answer_end': 5998,\n",
              "  'answer_start': 5756,\n",
              "  'text': 'how viral induced changes to the upper airway may contribute to chronic airway inflammatory disease exacerbations, to provide a unified elucidation of the potential exacerbation mechanisms initiated from predominantly upper airway infections.'},\n",
              " {'answer_end': 1369,\n",
              "  'answer_start': 1328,\n",
              "  'text': 'they inhibit cellular DNA/RNA polymerases'},\n",
              " {'answer_end': 67322, 'answer_start': 67310, 'text': ' nearly 40 %'},\n",
              " {'answer_end': 5316,\n",
              "  'answer_start': 5246,\n",
              "  'text': 'dysregulated inflammation in most chronic airway inflammatory diseases'},\n",
              " {'answer_end': 8570,\n",
              "  'answer_start': 8467,\n",
              "  'text': 'high proportion of\\ninfections not detected by health systems”7 and regular changes in testing policies,'},\n",
              " {'answer_end': 9642,\n",
              "  'answer_start': 9606,\n",
              "  'text': 'by applying JEV, YF, and DV cultures'},\n",
              " {'answer_end': 2141,\n",
              "  'answer_start': 2097,\n",
              "  'text': 'semi-mechanistic Bayesian hierarchical model'},\n",
              " {'answer_end': 3881,\n",
              "  'answer_start': 3841,\n",
              "  'text': 'A stratified multi stage random sampling'},\n",
              " {'answer_end': 5160,\n",
              "  'answer_start': 4790,\n",
              "  'text': ' Rare memory B cells producing antibodies specific for the EBV fusion machinery have been isolated; these can neutralize both B cell and epithelial cell infection (20). A new paradigm in malaria vaccine development is also emerging with the discovery of IgM+ and IgD+ memory B cells targeting the Merozoite Surface Protein 1, that rapidly respond to malaria re-infection'},\n",
              " {'answer_end': 9346,\n",
              "  'answer_start': 9226,\n",
              "  'text': 'antimicrobial resistance, global health and health security, drug repurposing and translational research infrastructure,'},\n",
              " {'answer_end': 2924, 'answer_start': 2923, 'text': '9'},\n",
              " {'answer_end': 4551,\n",
              "  'answer_start': 4315,\n",
              "  'text': 'Influenza also performs fusion through the stem region of the hemagglutinin protein, and the identification of B cells that target this relatively conserved site has spurred research on the development of a universal influenza vaccine ('},\n",
              " {'answer_end': 33620,\n",
              "  'answer_start': 33515,\n",
              "  'text': ' also may alter the management of the disease or confer resistance toward treatments that worked before. '},\n",
              " {'answer_end': 5279,\n",
              "  'answer_start': 5011,\n",
              "  'text': 'the virus transmitted among the bats, and then transmitted to unknown hosts (probably some wild animals). The hosts were hunted and sent to the seafood market which was defined as the reservoir of the virus. People exposed to the market got the risks of the infection '},\n",
              " {'answer_end': 674,\n",
              "  'answer_start': 645,\n",
              "  'text': 'long noncoding RNAs (lncRNAs)'},\n",
              " {'answer_end': 10154,\n",
              "  'answer_start': 10025,\n",
              "  'text': 'delivered to their site of action, be stable, enter the target cells, and be present in the cytoplasm at sufficient concentration'},\n",
              " {'answer_end': 5279,\n",
              "  'answer_start': 5079,\n",
              "  'text': 'it promotes germinal center formation, immunoglobulin (Ig) isotype switching, somatic hypermutation to enhance antigen affinity, and lastly, the formation of long-lived plasma cells and memory B cells'},\n",
              " {'answer_end': 625,\n",
              "  'answer_start': 563,\n",
              "  'text': 'The novel coronavirus (2019-nCoV) infection caused pneumonia. '},\n",
              " {'answer_end': 3818,\n",
              "  'answer_start': 3756,\n",
              "  'text': 'up to 81% of severe pneumonia deaths occur outside a hospital '},\n",
              " {'answer_end': 7661,\n",
              "  'answer_start': 7596,\n",
              "  'text': 'the ancient lineage of nucleocytosolic large DNA viruses (NCLDVs)'},\n",
              " {'answer_end': 22557,\n",
              "  'answer_start': 22206,\n",
              "  'text': ' IFV infection was found to induce oncostatin M (OSM) which causes tight junction opening (Pothoven et al., 2015; Tian et al., 2018) . Similarly, RV and RSV infections usually cause tight junction opening which may also increase the infiltration rate of eosinophils and thus worsening of the classical symptoms of chronic airway inflammatory diseases '},\n",
              " {'answer_end': 21479,\n",
              "  'answer_start': 21425,\n",
              "  'text': 'if only mild disease or asymptomatic infection results'},\n",
              " {'answer_end': 1780,\n",
              "  'answer_start': 1661,\n",
              "  'text': 'the importance of the virome within the airway microbiome environment and its impact on subsequent bacterial infection.'},\n",
              " {'answer_end': 31557,\n",
              "  'answer_start': 31344,\n",
              "  'text': ' viral infections such as RV and RSV alters DNA methylation and histone modifications in the airway epithelium which may alter inflammatory responses, driving chronic airway inflammatory diseases and exacerbations'},\n",
              " {'answer_end': 419, 'answer_start': 396, 'text': ' from person to person,'},\n",
              " {'answer_end': 6254,\n",
              "  'answer_start': 6208,\n",
              "  'text': ' five to six days, ranging from two to 16 days'},\n",
              " {'answer_end': 5523,\n",
              "  'answer_start': 5473,\n",
              "  'text': 'insulin [39] and low-molecular-weight heparin [40]'}]"
            ]
          },
          "metadata": {},
          "execution_count": 41
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ln5fFH5LIcIW"
      },
      "source": [
        "## Tokenizing"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "2YMCrUwoIYjl"
      },
      "source": [
        "# Initializing the tokenizer to be used\n",
        "model_name = \"distilbert-base-uncased\" \n",
        "tokenizer = DistilBertTokenizerFast.from_pretrained(model_name, model_max_length=7000)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "yKzEon-bIvnT"
      },
      "source": [
        "# Tokenizing\n",
        "train_encodings = tokenizer(train_contexts, train_questions, padding=True, truncation=True)\n",
        "val_encodings = tokenizer(val_contexts, val_questions, padding=True, truncation=True)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "wL3sC6FuJFTT"
      },
      "source": [
        "# function to add end_positions to the encodings\n",
        "def add_token_positions(encodings, answers):\n",
        "  start_positions = []\n",
        "  end_positions = []\n",
        "  for i in range(len(answers)):\n",
        "    start_positions.append(encodings.char_to_token(i, answers[i][\"answer_start\"]))\n",
        "    end_positions.append(encodings.char_to_token(i, answers[i][\"answer_end\"]))\n",
        "\n",
        "    go_back = 1\n",
        "    while start_positions[-1] is None:\n",
        "      start_positions[-1] =  encodings.char_to_token(i, answers[i]['answer_start'] - go_back)\n",
        "      go_back += 1\n",
        "\n",
        "    # go_back = 1\n",
        "    # while end_positions[-1] is None:\n",
        "    #   end_positions[-1] = encodings.char_to_token(i, answers[i]['answer_end'] - go_back)\n",
        "    #   go_back += 1\n",
        "    if end_positions[-1] is None:\n",
        "      end_positions[-1] = tokenizer.model_max_length\n",
        "\n",
        "  encodings.update({\n",
        "      \"start_positions\": start_positions,\n",
        "      \"end_positions\": end_positions\n",
        "  })"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "OglUX0WmKBPG"
      },
      "source": [
        "# add end_positions in the encodings\n",
        "add_token_positions(train_encodings, train_answers)\n",
        "add_token_positions(val_encodings, val_answers)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "1CMGFHLhsQOT",
        "outputId": "d6c326a8-fecc-48db-e3b7-00b07366e51b"
      },
      "source": [
        "print(train_encodings[\"start_positions\"])\n",
        "print(train_encodings[\"end_positions\"])"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[495, 501, 472, 496, 503, 503, 450, 496, 496, 504, 488, 490, 500, 491, 498, 498, 500, 492, 500, 280, 309, 493, 495, 503, 504, 487, 494, 500, 494, 499, 499, 492, 487, 500, 496, 493, 500, 498, 500, 500, 490, 492, 495, 500, 122, 499, 496, 489, 497, 501, 486, 494, 503, 502, 492, 239, 499, 494, 503, 499, 494, 496, 235, 500, 496, 495, 233, 487, 498, 500, 501, 486, 492, 498, 502, 496, 502, 494, 482, 495, 491, 493, 490, 503, 43, 487, 495, 486, 484, 491, 504, 496, 498, 177, 488, 494, 494, 500, 501, 502, 496, 299, 500, 500, 486, 502, 498, 502, 227, 495, 493, 484, 165, 501, 499, 499, 501, 423, 499, 495, 498, 492, 497, 494, 300, 496, 499, 488, 495, 498, 498, 492, 503, 494, 503, 487, 498, 490, 495, 488, 372, 495, 108, 493, 498, 490, 493, 495, 212, 502, 489, 458, 492, 497, 502, 286, 493, 496, 505, 493, 491, 503, 151, 500, 489, 497, 493, 496, 496, 171, 492, 492, 491, 498, 497, 495, 493, 490, 212, 487, 367, 489, 300, 492, 499, 492, 502, 492, 504, 498, 497, 459, 479, 502, 478, 496, 491, 496, 497, 250, 499, 499, 500, 501, 492, 489, 498, 421, 496, 487, 493, 500, 499, 493, 495, 494, 497, 488, 497, 498, 484, 467, 500, 486, 484, 501, 492, 485, 138, 493, 503, 496, 495, 492, 485, 486, 495, 497, 500, 493, 497, 497, 495, 500, 412, 501, 491, 501, 497, 497, 403, 468, 482, 497, 433, 305, 500, 502, 502, 494, 498, 497, 494, 496, 501, 494, 498, 499, 492, 382, 491, 500, 499, 490, 494, 503, 502, 492, 339, 495, 409, 495, 496, 500, 495, 498, 486, 497, 500, 433, 502, 503, 488, 495, 502, 186, 495, 148, 496, 500, 502, 501, 500, 502, 491, 501, 497, 505, 488, 491, 498, 495, 497, 489, 498, 501, 493, 502, 495, 490, 496, 499, 495, 501, 259, 502, 495, 501, 497, 477, 501, 495, 492, 499, 242, 491, 501, 495, 495, 500, 497, 499, 499, 501, 500, 496, 202, 489, 493, 490, 495, 492, 429, 498, 498, 499, 493, 496, 503, 492, 480, 497, 125, 490, 498, 500, 495, 500, 498, 498, 496, 494, 498, 498, 503, 490, 494, 500, 493, 502, 494, 499, 496, 496, 298, 492, 504, 497, 496, 486, 492, 171, 500, 501, 489, 499, 126, 503, 498, 503, 494, 497, 497, 490, 495, 504, 496, 483, 491, 352, 501, 492, 499, 269, 482, 499, 496, 501, 493, 497, 490, 492, 502, 503, 499, 502, 494, 501, 497, 492, 486, 492, 304, 496, 493, 486, 432, 486, 503, 490, 168, 499, 486, 491, 497, 497, 503, 490, 502, 501, 497, 223, 499, 495, 211, 489, 496, 484, 493, 500, 499, 502, 311, 486, 454, 309, 496, 361, 501, 447, 209, 502, 494, 501, 498, 477, 495, 476, 494, 497, 493, 488, 500, 498, 491, 498, 498, 137, 495, 500, 495, 396, 491, 225, 491, 502, 498, 502, 493, 495, 483, 503, 499, 503, 483, 502, 502, 413, 499, 494, 501, 496, 200, 501, 496, 493, 503, 499, 434, 497, 504, 408, 485, 495, 502, 355, 495, 499, 499, 502, 495, 503, 497, 497, 492, 492, 490, 503, 218, 489, 497, 219, 493, 502, 502, 494, 394, 502, 142, 498, 492]\n",
            "[512, 512, 475, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 129, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 47, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 168, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 306, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 140, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 154, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 308, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 479, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 450, 315, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 385, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 467, 512, 512, 512, 512, 512, 512, 512, 175, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 491, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 456, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 320, 512, 512, 512, 512, 512, 512, 184, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 442, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 232, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 321, 512, 512, 342, 512, 409, 512, 458, 215, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 418, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 205, 512, 512, 512, 512, 512, 447, 512, 512, 415, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 512, 233, 512, 512, 512, 512, 512, 512, 512, 512, 512]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "xK-wWN4vnOei",
        "outputId": "ab853666-72f9-4cf6-e418-7ae0e18e9f55"
      },
      "source": [
        "train_encodings[\"start_positions\"].count(None)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "0"
            ]
          },
          "metadata": {},
          "execution_count": 47
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "J_ZrF5U2Tm8j"
      },
      "source": [
        "## Transform into a PyTorch dataset"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "YlJLYCxwOQZ0"
      },
      "source": [
        "# class to create datasets from encodings\n",
        "class QADataset(torch.utils.data.Dataset):\n",
        "  def __init__(self, encodings):\n",
        "    self.encodings = encodings\n",
        "\n",
        "  def __getitem__(self, idx):\n",
        "    return {key: torch.tensor(val[idx]) for key, val in self.encodings.items()}\n",
        "\n",
        "  def __len__(self):\n",
        "    return len(self.encodings.input_ids)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "FnHUJXLDRngB"
      },
      "source": [
        "train_dataset = QADataset(train_encodings)\n",
        "val_dataset = QADataset(val_encodings)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "KFOiJjP5UO3Q"
      },
      "source": [
        "## Fine tuning the pretrained model"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "cS5YVGkMjZhv",
        "outputId": "4254db7a-b4ad-4853-eb8b-7a7c19e8a170"
      },
      "source": [
        "model = DistilBertForQuestionAnswering.from_pretrained(model_name)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Some weights of the model checkpoint at distilbert-base-uncased were not used when initializing DistilBertForQuestionAnswering: ['vocab_transform.weight', 'vocab_layer_norm.weight', 'vocab_transform.bias', 'vocab_projector.bias', 'vocab_layer_norm.bias', 'vocab_projector.weight']\n",
            "- This IS expected if you are initializing DistilBertForQuestionAnswering from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).\n",
            "- This IS NOT expected if you are initializing DistilBertForQuestionAnswering from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).\n",
            "Some weights of DistilBertForQuestionAnswering were not initialized from the model checkpoint at distilbert-base-uncased and are newly initialized: ['qa_outputs.bias', 'qa_outputs.weight']\n",
            "You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "UyhhsBJgRyXy"
      },
      "source": [
        "device = torch.device(\"cuda\") if torch.cuda.is_available() else torch.device(\"cpu\")\n",
        "model.to(device)\n",
        "model.train()\n",
        "optim = AdamW(model.parameters(), lr=5e-5)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "nWanQdr-krWL"
      },
      "source": [
        "train_loader = DataLoader(train_dataset, batch_size=16, shuffle=True)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "CG1gg3qtl1Px",
        "outputId": "cc8ef1a6-c594-4e8f-b269-7f3b484199fd"
      },
      "source": [
        "train_encodings.keys()"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "dict_keys(['input_ids', 'attention_mask', 'start_positions', 'end_positions'])"
            ]
          },
          "metadata": {},
          "execution_count": 53
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Ab7Oy7NplM-r",
        "outputId": "e4dd1116-98ad-4441-849d-8f73a1bbdecc"
      },
      "source": [
        "for epoch in range(5):\n",
        "  loop = tqdm(train_loader)\n",
        "  for batch in loop:\n",
        "    optim.zero_grad()\n",
        "\n",
        "    input_ids = batch[\"input_ids\"].to(device)\n",
        "    attention_mask = batch[\"attention_mask\"].to(device)\n",
        "    start_positions = batch[\"start_positions\"].to(device)\n",
        "    end_positions = batch[\"end_positions\"].to(device)\n",
        "\n",
        "    outputs = model(input_ids, \n",
        "                    attention_mask=attention_mask, \n",
        "                    start_positions = start_positions,\n",
        "                    end_positions = end_positions)\n",
        "    \n",
        "    loss = outputs[0]\n",
        "    loss.backward()\n",
        "    optim.step()\n",
        "    loop.set_description(f\"Epoch {epoch}\")\n",
        "    loop.set_postfix(loss = loss.item())"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Epoch 0: 100%|██████████| 35/35 [00:48<00:00,  1.38s/it, loss=3.77]\n",
            "Epoch 1: 100%|██████████| 35/35 [00:49<00:00,  1.40s/it, loss=0.067]\n",
            "Epoch 2: 100%|██████████| 35/35 [00:49<00:00,  1.40s/it, loss=0.0885]\n",
            "Epoch 3: 100%|██████████| 35/35 [00:49<00:00,  1.40s/it, loss=0.0813]\n",
            "Epoch 4: 100%|██████████| 35/35 [00:49<00:00,  1.40s/it, loss=0.49]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "d5-2eVXFm5If",
        "outputId": "4fd1e4d9-8d7f-4702-ebb7-a7fa780c21de"
      },
      "source": [
        "model.eval()"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "DistilBertForQuestionAnswering(\n",
              "  (distilbert): DistilBertModel(\n",
              "    (embeddings): Embeddings(\n",
              "      (word_embeddings): Embedding(30522, 768, padding_idx=0)\n",
              "      (position_embeddings): Embedding(512, 768)\n",
              "      (LayerNorm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "      (dropout): Dropout(p=0.1, inplace=False)\n",
              "    )\n",
              "    (transformer): Transformer(\n",
              "      (layer): ModuleList(\n",
              "        (0): TransformerBlock(\n",
              "          (attention): MultiHeadSelfAttention(\n",
              "            (dropout): Dropout(p=0.1, inplace=False)\n",
              "            (q_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (k_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (v_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (out_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "          )\n",
              "          (sa_layer_norm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "          (ffn): FFN(\n",
              "            (dropout): Dropout(p=0.1, inplace=False)\n",
              "            (lin1): Linear(in_features=768, out_features=3072, bias=True)\n",
              "            (lin2): Linear(in_features=3072, out_features=768, bias=True)\n",
              "          )\n",
              "          (output_layer_norm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "        )\n",
              "        (1): TransformerBlock(\n",
              "          (attention): MultiHeadSelfAttention(\n",
              "            (dropout): Dropout(p=0.1, inplace=False)\n",
              "            (q_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (k_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (v_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (out_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "          )\n",
              "          (sa_layer_norm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "          (ffn): FFN(\n",
              "            (dropout): Dropout(p=0.1, inplace=False)\n",
              "            (lin1): Linear(in_features=768, out_features=3072, bias=True)\n",
              "            (lin2): Linear(in_features=3072, out_features=768, bias=True)\n",
              "          )\n",
              "          (output_layer_norm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "        )\n",
              "        (2): TransformerBlock(\n",
              "          (attention): MultiHeadSelfAttention(\n",
              "            (dropout): Dropout(p=0.1, inplace=False)\n",
              "            (q_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (k_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (v_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (out_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "          )\n",
              "          (sa_layer_norm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "          (ffn): FFN(\n",
              "            (dropout): Dropout(p=0.1, inplace=False)\n",
              "            (lin1): Linear(in_features=768, out_features=3072, bias=True)\n",
              "            (lin2): Linear(in_features=3072, out_features=768, bias=True)\n",
              "          )\n",
              "          (output_layer_norm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "        )\n",
              "        (3): TransformerBlock(\n",
              "          (attention): MultiHeadSelfAttention(\n",
              "            (dropout): Dropout(p=0.1, inplace=False)\n",
              "            (q_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (k_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (v_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (out_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "          )\n",
              "          (sa_layer_norm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "          (ffn): FFN(\n",
              "            (dropout): Dropout(p=0.1, inplace=False)\n",
              "            (lin1): Linear(in_features=768, out_features=3072, bias=True)\n",
              "            (lin2): Linear(in_features=3072, out_features=768, bias=True)\n",
              "          )\n",
              "          (output_layer_norm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "        )\n",
              "        (4): TransformerBlock(\n",
              "          (attention): MultiHeadSelfAttention(\n",
              "            (dropout): Dropout(p=0.1, inplace=False)\n",
              "            (q_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (k_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (v_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (out_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "          )\n",
              "          (sa_layer_norm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "          (ffn): FFN(\n",
              "            (dropout): Dropout(p=0.1, inplace=False)\n",
              "            (lin1): Linear(in_features=768, out_features=3072, bias=True)\n",
              "            (lin2): Linear(in_features=3072, out_features=768, bias=True)\n",
              "          )\n",
              "          (output_layer_norm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "        )\n",
              "        (5): TransformerBlock(\n",
              "          (attention): MultiHeadSelfAttention(\n",
              "            (dropout): Dropout(p=0.1, inplace=False)\n",
              "            (q_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (k_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (v_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "            (out_lin): Linear(in_features=768, out_features=768, bias=True)\n",
              "          )\n",
              "          (sa_layer_norm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "          (ffn): FFN(\n",
              "            (dropout): Dropout(p=0.1, inplace=False)\n",
              "            (lin1): Linear(in_features=768, out_features=3072, bias=True)\n",
              "            (lin2): Linear(in_features=3072, out_features=768, bias=True)\n",
              "          )\n",
              "          (output_layer_norm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)\n",
              "        )\n",
              "      )\n",
              "    )\n",
              "  )\n",
              "  (qa_outputs): Linear(in_features=768, out_features=2, bias=True)\n",
              "  (dropout): Dropout(p=0.1, inplace=False)\n",
              ")"
            ]
          },
          "metadata": {},
          "execution_count": 55
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "eoI2eyeHzRGg",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "abdb177f-4565-45b1-f093-521d5dba377a"
      },
      "source": [
        "model_path = \"model/custom_distil_bert\"\n",
        "\n",
        "model.save_pretrained(model_path)\n",
        "tokenizer.save_pretrained(model_path)"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "('model/custom_distil_bert/tokenizer_config.json',\n",
              " 'model/custom_distil_bert/special_tokens_map.json',\n",
              " 'model/custom_distil_bert/vocab.txt',\n",
              " 'model/custom_distil_bert/added_tokens.json',\n",
              " 'model/custom_distil_bert/tokenizer.json')"
            ]
          },
          "metadata": {},
          "execution_count": 56
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "b2clUHbjb84R"
      },
      "source": [
        "model = DistilBertForQuestionAnswering.from_pretrained(model_path)\n",
        "tokenizer = AutoTokenizer.from_pretrained(model_path)\n",
        "\n",
        "nlp = pipeline('question-answering', model=model, tokenizer=tokenizer)"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "_P9FBW2P0HDz",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4ce37ea4-3417-4a70-9eb6-17c6e730945f"
      },
      "source": [
        "context = val_contexts[16]\n",
        "question = val_questions[16]\n",
        "print(question)\n",
        "print(val_answers[16])\n",
        "print(context)\n",
        "print(nlp({\n",
        "    \"question\" : question,\n",
        "     \"context\" : context\n",
        "}))"
      ],
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "What was mortality in  South Korea from MERS disease?\n",
            "{'text': 'ranged from 7 % among younger age groups to 40 % among those aged 60 years and above', 'answer_start': 8955, 'answer_end': 9039}\n",
            "MERS coronavirus: diagnostics, epidemiology and transmission\n",
            "\n",
            "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4687373/\n",
            "\n",
            "SHA: f6fcf1a99cbd073c5821d1c4ffa3f2c6daf8ae29\n",
            "\n",
            "Authors: Mackay, Ian M.; Arden, Katherine E.\n",
            "Date: 2015-12-22\n",
            "DOI: 10.1186/s12985-015-0439-5\n",
            "License: cc-by\n",
            "\n",
            "Abstract: The first known cases of Middle East respiratory syndrome (MERS), associated with infection by a novel coronavirus (CoV), occurred in 2012 in Jordan but were reported retrospectively. The case first to be publicly reported was from Jeddah, in the Kingdom of Saudi Arabia (KSA). Since then, MERS-CoV sequences have been found in a bat and in many dromedary camels (DC). MERS-CoV is enzootic in DC across the Arabian Peninsula and in parts of Africa, causing mild upper respiratory tract illness in its camel reservoir and sporadic, but relatively rare human infections. Precisely how virus transmits to humans remains unknown but close and lengthy exposure appears to be a requirement. The KSA is the focal point of MERS, with the majority of human cases. In humans, MERS is mostly known as a lower respiratory tract (LRT) disease involving fever, cough, breathing difficulties and pneumonia that may progress to acute respiratory distress syndrome, multiorgan failure and death in 20 % to 40 % of those infected. However, MERS-CoV has also been detected in mild and influenza-like illnesses and in those with no signs or symptoms. Older males most obviously suffer severe disease and MERS patients often have comorbidities. Compared to severe acute respiratory syndrome (SARS), another sometimes- fatal zoonotic coronavirus disease that has since disappeared, MERS progresses more rapidly to respiratory failure and acute kidney injury (it also has an affinity for growth in kidney cells under laboratory conditions), is more frequently reported in patients with underlying disease and is more often fatal. Most human cases of MERS have been linked to lapses in infection prevention and control (IPC) in healthcare settings, with approximately 20 % of all virus detections reported among healthcare workers (HCWs) and higher exposures in those with occupations that bring them into close contact with camels. Sero-surveys have found widespread evidence of past infection in adult camels and limited past exposure among humans. Sensitive, validated reverse transcriptase real-time polymerase chain reaction (RT-rtPCR)-based diagnostics have been available almost from the start of the emergence of MERS. While the basic virology of MERS-CoV has advanced over the past three years, understanding of the interplay between camel, environment, and human remains limited. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (doi:10.1186/s12985-015-0439-5) contains supplementary material, which is available to authorized users.\n",
            "\n",
            "Text: An email from Dr Ali Mohamed Zaki, an Egyptian virologist working at the Dr Soliman Fakeeh Hospital in Jeddah in the Kingdom of Saudi Arabia (KSA) announced the first culture of a new coronavirus to the world. The email was published on the website of the professional emerging diseases (ProMED) network on 20 th September 2012 [1] (Fig. 1) and described the first reported case, a 60 year old man from Bisha in the KSA. This information led to the rapid discovery of a second case of the virus, this time in an ill patient in the United Kingdom, who had been transferred from Qatar for care [2] . The new virus was initially called novel coronavirus (nCoV) and subsequentlty entitled the Middle East respiratoy syndrome coronavirus (MERS-CoV). As of 2 nd of September 2015, there have been 1,493 detections of viral RNA or virus-specific antibodies across 26 countries (Additional file 1: Figure S1 ) confirmed by the World Health Organization (WHO), with over a third of the positive people dying (at least 527, 35 %) [3] .\n",
            "\n",
            "Since that first report, a slow discovery process over the following two to three years revealed a virus that had infected over 90 % of adult dromedary camels (DC; Camelus dromedarius) in the KSA [4] , also DCs across the Arabian Peninsula and parts of Africa that are a source of DC imports for the KSA [5] . To date, MERS-CoV has not been detected in DCs tested in zoos or herds from other parts of the world [6] [7] [8] [9] . Occasionally, virus is transmitted from infected DCs to exposed humans. Subsequent transmission to other humans requires relatively close and prolonged exposure [10] .\n",
            "\n",
            "The first viral isolate was patented and concerns were raised that this would restrict access to both the virus and to viral diagnostics [11, 12] . However, sensitive, validated reverse transcriptase real-time polymerase chain reaction (RT-rtPCR)-based diagnostics were quickly described and virus was made freely available subject to routine biosafety considerations [13] . Subsequent epidemiology and research has identified the cell receptor as exopeptidase dipeptidyl peptidase 4 (DPP4; also called CD26); that MERS-CoV has a broad tropism, replicating better in some cells lines and eliciting a more proinflammatory response than SARS-CoV; is widespread in DCs; has the potential to infect other animals and that MERS kills its human host more often than SARS did (20-40 % versus 9 % for SARS [14] ) [15] [16] [17] [18] [19] .\n",
            "\n",
            "In humans, overt disease was given the name Middle East respiratory syndrome, with the acronym MERS. From intermittent animal-to-human spill-over events, the MERS-CoV spreads sporadically among people, causing more severe disease among older adults, especially males, with pre-existing diseases. The spread of MERS-CoV among humans has often been associated with outbreaks in hospitals, with around 20 % of all cases to date involving healthcare workers (HCWs).\n",
            "\n",
            "Although DCs appear to suffer the equivalent of a 'common cold' from MERS-CoV infection, in humans, the virus can be a more serious and opportunistic pathogen associated with the death of up to 40 % of reported cases. It has yet to be established whether infections thought to have been acquired from an animal source produce a more severe outcome than those spread between humans [20] . Studies have established that the mean incubation period for MERS is five to six days, ranging from two to 16 days, with 13 to 14 days between when illness begins in one person and subsequently spreads to another [21] [22] [23] [24] . Among those with progressive illness, the median time to death is 11 to 13 days, ranging from five to 27 days [23, 24] . Fever and gastrointestinal symptoms may form a prodrome, after which symptoms decline, only to be followed by a more severe systemic and respiratory syndrome [25, 26] .\n",
            "\n",
            "The first WHO case definition [27] defined probable cases of MERS based on the presence of febrile illness, cough and requirement for hospitalization with suspicion of lower respiratory tract (LRT) involvement. It also included roles for contact with a probable or confirmed case or for travel or residence within the Arabian Peninsula. If strictly adhered to, only the severe syndrome would be subject to laboratory testing, which was the paradigm early on [21] . From July 2013, the revised WHO case definition included the importance of seeking out and understanding the role of asymptomatic cases and from June 2014, the WHO definition more clearly stated that a confirmed case included any person whose sample was RT-PCR positive for MERS-CoV, or who produced a seroconversion, irrespective of clinical signs and symptoms. [28] [29] [30] Apart from the WHO and the KSA Ministry of Health reports, asymptomatic or subclinical cases of MERS-CoV infection were documented in the scientific literature although not always as often as occurred early on [31, 32] . The KSA definition of a case became more strict on 13 th May 2014, relying on the presence of both clinical features and laboratory confirmation [33] . Testing of asymptomatic people was recommended against from December 2014 [34] , reinforced by a case definition released by the KSA Ministry of Health in June 2015 [35] . The KSA has been the source of 79 % of human cases. Severe MERS is notable for its impact among older men with comorbid diseases including diabetes mellitus, cirrhosis and various lung, renal and cardiac conditions [36] [37] [38] . Interestingly in June 2015, an outbreak in South Korea followed a similar distribution [39, 40] . Among laboratory confirmed cases, fever, cough and upper respiratory tract (URT) signs and symptoms usually occur first, followed within a week by progressive LRT distress and lymphopaenia [37] . Patients often present to a hospital with pneumonia, or worse, and secondary bacterial infections have been reported [37, 41] . Disease can progress to acute respiratory distress syndrome and multiorgan system failure [37] . MERS has reportedly killed approximately 35 % of all reported cases, 42 % of cases in the KSA, yet only 19 % of cases in South Korea, where mortality ranged from 7 % among younger age groups to 40 % among those aged 60 years and above [42] ; all may be inflated values with asymptomatic or mild infections sometimes not sought or not reported [34] . General supportive care is key to managing severe cases [43] . Children under the age of 14 years are rarely reported to be positive for MERS-CoV, comprising only 1.1 % (n = 16) of total reported cases. Between 1 st September 2012 and 2 nd December 2013, a study described the then tally of paediatric cases in the KSA, which stood at 11 (two to 16 years of age; median 13 years); nine were asymptomatic (72 %) and one infant died [44] . In Amman, Jordan, 1,005 samples from hospitalized children under the age of two years with fever and/or respiratory signs and symptoms were tested but none were positive for MERS-CoV RNA, despite being collected at a similar time to the first known outbreak of MERS-CoV in the neighbouring town of Al-Zarqa [45] . A second trimester stillbirth occurred in a pregnant woman during an acute respiratory illness and while not RT-rtPCR positive, the mother did subsequently develop antibodies to MERS-CoV, suggestive of recent infection [46] . Her exposure history to a MERS-CoV RT-rtPCR positive relative and an antibody-reactive husband, her incubation period and her symptom history met the WHO criteria for being a probable MERS-CoV case [46] .\n",
            "\n",
            "Diagnostic methods were published within days of the ProMED email announcing the first MERS case [47] , including several now gold standard in-house RT-rtPCR assays (Fig. 2 ) as well as virus culture in Vero and LLC-MK2 cells [18, 47, 48] . A colorectal adenocarcinoma (Caco-2) epithelial cell line has since been recommended for isolation of infections MERS-CoV [49] . We previously [18] .). Open reading frames are indicated as yellow rectangles bracketed by terminal untranslated regions (UTR; grey rectangles). FS-frame-shift. Predicted regions encompassing recombination break-points are indicated by orange pills. Created using Geneious v8.1 [211] and annotated using Adobe Illustrator. Beneath this is a schematic depicting the location of RT-PCR primers (blue arrows indicate direction) and oligoprobes (green rectangles) used in the earliest RT-rtPCR screening assays and conventional, semi-nested (three primers) RT-PCR confirmatory sequencing assays [47, 48] . Publication order is noted by first [27 th September 2012; red] and second [6 th December 2012; orange] coloured rectangles; both from Corman et al. [47, 48] Those assays recommended by the WHO are highlighted underneath by yellow dots [53] . The NSeq reverse primer has consistently contained one sequence mismatch with some MERS-CoV variants. An altered version of that from Mackay IM, Arden KE. Middle East respiratory syndrome: An emerging coronavirus infection tracked by the crowd. Virus Res 2015 Vol 202:60-88 with permission from Elsevier [5] reviewed the broad tropism of MERS-CoV [5] . However, as is well described, cell culture is a slow, specialised and insensitive method [50] while PCR-based techniques are the preferred method for MERS-CoV detection.\n",
            "\n",
            "The first open reading frames (ORF 1a and 1b; Fig. 2 ) have become a key diagnostic and taxonomic target for CoV species identification. With less than 80 % identity between the amino acid sequence of MERS ORF 1ab and betacoronavirus relatives, Tylonycteris bat HKU4 and Pipistrellus bat HKU5, it can be concluded that it is a novel and distinct virus. MERS-CoV is predicted to encode ten open reading frames with 5' and 3' untranslated regions [51] . The structural proteins include the spike (S), envelope (E), membrane (M) and nucleocapsid (N) [52] . The products of ORF1a and ORF1b are predicted to encode nonstructural proteins.\n",
            "\n",
            "The majority of specimen testing to date has employed validated RT-rtPCR assays shown to be sensitive and specific [47, 48, 53] . The RealStar® kit uses these WHOrecommended assays [54] . The target sequences of these screening assays have not changed among genomes examined until at least mid-2015 (IMM observation). Other RT-rtPCR assays have been developed and validated for use as laboratory-based diagnostic tools [55] [56] [57] . Additionally, loop-mediated [58, 59] or recombinase polymerase [60] isothermal assays have been designed for field deployment.\n",
            "\n",
            "The detection of MERS-CoV antigen has not been common to date but the combination of short turnaround time from test to result, high throughput and identification of viral proteins makes this an attractive option. Detection of viral proteins rather than viral RNA indicates the likely presence of infectious virus. The first rapid immunochromatographic tool described could detect recombinant MERS-CoV nucleocapsid protein from DC nasal swabs with 94 % sensitivity and 100 % specificity compared to RT-rtPCR [61] . A different approach used a monoclonal antibody-based capture ELISA targeting the MERS-CoV nucleocapsid protein with a sensitivity of 10 3 TCID 50 and 100 % specificity [62] .\n",
            "\n",
            "Demonstration of a seroconversion to a MERS-CoV infection meets the current WHO definition of a case so optimized and thoroughly validated sero-assays employed alongside good clinical histories are useful to both identify prior MERS-CoV infection and help support transmission studies. Because serology testing is, by its nature, retrospective, it is usual to detect a viral footprint, in the form of antibodies, in the absence of any signs or symptoms of disease and often in the absence of any viral RNA [63] .\n",
            "\n",
            "Strategic, widespread sero-surveys of humans using samples collected after 2012 are infrequent. Much of the Arabian Peninsula and all of the Horn of Africa lack baseline data describing the proportion of the community who may have been infected by a MERS-CoV. However, sero-surveys have had widespread use in elucidating the role of DCs as a transmission source for MERS-CoV. Because of the identity shared between DC and human MERS-CoV (see Molecular epidemiology: using genomes to understand outbreaks), serological assays for DC sero-surveys should be transferrable to human screening with minimal re-configuration. Also, no diagnostically relevant variation in neutralization activity have been found from among a range of circulating tested MERS-CoV isolates and sera, so whole virus or specific protein-based sero-assays should perform equivalently in detecting serological responses to the single MERS-CoV serotype [49] . The development of robust serological assays requires reliable panels of wellcharacterized animal or human sera, including those positive for antibodies specific to MERS-CoV, as well as to likely sources of cross-reaction [64] . Obtaining these materials was problematic and slowed the development and commercialization of antibody detection assays for human testing [64] . A number of commercial ELISA kits, immunofluorescent assays (IFA) kits, recombinant proteins and monoclonal antibodies have been released [31, [65] [66] [67] [68] . Initially, conventional IFAs were used for human sero-surveys. These relied on MERS-CoV-infected cell culture as an antigen source, detecting the presence of human anti-MERS-CoV IgG, IgM or neutralizing antibodies in human samples [18, 48, 69] . No sign of MERS-CoV antibodies was found among 2,400 sera from patients visiting Hospital in Jeddah, from 2010 through 2012, prior to the description of MERS-CoV [18] . Nor did IFA methods detect any sign of prior MERS-CoV infection among a small sample of 130 healthy blood donors from another Hospital in Jeddah (collected between Jan and Dec 2012) [70] . Of 226 slaughterhouse workers, only eight (3.5 %) were positive by IFA, and those sera could not be confirmed by virus neutralization (NT) test. The study indicated that HCoV-HKU1 was a likely source of crossreactive antigen in the whole virus IFA [70] . Whole virus MERS-CoV IFA also suffered from some cross-reactivity with convalescent SARS patient sera and this could not be resolved by an NT test which was also cross-reactive [71] . IFA using recombinant proteins instead of whole-virus IFA, has been shown to be a more specific tool [31] . Since asymptomatic zoonoses have been posited [72] , an absence of antibodies to MERS-CoV among some humans who have regular and close contact with camels may reflect the rarity of actively infected animals at butcheries, a limited transmission risk associated with slaughtering DCs [70] , a pre-existing cross-protective immune status or some other factor(s) resulting in a low risk of disease and concurrent seroconversion developing after exposure in this group. IFA using recombinant proteins instead.\n",
            "\n",
            "Some sero-assays have bypassed the risks of working with infectious virus by creating transfected cells expressing recombinant portions of the MERS-CoV nucleocapsid and spike proteins [48, 73] , or using a recombinant lentivirus expressing MERS-CoV spike protein and luciferase [74, 75] . A pseudo particle neutralization (ppNT) assay has seen widespread used in animal studies and was at least as sensitive as the traditional microneutralization (MNT) test. [10, 74, [76] [77] [78] ] Studies using small sample numbers and ppNT found no evidence of MERS-CoV neutralizing antibody in sera from 158 children with LRT infections between May 2010 and May 2011, 110 sera from 19 to 52 year old male blood donors and 300 selfidentified animal workers from the Jazan Region of the KSA during 2012 [79, 80] . Similarly, a study of four herdsmen in contact with an infected DC herd in Al-Ahsa, eight people who had intermittent contact with the herd, 30 veterinary surgeons and support staff who were not exposed to the herd, three unprotected abattoir workers in Al-Ahsa and 146 controls who were not exposed to DCs in any professional role, found none with serological evidence of past MERS-CoV infection using the ppNT assay [10] . A delay in the neutralizing antibody response to MERS-CoV infection was associated with increased disease severity in South Korea cases with most responses detectable by week three of illness while others, even though disease was severe, did not respond for four or more weeks [81] . The implications for our ability to detect any response in mild or asymptomatic cases was not explored but may be a signifcant factor in understanding exposure in the wider community.\n",
            "\n",
            "A Jordanian outbreak of acute LRT disease in a hospital in 2012 was retrospectively found to be associated with MERS-CoV infection, initially using RT-rtPCR, but subsequently, and on a larger scale, through positivity by ELISA and IFA or MNT test. [46, 82, 83] This outbreak predated the first case of MERS in the KSA. The ELISA used a recombinant nucleocapsid protein from the group 2 betacoronavirus bat-CoV HKU5 to identify antibodies against the equivalent crossreactive MERS-CoV protein [71] . It was validated using 545 sera collected from people with prior HCoV-OC43, HCoV-229E, SARS-CoV, HCoV-NL63, HRV, HMPV or influenza A(H1N1) infections but was reportedly less specific than the recombinant IFA discussed above. It was still considered an applicable tool for screening large sample numbers [82] . A protein microarray expressing the S1 protein subunit has also been validated and widely used for DC testing [5, 84] . Detection of MERS-CoV infection using ELISA or S1 subunit protein microarray [84] is usually followed by confirmatory IFA and/ or a plaque-reduction neutralization (PRNT) [69, 70, 85] or MNT test. [74, 85, 86] This confirmatory process aims toensure the antibodies detected are able to specifically neutralize the intended virus and are not more broadly reactive to other coronaviruses found in DCs (bovine CoV, BCoV) or humans (HCoV-OC43, HCoV-229E, HCoV-NL63, HCoV-HKU1, SARS-CoV). In the largest study of human sera, a tiered diagnostic process assigned both recombinant IFA and recombinant ELISA positive sera to 'stage 1' seropositivity. A stage 2 seropositive result additionally required a suitably titred PRNT result [87] . The study found 15 sera collected in 2012 to 2013 from 10,009 (0.2 %) people in 13 KSA provinces contained MERS-CoV antibodies, but significantly higher proportions in occurred in camel shepherds (two of 87; 2.3 %) and slaughterhouse workers (five of 140; 3.6 %) [87] . Contemporary surveys are needed.\n",
            "\n",
            "MERS-CoV does not appear to be easily transmitted from DCs to humans, or perhaps it is [72] , but generally does not trigger a detectable immune response if only mild disease or asymptomatic infection results. Serology assays are in need of further validation in this area so care is required when moving newly developed diagnostic serology algorithms from a research setting to one that informs public health decisions. This was reinforced when a false positive US case, purported to have been infected after a handshake and two face-to-face meetings, did not withstand further confirmatory analysis using a more specific, NT assay and was subsequently retracted [88, 89] .\n",
            "\n",
            "The WHO recommends sampling from the LRT for MERS-CoV RT-rtPCR testing, especially when sample collection is delayed by a week or more after onset of symptoms. [53] LRT samples are also best for attempting isolation of infectious virus, although the success of culture is reduced when disease persists [49] . Recommended sample types include bronchoalveolar lavage (BAL), tracheal/tracheobronchial aspirate, pleural fluid and sputum [53, 90] . Fresh samples yield better diagnostic results than refrigerated material [69] and if delays in testing of ≥72 h are likely, samples (except for blood) should be frozen at −70°C [90] . If available, lung biopsy or autopsy tissues can also be tested [53] . The URT is a less invasive and more convenient sampling site however, and an oropharyngeal and throat swab or a nasopharyngeal aspirate/wash are recommended when URT sampling is to be conducted [90] . Paired sera, collected two to three weeks apart are preferable for serological testing while a single sample is suggested to be sufficient if collected two weeks after onset of disease or a single serum collected during the first 10-12 days if conducting RT-rtPCR [53, 90] . Human urine and stool have been found to contain MERS-CoV RNA 12 to 26 days after symptom onset [25, 69, 91] and are listed as samples that should be considered [53, 90] . In two cases that arrived in the Netherlands, urine was RT-rtPCR negative but faeces was weakly positive and sera were RT-rtPCR positive for five days or more [25] . The finding of MERS-CoV viral RNA in serum provides an avenue for retrospective PCR-based studies if respiratory samples are unavailable [83] . RNAaemia may also correlate with disease severity; signs of virus were cleared from the serum of a recovered patient, yet lingered until the death of another [92] .\n",
            "\n",
            "Clinically suspected MERS cases may return negative results by RT-rtPCR. Data have shown one or more negative URT samples may be contradicted by further URT sampling or the use of LRT samples, which is preferred [2, 43, 93] . Higher viral loads occur in the LRT compared to the URT. [22, 69, 88, 94] This fits with the observation that the majority of disease symptoms are reported to manifest as systemic and LRT disease [21] . However, on occasion, even LRT specimens from MERS cases may initially be negative, only to later become positive by RT-PCR [95] . This may be due to poor sampling when a cough is absent or non-productive or because the viral load is low [95] . Despite this both the largest human MERS-CoV studies [32, [96] [97] [98] and smaller ones [22, 25, 99] , use samples from the URT. It is then noteworthy that one study reported an association between higher loads in the URT and worse clinical outcome including intensive care and death [94] . At writing, no human data exist to define whether the virus replicates solely or preferentially in the LRT or URT, or replicates in other human tissues in vivo although MERS-CoV RNA has been detected from both the URT and LRT in a macaque monkey model [100] .The distribution of DPP4 in the human upper airways is also not well described.\n",
            "\n",
            "Individual human case studies report long periods of viral shedding, sometimes intermittently and not necessarily linked to the presence of disease symptoms. [25, 69, 99, 101] In one instance, a HCW shed viral RNA for 42 days in the absence of disease [99] . It is an area of high priority to better understand whether such cases are able to infect others. Over three quarters of MERS cases shed viral RNA in their LRT specimens (tracheal aspirates and sputum) for at least 30 days, while only 30 % of contacts were still shedding RNA in their URT specimens [91, 102] .\n",
            "\n",
            "In the only study to examine the effect of sample type on molecular analysis, 64 nasopharyngeal aspirates (NPA; an URT sample), 30 tracheal aspirates, 13 sputa and three BAL were examined. The tracheal aspirates and BAL returned the highest viral load values followed by NPA and sputum. Unsurprisingly, higher viral loads generally paralleled whole genome sequencing and culture success and, in NPA testing, were significantly correlated with severe disease and death [49, 94, 103] . This study demonstrated the importance of LRT sampling for whole genome sequencing.\n",
            "\n",
            "When tested, samples positive for MERS-CoV are often negative for other pathogens [2, 25, 93, 104] . However, many studies make no mention of additional testing for endemic human respiratory viruses [21, 23, 73, 105] . When viruses are sought, they have included human herpesvirus (HHV), rhinoviruses (HRV), enteroviruses (EV), respiratory syncytial virus (RSV), parainfluenzavirus types 1, 2 and 3 (PIVs),influenzaviruses (IFVs), endemic HCoVs, adenoviruses (AdVs) metapneumovirus (MPV) and influenza A\\H1N1 virus; co-detections with MERS-CoV have been found on occasion [2, 22, 37, 69, 97] . Bacterial testing is sometimes included (for example, for Legionella and Pneumococcus) but the impact of bacterial co-presence is also unclear [22, [104] [105] [106] . Further testing of the LRT sample from the first MERS case used IFA to screen for some viruses (negative for IFV, PIVs, RSV and AdVs) and RT-PCR for others (negative for AdV, EVs, MPV and HHVs) [18] . RT-PCR also detected MERS-CoV. The WHO strongly recommends testing for other respiratory pathogens [53] but with this recommendation often discounted, there are limited data to address the occurrence and impact of co-infections or alternative viral diagnoses among both MERS cases and their contacts. Little is known of other causes of MERS-like pneumonia in the KSA or of the general burden of disease due to the known classical respiratory viruses.\n",
            "\n",
            "Testing of adult pilgrims performing the Hajj in 2012 to 2014 has not detected any MERS-CoV. In 2012, nasal swabs from 154 pilgrims collected prior to leaving for or departing from the KSA were tested [47] . In 2013, testing was significantly scaled up with 5,235 nasopharyngeal swabs from 3,210 incoming pilgrims and 2,025 swabs from outgoing pilgrims tested [98] . It should be noted that most pilgrims arrived from MERS-free countries. A further 114 swabs were taken from pilgrims with influenza-like illness [96, 107] . In earlier Hajj gatherings, it was found that influenza viruses circulated widely, whilst other viruses, often rhinoviruses, circulated more selectively, interpreted as indicating their importation along with foreign pilgrims. [107] [108] [109] Over time, increased influenza vaccination has been credited for a fall in the prevalence of influenza like illnesses among Hajj pilgrims. [110] A LRT sample is often not collected for these studies [98, 107, 109] , so false negative findings are a possibility although little is known about the initial site of MERS-CoV infection and replication; it may have been assumed it was the LRT because disease was first noticed there but the URT may be the site of the earliest replication.\n",
            "\n",
            "In Jeddah between March and July 2014 (hereafter called the Jeddah-2014 outbreak; Fig. 3 ), there was a rapid increase in MERS cases, accompanied by intense screening; approximately 5,000 samples from in and around the region were tested in a month yielding around 140 MERS-CoV detections (~3 % prevalence) [111] . Among 5,065 individuals sampled and tested across the KSA between October 2012 and September 2013,108 (2.1 %) detections were made in a hospital-centric population which included hospitalized cases (n = 2,908; 57.4 %), their families (n = 462; 9.1 %) and associated HCWs (n = 1,695; 33.5 %) [32] . Among the detections, 19 (17.8 %) were HCWs and 10 (9.3 %) were family contacts [32] .\n",
            "\n",
            "The 2-3 % prevalence of active MERS-CoV infections is not dissimilar to the hospital-based prevalence of other human CoVs. [112] However, the proportion of deaths among those infected with MERS-CoV is much higher than that known for the HCoVs NL63, HKU1, 229E or OC43 in other countries, and even above that for SARS-CoV; it is not a virus that could reasonably be described as a \"storm in a teacup\". It is the low transmission rate that has prevented worldwide spread, despite many \"opportunities\".\n",
            "\n",
            "Very early in the MERS outbreak, some animals were highly regarded as either the reservoir or intermediate host(s) of MERS-CoV with three of the first five cases having contact with DCs [73, 113, 114] . Today, animal MERS-CoV infections must be reported to the world organization for animal health as an emerging disease [115] . A summary of the first MERS cases reported by the WHO defined animal contact with humans as being direct and within 10 days prior to symptom onset [20] . This definition made no specific allowance for acquisition from DCs through a droplet-based route, which is very likely route for acquisition of a virus that initially and predominantly causes respiratory disease [23] . Camels are known to produce high levels of MERS-CoV RNA in their URT and lungs [116] . Providing support for a droplet transmission route and perhaps indicating the presence of RNA in smaller, drier droplet nuclei, MERS-CoV RNA was identified in a high volume air sample collected from a barn housing an infected DC [117] . The precise source from which humans acquire MERS-CoV remains poorly studied but it seems likely that animal and human behavioural factors may play roles (Fig. 3) [118] . These factors may prove important for human cases who do not describe any DC contact [119] nor any contact with a confirmed case. Whether the WHO definition of animal contact is sufficient to identify exposure to this respiratory virus remains unclear. Wording focuses on consumption of DC products but does not specifically ascribe risk to a droplet route for acquisition of MERS-CoV from DC [120] . Some MERS patients are listed in WHO disease notices as being in proximity to DCs or farms, but the individuals have not described coming into contact with the animals. No alternative path for acquiring infection is reported in many of these instances. What constitutes a definition of \"contact\" during these interviews has been defined for one study [72] . Despite this lack of clarity, the WHO consider that evidence linking MERS-CoV transmission between DCs to humans is irrefutable (Fig. 4) [120] .\n",
            "\n",
            "The possibility that bats were an animal host of MERS-CoV was initially widely discussed because of the existing diversity of coronaviruses known to reside among them [121] [122] [123] [124] . Conclusive evidence supporting bats as a source for human infections by MERS-CoV has yet to be found, but bats do appear to host ancestral representatives [53, 125] . However, these are not variants of the same virus nor always within the same phylogenetic lineage as MERS-CoV; they are each a genetically distinct virus. Bat-to-human infection by MERS-CoV is a purely speculative event. The only piece of MERS-CoV-specific evidence pointing to bats originates from amplification of a 190 nt fragment of the RNAdependent RNA polymerase gene of the MERS-CoV genome, identified in a faecal pellet from an insectivorous Emballonuridae bat, Taphozous perforatus found in Bisha, the KSA [121] . While very short, the sequence of the fragment defined it as a diagnostic discovery. Subsequently a link to DCs was reported [85] and that link has matured into a verified association [38, 126] (Fig. 4) .\n",
            "\n",
            "(See figure on previous page.) Fig. 3 Monthly detections of MERS-CoV (blue bars) and of cases who died (red bars) with some dates of interest marked for 2012 to 4 th September 2015. An approximation of when DC calving season [128] and when recently born DCs are weaned is indicated. Spring (green) and summer (orange) in the Arabian Peninsula are also shaded. Note the left-hand y-axis scale for 2014 and 2015 which is greater than for 2012/13. Sources of these public data include the WHO, Ministries of Health and FluTrackers [207] [208] [209] . Earlier and subsequent versions of this chart are maintained on a personal blog [210] . Modified and reprinted from Mackay IM, Arden KE. Middle East respiratory syndrome: An emerging coronavirus infection tracked by the crowd. Virus Res 2015 Vol 202:60-88 with permission from Elsevier [5] DCs, which make up 95 % of all camels, have a central presence in the Arabian Peninsula where human-DC contact ranges from little to close [119] . Contact may be commonplace and could occur in variety of ways (Fig. 4a) . There are several large well-attended festivals, races, sales and parades which feature DCs and DCs are also kept and bred close to populated areas in the KSA [127, 128] . DC milk and meat are widely consumed and the older DC is an animal of ritual significance after the Hajj pilgrimage [129] . However, MERS-CoV infection frequency is reportedly much lower than is the widespread and frequent habit of eating, drinking and preparing DC products. Daily ingestion of fresh unpasteurized DC milk is common among the desert Bedouin and many others in the KSA. DC urine is also consumed or used for supposed health benefits. Despite camel butchery being a local occupation, neither butchers nor other at-risk groups are identifiable among MERS cases; this may simply be a reporting issue rather than an unexplainable absence of MERS. A small case-control study published in 2015 identified direct DC contact, and not ingestion of products, to be associated with onset of MERS [38] .\n",
            "\n",
            "The first sero-survey of livestock living in the Middle East region was conducted during 2012-2013 [85] . DCs were sampled from a mostly Canary Island-born herd and from Omani DCs (originally imported from the Horn of Africa) [85] . A neutralising antibody assay found only 10 % of strongly seropositive Canary Island [5] . b Camel-to-human infections appear to be infrequent, while human-to-human spread of infection is regularly facilitated by poor IPC in healthcare settings where transmission is amplified, accounting for the bulk of cases. There are human MERS cases that do not fall into either category of source and it is unclear if these acquired infection through some entirely separate route, or from cases that escaped diagnosis. c Hypothetical ways in which subclinical (when infection may not meet a previously defined clinical threshold of signs and/or symptoms) or asymptomatic (no obvious signs or measured, noticed or recalled symptoms of illness) MERS-CoV infection may be implicated in transmission DC sera could neutralise MERS-CoV while all Omani DC sera had high levels of specific MERS-CoV neutralizing antibody [85] . This indicated that DCs had in the past been infected by MERS-CoV, or a very similar virus.\n",
            "\n",
            "Since this study, a host of peer-reviewed reports have looked at both DCs and other animals, and the possibility that they may host MERS-CoV infection. Seropositive DCs have been found throughout the Arabian Peninsula including Oman, the KSA, Qatar, Jordan, the United Arab Emirates (UAE), Kuwait as well as Sudan, Somalia, Egypt, Tunisia, Nigeria, Kenya and Ethiopia in Africa and the Canary Islands [85, [130] [131] [132] [133] [134] . Other animals tested include sheep, cows, pigs, horses, donkeys, mules, birds, water buffalo, goats, Bactrian camels, llamas and guanaco (south American camelids) but none had detectable neutralising antibody against MERS-CoV [4, 74, 78, 85, 86, 135, 136] . No virology or serology studies of human samples from areas in Africa where there are camels with a history of MERS-CoV have been reported to date. However,an absence of unexplained pneumonia that may be attributable to MERS-CoV infection may not signal the absence of virus among humans in each country but simply reflect a lack of expensive epidemiology studies conducted by resource-poor countries. It is thus unclear whether MERS-CoV, or an antigenically related CoV, is an unrecognized pathogen in these regions, perhaps circulating for even longer than it has been known in the Arabian Peninsula [133] .\n",
            "\n",
            "MERS-CoV RNA has also been detected in DC samples, and recovery of infectious virus has also been achieved from DC samples [4, 77, 117, 132, [137] [138] [139] [140] [141] . From some of these, full or majority length genomes of MERS-CoV have been sequenced [77, 137, 138] . DC versions of MERS-CoV were found to be as similar to each other, as were variants detected from different humans over time and across distance.\n",
            "\n",
            "Antibody screening assays have also detected crossreactive antibodies in sera. These were identified as such by screening sera against similar viruses, for example BCoV or HCoV-OC43 (as an antigenic facsimile for BCoV). It is possible that other MERS-CoV-like viruses also reside within DCs, but this does not detract from the definitive finding of MERS-CoV genetic sequences in both DCs and humans [117, 142, 143] .\n",
            "\n",
            "Screening studies have shown that juvenile DCs are more often positive for virus or viral RNA while older DCs are more likely to be seropositive and RNA or virus negative [76, 77, 144] . In adult DCs, MERS-CoV RNA has been detected among animals with pre-existing antibody, suggesting re-infection is possible [77, 144] . Viral loads among positive DCs can be very high [4, 76, 77, 139, 144] and DCs have been found positive both when ill with URT respiratory signs [77, 117, 142, 145] or when apparently healthy [137] . These findings indicate DCs host natural MERS-CoV infections. Furthermore, stored DC sera have revealed signs of MERS-CoV in DCs which date back over three decades (the earliest collected in 1983) [4, 133, 135] . Older sera have not been tested and so precisely how long DCs have been afflicted by MERS-CoV, whether the virus is enzootic among them, introduced to them decades or centuries ago from bats in Africa or the Arabian Peninsula, or they are the subject of regular but short-lived viral incursions from an as yet unknown host, cannot be answered.\n",
            "\n",
            "Researchers sought to determine a direction for infection; were DCs transmitting virus to humans or were humans infecting DCs? At a Qatari site, a farm owner and his employee became ill in mid-October 2013 and tested positive for MERS-CoV RNA in a sputum and throat swab sample, respectively. RT-rtPCRs found MERS-CoV RNA in 11 of 14 positive DC nasal swabs at the farm; six (43 %) positive by two or more assays [138] . The results indicated a recent outbreak had occurred in this herd; the first indication of MERS-CoV RNA found within DCs with a temporal association to human infections. Three positive DC samples were confirmed by sequencing a 358 nt portion of the spike gene; these sequences were identical to each other, again with close homology to other human and DC MERS-CoV sequences [138] . The DCs and human contacts yielded ORF1a and ORF4b sequences differing by only a single nucleotide each, clustering closely with the Hafr-Al-Batin_1_2013 variant [138] . Subsequent case studies found evidence of a concurrent human and DC infection and the direction of that infection was inferred to be from the ill DCs and to their human owners [117, 142, 146] . Partial genome sequences indicated that a human and a MERS-CoV RT-rtPCR positive DC had been infected by a variant of the same virus, harbouring the same distinct pattern of nucleotide polymorphisms. [142] All nine DC in the owner's herd, serially sampled, reacted in a recombinant S1 antigen ELISA, with the two animals that had been RT-rtPCR positive showing a small, verifiable rise in antibody titre [142] . A rise in titre theoretically begins 10 to 21 days after DC infection [142] . The authors suggested that the rise in titre in DC sera which occurred alongside a declining RNA load, while the patient was actively ill and hospitalized, indicated that the DCs were infected first followed by the owner [117, 142] . BCoV antibodies were also present, and rising in one of the two RT-rtPCR positive animals but no animal's antibodies could neutralise BCoV infection [142] .\n",
            "\n",
            "Camel calving season occurs in the winter months (between late October and late February; Fig. 3 ) and this may be a time when there is increased risk to humans of spill-over due to new infections among naïve DC populations [128] . What role maternal camel antibody might play in delaying infection of calves remains unknown [128, 142] . Juvenile DCs appear to host active infection more often than adult DCs and thus the sacrificial slaughter of DCs, which must be five years of age or older (termed a thane), may not be accompanied by significant risk of exposure to infection. In contrast to earlier results, slaughterhouse workers who kill both younger and older DCs, may be an occupational group with significantly higher incidence of seropositivity to MERS-CoV when animals have active MERS-CoV infections [129, 139, [147] [148] [149] . Expanded virological investigations of African DCs may lead to more seropositive animals and geographic areas in which humans may be at risk. It is possible that there are areas where humans already harbour MERS-CoV infections that have not been identified because of an absence of laboratory surveillance. Virological investigations of bats may lead to findings of ancestral viruses and viral 'missing links' and identifying any other animal sources of zoonotic spread is important to inform options for reducing human exposures [56, 76] .\n",
            "\n",
            "Infectious MERS-CoV added to DC, goat or cow milk and stored at 4°C could be recovered at least 72 h later and, if stored at 22°C, recovery was possible for up to 48 h [150] . MERS-CoV titre decreased somewhat when recovered from milk at 22°C but pasteurization completely ablated MERS-CoV infectivity [150] . In a subsequent study, MERS-CoV RNA was identified in the milk, nasal secretion and faeces of DCs from Qatar [151] .\n",
            "\n",
            "A single study has examined the ability of MERS-CoV to survive in the environment [150] . Plastic or steel surfaces were inoculated with 10 6 TCID 50 of MERS-CoV at different temperature and relative humidity (RH) and virus recovery was attempted in cell culture. At high ambient temperature (30°C) and low RH (30 %) MERS-CoV remained viable for 24 h [150] . By comparison, a well known and efficently transmitted respiratory virus, influenza A virus, could not be recovered in culture beyond four hours under any conditions [150] . Aerosol experiments found MERS-CoV viability only decreased 7 % at low RH at 20°C. In comparison, influenza A virus decreased by 95 % [150] . MERS-CoV survival is inferior to that previously demonstrated for SARS-CoV [152] . For context, pathogenic bacteria can remain viable and airborne for 45 min in a coughed aerosol and can spread 4 m. MERS-CoV's ability to remain viable over long time periods gives it the capacity to thoroughly contaminate a room's surfaces when occupied by an infected and symptomatic patient [153] . Whether MERS-CoV can remain adrift and infectious for extended periods (truly airborne) remains unknown. Such findings expand our understanding of the possibilities for droplets to transmit respiratory viruses in many settings, including hospital waiting rooms, emergency departments, treatment rooms, open intensive care facilities and private patient rooms. The nature and quality of air exchange, circulation and filtration are important variables in risk measurement and reduction as is the use of negative pressure rooms to contain known cases. Droplet spread between humans is considered the mechanism of human-to-human transmission and the need for droplet precautions was emphasized after the Al-Ahsa hospital, the KSA and the South Korean outbreaks [21, 23, 154, 155] . By extrapolation, aerosol-generating events involving DCs (urination, defecation, and preparation and consumption of DC products) should be factored into risk measurement and reduction efforts and messaged using appropriate context. The provision of evidence supporting the best formulation of personal protective equipment to be worn by HCWs who receive, manage or conduct procedures on infectious cases remains a priority.\n",
            "\n",
            "MERS-CoV was found and characterized because of its apparent association with severe, and therefore more obvious, illness in humans; we were the canaries in the coal mine. Sero-assays and prospective cohort studies have yet to determine the extent to which milder or asymptomatic cases contribute to MERS-CoV transmission chains. However, transmission of MERS-CoV is defined as sporadic (not sustained), intra-familial, often healthcare associated, inefficient and requiring close and prolonged contact [22, 31, 63, 93, 97, 102, 156] In a household study, 14 of 280 (5 %) contacts of 26 MERS-CoV positive index patients were RNA or antibody positive; the rate of general transmission, even in outbreaks is around 3 % [31] . It seems that the majority of human cases of MERS-CoV, even when numbers appear to increase suddenly, do not readily transmit to more than one other human so to date, the localized epidemic of MERS-CoV has not been self-sustaining [157] [158] [159] [160] [161] . That is to say, the basic reproduction number (R 0 ) -the average number of infections caused by one infected individual in a fully susceptible populationhas been close to one throughout various clusters and outbreaks. If R 0 was greater than 1, a sustained increase in case numbers would be expected. Some R o calculations may be affected by incomplete case contact tracing, limited community testing and how a case is defined. That MERS has had a constant presence in the Arabian Peninsula since 2012 is due to ongoing, sporadic spill-over events from DCs amplified by poorly controlled hospital outbreaks.\n",
            "\n",
            "The first known MERS human-to-human transmission event was one characterized by acute LRT disease in a healthcare setting in Jordan. In stark contrast, a sero-survey of HCW who were sometimes in close and prolonged contact with the first, fatal MERS-CoV case in 2012 [162] , found none of the HCW had seroconverted four months later, despite an absence of eye protection and variable compliance with required PPE standards [162] .\n",
            "\n",
            "Early on in the MERS story, samples for testing were mostly collected from patients with severe illness and not those with milder acute respiratory tract infections. Contacts of confirmed MERS cases were often observed for clinical illness, but not tested. These omissions may have confounded our understanding of MERS-CoV transmission and biased early data towards higher numbers of seriously ill and hospitalized patients, inflating the apparent proportion of fatal cases. Case-control studies were not a focus. As testing paradigms changed and contacts were increasingly tested, more asymptomatic and mild infections were recognized [163] .\n",
            "\n",
            "A rise in the cases termed asymptomatic (which enlarge the denominator for calculations of the proportion of fatal cases, defined in [164] ) resulted in a drop in the proportion of fatal cases during the Jeddah-2014 outbreak. Historically, such rises are consistent with changing definitions and laboratory responses and clinical management of a newly discovered virus infection that was first noted only among the severely ill. Upon follow-up, over three-quarters of such MERS-CoV RNA positive people did recall having one or more symptoms at the time, despite being reported as asymptomatic [165] raising some question over the reliability of other reported data.\n",
            "\n",
            "The proportion of fatal MERS cases within the KSA compared to outside the KSA, as well as the age, and sex distribution change in different ways when comparing MERS outbreaks. Approximately 43 % of MERS cases (549 of 1277) in the KSA were fatal betwen 2012 and December 2015 while 21 % (72 of 330) died among those occurring outside of the KSA. The total number of male cases always outnumber females and the proportion of male deaths is always greater than the proportion of females who die. However the proportion of male deaths from total males with MERS is a similar figure to that for females. In the KSA, there is a greater proportion of younger males among cases and deaths than were observed from the 2015 South Korean or the Jeddah-2014 outbreaks (Additional file 2: Figure S2 ). Why these aspects have differed may be due to differences in the time to presentation and diagnosis, the nature and quality of supportive care, the way a person became infected (habits, exposure to a human or zoonotic source, viral load, route of infection) or the extent to which different populations are burdened by underlying diseases [40] .\n",
            "\n",
            "As a group, HCWs comprised 16 % of MERS cases in the KSA and South Korea. It is apparent that the weekly proportion of infected HCWs increases alongside each steep rise in overall detections (Fig. 5) . In May 2013, the WHO published guidelines for IPC during care of probable or confirmed cases of MERS-CoV infection in a healthcare setting [166] . This is explainable because to date, each case rise has been intimately associated with healthcare-facility related outbreaks [118] . These rises in MERS-CoV detections can decrease the average age during each event because HCWs are usually younger than inpatients with MERS. Healthcare facilities have been a regular target for suggested improvements aimed at improving infection prevention and control (IPC) procedures [115, 118] .\n",
            "\n",
            "Most of the analysis of MERS-CoV genetics has been performed using high throughput or \"deep\" sequencing methods for complete genome deduction [167] [168] [169] . MERS-CoV was the first subject of such widespread use of deep sequencing to study an emerging viral outbreak with global reach. The technique can produce genomic [207] [208] [209] . Earlier and subsequent versions of this chart are maintained on a personal blog [210] length coverage in a single experiment with highly repetitious measurement of each nucleotide position [52, 140] . Despite assays having been published early on, subgenomic sequencing, once the mainstay of viral outbreak studies, has less often been published during MERS-CoV characterization [48] . As more genomes from both humans and DCs have been characterized, two clades have become apparent; A and B (Fig. 6) . Clade A contains only human-derived MERS-CoV genomes from Jordan, while Clade B comprises the majority of human and camel genomes deduced thus far [168] .\n",
            "\n",
            "Two studies during 2015, one looking at Jeddah-2014 MERS-CoV variants and another looking at a variant exported from South Korea to China, have now identified signs of genetic recombination among MERS-CoV variants. While human and camel whole genome sequences have retained >99 % identity with each other, members of genetically distinct lineages can and do swap genetic material when suitable conditions and coinfections co-occur [170] [171] [172] . Shared identity implies that the major source for human acquisition is the DC, rather than another animal, although more testing of other animal species is needed to confirm that conclusion. Over a month, a DC virus sequenced on different occasions did not change at all indicating a degree of genomic stability in its host, supporting that DCs are the natural, rather than intermediate, host for the MERS-CoV we know today [77] . To date, recombination has been localised to breakpoints near the boundary between ORF1a and ORF1b regions, within the spike gene [170] and in the ORF1b region (Fig. 2) [172] . It is not unexpected that recombination should occur since it is well known among other CoVs [124] and because the majority of MERS-CoV whole genomes collected from samples spanning three years (2012-2015) and from humans, camels and different countries have shown close genetic identity to each other, with just enough subtle variation to support outbreak investigations so long as whole genome sequencing is applied [52, 77, 135, 138, 168, [173] [174] [175] .\n",
            "\n",
            "Changes in genome sequence may herald alterations to virus transmissibility, replication, persistence, lethality or response to future drugs. If we have prior knowledge of the impact of genetic changes because of thorough characterization studies, we can closely Fig. 6 The genetic relationship between MERS-CoV nucleotide sequences (downloaded from GenBank using the listed accession numbers and from virological.org [212] ). This neighbour joining tree was created in MEGA v6 using an alignment of human and DCderived MERS-CoV sequences (Geneious v8.1 [211] ). Clades are indicated next to dark (Clade A) or pale (Clade B) blue vertical bars. Camel icons denote genomes from DCs. Healthcare or community outbreaks are boxed and labelled using previously described schemes [212, 213] monitor the genomic regions and better understand any changes in transmission or disease patterns as they occur. Genetic mutations noted during the largest of human outbreaks, Jeddah-2014, did not impart any major replicative or immunomodulatory changes when compared to earlier viral variants in vitro [156, 176] . However, we understand very little of the phenotypic outcomes that result from subtle genetic change in MERS-CoV genomes. To date no clinical relevance or obvious in vivo changes to viral replication, shedding or transmission has been reported or attributed to mutations or to new recombinant viruses [156] . But vigilance and larger, more contemporary and in vivo studies are needed.\n",
            "\n",
            "Genome sequence located to a distinct clade were identified from an Egyptian DC that was probably imported from Sudan. This does not fit into either of the current clades [125, 168, 177] . A virus sequenced from a Neoromicia capensis bat was more closely related to MERS-CoV than other large bat-derived sequences had been to that point, but the genome of a variant of a MERS-CoV has yet to be discovered and deduced from any bat [125] .\n",
            "\n",
            "Analyses of MERS-CoV genomes have shown that most single nucleotide differences among variants were located in the last third of the genome (Fig. 2) , which encodes the spike protein and accessory proteins [168] . At least nine MERS-CoV genomes contained amino acid substitutions in the receptor binding domain (RBD) of the spike protein and codons 158 (N-terminal region), 460 (RBD), 1020 (in heptad repeat 1), 1202 and 1208 bear investigation as markers of adaptive change [140, 169] . The spike protein had not changed in the recombinant MERS-CoV genome identified in China in 2015 but was reported to have varied at a higher rate than that for complete MERS-CoV genomes, among South Korean variants [172, 178] . This highlights that subgenomic regions may not always contain enough genetic diversity to prove useful for differentiating viral variants. Despite this, one assay amplifying a 615 nucleotide fragment of the spike S2 domain gene for Sanger sequencing agreed with the results generated by the sequencing of a some full genomes and was useful to define additional sequence groupings [177] .\n",
            "\n",
            "Genomic sequence can also be used to define the geographic boundaries of a cluster or outbreak and monitor its progress, based on the similarity of the variants found among infected humans and animals when occurring together, or between different sites and times (Fig. 6 ) [169] . This approach was employed when defining the geographically constrained MERS hospital outbreak in Al-Ahsa, which occurred between 1 st April and 23 rd May 2013, as well as clusters in Buraidah and a community outbreak in Hafr Al-Batin, the KSA. Genomic sequencing identified that approximately 12 MERS-CoV detections from a community outbreak in Hafr Al-Batin between June and August 2013 may have been triggered by an index case becoming infected through DC contact [175] . Sequencing MERS-CoV genomes from the 2013 Al-Ahsa hospital outbreak indicated that multiple viral variants contributed to the cases but that most were similar enough to each other to be consistent with human-tohuman transmission. Molecular epidemiology has revealed otherwise hidden links in transmission chains encompassing a period of up to five months [179] . However, most outbreaks have not continued for longer than two to three months and so opportunities for the virus to adapt further to humans through co-infection and sustained serial passage have been rare [169] . In Riyadh-2014, genetic evidence supported the likelihood of multiple external introductions of virus, implicating a range of healthcare facilities in an event that otherwise looked contiguous [23, 168, 179] . Riyadh is a nexus for camel and human travel and has had more MERS cases than any other region of the KSA to date but also harbours a wide range of MERS-CoV variants [128, 167, 179] . However the South Korean outbreak originated from a single infected person, resulting in three to four generations of cases [180, 181] . Studies of this apparently recombinant viral variant did not find an increased evolutionary rate and no sign of virus adaptation thus the outbreak seems to have been driven by circumstance rather than circumstance together with mutation [181] .\n",
            "\n",
            "For many MERS cases detected outside the Arabian Peninsula, extensive contact tracing has been performed and the results described in detail. Contact tracing is essential to contain the emergence and transmission of a new virus and today it is supported by molecular epidemiology. Although it is an expensive and time consuming process, contact tracing can identify potential new infections and through active or passive monitoring, react more rapidly if disease does develop. Results of contact tracing to date have found that onward transmission among humans is an infrequent event. For example, there were 83 contacts, both symptomatic and asymptomatic, of a case treated in Germany who travelled from the UAE but no sign of virus or antibody were found in any of them [73] . The very first MERS case had made contact with 56 HCWs and 48 others, but none developed any indication of infection [162] . In a study of 123 contacts of a case treated in France, only seven matched the definition for a possible case and were tested; one who had shared a 20 m 2 hospital room while in a bed 1.5 m away from the index case for a prolonged period was positive [26] . None of the contacts of the first two MERS cases imported into the USA in 2014 contained any MERS-CoV footprint [182] and none of the 131 contacts of two travellers returning to the Netherlands developed MERS-CoV antibodies or tested RNA positive [25, 183] . Analyses of public data reveal many likely instances of nosocomial acquisition of infection in the Arabian Peninsula and these data may be accompanied by some details noting contact with a known case or facility. One example identified the likely role of a patient with a subclinical infection, present in a hospital during their admission for other reasons, as the likeliest index case triggering a family cluster [93] . Contact tracing was a significant factor in the termination of a 2015 outbreak involving multiple South Korean hospitals [184] . Such studies demonstrate the necessity of finding and understanding a role for mild and asymptomatic cases, together with restricting close contact or prolonged exposure of infected people to others, especially older family members and friends with underlying disease (Fig. 4c) .\n",
            "\n",
            "The hospital-associated outbreak in Jeddah in 2014 was the largest and most rapid accumulation of MERS-CoV detections to date. The greatest number of MERS-CoV detections of any month on record occurred in Jeddah in April. The outbreak was mostly (>60 % of cases) associated with human-to-human spread within hospital environments and resulted from a lack of, or breakdown in, infection prevention and control [37, 185, 186] . A rise in fatalities followed the rapid increase in case numbers.\n",
            "\n",
            "In 2015 two large outbreaks occurred. South Korea was the site of the first large scale outbreak outside the Arabian Peninsula and produced the first cases in both South Korea and China, occurring between May and July 2015. This was closely followed by a distinct outbreak in Ar Riyad province in the KSA which appeared to come under control in early November.\n",
            "\n",
            "After staying in Bahrain for two weeks, a 68 year old male (68 M) travelled home to South Korea via Qatar, arriving free of symptoms on the 4 th May 2015 [187] . He developed fever, myalgia and a cough nearly a week later (11 th ). He visited a clinic as an outpatient between the 12 th and 15 th of May and was admitted to Hospital A on the 15 th [188] . He was discharged from Hospital A on the 17 th then visited and was admitted to the emergency department of Hospital B on the 18 th . During this second stay, a sputum sample was taken and tested positive for MERS-CoV on the 20 th [187, 188] , triggering transfer to the designated isolation treatment facility. Over a period of 10 days, the index case was seen at three different hospitals, demonstrating a key feature of \"hospital shopping\" that shaped the South Korean outbreak. Approximately 34 people were infected during this time [187] . In total 186 cases were generated in this outbreak, all linked through a single transmission chain to 68 M; 37 cases died [189] . In South Korea, the national health insurance system provides for relatively low cost medical care, defraying some costs by making family members responsible for a portion of the ministration of the sick, resulting in them sometimes staying for long periods in the rooms that often have more than four beds in them [24] . Other factors thought to have enabled this outbreak included unfamiliarity of local clinicians with MERS, ease with which the public can visit and be treated by tertiary hospitals, the custom of visiting sick friends and relatives in hospitals, the hierarchical nature of Korean society, crowded emergency rooms, poor IPC measures, a lack of negative pressure isolation rooms and poor inter-hospital communication of patient disease histories [24, [190] [191] [192] . All of the reported transmission occurred across three or four generations and apart from one unknown source, were all hospital-acquired [24, 120, 181, [193] [194] [195] . Few clinical details about these cases have been reported to date and detail on transmission and contact tracing is minimal. The hospitals involved were initially not identified, governmental guidance and actions produced confusing messages and there was very limited communication at all early on which resulted in unnecessary concern, distrust and a distinct economic impact [191, [196] [197] [198] . Early in the outbreak, a infected traveller, the son of an identified case in South Korea, passed through Hong Kong on his way to China where he was located, isolated and cared for in China [91, 199, 200] . No contacts became ill. The outbreak was brought under control in late July/ early August [201] after improved IPC measures were employed, strong contact tracing monitoring and quarantine, expanded laboratory testing, hospitals were better secured, specialized personnel were dispatched to manage cases and international cooperation increased [202, 203] . A review of public data showed that, as for MERS in the KSA, older age and the presence of underlying disease were significantly associated with a fatal outcome in South Korea. [40] Even though R 0 is <1, super-spreading events facilitated by circumstances created in healthcare settings and characterized by cluster sizes over 150, such as this one, are not unexpected from MERS-CoV infection [204] . The dynamic of an outbreak depends on the R 0 and an individual's viral shedding patterns, contact type and frequency, hospital procedures and population structure and density [204] .\n",
            "\n",
            "In the region of Ar Riyad, including the capital city of Riyadh, a hospital based cluster began, within a single hospital, from late June 2015 [205] . By mid-September there had been approximately170 cases reported but the outbreak appeared to been brought under control in November.\n",
            "\n",
            "It became apparent early on that MERS-CoV spread relatively ineffectively from human-to-human. Despite ongoing and possibly seasonal introduction of virus to the human population via infected DCs and perhaps other animals yet to be identified, the vast majority of MERS-CoV transmission has occurred from infected to uninfected humans in close and prolonged contact through circumstances created by poor infection control in health care settings. This opportunistic virus has had its greatest impact on those with underlying diseases and such vulnerable people, sometimes suffering multiple comorbidities, have been most often associated with hospitals, creating a perfect storm of exposure, transmission and mortality. It remains unclear if this group are uniquely affected by MERS-CoV or if other respiratory virus infections, including those from HCoVs, produce a similarly serious impact. In South Korea, a single imported case created an outbreak of 185 cases and 36 deaths that had a disproportionate impact on economic performance, community behaviour and trust in government and the health care system. Household human-to human transmission occurs but is also limited. Educational programs will be essential tools for combatting the spread of MERS-CoV both within urban and regional communities and for the health care setting.\n",
            "\n",
            "Vigilance remains important for containment since MERS-CoV is a virus with a genetic makeup that has been observed for only three years and is not stable. Among all humans reported to be infected, nearly 40 % have died. Continued laboratory testing, sequencing, analysis, timely data sharing and clear communication are essential for such vigilance to be effective. Global alignment of case definitions would further aid accurate calculation of a case fatality ratio by including subclinical case numbers. Whole genome sequencing has been used extensively to study MERS-CoV travel and variation and although it remains a tool for experts, it appears to be the best tool for the job.\n",
            "\n",
            "MERS and SARS have some clinical similarities but they also diverge significantly [206] . Defining characteristics include the higher PFC among MERS cases (above 50 % in 2013 and currently at 30-40 %; well above the 9 % of SARS) and the higher association between fatal MERS and older males with underlying comorbidities. For the viruses, MERS-CoV has a broader tropism, grows more rapidly in vitro, more rapidly induces cytopathogenic change, triggers distinct transcriptional responses, makes use of a different receptor, induces a more proinflammatory state and has a delayed innate antiviral response compared to SARS-CoV.\n",
            "\n",
            "There appears to be a 2-3 % prevalence of MERS-CoV in the KSA with a 5 % chance of secondary transmission within the household. There is an increased risk of infection through certain occupations at certain times and a much greater chance for spread to other humans during circumstances created by humans, which drives more effective transmission than any R 0 would predict on face value. Nonetheless, despite multiple mass gatherings that have afforded the virus many millions of opportunities to spread, there have remarkably been no reported outbreaks of MERS or MERS-CoV during or immediately after these events. There is no evidence that MERS-CoV is a virus of pandemic concern. Nonetheless, hospital settings continue to describe MERS cases and outbreaks in the Arabian Peninsula. As long as we facilitate the spread of MERS-CoV among our most vulnerable populations, the world must remain on alert for cases which may be exported more frequently when a host country with infected camel reservoirs is experiencing human clusters or outbreaks.\n",
            "\n",
            "The MERS-CoV appears to be an enzootic virus infecting the DC URT with evidence of recent genetic recombination. It may once have had its origins among bats, but evidence is lacking and the relevance of that to today's ongoing epidemic is academic. Thanks to quick action, the sensitive and rapid molecular diagnostic tools required to achieve rapid and sensitive detection goal have been in place and made widely available since the virus was reported in 2012. RT-PCR testing of LRT samples remains the gold standard for MERS-CoV confirmation. Serological tools continue to emerge but they are in need of further validation using samples from mild and asymptomatic infections and a densely sampled cohort study to follow contacts of new cases may address this need. Similarly, the important question of whether those who do shed MERS-CoV RNA for extended periods are infectious while appearing well, continues to go unanswered. It is even unclear just how many 'asymptomatic' infections have been described and reported correctly which in turn raises questions about the reliability of other clinical data collection to date. While the basic virology of MERS-CoV has advanced over the course of the past three years, understanding what is happening in, and the interplay between, camel, environment and human is still in its infancy.\n",
            "\n",
            "Additional file 1: Figure S1 . The\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.7/dist-packages/numpy/core/_asarray.py:83: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray\n",
            "  return array(a, dtype, copy=False, order=order)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{'score': 0.02347426302731037, 'start': 70843, 'end': 70846, 'answer': 'The'}\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "P_XC0k7IdqSc"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "bX9P4c5Vduk2"
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    }
  ]
}